{
 "cells": [
  {
   "cell_type": "code",
   "execution_count": 1,
   "metadata": {},
   "outputs": [
    {
     "name": "stdout",
     "output_type": "stream",
     "text": [
      "TODAY'S DATE\n",
      "Thu Jan 25 08:57:57 PST 2018\n",
      "------------\n",
      "\n",
      "Distributor ID:\tUbuntu\n",
      "Description:\tUbuntu 16.04.3 LTS\n",
      "Release:\t16.04\n",
      "Codename:\txenial\n",
      "\n",
      "------------\n",
      "HOSTNAME: \n",
      "roadrunner\n",
      "\n",
      "------------\n",
      "Computer Specs:\n",
      "\n",
      "Architecture:          x86_64\n",
      "CPU op-mode(s):        32-bit, 64-bit\n",
      "Byte Order:            Little Endian\n",
      "CPU(s):                16\n",
      "On-line CPU(s) list:   0-15\n",
      "Thread(s) per core:    2\n",
      "Core(s) per socket:    4\n",
      "Socket(s):             2\n",
      "NUMA node(s):          1\n",
      "Vendor ID:             GenuineIntel\n",
      "CPU family:            6\n",
      "Model:                 26\n",
      "Model name:            Intel(R) Xeon(R) CPU           E5520  @ 2.27GHz\n",
      "Stepping:              5\n",
      "CPU MHz:               1596.000\n",
      "CPU max MHz:           2394.0000\n",
      "CPU min MHz:           1596.0000\n",
      "BogoMIPS:              4521.75\n",
      "Virtualization:        VT-x\n",
      "L1d cache:             32K\n",
      "L1i cache:             32K\n",
      "L2 cache:              256K\n",
      "L3 cache:              8192K\n",
      "NUMA node0 CPU(s):     0-15\n",
      "Flags:                 fpu vme de pse tsc msr pae mce cx8 apic sep mtrr pge mca cmov pat pse36 clflush dts acpi mmx fxsr sse sse2 ss ht tm pbe syscall nx rdtscp lm constant_tsc arch_perfmon pebs bts nopl xtopology nonstop_tsc aperfmperf pni dtes64 monitor ds_cpl vmx est tm2 ssse3 cx16 xtpr pdcm dca sse4_1 sse4_2 popcnt lahf_lm kaiser tpr_shadow vnmi flexpriority ept vpid dtherm ida\n",
      "\n",
      "------------\n",
      "\n",
      "Memory Specs\n",
      "\n",
      "              total        used        free      shared  buff/cache   available\n",
      "Mem:            47G        556M         18G         77M         28G         46G\n",
      "Swap:           47G          0B         47G\n"
     ]
    },
    {
     "name": "stderr",
     "output_type": "stream",
     "text": [
      "No LSB modules are available.\n"
     ]
    }
   ],
   "source": [
    "%%bash\n",
    "echo \"TODAY'S DATE\"\n",
    "date\n",
    "echo \"------------\"\n",
    "echo \"\"\n",
    "lsb_release -a\n",
    "echo \"\"\n",
    "echo \"------------\"\n",
    "echo \"HOSTNAME: \"\n",
    "hostname\n",
    "echo \"\"\n",
    "echo \"------------\"\n",
    "echo \"Computer Specs:\"\n",
    "echo \"\"\n",
    "lscpu\n",
    "echo \"\"\n",
    "echo \"------------\"\n",
    "echo \"\"\n",
    "echo \"Memory Specs\"\n",
    "echo \"\"\n",
    "free -mh"
   ]
  },
  {
   "cell_type": "code",
   "execution_count": 2,
   "metadata": {},
   "outputs": [
    {
     "name": "stderr",
     "output_type": "stream",
     "text": [
      "\n",
      "real\t149m27.558s\n",
      "user\t0m6.264s\n",
      "sys\t10m11.356s\n"
     ]
    }
   ],
   "source": [
    "%%bash\n",
    "time cp /home/sam/owl/nightingales/P_generosa/[AN][DR]0*.gz /home/sam/data/geoduck_illumina/"
   ]
  },
  {
   "cell_type": "code",
   "execution_count": 3,
   "metadata": {},
   "outputs": [
    {
     "name": "stderr",
     "output_type": "stream",
     "text": [
      "bash: line 3: emailme: command not found\n"
     ]
    }
   ],
   "source": [
    "%%bash\n",
    "source /etc/environment\n",
    "source ~/.bashrc\n",
    "source ~/.bash_aliases\n",
    "emailme"
   ]
  },
  {
   "cell_type": "markdown",
   "metadata": {},
   "source": [
    "Steven previously ran these through [MultiQC](http://multiqc.info/) and they failed the adapter sequence contamination check. See his results here: [http://owl.fish.washington.edu/halfshell/bu-alanine-wd/17-09-15b/multiqc_report.html#fastqc_adapter_content](http://owl.fish.washington.edu/halfshell/bu-alanine-wd/17-09-15b/multiqc_report.html#fastqc_adapter_content)\n",
    "\n",
    "That means we need to trim them. Will use ```trim_galore``` for this; it has built-in adapter sequences for standar Illumina libraries, as well as Nextera libraries (will probably need to trim for both)."
   ]
  },
  {
   "cell_type": "code",
   "execution_count": 4,
   "metadata": {},
   "outputs": [
    {
     "name": "stdout",
     "output_type": "stream",
     "text": [
      "total 541G\n",
      "-rwxr-xr-x 1 sam sam  15G Jan 25 09:05 AD002_S9_L001_R1_001.fastq.gz\n",
      "-rwxr-xr-x 1 sam sam  15G Jan 25 09:08 AD002_S9_L001_R2_001.fastq.gz\n",
      "-rwxr-xr-x 1 sam sam  17G Jan 25 09:13 AD002_S9_L002_R1_001.fastq.gz\n",
      "-rwxr-xr-x 1 sam sam  18G Jan 25 09:17 AD002_S9_L002_R2_001.fastq.gz\n",
      "-rwxr-xr-x 1 sam sam 3.4M Jan 25 09:17 NR005_S4_L001_R1_001.fastq.gz\n",
      "-rwxr-xr-x 1 sam sam 3.5M Jan 25 09:17 NR005_S4_L001_R2_001.fastq.gz\n",
      "-rwxr-xr-x 1 sam sam 2.8M Jan 25 09:17 NR005_S4_L002_R1_001.fastq.gz\n",
      "-rwxr-xr-x 1 sam sam 2.9M Jan 25 09:17 NR005_S4_L002_R2_001.fastq.gz\n",
      "-rwxr-xr-x 1 sam sam 262M Jan 25 09:17 NR006_S3_L001_R1_001.fastq.gz\n",
      "-rwxr-xr-x 1 sam sam 271M Jan 25 09:17 NR006_S3_L001_R2_001.fastq.gz\n",
      "-rwxr-xr-x 1 sam sam 217M Jan 25 09:17 NR006_S3_L002_R1_001.fastq.gz\n",
      "-rwxr-xr-x 1 sam sam 226M Jan 25 09:17 NR006_S3_L002_R2_001.fastq.gz\n",
      "-rwxr-xr-x 1 sam sam  25M Jan 25 09:17 NR012_S1_L001_R1_001.fastq.gz\n",
      "-rwxr-xr-x 1 sam sam  27M Jan 25 09:17 NR012_S1_L001_R2_001.fastq.gz\n",
      "-rwxr-xr-x 1 sam sam  28M Jan 25 09:17 NR012_S1_L002_R1_001.fastq.gz\n",
      "-rwxr-xr-x 1 sam sam  29M Jan 25 09:17 NR012_S1_L002_R2_001.fastq.gz\n",
      "-rwxr-xr-x 1 sam sam  26G Jan 25 09:23 NR013_AD013_S2_L001_R1_001.fastq.gz\n",
      "-rwxr-xr-x 1 sam sam  28G Jan 25 09:30 NR013_AD013_S2_L001_R2_001.fastq.gz\n",
      "-rwxr-xr-x 1 sam sam  30G Jan 25 09:37 NR013_AD013_S2_L002_R1_001.fastq.gz\n",
      "-rwxr-xr-x 1 sam sam  32G Jan 25 09:44 NR013_AD013_S2_L002_R2_001.fastq.gz\n",
      "-rwxr-xr-x 1 sam sam  64G Jan 25 10:02 NR014_AD014_S5_L001_R1_001.fastq.gz\n",
      "-rwxr-xr-x 1 sam sam  66G Jan 25 10:20 NR014_AD014_S5_L001_R2_001.fastq.gz\n",
      "-rwxr-xr-x 1 sam sam  70G Jan 25 10:39 NR014_AD014_S5_L002_R1_001.fastq.gz\n",
      "-rwxr-xr-x 1 sam sam  73G Jan 25 11:01 NR014_AD014_S5_L002_R2_001.fastq.gz\n",
      "-rwxr-xr-x 1 sam sam  22G Jan 25 11:07 NR015_AD015_S6_L001_R1_001.fastq.gz\n",
      "-rwxr-xr-x 1 sam sam  23G Jan 25 11:15 NR015_AD015_S6_L001_R2_001.fastq.gz\n",
      "-rwxr-xr-x 1 sam sam  23G Jan 25 11:23 NR015_AD015_S6_L002_R1_001.fastq.gz\n",
      "-rwxr-xr-x 1 sam sam  24G Jan 25 11:31 NR015_AD015_S6_L002_R2_001.fastq.gz\n",
      "-rwxr-xr-x 1 sam sam 4.5M Jan 25 11:31 NR019_S7_L001_R1_001.fastq.gz\n",
      "-rwxr-xr-x 1 sam sam 4.5M Jan 25 11:31 NR019_S7_L001_R2_001.fastq.gz\n",
      "-rwxr-xr-x 1 sam sam 2.5M Jan 25 11:31 NR019_S7_L002_R1_001.fastq.gz\n",
      "-rwxr-xr-x 1 sam sam 2.5M Jan 25 11:31 NR019_S7_L002_R2_001.fastq.gz\n",
      "-rwxr-xr-x 1 sam sam 319M Jan 25 11:31 NR021_S8_L001_R1_001.fastq.gz\n",
      "-rwxr-xr-x 1 sam sam 333M Jan 25 11:31 NR021_S8_L001_R2_001.fastq.gz\n",
      "-rwxr-xr-x 1 sam sam 265M Jan 25 11:31 NR021_S8_L002_R1_001.fastq.gz\n",
      "-rwxr-xr-x 1 sam sam 276M Jan 25 11:31 NR021_S8_L002_R2_001.fastq.gz\n"
     ]
    }
   ],
   "source": [
    "%%bash\n",
    "ls -lh /home/sam/data/geoduck_illumina/"
   ]
  },
  {
   "cell_type": "markdown",
   "metadata": {
    "collapsed": true
   },
   "source": [
    "Restarted notebook to load in fixes for email notifications."
   ]
  },
  {
   "cell_type": "code",
   "execution_count": 1,
   "metadata": {},
   "outputs": [
    {
     "name": "stdout",
     "output_type": "stream",
     "text": [
      "Thu Jan 25 11:52:01 PST 2018\n"
     ]
    }
   ],
   "source": [
    "%%bash\n",
    "date"
   ]
  },
  {
   "cell_type": "code",
   "execution_count": 2,
   "metadata": {
    "collapsed": true
   },
   "outputs": [],
   "source": [
    "%%bash\n",
    "mkdir /home/sam/data/geoduck_illumina/trimmed"
   ]
  },
  {
   "cell_type": "code",
   "execution_count": 3,
   "metadata": {},
   "outputs": [
    {
     "name": "stderr",
     "output_type": "stream",
     "text": [
      "No quality encoding type selected. Assuming that the data provided uses Sanger encoded Phred scores (default)\n",
      "\n",
      "\n",
      "Please provide the filename(s) of one or more FastQ file(s) to launch Trim Galore!\n",
      "\n",
      "USAGE:  'trim_galore [options] <filename(s)>'    or    'trim_galore --help'    for more options\n",
      "\n",
      "\n",
      "real\t0m1.499s\n",
      "user\t0m0.036s\n",
      "sys\t0m0.012s\n",
      "bash: line 4: emailme: command not found\n"
     ]
    }
   ],
   "source": [
    "%%bash\n",
    "time /home/shared/trimgalore/trim_galore \\\n",
    "--paired \\\n",
    "--output_dir /home/sam/data/geoduck_illumina/trimmed/\n",
    "emailme"
   ]
  },
  {
   "cell_type": "code",
   "execution_count": 4,
   "metadata": {},
   "outputs": [
    {
     "name": "stdout",
     "output_type": "stream",
     "text": [
      "samwhite@uw.edu\n"
     ]
    }
   ],
   "source": [
    "%%bash\n",
    "echo $EMAIL"
   ]
  },
  {
   "cell_type": "code",
   "execution_count": 5,
   "metadata": {},
   "outputs": [
    {
     "name": "stdout",
     "output_type": "stream",
     "text": [
      "testing\n"
     ]
    },
    {
     "name": "stderr",
     "output_type": "stream",
     "text": [
      "bash: line 1: emailme: command not found\n"
     ]
    }
   ],
   "source": [
    "%%bash\n",
    "echo \"testing\"; emailme"
   ]
  },
  {
   "cell_type": "code",
   "execution_count": 6,
   "metadata": {
    "collapsed": true
   },
   "outputs": [],
   "source": [
    "%%bash\n",
    "source /home/sam/.bash_aliases"
   ]
  },
  {
   "cell_type": "code",
   "execution_count": 7,
   "metadata": {},
   "outputs": [
    {
     "name": "stdout",
     "output_type": "stream",
     "text": [
      "testing\n"
     ]
    },
    {
     "name": "stderr",
     "output_type": "stream",
     "text": [
      "bash: line 1: emailme: command not found\n"
     ]
    }
   ],
   "source": [
    "%%bash\n",
    "echo \"testing\"; emailme"
   ]
  },
  {
   "cell_type": "code",
   "execution_count": 8,
   "metadata": {},
   "outputs": [
    {
     "name": "stdout",
     "output_type": "stream",
     "text": [
      "testing\n"
     ]
    },
    {
     "name": "stderr",
     "output_type": "stream",
     "text": [
      "bash: line 2: emailme: command not found\n"
     ]
    }
   ],
   "source": [
    "%%bash\n",
    "source /home/sam/.bash_aliases\n",
    "echo \"testing\"; emailme"
   ]
  },
  {
   "cell_type": "code",
   "execution_count": 9,
   "metadata": {},
   "outputs": [
    {
     "name": "stdout",
     "output_type": "stream",
     "text": [
      "# Alias for emailing with default email sender/subject\n",
      "alias emailme='cat ~/.default_subject.mail | msmtp \"$EMAIL\"'\n"
     ]
    }
   ],
   "source": [
    "%%bash\n",
    "cat /home/sam/.bash_aliases"
   ]
  },
  {
   "cell_type": "code",
   "execution_count": 10,
   "metadata": {},
   "outputs": [
    {
     "name": "stdout",
     "output_type": "stream",
     "text": [
      "# Alias for emailing with default email sender/subject\n",
      "alias emailme='cat ~/.default-subject.mail | msmtp \"$EMAIL\"'\n"
     ]
    }
   ],
   "source": [
    "%%bash\n",
    "cat /home/sam/.bash_aliases"
   ]
  },
  {
   "cell_type": "code",
   "execution_count": 11,
   "metadata": {},
   "outputs": [
    {
     "name": "stdout",
     "output_type": "stream",
     "text": [
      "testing\n"
     ]
    },
    {
     "name": "stderr",
     "output_type": "stream",
     "text": [
      "bash: line 2: emailme: command not found\n"
     ]
    }
   ],
   "source": [
    "%%bash\n",
    "source /home/sam/.bash_aliases\n",
    "echo \"testing\"; emailme"
   ]
  },
  {
   "cell_type": "code",
   "execution_count": 12,
   "metadata": {},
   "outputs": [
    {
     "name": "stderr",
     "output_type": "stream",
     "text": [
      "bash: line 3: unexpected EOF while looking for matching `\"'\n",
      "bash: line 5: syntax error: unexpected end of file\n"
     ]
    }
   ],
   "source": [
    "%%bash\n",
    "for file in /home/sam/data/geoduck_illumina/*.gz\n",
    "do\n",
    "printf %s/n \"$file \\\"\n",
    "done"
   ]
  },
  {
   "cell_type": "code",
   "execution_count": 13,
   "metadata": {},
   "outputs": [
    {
     "name": "stderr",
     "output_type": "stream",
     "text": [
      "bash: line 3: unexpected EOF while looking for matching `\"'\n",
      "bash: line 5: syntax error: unexpected end of file\n"
     ]
    }
   ],
   "source": [
    "%%bash\n",
    "for file in /home/sam/data/geoduck_illumina/*.gz\n",
    "do\n",
    "printf \"\"%s\\n\"\" \"$file \\\"\n",
    "done"
   ]
  },
  {
   "cell_type": "code",
   "execution_count": 14,
   "metadata": {},
   "outputs": [
    {
     "name": "stderr",
     "output_type": "stream",
     "text": [
      "bash: line 3: unexpected EOF while looking for matching `\"'\n",
      "bash: line 5: syntax error: unexpected end of file\n"
     ]
    }
   ],
   "source": [
    "%%bash\n",
    "for file in /home/sam/data/geoduck_illumina/*.gz\n",
    "do\n",
    "printf \"%s\\n\" \"$file \\\"\n",
    "done"
   ]
  },
  {
   "cell_type": "code",
   "execution_count": 15,
   "metadata": {},
   "outputs": [
    {
     "name": "stdout",
     "output_type": "stream",
     "text": [
      "/home/sam/data/geoduck_illumina/AD002_S9_L001_R1_001.fastq.gz \\\n",
      "/home/sam/data/geoduck_illumina/AD002_S9_L001_R2_001.fastq.gz \\\n",
      "/home/sam/data/geoduck_illumina/AD002_S9_L002_R1_001.fastq.gz \\\n",
      "/home/sam/data/geoduck_illumina/AD002_S9_L002_R2_001.fastq.gz \\\n",
      "/home/sam/data/geoduck_illumina/NR005_S4_L001_R1_001.fastq.gz \\\n",
      "/home/sam/data/geoduck_illumina/NR005_S4_L001_R2_001.fastq.gz \\\n",
      "/home/sam/data/geoduck_illumina/NR005_S4_L002_R1_001.fastq.gz \\\n",
      "/home/sam/data/geoduck_illumina/NR005_S4_L002_R2_001.fastq.gz \\\n",
      "/home/sam/data/geoduck_illumina/NR006_S3_L001_R1_001.fastq.gz \\\n",
      "/home/sam/data/geoduck_illumina/NR006_S3_L001_R2_001.fastq.gz \\\n",
      "/home/sam/data/geoduck_illumina/NR006_S3_L002_R1_001.fastq.gz \\\n",
      "/home/sam/data/geoduck_illumina/NR006_S3_L002_R2_001.fastq.gz \\\n",
      "/home/sam/data/geoduck_illumina/NR012_S1_L001_R1_001.fastq.gz \\\n",
      "/home/sam/data/geoduck_illumina/NR012_S1_L001_R2_001.fastq.gz \\\n",
      "/home/sam/data/geoduck_illumina/NR012_S1_L002_R1_001.fastq.gz \\\n",
      "/home/sam/data/geoduck_illumina/NR012_S1_L002_R2_001.fastq.gz \\\n",
      "/home/sam/data/geoduck_illumina/NR013_AD013_S2_L001_R1_001.fastq.gz \\\n",
      "/home/sam/data/geoduck_illumina/NR013_AD013_S2_L001_R2_001.fastq.gz \\\n",
      "/home/sam/data/geoduck_illumina/NR013_AD013_S2_L002_R1_001.fastq.gz \\\n",
      "/home/sam/data/geoduck_illumina/NR013_AD013_S2_L002_R2_001.fastq.gz \\\n",
      "/home/sam/data/geoduck_illumina/NR014_AD014_S5_L001_R1_001.fastq.gz \\\n",
      "/home/sam/data/geoduck_illumina/NR014_AD014_S5_L001_R2_001.fastq.gz \\\n",
      "/home/sam/data/geoduck_illumina/NR014_AD014_S5_L002_R1_001.fastq.gz \\\n",
      "/home/sam/data/geoduck_illumina/NR014_AD014_S5_L002_R2_001.fastq.gz \\\n",
      "/home/sam/data/geoduck_illumina/NR015_AD015_S6_L001_R1_001.fastq.gz \\\n",
      "/home/sam/data/geoduck_illumina/NR015_AD015_S6_L001_R2_001.fastq.gz \\\n",
      "/home/sam/data/geoduck_illumina/NR015_AD015_S6_L002_R1_001.fastq.gz \\\n",
      "/home/sam/data/geoduck_illumina/NR015_AD015_S6_L002_R2_001.fastq.gz \\\n",
      "/home/sam/data/geoduck_illumina/NR019_S7_L001_R1_001.fastq.gz \\\n",
      "/home/sam/data/geoduck_illumina/NR019_S7_L001_R2_001.fastq.gz \\\n",
      "/home/sam/data/geoduck_illumina/NR019_S7_L002_R1_001.fastq.gz \\\n",
      "/home/sam/data/geoduck_illumina/NR019_S7_L002_R2_001.fastq.gz \\\n",
      "/home/sam/data/geoduck_illumina/NR021_S8_L001_R1_001.fastq.gz \\\n",
      "/home/sam/data/geoduck_illumina/NR021_S8_L001_R2_001.fastq.gz \\\n",
      "/home/sam/data/geoduck_illumina/NR021_S8_L002_R1_001.fastq.gz \\\n",
      "/home/sam/data/geoduck_illumina/NR021_S8_L002_R2_001.fastq.gz \\\n"
     ]
    }
   ],
   "source": [
    "%%bash\n",
    "for file in /home/sam/data/geoduck_illumina/*.gz\n",
    "do\n",
    "printf \"%s\\n\" \"$file \\\\\"\n",
    "done"
   ]
  },
  {
   "cell_type": "code",
   "execution_count": 16,
   "metadata": {},
   "outputs": [
    {
     "name": "stdout",
     "output_type": "stream",
     "text": [
      "AD002_S9_L001_R1_001.fastq.gz \\\n",
      "AD002_S9_L001_R2_001.fastq.gz \\\n",
      "AD002_S9_L002_R1_001.fastq.gz \\\n",
      "AD002_S9_L002_R2_001.fastq.gz \\\n",
      "NR005_S4_L001_R1_001.fastq.gz \\\n",
      "NR005_S4_L001_R2_001.fastq.gz \\\n",
      "NR005_S4_L002_R1_001.fastq.gz \\\n",
      "NR005_S4_L002_R2_001.fastq.gz \\\n",
      "NR006_S3_L001_R1_001.fastq.gz \\\n",
      "NR006_S3_L001_R2_001.fastq.gz \\\n",
      "NR006_S3_L002_R1_001.fastq.gz \\\n",
      "NR006_S3_L002_R2_001.fastq.gz \\\n",
      "NR012_S1_L001_R1_001.fastq.gz \\\n",
      "NR012_S1_L001_R2_001.fastq.gz \\\n",
      "NR012_S1_L002_R1_001.fastq.gz \\\n",
      "NR012_S1_L002_R2_001.fastq.gz \\\n",
      "NR013_AD013_S2_L001_R1_001.fastq.gz \\\n",
      "NR013_AD013_S2_L001_R2_001.fastq.gz \\\n",
      "NR013_AD013_S2_L002_R1_001.fastq.gz \\\n",
      "NR013_AD013_S2_L002_R2_001.fastq.gz \\\n",
      "NR014_AD014_S5_L001_R1_001.fastq.gz \\\n",
      "NR014_AD014_S5_L001_R2_001.fastq.gz \\\n",
      "NR014_AD014_S5_L002_R1_001.fastq.gz \\\n",
      "NR014_AD014_S5_L002_R2_001.fastq.gz \\\n",
      "NR015_AD015_S6_L001_R1_001.fastq.gz \\\n",
      "NR015_AD015_S6_L001_R2_001.fastq.gz \\\n",
      "NR015_AD015_S6_L002_R1_001.fastq.gz \\\n",
      "NR015_AD015_S6_L002_R2_001.fastq.gz \\\n",
      "NR019_S7_L001_R1_001.fastq.gz \\\n",
      "NR019_S7_L001_R2_001.fastq.gz \\\n",
      "NR019_S7_L002_R1_001.fastq.gz \\\n",
      "NR019_S7_L002_R2_001.fastq.gz \\\n",
      "NR021_S8_L001_R1_001.fastq.gz \\\n",
      "NR021_S8_L001_R2_001.fastq.gz \\\n",
      "NR021_S8_L002_R1_001.fastq.gz \\\n",
      "NR021_S8_L002_R2_001.fastq.gz \\\n"
     ]
    }
   ],
   "source": [
    "%%bash\n",
    "for file in /home/sam/data/geoduck_illumina/*.gz\n",
    "do\n",
    "filename=${file##*/}\n",
    "printf \"%s\\n\" \"$filename \\\\\"\n",
    "done"
   ]
  },
  {
   "cell_type": "code",
   "execution_count": 17,
   "metadata": {},
   "outputs": [
    {
     "name": "stdout",
     "output_type": "stream",
     "text": [
      "1.9.1\n"
     ]
    },
    {
     "name": "stderr",
     "output_type": "stream",
     "text": [
      "No quality encoding type selected. Assuming that the data provided uses Sanger encoded Phred scores (default)\n",
      "\n",
      "gzip: AD002_S9_L001_R1_001.fastq.gz: No such file or directory\n",
      "Input file 'AD002_S9_L001_R1_001.fastq.gz' seems to be completely empty. Consider respecifying!\n",
      "\n",
      "Path to Cutadapt set as: 'cutadapt' (default)\n",
      "Cutadapt seems to be working fine (tested command 'cutadapt --version')\n",
      "\n",
      "\n",
      "AUTO-DETECTING ADAPTER TYPE\n",
      "===========================\n",
      "Attempting to auto-detect adapter type from the first 1 million sequences of the first file (>> AD002_S9_L001_R1_001.fastq.gz <<)\n",
      "\n",
      "gzip: AD002_S9_L001_R1_001.fastq.gz: No such file or directory\n",
      "Found perfect matches for the following adapter sequences:\n",
      "Adapter type\tCount\tSequence\tSequences analysed\tPercentage\n",
      "Illegal division by zero at /home/shared/trimgalore/trim_galore line 1229.\n",
      "\n",
      "real\t0m1.615s\n",
      "user\t0m0.112s\n",
      "sys\t0m0.024s\n"
     ]
    }
   ],
   "source": [
    "%%bash\n",
    "time /home/shared/trimgalore/trim_galore \\\n",
    "--paired \\\n",
    "--output_dir /home/sam/data/geoduck_illumina/trimmed/ \\\n",
    "AD002_S9_L001_R1_001.fastq.gz \\\n",
    "AD002_S9_L001_R2_001.fastq.gz \\\n",
    "AD002_S9_L002_R1_001.fastq.gz \\\n",
    "AD002_S9_L002_R2_001.fastq.gz \\\n",
    "NR005_S4_L001_R1_001.fastq.gz \\\n",
    "NR005_S4_L001_R2_001.fastq.gz \\\n",
    "NR005_S4_L002_R1_001.fastq.gz \\\n",
    "NR005_S4_L002_R2_001.fastq.gz \\\n",
    "NR006_S3_L001_R1_001.fastq.gz \\\n",
    "NR006_S3_L001_R2_001.fastq.gz \\\n",
    "NR006_S3_L002_R1_001.fastq.gz \\\n",
    "NR006_S3_L002_R2_001.fastq.gz \\\n",
    "NR012_S1_L001_R1_001.fastq.gz \\\n",
    "NR012_S1_L001_R2_001.fastq.gz \\\n",
    "NR012_S1_L002_R1_001.fastq.gz \\\n",
    "NR012_S1_L002_R2_001.fastq.gz \\\n",
    "NR013_AD013_S2_L001_R1_001.fastq.gz \\\n",
    "NR013_AD013_S2_L001_R2_001.fastq.gz \\\n",
    "NR013_AD013_S2_L002_R1_001.fastq.gz \\\n",
    "NR013_AD013_S2_L002_R2_001.fastq.gz \\\n",
    "NR014_AD014_S5_L001_R1_001.fastq.gz \\\n",
    "NR014_AD014_S5_L001_R2_001.fastq.gz \\\n",
    "NR014_AD014_S5_L002_R1_001.fastq.gz \\\n",
    "NR014_AD014_S5_L002_R2_001.fastq.gz \\\n",
    "NR015_AD015_S6_L001_R1_001.fastq.gz \\\n",
    "NR015_AD015_S6_L001_R2_001.fastq.gz \\\n",
    "NR015_AD015_S6_L002_R1_001.fastq.gz \\\n",
    "NR015_AD015_S6_L002_R2_001.fastq.gz \\\n",
    "NR019_S7_L001_R1_001.fastq.gz \\\n",
    "NR019_S7_L001_R2_001.fastq.gz \\\n",
    "NR019_S7_L002_R1_001.fastq.gz \\\n",
    "NR019_S7_L002_R2_001.fastq.gz \\\n",
    "NR021_S8_L001_R1_001.fastq.gz \\\n",
    "NR021_S8_L001_R2_001.fastq.gz \\\n",
    "NR021_S8_L002_R1_001.fastq.gz \\\n",
    "NR021_S8_L002_R2_001.fastq.gz"
   ]
  },
  {
   "cell_type": "code",
   "execution_count": 18,
   "metadata": {
    "collapsed": true
   },
   "outputs": [],
   "source": [
    "cat ~/.default-subject.mail | msmtp \"$EMAIL\""
   ]
  },
  {
   "cell_type": "code",
   "execution_count": 19,
   "metadata": {},
   "outputs": [
    {
     "name": "stdout",
     "output_type": "stream",
     "text": [
      "1.9.1\n"
     ]
    },
    {
     "name": "stderr",
     "output_type": "stream",
     "text": [
      "No quality encoding type selected. Assuming that the data provided uses Sanger encoded Phred scores (default)\n",
      "\n",
      "Path to Cutadapt set as: 'cutadapt' (default)\n",
      "Cutadapt seems to be working fine (tested command 'cutadapt --version')\n",
      "\n",
      "\n",
      "AUTO-DETECTING ADAPTER TYPE\n",
      "===========================\n",
      "Attempting to auto-detect adapter type from the first 1 million sequences of the first file (>> /home/sam/data/geoduck_illumina/AD002_S9_L001_R1_001.fastq.gz <<)\n",
      "\n",
      "Found perfect matches for the following adapter sequences:\n",
      "Adapter type\tCount\tSequence\tSequences analysed\tPercentage\n",
      "Nextera\t373733\tCTGTCTCTTATA\t1000000\t37.37\n",
      "Illumina\t15621\tAGATCGGAAGAGC\t1000000\t1.56\n",
      "smallRNA\t4\tTGGAATTCTCGG\t1000000\t0.00\n",
      "Using Nextera adapter for trimming (count: 373733). Second best hit was Illumina (count: 15621)\n",
      "\n",
      "Writing report to '/home/sam/data/geoduck_illumina/trimmed/AD002_S9_L001_R1_001.fastq.gz_trimming_report.txt'\n",
      "\n",
      "SUMMARISING RUN PARAMETERS\n",
      "==========================\n",
      "Input filename: /home/sam/data/geoduck_illumina/AD002_S9_L001_R1_001.fastq.gz\n",
      "Trimming mode: paired-end\n",
      "Trim Galore version: 0.4.4\n",
      "Cutadapt version: 1.9.1\n",
      "Quality Phred score cutoff: 20\n",
      "Quality encoding type selected: ASCII+33\n",
      "Adapter sequence: 'CTGTCTCTTATA' (Nextera Transposase sequence; auto-detected)\n",
      "Maximum trimming error rate: 0.1 (default)\n",
      "Minimum required adapter overlap (stringency): 1 bp\n",
      "Minimum required sequence length for both reads before a sequence pair gets removed: 20 bp\n",
      "Output file(s) will be GZIP compressed\n",
      "\n",
      "Writing final adapter and quality trimmed output to AD002_S9_L001_R1_001_trimmed.fq.gz\n",
      "\n",
      "\n",
      "  >>> Now performing quality (cutoff 20) and adapter trimming in a single pass for the adapter sequence: 'CTGTCTCTTATA' from file /home/sam/data/geoduck_illumina/AD002_S9_L001_R1_001.fastq.gz <<< \n",
      "10000000 sequences processed\n",
      "20000000 sequences processed\n",
      "30000000 sequences processed\n",
      "40000000 sequences processed\n",
      "50000000 sequences processed\n",
      "60000000 sequences processed\n",
      "70000000 sequences processed\n",
      "80000000 sequences processed\n",
      "90000000 sequences processed\n",
      "100000000 sequences processed\n",
      "110000000 sequences processed\n",
      "120000000 sequences processed\n",
      "130000000 sequences processed\n",
      "140000000 sequences processed\n",
      "150000000 sequences processed\n",
      "160000000 sequences processed\n",
      "170000000 sequences processed\n",
      "180000000 sequences processed\n",
      "This is cutadapt 1.9.1 with Python 2.7.12\n",
      "Command line parameters: -f fastq -e 0.1 -q 20 -O 1 -a CTGTCTCTTATA /home/sam/data/geoduck_illumina/AD002_S9_L001_R1_001.fastq.gz\n",
      "Trimming 1 adapter with at most 10.0% errors in single-end mode ...\n",
      "Finished in 4482.08 s (24 us/read; 2.51 M reads/minute).\n",
      "\n",
      "=== Summary ===\n",
      "\n",
      "Total reads processed:             187,796,434\n",
      "Reads with adapters:               108,648,810 (57.9%)\n",
      "Reads written (passing filters):   187,796,434 (100.0%)\n",
      "\n",
      "Total basepairs processed: 28,357,261,534 bp\n",
      "Quality-trimmed:              69,695,707 bp (0.2%)\n",
      "Total written (filtered):  21,758,478,183 bp (76.7%)\n",
      "\n",
      "=== Adapter 1 ===\n",
      "\n",
      "Sequence: CTGTCTCTTATA; Type: regular 3'; Length: 12; Trimmed: 108648810 times.\n",
      "\n",
      "No. of allowed errors:\n",
      "0-9 bp: 0; 10-12 bp: 1\n",
      "\n",
      "Bases preceding removed adapters:\n",
      "  A: 17.6%\n",
      "  C: 35.0%\n",
      "  G: 20.6%\n",
      "  T: 26.7%\n",
      "  none/other: 0.1%\n",
      "\n",
      "Overview of removed sequences\n",
      "length\tcount\texpect\tmax.err\terror counts\n",
      "1\t19862890\t46949108.5\t0\t19862890\n",
      "2\t5913069\t11737277.1\t0\t5913069\n",
      "3\t2045500\t2934319.3\t0\t2045500\n",
      "4\t923646\t733579.8\t0\t923646\n",
      "5\t581270\t183395.0\t0\t581270\n",
      "6\t473952\t45848.7\t0\t473952\n",
      "7\t477046\t11462.2\t0\t477046\n",
      "8\t506207\t2865.5\t0\t506207\n",
      "9\t429336\t716.4\t0\t425000 4336\n",
      "10\t477601\t179.1\t1\t446591 31010\n",
      "11\t496442\t44.8\t1\t467811 28631\n",
      "12\t470709\t11.2\t1\t439639 31070\n",
      "13\t485921\t11.2\t1\t457188 28733\n",
      "14\t445391\t11.2\t1\t418393 26998\n",
      "15\t536842\t11.2\t1\t502591 34251\n",
      "16\t578317\t11.2\t1\t543340 34977\n",
      "17\t347969\t11.2\t1\t327499 20470\n",
      "18\t491372\t11.2\t1\t463777 27595\n",
      "19\t504486\t11.2\t1\t478695 25791\n",
      "20\t446679\t11.2\t1\t419960 26719\n",
      "21\t493397\t11.2\t1\t467370 26027\n",
      "22\t545771\t11.2\t1\t515112 30659\n",
      "23\t404368\t11.2\t1\t380754 23614\n",
      "24\t504717\t11.2\t1\t478368 26349\n",
      "25\t438793\t11.2\t1\t416645 22148\n",
      "26\t518329\t11.2\t1\t488120 30209\n",
      "27\t491943\t11.2\t1\t466877 25066\n",
      "28\t464468\t11.2\t1\t440682 23786\n",
      "29\t481248\t11.2\t1\t458288 22960\n",
      "30\t528107\t11.2\t1\t500993 27114\n",
      "31\t489811\t11.2\t1\t464264 25547\n",
      "32\t474388\t11.2\t1\t451937 22451\n",
      "33\t478470\t11.2\t1\t454082 24388\n",
      "34\t480220\t11.2\t1\t457224 22996\n",
      "35\t488796\t11.2\t1\t463539 25257\n",
      "36\t528715\t11.2\t1\t504098 24617\n",
      "37\t468294\t11.2\t1\t443393 24901\n",
      "38\t516389\t11.2\t1\t491239 25150\n",
      "39\t504166\t11.2\t1\t479708 24458\n",
      "40\t504772\t11.2\t1\t480743 24029\n",
      "41\t497388\t11.2\t1\t472917 24471\n",
      "42\t503256\t11.2\t1\t479203 24053\n",
      "43\t500098\t11.2\t1\t476330 23768\n",
      "44\t499772\t11.2\t1\t476045 23727\n",
      "45\t502817\t11.2\t1\t479398 23419\n",
      "46\t499435\t11.2\t1\t475501 23934\n",
      "47\t508746\t11.2\t1\t485077 23669\n",
      "48\t504236\t11.2\t1\t480644 23592\n",
      "49\t510124\t11.2\t1\t486308 23816\n",
      "50\t507278\t11.2\t1\t483978 23300\n",
      "51\t509992\t11.2\t1\t486588 23404\n",
      "52\t510956\t11.2\t1\t488137 22819\n",
      "53\t513247\t11.2\t1\t490293 22954\n",
      "54\t511277\t11.2\t1\t488419 22858\n",
      "55\t512173\t11.2\t1\t489481 22692\n",
      "56\t514152\t11.2\t1\t491750 22402\n",
      "57\t516243\t11.2\t1\t493679 22564\n",
      "58\t516952\t11.2\t1\t494663 22289\n",
      "59\t520137\t11.2\t1\t497533 22604\n",
      "60\t521470\t11.2\t1\t499230 22240\n",
      "61\t523017\t11.2\t1\t500690 22327\n",
      "62\t521644\t11.2\t1\t499715 21929\n",
      "63\t522362\t11.2\t1\t500517 21845\n",
      "64\t521660\t11.2\t1\t500073 21587\n",
      "65\t523669\t11.2\t1\t501771 21898\n",
      "66\t523555\t11.2\t1\t501965 21590\n",
      "67\t525958\t11.2\t1\t504466 21492\n",
      "68\t527930\t11.2\t1\t506663 21267\n",
      "69\t531548\t11.2\t1\t510020 21528\n",
      "70\t531427\t11.2\t1\t509837 21590\n",
      "71\t535318\t11.2\t1\t513792 21526\n",
      "72\t533752\t11.2\t1\t512805 20947\n",
      "73\t535054\t11.2\t1\t513827 21227\n",
      "74\t534874\t11.2\t1\t513683 21191\n",
      "75\t534350\t11.2\t1\t513385 20965\n",
      "76\t537816\t11.2\t1\t516778 21038\n",
      "77\t538471\t11.2\t1\t517588 20883\n",
      "78\t540612\t11.2\t1\t519613 20999\n",
      "79\t544380\t11.2\t1\t523348 21032\n",
      "80\t546669\t11.2\t1\t525727 20942\n",
      "81\t547518\t11.2\t1\t526685 20833\n",
      "82\t548135\t11.2\t1\t527694 20441\n",
      "83\t548870\t11.2\t1\t528385 20485\n",
      "84\t548717\t11.2\t1\t528375 20342\n",
      "85\t549117\t11.2\t1\t528665 20452\n",
      "86\t549211\t11.2\t1\t528761 20450\n",
      "87\t552056\t11.2\t1\t531833 20223\n",
      "88\t553878\t11.2\t1\t533813 20065\n",
      "89\t558989\t11.2\t1\t539020 19969\n",
      "90\t560248\t11.2\t1\t539997 20251\n",
      "91\t564403\t11.2\t1\t544171 20232\n",
      "92\t565138\t11.2\t1\t545285 19853\n",
      "93\t563567\t11.2\t1\t543638 19929\n",
      "94\t562873\t11.2\t1\t543490 19383\n",
      "95\t564022\t11.2\t1\t544851 19171\n",
      "96\t565332\t11.2\t1\t546247 19085\n",
      "97\t565694\t11.2\t1\t546817 18877\n",
      "98\t567802\t11.2\t1\t549213 18589\n",
      "99\t571052\t11.2\t1\t551955 19097\n",
      "100\t578310\t11.2\t1\t559322 18988\n",
      "101\t579982\t11.2\t1\t560847 19135\n",
      "102\t581851\t11.2\t1\t562931 18920\n",
      "103\t582338\t11.2\t1\t563504 18834\n",
      "104\t581897\t11.2\t1\t563677 18220\n",
      "105\t582162\t11.2\t1\t563575 18587\n",
      "106\t584452\t11.2\t1\t565201 19251\n",
      "107\t585199\t11.2\t1\t566128 19071\n",
      "108\t585585\t11.2\t1\t566374 19211\n",
      "109\t590983\t11.2\t1\t572016 18967\n",
      "110\t597596\t11.2\t1\t578755 18841\n",
      "111\t602875\t11.2\t1\t583907 18968\n",
      "112\t604774\t11.2\t1\t585834 18940\n",
      "113\t606458\t11.2\t1\t587404 19054\n",
      "114\t603940\t11.2\t1\t585325 18615\n",
      "115\t602671\t11.2\t1\t584245 18426\n",
      "116\t603098\t11.2\t1\t584684 18414\n",
      "117\t601322\t11.2\t1\t583229 18093\n",
      "118\t602874\t11.2\t1\t585432 17442\n",
      "119\t607469\t11.2\t1\t589940 17529\n",
      "120\t614394\t11.2\t1\t597046 17348\n",
      "121\t621318\t11.2\t1\t603834 17484\n",
      "122\t627635\t11.2\t1\t610100 17535\n",
      "123\t628652\t11.2\t1\t611112 17540\n",
      "124\t623745\t11.2\t1\t606579 17166\n",
      "125\t623213\t11.2\t1\t605905 17308\n",
      "126\t621191\t11.2\t1\t604400 16791\n",
      "127\t623732\t11.2\t1\t607347 16385\n",
      "128\t627258\t11.2\t1\t611387 15871\n",
      "129\t629912\t11.2\t1\t614647 15265\n",
      "130\t639245\t11.2\t1\t624973 14272\n",
      "131\t647229\t11.2\t1\t633032 14197\n",
      "132\t650488\t11.2\t1\t636946 13542\n",
      "133\t648379\t11.2\t1\t635925 12454\n",
      "134\t623141\t11.2\t1\t611943 11198\n",
      "135\t618297\t11.2\t1\t607766 10531\n",
      "136\t606623\t11.2\t1\t596528 10095\n",
      "137\t596508\t11.2\t1\t587014 9494\n",
      "138\t576880\t11.2\t1\t567742 9138\n",
      "139\t578999\t11.2\t1\t569999 9000\n",
      "140\t604758\t11.2\t1\t595394 9364\n",
      "141\t631799\t11.2\t1\t622180 9619\n",
      "142\t562420\t11.2\t1\t553470 8950\n",
      "143\t560808\t11.2\t1\t552100 8708\n",
      "144\t607571\t11.2\t1\t598417 9154\n",
      "145\t618301\t11.2\t1\t608953 9348\n",
      "146\t692057\t11.2\t1\t681341 10716\n",
      "147\t606471\t11.2\t1\t596463 10008\n",
      "148\t616191\t11.2\t1\t605739 10452\n",
      "149\t535095\t11.2\t1\t525226 9869\n",
      "150\t470574\t11.2\t1\t460819 9755\n",
      "151\t129846\t11.2\t1\t118414 11432\n",
      "\n",
      "\n",
      "RUN STATISTICS FOR INPUT FILE: /home/sam/data/geoduck_illumina/AD002_S9_L001_R1_001.fastq.gz\n",
      "=============================================\n",
      "187796434 sequences processed in total\n",
      "The length threshold of paired-end sequences gets evaluated later on (in the validation step)\n",
      "\n",
      "Writing report to '/home/sam/data/geoduck_illumina/trimmed/AD002_S9_L001_R2_001.fastq.gz_trimming_report.txt'\n",
      "\n",
      "SUMMARISING RUN PARAMETERS\n",
      "==========================\n",
      "Input filename: /home/sam/data/geoduck_illumina/AD002_S9_L001_R2_001.fastq.gz\n",
      "Trimming mode: paired-end\n",
      "Trim Galore version: 0.4.4\n",
      "Cutadapt version: 1.9.1\n",
      "Quality Phred score cutoff: 20\n",
      "Quality encoding type selected: ASCII+33\n",
      "Adapter sequence: 'CTGTCTCTTATA' (Nextera Transposase sequence; auto-detected)\n",
      "Maximum trimming error rate: 0.1 (default)\n",
      "Minimum required adapter overlap (stringency): 1 bp\n",
      "Minimum required sequence length for both reads before a sequence pair gets removed: 20 bp\n",
      "Output file(s) will be GZIP compressed\n",
      "\n",
      "Writing final adapter and quality trimmed output to AD002_S9_L001_R2_001_trimmed.fq.gz\n",
      "\n",
      "\n",
      "  >>> Now performing quality (cutoff 20) and adapter trimming in a single pass for the adapter sequence: 'CTGTCTCTTATA' from file /home/sam/data/geoduck_illumina/AD002_S9_L001_R2_001.fastq.gz <<< \n",
      "10000000 sequences processed\n",
      "20000000 sequences processed\n",
      "30000000 sequences processed\n",
      "40000000 sequences processed\n",
      "50000000 sequences processed\n",
      "60000000 sequences processed\n",
      "70000000 sequences processed\n",
      "80000000 sequences processed\n",
      "90000000 sequences processed\n",
      "100000000 sequences processed\n",
      "110000000 sequences processed\n",
      "120000000 sequences processed\n",
      "130000000 sequences processed\n",
      "140000000 sequences processed\n",
      "150000000 sequences processed\n",
      "160000000 sequences processed\n",
      "170000000 sequences processed\n",
      "180000000 sequences processed\n",
      "This is cutadapt 1.9.1 with Python 2.7.12\n",
      "Command line parameters: -f fastq -e 0.1 -q 20 -O 1 -a CTGTCTCTTATA /home/sam/data/geoduck_illumina/AD002_S9_L001_R2_001.fastq.gz\n",
      "Trimming 1 adapter with at most 10.0% errors in single-end mode ...\n",
      "Finished in 4531.42 s (24 us/read; 2.49 M reads/minute).\n",
      "\n",
      "=== Summary ===\n",
      "\n",
      "Total reads processed:             187,796,434\n",
      "Reads with adapters:               100,915,965 (53.7%)\n",
      "Reads written (passing filters):   187,796,434 (100.0%)\n",
      "\n",
      "Total basepairs processed: 28,357,261,534 bp\n",
      "Quality-trimmed:              94,931,330 bp (0.3%)\n",
      "Total written (filtered):  22,585,496,126 bp (79.6%)\n",
      "\n",
      "=== Adapter 1 ===\n",
      "\n",
      "Sequence: CTGTCTCTTATA; Type: regular 3'; Length: 12; Trimmed: 100915965 times.\n",
      "\n",
      "No. of allowed errors:\n",
      "0-9 bp: 0; 10-12 bp: 1\n",
      "\n",
      "Bases preceding removed adapters:\n",
      "  A: 18.4%\n",
      "  C: 33.9%\n",
      "  G: 20.7%\n",
      "  T: 26.9%\n",
      "  none/other: 0.1%\n",
      "\n",
      "Overview of removed sequences\n",
      "length\tcount\texpect\tmax.err\terror counts\n",
      "1\t20650020\t46949108.5\t0\t20650020\n",
      "2\t6146591\t11737277.1\t0\t6146591\n",
      "3\t2063660\t2934319.3\t0\t2063660\n",
      "4\t928498\t733579.8\t0\t928498\n",
      "5\t556966\t183395.0\t0\t556966\n",
      "6\t447817\t45848.7\t0\t447817\n",
      "7\t447043\t11462.2\t0\t447043\n",
      "8\t483777\t2865.5\t0\t483777\n",
      "9\t396818\t716.4\t0\t391992 4826\n",
      "10\t450848\t179.1\t1\t409394 41454\n",
      "11\t475250\t44.8\t1\t433221 42029\n",
      "12\t431724\t11.2\t1\t391413 40311\n",
      "13\t464934\t11.2\t1\t421617 43317\n",
      "14\t408441\t11.2\t1\t373001 35440\n",
      "15\t511046\t11.2\t1\t461934 49112\n",
      "16\t519299\t11.2\t1\t474033 45266\n",
      "17\t344217\t11.2\t1\t314891 29326\n",
      "18\t462589\t11.2\t1\t422724 39865\n",
      "19\t480574\t11.2\t1\t442349 38225\n",
      "20\t419028\t11.2\t1\t385224 33804\n",
      "21\t460933\t11.2\t1\t421638 39295\n",
      "22\t500587\t11.2\t1\t458801 41786\n",
      "23\t390996\t11.2\t1\t358743 32253\n",
      "24\t465914\t11.2\t1\t425972 39942\n",
      "25\t423719\t11.2\t1\t391596 32123\n",
      "26\t485401\t11.2\t1\t441113 44288\n",
      "27\t468114\t11.2\t1\t430294 37820\n",
      "28\t434335\t11.2\t1\t400519 33816\n",
      "29\t455764\t11.2\t1\t420992 34772\n",
      "30\t486842\t11.2\t1\t449922 36920\n",
      "31\t448704\t11.2\t1\t415257 33447\n",
      "32\t448446\t11.2\t1\t413280 35166\n",
      "33\t448111\t11.2\t1\t415051 33060\n",
      "34\t456901\t11.2\t1\t421409 35492\n",
      "35\t439833\t11.2\t1\t407311 32522\n",
      "36\t509049\t11.2\t1\t472481 36568\n",
      "37\t424417\t11.2\t1\t394152 30265\n",
      "38\t486419\t11.2\t1\t450966 35453\n",
      "39\t471960\t11.2\t1\t437628 34332\n",
      "40\t471765\t11.2\t1\t439658 32107\n",
      "41\t462077\t11.2\t1\t430652 31425\n",
      "42\t472140\t11.2\t1\t440570 31570\n",
      "43\t465706\t11.2\t1\t434911 30795\n",
      "44\t464453\t11.2\t1\t433263 31190\n",
      "45\t469081\t11.2\t1\t437985 31096\n",
      "46\t459298\t11.2\t1\t429257 30041\n",
      "47\t474981\t11.2\t1\t443732 31249\n",
      "48\t469209\t11.2\t1\t439383 29826\n",
      "49\t473557\t11.2\t1\t442818 30739\n",
      "50\t470041\t11.2\t1\t440425 29616\n",
      "51\t473783\t11.2\t1\t443509 30274\n",
      "52\t474757\t11.2\t1\t444462 30295\n",
      "53\t475307\t11.2\t1\t445128 30179\n",
      "54\t472071\t11.2\t1\t442640 29431\n",
      "55\t475395\t11.2\t1\t445582 29813\n",
      "56\t475724\t11.2\t1\t445975 29749\n",
      "57\t476269\t11.2\t1\t446159 30110\n",
      "58\t476959\t11.2\t1\t447093 29866\n",
      "59\t478099\t11.2\t1\t448834 29265\n",
      "60\t478576\t11.2\t1\t449612 28964\n",
      "61\t479282\t11.2\t1\t450016 29266\n",
      "62\t481457\t11.2\t1\t452783 28674\n",
      "63\t478247\t11.2\t1\t449894 28353\n",
      "64\t478841\t11.2\t1\t450466 28375\n",
      "65\t478549\t11.2\t1\t450611 27938\n",
      "66\t479979\t11.2\t1\t451904 28075\n",
      "67\t481071\t11.2\t1\t453308 27763\n",
      "68\t482818\t11.2\t1\t455896 26922\n",
      "69\t484993\t11.2\t1\t457915 27078\n",
      "70\t486889\t11.2\t1\t460560 26329\n",
      "71\t487578\t11.2\t1\t461120 26458\n",
      "72\t485738\t11.2\t1\t459656 26082\n",
      "73\t488282\t11.2\t1\t461838 26444\n",
      "74\t487590\t11.2\t1\t461697 25893\n",
      "75\t487303\t11.2\t1\t461346 25957\n",
      "76\t486311\t11.2\t1\t460232 26079\n",
      "77\t488979\t11.2\t1\t463045 25934\n",
      "78\t490213\t11.2\t1\t464154 26059\n",
      "79\t492178\t11.2\t1\t466091 26087\n",
      "80\t492827\t11.2\t1\t466797 26030\n",
      "81\t492482\t11.2\t1\t467029 25453\n",
      "82\t491966\t11.2\t1\t466587 25379\n",
      "83\t493081\t11.2\t1\t467963 25118\n",
      "84\t491190\t11.2\t1\t466462 24728\n",
      "85\t492594\t11.2\t1\t468151 24443\n",
      "86\t490583\t11.2\t1\t465975 24608\n",
      "87\t493010\t11.2\t1\t469115 23895\n",
      "88\t493945\t11.2\t1\t470101 23844\n",
      "89\t497734\t11.2\t1\t473992 23742\n",
      "90\t501831\t11.2\t1\t478080 23751\n",
      "91\t503337\t11.2\t1\t479396 23941\n",
      "92\t501925\t11.2\t1\t478668 23257\n",
      "93\t501448\t11.2\t1\t478337 23111\n",
      "94\t501486\t11.2\t1\t478357 23129\n",
      "95\t502017\t11.2\t1\t478999 23018\n",
      "96\t500520\t11.2\t1\t477575 22945\n",
      "97\t499635\t11.2\t1\t477000 22635\n",
      "98\t501649\t11.2\t1\t479271 22378\n",
      "99\t505010\t11.2\t1\t482347 22663\n",
      "100\t509609\t11.2\t1\t486981 22628\n",
      "101\t511840\t11.2\t1\t489583 22257\n",
      "102\t511708\t11.2\t1\t489794 21914\n",
      "103\t510594\t11.2\t1\t488660 21934\n",
      "104\t507823\t11.2\t1\t486170 21653\n",
      "105\t508658\t11.2\t1\t486649 22009\n",
      "106\t507271\t11.2\t1\t485790 21481\n",
      "107\t509055\t11.2\t1\t487589 21466\n",
      "108\t510254\t11.2\t1\t489106 21148\n",
      "109\t514493\t11.2\t1\t492907 21586\n",
      "110\t520231\t11.2\t1\t498699 21532\n",
      "111\t522522\t11.2\t1\t500937 21585\n",
      "112\t523991\t11.2\t1\t502802 21189\n",
      "113\t523643\t11.2\t1\t502433 21210\n",
      "114\t520343\t11.2\t1\t499062 21281\n",
      "115\t517356\t11.2\t1\t496367 20989\n",
      "116\t517419\t11.2\t1\t496877 20542\n",
      "117\t516465\t11.2\t1\t496247 20218\n",
      "118\t516238\t11.2\t1\t496476 19762\n",
      "119\t518995\t11.2\t1\t498684 20311\n",
      "120\t523000\t11.2\t1\t502330 20670\n",
      "121\t529423\t11.2\t1\t508680 20743\n",
      "122\t533679\t11.2\t1\t513238 20441\n",
      "123\t532550\t11.2\t1\t513115 19435\n",
      "124\t529611\t11.2\t1\t510522 19089\n",
      "125\t525244\t11.2\t1\t505674 19570\n",
      "126\t524042\t11.2\t1\t505297 18745\n",
      "127\t523692\t11.2\t1\t505543 18149\n",
      "128\t524364\t11.2\t1\t506458 17906\n",
      "129\t527599\t11.2\t1\t509650 17949\n",
      "130\t535228\t11.2\t1\t517692 17536\n",
      "131\t539486\t11.2\t1\t521571 17915\n",
      "132\t540903\t11.2\t1\t523267 17636\n",
      "133\t538194\t11.2\t1\t521637 16557\n",
      "134\t518674\t11.2\t1\t502197 16477\n",
      "135\t510992\t11.2\t1\t495319 15673\n",
      "136\t498733\t11.2\t1\t483585 15148\n",
      "137\t485446\t11.2\t1\t470312 15134\n",
      "138\t466869\t11.2\t1\t452108 14761\n",
      "139\t468212\t11.2\t1\t453288 14924\n",
      "140\t490235\t11.2\t1\t475108 15127\n",
      "141\t508536\t11.2\t1\t493732 14804\n",
      "142\t459703\t11.2\t1\t446002 13701\n",
      "143\t463847\t11.2\t1\t449849 13998\n",
      "144\t504000\t11.2\t1\t488925 15075\n",
      "145\t519512\t11.2\t1\t504195 15317\n",
      "146\t567962\t11.2\t1\t551637 16325\n",
      "147\t502594\t11.2\t1\t488956 13638\n",
      "148\t499738\t11.2\t1\t487156 12582\n",
      "149\t408142\t11.2\t1\t397294 10848\n",
      "150\t360984\t11.2\t1\t350911 10073\n",
      "151\t96862\t11.2\t1\t86400 10462\n",
      "\n",
      "\n",
      "RUN STATISTICS FOR INPUT FILE: /home/sam/data/geoduck_illumina/AD002_S9_L001_R2_001.fastq.gz\n",
      "=============================================\n",
      "187796434 sequences processed in total\n",
      "The length threshold of paired-end sequences gets evaluated later on (in the validation step)\n",
      "\n",
      "Validate paired-end files AD002_S9_L001_R1_001_trimmed.fq.gz and AD002_S9_L001_R2_001_trimmed.fq.gz\n",
      "file_1: AD002_S9_L001_R1_001_trimmed.fq.gz, file_2: AD002_S9_L001_R2_001_trimmed.fq.gz\n",
      "\n",
      "\n",
      ">>>>> Now validing the length of the 2 paired-end infiles: AD002_S9_L001_R1_001_trimmed.fq.gz and AD002_S9_L001_R2_001_trimmed.fq.gz <<<<<\n",
      "Writing validated paired-end read 1 reads to AD002_S9_L001_R1_001_val_1.fq.gz\n",
      "Writing validated paired-end read 2 reads to AD002_S9_L001_R2_001_val_2.fq.gz\n",
      "\n",
      "Total number of sequences analysed: 187796434\n",
      "\n",
      "Number of sequence pairs removed because at least one read was shorter than the length cutoff (20 bp): 21276885 (11.33%)\n",
      "\n",
      "Deleting both intermediate output files AD002_S9_L001_R1_001_trimmed.fq.gz and AD002_S9_L001_R2_001_trimmed.fq.gz\n",
      "\n",
      "====================================================================================================\n",
      "\n",
      "Writing report to '/home/sam/data/geoduck_illumina/trimmed/AD002_S9_L002_R1_001.fastq.gz_trimming_report.txt'\n",
      "\n",
      "SUMMARISING RUN PARAMETERS\n",
      "==========================\n",
      "Input filename: /home/sam/data/geoduck_illumina/AD002_S9_L002_R1_001.fastq.gz\n",
      "Trimming mode: paired-end\n",
      "Trim Galore version: 0.4.4\n",
      "Cutadapt version: 1.9.1\n",
      "Quality Phred score cutoff: 20\n",
      "Quality encoding type selected: ASCII+33\n",
      "Adapter sequence: 'CTGTCTCTTATA' (Nextera Transposase sequence; auto-detected)\n",
      "Maximum trimming error rate: 0.1 (default)\n",
      "Minimum required adapter overlap (stringency): 1 bp\n",
      "Minimum required sequence length for both reads before a sequence pair gets removed: 20 bp\n",
      "Output file(s) will be GZIP compressed\n",
      "\n",
      "Writing final adapter and quality trimmed output to AD002_S9_L002_R1_001_trimmed.fq.gz\n",
      "\n",
      "\n",
      "  >>> Now performing quality (cutoff 20) and adapter trimming in a single pass for the adapter sequence: 'CTGTCTCTTATA' from file /home/sam/data/geoduck_illumina/AD002_S9_L002_R1_001.fastq.gz <<< \n",
      "10000000 sequences processed\n",
      "20000000 sequences processed\n",
      "30000000 sequences processed\n",
      "40000000 sequences processed\n",
      "50000000 sequences processed\n",
      "60000000 sequences processed\n",
      "70000000 sequences processed\n",
      "80000000 sequences processed\n",
      "90000000 sequences processed\n",
      "100000000 sequences processed\n",
      "110000000 sequences processed\n",
      "120000000 sequences processed\n",
      "130000000 sequences processed\n",
      "140000000 sequences processed\n",
      "150000000 sequences processed\n",
      "160000000 sequences processed\n",
      "170000000 sequences processed\n",
      "180000000 sequences processed\n",
      "190000000 sequences processed\n",
      "200000000 sequences processed\n",
      "210000000 sequences processed\n",
      "220000000 sequences processed\n",
      "230000000 sequences processed\n",
      "This is cutadapt 1.9.1 with Python 2.7.12\n",
      "Command line parameters: -f fastq -e 0.1 -q 20 -O 1 -a CTGTCTCTTATA /home/sam/data/geoduck_illumina/AD002_S9_L002_R1_001.fastq.gz\n",
      "Trimming 1 adapter with at most 10.0% errors in single-end mode ...\n",
      "Finished in 5604.32 s (24 us/read; 2.54 M reads/minute).\n",
      "\n",
      "=== Summary ===\n",
      "\n",
      "Total reads processed:             236,917,763\n",
      "Reads with adapters:               130,979,298 (55.3%)\n",
      "Reads written (passing filters):   236,917,763 (100.0%)\n",
      "\n",
      "Total basepairs processed: 35,774,582,213 bp\n",
      "Quality-trimmed:              88,859,002 bp (0.2%)\n",
      "Total written (filtered):  28,048,913,479 bp (78.4%)\n",
      "\n",
      "=== Adapter 1 ===\n",
      "\n",
      "Sequence: CTGTCTCTTATA; Type: regular 3'; Length: 12; Trimmed: 130979298 times.\n",
      "\n",
      "No. of allowed errors:\n",
      "0-9 bp: 0; 10-12 bp: 1\n",
      "\n",
      "Bases preceding removed adapters:\n",
      "  A: 18.0%\n",
      "  C: 34.6%\n",
      "  G: 20.6%\n",
      "  T: 26.7%\n",
      "  none/other: 0.1%\n",
      "\n",
      "Overview of removed sequences\n",
      "length\tcount\texpect\tmax.err\terror counts\n",
      "1\t26072283\t59229440.8\t0\t26072283\n",
      "2\t7754424\t14807360.2\t0\t7754424\n",
      "3\t2615393\t3701840.0\t0\t2615393\n",
      "4\t1158233\t925460.0\t0\t1158233\n",
      "5\t687215\t231365.0\t0\t687215\n",
      "6\t571963\t57841.3\t0\t571963\n",
      "7\t562880\t14460.3\t0\t562880\n",
      "8\t589858\t3615.1\t0\t589858\n",
      "9\t515603\t903.8\t0\t510820 4783\n",
      "10\t558528\t225.9\t1\t533113 25415\n",
      "11\t579007\t56.5\t1\t556144 22863\n",
      "12\t550249\t14.1\t1\t527149 23100\n",
      "13\t569428\t14.1\t1\t547481 21947\n",
      "14\t530109\t14.1\t1\t510052 20057\n",
      "15\t609273\t14.1\t1\t583556 25717\n",
      "16\t669393\t14.1\t1\t643174 26219\n",
      "17\t420062\t14.1\t1\t404987 15075\n",
      "18\t572732\t14.1\t1\t552523 20209\n",
      "19\t584470\t14.1\t1\t565824 18646\n",
      "20\t527803\t14.1\t1\t508276 19527\n",
      "21\t579340\t14.1\t1\t559307 20033\n",
      "22\t634357\t14.1\t1\t611402 22955\n",
      "23\t476587\t14.1\t1\t459153 17434\n",
      "24\t590952\t14.1\t1\t569969 20983\n",
      "25\t523353\t14.1\t1\t506915 16438\n",
      "26\t603228\t14.1\t1\t578877 24351\n",
      "27\t574554\t14.1\t1\t555345 19209\n",
      "28\t549718\t14.1\t1\t531929 17789\n",
      "29\t564848\t14.1\t1\t547635 17213\n",
      "30\t603054\t14.1\t1\t583440 19614\n",
      "31\t572748\t14.1\t1\t553819 18929\n",
      "32\t561827\t14.1\t1\t544923 16904\n",
      "33\t555924\t14.1\t1\t537944 17980\n",
      "34\t567520\t14.1\t1\t549405 18115\n",
      "35\t570660\t14.1\t1\t551916 18744\n",
      "36\t606452\t14.1\t1\t588048 18404\n",
      "37\t551655\t14.1\t1\t533363 18292\n",
      "38\t601494\t14.1\t1\t582405 19089\n",
      "39\t589056\t14.1\t1\t570060 18996\n",
      "40\t587735\t14.1\t1\t569593 18142\n",
      "41\t579628\t14.1\t1\t561509 18119\n",
      "42\t586109\t14.1\t1\t568279 17830\n",
      "43\t584440\t14.1\t1\t566783 17657\n",
      "44\t584969\t14.1\t1\t566818 18151\n",
      "45\t586075\t14.1\t1\t568309 17766\n",
      "46\t582480\t14.1\t1\t564331 18149\n",
      "47\t592528\t14.1\t1\t574820 17708\n",
      "48\t590483\t14.1\t1\t572804 17679\n",
      "49\t593997\t14.1\t1\t576332 17665\n",
      "50\t593229\t14.1\t1\t575839 17390\n",
      "51\t596082\t14.1\t1\t578259 17823\n",
      "52\t595558\t14.1\t1\t578372 17186\n",
      "53\t598320\t14.1\t1\t580967 17353\n",
      "54\t595790\t14.1\t1\t578760 17030\n",
      "55\t597323\t14.1\t1\t580229 17094\n",
      "56\t599762\t14.1\t1\t582868 16894\n",
      "57\t602795\t14.1\t1\t585705 17090\n",
      "58\t604365\t14.1\t1\t587139 17226\n",
      "59\t605710\t14.1\t1\t588517 17193\n",
      "60\t606801\t14.1\t1\t588958 17843\n",
      "61\t609421\t14.1\t1\t591688 17733\n",
      "62\t608612\t14.1\t1\t591172 17440\n",
      "63\t608648\t14.1\t1\t591196 17452\n",
      "64\t608196\t14.1\t1\t590905 17291\n",
      "65\t609725\t14.1\t1\t592600 17125\n",
      "66\t609413\t14.1\t1\t592260 17153\n",
      "67\t612898\t14.1\t1\t595333 17565\n",
      "68\t616938\t14.1\t1\t599377 17561\n",
      "69\t617724\t14.1\t1\t600854 16870\n",
      "70\t619077\t14.1\t1\t601734 17343\n",
      "71\t624176\t14.1\t1\t607084 17092\n",
      "72\t622177\t14.1\t1\t605239 16938\n",
      "73\t622519\t14.1\t1\t605669 16850\n",
      "74\t623572\t14.1\t1\t606594 16978\n",
      "75\t624251\t14.1\t1\t607245 17006\n",
      "76\t625248\t14.1\t1\t608642 16606\n",
      "77\t627801\t14.1\t1\t611028 16773\n",
      "78\t630419\t14.1\t1\t613444 16975\n",
      "79\t634153\t14.1\t1\t617371 16782\n",
      "80\t635881\t14.1\t1\t618818 17063\n",
      "81\t638379\t14.1\t1\t621561 16818\n",
      "82\t637956\t14.1\t1\t621092 16864\n",
      "83\t640101\t14.1\t1\t623416 16685\n",
      "84\t637670\t14.1\t1\t621708 15962\n",
      "85\t639728\t14.1\t1\t623623 16105\n",
      "86\t640254\t14.1\t1\t624253 16001\n",
      "87\t643421\t14.1\t1\t627535 15886\n",
      "88\t644225\t14.1\t1\t628545 15680\n",
      "89\t651062\t14.1\t1\t635406 15656\n",
      "90\t652219\t14.1\t1\t636697 15522\n",
      "91\t658193\t14.1\t1\t642736 15457\n",
      "92\t659828\t14.1\t1\t644294 15534\n",
      "93\t656037\t14.1\t1\t640796 15241\n",
      "94\t658274\t14.1\t1\t642974 15300\n",
      "95\t658259\t14.1\t1\t643337 14922\n",
      "96\t656656\t14.1\t1\t641988 14668\n",
      "97\t660650\t14.1\t1\t645915 14735\n",
      "98\t662431\t14.1\t1\t647874 14557\n",
      "99\t668276\t14.1\t1\t653707 14569\n",
      "100\t671359\t14.1\t1\t656857 14502\n",
      "101\t677681\t14.1\t1\t663080 14601\n",
      "102\t679269\t14.1\t1\t664931 14338\n",
      "103\t679433\t14.1\t1\t665364 14069\n",
      "104\t679108\t14.1\t1\t665031 14077\n",
      "105\t679269\t14.1\t1\t665256 14013\n",
      "106\t679618\t14.1\t1\t665694 13924\n",
      "107\t682355\t14.1\t1\t668119 14236\n",
      "108\t684217\t14.1\t1\t670229 13988\n",
      "109\t691454\t14.1\t1\t677286 14168\n",
      "110\t697612\t14.1\t1\t683244 14368\n",
      "111\t703722\t14.1\t1\t689434 14288\n",
      "112\t707554\t14.1\t1\t693554 14000\n",
      "113\t709388\t14.1\t1\t695376 14012\n",
      "114\t705346\t14.1\t1\t691737 13609\n",
      "115\t706472\t14.1\t1\t692603 13869\n",
      "116\t703507\t14.1\t1\t689984 13523\n",
      "117\t703149\t14.1\t1\t689920 13229\n",
      "118\t705069\t14.1\t1\t691583 13486\n",
      "119\t709943\t14.1\t1\t696568 13375\n",
      "120\t719130\t14.1\t1\t705483 13647\n",
      "121\t726635\t14.1\t1\t712989 13646\n",
      "122\t730601\t14.1\t1\t716906 13695\n",
      "123\t733595\t14.1\t1\t720128 13467\n",
      "124\t730327\t14.1\t1\t716993 13334\n",
      "125\t726890\t14.1\t1\t713700 13190\n",
      "126\t726378\t14.1\t1\t713447 12931\n",
      "127\t726914\t14.1\t1\t714379 12535\n",
      "128\t728880\t14.1\t1\t716716 12164\n",
      "129\t733607\t14.1\t1\t722057 11550\n",
      "130\t743886\t14.1\t1\t732788 11098\n",
      "131\t750672\t14.1\t1\t739882 10790\n",
      "132\t755266\t14.1\t1\t745280 9986\n",
      "133\t754441\t14.1\t1\t744908 9533\n",
      "134\t725363\t14.1\t1\t716616 8747\n",
      "135\t720274\t14.1\t1\t712044 8230\n",
      "136\t704731\t14.1\t1\t697012 7719\n",
      "137\t694080\t14.1\t1\t686893 7187\n",
      "138\t669855\t14.1\t1\t663062 6793\n",
      "139\t671478\t14.1\t1\t664565 6913\n",
      "140\t701689\t14.1\t1\t694670 7019\n",
      "141\t738780\t14.1\t1\t731568 7212\n",
      "142\t658836\t14.1\t1\t652082 6754\n",
      "143\t659728\t14.1\t1\t653259 6469\n",
      "144\t714783\t14.1\t1\t708059 6724\n",
      "145\t730774\t14.1\t1\t723961 6813\n",
      "146\t825298\t14.1\t1\t817454 7844\n",
      "147\t722526\t14.1\t1\t715234 7292\n",
      "148\t735852\t14.1\t1\t728488 7364\n",
      "149\t635235\t14.1\t1\t627859 7376\n",
      "150\t572343\t14.1\t1\t564528 7815\n",
      "151\t160046\t14.1\t1\t147090 12956\n",
      "\n",
      "\n",
      "RUN STATISTICS FOR INPUT FILE: /home/sam/data/geoduck_illumina/AD002_S9_L002_R1_001.fastq.gz\n",
      "=============================================\n",
      "236917763 sequences processed in total\n",
      "The length threshold of paired-end sequences gets evaluated later on (in the validation step)\n",
      "\n",
      "Writing report to '/home/sam/data/geoduck_illumina/trimmed/AD002_S9_L002_R2_001.fastq.gz_trimming_report.txt'\n",
      "\n",
      "SUMMARISING RUN PARAMETERS\n",
      "==========================\n",
      "Input filename: /home/sam/data/geoduck_illumina/AD002_S9_L002_R2_001.fastq.gz\n",
      "Trimming mode: paired-end\n",
      "Trim Galore version: 0.4.4\n",
      "Cutadapt version: 1.9.1\n",
      "Quality Phred score cutoff: 20\n",
      "Quality encoding type selected: ASCII+33\n",
      "Adapter sequence: 'CTGTCTCTTATA' (Nextera Transposase sequence; auto-detected)\n",
      "Maximum trimming error rate: 0.1 (default)\n",
      "Minimum required adapter overlap (stringency): 1 bp\n",
      "Minimum required sequence length for both reads before a sequence pair gets removed: 20 bp\n",
      "Output file(s) will be GZIP compressed\n",
      "\n",
      "Writing final adapter and quality trimmed output to AD002_S9_L002_R2_001_trimmed.fq.gz\n",
      "\n",
      "\n",
      "  >>> Now performing quality (cutoff 20) and adapter trimming in a single pass for the adapter sequence: 'CTGTCTCTTATA' from file /home/sam/data/geoduck_illumina/AD002_S9_L002_R2_001.fastq.gz <<< \n",
      "10000000 sequences processed\n",
      "20000000 sequences processed\n",
      "30000000 sequences processed\n",
      "40000000 sequences processed\n",
      "50000000 sequences processed\n",
      "60000000 sequences processed\n",
      "70000000 sequences processed\n",
      "80000000 sequences processed\n",
      "90000000 sequences processed\n",
      "100000000 sequences processed\n",
      "110000000 sequences processed\n",
      "120000000 sequences processed\n",
      "130000000 sequences processed\n",
      "140000000 sequences processed\n",
      "150000000 sequences processed\n",
      "160000000 sequences processed\n",
      "170000000 sequences processed\n",
      "180000000 sequences processed\n",
      "190000000 sequences processed\n",
      "200000000 sequences processed\n",
      "210000000 sequences processed\n",
      "220000000 sequences processed\n",
      "230000000 sequences processed\n",
      "This is cutadapt 1.9.1 with Python 2.7.12\n",
      "Command line parameters: -f fastq -e 0.1 -q 20 -O 1 -a CTGTCTCTTATA /home/sam/data/geoduck_illumina/AD002_S9_L002_R2_001.fastq.gz\n",
      "Trimming 1 adapter with at most 10.0% errors in single-end mode ...\n",
      "Finished in 5672.61 s (24 us/read; 2.51 M reads/minute).\n",
      "\n",
      "=== Summary ===\n",
      "\n",
      "Total reads processed:             236,917,763\n",
      "Reads with adapters:               119,760,324 (50.5%)\n",
      "Reads written (passing filters):   236,917,763 (100.0%)\n",
      "\n",
      "Total basepairs processed: 35,774,582,213 bp\n",
      "Quality-trimmed:             122,766,404 bp (0.3%)\n",
      "Total written (filtered):  29,261,997,335 bp (81.8%)\n",
      "\n",
      "=== Adapter 1 ===\n",
      "\n",
      "Sequence: CTGTCTCTTATA; Type: regular 3'; Length: 12; Trimmed: 119760324 times.\n",
      "\n",
      "No. of allowed errors:\n",
      "0-9 bp: 0; 10-12 bp: 1\n",
      "\n",
      "Bases preceding removed adapters:\n",
      "  A: 18.9%\n",
      "  C: 33.4%\n",
      "  G: 20.7%\n",
      "  T: 26.9%\n",
      "  none/other: 0.1%\n",
      "\n",
      "Overview of removed sequences\n",
      "length\tcount\texpect\tmax.err\terror counts\n",
      "1\t27184123\t59229440.8\t0\t27184123\n",
      "2\t8041586\t14807360.2\t0\t8041586\n",
      "3\t2666576\t3701840.0\t0\t2666576\n",
      "4\t1168313\t925460.0\t0\t1168313\n",
      "5\t667985\t231365.0\t0\t667985\n",
      "6\t538222\t57841.3\t0\t538222\n",
      "7\t530151\t14460.3\t0\t530151\n",
      "8\t564228\t3615.1\t0\t564228\n",
      "9\t475542\t903.8\t0\t470308 5234\n",
      "10\t522966\t225.9\t1\t487785 35181\n",
      "11\t553066\t56.5\t1\t517586 35480\n",
      "12\t498888\t14.1\t1\t466969 31919\n",
      "13\t543535\t14.1\t1\t507568 35967\n",
      "14\t480829\t14.1\t1\t452490 28339\n",
      "15\t587923\t14.1\t1\t546194 41729\n",
      "16\t601689\t14.1\t1\t565659 36030\n",
      "17\t400438\t14.1\t1\t377348 23090\n",
      "18\t539995\t14.1\t1\t506401 33594\n",
      "19\t553585\t14.1\t1\t523066 30519\n",
      "20\t480346\t14.1\t1\t454275 26071\n",
      "21\t543014\t14.1\t1\t507807 35207\n",
      "22\t582207\t14.1\t1\t547563 34644\n",
      "23\t438809\t14.1\t1\t413367 25442\n",
      "24\t551698\t14.1\t1\t516672 35026\n",
      "25\t484624\t14.1\t1\t459539 25085\n",
      "26\t573140\t14.1\t1\t532542 40598\n",
      "27\t543672\t14.1\t1\t512212 31460\n",
      "28\t502380\t14.1\t1\t474888 27492\n",
      "29\t525406\t14.1\t1\t496963 28443\n",
      "30\t555686\t14.1\t1\t524310 31376\n",
      "31\t507209\t14.1\t1\t481126 26083\n",
      "32\t532674\t14.1\t1\t500832 31842\n",
      "33\t506897\t14.1\t1\t480620 26277\n",
      "34\t538904\t14.1\t1\t506428 32476\n",
      "35\t510614\t14.1\t1\t483465 27149\n",
      "36\t581646\t14.1\t1\t550317 31329\n",
      "37\t486866\t14.1\t1\t462566 24300\n",
      "38\t562896\t14.1\t1\t531710 31186\n",
      "39\t546810\t14.1\t1\t515949 30861\n",
      "40\t542171\t14.1\t1\t514164 28007\n",
      "41\t531228\t14.1\t1\t503089 28139\n",
      "42\t541915\t14.1\t1\t514053 27862\n",
      "43\t534303\t14.1\t1\t507549 26754\n",
      "44\t533769\t14.1\t1\t506552 27217\n",
      "45\t538343\t14.1\t1\t510751 27592\n",
      "46\t526920\t14.1\t1\t500509 26411\n",
      "47\t546871\t14.1\t1\t518567 28304\n",
      "48\t537810\t14.1\t1\t511582 26228\n",
      "49\t544217\t14.1\t1\t516812 27405\n",
      "50\t538442\t14.1\t1\t512132 26310\n",
      "51\t543023\t14.1\t1\t516151 26872\n",
      "52\t542906\t14.1\t1\t516336 26570\n",
      "53\t543461\t14.1\t1\t516903 26558\n",
      "54\t540855\t14.1\t1\t514609 26246\n",
      "55\t542847\t14.1\t1\t516385 26462\n",
      "56\t544020\t14.1\t1\t517541 26479\n",
      "57\t544660\t14.1\t1\t518095 26565\n",
      "58\t545208\t14.1\t1\t518699 26509\n",
      "59\t547713\t14.1\t1\t521458 26255\n",
      "60\t546919\t14.1\t1\t520881 26038\n",
      "61\t546617\t14.1\t1\t520845 25772\n",
      "62\t546592\t14.1\t1\t521095 25497\n",
      "63\t544423\t14.1\t1\t518758 25665\n",
      "64\t547012\t14.1\t1\t521324 25688\n",
      "65\t545326\t14.1\t1\t519438 25888\n",
      "66\t545967\t14.1\t1\t520268 25699\n",
      "67\t548645\t14.1\t1\t523426 25219\n",
      "68\t550206\t14.1\t1\t525275 24931\n",
      "69\t552617\t14.1\t1\t528189 24428\n",
      "70\t553674\t14.1\t1\t529313 24361\n",
      "71\t555862\t14.1\t1\t531442 24420\n",
      "72\t552489\t14.1\t1\t528475 24014\n",
      "73\t552862\t14.1\t1\t528692 24170\n",
      "74\t553292\t14.1\t1\t529511 23781\n",
      "75\t553465\t14.1\t1\t529439 24026\n",
      "76\t553976\t14.1\t1\t530186 23790\n",
      "77\t553197\t14.1\t1\t529355 23842\n",
      "78\t555113\t14.1\t1\t531602 23511\n",
      "79\t558745\t14.1\t1\t535383 23362\n",
      "80\t558122\t14.1\t1\t534714 23408\n",
      "81\t559044\t14.1\t1\t535496 23548\n",
      "82\t557981\t14.1\t1\t534901 23080\n",
      "83\t557826\t14.1\t1\t534758 23068\n",
      "84\t555966\t14.1\t1\t533296 22670\n",
      "85\t554929\t14.1\t1\t532240 22689\n",
      "86\t554125\t14.1\t1\t531736 22389\n",
      "87\t556244\t14.1\t1\t534083 22161\n",
      "88\t557437\t14.1\t1\t535075 22362\n",
      "89\t563161\t14.1\t1\t540630 22531\n",
      "90\t564419\t14.1\t1\t542124 22295\n",
      "91\t568529\t14.1\t1\t546368 22161\n",
      "92\t566760\t14.1\t1\t544580 22180\n",
      "93\t566536\t14.1\t1\t544341 22195\n",
      "94\t565746\t14.1\t1\t543984 21762\n",
      "95\t564054\t14.1\t1\t542487 21567\n",
      "96\t563055\t14.1\t1\t541337 21718\n",
      "97\t563442\t14.1\t1\t541848 21594\n",
      "98\t564578\t14.1\t1\t542893 21685\n",
      "99\t569033\t14.1\t1\t547266 21767\n",
      "100\t571529\t14.1\t1\t550147 21382\n",
      "101\t572792\t14.1\t1\t551940 20852\n",
      "102\t574962\t14.1\t1\t554356 20606\n",
      "103\t574068\t14.1\t1\t553643 20425\n",
      "104\t570390\t14.1\t1\t550099 20291\n",
      "105\t570644\t14.1\t1\t550442 20202\n",
      "106\t570156\t14.1\t1\t550276 19880\n",
      "107\t568786\t14.1\t1\t549037 19749\n",
      "108\t571635\t14.1\t1\t551875 19760\n",
      "109\t576550\t14.1\t1\t556970 19580\n",
      "110\t580922\t14.1\t1\t561141 19781\n",
      "111\t583973\t14.1\t1\t564217 19756\n",
      "112\t586948\t14.1\t1\t567436 19512\n",
      "113\t586672\t14.1\t1\t567094 19578\n",
      "114\t579669\t14.1\t1\t560557 19112\n",
      "115\t578056\t14.1\t1\t559029 19027\n",
      "116\t576319\t14.1\t1\t557223 19096\n",
      "117\t575447\t14.1\t1\t556765 18682\n",
      "118\t575524\t14.1\t1\t557025 18499\n",
      "119\t577752\t14.1\t1\t558883 18869\n",
      "120\t584731\t14.1\t1\t565703 19028\n",
      "121\t592424\t14.1\t1\t573182 19242\n",
      "122\t593492\t14.1\t1\t574808 18684\n",
      "123\t595523\t14.1\t1\t577343 18180\n",
      "124\t588624\t14.1\t1\t570955 17669\n",
      "125\t586601\t14.1\t1\t568934 17667\n",
      "126\t583242\t14.1\t1\t565959 17283\n",
      "127\t583398\t14.1\t1\t566484 16914\n",
      "128\t585615\t14.1\t1\t568906 16709\n",
      "129\t586789\t14.1\t1\t570207 16582\n",
      "130\t595337\t14.1\t1\t579228 16109\n",
      "131\t599699\t14.1\t1\t583358 16341\n",
      "132\t602201\t14.1\t1\t586199 16002\n",
      "133\t598589\t14.1\t1\t583429 15160\n",
      "134\t576993\t14.1\t1\t562472 14521\n",
      "135\t568948\t14.1\t1\t555235 13713\n",
      "136\t553101\t14.1\t1\t540371 12730\n",
      "137\t540774\t14.1\t1\t528222 12552\n",
      "138\t517048\t14.1\t1\t505103 11945\n",
      "139\t519527\t14.1\t1\t507776 11751\n",
      "140\t542939\t14.1\t1\t530762 12177\n",
      "141\t560987\t14.1\t1\t549293 11694\n",
      "142\t508971\t14.1\t1\t497610 11361\n",
      "143\t511483\t14.1\t1\t500236 11247\n",
      "144\t555570\t14.1\t1\t543422 12148\n",
      "145\t575167\t14.1\t1\t562620 12547\n",
      "146\t629380\t14.1\t1\t616367 13013\n",
      "147\t552383\t14.1\t1\t540777 11606\n",
      "148\t552853\t14.1\t1\t541802 11051\n",
      "149\t451659\t14.1\t1\t442111 9548\n",
      "150\t399813\t14.1\t1\t390507 9306\n",
      "151\t109863\t14.1\t1\t98224 11639\n",
      "\n",
      "\n",
      "RUN STATISTICS FOR INPUT FILE: /home/sam/data/geoduck_illumina/AD002_S9_L002_R2_001.fastq.gz\n",
      "=============================================\n",
      "236917763 sequences processed in total\n",
      "The length threshold of paired-end sequences gets evaluated later on (in the validation step)\n",
      "\n",
      "Validate paired-end files AD002_S9_L002_R1_001_trimmed.fq.gz and AD002_S9_L002_R2_001_trimmed.fq.gz\n",
      "file_1: AD002_S9_L002_R1_001_trimmed.fq.gz, file_2: AD002_S9_L002_R2_001_trimmed.fq.gz\n",
      "\n",
      "\n",
      ">>>>> Now validing the length of the 2 paired-end infiles: AD002_S9_L002_R1_001_trimmed.fq.gz and AD002_S9_L002_R2_001_trimmed.fq.gz <<<<<\n",
      "Writing validated paired-end read 1 reads to AD002_S9_L002_R1_001_val_1.fq.gz\n",
      "Writing validated paired-end read 2 reads to AD002_S9_L002_R2_001_val_2.fq.gz\n",
      "\n",
      "Total number of sequences analysed: 236917763\n",
      "\n",
      "Number of sequence pairs removed because at least one read was shorter than the length cutoff (20 bp): 24346749 (10.28%)\n",
      "\n",
      "Deleting both intermediate output files AD002_S9_L002_R1_001_trimmed.fq.gz and AD002_S9_L002_R2_001_trimmed.fq.gz\n",
      "\n",
      "====================================================================================================\n",
      "\n",
      "Writing report to '/home/sam/data/geoduck_illumina/trimmed/NR005_S4_L001_R1_001.fastq.gz_trimming_report.txt'\n",
      "\n",
      "SUMMARISING RUN PARAMETERS\n",
      "==========================\n",
      "Input filename: /home/sam/data/geoduck_illumina/NR005_S4_L001_R1_001.fastq.gz\n",
      "Trimming mode: paired-end\n",
      "Trim Galore version: 0.4.4\n",
      "Cutadapt version: 1.9.1\n",
      "Quality Phred score cutoff: 20\n",
      "Quality encoding type selected: ASCII+33\n",
      "Adapter sequence: 'CTGTCTCTTATA' (Nextera Transposase sequence; auto-detected)\n",
      "Maximum trimming error rate: 0.1 (default)\n",
      "Minimum required adapter overlap (stringency): 1 bp\n",
      "Minimum required sequence length for both reads before a sequence pair gets removed: 20 bp\n",
      "Output file(s) will be GZIP compressed\n",
      "\n",
      "Writing final adapter and quality trimmed output to NR005_S4_L001_R1_001_trimmed.fq.gz\n",
      "\n",
      "\n",
      "  >>> Now performing quality (cutoff 20) and adapter trimming in a single pass for the adapter sequence: 'CTGTCTCTTATA' from file /home/sam/data/geoduck_illumina/NR005_S4_L001_R1_001.fastq.gz <<< \n",
      "This is cutadapt 1.9.1 with Python 2.7.12\n",
      "Command line parameters: -f fastq -e 0.1 -q 20 -O 1 -a CTGTCTCTTATA /home/sam/data/geoduck_illumina/NR005_S4_L001_R1_001.fastq.gz\n",
      "Trimming 1 adapter with at most 10.0% errors in single-end mode ...\n",
      "Finished in 1.14 s (27 us/read; 2.19 M reads/minute).\n",
      "\n",
      "=== Summary ===\n",
      "\n",
      "Total reads processed:                  41,570\n",
      "Reads with adapters:                    12,428 (29.9%)\n",
      "Reads written (passing filters):        41,570 (100.0%)\n",
      "\n",
      "Total basepairs processed:     6,277,070 bp\n",
      "Quality-trimmed:                   7,538 bp (0.1%)\n",
      "Total written (filtered):      6,243,003 bp (99.5%)\n",
      "\n",
      "=== Adapter 1 ===\n",
      "\n",
      "Sequence: CTGTCTCTTATA; Type: regular 3'; Length: 12; Trimmed: 12428 times.\n",
      "\n",
      "No. of allowed errors:\n",
      "0-9 bp: 0; 10-12 bp: 1\n",
      "\n",
      "Bases preceding removed adapters:\n",
      "  A: 31.0%\n",
      "  C: 14.7%\n",
      "  G: 14.9%\n",
      "  T: 39.3%\n",
      "  none/other: 0.0%\n",
      "\n",
      "Overview of removed sequences\n",
      "length\tcount\texpect\tmax.err\terror counts\n",
      "1\t9041\t10392.5\t0\t9041\n",
      "2\t2328\t2598.1\t0\t2328\n",
      "3\t716\t649.5\t0\t716\n",
      "4\t170\t162.4\t0\t170\n",
      "5\t33\t40.6\t0\t33\n",
      "6\t10\t10.1\t0\t10\n",
      "7\t5\t2.5\t0\t5\n",
      "9\t2\t0.2\t0\t0 2\n",
      "10\t2\t0.0\t1\t1 1\n",
      "11\t1\t0.0\t1\t0 1\n",
      "12\t1\t0.0\t1\t1\n",
      "14\t1\t0.0\t1\t1\n",
      "16\t3\t0.0\t1\t2 1\n",
      "19\t1\t0.0\t1\t1\n",
      "21\t1\t0.0\t1\t1\n",
      "23\t1\t0.0\t1\t1\n",
      "24\t1\t0.0\t1\t0 1\n",
      "25\t1\t0.0\t1\t1\n",
      "29\t2\t0.0\t1\t1 1\n",
      "30\t1\t0.0\t1\t0 1\n",
      "31\t3\t0.0\t1\t1 2\n",
      "32\t1\t0.0\t1\t0 1\n",
      "34\t3\t0.0\t1\t1 2\n",
      "36\t3\t0.0\t1\t1 2\n",
      "38\t3\t0.0\t1\t3\n",
      "40\t1\t0.0\t1\t0 1\n",
      "41\t3\t0.0\t1\t1 2\n",
      "42\t1\t0.0\t1\t0 1\n",
      "44\t1\t0.0\t1\t1\n",
      "45\t1\t0.0\t1\t0 1\n",
      "46\t1\t0.0\t1\t0 1\n",
      "47\t1\t0.0\t1\t0 1\n",
      "49\t1\t0.0\t1\t1\n",
      "51\t2\t0.0\t1\t1 1\n",
      "54\t2\t0.0\t1\t1 1\n",
      "58\t1\t0.0\t1\t0 1\n",
      "62\t1\t0.0\t1\t0 1\n",
      "63\t2\t0.0\t1\t0 2\n",
      "65\t3\t0.0\t1\t1 2\n",
      "67\t1\t0.0\t1\t0 1\n",
      "70\t2\t0.0\t1\t2\n",
      "76\t2\t0.0\t1\t2\n",
      "78\t1\t0.0\t1\t1\n",
      "81\t1\t0.0\t1\t1\n",
      "82\t3\t0.0\t1\t2 1\n",
      "86\t1\t0.0\t1\t0 1\n",
      "87\t1\t0.0\t1\t0 1\n",
      "88\t1\t0.0\t1\t0 1\n",
      "89\t3\t0.0\t1\t1 2\n",
      "90\t1\t0.0\t1\t0 1\n",
      "92\t1\t0.0\t1\t1\n",
      "93\t3\t0.0\t1\t2 1\n",
      "94\t2\t0.0\t1\t0 2\n",
      "96\t2\t0.0\t1\t0 2\n",
      "97\t1\t0.0\t1\t1\n",
      "98\t1\t0.0\t1\t0 1\n",
      "99\t1\t0.0\t1\t0 1\n",
      "101\t4\t0.0\t1\t3 1\n",
      "102\t2\t0.0\t1\t1 1\n",
      "103\t2\t0.0\t1\t2\n",
      "105\t1\t0.0\t1\t0 1\n",
      "106\t1\t0.0\t1\t0 1\n",
      "107\t2\t0.0\t1\t0 2\n",
      "108\t1\t0.0\t1\t0 1\n",
      "111\t1\t0.0\t1\t1\n",
      "112\t1\t0.0\t1\t1\n",
      "113\t1\t0.0\t1\t0 1\n",
      "114\t1\t0.0\t1\t0 1\n",
      "116\t1\t0.0\t1\t0 1\n",
      "117\t1\t0.0\t1\t1\n",
      "118\t4\t0.0\t1\t3 1\n",
      "119\t2\t0.0\t1\t2\n",
      "121\t1\t0.0\t1\t1\n",
      "122\t2\t0.0\t1\t0 2\n",
      "124\t1\t0.0\t1\t1\n",
      "127\t1\t0.0\t1\t1\n",
      "128\t1\t0.0\t1\t1\n",
      "130\t1\t0.0\t1\t1\n",
      "131\t1\t0.0\t1\t1\n",
      "132\t1\t0.0\t1\t1\n",
      "135\t2\t0.0\t1\t1 1\n",
      "141\t1\t0.0\t1\t1\n",
      "142\t1\t0.0\t1\t1\n",
      "143\t1\t0.0\t1\t0 1\n",
      "144\t1\t0.0\t1\t1\n",
      "146\t1\t0.0\t1\t0 1\n",
      "147\t1\t0.0\t1\t1\n",
      "148\t1\t0.0\t1\t0 1\n",
      "149\t1\t0.0\t1\t1\n",
      "151\t1\t0.0\t1\t0 1\n",
      "\n",
      "\n",
      "RUN STATISTICS FOR INPUT FILE: /home/sam/data/geoduck_illumina/NR005_S4_L001_R1_001.fastq.gz\n",
      "=============================================\n",
      "41570 sequences processed in total\n",
      "The length threshold of paired-end sequences gets evaluated later on (in the validation step)\n",
      "\n",
      "Writing report to '/home/sam/data/geoduck_illumina/trimmed/NR005_S4_L001_R2_001.fastq.gz_trimming_report.txt'\n",
      "\n",
      "SUMMARISING RUN PARAMETERS\n",
      "==========================\n",
      "Input filename: /home/sam/data/geoduck_illumina/NR005_S4_L001_R2_001.fastq.gz\n",
      "Trimming mode: paired-end\n",
      "Trim Galore version: 0.4.4\n",
      "Cutadapt version: 1.9.1\n",
      "Quality Phred score cutoff: 20\n",
      "Quality encoding type selected: ASCII+33\n",
      "Adapter sequence: 'CTGTCTCTTATA' (Nextera Transposase sequence; auto-detected)\n",
      "Maximum trimming error rate: 0.1 (default)\n",
      "Minimum required adapter overlap (stringency): 1 bp\n",
      "Minimum required sequence length for both reads before a sequence pair gets removed: 20 bp\n",
      "Output file(s) will be GZIP compressed\n",
      "\n",
      "Writing final adapter and quality trimmed output to NR005_S4_L001_R2_001_trimmed.fq.gz\n",
      "\n",
      "\n",
      "  >>> Now performing quality (cutoff 20) and adapter trimming in a single pass for the adapter sequence: 'CTGTCTCTTATA' from file /home/sam/data/geoduck_illumina/NR005_S4_L001_R2_001.fastq.gz <<< \n",
      "This is cutadapt 1.9.1 with Python 2.7.12\n",
      "Command line parameters: -f fastq -e 0.1 -q 20 -O 1 -a CTGTCTCTTATA /home/sam/data/geoduck_illumina/NR005_S4_L001_R2_001.fastq.gz\n",
      "Trimming 1 adapter with at most 10.0% errors in single-end mode ...\n",
      "Finished in 1.05 s (25 us/read; 2.38 M reads/minute).\n",
      "\n",
      "=== Summary ===\n",
      "\n",
      "Total reads processed:                  41,570\n",
      "Reads with adapters:                     7,910 (19.0%)\n",
      "Reads written (passing filters):        41,570 (100.0%)\n",
      "\n",
      "Total basepairs processed:     6,277,070 bp\n",
      "Quality-trimmed:                  37,349 bp (0.6%)\n",
      "Total written (filtered):      6,222,043 bp (99.1%)\n",
      "\n",
      "=== Adapter 1 ===\n",
      "\n",
      "Sequence: CTGTCTCTTATA; Type: regular 3'; Length: 12; Trimmed: 7910 times.\n",
      "\n",
      "No. of allowed errors:\n",
      "0-9 bp: 0; 10-12 bp: 1\n",
      "\n",
      "Bases preceding removed adapters:\n",
      "  A: 28.6%\n",
      "  C: 15.8%\n",
      "  G: 21.2%\n",
      "  T: 34.4%\n",
      "  none/other: 0.0%\n",
      "\n",
      "Overview of removed sequences\n",
      "length\tcount\texpect\tmax.err\terror counts\n",
      "1\t5880\t10392.5\t0\t5880\n",
      "2\t1384\t2598.1\t0\t1384\n",
      "3\t385\t649.5\t0\t385\n",
      "4\t143\t162.4\t0\t143\n",
      "5\t20\t40.6\t0\t20\n",
      "6\t9\t10.1\t0\t9\n",
      "7\t2\t2.5\t0\t2\n",
      "10\t1\t0.0\t1\t1\n",
      "14\t1\t0.0\t1\t1\n",
      "15\t1\t0.0\t1\t0 1\n",
      "16\t1\t0.0\t1\t0 1\n",
      "18\t1\t0.0\t1\t1\n",
      "26\t1\t0.0\t1\t1\n",
      "27\t1\t0.0\t1\t1\n",
      "28\t2\t0.0\t1\t2\n",
      "29\t1\t0.0\t1\t1\n",
      "30\t1\t0.0\t1\t0 1\n",
      "33\t1\t0.0\t1\t1\n",
      "37\t1\t0.0\t1\t0 1\n",
      "38\t2\t0.0\t1\t2\n",
      "43\t1\t0.0\t1\t1\n",
      "44\t2\t0.0\t1\t1 1\n",
      "51\t1\t0.0\t1\t0 1\n",
      "52\t1\t0.0\t1\t0 1\n",
      "53\t1\t0.0\t1\t1\n",
      "55\t1\t0.0\t1\t0 1\n",
      "56\t1\t0.0\t1\t0 1\n",
      "57\t1\t0.0\t1\t1\n",
      "62\t1\t0.0\t1\t1\n",
      "63\t1\t0.0\t1\t1\n",
      "64\t2\t0.0\t1\t2\n",
      "65\t1\t0.0\t1\t1\n",
      "66\t1\t0.0\t1\t1\n",
      "69\t1\t0.0\t1\t0 1\n",
      "70\t1\t0.0\t1\t0 1\n",
      "73\t1\t0.0\t1\t1\n",
      "74\t2\t0.0\t1\t1 1\n",
      "76\t1\t0.0\t1\t1\n",
      "77\t1\t0.0\t1\t1\n",
      "78\t2\t0.0\t1\t0 2\n",
      "79\t1\t0.0\t1\t1\n",
      "80\t2\t0.0\t1\t2\n",
      "81\t2\t0.0\t1\t2\n",
      "85\t2\t0.0\t1\t2\n",
      "87\t1\t0.0\t1\t1\n",
      "88\t1\t0.0\t1\t1\n",
      "89\t1\t0.0\t1\t1\n",
      "92\t1\t0.0\t1\t1\n",
      "93\t1\t0.0\t1\t0 1\n",
      "94\t2\t0.0\t1\t1 1\n",
      "95\t1\t0.0\t1\t0 1\n",
      "97\t1\t0.0\t1\t1\n",
      "98\t1\t0.0\t1\t0 1\n",
      "99\t1\t0.0\t1\t0 1\n",
      "100\t1\t0.0\t1\t0 1\n",
      "101\t1\t0.0\t1\t1\n",
      "106\t1\t0.0\t1\t0 1\n",
      "107\t1\t0.0\t1\t1\n",
      "108\t2\t0.0\t1\t2\n",
      "110\t1\t0.0\t1\t1\n",
      "111\t1\t0.0\t1\t1\n",
      "113\t2\t0.0\t1\t1 1\n",
      "114\t1\t0.0\t1\t1\n",
      "115\t2\t0.0\t1\t2\n",
      "116\t1\t0.0\t1\t0 1\n",
      "118\t1\t0.0\t1\t1\n",
      "120\t2\t0.0\t1\t2\n",
      "126\t1\t0.0\t1\t1\n",
      "129\t2\t0.0\t1\t1 1\n",
      "130\t1\t0.0\t1\t1\n",
      "131\t2\t0.0\t1\t2\n",
      "134\t2\t0.0\t1\t1 1\n",
      "137\t1\t0.0\t1\t1\n",
      "138\t1\t0.0\t1\t0 1\n",
      "141\t1\t0.0\t1\t1\n",
      "143\t1\t0.0\t1\t1\n",
      "145\t1\t0.0\t1\t1\n",
      "\n",
      "\n",
      "RUN STATISTICS FOR INPUT FILE: /home/sam/data/geoduck_illumina/NR005_S4_L001_R2_001.fastq.gz\n",
      "=============================================\n",
      "41570 sequences processed in total\n",
      "The length threshold of paired-end sequences gets evaluated later on (in the validation step)\n",
      "\n",
      "Validate paired-end files NR005_S4_L001_R1_001_trimmed.fq.gz and NR005_S4_L001_R2_001_trimmed.fq.gz\n",
      "file_1: NR005_S4_L001_R1_001_trimmed.fq.gz, file_2: NR005_S4_L001_R2_001_trimmed.fq.gz\n",
      "\n",
      "\n",
      ">>>>> Now validing the length of the 2 paired-end infiles: NR005_S4_L001_R1_001_trimmed.fq.gz and NR005_S4_L001_R2_001_trimmed.fq.gz <<<<<\n",
      "Writing validated paired-end read 1 reads to NR005_S4_L001_R1_001_val_1.fq.gz\n",
      "Writing validated paired-end read 2 reads to NR005_S4_L001_R2_001_val_2.fq.gz\n",
      "\n",
      "Total number of sequences analysed: 41570\n",
      "\n",
      "Number of sequence pairs removed because at least one read was shorter than the length cutoff (20 bp): 110 (0.26%)\n",
      "\n",
      "Deleting both intermediate output files NR005_S4_L001_R1_001_trimmed.fq.gz and NR005_S4_L001_R2_001_trimmed.fq.gz\n",
      "\n",
      "====================================================================================================\n",
      "\n",
      "Writing report to '/home/sam/data/geoduck_illumina/trimmed/NR005_S4_L002_R1_001.fastq.gz_trimming_report.txt'\n",
      "\n",
      "SUMMARISING RUN PARAMETERS\n",
      "==========================\n",
      "Input filename: /home/sam/data/geoduck_illumina/NR005_S4_L002_R1_001.fastq.gz\n",
      "Trimming mode: paired-end\n",
      "Trim Galore version: 0.4.4\n",
      "Cutadapt version: 1.9.1\n",
      "Quality Phred score cutoff: 20\n",
      "Quality encoding type selected: ASCII+33\n",
      "Adapter sequence: 'CTGTCTCTTATA' (Nextera Transposase sequence; auto-detected)\n",
      "Maximum trimming error rate: 0.1 (default)\n",
      "Minimum required adapter overlap (stringency): 1 bp\n",
      "Minimum required sequence length for both reads before a sequence pair gets removed: 20 bp\n",
      "Output file(s) will be GZIP compressed\n",
      "\n",
      "Writing final adapter and quality trimmed output to NR005_S4_L002_R1_001_trimmed.fq.gz\n",
      "\n",
      "\n",
      "  >>> Now performing quality (cutoff 20) and adapter trimming in a single pass for the adapter sequence: 'CTGTCTCTTATA' from file /home/sam/data/geoduck_illumina/NR005_S4_L002_R1_001.fastq.gz <<< \n",
      "This is cutadapt 1.9.1 with Python 2.7.12\n",
      "Command line parameters: -f fastq -e 0.1 -q 20 -O 1 -a CTGTCTCTTATA /home/sam/data/geoduck_illumina/NR005_S4_L002_R1_001.fastq.gz\n",
      "Trimming 1 adapter with at most 10.0% errors in single-end mode ...\n",
      "Finished in 0.96 s (26 us/read; 2.30 M reads/minute).\n",
      "\n",
      "=== Summary ===\n",
      "\n",
      "Total reads processed:                  36,577\n",
      "Reads with adapters:                    11,175 (30.6%)\n",
      "Reads written (passing filters):        36,577 (100.0%)\n",
      "\n",
      "Total basepairs processed:     5,523,127 bp\n",
      "Quality-trimmed:                   4,381 bp (0.1%)\n",
      "Total written (filtered):      5,492,464 bp (99.4%)\n",
      "\n",
      "=== Adapter 1 ===\n",
      "\n",
      "Sequence: CTGTCTCTTATA; Type: regular 3'; Length: 12; Trimmed: 11175 times.\n",
      "\n",
      "No. of allowed errors:\n",
      "0-9 bp: 0; 10-12 bp: 1\n",
      "\n",
      "Bases preceding removed adapters:\n",
      "  A: 29.9%\n",
      "  C: 15.0%\n",
      "  G: 15.2%\n",
      "  T: 39.9%\n",
      "  none/other: 0.0%\n",
      "\n",
      "Overview of removed sequences\n",
      "length\tcount\texpect\tmax.err\terror counts\n",
      "1\t8079\t9144.2\t0\t8079\n",
      "2\t2102\t2286.1\t0\t2102\n",
      "3\t665\t571.5\t0\t665\n",
      "4\t132\t142.9\t0\t132\n",
      "5\t37\t35.7\t0\t37\n",
      "6\t8\t8.9\t0\t8\n",
      "7\t7\t2.2\t0\t7\n",
      "8\t1\t0.6\t0\t1\n",
      "10\t1\t0.0\t1\t0 1\n",
      "11\t1\t0.0\t1\t1\n",
      "20\t1\t0.0\t1\t1\n",
      "21\t2\t0.0\t1\t1 1\n",
      "22\t1\t0.0\t1\t0 1\n",
      "25\t2\t0.0\t1\t2\n",
      "26\t1\t0.0\t1\t0 1\n",
      "27\t1\t0.0\t1\t1\n",
      "28\t1\t0.0\t1\t0 1\n",
      "29\t2\t0.0\t1\t1 1\n",
      "31\t2\t0.0\t1\t2\n",
      "32\t2\t0.0\t1\t2\n",
      "33\t3\t0.0\t1\t3\n",
      "35\t1\t0.0\t1\t1\n",
      "37\t2\t0.0\t1\t1 1\n",
      "38\t2\t0.0\t1\t2\n",
      "39\t4\t0.0\t1\t3 1\n",
      "41\t2\t0.0\t1\t0 2\n",
      "42\t1\t0.0\t1\t0 1\n",
      "44\t3\t0.0\t1\t2 1\n",
      "45\t5\t0.0\t1\t3 2\n",
      "46\t3\t0.0\t1\t1 2\n",
      "47\t2\t0.0\t1\t1 1\n",
      "49\t2\t0.0\t1\t0 2\n",
      "50\t1\t0.0\t1\t1\n",
      "52\t2\t0.0\t1\t2\n",
      "54\t3\t0.0\t1\t2 1\n",
      "57\t2\t0.0\t1\t2\n",
      "58\t2\t0.0\t1\t1 1\n",
      "59\t2\t0.0\t1\t2\n",
      "60\t1\t0.0\t1\t1\n",
      "62\t2\t0.0\t1\t1 1\n",
      "65\t2\t0.0\t1\t1 1\n",
      "66\t1\t0.0\t1\t0 1\n",
      "67\t1\t0.0\t1\t0 1\n",
      "69\t3\t0.0\t1\t3\n",
      "70\t1\t0.0\t1\t1\n",
      "71\t1\t0.0\t1\t1\n",
      "72\t3\t0.0\t1\t3\n",
      "73\t1\t0.0\t1\t1\n",
      "74\t1\t0.0\t1\t1\n",
      "75\t2\t0.0\t1\t1 1\n",
      "76\t2\t0.0\t1\t1 1\n",
      "77\t2\t0.0\t1\t2\n",
      "80\t1\t0.0\t1\t0 1\n",
      "81\t4\t0.0\t1\t4\n",
      "83\t2\t0.0\t1\t1 1\n",
      "86\t1\t0.0\t1\t1\n",
      "87\t1\t0.0\t1\t1\n",
      "94\t2\t0.0\t1\t1 1\n",
      "95\t1\t0.0\t1\t1\n",
      "96\t1\t0.0\t1\t1\n",
      "97\t2\t0.0\t1\t0 2\n",
      "98\t2\t0.0\t1\t2\n",
      "102\t1\t0.0\t1\t1\n",
      "106\t2\t0.0\t1\t0 2\n",
      "107\t1\t0.0\t1\t1\n",
      "110\t1\t0.0\t1\t1\n",
      "111\t3\t0.0\t1\t3\n",
      "112\t3\t0.0\t1\t0 3\n",
      "114\t1\t0.0\t1\t0 1\n",
      "115\t1\t0.0\t1\t1\n",
      "116\t3\t0.0\t1\t2 1\n",
      "118\t2\t0.0\t1\t1 1\n",
      "122\t1\t0.0\t1\t0 1\n",
      "125\t1\t0.0\t1\t0 1\n",
      "126\t1\t0.0\t1\t1\n",
      "127\t1\t0.0\t1\t0 1\n",
      "129\t2\t0.0\t1\t2\n",
      "130\t1\t0.0\t1\t1\n",
      "131\t2\t0.0\t1\t2\n",
      "134\t2\t0.0\t1\t1 1\n",
      "137\t1\t0.0\t1\t0 1\n",
      "138\t2\t0.0\t1\t1 1\n",
      "139\t2\t0.0\t1\t1 1\n",
      "142\t1\t0.0\t1\t1\n",
      "143\t1\t0.0\t1\t0 1\n",
      "145\t4\t0.0\t1\t3 1\n",
      "147\t2\t0.0\t1\t2\n",
      "148\t3\t0.0\t1\t3\n",
      "\n",
      "\n",
      "RUN STATISTICS FOR INPUT FILE: /home/sam/data/geoduck_illumina/NR005_S4_L002_R1_001.fastq.gz\n",
      "=============================================\n",
      "36577 sequences processed in total\n",
      "The length threshold of paired-end sequences gets evaluated later on (in the validation step)\n",
      "\n",
      "Writing report to '/home/sam/data/geoduck_illumina/trimmed/NR005_S4_L002_R2_001.fastq.gz_trimming_report.txt'\n",
      "\n",
      "SUMMARISING RUN PARAMETERS\n",
      "==========================\n",
      "Input filename: /home/sam/data/geoduck_illumina/NR005_S4_L002_R2_001.fastq.gz\n",
      "Trimming mode: paired-end\n",
      "Trim Galore version: 0.4.4\n",
      "Cutadapt version: 1.9.1\n",
      "Quality Phred score cutoff: 20\n",
      "Quality encoding type selected: ASCII+33\n",
      "Adapter sequence: 'CTGTCTCTTATA' (Nextera Transposase sequence; auto-detected)\n",
      "Maximum trimming error rate: 0.1 (default)\n",
      "Minimum required adapter overlap (stringency): 1 bp\n",
      "Minimum required sequence length for both reads before a sequence pair gets removed: 20 bp\n",
      "Output file(s) will be GZIP compressed\n",
      "\n",
      "Writing final adapter and quality trimmed output to NR005_S4_L002_R2_001_trimmed.fq.gz\n",
      "\n",
      "\n",
      "  >>> Now performing quality (cutoff 20) and adapter trimming in a single pass for the adapter sequence: 'CTGTCTCTTATA' from file /home/sam/data/geoduck_illumina/NR005_S4_L002_R2_001.fastq.gz <<< \n",
      "This is cutadapt 1.9.1 with Python 2.7.12\n",
      "Command line parameters: -f fastq -e 0.1 -q 20 -O 1 -a CTGTCTCTTATA /home/sam/data/geoduck_illumina/NR005_S4_L002_R2_001.fastq.gz\n",
      "Trimming 1 adapter with at most 10.0% errors in single-end mode ...\n",
      "Finished in 0.93 s (25 us/read; 2.37 M reads/minute).\n",
      "\n",
      "=== Summary ===\n",
      "\n",
      "Total reads processed:                  36,577\n",
      "Reads with adapters:                     6,876 (18.8%)\n",
      "Reads written (passing filters):        36,577 (100.0%)\n",
      "\n",
      "Total basepairs processed:     5,523,127 bp\n",
      "Quality-trimmed:                  34,645 bp (0.6%)\n",
      "Total written (filtered):      5,471,904 bp (99.1%)\n",
      "\n",
      "=== Adapter 1 ===\n",
      "\n",
      "Sequence: CTGTCTCTTATA; Type: regular 3'; Length: 12; Trimmed: 6876 times.\n",
      "\n",
      "No. of allowed errors:\n",
      "0-9 bp: 0; 10-12 bp: 1\n",
      "\n",
      "Bases preceding removed adapters:\n",
      "  A: 29.6%\n",
      "  C: 16.2%\n",
      "  G: 21.3%\n",
      "  T: 33.0%\n",
      "  none/other: 0.0%\n",
      "\n",
      "Overview of removed sequences\n",
      "length\tcount\texpect\tmax.err\terror counts\n",
      "1\t5020\t9144.2\t0\t5020\n",
      "2\t1245\t2286.1\t0\t1245\n",
      "3\t388\t571.5\t0\t388\n",
      "4\t104\t142.9\t0\t104\n",
      "5\t19\t35.7\t0\t19\n",
      "6\t9\t8.9\t0\t9\n",
      "8\t2\t0.6\t0\t2\n",
      "10\t1\t0.0\t1\t0 1\n",
      "16\t2\t0.0\t1\t1 1\n",
      "18\t1\t0.0\t1\t1\n",
      "20\t4\t0.0\t1\t3 1\n",
      "21\t1\t0.0\t1\t1\n",
      "22\t1\t0.0\t1\t1\n",
      "26\t1\t0.0\t1\t1\n",
      "27\t1\t0.0\t1\t1\n",
      "28\t1\t0.0\t1\t0 1\n",
      "32\t1\t0.0\t1\t1\n",
      "36\t2\t0.0\t1\t2\n",
      "37\t1\t0.0\t1\t1\n",
      "46\t1\t0.0\t1\t1\n",
      "48\t1\t0.0\t1\t1\n",
      "50\t3\t0.0\t1\t2 1\n",
      "51\t2\t0.0\t1\t2\n",
      "52\t1\t0.0\t1\t1\n",
      "53\t1\t0.0\t1\t0 1\n",
      "55\t1\t0.0\t1\t1\n",
      "56\t1\t0.0\t1\t0 1\n",
      "57\t1\t0.0\t1\t1\n",
      "58\t1\t0.0\t1\t1\n",
      "61\t2\t0.0\t1\t2\n",
      "63\t1\t0.0\t1\t1\n",
      "64\t2\t0.0\t1\t2\n",
      "66\t1\t0.0\t1\t0 1\n",
      "67\t3\t0.0\t1\t2 1\n",
      "70\t1\t0.0\t1\t1\n",
      "75\t1\t0.0\t1\t1\n",
      "81\t1\t0.0\t1\t0 1\n",
      "82\t1\t0.0\t1\t1\n",
      "84\t1\t0.0\t1\t1\n",
      "86\t1\t0.0\t1\t1\n",
      "87\t2\t0.0\t1\t1 1\n",
      "89\t1\t0.0\t1\t1\n",
      "90\t2\t0.0\t1\t2\n",
      "91\t2\t0.0\t1\t2\n",
      "95\t1\t0.0\t1\t1\n",
      "96\t1\t0.0\t1\t1\n",
      "97\t1\t0.0\t1\t1\n",
      "102\t1\t0.0\t1\t1\n",
      "107\t2\t0.0\t1\t2\n",
      "110\t1\t0.0\t1\t1\n",
      "111\t1\t0.0\t1\t1\n",
      "112\t1\t0.0\t1\t0 1\n",
      "113\t1\t0.0\t1\t0 1\n",
      "114\t1\t0.0\t1\t0 1\n",
      "115\t1\t0.0\t1\t1\n",
      "117\t1\t0.0\t1\t1\n",
      "120\t2\t0.0\t1\t2\n",
      "121\t3\t0.0\t1\t3\n",
      "122\t1\t0.0\t1\t1\n",
      "123\t2\t0.0\t1\t2\n",
      "125\t1\t0.0\t1\t1\n",
      "126\t1\t0.0\t1\t1\n",
      "128\t1\t0.0\t1\t1\n",
      "129\t1\t0.0\t1\t0 1\n",
      "130\t2\t0.0\t1\t1 1\n",
      "132\t2\t0.0\t1\t2\n",
      "135\t1\t0.0\t1\t1\n",
      "142\t1\t0.0\t1\t1\n",
      "144\t1\t0.0\t1\t1\n",
      "146\t1\t0.0\t1\t1\n",
      "147\t2\t0.0\t1\t1 1\n",
      "148\t1\t0.0\t1\t1\n",
      "150\t1\t0.0\t1\t1\n",
      "\n",
      "\n",
      "RUN STATISTICS FOR INPUT FILE: /home/sam/data/geoduck_illumina/NR005_S4_L002_R2_001.fastq.gz\n",
      "=============================================\n",
      "36577 sequences processed in total\n",
      "The length threshold of paired-end sequences gets evaluated later on (in the validation step)\n",
      "\n",
      "Validate paired-end files NR005_S4_L002_R1_001_trimmed.fq.gz and NR005_S4_L002_R2_001_trimmed.fq.gz\n",
      "file_1: NR005_S4_L002_R1_001_trimmed.fq.gz, file_2: NR005_S4_L002_R2_001_trimmed.fq.gz\n",
      "\n",
      "\n",
      ">>>>> Now validing the length of the 2 paired-end infiles: NR005_S4_L002_R1_001_trimmed.fq.gz and NR005_S4_L002_R2_001_trimmed.fq.gz <<<<<\n",
      "Writing validated paired-end read 1 reads to NR005_S4_L002_R1_001_val_1.fq.gz\n",
      "Writing validated paired-end read 2 reads to NR005_S4_L002_R2_001_val_2.fq.gz\n",
      "\n",
      "Total number of sequences analysed: 36577\n",
      "\n",
      "Number of sequence pairs removed because at least one read was shorter than the length cutoff (20 bp): 121 (0.33%)\n",
      "\n",
      "Deleting both intermediate output files NR005_S4_L002_R1_001_trimmed.fq.gz and NR005_S4_L002_R2_001_trimmed.fq.gz\n",
      "\n",
      "====================================================================================================\n",
      "\n",
      "Writing report to '/home/sam/data/geoduck_illumina/trimmed/NR006_S3_L001_R1_001.fastq.gz_trimming_report.txt'\n",
      "\n",
      "SUMMARISING RUN PARAMETERS\n",
      "==========================\n",
      "Input filename: /home/sam/data/geoduck_illumina/NR006_S3_L001_R1_001.fastq.gz\n",
      "Trimming mode: paired-end\n",
      "Trim Galore version: 0.4.4\n",
      "Cutadapt version: 1.9.1\n",
      "Quality Phred score cutoff: 20\n",
      "Quality encoding type selected: ASCII+33\n",
      "Adapter sequence: 'CTGTCTCTTATA' (Nextera Transposase sequence; auto-detected)\n",
      "Maximum trimming error rate: 0.1 (default)\n",
      "Minimum required adapter overlap (stringency): 1 bp\n",
      "Minimum required sequence length for both reads before a sequence pair gets removed: 20 bp\n",
      "Output file(s) will be GZIP compressed\n",
      "\n",
      "Writing final adapter and quality trimmed output to NR006_S3_L001_R1_001_trimmed.fq.gz\n",
      "\n",
      "\n",
      "  >>> Now performing quality (cutoff 20) and adapter trimming in a single pass for the adapter sequence: 'CTGTCTCTTATA' from file /home/sam/data/geoduck_illumina/NR006_S3_L001_R1_001.fastq.gz <<< \n",
      "This is cutadapt 1.9.1 with Python 2.7.12\n",
      "Command line parameters: -f fastq -e 0.1 -q 20 -O 1 -a CTGTCTCTTATA /home/sam/data/geoduck_illumina/NR006_S3_L001_R1_001.fastq.gz\n",
      "Trimming 1 adapter with at most 10.0% errors in single-end mode ...\n",
      "Finished in 88.92 s (26 us/read; 2.31 M reads/minute).\n",
      "\n",
      "=== Summary ===\n",
      "\n",
      "Total reads processed:               3,427,897\n",
      "Reads with adapters:                   993,908 (29.0%)\n",
      "Reads written (passing filters):     3,427,897 (100.0%)\n",
      "\n",
      "Total basepairs processed:   517,612,447 bp\n",
      "Quality-trimmed:                 649,955 bp (0.1%)\n",
      "Total written (filtered):    515,115,749 bp (99.5%)\n",
      "\n",
      "=== Adapter 1 ===\n",
      "\n",
      "Sequence: CTGTCTCTTATA; Type: regular 3'; Length: 12; Trimmed: 993908 times.\n",
      "\n",
      "No. of allowed errors:\n",
      "0-9 bp: 0; 10-12 bp: 1\n",
      "\n",
      "Bases preceding removed adapters:\n",
      "  A: 30.6%\n",
      "  C: 15.9%\n",
      "  G: 15.8%\n",
      "  T: 37.7%\n",
      "  none/other: 0.0%\n",
      "\n",
      "Overview of removed sequences\n",
      "length\tcount\texpect\tmax.err\terror counts\n",
      "1\t729743\t856974.2\t0\t729743\n",
      "2\t182904\t214243.6\t0\t182904\n",
      "3\t59512\t53560.9\t0\t59512\n",
      "4\t12113\t13390.2\t0\t12113\n",
      "5\t2394\t3347.6\t0\t2394\n",
      "6\t759\t836.9\t0\t759\n",
      "7\t197\t209.2\t0\t197\n",
      "8\t84\t52.3\t0\t84\n",
      "9\t67\t13.1\t0\t14 53\n",
      "10\t95\t3.3\t1\t8 87\n",
      "11\t68\t0.8\t1\t11 57\n",
      "12\t24\t0.2\t1\t9 15\n",
      "13\t29\t0.2\t1\t9 20\n",
      "14\t31\t0.2\t1\t12 19\n",
      "15\t22\t0.2\t1\t10 12\n",
      "16\t36\t0.2\t1\t11 25\n",
      "17\t8\t0.2\t1\t4 4\n",
      "18\t38\t0.2\t1\t10 28\n",
      "19\t30\t0.2\t1\t6 24\n",
      "20\t36\t0.2\t1\t9 27\n",
      "21\t36\t0.2\t1\t14 22\n",
      "22\t32\t0.2\t1\t10 22\n",
      "23\t32\t0.2\t1\t4 28\n",
      "24\t51\t0.2\t1\t18 33\n",
      "25\t42\t0.2\t1\t5 37\n",
      "26\t30\t0.2\t1\t13 17\n",
      "27\t36\t0.2\t1\t11 25\n",
      "28\t35\t0.2\t1\t7 28\n",
      "29\t49\t0.2\t1\t11 38\n",
      "30\t37\t0.2\t1\t13 24\n",
      "31\t48\t0.2\t1\t15 33\n",
      "32\t72\t0.2\t1\t13 59\n",
      "33\t55\t0.2\t1\t14 41\n",
      "34\t33\t0.2\t1\t5 28\n",
      "35\t34\t0.2\t1\t7 27\n",
      "36\t59\t0.2\t1\t13 46\n",
      "37\t46\t0.2\t1\t11 35\n",
      "38\t41\t0.2\t1\t11 30\n",
      "39\t33\t0.2\t1\t10 23\n",
      "40\t55\t0.2\t1\t15 40\n",
      "41\t66\t0.2\t1\t13 53\n",
      "42\t46\t0.2\t1\t13 33\n",
      "43\t42\t0.2\t1\t14 28\n",
      "44\t51\t0.2\t1\t17 34\n",
      "45\t68\t0.2\t1\t12 56\n",
      "46\t51\t0.2\t1\t11 40\n",
      "47\t30\t0.2\t1\t8 22\n",
      "48\t29\t0.2\t1\t11 18\n",
      "49\t33\t0.2\t1\t9 24\n",
      "50\t33\t0.2\t1\t11 22\n",
      "51\t29\t0.2\t1\t12 17\n",
      "52\t44\t0.2\t1\t14 30\n",
      "53\t23\t0.2\t1\t9 14\n",
      "54\t41\t0.2\t1\t17 24\n",
      "55\t25\t0.2\t1\t12 13\n",
      "56\t49\t0.2\t1\t9 40\n",
      "57\t38\t0.2\t1\t18 20\n",
      "58\t31\t0.2\t1\t13 18\n",
      "59\t20\t0.2\t1\t13 7\n",
      "60\t25\t0.2\t1\t17 8\n",
      "61\t22\t0.2\t1\t11 11\n",
      "62\t25\t0.2\t1\t10 15\n",
      "63\t34\t0.2\t1\t10 24\n",
      "64\t62\t0.2\t1\t20 42\n",
      "65\t61\t0.2\t1\t17 44\n",
      "66\t43\t0.2\t1\t17 26\n",
      "67\t36\t0.2\t1\t17 19\n",
      "68\t42\t0.2\t1\t20 22\n",
      "69\t32\t0.2\t1\t13 19\n",
      "70\t25\t0.2\t1\t15 10\n",
      "71\t34\t0.2\t1\t15 19\n",
      "72\t29\t0.2\t1\t11 18\n",
      "73\t20\t0.2\t1\t13 7\n",
      "74\t46\t0.2\t1\t20 26\n",
      "75\t35\t0.2\t1\t9 26\n",
      "76\t37\t0.2\t1\t20 17\n",
      "77\t36\t0.2\t1\t10 26\n",
      "78\t32\t0.2\t1\t16 16\n",
      "79\t23\t0.2\t1\t12 11\n",
      "80\t37\t0.2\t1\t25 12\n",
      "81\t28\t0.2\t1\t8 20\n",
      "82\t73\t0.2\t1\t19 54\n",
      "83\t37\t0.2\t1\t19 18\n",
      "84\t36\t0.2\t1\t19 17\n",
      "85\t51\t0.2\t1\t16 35\n",
      "86\t35\t0.2\t1\t16 19\n",
      "87\t44\t0.2\t1\t17 27\n",
      "88\t43\t0.2\t1\t15 28\n",
      "89\t57\t0.2\t1\t22 35\n",
      "90\t60\t0.2\t1\t15 45\n",
      "91\t47\t0.2\t1\t26 21\n",
      "92\t46\t0.2\t1\t17 29\n",
      "93\t45\t0.2\t1\t25 20\n",
      "94\t55\t0.2\t1\t19 36\n",
      "95\t58\t0.2\t1\t17 41\n",
      "96\t87\t0.2\t1\t18 69\n",
      "97\t44\t0.2\t1\t19 25\n",
      "98\t39\t0.2\t1\t17 22\n",
      "99\t33\t0.2\t1\t15 18\n",
      "100\t44\t0.2\t1\t18 26\n",
      "101\t54\t0.2\t1\t12 42\n",
      "102\t76\t0.2\t1\t18 58\n",
      "103\t49\t0.2\t1\t22 27\n",
      "104\t53\t0.2\t1\t25 28\n",
      "105\t52\t0.2\t1\t18 34\n",
      "106\t125\t0.2\t1\t10 115\n",
      "107\t71\t0.2\t1\t25 46\n",
      "108\t86\t0.2\t1\t10 76\n",
      "109\t71\t0.2\t1\t25 46\n",
      "110\t40\t0.2\t1\t20 20\n",
      "111\t69\t0.2\t1\t19 50\n",
      "112\t43\t0.2\t1\t17 26\n",
      "113\t53\t0.2\t1\t13 40\n",
      "114\t66\t0.2\t1\t27 39\n",
      "115\t55\t0.2\t1\t15 40\n",
      "116\t37\t0.2\t1\t18 19\n",
      "117\t45\t0.2\t1\t12 33\n",
      "118\t41\t0.2\t1\t14 27\n",
      "119\t41\t0.2\t1\t12 29\n",
      "120\t44\t0.2\t1\t22 22\n",
      "121\t45\t0.2\t1\t12 33\n",
      "122\t37\t0.2\t1\t12 25\n",
      "123\t41\t0.2\t1\t23 18\n",
      "124\t35\t0.2\t1\t18 17\n",
      "125\t40\t0.2\t1\t22 18\n",
      "126\t45\t0.2\t1\t27 18\n",
      "127\t35\t0.2\t1\t20 15\n",
      "128\t43\t0.2\t1\t26 17\n",
      "129\t42\t0.2\t1\t25 17\n",
      "130\t37\t0.2\t1\t18 19\n",
      "131\t35\t0.2\t1\t26 9\n",
      "132\t37\t0.2\t1\t18 19\n",
      "133\t39\t0.2\t1\t24 15\n",
      "134\t28\t0.2\t1\t14 14\n",
      "135\t43\t0.2\t1\t20 23\n",
      "136\t33\t0.2\t1\t24 9\n",
      "137\t42\t0.2\t1\t19 23\n",
      "138\t56\t0.2\t1\t24 32\n",
      "139\t33\t0.2\t1\t19 14\n",
      "140\t69\t0.2\t1\t27 42\n",
      "141\t40\t0.2\t1\t17 23\n",
      "142\t30\t0.2\t1\t15 15\n",
      "143\t52\t0.2\t1\t21 31\n",
      "144\t42\t0.2\t1\t17 25\n",
      "145\t55\t0.2\t1\t28 27\n",
      "146\t48\t0.2\t1\t16 32\n",
      "147\t54\t0.2\t1\t22 32\n",
      "148\t33\t0.2\t1\t17 16\n",
      "149\t20\t0.2\t1\t11 9\n",
      "150\t36\t0.2\t1\t12 24\n",
      "151\t19\t0.2\t1\t3 16\n",
      "\n",
      "\n",
      "RUN STATISTICS FOR INPUT FILE: /home/sam/data/geoduck_illumina/NR006_S3_L001_R1_001.fastq.gz\n",
      "=============================================\n",
      "3427897 sequences processed in total\n",
      "The length threshold of paired-end sequences gets evaluated later on (in the validation step)\n",
      "\n",
      "Writing report to '/home/sam/data/geoduck_illumina/trimmed/NR006_S3_L001_R2_001.fastq.gz_trimming_report.txt'\n",
      "\n",
      "SUMMARISING RUN PARAMETERS\n",
      "==========================\n",
      "Input filename: /home/sam/data/geoduck_illumina/NR006_S3_L001_R2_001.fastq.gz\n",
      "Trimming mode: paired-end\n",
      "Trim Galore version: 0.4.4\n",
      "Cutadapt version: 1.9.1\n",
      "Quality Phred score cutoff: 20\n",
      "Quality encoding type selected: ASCII+33\n",
      "Adapter sequence: 'CTGTCTCTTATA' (Nextera Transposase sequence; auto-detected)\n",
      "Maximum trimming error rate: 0.1 (default)\n",
      "Minimum required adapter overlap (stringency): 1 bp\n",
      "Minimum required sequence length for both reads before a sequence pair gets removed: 20 bp\n",
      "Output file(s) will be GZIP compressed\n",
      "\n",
      "Writing final adapter and quality trimmed output to NR006_S3_L001_R2_001_trimmed.fq.gz\n",
      "\n",
      "\n",
      "  >>> Now performing quality (cutoff 20) and adapter trimming in a single pass for the adapter sequence: 'CTGTCTCTTATA' from file /home/sam/data/geoduck_illumina/NR006_S3_L001_R2_001.fastq.gz <<< \n",
      "This is cutadapt 1.9.1 with Python 2.7.12\n",
      "Command line parameters: -f fastq -e 0.1 -q 20 -O 1 -a CTGTCTCTTATA /home/sam/data/geoduck_illumina/NR006_S3_L001_R2_001.fastq.gz\n",
      "Trimming 1 adapter with at most 10.0% errors in single-end mode ...\n",
      "Finished in 84.08 s (25 us/read; 2.45 M reads/minute).\n",
      "\n",
      "=== Summary ===\n",
      "\n",
      "Total reads processed:               3,427,897\n",
      "Reads with adapters:                   655,420 (19.1%)\n",
      "Reads written (passing filters):     3,427,897 (100.0%)\n",
      "\n",
      "Total basepairs processed:   517,612,447 bp\n",
      "Quality-trimmed:               4,287,079 bp (0.8%)\n",
      "Total written (filtered):    512,054,481 bp (98.9%)\n",
      "\n",
      "=== Adapter 1 ===\n",
      "\n",
      "Sequence: CTGTCTCTTATA; Type: regular 3'; Length: 12; Trimmed: 655420 times.\n",
      "\n",
      "No. of allowed errors:\n",
      "0-9 bp: 0; 10-12 bp: 1\n",
      "\n",
      "Bases preceding removed adapters:\n",
      "  A: 28.9%\n",
      "  C: 16.1%\n",
      "  G: 19.6%\n",
      "  T: 35.4%\n",
      "  none/other: 0.0%\n",
      "\n",
      "Overview of removed sequences\n",
      "length\tcount\texpect\tmax.err\terror counts\n",
      "1\t484728\t856974.2\t0\t484728\n",
      "2\t116897\t214243.6\t0\t116897\n",
      "3\t35120\t53560.9\t0\t35120\n",
      "4\t11002\t13390.2\t0\t11002\n",
      "5\t1997\t3347.6\t0\t1997\n",
      "6\t692\t836.9\t0\t692\n",
      "7\t182\t209.2\t0\t182\n",
      "8\t67\t52.3\t0\t67\n",
      "9\t51\t13.1\t0\t12 39\n",
      "10\t99\t3.3\t1\t12 87\n",
      "11\t44\t0.8\t1\t7 37\n",
      "12\t26\t0.2\t1\t12 14\n",
      "13\t42\t0.2\t1\t10 32\n",
      "14\t23\t0.2\t1\t7 16\n",
      "15\t25\t0.2\t1\t9 16\n",
      "16\t27\t0.2\t1\t9 18\n",
      "17\t18\t0.2\t1\t4 14\n",
      "18\t23\t0.2\t1\t11 12\n",
      "19\t29\t0.2\t1\t7 22\n",
      "20\t17\t0.2\t1\t6 11\n",
      "21\t27\t0.2\t1\t9 18\n",
      "22\t30\t0.2\t1\t8 22\n",
      "23\t23\t0.2\t1\t7 16\n",
      "24\t31\t0.2\t1\t11 20\n",
      "25\t36\t0.2\t1\t9 27\n",
      "26\t39\t0.2\t1\t15 24\n",
      "27\t38\t0.2\t1\t15 23\n",
      "28\t35\t0.2\t1\t12 23\n",
      "29\t22\t0.2\t1\t2 20\n",
      "30\t44\t0.2\t1\t17 27\n",
      "31\t22\t0.2\t1\t10 12\n",
      "32\t51\t0.2\t1\t15 36\n",
      "33\t31\t0.2\t1\t5 26\n",
      "34\t24\t0.2\t1\t8 16\n",
      "35\t18\t0.2\t1\t6 12\n",
      "36\t34\t0.2\t1\t15 19\n",
      "37\t32\t0.2\t1\t9 23\n",
      "38\t32\t0.2\t1\t12 20\n",
      "39\t30\t0.2\t1\t16 14\n",
      "40\t42\t0.2\t1\t14 28\n",
      "41\t22\t0.2\t1\t11 11\n",
      "42\t27\t0.2\t1\t10 17\n",
      "43\t29\t0.2\t1\t12 17\n",
      "44\t40\t0.2\t1\t12 28\n",
      "45\t39\t0.2\t1\t13 26\n",
      "46\t23\t0.2\t1\t9 14\n",
      "47\t21\t0.2\t1\t15 6\n",
      "48\t25\t0.2\t1\t12 13\n",
      "49\t35\t0.2\t1\t11 24\n",
      "50\t26\t0.2\t1\t12 14\n",
      "51\t31\t0.2\t1\t16 15\n",
      "52\t33\t0.2\t1\t12 21\n",
      "53\t31\t0.2\t1\t14 17\n",
      "54\t27\t0.2\t1\t15 12\n",
      "55\t33\t0.2\t1\t15 18\n",
      "56\t25\t0.2\t1\t13 12\n",
      "57\t21\t0.2\t1\t14 7\n",
      "58\t18\t0.2\t1\t9 9\n",
      "59\t24\t0.2\t1\t18 6\n",
      "60\t33\t0.2\t1\t9 24\n",
      "61\t25\t0.2\t1\t9 16\n",
      "62\t19\t0.2\t1\t12 7\n",
      "63\t28\t0.2\t1\t14 14\n",
      "64\t37\t0.2\t1\t17 20\n",
      "65\t36\t0.2\t1\t10 26\n",
      "66\t23\t0.2\t1\t12 11\n",
      "67\t34\t0.2\t1\t16 18\n",
      "68\t35\t0.2\t1\t15 20\n",
      "69\t34\t0.2\t1\t13 21\n",
      "70\t30\t0.2\t1\t17 13\n",
      "71\t26\t0.2\t1\t16 10\n",
      "72\t31\t0.2\t1\t19 12\n",
      "73\t25\t0.2\t1\t13 12\n",
      "74\t20\t0.2\t1\t11 9\n",
      "75\t23\t0.2\t1\t9 14\n",
      "76\t34\t0.2\t1\t15 19\n",
      "77\t30\t0.2\t1\t16 14\n",
      "78\t30\t0.2\t1\t12 18\n",
      "79\t33\t0.2\t1\t15 18\n",
      "80\t33\t0.2\t1\t8 25\n",
      "81\t24\t0.2\t1\t13 11\n",
      "82\t29\t0.2\t1\t10 19\n",
      "83\t30\t0.2\t1\t8 22\n",
      "84\t29\t0.2\t1\t9 20\n",
      "85\t35\t0.2\t1\t11 24\n",
      "86\t31\t0.2\t1\t16 15\n",
      "87\t27\t0.2\t1\t10 17\n",
      "88\t30\t0.2\t1\t13 17\n",
      "89\t33\t0.2\t1\t11 22\n",
      "90\t33\t0.2\t1\t10 23\n",
      "91\t26\t0.2\t1\t12 14\n",
      "92\t51\t0.2\t1\t19 32\n",
      "93\t38\t0.2\t1\t10 28\n",
      "94\t30\t0.2\t1\t17 13\n",
      "95\t46\t0.2\t1\t13 33\n",
      "96\t31\t0.2\t1\t14 17\n",
      "97\t23\t0.2\t1\t12 11\n",
      "98\t48\t0.2\t1\t16 32\n",
      "99\t36\t0.2\t1\t16 20\n",
      "100\t36\t0.2\t1\t14 22\n",
      "101\t54\t0.2\t1\t18 36\n",
      "102\t53\t0.2\t1\t18 35\n",
      "103\t39\t0.2\t1\t20 19\n",
      "104\t35\t0.2\t1\t14 21\n",
      "105\t33\t0.2\t1\t10 23\n",
      "106\t61\t0.2\t1\t19 42\n",
      "107\t43\t0.2\t1\t20 23\n",
      "108\t52\t0.2\t1\t16 36\n",
      "109\t39\t0.2\t1\t17 22\n",
      "110\t38\t0.2\t1\t18 20\n",
      "111\t40\t0.2\t1\t19 21\n",
      "112\t28\t0.2\t1\t14 14\n",
      "113\t35\t0.2\t1\t22 13\n",
      "114\t36\t0.2\t1\t18 18\n",
      "115\t36\t0.2\t1\t18 18\n",
      "116\t36\t0.2\t1\t9 27\n",
      "117\t29\t0.2\t1\t13 16\n",
      "118\t39\t0.2\t1\t17 22\n",
      "119\t37\t0.2\t1\t16 21\n",
      "120\t46\t0.2\t1\t25 21\n",
      "121\t32\t0.2\t1\t22 10\n",
      "122\t33\t0.2\t1\t10 23\n",
      "123\t33\t0.2\t1\t17 16\n",
      "124\t20\t0.2\t1\t7 13\n",
      "125\t31\t0.2\t1\t14 17\n",
      "126\t33\t0.2\t1\t19 14\n",
      "127\t22\t0.2\t1\t11 11\n",
      "128\t21\t0.2\t1\t11 10\n",
      "129\t32\t0.2\t1\t13 19\n",
      "130\t39\t0.2\t1\t21 18\n",
      "131\t34\t0.2\t1\t18 16\n",
      "132\t42\t0.2\t1\t15 27\n",
      "133\t43\t0.2\t1\t22 21\n",
      "134\t30\t0.2\t1\t12 18\n",
      "135\t39\t0.2\t1\t16 23\n",
      "136\t23\t0.2\t1\t13 10\n",
      "137\t38\t0.2\t1\t16 22\n",
      "138\t24\t0.2\t1\t12 12\n",
      "139\t38\t0.2\t1\t25 13\n",
      "140\t54\t0.2\t1\t11 43\n",
      "141\t45\t0.2\t1\t22 23\n",
      "142\t35\t0.2\t1\t16 19\n",
      "143\t43\t0.2\t1\t16 27\n",
      "144\t48\t0.2\t1\t19 29\n",
      "145\t30\t0.2\t1\t10 20\n",
      "146\t55\t0.2\t1\t13 42\n",
      "147\t34\t0.2\t1\t11 23\n",
      "148\t28\t0.2\t1\t15 13\n",
      "149\t17\t0.2\t1\t8 9\n",
      "150\t38\t0.2\t1\t13 25\n",
      "151\t12\t0.2\t1\t4 8\n",
      "\n",
      "\n",
      "RUN STATISTICS FOR INPUT FILE: /home/sam/data/geoduck_illumina/NR006_S3_L001_R2_001.fastq.gz\n",
      "=============================================\n",
      "3427897 sequences processed in total\n",
      "The length threshold of paired-end sequences gets evaluated later on (in the validation step)\n",
      "\n",
      "Validate paired-end files NR006_S3_L001_R1_001_trimmed.fq.gz and NR006_S3_L001_R2_001_trimmed.fq.gz\n",
      "file_1: NR006_S3_L001_R1_001_trimmed.fq.gz, file_2: NR006_S3_L001_R2_001_trimmed.fq.gz\n",
      "\n",
      "\n",
      ">>>>> Now validing the length of the 2 paired-end infiles: NR006_S3_L001_R1_001_trimmed.fq.gz and NR006_S3_L001_R2_001_trimmed.fq.gz <<<<<\n",
      "Writing validated paired-end read 1 reads to NR006_S3_L001_R1_001_val_1.fq.gz\n",
      "Writing validated paired-end read 2 reads to NR006_S3_L001_R2_001_val_2.fq.gz\n",
      "\n",
      "Total number of sequences analysed: 3427897\n",
      "\n",
      "Number of sequence pairs removed because at least one read was shorter than the length cutoff (20 bp): 13219 (0.39%)\n",
      "\n",
      "Deleting both intermediate output files NR006_S3_L001_R1_001_trimmed.fq.gz and NR006_S3_L001_R2_001_trimmed.fq.gz\n",
      "\n",
      "====================================================================================================\n",
      "\n",
      "Writing report to '/home/sam/data/geoduck_illumina/trimmed/NR006_S3_L002_R1_001.fastq.gz_trimming_report.txt'\n",
      "\n",
      "SUMMARISING RUN PARAMETERS\n",
      "==========================\n",
      "Input filename: /home/sam/data/geoduck_illumina/NR006_S3_L002_R1_001.fastq.gz\n",
      "Trimming mode: paired-end\n",
      "Trim Galore version: 0.4.4\n",
      "Cutadapt version: 1.9.1\n",
      "Quality Phred score cutoff: 20\n",
      "Quality encoding type selected: ASCII+33\n",
      "Adapter sequence: 'CTGTCTCTTATA' (Nextera Transposase sequence; auto-detected)\n",
      "Maximum trimming error rate: 0.1 (default)\n",
      "Minimum required adapter overlap (stringency): 1 bp\n",
      "Minimum required sequence length for both reads before a sequence pair gets removed: 20 bp\n",
      "Output file(s) will be GZIP compressed\n",
      "\n",
      "Writing final adapter and quality trimmed output to NR006_S3_L002_R1_001_trimmed.fq.gz\n",
      "\n",
      "\n",
      "  >>> Now performing quality (cutoff 20) and adapter trimming in a single pass for the adapter sequence: 'CTGTCTCTTATA' from file /home/sam/data/geoduck_illumina/NR006_S3_L002_R1_001.fastq.gz <<< \n",
      "This is cutadapt 1.9.1 with Python 2.7.12\n",
      "Command line parameters: -f fastq -e 0.1 -q 20 -O 1 -a CTGTCTCTTATA /home/sam/data/geoduck_illumina/NR006_S3_L002_R1_001.fastq.gz\n",
      "Trimming 1 adapter with at most 10.0% errors in single-end mode ...\n",
      "Finished in 77.02 s (25 us/read; 2.37 M reads/minute).\n",
      "\n",
      "=== Summary ===\n",
      "\n",
      "Total reads processed:               3,045,976\n",
      "Reads with adapters:                   866,851 (28.5%)\n",
      "Reads written (passing filters):     3,045,976 (100.0%)\n",
      "\n",
      "Total basepairs processed:   459,942,376 bp\n",
      "Quality-trimmed:                 610,758 bp (0.1%)\n",
      "Total written (filtered):    457,563,475 bp (99.5%)\n",
      "\n",
      "=== Adapter 1 ===\n",
      "\n",
      "Sequence: CTGTCTCTTATA; Type: regular 3'; Length: 12; Trimmed: 866851 times.\n",
      "\n",
      "No. of allowed errors:\n",
      "0-9 bp: 0; 10-12 bp: 1\n",
      "\n",
      "Bases preceding removed adapters:\n",
      "  A: 30.0%\n",
      "  C: 16.1%\n",
      "  G: 16.1%\n",
      "  T: 37.9%\n",
      "  none/other: 0.0%\n",
      "\n",
      "Overview of removed sequences\n",
      "length\tcount\texpect\tmax.err\terror counts\n",
      "1\t633365\t761494.0\t0\t633365\n",
      "2\t160558\t190373.5\t0\t160558\n",
      "3\t52444\t47593.4\t0\t52444\n",
      "4\t10136\t11898.3\t0\t10136\n",
      "5\t2116\t2974.6\t0\t2116\n",
      "6\t771\t743.6\t0\t771\n",
      "7\t216\t185.9\t0\t216\n",
      "8\t70\t46.5\t0\t70\n",
      "9\t62\t11.6\t0\t16 46\n",
      "10\t99\t2.9\t1\t21 78\n",
      "11\t83\t0.7\t1\t25 58\n",
      "12\t18\t0.2\t1\t8 10\n",
      "13\t34\t0.2\t1\t20 14\n",
      "14\t28\t0.2\t1\t14 14\n",
      "15\t40\t0.2\t1\t21 19\n",
      "16\t39\t0.2\t1\t18 21\n",
      "17\t31\t0.2\t1\t17 14\n",
      "18\t46\t0.2\t1\t24 22\n",
      "19\t38\t0.2\t1\t14 24\n",
      "20\t36\t0.2\t1\t18 18\n",
      "21\t51\t0.2\t1\t23 28\n",
      "22\t44\t0.2\t1\t25 19\n",
      "23\t30\t0.2\t1\t17 13\n",
      "24\t55\t0.2\t1\t21 34\n",
      "25\t31\t0.2\t1\t15 16\n",
      "26\t38\t0.2\t1\t23 15\n",
      "27\t36\t0.2\t1\t15 21\n",
      "28\t45\t0.2\t1\t21 24\n",
      "29\t36\t0.2\t1\t22 14\n",
      "30\t42\t0.2\t1\t17 25\n",
      "31\t59\t0.2\t1\t25 34\n",
      "32\t78\t0.2\t1\t28 50\n",
      "33\t51\t0.2\t1\t16 35\n",
      "34\t47\t0.2\t1\t30 17\n",
      "35\t38\t0.2\t1\t17 21\n",
      "36\t62\t0.2\t1\t9 53\n",
      "37\t40\t0.2\t1\t19 21\n",
      "38\t56\t0.2\t1\t26 30\n",
      "39\t51\t0.2\t1\t26 25\n",
      "40\t46\t0.2\t1\t16 30\n",
      "41\t40\t0.2\t1\t12 28\n",
      "42\t39\t0.2\t1\t20 19\n",
      "43\t52\t0.2\t1\t18 34\n",
      "44\t47\t0.2\t1\t12 35\n",
      "45\t62\t0.2\t1\t18 44\n",
      "46\t47\t0.2\t1\t18 29\n",
      "47\t37\t0.2\t1\t16 21\n",
      "48\t54\t0.2\t1\t23 31\n",
      "49\t34\t0.2\t1\t20 14\n",
      "50\t41\t0.2\t1\t14 27\n",
      "51\t40\t0.2\t1\t19 21\n",
      "52\t63\t0.2\t1\t28 35\n",
      "53\t33\t0.2\t1\t21 12\n",
      "54\t46\t0.2\t1\t27 19\n",
      "55\t39\t0.2\t1\t25 14\n",
      "56\t52\t0.2\t1\t28 24\n",
      "57\t32\t0.2\t1\t15 17\n",
      "58\t37\t0.2\t1\t26 11\n",
      "59\t23\t0.2\t1\t19 4\n",
      "60\t30\t0.2\t1\t19 11\n",
      "61\t44\t0.2\t1\t21 23\n",
      "62\t48\t0.2\t1\t32 16\n",
      "63\t32\t0.2\t1\t26 6\n",
      "64\t51\t0.2\t1\t24 27\n",
      "65\t48\t0.2\t1\t16 32\n",
      "66\t38\t0.2\t1\t24 14\n",
      "67\t42\t0.2\t1\t24 18\n",
      "68\t46\t0.2\t1\t29 17\n",
      "69\t40\t0.2\t1\t24 16\n",
      "70\t35\t0.2\t1\t24 11\n",
      "71\t40\t0.2\t1\t21 19\n",
      "72\t40\t0.2\t1\t27 13\n",
      "73\t42\t0.2\t1\t27 15\n",
      "74\t49\t0.2\t1\t19 30\n",
      "75\t39\t0.2\t1\t28 11\n",
      "76\t36\t0.2\t1\t18 18\n",
      "77\t42\t0.2\t1\t28 14\n",
      "78\t42\t0.2\t1\t31 11\n",
      "79\t54\t0.2\t1\t41 13\n",
      "80\t21\t0.2\t1\t16 5\n",
      "81\t36\t0.2\t1\t23 13\n",
      "82\t74\t0.2\t1\t30 44\n",
      "83\t34\t0.2\t1\t18 16\n",
      "84\t44\t0.2\t1\t28 16\n",
      "85\t49\t0.2\t1\t23 26\n",
      "86\t29\t0.2\t1\t16 13\n",
      "87\t52\t0.2\t1\t28 24\n",
      "88\t57\t0.2\t1\t43 14\n",
      "89\t59\t0.2\t1\t26 33\n",
      "90\t59\t0.2\t1\t27 32\n",
      "91\t51\t0.2\t1\t24 27\n",
      "92\t50\t0.2\t1\t20 30\n",
      "93\t40\t0.2\t1\t25 15\n",
      "94\t76\t0.2\t1\t29 47\n",
      "95\t49\t0.2\t1\t25 24\n",
      "96\t102\t0.2\t1\t34 68\n",
      "97\t56\t0.2\t1\t34 22\n",
      "98\t49\t0.2\t1\t27 22\n",
      "99\t45\t0.2\t1\t25 20\n",
      "100\t57\t0.2\t1\t31 26\n",
      "101\t66\t0.2\t1\t34 32\n",
      "102\t82\t0.2\t1\t23 59\n",
      "103\t53\t0.2\t1\t24 29\n",
      "104\t48\t0.2\t1\t31 17\n",
      "105\t68\t0.2\t1\t26 42\n",
      "106\t125\t0.2\t1\t28 97\n",
      "107\t68\t0.2\t1\t33 35\n",
      "108\t76\t0.2\t1\t26 50\n",
      "109\t58\t0.2\t1\t28 30\n",
      "110\t53\t0.2\t1\t38 15\n",
      "111\t60\t0.2\t1\t32 28\n",
      "112\t63\t0.2\t1\t31 32\n",
      "113\t85\t0.2\t1\t45 40\n",
      "114\t51\t0.2\t1\t30 21\n",
      "115\t70\t0.2\t1\t28 42\n",
      "116\t63\t0.2\t1\t41 22\n",
      "117\t87\t0.2\t1\t34 53\n",
      "118\t54\t0.2\t1\t30 24\n",
      "119\t64\t0.2\t1\t38 26\n",
      "120\t47\t0.2\t1\t31 16\n",
      "121\t71\t0.2\t1\t36 35\n",
      "122\t49\t0.2\t1\t26 23\n",
      "123\t64\t0.2\t1\t45 19\n",
      "124\t47\t0.2\t1\t30 17\n",
      "125\t70\t0.2\t1\t43 27\n",
      "126\t41\t0.2\t1\t24 17\n",
      "127\t45\t0.2\t1\t34 11\n",
      "128\t36\t0.2\t1\t28 8\n",
      "129\t64\t0.2\t1\t34 30\n",
      "130\t58\t0.2\t1\t38 20\n",
      "131\t53\t0.2\t1\t37 16\n",
      "132\t59\t0.2\t1\t37 22\n",
      "133\t49\t0.2\t1\t37 12\n",
      "134\t50\t0.2\t1\t39 11\n",
      "135\t68\t0.2\t1\t36 32\n",
      "136\t43\t0.2\t1\t28 15\n",
      "137\t44\t0.2\t1\t32 12\n",
      "138\t49\t0.2\t1\t29 20\n",
      "139\t44\t0.2\t1\t33 11\n",
      "140\t62\t0.2\t1\t29 33\n",
      "141\t50\t0.2\t1\t28 22\n",
      "142\t41\t0.2\t1\t31 10\n",
      "143\t58\t0.2\t1\t30 28\n",
      "144\t59\t0.2\t1\t38 21\n",
      "145\t55\t0.2\t1\t39 16\n",
      "146\t67\t0.2\t1\t36 31\n",
      "147\t49\t0.2\t1\t31 18\n",
      "148\t62\t0.2\t1\t38 24\n",
      "149\t31\t0.2\t1\t23 8\n",
      "150\t44\t0.2\t1\t20 24\n",
      "151\t17\t0.2\t1\t5 12\n",
      "\n",
      "\n",
      "RUN STATISTICS FOR INPUT FILE: /home/sam/data/geoduck_illumina/NR006_S3_L002_R1_001.fastq.gz\n",
      "=============================================\n",
      "3045976 sequences processed in total\n",
      "The length threshold of paired-end sequences gets evaluated later on (in the validation step)\n",
      "\n",
      "Writing report to '/home/sam/data/geoduck_illumina/trimmed/NR006_S3_L002_R2_001.fastq.gz_trimming_report.txt'\n",
      "\n",
      "SUMMARISING RUN PARAMETERS\n",
      "==========================\n",
      "Input filename: /home/sam/data/geoduck_illumina/NR006_S3_L002_R2_001.fastq.gz\n",
      "Trimming mode: paired-end\n",
      "Trim Galore version: 0.4.4\n",
      "Cutadapt version: 1.9.1\n",
      "Quality Phred score cutoff: 20\n",
      "Quality encoding type selected: ASCII+33\n",
      "Adapter sequence: 'CTGTCTCTTATA' (Nextera Transposase sequence; auto-detected)\n",
      "Maximum trimming error rate: 0.1 (default)\n",
      "Minimum required adapter overlap (stringency): 1 bp\n",
      "Minimum required sequence length for both reads before a sequence pair gets removed: 20 bp\n",
      "Output file(s) will be GZIP compressed\n",
      "\n",
      "Writing final adapter and quality trimmed output to NR006_S3_L002_R2_001_trimmed.fq.gz\n",
      "\n",
      "\n",
      "  >>> Now performing quality (cutoff 20) and adapter trimming in a single pass for the adapter sequence: 'CTGTCTCTTATA' from file /home/sam/data/geoduck_illumina/NR006_S3_L002_R2_001.fastq.gz <<< \n",
      "This is cutadapt 1.9.1 with Python 2.7.12\n",
      "Command line parameters: -f fastq -e 0.1 -q 20 -O 1 -a CTGTCTCTTATA /home/sam/data/geoduck_illumina/NR006_S3_L002_R2_001.fastq.gz\n",
      "Trimming 1 adapter with at most 10.0% errors in single-end mode ...\n",
      "Finished in 73.98 s (24 us/read; 2.47 M reads/minute).\n",
      "\n",
      "=== Summary ===\n",
      "\n",
      "Total reads processed:               3,045,976\n",
      "Reads with adapters:                   565,031 (18.6%)\n",
      "Reads written (passing filters):     3,045,976 (100.0%)\n",
      "\n",
      "Total basepairs processed:   459,942,376 bp\n",
      "Quality-trimmed:               4,313,218 bp (0.9%)\n",
      "Total written (filtered):    454,437,558 bp (98.8%)\n",
      "\n",
      "=== Adapter 1 ===\n",
      "\n",
      "Sequence: CTGTCTCTTATA; Type: regular 3'; Length: 12; Trimmed: 565031 times.\n",
      "\n",
      "No. of allowed errors:\n",
      "0-9 bp: 0; 10-12 bp: 1\n",
      "\n",
      "Bases preceding removed adapters:\n",
      "  A: 28.3%\n",
      "  C: 16.5%\n",
      "  G: 20.0%\n",
      "  T: 35.2%\n",
      "  none/other: 0.0%\n",
      "\n",
      "Overview of removed sequences\n",
      "length\tcount\texpect\tmax.err\terror counts\n",
      "1\t417639\t761494.0\t0\t417639\n",
      "2\t100209\t190373.5\t0\t100209\n",
      "3\t29961\t47593.4\t0\t29961\n",
      "4\t9506\t11898.3\t0\t9506\n",
      "5\t1704\t2974.6\t0\t1704\n",
      "6\t588\t743.6\t0\t588\n",
      "7\t164\t185.9\t0\t164\n",
      "8\t37\t46.5\t0\t37\n",
      "9\t68\t11.6\t0\t22 46\n",
      "10\t73\t2.9\t1\t18 55\n",
      "11\t50\t0.7\t1\t12 38\n",
      "12\t24\t0.2\t1\t15 9\n",
      "13\t44\t0.2\t1\t19 25\n",
      "14\t23\t0.2\t1\t14 9\n",
      "15\t25\t0.2\t1\t12 13\n",
      "16\t37\t0.2\t1\t21 16\n",
      "17\t23\t0.2\t1\t13 10\n",
      "18\t33\t0.2\t1\t19 14\n",
      "19\t31\t0.2\t1\t13 18\n",
      "20\t32\t0.2\t1\t20 12\n",
      "21\t22\t0.2\t1\t9 13\n",
      "22\t27\t0.2\t1\t18 9\n",
      "23\t26\t0.2\t1\t11 15\n",
      "24\t34\t0.2\t1\t22 12\n",
      "25\t36\t0.2\t1\t14 22\n",
      "26\t24\t0.2\t1\t16 8\n",
      "27\t31\t0.2\t1\t14 17\n",
      "28\t34\t0.2\t1\t13 21\n",
      "29\t31\t0.2\t1\t21 10\n",
      "30\t37\t0.2\t1\t14 23\n",
      "31\t38\t0.2\t1\t21 17\n",
      "32\t38\t0.2\t1\t11 27\n",
      "33\t43\t0.2\t1\t19 24\n",
      "34\t16\t0.2\t1\t12 4\n",
      "35\t43\t0.2\t1\t23 20\n",
      "36\t38\t0.2\t1\t19 19\n",
      "37\t36\t0.2\t1\t20 16\n",
      "38\t41\t0.2\t1\t23 18\n",
      "39\t26\t0.2\t1\t12 14\n",
      "40\t29\t0.2\t1\t16 13\n",
      "41\t20\t0.2\t1\t12 8\n",
      "42\t33\t0.2\t1\t17 16\n",
      "43\t32\t0.2\t1\t19 13\n",
      "44\t30\t0.2\t1\t15 15\n",
      "45\t35\t0.2\t1\t11 24\n",
      "46\t27\t0.2\t1\t14 13\n",
      "47\t24\t0.2\t1\t16 8\n",
      "48\t24\t0.2\t1\t20 4\n",
      "49\t37\t0.2\t1\t22 15\n",
      "50\t41\t0.2\t1\t22 19\n",
      "51\t29\t0.2\t1\t20 9\n",
      "52\t35\t0.2\t1\t17 18\n",
      "53\t33\t0.2\t1\t13 20\n",
      "54\t31\t0.2\t1\t17 14\n",
      "55\t30\t0.2\t1\t22 8\n",
      "56\t28\t0.2\t1\t12 16\n",
      "57\t23\t0.2\t1\t13 10\n",
      "58\t29\t0.2\t1\t18 11\n",
      "59\t23\t0.2\t1\t19 4\n",
      "60\t39\t0.2\t1\t22 17\n",
      "61\t33\t0.2\t1\t18 15\n",
      "62\t34\t0.2\t1\t22 12\n",
      "63\t29\t0.2\t1\t17 12\n",
      "64\t29\t0.2\t1\t18 11\n",
      "65\t51\t0.2\t1\t18 33\n",
      "66\t27\t0.2\t1\t11 16\n",
      "67\t45\t0.2\t1\t27 18\n",
      "68\t38\t0.2\t1\t18 20\n",
      "69\t40\t0.2\t1\t16 24\n",
      "70\t25\t0.2\t1\t17 8\n",
      "71\t29\t0.2\t1\t19 10\n",
      "72\t36\t0.2\t1\t23 13\n",
      "73\t29\t0.2\t1\t19 10\n",
      "74\t36\t0.2\t1\t20 16\n",
      "75\t37\t0.2\t1\t22 15\n",
      "76\t33\t0.2\t1\t20 13\n",
      "77\t35\t0.2\t1\t26 9\n",
      "78\t38\t0.2\t1\t14 24\n",
      "79\t33\t0.2\t1\t23 10\n",
      "80\t33\t0.2\t1\t17 16\n",
      "81\t29\t0.2\t1\t20 9\n",
      "82\t44\t0.2\t1\t17 27\n",
      "83\t38\t0.2\t1\t19 19\n",
      "84\t41\t0.2\t1\t24 17\n",
      "85\t29\t0.2\t1\t17 12\n",
      "86\t24\t0.2\t1\t18 6\n",
      "87\t36\t0.2\t1\t22 14\n",
      "88\t29\t0.2\t1\t17 12\n",
      "89\t33\t0.2\t1\t24 9\n",
      "90\t37\t0.2\t1\t21 16\n",
      "91\t42\t0.2\t1\t29 13\n",
      "92\t35\t0.2\t1\t17 18\n",
      "93\t42\t0.2\t1\t25 17\n",
      "94\t31\t0.2\t1\t19 12\n",
      "95\t44\t0.2\t1\t21 23\n",
      "96\t46\t0.2\t1\t25 21\n",
      "97\t40\t0.2\t1\t22 18\n",
      "98\t45\t0.2\t1\t19 26\n",
      "99\t42\t0.2\t1\t19 23\n",
      "100\t42\t0.2\t1\t23 19\n",
      "101\t37\t0.2\t1\t16 21\n",
      "102\t52\t0.2\t1\t29 23\n",
      "103\t55\t0.2\t1\t33 22\n",
      "104\t35\t0.2\t1\t20 15\n",
      "105\t41\t0.2\t1\t20 21\n",
      "106\t50\t0.2\t1\t15 35\n",
      "107\t65\t0.2\t1\t29 36\n",
      "108\t45\t0.2\t1\t24 21\n",
      "109\t41\t0.2\t1\t24 17\n",
      "110\t38\t0.2\t1\t26 12\n",
      "111\t48\t0.2\t1\t32 16\n",
      "112\t32\t0.2\t1\t24 8\n",
      "113\t45\t0.2\t1\t25 20\n",
      "114\t43\t0.2\t1\t26 17\n",
      "115\t40\t0.2\t1\t20 20\n",
      "116\t45\t0.2\t1\t24 21\n",
      "117\t33\t0.2\t1\t23 10\n",
      "118\t42\t0.2\t1\t24 18\n",
      "119\t35\t0.2\t1\t22 13\n",
      "120\t60\t0.2\t1\t34 26\n",
      "121\t50\t0.2\t1\t35 15\n",
      "122\t34\t0.2\t1\t20 14\n",
      "123\t33\t0.2\t1\t14 19\n",
      "124\t28\t0.2\t1\t20 8\n",
      "125\t34\t0.2\t1\t24 10\n",
      "126\t32\t0.2\t1\t20 12\n",
      "127\t40\t0.2\t1\t28 12\n",
      "128\t28\t0.2\t1\t18 10\n",
      "129\t37\t0.2\t1\t21 16\n",
      "130\t59\t0.2\t1\t31 28\n",
      "131\t46\t0.2\t1\t25 21\n",
      "132\t48\t0.2\t1\t30 18\n",
      "133\t41\t0.2\t1\t28 13\n",
      "134\t38\t0.2\t1\t21 17\n",
      "135\t45\t0.2\t1\t22 23\n",
      "136\t25\t0.2\t1\t13 12\n",
      "137\t30\t0.2\t1\t21 9\n",
      "138\t27\t0.2\t1\t17 10\n",
      "139\t32\t0.2\t1\t20 12\n",
      "140\t59\t0.2\t1\t32 27\n",
      "141\t30\t0.2\t1\t22 8\n",
      "142\t36\t0.2\t1\t25 11\n",
      "143\t47\t0.2\t1\t31 16\n",
      "144\t48\t0.2\t1\t30 18\n",
      "145\t40\t0.2\t1\t23 17\n",
      "146\t66\t0.2\t1\t33 33\n",
      "147\t35\t0.2\t1\t19 16\n",
      "148\t33\t0.2\t1\t14 19\n",
      "149\t25\t0.2\t1\t17 8\n",
      "150\t46\t0.2\t1\t19 27\n",
      "151\t14\t0.2\t1\t5 9\n",
      "\n",
      "\n",
      "RUN STATISTICS FOR INPUT FILE: /home/sam/data/geoduck_illumina/NR006_S3_L002_R2_001.fastq.gz\n",
      "=============================================\n",
      "3045976 sequences processed in total\n",
      "The length threshold of paired-end sequences gets evaluated later on (in the validation step)\n",
      "\n",
      "Validate paired-end files NR006_S3_L002_R1_001_trimmed.fq.gz and NR006_S3_L002_R2_001_trimmed.fq.gz\n",
      "file_1: NR006_S3_L002_R1_001_trimmed.fq.gz, file_2: NR006_S3_L002_R2_001_trimmed.fq.gz\n",
      "\n",
      "\n",
      ">>>>> Now validing the length of the 2 paired-end infiles: NR006_S3_L002_R1_001_trimmed.fq.gz and NR006_S3_L002_R2_001_trimmed.fq.gz <<<<<\n",
      "Writing validated paired-end read 1 reads to NR006_S3_L002_R1_001_val_1.fq.gz\n",
      "Writing validated paired-end read 2 reads to NR006_S3_L002_R2_001_val_2.fq.gz\n",
      "\n",
      "Total number of sequences analysed: 3045976\n",
      "\n",
      "Number of sequence pairs removed because at least one read was shorter than the length cutoff (20 bp): 14951 (0.49%)\n",
      "\n",
      "Deleting both intermediate output files NR006_S3_L002_R1_001_trimmed.fq.gz and NR006_S3_L002_R2_001_trimmed.fq.gz\n",
      "\n",
      "====================================================================================================\n",
      "\n",
      "Writing report to '/home/sam/data/geoduck_illumina/trimmed/NR012_S1_L001_R1_001.fastq.gz_trimming_report.txt'\n",
      "\n",
      "SUMMARISING RUN PARAMETERS\n",
      "==========================\n",
      "Input filename: /home/sam/data/geoduck_illumina/NR012_S1_L001_R1_001.fastq.gz\n",
      "Trimming mode: paired-end\n",
      "Trim Galore version: 0.4.4\n",
      "Cutadapt version: 1.9.1\n",
      "Quality Phred score cutoff: 20\n",
      "Quality encoding type selected: ASCII+33\n",
      "Adapter sequence: 'CTGTCTCTTATA' (Nextera Transposase sequence; auto-detected)\n",
      "Maximum trimming error rate: 0.1 (default)\n",
      "Minimum required adapter overlap (stringency): 1 bp\n",
      "Minimum required sequence length for both reads before a sequence pair gets removed: 20 bp\n",
      "Output file(s) will be GZIP compressed\n",
      "\n",
      "Writing final adapter and quality trimmed output to NR012_S1_L001_R1_001_trimmed.fq.gz\n",
      "\n",
      "\n",
      "  >>> Now performing quality (cutoff 20) and adapter trimming in a single pass for the adapter sequence: 'CTGTCTCTTATA' from file /home/sam/data/geoduck_illumina/NR012_S1_L001_R1_001.fastq.gz <<< \n",
      "This is cutadapt 1.9.1 with Python 2.7.12\n",
      "Command line parameters: -f fastq -e 0.1 -q 20 -O 1 -a CTGTCTCTTATA /home/sam/data/geoduck_illumina/NR012_S1_L001_R1_001.fastq.gz\n",
      "Trimming 1 adapter with at most 10.0% errors in single-end mode ...\n",
      "Finished in 7.64 s (25 us/read; 2.44 M reads/minute).\n",
      "\n",
      "=== Summary ===\n",
      "\n",
      "Total reads processed:                 310,937\n",
      "Reads with adapters:                   165,260 (53.1%)\n",
      "Reads written (passing filters):       310,937 (100.0%)\n",
      "\n",
      "Total basepairs processed:    46,951,487 bp\n",
      "Quality-trimmed:                 110,996 bp (0.2%)\n",
      "Total written (filtered):     37,726,449 bp (80.4%)\n",
      "\n",
      "=== Adapter 1 ===\n",
      "\n",
      "Sequence: CTGTCTCTTATA; Type: regular 3'; Length: 12; Trimmed: 165260 times.\n",
      "\n",
      "No. of allowed errors:\n",
      "0-9 bp: 0; 10-12 bp: 1\n",
      "\n",
      "Bases preceding removed adapters:\n",
      "  A: 19.2%\n",
      "  C: 33.1%\n",
      "  G: 19.9%\n",
      "  T: 27.7%\n",
      "  none/other: 0.1%\n",
      "\n",
      "Overview of removed sequences\n",
      "length\tcount\texpect\tmax.err\terror counts\n",
      "1\t38006\t77734.2\t0\t38006\n",
      "2\t11189\t19433.6\t0\t11189\n",
      "3\t3721\t4858.4\t0\t3721\n",
      "4\t1394\t1214.6\t0\t1394\n",
      "5\t833\t303.6\t0\t833\n",
      "6\t691\t75.9\t0\t691\n",
      "7\t644\t19.0\t0\t644\n",
      "8\t689\t4.7\t0\t689\n",
      "9\t629\t1.2\t0\t624 5\n",
      "10\t647\t0.3\t1\t617 30\n",
      "11\t630\t0.1\t1\t598 32\n",
      "12\t636\t0.0\t1\t598 38\n",
      "13\t650\t0.0\t1\t617 33\n",
      "14\t609\t0.0\t1\t570 39\n",
      "15\t727\t0.0\t1\t687 40\n",
      "16\t756\t0.0\t1\t716 40\n",
      "17\t504\t0.0\t1\t478 26\n",
      "18\t642\t0.0\t1\t613 29\n",
      "19\t650\t0.0\t1\t624 26\n",
      "20\t603\t0.0\t1\t562 41\n",
      "21\t622\t0.0\t1\t586 36\n",
      "22\t753\t0.0\t1\t705 48\n",
      "23\t583\t0.0\t1\t545 38\n",
      "24\t673\t0.0\t1\t640 33\n",
      "25\t582\t0.0\t1\t550 32\n",
      "26\t721\t0.0\t1\t692 29\n",
      "27\t658\t0.0\t1\t617 41\n",
      "28\t630\t0.0\t1\t599 31\n",
      "29\t651\t0.0\t1\t631 20\n",
      "30\t684\t0.0\t1\t655 29\n",
      "31\t668\t0.0\t1\t641 27\n",
      "32\t616\t0.0\t1\t587 29\n",
      "33\t643\t0.0\t1\t628 15\n",
      "34\t624\t0.0\t1\t596 28\n",
      "35\t673\t0.0\t1\t631 42\n",
      "36\t695\t0.0\t1\t669 26\n",
      "37\t651\t0.0\t1\t615 36\n",
      "38\t666\t0.0\t1\t632 34\n",
      "39\t697\t0.0\t1\t673 24\n",
      "40\t730\t0.0\t1\t695 35\n",
      "41\t679\t0.0\t1\t648 31\n",
      "42\t669\t0.0\t1\t639 30\n",
      "43\t697\t0.0\t1\t655 42\n",
      "44\t718\t0.0\t1\t684 34\n",
      "45\t703\t0.0\t1\t678 25\n",
      "46\t669\t0.0\t1\t637 32\n",
      "47\t724\t0.0\t1\t699 25\n",
      "48\t659\t0.0\t1\t618 41\n",
      "49\t758\t0.0\t1\t727 31\n",
      "50\t701\t0.0\t1\t682 19\n",
      "51\t701\t0.0\t1\t670 31\n",
      "52\t699\t0.0\t1\t667 32\n",
      "53\t704\t0.0\t1\t673 31\n",
      "54\t728\t0.0\t1\t710 18\n",
      "55\t693\t0.0\t1\t672 21\n",
      "56\t736\t0.0\t1\t701 35\n",
      "57\t701\t0.0\t1\t678 23\n",
      "58\t692\t0.0\t1\t663 29\n",
      "59\t690\t0.0\t1\t663 27\n",
      "60\t712\t0.0\t1\t678 34\n",
      "61\t759\t0.0\t1\t723 36\n",
      "62\t749\t0.0\t1\t723 26\n",
      "63\t761\t0.0\t1\t737 24\n",
      "64\t764\t0.0\t1\t734 30\n",
      "65\t750\t0.0\t1\t718 32\n",
      "66\t729\t0.0\t1\t705 24\n",
      "67\t774\t0.0\t1\t742 32\n",
      "68\t741\t0.0\t1\t710 31\n",
      "69\t734\t0.0\t1\t699 35\n",
      "70\t708\t0.0\t1\t684 24\n",
      "71\t762\t0.0\t1\t740 22\n",
      "72\t782\t0.0\t1\t760 22\n",
      "73\t783\t0.0\t1\t763 20\n",
      "74\t729\t0.0\t1\t699 30\n",
      "75\t736\t0.0\t1\t711 25\n",
      "76\t763\t0.0\t1\t729 34\n",
      "77\t780\t0.0\t1\t743 37\n",
      "78\t749\t0.0\t1\t716 33\n",
      "79\t778\t0.0\t1\t752 26\n",
      "80\t804\t0.0\t1\t771 33\n",
      "81\t758\t0.0\t1\t738 20\n",
      "82\t752\t0.0\t1\t723 29\n",
      "83\t749\t0.0\t1\t725 24\n",
      "84\t761\t0.0\t1\t740 21\n",
      "85\t801\t0.0\t1\t771 30\n",
      "86\t826\t0.0\t1\t799 27\n",
      "87\t783\t0.0\t1\t752 31\n",
      "88\t803\t0.0\t1\t777 26\n",
      "89\t831\t0.0\t1\t803 28\n",
      "90\t796\t0.0\t1\t773 23\n",
      "91\t789\t0.0\t1\t767 22\n",
      "92\t768\t0.0\t1\t739 29\n",
      "93\t780\t0.0\t1\t751 29\n",
      "94\t757\t0.0\t1\t738 19\n",
      "95\t797\t0.0\t1\t771 26\n",
      "96\t815\t0.0\t1\t783 32\n",
      "97\t779\t0.0\t1\t754 25\n",
      "98\t810\t0.0\t1\t796 14\n",
      "99\t872\t0.0\t1\t848 24\n",
      "100\t826\t0.0\t1\t799 27\n",
      "101\t858\t0.0\t1\t833 25\n",
      "102\t846\t0.0\t1\t826 20\n",
      "103\t819\t0.0\t1\t798 21\n",
      "104\t818\t0.0\t1\t802 16\n",
      "105\t806\t0.0\t1\t786 20\n",
      "106\t789\t0.0\t1\t768 21\n",
      "107\t843\t0.0\t1\t817 26\n",
      "108\t876\t0.0\t1\t846 30\n",
      "109\t841\t0.0\t1\t815 26\n",
      "110\t823\t0.0\t1\t796 27\n",
      "111\t824\t0.0\t1\t802 22\n",
      "112\t883\t0.0\t1\t863 20\n",
      "113\t884\t0.0\t1\t846 38\n",
      "114\t798\t0.0\t1\t780 18\n",
      "115\t802\t0.0\t1\t780 22\n",
      "116\t815\t0.0\t1\t794 21\n",
      "117\t872\t0.0\t1\t844 28\n",
      "118\t846\t0.0\t1\t821 25\n",
      "119\t875\t0.0\t1\t850 25\n",
      "120\t856\t0.0\t1\t840 16\n",
      "121\t842\t0.0\t1\t818 24\n",
      "122\t866\t0.0\t1\t844 22\n",
      "123\t820\t0.0\t1\t788 32\n",
      "124\t825\t0.0\t1\t804 21\n",
      "125\t930\t0.0\t1\t912 18\n",
      "126\t879\t0.0\t1\t864 15\n",
      "127\t929\t0.0\t1\t906 23\n",
      "128\t865\t0.0\t1\t848 17\n",
      "129\t919\t0.0\t1\t896 23\n",
      "130\t891\t0.0\t1\t870 21\n",
      "131\t976\t0.0\t1\t960 16\n",
      "132\t857\t0.0\t1\t839 18\n",
      "133\t908\t0.0\t1\t884 24\n",
      "134\t836\t0.0\t1\t823 13\n",
      "135\t781\t0.0\t1\t772 9\n",
      "136\t771\t0.0\t1\t758 13\n",
      "137\t828\t0.0\t1\t819 9\n",
      "138\t807\t0.0\t1\t794 13\n",
      "139\t809\t0.0\t1\t802 7\n",
      "140\t873\t0.0\t1\t862 11\n",
      "141\t908\t0.0\t1\t897 11\n",
      "142\t778\t0.0\t1\t766 12\n",
      "143\t765\t0.0\t1\t753 12\n",
      "144\t850\t0.0\t1\t840 10\n",
      "145\t802\t0.0\t1\t795 7\n",
      "146\t901\t0.0\t1\t887 14\n",
      "147\t797\t0.0\t1\t782 15\n",
      "148\t843\t0.0\t1\t826 17\n",
      "149\t686\t0.0\t1\t670 16\n",
      "150\t675\t0.0\t1\t663 12\n",
      "151\t178\t0.0\t1\t159 19\n",
      "\n",
      "\n",
      "RUN STATISTICS FOR INPUT FILE: /home/sam/data/geoduck_illumina/NR012_S1_L001_R1_001.fastq.gz\n",
      "=============================================\n",
      "310937 sequences processed in total\n",
      "The length threshold of paired-end sequences gets evaluated later on (in the validation step)\n",
      "\n",
      "Writing report to '/home/sam/data/geoduck_illumina/trimmed/NR012_S1_L001_R2_001.fastq.gz_trimming_report.txt'\n",
      "\n",
      "SUMMARISING RUN PARAMETERS\n",
      "==========================\n",
      "Input filename: /home/sam/data/geoduck_illumina/NR012_S1_L001_R2_001.fastq.gz\n",
      "Trimming mode: paired-end\n",
      "Trim Galore version: 0.4.4\n",
      "Cutadapt version: 1.9.1\n",
      "Quality Phred score cutoff: 20\n",
      "Quality encoding type selected: ASCII+33\n",
      "Adapter sequence: 'CTGTCTCTTATA' (Nextera Transposase sequence; auto-detected)\n",
      "Maximum trimming error rate: 0.1 (default)\n",
      "Minimum required adapter overlap (stringency): 1 bp\n",
      "Minimum required sequence length for both reads before a sequence pair gets removed: 20 bp\n",
      "Output file(s) will be GZIP compressed\n",
      "\n",
      "Writing final adapter and quality trimmed output to NR012_S1_L001_R2_001_trimmed.fq.gz\n",
      "\n",
      "\n",
      "  >>> Now performing quality (cutoff 20) and adapter trimming in a single pass for the adapter sequence: 'CTGTCTCTTATA' from file /home/sam/data/geoduck_illumina/NR012_S1_L001_R2_001.fastq.gz <<< \n",
      "This is cutadapt 1.9.1 with Python 2.7.12\n",
      "Command line parameters: -f fastq -e 0.1 -q 20 -O 1 -a CTGTCTCTTATA /home/sam/data/geoduck_illumina/NR012_S1_L001_R2_001.fastq.gz\n",
      "Trimming 1 adapter with at most 10.0% errors in single-end mode ...\n",
      "Finished in 7.84 s (25 us/read; 2.38 M reads/minute).\n",
      "\n",
      "=== Summary ===\n",
      "\n",
      "Total reads processed:                 310,937\n",
      "Reads with adapters:                   148,571 (47.8%)\n",
      "Reads written (passing filters):       310,937 (100.0%)\n",
      "\n",
      "Total basepairs processed:    46,951,487 bp\n",
      "Quality-trimmed:                 185,152 bp (0.4%)\n",
      "Total written (filtered):     38,811,744 bp (82.7%)\n",
      "\n",
      "=== Adapter 1 ===\n",
      "\n",
      "Sequence: CTGTCTCTTATA; Type: regular 3'; Length: 12; Trimmed: 148571 times.\n",
      "\n",
      "No. of allowed errors:\n",
      "0-9 bp: 0; 10-12 bp: 1\n",
      "\n",
      "Bases preceding removed adapters:\n",
      "  A: 19.2%\n",
      "  C: 32.6%\n",
      "  G: 20.8%\n",
      "  T: 27.3%\n",
      "  none/other: 0.1%\n",
      "\n",
      "Overview of removed sequences\n",
      "length\tcount\texpect\tmax.err\terror counts\n",
      "1\t35513\t77734.2\t0\t35513\n",
      "2\t9836\t19433.6\t0\t9836\n",
      "3\t3265\t4858.4\t0\t3265\n",
      "4\t1404\t1214.6\t0\t1404\n",
      "5\t753\t303.6\t0\t753\n",
      "6\t576\t75.9\t0\t576\n",
      "7\t551\t19.0\t0\t551\n",
      "8\t641\t4.7\t0\t641\n",
      "9\t580\t1.2\t0\t570 10\n",
      "10\t609\t0.3\t1\t544 65\n",
      "11\t588\t0.1\t1\t527 61\n",
      "12\t603\t0.0\t1\t558 45\n",
      "13\t631\t0.0\t1\t575 56\n",
      "14\t520\t0.0\t1\t472 48\n",
      "15\t654\t0.0\t1\t576 78\n",
      "16\t708\t0.0\t1\t638 70\n",
      "17\t453\t0.0\t1\t419 34\n",
      "18\t629\t0.0\t1\t583 46\n",
      "19\t675\t0.0\t1\t636 39\n",
      "20\t567\t0.0\t1\t509 58\n",
      "21\t651\t0.0\t1\t586 65\n",
      "22\t705\t0.0\t1\t647 58\n",
      "23\t509\t0.0\t1\t459 50\n",
      "24\t603\t0.0\t1\t558 45\n",
      "25\t620\t0.0\t1\t574 46\n",
      "26\t675\t0.0\t1\t608 67\n",
      "27\t649\t0.0\t1\t607 42\n",
      "28\t559\t0.0\t1\t512 47\n",
      "29\t649\t0.0\t1\t605 44\n",
      "30\t677\t0.0\t1\t626 51\n",
      "31\t577\t0.0\t1\t536 41\n",
      "32\t609\t0.0\t1\t550 59\n",
      "33\t566\t0.0\t1\t508 58\n",
      "34\t630\t0.0\t1\t573 57\n",
      "35\t563\t0.0\t1\t518 45\n",
      "36\t696\t0.0\t1\t647 49\n",
      "37\t551\t0.0\t1\t506 45\n",
      "38\t672\t0.0\t1\t612 60\n",
      "39\t661\t0.0\t1\t613 48\n",
      "40\t628\t0.0\t1\t586 42\n",
      "41\t627\t0.0\t1\t573 54\n",
      "42\t676\t0.0\t1\t641 35\n",
      "43\t616\t0.0\t1\t578 38\n",
      "44\t579\t0.0\t1\t547 32\n",
      "45\t645\t0.0\t1\t603 42\n",
      "46\t601\t0.0\t1\t556 45\n",
      "47\t652\t0.0\t1\t610 42\n",
      "48\t669\t0.0\t1\t627 42\n",
      "49\t600\t0.0\t1\t544 56\n",
      "50\t679\t0.0\t1\t625 54\n",
      "51\t618\t0.0\t1\t570 48\n",
      "52\t611\t0.0\t1\t557 54\n",
      "53\t601\t0.0\t1\t574 27\n",
      "54\t646\t0.0\t1\t596 50\n",
      "55\t637\t0.0\t1\t594 43\n",
      "56\t658\t0.0\t1\t618 40\n",
      "57\t611\t0.0\t1\t576 35\n",
      "58\t652\t0.0\t1\t613 39\n",
      "59\t670\t0.0\t1\t624 46\n",
      "60\t658\t0.0\t1\t619 39\n",
      "61\t645\t0.0\t1\t611 34\n",
      "62\t678\t0.0\t1\t641 37\n",
      "63\t714\t0.0\t1\t668 46\n",
      "64\t617\t0.0\t1\t581 36\n",
      "65\t669\t0.0\t1\t634 35\n",
      "66\t673\t0.0\t1\t633 40\n",
      "67\t624\t0.0\t1\t585 39\n",
      "68\t650\t0.0\t1\t606 44\n",
      "69\t650\t0.0\t1\t610 40\n",
      "70\t725\t0.0\t1\t679 46\n",
      "71\t662\t0.0\t1\t628 34\n",
      "72\t691\t0.0\t1\t652 39\n",
      "73\t701\t0.0\t1\t670 31\n",
      "74\t693\t0.0\t1\t657 36\n",
      "75\t709\t0.0\t1\t679 30\n",
      "76\t682\t0.0\t1\t637 45\n",
      "77\t749\t0.0\t1\t706 43\n",
      "78\t674\t0.0\t1\t635 39\n",
      "79\t685\t0.0\t1\t654 31\n",
      "80\t646\t0.0\t1\t613 33\n",
      "81\t697\t0.0\t1\t665 32\n",
      "82\t690\t0.0\t1\t651 39\n",
      "83\t714\t0.0\t1\t671 43\n",
      "84\t686\t0.0\t1\t657 29\n",
      "85\t692\t0.0\t1\t657 35\n",
      "86\t688\t0.0\t1\t660 28\n",
      "87\t657\t0.0\t1\t618 39\n",
      "88\t700\t0.0\t1\t668 32\n",
      "89\t724\t0.0\t1\t689 35\n",
      "90\t655\t0.0\t1\t624 31\n",
      "91\t740\t0.0\t1\t700 40\n",
      "92\t671\t0.0\t1\t641 30\n",
      "93\t724\t0.0\t1\t691 33\n",
      "94\t738\t0.0\t1\t715 23\n",
      "95\t703\t0.0\t1\t671 32\n",
      "96\t737\t0.0\t1\t700 37\n",
      "97\t685\t0.0\t1\t648 37\n",
      "98\t685\t0.0\t1\t655 30\n",
      "99\t731\t0.0\t1\t702 29\n",
      "100\t744\t0.0\t1\t717 27\n",
      "101\t726\t0.0\t1\t690 36\n",
      "102\t738\t0.0\t1\t718 20\n",
      "103\t713\t0.0\t1\t690 23\n",
      "104\t722\t0.0\t1\t680 42\n",
      "105\t700\t0.0\t1\t675 25\n",
      "106\t703\t0.0\t1\t674 29\n",
      "107\t731\t0.0\t1\t696 35\n",
      "108\t783\t0.0\t1\t744 39\n",
      "109\t724\t0.0\t1\t694 30\n",
      "110\t696\t0.0\t1\t672 24\n",
      "111\t802\t0.0\t1\t768 34\n",
      "112\t737\t0.0\t1\t719 18\n",
      "113\t752\t0.0\t1\t722 30\n",
      "114\t742\t0.0\t1\t717 25\n",
      "115\t715\t0.0\t1\t691 24\n",
      "116\t732\t0.0\t1\t701 31\n",
      "117\t743\t0.0\t1\t697 46\n",
      "118\t745\t0.0\t1\t705 40\n",
      "119\t726\t0.0\t1\t703 23\n",
      "120\t752\t0.0\t1\t730 22\n",
      "121\t723\t0.0\t1\t701 22\n",
      "122\t779\t0.0\t1\t745 34\n",
      "123\t787\t0.0\t1\t760 27\n",
      "124\t720\t0.0\t1\t689 31\n",
      "125\t752\t0.0\t1\t731 21\n",
      "126\t723\t0.0\t1\t689 34\n",
      "127\t719\t0.0\t1\t689 30\n",
      "128\t714\t0.0\t1\t682 32\n",
      "129\t788\t0.0\t1\t750 38\n",
      "130\t737\t0.0\t1\t714 23\n",
      "131\t796\t0.0\t1\t771 25\n",
      "132\t770\t0.0\t1\t739 31\n",
      "133\t755\t0.0\t1\t735 20\n",
      "134\t760\t0.0\t1\t739 21\n",
      "135\t705\t0.0\t1\t688 17\n",
      "136\t652\t0.0\t1\t632 20\n",
      "137\t687\t0.0\t1\t669 18\n",
      "138\t664\t0.0\t1\t641 23\n",
      "139\t658\t0.0\t1\t632 26\n",
      "140\t686\t0.0\t1\t670 16\n",
      "141\t742\t0.0\t1\t721 21\n",
      "142\t638\t0.0\t1\t615 23\n",
      "143\t663\t0.0\t1\t636 27\n",
      "144\t719\t0.0\t1\t700 19\n",
      "145\t746\t0.0\t1\t737 9\n",
      "146\t766\t0.0\t1\t748 18\n",
      "147\t737\t0.0\t1\t720 17\n",
      "148\t632\t0.0\t1\t619 13\n",
      "149\t536\t0.0\t1\t527 9\n",
      "150\t507\t0.0\t1\t492 15\n",
      "151\t118\t0.0\t1\t104 14\n",
      "\n",
      "\n",
      "RUN STATISTICS FOR INPUT FILE: /home/sam/data/geoduck_illumina/NR012_S1_L001_R2_001.fastq.gz\n",
      "=============================================\n",
      "310937 sequences processed in total\n",
      "The length threshold of paired-end sequences gets evaluated later on (in the validation step)\n",
      "\n",
      "Validate paired-end files NR012_S1_L001_R1_001_trimmed.fq.gz and NR012_S1_L001_R2_001_trimmed.fq.gz\n",
      "file_1: NR012_S1_L001_R1_001_trimmed.fq.gz, file_2: NR012_S1_L001_R2_001_trimmed.fq.gz\n",
      "\n",
      "\n",
      ">>>>> Now validing the length of the 2 paired-end infiles: NR012_S1_L001_R1_001_trimmed.fq.gz and NR012_S1_L001_R2_001_trimmed.fq.gz <<<<<\n",
      "Writing validated paired-end read 1 reads to NR012_S1_L001_R1_001_val_1.fq.gz\n",
      "Writing validated paired-end read 2 reads to NR012_S1_L001_R2_001_val_2.fq.gz\n",
      "\n",
      "Total number of sequences analysed: 310937\n",
      "\n",
      "Number of sequence pairs removed because at least one read was shorter than the length cutoff (20 bp): 29465 (9.48%)\n",
      "\n",
      "Deleting both intermediate output files NR012_S1_L001_R1_001_trimmed.fq.gz and NR012_S1_L001_R2_001_trimmed.fq.gz\n",
      "\n",
      "====================================================================================================\n",
      "\n",
      "Writing report to '/home/sam/data/geoduck_illumina/trimmed/NR012_S1_L002_R1_001.fastq.gz_trimming_report.txt'\n",
      "\n",
      "SUMMARISING RUN PARAMETERS\n",
      "==========================\n",
      "Input filename: /home/sam/data/geoduck_illumina/NR012_S1_L002_R1_001.fastq.gz\n",
      "Trimming mode: paired-end\n",
      "Trim Galore version: 0.4.4\n",
      "Cutadapt version: 1.9.1\n",
      "Quality Phred score cutoff: 20\n",
      "Quality encoding type selected: ASCII+33\n",
      "Adapter sequence: 'CTGTCTCTTATA' (Nextera Transposase sequence; auto-detected)\n",
      "Maximum trimming error rate: 0.1 (default)\n",
      "Minimum required adapter overlap (stringency): 1 bp\n",
      "Minimum required sequence length for both reads before a sequence pair gets removed: 20 bp\n",
      "Output file(s) will be GZIP compressed\n",
      "\n",
      "Writing final adapter and quality trimmed output to NR012_S1_L002_R1_001_trimmed.fq.gz\n",
      "\n",
      "\n",
      "  >>> Now performing quality (cutoff 20) and adapter trimming in a single pass for the adapter sequence: 'CTGTCTCTTATA' from file /home/sam/data/geoduck_illumina/NR012_S1_L002_R1_001.fastq.gz <<< \n",
      "This is cutadapt 1.9.1 with Python 2.7.12\n",
      "Command line parameters: -f fastq -e 0.1 -q 20 -O 1 -a CTGTCTCTTATA /home/sam/data/geoduck_illumina/NR012_S1_L002_R1_001.fastq.gz\n",
      "Trimming 1 adapter with at most 10.0% errors in single-end mode ...\n",
      "Finished in 8.82 s (25 us/read; 2.44 M reads/minute).\n",
      "\n",
      "=== Summary ===\n",
      "\n",
      "Total reads processed:                 358,756\n",
      "Reads with adapters:                   186,719 (52.0%)\n",
      "Reads written (passing filters):       358,756 (100.0%)\n",
      "\n",
      "Total basepairs processed:    54,172,156 bp\n",
      "Quality-trimmed:                 132,160 bp (0.2%)\n",
      "Total written (filtered):     43,801,446 bp (80.9%)\n",
      "\n",
      "=== Adapter 1 ===\n",
      "\n",
      "Sequence: CTGTCTCTTATA; Type: regular 3'; Length: 12; Trimmed: 186719 times.\n",
      "\n",
      "No. of allowed errors:\n",
      "0-9 bp: 0; 10-12 bp: 1\n",
      "\n",
      "Bases preceding removed adapters:\n",
      "  A: 19.2%\n",
      "  C: 33.2%\n",
      "  G: 20.2%\n",
      "  T: 27.3%\n",
      "  none/other: 0.1%\n",
      "\n",
      "Overview of removed sequences\n",
      "length\tcount\texpect\tmax.err\terror counts\n",
      "1\t43381\t89689.0\t0\t43381\n",
      "2\t12814\t22422.2\t0\t12814\n",
      "3\t4159\t5605.6\t0\t4159\n",
      "4\t1624\t1401.4\t0\t1624\n",
      "5\t948\t350.3\t0\t948\n",
      "6\t706\t87.6\t0\t706\n",
      "7\t729\t21.9\t0\t729\n",
      "8\t787\t5.5\t0\t787\n",
      "9\t700\t1.4\t0\t694 6\n",
      "10\t769\t0.3\t1\t729 40\n",
      "11\t713\t0.1\t1\t687 26\n",
      "12\t671\t0.0\t1\t648 23\n",
      "13\t690\t0.0\t1\t668 22\n",
      "14\t669\t0.0\t1\t637 32\n",
      "15\t754\t0.0\t1\t725 29\n",
      "16\t907\t0.0\t1\t870 37\n",
      "17\t503\t0.0\t1\t488 15\n",
      "18\t788\t0.0\t1\t770 18\n",
      "19\t765\t0.0\t1\t749 16\n",
      "20\t744\t0.0\t1\t716 28\n",
      "21\t734\t0.0\t1\t715 19\n",
      "22\t834\t0.0\t1\t810 24\n",
      "23\t630\t0.0\t1\t608 22\n",
      "24\t727\t0.0\t1\t705 22\n",
      "25\t685\t0.0\t1\t664 21\n",
      "26\t823\t0.0\t1\t794 29\n",
      "27\t776\t0.0\t1\t750 26\n",
      "28\t785\t0.0\t1\t758 27\n",
      "29\t776\t0.0\t1\t759 17\n",
      "30\t763\t0.0\t1\t738 25\n",
      "31\t734\t0.0\t1\t711 23\n",
      "32\t759\t0.0\t1\t740 19\n",
      "33\t708\t0.0\t1\t686 22\n",
      "34\t712\t0.0\t1\t696 16\n",
      "35\t821\t0.0\t1\t793 28\n",
      "36\t796\t0.0\t1\t767 29\n",
      "37\t726\t0.0\t1\t703 23\n",
      "38\t817\t0.0\t1\t797 20\n",
      "39\t766\t0.0\t1\t738 28\n",
      "40\t788\t0.0\t1\t766 22\n",
      "41\t751\t0.0\t1\t726 25\n",
      "42\t779\t0.0\t1\t757 22\n",
      "43\t750\t0.0\t1\t721 29\n",
      "44\t816\t0.0\t1\t793 23\n",
      "45\t746\t0.0\t1\t724 22\n",
      "46\t780\t0.0\t1\t749 31\n",
      "47\t825\t0.0\t1\t802 23\n",
      "48\t798\t0.0\t1\t781 17\n",
      "49\t790\t0.0\t1\t771 19\n",
      "50\t791\t0.0\t1\t781 10\n",
      "51\t804\t0.0\t1\t780 24\n",
      "52\t790\t0.0\t1\t765 25\n",
      "53\t755\t0.0\t1\t738 17\n",
      "54\t805\t0.0\t1\t780 25\n",
      "55\t739\t0.0\t1\t723 16\n",
      "56\t802\t0.0\t1\t776 26\n",
      "57\t779\t0.0\t1\t747 32\n",
      "58\t837\t0.0\t1\t809 28\n",
      "59\t823\t0.0\t1\t802 21\n",
      "60\t813\t0.0\t1\t796 17\n",
      "61\t816\t0.0\t1\t796 20\n",
      "62\t845\t0.0\t1\t829 16\n",
      "63\t802\t0.0\t1\t779 23\n",
      "64\t770\t0.0\t1\t739 31\n",
      "65\t845\t0.0\t1\t822 23\n",
      "66\t766\t0.0\t1\t744 22\n",
      "67\t834\t0.0\t1\t804 30\n",
      "68\t905\t0.0\t1\t879 26\n",
      "69\t789\t0.0\t1\t771 18\n",
      "70\t851\t0.0\t1\t829 22\n",
      "71\t871\t0.0\t1\t850 21\n",
      "72\t844\t0.0\t1\t821 23\n",
      "73\t860\t0.0\t1\t839 21\n",
      "74\t856\t0.0\t1\t845 11\n",
      "75\t838\t0.0\t1\t812 26\n",
      "76\t900\t0.0\t1\t874 26\n",
      "77\t870\t0.0\t1\t854 16\n",
      "78\t917\t0.0\t1\t881 36\n",
      "79\t820\t0.0\t1\t807 13\n",
      "80\t844\t0.0\t1\t821 23\n",
      "81\t875\t0.0\t1\t850 25\n",
      "82\t886\t0.0\t1\t865 21\n",
      "83\t842\t0.0\t1\t817 25\n",
      "84\t872\t0.0\t1\t850 22\n",
      "85\t857\t0.0\t1\t839 18\n",
      "86\t803\t0.0\t1\t784 19\n",
      "87\t880\t0.0\t1\t856 24\n",
      "88\t926\t0.0\t1\t902 24\n",
      "89\t882\t0.0\t1\t860 22\n",
      "90\t973\t0.0\t1\t953 20\n",
      "91\t893\t0.0\t1\t874 19\n",
      "92\t873\t0.0\t1\t858 15\n",
      "93\t846\t0.0\t1\t834 12\n",
      "94\t851\t0.0\t1\t832 19\n",
      "95\t867\t0.0\t1\t848 19\n",
      "96\t914\t0.0\t1\t898 16\n",
      "97\t920\t0.0\t1\t898 22\n",
      "98\t895\t0.0\t1\t872 23\n",
      "99\t918\t0.0\t1\t906 12\n",
      "100\t932\t0.0\t1\t918 14\n",
      "101\t945\t0.0\t1\t925 20\n",
      "102\t914\t0.0\t1\t901 13\n",
      "103\t906\t0.0\t1\t891 15\n",
      "104\t889\t0.0\t1\t873 16\n",
      "105\t927\t0.0\t1\t911 16\n",
      "106\t960\t0.0\t1\t946 14\n",
      "107\t958\t0.0\t1\t948 10\n",
      "108\t934\t0.0\t1\t917 17\n",
      "109\t917\t0.0\t1\t892 25\n",
      "110\t928\t0.0\t1\t908 20\n",
      "111\t998\t0.0\t1\t975 23\n",
      "112\t930\t0.0\t1\t903 27\n",
      "113\t966\t0.0\t1\t952 14\n",
      "114\t997\t0.0\t1\t982 15\n",
      "115\t917\t0.0\t1\t892 25\n",
      "116\t893\t0.0\t1\t875 18\n",
      "117\t956\t0.0\t1\t941 15\n",
      "118\t963\t0.0\t1\t943 20\n",
      "119\t931\t0.0\t1\t908 23\n",
      "120\t1010\t0.0\t1\t998 12\n",
      "121\t1004\t0.0\t1\t992 12\n",
      "122\t964\t0.0\t1\t953 11\n",
      "123\t964\t0.0\t1\t943 21\n",
      "124\t988\t0.0\t1\t971 17\n",
      "125\t986\t0.0\t1\t970 16\n",
      "126\t958\t0.0\t1\t950 8\n",
      "127\t1019\t0.0\t1\t1003 16\n",
      "128\t944\t0.0\t1\t930 14\n",
      "129\t1032\t0.0\t1\t1014 18\n",
      "130\t970\t0.0\t1\t956 14\n",
      "131\t1060\t0.0\t1\t1050 10\n",
      "132\t1003\t0.0\t1\t998 5\n",
      "133\t989\t0.0\t1\t980 9\n",
      "134\t966\t0.0\t1\t955 11\n",
      "135\t977\t0.0\t1\t972 5\n",
      "136\t954\t0.0\t1\t945 9\n",
      "137\t886\t0.0\t1\t871 15\n",
      "138\t886\t0.0\t1\t877 9\n",
      "139\t850\t0.0\t1\t842 8\n",
      "140\t898\t0.0\t1\t892 6\n",
      "141\t943\t0.0\t1\t933 10\n",
      "142\t864\t0.0\t1\t851 13\n",
      "143\t889\t0.0\t1\t883 6\n",
      "144\t1000\t0.0\t1\t997 3\n",
      "145\t936\t0.0\t1\t928 8\n",
      "146\t1039\t0.0\t1\t1025 14\n",
      "147\t934\t0.0\t1\t927 7\n",
      "148\t944\t0.0\t1\t939 5\n",
      "149\t754\t0.0\t1\t746 8\n",
      "150\t776\t0.0\t1\t765 11\n",
      "151\t201\t0.0\t1\t181 20\n",
      "\n",
      "\n",
      "RUN STATISTICS FOR INPUT FILE: /home/sam/data/geoduck_illumina/NR012_S1_L002_R1_001.fastq.gz\n",
      "=============================================\n",
      "358756 sequences processed in total\n",
      "The length threshold of paired-end sequences gets evaluated later on (in the validation step)\n",
      "\n",
      "Writing report to '/home/sam/data/geoduck_illumina/trimmed/NR012_S1_L002_R2_001.fastq.gz_trimming_report.txt'\n",
      "\n",
      "SUMMARISING RUN PARAMETERS\n",
      "==========================\n",
      "Input filename: /home/sam/data/geoduck_illumina/NR012_S1_L002_R2_001.fastq.gz\n",
      "Trimming mode: paired-end\n",
      "Trim Galore version: 0.4.4\n",
      "Cutadapt version: 1.9.1\n",
      "Quality Phred score cutoff: 20\n",
      "Quality encoding type selected: ASCII+33\n",
      "Adapter sequence: 'CTGTCTCTTATA' (Nextera Transposase sequence; auto-detected)\n",
      "Maximum trimming error rate: 0.1 (default)\n",
      "Minimum required adapter overlap (stringency): 1 bp\n",
      "Minimum required sequence length for both reads before a sequence pair gets removed: 20 bp\n",
      "Output file(s) will be GZIP compressed\n",
      "\n",
      "Writing final adapter and quality trimmed output to NR012_S1_L002_R2_001_trimmed.fq.gz\n",
      "\n",
      "\n",
      "  >>> Now performing quality (cutoff 20) and adapter trimming in a single pass for the adapter sequence: 'CTGTCTCTTATA' from file /home/sam/data/geoduck_illumina/NR012_S1_L002_R2_001.fastq.gz <<< \n",
      "This is cutadapt 1.9.1 with Python 2.7.12\n",
      "Command line parameters: -f fastq -e 0.1 -q 20 -O 1 -a CTGTCTCTTATA /home/sam/data/geoduck_illumina/NR012_S1_L002_R2_001.fastq.gz\n",
      "Trimming 1 adapter with at most 10.0% errors in single-end mode ...\n",
      "Finished in 8.62 s (24 us/read; 2.50 M reads/minute).\n",
      "\n",
      "=== Summary ===\n",
      "\n",
      "Total reads processed:                 358,756\n",
      "Reads with adapters:                   168,067 (46.8%)\n",
      "Reads written (passing filters):       358,756 (100.0%)\n",
      "\n",
      "Total basepairs processed:    54,172,156 bp\n",
      "Quality-trimmed:                 213,647 bp (0.4%)\n",
      "Total written (filtered):     45,232,988 bp (83.5%)\n",
      "\n",
      "=== Adapter 1 ===\n",
      "\n",
      "Sequence: CTGTCTCTTATA; Type: regular 3'; Length: 12; Trimmed: 168067 times.\n",
      "\n",
      "No. of allowed errors:\n",
      "0-9 bp: 0; 10-12 bp: 1\n",
      "\n",
      "Bases preceding removed adapters:\n",
      "  A: 19.6%\n",
      "  C: 32.2%\n",
      "  G: 20.8%\n",
      "  T: 27.2%\n",
      "  none/other: 0.1%\n",
      "\n",
      "Overview of removed sequences\n",
      "length\tcount\texpect\tmax.err\terror counts\n",
      "1\t41199\t89689.0\t0\t41199\n",
      "2\t12173\t22422.2\t0\t12173\n",
      "3\t3946\t5605.6\t0\t3946\n",
      "4\t1645\t1401.4\t0\t1645\n",
      "5\t906\t350.3\t0\t906\n",
      "6\t713\t87.6\t0\t713\n",
      "7\t700\t21.9\t0\t700\n",
      "8\t706\t5.5\t0\t706\n",
      "9\t594\t1.4\t0\t584 10\n",
      "10\t660\t0.3\t1\t611 49\n",
      "11\t689\t0.1\t1\t642 47\n",
      "12\t649\t0.0\t1\t604 45\n",
      "13\t754\t0.0\t1\t699 55\n",
      "14\t629\t0.0\t1\t597 32\n",
      "15\t819\t0.0\t1\t771 48\n",
      "16\t829\t0.0\t1\t770 59\n",
      "17\t469\t0.0\t1\t443 26\n",
      "18\t684\t0.0\t1\t634 50\n",
      "19\t702\t0.0\t1\t661 41\n",
      "20\t591\t0.0\t1\t556 35\n",
      "21\t736\t0.0\t1\t690 46\n",
      "22\t768\t0.0\t1\t735 33\n",
      "23\t566\t0.0\t1\t539 27\n",
      "24\t693\t0.0\t1\t642 51\n",
      "25\t595\t0.0\t1\t561 34\n",
      "26\t741\t0.0\t1\t680 61\n",
      "27\t748\t0.0\t1\t705 43\n",
      "28\t660\t0.0\t1\t627 33\n",
      "29\t757\t0.0\t1\t718 39\n",
      "30\t710\t0.0\t1\t678 32\n",
      "31\t701\t0.0\t1\t666 35\n",
      "32\t714\t0.0\t1\t667 47\n",
      "33\t669\t0.0\t1\t633 36\n",
      "34\t711\t0.0\t1\t675 36\n",
      "35\t693\t0.0\t1\t660 33\n",
      "36\t793\t0.0\t1\t754 39\n",
      "37\t680\t0.0\t1\t653 27\n",
      "38\t755\t0.0\t1\t715 40\n",
      "39\t716\t0.0\t1\t680 36\n",
      "40\t696\t0.0\t1\t665 31\n",
      "41\t739\t0.0\t1\t699 40\n",
      "42\t681\t0.0\t1\t640 41\n",
      "43\t737\t0.0\t1\t700 37\n",
      "44\t696\t0.0\t1\t656 40\n",
      "45\t742\t0.0\t1\t706 36\n",
      "46\t697\t0.0\t1\t652 45\n",
      "47\t723\t0.0\t1\t684 39\n",
      "48\t680\t0.0\t1\t656 24\n",
      "49\t695\t0.0\t1\t670 25\n",
      "50\t705\t0.0\t1\t664 41\n",
      "51\t708\t0.0\t1\t677 31\n",
      "52\t715\t0.0\t1\t680 35\n",
      "53\t715\t0.0\t1\t682 33\n",
      "54\t710\t0.0\t1\t684 26\n",
      "55\t701\t0.0\t1\t674 27\n",
      "56\t736\t0.0\t1\t691 45\n",
      "57\t705\t0.0\t1\t680 25\n",
      "58\t746\t0.0\t1\t712 34\n",
      "59\t692\t0.0\t1\t656 36\n",
      "60\t699\t0.0\t1\t665 34\n",
      "61\t732\t0.0\t1\t702 30\n",
      "62\t766\t0.0\t1\t730 36\n",
      "63\t697\t0.0\t1\t670 27\n",
      "64\t749\t0.0\t1\t719 30\n",
      "65\t798\t0.0\t1\t765 33\n",
      "66\t735\t0.0\t1\t701 34\n",
      "67\t791\t0.0\t1\t756 35\n",
      "68\t804\t0.0\t1\t757 47\n",
      "69\t751\t0.0\t1\t724 27\n",
      "70\t711\t0.0\t1\t680 31\n",
      "71\t750\t0.0\t1\t712 38\n",
      "72\t785\t0.0\t1\t758 27\n",
      "73\t803\t0.0\t1\t762 41\n",
      "74\t759\t0.0\t1\t728 31\n",
      "75\t730\t0.0\t1\t698 32\n",
      "76\t765\t0.0\t1\t737 28\n",
      "77\t768\t0.0\t1\t729 39\n",
      "78\t715\t0.0\t1\t683 32\n",
      "79\t812\t0.0\t1\t775 37\n",
      "80\t771\t0.0\t1\t732 39\n",
      "81\t731\t0.0\t1\t707 24\n",
      "82\t719\t0.0\t1\t695 24\n",
      "83\t771\t0.0\t1\t730 41\n",
      "84\t737\t0.0\t1\t708 29\n",
      "85\t816\t0.0\t1\t784 32\n",
      "86\t794\t0.0\t1\t765 29\n",
      "87\t763\t0.0\t1\t732 31\n",
      "88\t736\t0.0\t1\t719 17\n",
      "89\t764\t0.0\t1\t726 38\n",
      "90\t720\t0.0\t1\t684 36\n",
      "91\t792\t0.0\t1\t770 22\n",
      "92\t802\t0.0\t1\t774 28\n",
      "93\t770\t0.0\t1\t731 39\n",
      "94\t799\t0.0\t1\t772 27\n",
      "95\t757\t0.0\t1\t727 30\n",
      "96\t771\t0.0\t1\t744 27\n",
      "97\t746\t0.0\t1\t715 31\n",
      "98\t803\t0.0\t1\t782 21\n",
      "99\t794\t0.0\t1\t764 30\n",
      "100\t799\t0.0\t1\t775 24\n",
      "101\t798\t0.0\t1\t766 32\n",
      "102\t755\t0.0\t1\t728 27\n",
      "103\t803\t0.0\t1\t779 24\n",
      "104\t765\t0.0\t1\t740 25\n",
      "105\t745\t0.0\t1\t723 22\n",
      "106\t814\t0.0\t1\t788 26\n",
      "107\t793\t0.0\t1\t767 26\n",
      "108\t778\t0.0\t1\t755 23\n",
      "109\t858\t0.0\t1\t825 33\n",
      "110\t813\t0.0\t1\t785 28\n",
      "111\t823\t0.0\t1\t797 26\n",
      "112\t777\t0.0\t1\t760 17\n",
      "113\t790\t0.0\t1\t768 22\n",
      "114\t797\t0.0\t1\t777 20\n",
      "115\t880\t0.0\t1\t840 40\n",
      "116\t775\t0.0\t1\t758 17\n",
      "117\t792\t0.0\t1\t772 20\n",
      "118\t789\t0.0\t1\t769 20\n",
      "119\t839\t0.0\t1\t816 23\n",
      "120\t825\t0.0\t1\t799 26\n",
      "121\t818\t0.0\t1\t789 29\n",
      "122\t776\t0.0\t1\t758 18\n",
      "123\t883\t0.0\t1\t858 25\n",
      "124\t744\t0.0\t1\t722 22\n",
      "125\t857\t0.0\t1\t834 23\n",
      "126\t805\t0.0\t1\t787 18\n",
      "127\t783\t0.0\t1\t763 20\n",
      "128\t815\t0.0\t1\t798 17\n",
      "129\t805\t0.0\t1\t788 17\n",
      "130\t845\t0.0\t1\t820 25\n",
      "131\t862\t0.0\t1\t843 19\n",
      "132\t816\t0.0\t1\t803 13\n",
      "133\t823\t0.0\t1\t802 21\n",
      "134\t824\t0.0\t1\t804 20\n",
      "135\t785\t0.0\t1\t767 18\n",
      "136\t734\t0.0\t1\t714 20\n",
      "137\t712\t0.0\t1\t697 15\n",
      "138\t704\t0.0\t1\t687 17\n",
      "139\t695\t0.0\t1\t678 17\n",
      "140\t723\t0.0\t1\t706 17\n",
      "141\t737\t0.0\t1\t723 14\n",
      "142\t696\t0.0\t1\t683 13\n",
      "143\t711\t0.0\t1\t696 15\n",
      "144\t799\t0.0\t1\t788 11\n",
      "145\t804\t0.0\t1\t786 18\n",
      "146\t815\t0.0\t1\t794 21\n",
      "147\t716\t0.0\t1\t695 21\n",
      "148\t703\t0.0\t1\t688 15\n",
      "149\t592\t0.0\t1\t574 18\n",
      "150\t572\t0.0\t1\t560 12\n",
      "151\t147\t0.0\t1\t134 13\n",
      "\n",
      "\n",
      "RUN STATISTICS FOR INPUT FILE: /home/sam/data/geoduck_illumina/NR012_S1_L002_R2_001.fastq.gz\n",
      "=============================================\n",
      "358756 sequences processed in total\n",
      "The length threshold of paired-end sequences gets evaluated later on (in the validation step)\n",
      "\n",
      "Validate paired-end files NR012_S1_L002_R1_001_trimmed.fq.gz and NR012_S1_L002_R2_001_trimmed.fq.gz\n",
      "file_1: NR012_S1_L002_R1_001_trimmed.fq.gz, file_2: NR012_S1_L002_R2_001_trimmed.fq.gz\n",
      "\n",
      "\n",
      ">>>>> Now validing the length of the 2 paired-end infiles: NR012_S1_L002_R1_001_trimmed.fq.gz and NR012_S1_L002_R2_001_trimmed.fq.gz <<<<<\n",
      "Writing validated paired-end read 1 reads to NR012_S1_L002_R1_001_val_1.fq.gz\n",
      "Writing validated paired-end read 2 reads to NR012_S1_L002_R2_001_val_2.fq.gz\n",
      "\n",
      "Total number of sequences analysed: 358756\n",
      "\n",
      "Number of sequence pairs removed because at least one read was shorter than the length cutoff (20 bp): 32504 (9.06%)\n",
      "\n",
      "Deleting both intermediate output files NR012_S1_L002_R1_001_trimmed.fq.gz and NR012_S1_L002_R2_001_trimmed.fq.gz\n",
      "\n",
      "====================================================================================================\n",
      "\n",
      "Writing report to '/home/sam/data/geoduck_illumina/trimmed/NR013_AD013_S2_L001_R1_001.fastq.gz_trimming_report.txt'\n",
      "\n",
      "SUMMARISING RUN PARAMETERS\n",
      "==========================\n",
      "Input filename: /home/sam/data/geoduck_illumina/NR013_AD013_S2_L001_R1_001.fastq.gz\n",
      "Trimming mode: paired-end\n",
      "Trim Galore version: 0.4.4\n",
      "Cutadapt version: 1.9.1\n",
      "Quality Phred score cutoff: 20\n",
      "Quality encoding type selected: ASCII+33\n",
      "Adapter sequence: 'CTGTCTCTTATA' (Nextera Transposase sequence; auto-detected)\n",
      "Maximum trimming error rate: 0.1 (default)\n",
      "Minimum required adapter overlap (stringency): 1 bp\n",
      "Minimum required sequence length for both reads before a sequence pair gets removed: 20 bp\n",
      "Output file(s) will be GZIP compressed\n",
      "\n",
      "Writing final adapter and quality trimmed output to NR013_AD013_S2_L001_R1_001_trimmed.fq.gz\n",
      "\n",
      "\n",
      "  >>> Now performing quality (cutoff 20) and adapter trimming in a single pass for the adapter sequence: 'CTGTCTCTTATA' from file /home/sam/data/geoduck_illumina/NR013_AD013_S2_L001_R1_001.fastq.gz <<< \n",
      "10000000 sequences processed\n",
      "20000000 sequences processed\n",
      "30000000 sequences processed\n",
      "40000000 sequences processed\n",
      "50000000 sequences processed\n",
      "60000000 sequences processed\n",
      "70000000 sequences processed\n",
      "80000000 sequences processed\n",
      "90000000 sequences processed\n",
      "100000000 sequences processed\n",
      "110000000 sequences processed\n",
      "120000000 sequences processed\n",
      "130000000 sequences processed\n",
      "140000000 sequences processed\n",
      "150000000 sequences processed\n",
      "160000000 sequences processed\n",
      "170000000 sequences processed\n",
      "180000000 sequences processed\n",
      "190000000 sequences processed\n",
      "200000000 sequences processed\n",
      "210000000 sequences processed\n",
      "220000000 sequences processed\n",
      "230000000 sequences processed\n",
      "240000000 sequences processed\n",
      "250000000 sequences processed\n",
      "260000000 sequences processed\n",
      "270000000 sequences processed\n",
      "280000000 sequences processed\n",
      "290000000 sequences processed\n",
      "300000000 sequences processed\n",
      "310000000 sequences processed\n",
      "320000000 sequences processed\n",
      "330000000 sequences processed\n",
      "340000000 sequences processed\n",
      "This is cutadapt 1.9.1 with Python 2.7.12\n",
      "Command line parameters: -f fastq -e 0.1 -q 20 -O 1 -a CTGTCTCTTATA /home/sam/data/geoduck_illumina/NR013_AD013_S2_L001_R1_001.fastq.gz\n",
      "Trimming 1 adapter with at most 10.0% errors in single-end mode ...\n",
      "Finished in 8220.39 s (24 us/read; 2.50 M reads/minute).\n",
      "\n",
      "=== Summary ===\n",
      "\n",
      "Total reads processed:             342,277,558\n",
      "Reads with adapters:               183,891,810 (53.7%)\n",
      "Reads written (passing filters):   342,277,558 (100.0%)\n",
      "\n",
      "Total basepairs processed: 51,683,911,258 bp\n",
      "Quality-trimmed:             177,870,597 bp (0.3%)\n",
      "Total written (filtered):  41,309,193,256 bp (79.9%)\n",
      "\n",
      "=== Adapter 1 ===\n",
      "\n",
      "Sequence: CTGTCTCTTATA; Type: regular 3'; Length: 12; Trimmed: 183891810 times.\n",
      "\n",
      "No. of allowed errors:\n",
      "0-9 bp: 0; 10-12 bp: 1\n",
      "\n",
      "Bases preceding removed adapters:\n",
      "  A: 19.7%\n",
      "  C: 32.7%\n",
      "  G: 20.4%\n",
      "  T: 27.0%\n",
      "  none/other: 0.2%\n",
      "\n",
      "Overview of removed sequences\n",
      "length\tcount\texpect\tmax.err\terror counts\n",
      "1\t40388880\t85569389.5\t0\t40388880\n",
      "2\t11683799\t21392347.4\t0\t11683799\n",
      "3\t4010339\t5348086.8\t0\t4010339\n",
      "4\t1712947\t1337021.7\t0\t1712947\n",
      "5\t995397\t334255.4\t0\t995397\n",
      "6\t786526\t83563.9\t0\t786526\n",
      "7\t784662\t20891.0\t0\t784662\n",
      "8\t827819\t5222.7\t0\t827819\n",
      "9\t695311\t1305.7\t0\t686951 8360\n",
      "10\t779196\t326.4\t1\t723277 55919\n",
      "11\t806919\t81.6\t1\t755613 51306\n",
      "12\t763544\t20.4\t1\t709569 53975\n",
      "13\t779656\t20.4\t1\t731038 48618\n",
      "14\t727535\t20.4\t1\t680076 47459\n",
      "15\t852162\t20.4\t1\t793739 58423\n",
      "16\t927854\t20.4\t1\t865786 62068\n",
      "17\t595097\t20.4\t1\t559071 36026\n",
      "18\t793308\t20.4\t1\t745158 48150\n",
      "19\t820813\t20.4\t1\t774999 45814\n",
      "20\t726618\t20.4\t1\t680830 45788\n",
      "21\t797575\t20.4\t1\t751754 45821\n",
      "22\t875656\t20.4\t1\t822159 53497\n",
      "23\t670794\t20.4\t1\t629734 41060\n",
      "24\t815418\t20.4\t1\t769524 45894\n",
      "25\t718144\t20.4\t1\t679059 39085\n",
      "26\t838493\t20.4\t1\t785154 53339\n",
      "27\t796979\t20.4\t1\t753147 43832\n",
      "28\t752459\t20.4\t1\t710800 41659\n",
      "29\t779708\t20.4\t1\t739042 40666\n",
      "30\t846865\t20.4\t1\t800159 46706\n",
      "31\t789467\t20.4\t1\t745353 44114\n",
      "32\t766285\t20.4\t1\t726684 39601\n",
      "33\t775956\t20.4\t1\t732627 43329\n",
      "34\t777116\t20.4\t1\t735561 41555\n",
      "35\t789816\t20.4\t1\t744671 45145\n",
      "36\t847851\t20.4\t1\t804739 43112\n",
      "37\t758734\t20.4\t1\t715110 43624\n",
      "38\t832663\t20.4\t1\t788304 44359\n",
      "39\t812276\t20.4\t1\t768737 43539\n",
      "40\t810942\t20.4\t1\t768401 42541\n",
      "41\t800677\t20.4\t1\t757743 42934\n",
      "42\t808457\t20.4\t1\t766837 41620\n",
      "43\t801371\t20.4\t1\t760095 41276\n",
      "44\t805830\t20.4\t1\t764204 41626\n",
      "45\t810657\t20.4\t1\t769292 41365\n",
      "46\t803540\t20.4\t1\t761439 42101\n",
      "47\t817900\t20.4\t1\t776415 41485\n",
      "48\t813316\t20.4\t1\t772060 41256\n",
      "49\t818804\t20.4\t1\t777318 41486\n",
      "50\t815442\t20.4\t1\t774692 40750\n",
      "51\t818321\t20.4\t1\t777259 41062\n",
      "52\t818382\t20.4\t1\t778490 39892\n",
      "53\t823951\t20.4\t1\t783815 40136\n",
      "54\t818968\t20.4\t1\t779047 39921\n",
      "55\t822143\t20.4\t1\t781895 40248\n",
      "56\t823609\t20.4\t1\t784267 39342\n",
      "57\t825029\t20.4\t1\t785368 39661\n",
      "58\t827704\t20.4\t1\t788128 39576\n",
      "59\t833104\t20.4\t1\t793259 39845\n",
      "60\t831235\t20.4\t1\t792380 38855\n",
      "61\t832657\t20.4\t1\t793431 39226\n",
      "62\t831832\t20.4\t1\t793357 38475\n",
      "63\t833288\t20.4\t1\t794795 38493\n",
      "64\t833485\t20.4\t1\t795306 38179\n",
      "65\t834389\t20.4\t1\t796142 38247\n",
      "66\t837347\t20.4\t1\t799230 38117\n",
      "67\t840684\t20.4\t1\t802647 38037\n",
      "68\t845216\t20.4\t1\t807252 37964\n",
      "69\t847361\t20.4\t1\t809367 37994\n",
      "70\t848360\t20.4\t1\t810649 37711\n",
      "71\t851310\t20.4\t1\t813436 37874\n",
      "72\t849201\t20.4\t1\t811832 37369\n",
      "73\t848981\t20.4\t1\t811750 37231\n",
      "74\t848610\t20.4\t1\t811471 37139\n",
      "75\t851947\t20.4\t1\t814603 37344\n",
      "76\t851151\t20.4\t1\t814602 36549\n",
      "77\t855712\t20.4\t1\t818946 36766\n",
      "78\t857694\t20.4\t1\t820917 36777\n",
      "79\t867771\t20.4\t1\t830904 36867\n",
      "80\t868545\t20.4\t1\t831831 36714\n",
      "81\t870150\t20.4\t1\t833875 36275\n",
      "82\t869271\t20.4\t1\t832978 36293\n",
      "83\t868005\t20.4\t1\t832273 35732\n",
      "84\t865449\t20.4\t1\t829813 35636\n",
      "85\t868387\t20.4\t1\t832417 35970\n",
      "86\t866211\t20.4\t1\t830618 35593\n",
      "87\t870208\t20.4\t1\t835273 34935\n",
      "88\t874875\t20.4\t1\t839361 35514\n",
      "89\t881784\t20.4\t1\t846527 35257\n",
      "90\t886150\t20.4\t1\t851230 34920\n",
      "91\t893644\t20.4\t1\t858759 34885\n",
      "92\t887519\t20.4\t1\t853104 34415\n",
      "93\t888386\t20.4\t1\t854265 34121\n",
      "94\t886135\t20.4\t1\t852195 33940\n",
      "95\t886285\t20.4\t1\t853201 33084\n",
      "96\t884044\t20.4\t1\t851257 32787\n",
      "97\t887292\t20.4\t1\t854429 32863\n",
      "98\t893376\t20.4\t1\t860471 32905\n",
      "99\t901175\t20.4\t1\t868374 32801\n",
      "100\t907164\t20.4\t1\t874405 32759\n",
      "101\t913618\t20.4\t1\t880838 32780\n",
      "102\t913959\t20.4\t1\t881754 32205\n",
      "103\t911739\t20.4\t1\t879647 32092\n",
      "104\t905964\t20.4\t1\t874314 31650\n",
      "105\t905993\t20.4\t1\t874258 31735\n",
      "106\t906182\t20.4\t1\t873559 32623\n",
      "107\t908110\t20.4\t1\t876186 31924\n",
      "108\t913564\t20.4\t1\t881156 32408\n",
      "109\t925991\t20.4\t1\t893692 32299\n",
      "110\t931480\t20.4\t1\t899357 32123\n",
      "111\t941940\t20.4\t1\t910234 31706\n",
      "112\t946848\t20.4\t1\t915100 31748\n",
      "113\t944143\t20.4\t1\t912658 31485\n",
      "114\t937208\t20.4\t1\t905801 31407\n",
      "115\t931563\t20.4\t1\t901242 30321\n",
      "116\t929805\t20.4\t1\t899326 30479\n",
      "117\t928394\t20.4\t1\t898673 29721\n",
      "118\t929624\t20.4\t1\t901129 28495\n",
      "119\t936256\t20.4\t1\t907871 28385\n",
      "120\t947616\t20.4\t1\t918957 28659\n",
      "121\t961453\t20.4\t1\t932789 28664\n",
      "122\t970862\t20.4\t1\t942499 28363\n",
      "123\t968427\t20.4\t1\t939973 28454\n",
      "124\t963612\t20.4\t1\t935591 28021\n",
      "125\t957482\t20.4\t1\t929511 27971\n",
      "126\t953436\t20.4\t1\t926550 26886\n",
      "127\t954852\t20.4\t1\t928741 26111\n",
      "128\t962979\t20.4\t1\t938243 24736\n",
      "129\t971364\t20.4\t1\t947262 24102\n",
      "130\t991716\t20.4\t1\t968746 22970\n",
      "131\t1010119\t20.4\t1\t987255 22864\n",
      "132\t1017220\t20.4\t1\t996130 21090\n",
      "133\t1015734\t20.4\t1\t995762 19972\n",
      "134\t965026\t20.4\t1\t946900 18126\n",
      "135\t947123\t20.4\t1\t930865 16258\n",
      "136\t919019\t20.4\t1\t903569 15450\n",
      "137\t895405\t20.4\t1\t881112 14293\n",
      "138\t860977\t20.4\t1\t847091 13886\n",
      "139\t851175\t20.4\t1\t837420 13755\n",
      "140\t918814\t20.4\t1\t904007 14807\n",
      "141\t977689\t20.4\t1\t962530 15159\n",
      "142\t886262\t20.4\t1\t872083 14179\n",
      "143\t919381\t20.4\t1\t905178 14203\n",
      "144\t975731\t20.4\t1\t960540 15191\n",
      "145\t945359\t20.4\t1\t930332 15027\n",
      "146\t990217\t20.4\t1\t973971 16246\n",
      "147\t834988\t20.4\t1\t820525 14463\n",
      "148\t851341\t20.4\t1\t835999 15342\n",
      "149\t716897\t20.4\t1\t702235 14662\n",
      "150\t690547\t20.4\t1\t673848 16699\n",
      "151\t315536\t20.4\t1\t286994 28542\n",
      "\n",
      "\n",
      "RUN STATISTICS FOR INPUT FILE: /home/sam/data/geoduck_illumina/NR013_AD013_S2_L001_R1_001.fastq.gz\n",
      "=============================================\n",
      "342277558 sequences processed in total\n",
      "The length threshold of paired-end sequences gets evaluated later on (in the validation step)\n",
      "\n",
      "Writing report to '/home/sam/data/geoduck_illumina/trimmed/NR013_AD013_S2_L001_R2_001.fastq.gz_trimming_report.txt'\n",
      "\n",
      "SUMMARISING RUN PARAMETERS\n",
      "==========================\n",
      "Input filename: /home/sam/data/geoduck_illumina/NR013_AD013_S2_L001_R2_001.fastq.gz\n",
      "Trimming mode: paired-end\n",
      "Trim Galore version: 0.4.4\n",
      "Cutadapt version: 1.9.1\n",
      "Quality Phred score cutoff: 20\n",
      "Quality encoding type selected: ASCII+33\n",
      "Adapter sequence: 'CTGTCTCTTATA' (Nextera Transposase sequence; auto-detected)\n",
      "Maximum trimming error rate: 0.1 (default)\n",
      "Minimum required adapter overlap (stringency): 1 bp\n",
      "Minimum required sequence length for both reads before a sequence pair gets removed: 20 bp\n",
      "Output file(s) will be GZIP compressed\n",
      "\n",
      "Writing final adapter and quality trimmed output to NR013_AD013_S2_L001_R2_001_trimmed.fq.gz\n",
      "\n",
      "\n",
      "  >>> Now performing quality (cutoff 20) and adapter trimming in a single pass for the adapter sequence: 'CTGTCTCTTATA' from file /home/sam/data/geoduck_illumina/NR013_AD013_S2_L001_R2_001.fastq.gz <<< \n",
      "10000000 sequences processed\n",
      "20000000 sequences processed\n",
      "30000000 sequences processed\n",
      "40000000 sequences processed\n",
      "50000000 sequences processed\n",
      "60000000 sequences processed\n",
      "70000000 sequences processed\n",
      "80000000 sequences processed\n",
      "90000000 sequences processed\n",
      "100000000 sequences processed\n",
      "110000000 sequences processed\n",
      "120000000 sequences processed\n",
      "130000000 sequences processed\n",
      "140000000 sequences processed\n",
      "150000000 sequences processed\n",
      "160000000 sequences processed\n",
      "170000000 sequences processed\n",
      "180000000 sequences processed\n",
      "190000000 sequences processed\n",
      "200000000 sequences processed\n",
      "210000000 sequences processed\n",
      "220000000 sequences processed\n",
      "230000000 sequences processed\n",
      "240000000 sequences processed\n",
      "250000000 sequences processed\n",
      "260000000 sequences processed\n",
      "270000000 sequences processed\n",
      "280000000 sequences processed\n",
      "290000000 sequences processed\n",
      "300000000 sequences processed\n",
      "310000000 sequences processed\n",
      "320000000 sequences processed\n",
      "330000000 sequences processed\n",
      "340000000 sequences processed\n",
      "This is cutadapt 1.9.1 with Python 2.7.12\n",
      "Command line parameters: -f fastq -e 0.1 -q 20 -O 1 -a CTGTCTCTTATA /home/sam/data/geoduck_illumina/NR013_AD013_S2_L001_R2_001.fastq.gz\n",
      "Trimming 1 adapter with at most 10.0% errors in single-end mode ...\n",
      "Finished in 8511.76 s (25 us/read; 2.41 M reads/minute).\n",
      "\n",
      "=== Summary ===\n",
      "\n",
      "Total reads processed:             342,277,558\n",
      "Reads with adapters:               169,073,071 (49.4%)\n",
      "Reads written (passing filters):   342,277,558 (100.0%)\n",
      "\n",
      "Total basepairs processed: 51,683,911,258 bp\n",
      "Quality-trimmed:             235,733,156 bp (0.5%)\n",
      "Total written (filtered):  42,650,220,228 bp (82.5%)\n",
      "\n",
      "=== Adapter 1 ===\n",
      "\n",
      "Sequence: CTGTCTCTTATA; Type: regular 3'; Length: 12; Trimmed: 169073071 times.\n",
      "\n",
      "No. of allowed errors:\n",
      "0-9 bp: 0; 10-12 bp: 1\n",
      "\n",
      "Bases preceding removed adapters:\n",
      "  A: 20.3%\n",
      "  C: 31.7%\n",
      "  G: 20.6%\n",
      "  T: 27.2%\n",
      "  none/other: 0.2%\n",
      "\n",
      "Overview of removed sequences\n",
      "length\tcount\texpect\tmax.err\terror counts\n",
      "1\t40820229\t85569389.5\t0\t40820229\n",
      "2\t11782648\t21392347.4\t0\t11782648\n",
      "3\t4020450\t5348086.8\t0\t4020450\n",
      "4\t1715428\t1337021.7\t0\t1715428\n",
      "5\t935513\t334255.4\t0\t935513\n",
      "6\t719155\t83563.9\t0\t719155\n",
      "7\t714435\t20891.0\t0\t714435\n",
      "8\t756165\t5222.7\t0\t756165\n",
      "9\t624028\t1305.7\t0\t614695 9333\n",
      "10\t706778\t326.4\t1\t624616 82162\n",
      "11\t759782\t81.6\t1\t674567 85215\n",
      "12\t671199\t20.4\t1\t593380 77819\n",
      "13\t736821\t20.4\t1\t651009 85812\n",
      "14\t633841\t20.4\t1\t564162 69679\n",
      "15\t820538\t20.4\t1\t722627 97911\n",
      "16\t806259\t20.4\t1\t718905 87354\n",
      "17\t536268\t20.4\t1\t477709 58559\n",
      "18\t735258\t20.4\t1\t655106 80152\n",
      "19\t758719\t20.4\t1\t682265 76454\n",
      "20\t635894\t20.4\t1\t571100 64794\n",
      "21\t742006\t20.4\t1\t661359 80647\n",
      "22\t782074\t20.4\t1\t701445 80629\n",
      "23\t605414\t20.4\t1\t541973 63441\n",
      "24\t744945\t20.4\t1\t664620 80325\n",
      "25\t655255\t20.4\t1\t591736 63519\n",
      "26\t776270\t20.4\t1\t686776 89494\n",
      "27\t741583\t20.4\t1\t665113 76470\n",
      "28\t677788\t20.4\t1\t611417 66371\n",
      "29\t718819\t20.4\t1\t648277 70542\n",
      "30\t764766\t20.4\t1\t691730 73036\n",
      "31\t682123\t20.4\t1\t617636 64487\n",
      "32\t719464\t20.4\t1\t647349 72115\n",
      "33\t691777\t20.4\t1\t627625 64152\n",
      "34\t729627\t20.4\t1\t656813 72814\n",
      "35\t687137\t20.4\t1\t621884 65253\n",
      "36\t805068\t20.4\t1\t731891 73177\n",
      "37\t655611\t20.4\t1\t595685 59926\n",
      "38\t768117\t20.4\t1\t696101 72016\n",
      "39\t748806\t20.4\t1\t678859 69947\n",
      "40\t741903\t20.4\t1\t676610 65293\n",
      "41\t725713\t20.4\t1\t661915 63798\n",
      "42\t741386\t20.4\t1\t677245 64141\n",
      "43\t729434\t20.4\t1\t666505 62929\n",
      "44\t728751\t20.4\t1\t665841 62910\n",
      "45\t737282\t20.4\t1\t673834 63448\n",
      "46\t722712\t20.4\t1\t661574 61138\n",
      "47\t753002\t20.4\t1\t688608 64394\n",
      "48\t732571\t20.4\t1\t672457 60114\n",
      "49\t745387\t20.4\t1\t683334 62053\n",
      "50\t737709\t20.4\t1\t677391 60318\n",
      "51\t742692\t20.4\t1\t681024 61668\n",
      "52\t743725\t20.4\t1\t682869 60856\n",
      "53\t747773\t20.4\t1\t686419 61354\n",
      "54\t741385\t20.4\t1\t681898 59487\n",
      "55\t744573\t20.4\t1\t684320 60253\n",
      "56\t746636\t20.4\t1\t686510 60126\n",
      "57\t747951\t20.4\t1\t687440 60511\n",
      "58\t750971\t20.4\t1\t690584 60387\n",
      "59\t751688\t20.4\t1\t692024 59664\n",
      "60\t752003\t20.4\t1\t692912 59091\n",
      "61\t753403\t20.4\t1\t694485 58918\n",
      "62\t749599\t20.4\t1\t691886 57713\n",
      "63\t750095\t20.4\t1\t692505 57590\n",
      "64\t751757\t20.4\t1\t694286 57471\n",
      "65\t748781\t20.4\t1\t691588 57193\n",
      "66\t751316\t20.4\t1\t694429 56887\n",
      "67\t755739\t20.4\t1\t699092 56647\n",
      "68\t757346\t20.4\t1\t702379 54967\n",
      "69\t761461\t20.4\t1\t706811 54650\n",
      "70\t763474\t20.4\t1\t709206 54268\n",
      "71\t762579\t20.4\t1\t708069 54510\n",
      "72\t761282\t20.4\t1\t708276 53006\n",
      "73\t760388\t20.4\t1\t706805 53583\n",
      "74\t760282\t20.4\t1\t707097 53185\n",
      "75\t760919\t20.4\t1\t708085 52834\n",
      "76\t761360\t20.4\t1\t708364 52996\n",
      "77\t765345\t20.4\t1\t712241 53104\n",
      "78\t765972\t20.4\t1\t712751 53221\n",
      "79\t769659\t20.4\t1\t716379 53280\n",
      "80\t771993\t20.4\t1\t719829 52164\n",
      "81\t772403\t20.4\t1\t720381 52022\n",
      "82\t770945\t20.4\t1\t719545 51400\n",
      "83\t766876\t20.4\t1\t715360 51516\n",
      "84\t766959\t20.4\t1\t716014 50945\n",
      "85\t765578\t20.4\t1\t715552 50026\n",
      "86\t765398\t20.4\t1\t715395 50003\n",
      "87\t769546\t20.4\t1\t719845 49701\n",
      "88\t770661\t20.4\t1\t722014 48647\n",
      "89\t779324\t20.4\t1\t730585 48739\n",
      "90\t782440\t20.4\t1\t733139 49301\n",
      "91\t785621\t20.4\t1\t736955 48666\n",
      "92\t784128\t20.4\t1\t735042 49086\n",
      "93\t783323\t20.4\t1\t735462 47861\n",
      "94\t777276\t20.4\t1\t729989 47287\n",
      "95\t776780\t20.4\t1\t729845 46935\n",
      "96\t775301\t20.4\t1\t728847 46454\n",
      "97\t778120\t20.4\t1\t731379 46741\n",
      "98\t782054\t20.4\t1\t735883 46171\n",
      "99\t787491\t20.4\t1\t741122 46369\n",
      "100\t792380\t20.4\t1\t746352 46028\n",
      "101\t797343\t20.4\t1\t751323 46020\n",
      "102\t793873\t20.4\t1\t749291 44582\n",
      "103\t791473\t20.4\t1\t747107 44366\n",
      "104\t786531\t20.4\t1\t742401 44130\n",
      "105\t784956\t20.4\t1\t741447 43509\n",
      "106\t784602\t20.4\t1\t741267 43335\n",
      "107\t785094\t20.4\t1\t741639 43455\n",
      "108\t788685\t20.4\t1\t746409 42276\n",
      "109\t798037\t20.4\t1\t754775 43262\n",
      "110\t805426\t20.4\t1\t762153 43273\n",
      "111\t810956\t20.4\t1\t767929 43027\n",
      "112\t813814\t20.4\t1\t771545 42269\n",
      "113\t811795\t20.4\t1\t769721 42074\n",
      "114\t801975\t20.4\t1\t760023 41952\n",
      "115\t798716\t20.4\t1\t757606 41110\n",
      "116\t793490\t20.4\t1\t753519 39971\n",
      "117\t792405\t20.4\t1\t752759 39646\n",
      "118\t792182\t20.4\t1\t752956 39226\n",
      "119\t798086\t20.4\t1\t759124 38962\n",
      "120\t807653\t20.4\t1\t768894 38759\n",
      "121\t818403\t20.4\t1\t778685 39718\n",
      "122\t819793\t20.4\t1\t780823 38970\n",
      "123\t820755\t20.4\t1\t782414 38341\n",
      "124\t813872\t20.4\t1\t775571 38301\n",
      "125\t805064\t20.4\t1\t767871 37193\n",
      "126\t801000\t20.4\t1\t764847 36153\n",
      "127\t804159\t20.4\t1\t768491 35668\n",
      "128\t808143\t20.4\t1\t773386 34757\n",
      "129\t816192\t20.4\t1\t781321 34871\n",
      "130\t834445\t20.4\t1\t799613 34832\n",
      "131\t847004\t20.4\t1\t812141 34863\n",
      "132\t848895\t20.4\t1\t815313 33582\n",
      "133\t841281\t20.4\t1\t808989 32292\n",
      "134\t799405\t20.4\t1\t768827 30578\n",
      "135\t783067\t20.4\t1\t753818 29249\n",
      "136\t757259\t20.4\t1\t728681 28578\n",
      "137\t733223\t20.4\t1\t705331 27892\n",
      "138\t701751\t20.4\t1\t674466 27285\n",
      "139\t697866\t20.4\t1\t671317 26549\n",
      "140\t750187\t20.4\t1\t722013 28174\n",
      "141\t791694\t20.4\t1\t763209 28485\n",
      "142\t729754\t20.4\t1\t702984 26770\n",
      "143\t760344\t20.4\t1\t733107 27237\n",
      "144\t802331\t20.4\t1\t774018 28313\n",
      "145\t791346\t20.4\t1\t764886 26460\n",
      "146\t804731\t20.4\t1\t777907 26824\n",
      "147\t686659\t20.4\t1\t664329 22330\n",
      "148\t686872\t20.4\t1\t665692 21180\n",
      "149\t550444\t20.4\t1\t532595 17849\n",
      "150\t529171\t20.4\t1\t510636 18535\n",
      "151\t234210\t20.4\t1\t208660 25550\n",
      "\n",
      "\n",
      "RUN STATISTICS FOR INPUT FILE: /home/sam/data/geoduck_illumina/NR013_AD013_S2_L001_R2_001.fastq.gz\n",
      "=============================================\n",
      "342277558 sequences processed in total\n",
      "The length threshold of paired-end sequences gets evaluated later on (in the validation step)\n",
      "\n",
      "Validate paired-end files NR013_AD013_S2_L001_R1_001_trimmed.fq.gz and NR013_AD013_S2_L001_R2_001_trimmed.fq.gz\n",
      "file_1: NR013_AD013_S2_L001_R1_001_trimmed.fq.gz, file_2: NR013_AD013_S2_L001_R2_001_trimmed.fq.gz\n",
      "\n",
      "\n",
      ">>>>> Now validing the length of the 2 paired-end infiles: NR013_AD013_S2_L001_R1_001_trimmed.fq.gz and NR013_AD013_S2_L001_R2_001_trimmed.fq.gz <<<<<\n",
      "Writing validated paired-end read 1 reads to NR013_AD013_S2_L001_R1_001_val_1.fq.gz\n",
      "Writing validated paired-end read 2 reads to NR013_AD013_S2_L001_R2_001_val_2.fq.gz\n",
      "\n",
      "Total number of sequences analysed: 342277558\n",
      "\n",
      "Number of sequence pairs removed because at least one read was shorter than the length cutoff (20 bp): 32399854 (9.47%)\n",
      "\n",
      "Deleting both intermediate output files NR013_AD013_S2_L001_R1_001_trimmed.fq.gz and NR013_AD013_S2_L001_R2_001_trimmed.fq.gz\n",
      "\n",
      "====================================================================================================\n",
      "\n",
      "Writing report to '/home/sam/data/geoduck_illumina/trimmed/NR013_AD013_S2_L002_R1_001.fastq.gz_trimming_report.txt'\n",
      "\n",
      "SUMMARISING RUN PARAMETERS\n",
      "==========================\n",
      "Input filename: /home/sam/data/geoduck_illumina/NR013_AD013_S2_L002_R1_001.fastq.gz\n",
      "Trimming mode: paired-end\n",
      "Trim Galore version: 0.4.4\n",
      "Cutadapt version: 1.9.1\n",
      "Quality Phred score cutoff: 20\n",
      "Quality encoding type selected: ASCII+33\n",
      "Adapter sequence: 'CTGTCTCTTATA' (Nextera Transposase sequence; auto-detected)\n",
      "Maximum trimming error rate: 0.1 (default)\n",
      "Minimum required adapter overlap (stringency): 1 bp\n",
      "Minimum required sequence length for both reads before a sequence pair gets removed: 20 bp\n",
      "Output file(s) will be GZIP compressed\n",
      "\n",
      "Writing final adapter and quality trimmed output to NR013_AD013_S2_L002_R1_001_trimmed.fq.gz\n",
      "\n",
      "\n",
      "  >>> Now performing quality (cutoff 20) and adapter trimming in a single pass for the adapter sequence: 'CTGTCTCTTATA' from file /home/sam/data/geoduck_illumina/NR013_AD013_S2_L002_R1_001.fastq.gz <<< \n",
      "10000000 sequences processed\n",
      "20000000 sequences processed\n",
      "30000000 sequences processed\n",
      "40000000 sequences processed\n",
      "50000000 sequences processed\n",
      "60000000 sequences processed\n",
      "70000000 sequences processed\n",
      "80000000 sequences processed\n",
      "90000000 sequences processed\n",
      "100000000 sequences processed\n",
      "110000000 sequences processed\n",
      "120000000 sequences processed\n",
      "130000000 sequences processed\n",
      "140000000 sequences processed\n",
      "150000000 sequences processed\n",
      "160000000 sequences processed\n",
      "170000000 sequences processed\n",
      "180000000 sequences processed\n",
      "190000000 sequences processed\n",
      "200000000 sequences processed\n",
      "210000000 sequences processed\n",
      "220000000 sequences processed\n",
      "230000000 sequences processed\n",
      "240000000 sequences processed\n",
      "250000000 sequences processed\n",
      "260000000 sequences processed\n",
      "270000000 sequences processed\n",
      "280000000 sequences processed\n",
      "290000000 sequences processed\n",
      "300000000 sequences processed\n",
      "310000000 sequences processed\n",
      "320000000 sequences processed\n",
      "330000000 sequences processed\n",
      "340000000 sequences processed\n",
      "350000000 sequences processed\n",
      "360000000 sequences processed\n",
      "370000000 sequences processed\n",
      "380000000 sequences processed\n",
      "390000000 sequences processed\n",
      "400000000 sequences processed\n",
      "410000000 sequences processed\n",
      "This is cutadapt 1.9.1 with Python 2.7.12\n",
      "Command line parameters: -f fastq -e 0.1 -q 20 -O 1 -a CTGTCTCTTATA /home/sam/data/geoduck_illumina/NR013_AD013_S2_L002_R1_001.fastq.gz\n",
      "Trimming 1 adapter with at most 10.0% errors in single-end mode ...\n",
      "Finished in 9950.16 s (24 us/read; 2.51 M reads/minute).\n",
      "\n",
      "=== Summary ===\n",
      "\n",
      "Total reads processed:             415,822,852\n",
      "Reads with adapters:               214,614,887 (51.6%)\n",
      "Reads written (passing filters):   415,822,852 (100.0%)\n",
      "\n",
      "Total basepairs processed: 62,789,250,652 bp\n",
      "Quality-trimmed:             237,916,547 bp (0.4%)\n",
      "Total written (filtered):  50,986,157,920 bp (81.2%)\n",
      "\n",
      "=== Adapter 1 ===\n",
      "\n",
      "Sequence: CTGTCTCTTATA; Type: regular 3'; Length: 12; Trimmed: 214614887 times.\n",
      "\n",
      "No. of allowed errors:\n",
      "0-9 bp: 0; 10-12 bp: 1\n",
      "\n",
      "Bases preceding removed adapters:\n",
      "  A: 20.0%\n",
      "  C: 32.4%\n",
      "  G: 20.4%\n",
      "  T: 27.1%\n",
      "  none/other: 0.2%\n",
      "\n",
      "Overview of removed sequences\n",
      "length\tcount\texpect\tmax.err\terror counts\n",
      "1\t50386724\t103955713.0\t0\t50386724\n",
      "2\t14542061\t25988928.2\t0\t14542061\n",
      "3\t4910218\t6497232.1\t0\t4910218\n",
      "4\t2070214\t1624308.0\t0\t2070214\n",
      "5\t1141353\t406077.0\t0\t1141353\n",
      "6\t914915\t101519.3\t0\t914915\n",
      "7\t892217\t25379.8\t0\t892217\n",
      "8\t927453\t6345.0\t0\t927453\n",
      "9\t808460\t1586.2\t0\t799470 8990\n",
      "10\t881043\t396.6\t1\t835188 45855\n",
      "11\t910256\t99.1\t1\t869754 40502\n",
      "12\t863077\t24.8\t1\t822517 40560\n",
      "13\t884312\t24.8\t1\t847815 36497\n",
      "14\t833437\t24.8\t1\t797878 35559\n",
      "15\t943139\t24.8\t1\t899833 43306\n",
      "16\t1031643\t24.8\t1\t985067 46576\n",
      "17\t693776\t24.8\t1\t666899 26877\n",
      "18\t890739\t24.8\t1\t856113 34626\n",
      "19\t919240\t24.8\t1\t886166 33074\n",
      "20\t830740\t24.8\t1\t797012 33728\n",
      "21\t904625\t24.8\t1\t869090 35535\n",
      "22\t980608\t24.8\t1\t940157 40451\n",
      "23\t758414\t24.8\t1\t727750 30664\n",
      "24\t922246\t24.8\t1\t886504 35742\n",
      "25\t821687\t24.8\t1\t792410 29277\n",
      "26\t940516\t24.8\t1\t898312 42204\n",
      "27\t895290\t24.8\t1\t862092 33198\n",
      "28\t861508\t24.8\t1\t829893 31615\n",
      "29\t878923\t24.8\t1\t848680 30243\n",
      "30\t935661\t24.8\t1\t900959 34702\n",
      "31\t890427\t24.8\t1\t856911 33516\n",
      "32\t876170\t24.8\t1\t845708 30462\n",
      "33\t871818\t24.8\t1\t839349 32469\n",
      "34\t888056\t24.8\t1\t855472 32584\n",
      "35\t893985\t24.8\t1\t860462 33523\n",
      "36\t941283\t24.8\t1\t908816 32467\n",
      "37\t866803\t24.8\t1\t833795 33008\n",
      "38\t938084\t24.8\t1\t903662 34422\n",
      "39\t917384\t24.8\t1\t883803 33581\n",
      "40\t915553\t24.8\t1\t883357 32196\n",
      "41\t903529\t24.8\t1\t871137 32392\n",
      "42\t912337\t24.8\t1\t880927 31410\n",
      "43\t903619\t24.8\t1\t872819 30800\n",
      "44\t908618\t24.8\t1\t876636 31982\n",
      "45\t913382\t24.8\t1\t881338 32044\n",
      "46\t904794\t24.8\t1\t872879 31915\n",
      "47\t921905\t24.8\t1\t889613 32292\n",
      "48\t917471\t24.8\t1\t885935 31536\n",
      "49\t925145\t24.8\t1\t892957 32188\n",
      "50\t917758\t24.8\t1\t886669 31089\n",
      "51\t923602\t24.8\t1\t892274 31328\n",
      "52\t922737\t24.8\t1\t892327 30410\n",
      "53\t927239\t24.8\t1\t896380 30859\n",
      "54\t924549\t24.8\t1\t894108 30441\n",
      "55\t927519\t24.8\t1\t897253 30266\n",
      "56\t926828\t24.8\t1\t896613 30215\n",
      "57\t930704\t24.8\t1\t900241 30463\n",
      "58\t933679\t24.8\t1\t903258 30421\n",
      "59\t937434\t24.8\t1\t907011 30423\n",
      "60\t938844\t24.8\t1\t906975 31869\n",
      "61\t941495\t24.8\t1\t910132 31363\n",
      "62\t937993\t24.8\t1\t906915 31078\n",
      "63\t939883\t24.8\t1\t909197 30686\n",
      "64\t938737\t24.8\t1\t907708 31029\n",
      "65\t941343\t24.8\t1\t910953 30390\n",
      "66\t941832\t24.8\t1\t911016 30816\n",
      "67\t948012\t24.8\t1\t917423 30589\n",
      "68\t952097\t24.8\t1\t920730 31367\n",
      "69\t954987\t24.8\t1\t924503 30484\n",
      "70\t957511\t24.8\t1\t926593 30918\n",
      "71\t960740\t24.8\t1\t929915 30825\n",
      "72\t957389\t24.8\t1\t927925 29464\n",
      "73\t956968\t24.8\t1\t927378 29590\n",
      "74\t957044\t24.8\t1\t926974 30070\n",
      "75\t958262\t24.8\t1\t928562 29700\n",
      "76\t960471\t24.8\t1\t930347 30124\n",
      "77\t962961\t24.8\t1\t933258 29703\n",
      "78\t967356\t24.8\t1\t937373 29983\n",
      "79\t975461\t24.8\t1\t945341 30120\n",
      "80\t978503\t24.8\t1\t948386 30117\n",
      "81\t980057\t24.8\t1\t950376 29681\n",
      "82\t977649\t24.8\t1\t947985 29664\n",
      "83\t979192\t24.8\t1\t949570 29622\n",
      "84\t976875\t24.8\t1\t948299 28576\n",
      "85\t975691\t24.8\t1\t947442 28249\n",
      "86\t977587\t24.8\t1\t949341 28246\n",
      "87\t980919\t24.8\t1\t952844 28075\n",
      "88\t984578\t24.8\t1\t956968 27610\n",
      "89\t997379\t24.8\t1\t969773 27606\n",
      "90\t1000000\t24.8\t1\t972394 27606\n",
      "91\t1003424\t24.8\t1\t976109 27315\n",
      "92\t1004385\t24.8\t1\t977118 27267\n",
      "93\t1003119\t24.8\t1\t976170 26949\n",
      "94\t999184\t24.8\t1\t972804 26380\n",
      "95\t998306\t24.8\t1\t972314 25992\n",
      "96\t998683\t24.8\t1\t973128 25555\n",
      "97\t1002572\t24.8\t1\t976999 25573\n",
      "98\t1006682\t24.8\t1\t981162 25520\n",
      "99\t1016517\t24.8\t1\t991004 25513\n",
      "100\t1022183\t24.8\t1\t996830 25353\n",
      "101\t1031521\t24.8\t1\t1006155 25366\n",
      "102\t1030657\t24.8\t1\t1005449 25208\n",
      "103\t1027733\t24.8\t1\t1003218 24515\n",
      "104\t1025713\t24.8\t1\t1000982 24731\n",
      "105\t1023390\t24.8\t1\t999444 23946\n",
      "106\t1024626\t24.8\t1\t1000157 24469\n",
      "107\t1026709\t24.8\t1\t1002684 24025\n",
      "108\t1031646\t24.8\t1\t1007518 24128\n",
      "109\t1044142\t24.8\t1\t1020064 24078\n",
      "110\t1054697\t24.8\t1\t1030442 24255\n",
      "111\t1066868\t24.8\t1\t1042597 24271\n",
      "112\t1071797\t24.8\t1\t1048108 23689\n",
      "113\t1068927\t24.8\t1\t1045243 23684\n",
      "114\t1061437\t24.8\t1\t1038079 23358\n",
      "115\t1057880\t24.8\t1\t1034973 22907\n",
      "116\t1053289\t24.8\t1\t1030728 22561\n",
      "117\t1050856\t24.8\t1\t1028684 22172\n",
      "118\t1053560\t24.8\t1\t1031621 21939\n",
      "119\t1061908\t24.8\t1\t1039826 22082\n",
      "120\t1075671\t24.8\t1\t1053660 22011\n",
      "121\t1088488\t24.8\t1\t1066958 21530\n",
      "122\t1099624\t24.8\t1\t1077807 21817\n",
      "123\t1100307\t24.8\t1\t1078581 21726\n",
      "124\t1093196\t24.8\t1\t1071906 21290\n",
      "125\t1088872\t24.8\t1\t1067782 21090\n",
      "126\t1080913\t24.8\t1\t1060665 20248\n",
      "127\t1082559\t24.8\t1\t1062971 19588\n",
      "128\t1091522\t24.8\t1\t1072804 18718\n",
      "129\t1099304\t24.8\t1\t1081305 17999\n",
      "130\t1123433\t24.8\t1\t1106053 17380\n",
      "131\t1143425\t24.8\t1\t1126843 16582\n",
      "132\t1152356\t24.8\t1\t1136746 15610\n",
      "133\t1150847\t24.8\t1\t1136490 14357\n",
      "134\t1092790\t24.8\t1\t1079514 13276\n",
      "135\t1077084\t24.8\t1\t1064836 12248\n",
      "136\t1041814\t24.8\t1\t1030419 11395\n",
      "137\t1020805\t24.8\t1\t1010103 10702\n",
      "138\t975771\t24.8\t1\t965638 10133\n",
      "139\t963770\t24.8\t1\t953785 9985\n",
      "140\t1038313\t24.8\t1\t1027735 10578\n",
      "141\t1114144\t24.8\t1\t1103362 10782\n",
      "142\t1011252\t24.8\t1\t1001043 10209\n",
      "143\t1051290\t24.8\t1\t1041028 10262\n",
      "144\t1119587\t24.8\t1\t1109095 10492\n",
      "145\t1088092\t24.8\t1\t1077647 10445\n",
      "146\t1146022\t24.8\t1\t1135202 10820\n",
      "147\t965267\t24.8\t1\t955299 9968\n",
      "148\t985875\t24.8\t1\t974918 10957\n",
      "149\t829994\t24.8\t1\t819882 10112\n",
      "150\t809616\t24.8\t1\t796770 12846\n",
      "151\t375750\t24.8\t1\t344451 31299\n",
      "\n",
      "\n",
      "RUN STATISTICS FOR INPUT FILE: /home/sam/data/geoduck_illumina/NR013_AD013_S2_L002_R1_001.fastq.gz\n",
      "=============================================\n",
      "415822852 sequences processed in total\n",
      "The length threshold of paired-end sequences gets evaluated later on (in the validation step)\n",
      "\n",
      "Writing report to '/home/sam/data/geoduck_illumina/trimmed/NR013_AD013_S2_L002_R2_001.fastq.gz_trimming_report.txt'\n",
      "\n",
      "SUMMARISING RUN PARAMETERS\n",
      "==========================\n",
      "Input filename: /home/sam/data/geoduck_illumina/NR013_AD013_S2_L002_R2_001.fastq.gz\n",
      "Trimming mode: paired-end\n",
      "Trim Galore version: 0.4.4\n",
      "Cutadapt version: 1.9.1\n",
      "Quality Phred score cutoff: 20\n",
      "Quality encoding type selected: ASCII+33\n",
      "Adapter sequence: 'CTGTCTCTTATA' (Nextera Transposase sequence; auto-detected)\n",
      "Maximum trimming error rate: 0.1 (default)\n",
      "Minimum required adapter overlap (stringency): 1 bp\n",
      "Minimum required sequence length for both reads before a sequence pair gets removed: 20 bp\n",
      "Output file(s) will be GZIP compressed\n",
      "\n",
      "Writing final adapter and quality trimmed output to NR013_AD013_S2_L002_R2_001_trimmed.fq.gz\n",
      "\n",
      "\n",
      "  >>> Now performing quality (cutoff 20) and adapter trimming in a single pass for the adapter sequence: 'CTGTCTCTTATA' from file /home/sam/data/geoduck_illumina/NR013_AD013_S2_L002_R2_001.fastq.gz <<< \n",
      "10000000 sequences processed\n",
      "20000000 sequences processed\n",
      "30000000 sequences processed\n",
      "40000000 sequences processed\n",
      "50000000 sequences processed\n",
      "60000000 sequences processed\n",
      "70000000 sequences processed\n",
      "80000000 sequences processed\n",
      "90000000 sequences processed\n",
      "100000000 sequences processed\n",
      "110000000 sequences processed\n",
      "120000000 sequences processed\n",
      "130000000 sequences processed\n",
      "140000000 sequences processed\n",
      "150000000 sequences processed\n",
      "160000000 sequences processed\n",
      "170000000 sequences processed\n",
      "180000000 sequences processed\n",
      "190000000 sequences processed\n",
      "200000000 sequences processed\n",
      "210000000 sequences processed\n",
      "220000000 sequences processed\n",
      "230000000 sequences processed\n",
      "240000000 sequences processed\n",
      "250000000 sequences processed\n",
      "260000000 sequences processed\n",
      "270000000 sequences processed\n",
      "280000000 sequences processed\n",
      "290000000 sequences processed\n",
      "300000000 sequences processed\n",
      "310000000 sequences processed\n",
      "320000000 sequences processed\n",
      "330000000 sequences processed\n",
      "340000000 sequences processed\n",
      "350000000 sequences processed\n",
      "360000000 sequences processed\n",
      "370000000 sequences processed\n",
      "380000000 sequences processed\n",
      "390000000 sequences processed\n",
      "400000000 sequences processed\n",
      "410000000 sequences processed\n",
      "This is cutadapt 1.9.1 with Python 2.7.12\n",
      "Command line parameters: -f fastq -e 0.1 -q 20 -O 1 -a CTGTCTCTTATA /home/sam/data/geoduck_illumina/NR013_AD013_S2_L002_R2_001.fastq.gz\n",
      "Trimming 1 adapter with at most 10.0% errors in single-end mode ...\n",
      "Finished in 10000.93 s (24 us/read; 2.49 M reads/minute).\n",
      "\n",
      "=== Summary ===\n",
      "\n",
      "Total reads processed:             415,822,852\n",
      "Reads with adapters:               195,051,478 (46.9%)\n",
      "Reads written (passing filters):   415,822,852 (100.0%)\n",
      "\n",
      "Total basepairs processed: 62,789,250,652 bp\n",
      "Quality-trimmed:             300,902,002 bp (0.5%)\n",
      "Total written (filtered):  52,776,478,423 bp (84.1%)\n",
      "\n",
      "=== Adapter 1 ===\n",
      "\n",
      "Sequence: CTGTCTCTTATA; Type: regular 3'; Length: 12; Trimmed: 195051478 times.\n",
      "\n",
      "No. of allowed errors:\n",
      "0-9 bp: 0; 10-12 bp: 1\n",
      "\n",
      "Bases preceding removed adapters:\n",
      "  A: 20.6%\n",
      "  C: 31.4%\n",
      "  G: 20.7%\n",
      "  T: 27.1%\n",
      "  none/other: 0.2%\n",
      "\n",
      "Overview of removed sequences\n",
      "length\tcount\texpect\tmax.err\terror counts\n",
      "1\t50376644\t103955713.0\t0\t50376644\n",
      "2\t14530385\t25988928.2\t0\t14530385\n",
      "3\t4919981\t6497232.1\t0\t4919981\n",
      "4\t2063073\t1624308.0\t0\t2063073\n",
      "5\t1089623\t406077.0\t0\t1089623\n",
      "6\t846907\t101519.3\t0\t846907\n",
      "7\t828973\t25379.8\t0\t828973\n",
      "8\t871760\t6345.0\t0\t871760\n",
      "9\t733129\t1586.2\t0\t723427 9702\n",
      "10\t805998\t396.6\t1\t740838 65160\n",
      "11\t866553\t99.1\t1\t798189 68364\n",
      "12\t766453\t24.8\t1\t707500 58953\n",
      "13\t842509\t24.8\t1\t775794 66715\n",
      "14\t738722\t24.8\t1\t685925 52797\n",
      "15\t919245\t24.8\t1\t842448 76797\n",
      "16\t922621\t24.8\t1\t856628 65993\n",
      "17\t611188\t24.8\t1\t567954 43234\n",
      "18\t840028\t24.8\t1\t777999 62029\n",
      "19\t855634\t24.8\t1\t798944 56690\n",
      "20\t721068\t24.8\t1\t673814 47254\n",
      "21\t848419\t24.8\t1\t782227 66192\n",
      "22\t891577\t24.8\t1\t829103 62474\n",
      "23\t672434\t24.8\t1\t625447 46987\n",
      "24\t856230\t24.8\t1\t790616 65614\n",
      "25\t738811\t24.8\t1\t691994 46817\n",
      "26\t893273\t24.8\t1\t817121 76152\n",
      "27\t840898\t24.8\t1\t781937 58961\n",
      "28\t770808\t24.8\t1\t720250 50558\n",
      "29\t809273\t24.8\t1\t755595 53678\n",
      "30\t853030\t24.8\t1\t796489 56541\n",
      "31\t764530\t24.8\t1\t716938 47592\n",
      "32\t825283\t24.8\t1\t765344 59939\n",
      "33\t770499\t24.8\t1\t722594 47905\n",
      "34\t839651\t24.8\t1\t777504 62147\n",
      "35\t783513\t24.8\t1\t732634 50879\n",
      "36\t897953\t24.8\t1\t840382 57571\n",
      "37\t742736\t24.8\t1\t698140 44596\n",
      "38\t869146\t24.8\t1\t809777 59369\n",
      "39\t846334\t24.8\t1\t788617 57717\n",
      "40\t835033\t24.8\t1\t782357 52676\n",
      "41\t819318\t24.8\t1\t766923 52395\n",
      "42\t832665\t24.8\t1\t780899 51766\n",
      "43\t818450\t24.8\t1\t768055 50395\n",
      "44\t820542\t24.8\t1\t769251 51291\n",
      "45\t831004\t24.8\t1\t779335 51669\n",
      "46\t811273\t24.8\t1\t761779 49494\n",
      "47\t846143\t24.8\t1\t793562 52581\n",
      "48\t822612\t24.8\t1\t773646 48966\n",
      "49\t838768\t24.8\t1\t787145 51623\n",
      "50\t829239\t24.8\t1\t779651 49588\n",
      "51\t835080\t24.8\t1\t784644 50436\n",
      "52\t837242\t24.8\t1\t787293 49949\n",
      "53\t837448\t24.8\t1\t787925 49523\n",
      "54\t832022\t24.8\t1\t782853 49169\n",
      "55\t835247\t24.8\t1\t785871 49376\n",
      "56\t835097\t24.8\t1\t785426 49671\n",
      "57\t837995\t24.8\t1\t788349 49646\n",
      "58\t840020\t24.8\t1\t790213 49807\n",
      "59\t838925\t24.8\t1\t789775 49150\n",
      "60\t838163\t24.8\t1\t789473 48690\n",
      "61\t840633\t24.8\t1\t792371 48262\n",
      "62\t836051\t24.8\t1\t788239 47812\n",
      "63\t836058\t24.8\t1\t787238 48820\n",
      "64\t836865\t24.8\t1\t788863 48002\n",
      "65\t836861\t24.8\t1\t788598 48263\n",
      "66\t837232\t24.8\t1\t789580 47652\n",
      "67\t843818\t24.8\t1\t795801 48017\n",
      "68\t843955\t24.8\t1\t797710 46245\n",
      "69\t850044\t24.8\t1\t804229 45815\n",
      "70\t850165\t24.8\t1\t804166 45999\n",
      "71\t850458\t24.8\t1\t804615 45843\n",
      "72\t845184\t24.8\t1\t800242 44942\n",
      "73\t844426\t24.8\t1\t799306 45120\n",
      "74\t844512\t24.8\t1\t800028 44484\n",
      "75\t844998\t24.8\t1\t800235 44763\n",
      "76\t845256\t24.8\t1\t800467 44789\n",
      "77\t849297\t24.8\t1\t804447 44850\n",
      "78\t851261\t24.8\t1\t806903 44358\n",
      "79\t853746\t24.8\t1\t809400 44346\n",
      "80\t855327\t24.8\t1\t811613 43714\n",
      "81\t854922\t24.8\t1\t811331 43591\n",
      "82\t852487\t24.8\t1\t809322 43165\n",
      "83\t849815\t24.8\t1\t807066 42749\n",
      "84\t847575\t24.8\t1\t804804 42771\n",
      "85\t847253\t24.8\t1\t804749 42504\n",
      "86\t845785\t24.8\t1\t804397 41388\n",
      "87\t851200\t24.8\t1\t809416 41784\n",
      "88\t853308\t24.8\t1\t811548 41760\n",
      "89\t860452\t24.8\t1\t818777 41675\n",
      "90\t862644\t24.8\t1\t820913 41731\n",
      "91\t866608\t24.8\t1\t824895 41713\n",
      "92\t862286\t24.8\t1\t821023 41263\n",
      "93\t859686\t24.8\t1\t818521 41165\n",
      "94\t855080\t24.8\t1\t814910 40170\n",
      "95\t854072\t24.8\t1\t813903 40169\n",
      "96\t854289\t24.8\t1\t814606 39683\n",
      "97\t858145\t24.8\t1\t818021 40124\n",
      "98\t859297\t24.8\t1\t819787 39510\n",
      "99\t868611\t24.8\t1\t828295 40316\n",
      "100\t870352\t24.8\t1\t830927 39425\n",
      "101\t874720\t24.8\t1\t835812 38908\n",
      "102\t872775\t24.8\t1\t834595 38180\n",
      "103\t868011\t24.8\t1\t830697 37314\n",
      "104\t862209\t24.8\t1\t824704 37505\n",
      "105\t860907\t24.8\t1\t824179 36728\n",
      "106\t857830\t24.8\t1\t821703 36127\n",
      "107\t861400\t24.8\t1\t824886 36514\n",
      "108\t865493\t24.8\t1\t829437 36056\n",
      "109\t874548\t24.8\t1\t838358 36190\n",
      "110\t881911\t24.8\t1\t846025 35886\n",
      "111\t889032\t24.8\t1\t852858 36174\n",
      "112\t891732\t24.8\t1\t855803 35929\n",
      "113\t887699\t24.8\t1\t852625 35074\n",
      "114\t879159\t24.8\t1\t844446 34713\n",
      "115\t873492\t24.8\t1\t839178 34314\n",
      "116\t867474\t24.8\t1\t833590 33884\n",
      "117\t867596\t24.8\t1\t834348 33248\n",
      "118\t867927\t24.8\t1\t835268 32659\n",
      "119\t872419\t24.8\t1\t839444 32975\n",
      "120\t882790\t24.8\t1\t850657 32133\n",
      "121\t893102\t24.8\t1\t860255 32847\n",
      "122\t897101\t24.8\t1\t864400 32701\n",
      "123\t896660\t24.8\t1\t864496 32164\n",
      "124\t889027\t24.8\t1\t857300 31727\n",
      "125\t882687\t24.8\t1\t851900 30787\n",
      "126\t875583\t24.8\t1\t845434 30149\n",
      "127\t878522\t24.8\t1\t848723 29799\n",
      "128\t880624\t24.8\t1\t851524 29100\n",
      "129\t891696\t24.8\t1\t862630 29066\n",
      "130\t909308\t24.8\t1\t880950 28358\n",
      "131\t924788\t24.8\t1\t896377 28411\n",
      "132\t928656\t24.8\t1\t901311 27345\n",
      "133\t923575\t24.8\t1\t897579 25996\n",
      "134\t875339\t24.8\t1\t851275 24064\n",
      "135\t859221\t24.8\t1\t836132 23089\n",
      "136\t827125\t24.8\t1\t805178 21947\n",
      "137\t803333\t24.8\t1\t782677 20656\n",
      "138\t764232\t24.8\t1\t744812 19420\n",
      "139\t761529\t24.8\t1\t742327 19202\n",
      "140\t818179\t24.8\t1\t798006 20173\n",
      "141\t864027\t24.8\t1\t843726 20301\n",
      "142\t793172\t24.8\t1\t774186 18986\n",
      "143\t825833\t24.8\t1\t806330 19503\n",
      "144\t873372\t24.8\t1\t853791 19581\n",
      "145\t863405\t24.8\t1\t843829 19576\n",
      "146\t882008\t24.8\t1\t862794 19214\n",
      "147\t746019\t24.8\t1\t728926 17093\n",
      "148\t750025\t24.8\t1\t733057 16968\n",
      "149\t601951\t24.8\t1\t587440 14511\n",
      "150\t581023\t24.8\t1\t565019 16004\n",
      "151\t264040\t24.8\t1\t236618 27422\n",
      "\n",
      "\n",
      "RUN STATISTICS FOR INPUT FILE: /home/sam/data/geoduck_illumina/NR013_AD013_S2_L002_R2_001.fastq.gz\n",
      "=============================================\n",
      "415822852 sequences processed in total\n",
      "The length threshold of paired-end sequences gets evaluated later on (in the validation step)\n",
      "\n",
      "Validate paired-end files NR013_AD013_S2_L002_R1_001_trimmed.fq.gz and NR013_AD013_S2_L002_R2_001_trimmed.fq.gz\n",
      "file_1: NR013_AD013_S2_L002_R1_001_trimmed.fq.gz, file_2: NR013_AD013_S2_L002_R2_001_trimmed.fq.gz\n",
      "\n",
      "\n",
      ">>>>> Now validing the length of the 2 paired-end infiles: NR013_AD013_S2_L002_R1_001_trimmed.fq.gz and NR013_AD013_S2_L002_R2_001_trimmed.fq.gz <<<<<\n",
      "Writing validated paired-end read 1 reads to NR013_AD013_S2_L002_R1_001_val_1.fq.gz\n",
      "Writing validated paired-end read 2 reads to NR013_AD013_S2_L002_R2_001_val_2.fq.gz\n",
      "\n",
      "Total number of sequences analysed: 415822852\n",
      "\n",
      "Number of sequence pairs removed because at least one read was shorter than the length cutoff (20 bp): 36287376 (8.73%)\n",
      "\n",
      "Deleting both intermediate output files NR013_AD013_S2_L002_R1_001_trimmed.fq.gz and NR013_AD013_S2_L002_R2_001_trimmed.fq.gz\n",
      "\n",
      "====================================================================================================\n",
      "\n",
      "Writing report to '/home/sam/data/geoduck_illumina/trimmed/NR014_AD014_S5_L001_R1_001.fastq.gz_trimming_report.txt'\n",
      "\n",
      "SUMMARISING RUN PARAMETERS\n",
      "==========================\n",
      "Input filename: /home/sam/data/geoduck_illumina/NR014_AD014_S5_L001_R1_001.fastq.gz\n",
      "Trimming mode: paired-end\n",
      "Trim Galore version: 0.4.4\n",
      "Cutadapt version: 1.9.1\n",
      "Quality Phred score cutoff: 20\n",
      "Quality encoding type selected: ASCII+33\n",
      "Adapter sequence: 'CTGTCTCTTATA' (Nextera Transposase sequence; auto-detected)\n",
      "Maximum trimming error rate: 0.1 (default)\n",
      "Minimum required adapter overlap (stringency): 1 bp\n",
      "Minimum required sequence length for both reads before a sequence pair gets removed: 20 bp\n",
      "Output file(s) will be GZIP compressed\n",
      "\n",
      "Writing final adapter and quality trimmed output to NR014_AD014_S5_L001_R1_001_trimmed.fq.gz\n",
      "\n",
      "\n",
      "  >>> Now performing quality (cutoff 20) and adapter trimming in a single pass for the adapter sequence: 'CTGTCTCTTATA' from file /home/sam/data/geoduck_illumina/NR014_AD014_S5_L001_R1_001.fastq.gz <<< \n",
      "10000000 sequences processed\n",
      "20000000 sequences processed\n",
      "30000000 sequences processed\n",
      "40000000 sequences processed\n",
      "50000000 sequences processed\n",
      "60000000 sequences processed\n",
      "70000000 sequences processed\n",
      "80000000 sequences processed\n",
      "90000000 sequences processed\n",
      "100000000 sequences processed\n",
      "110000000 sequences processed\n",
      "120000000 sequences processed\n",
      "130000000 sequences processed\n",
      "140000000 sequences processed\n",
      "150000000 sequences processed\n",
      "160000000 sequences processed\n",
      "170000000 sequences processed\n",
      "180000000 sequences processed\n",
      "190000000 sequences processed\n",
      "200000000 sequences processed\n",
      "210000000 sequences processed\n",
      "220000000 sequences processed\n",
      "230000000 sequences processed\n",
      "240000000 sequences processed\n",
      "250000000 sequences processed\n",
      "260000000 sequences processed\n",
      "270000000 sequences processed\n",
      "280000000 sequences processed\n",
      "290000000 sequences processed\n",
      "300000000 sequences processed\n",
      "310000000 sequences processed\n",
      "320000000 sequences processed\n",
      "330000000 sequences processed\n",
      "340000000 sequences processed\n",
      "350000000 sequences processed\n",
      "360000000 sequences processed\n",
      "370000000 sequences processed\n",
      "380000000 sequences processed\n",
      "390000000 sequences processed\n",
      "400000000 sequences processed\n",
      "410000000 sequences processed\n",
      "420000000 sequences processed\n",
      "430000000 sequences processed\n",
      "440000000 sequences processed\n",
      "450000000 sequences processed\n",
      "460000000 sequences processed\n",
      "470000000 sequences processed\n",
      "480000000 sequences processed\n",
      "490000000 sequences processed\n",
      "500000000 sequences processed\n",
      "510000000 sequences processed\n",
      "520000000 sequences processed\n",
      "530000000 sequences processed\n",
      "540000000 sequences processed\n",
      "550000000 sequences processed\n",
      "560000000 sequences processed\n",
      "570000000 sequences processed\n",
      "580000000 sequences processed\n",
      "590000000 sequences processed\n",
      "600000000 sequences processed\n",
      "610000000 sequences processed\n",
      "620000000 sequences processed\n",
      "630000000 sequences processed\n",
      "640000000 sequences processed\n",
      "650000000 sequences processed\n",
      "660000000 sequences processed\n",
      "670000000 sequences processed\n",
      "680000000 sequences processed\n",
      "690000000 sequences processed\n",
      "700000000 sequences processed\n",
      "710000000 sequences processed\n",
      "720000000 sequences processed\n",
      "730000000 sequences processed\n",
      "740000000 sequences processed\n",
      "750000000 sequences processed\n",
      "760000000 sequences processed\n",
      "770000000 sequences processed\n",
      "780000000 sequences processed\n",
      "790000000 sequences processed\n",
      "800000000 sequences processed\n",
      "810000000 sequences processed\n",
      "820000000 sequences processed\n",
      "830000000 sequences processed\n",
      "840000000 sequences processed\n",
      "850000000 sequences processed\n",
      "860000000 sequences processed\n",
      "This is cutadapt 1.9.1 with Python 2.7.12\n",
      "Command line parameters: -f fastq -e 0.1 -q 20 -O 1 -a CTGTCTCTTATA /home/sam/data/geoduck_illumina/NR014_AD014_S5_L001_R1_001.fastq.gz\n",
      "Trimming 1 adapter with at most 10.0% errors in single-end mode ...\n",
      "Finished in 20359.64 s (24 us/read; 2.54 M reads/minute).\n",
      "\n",
      "=== Summary ===\n",
      "\n",
      "Total reads processed:             862,125,010\n",
      "Reads with adapters:               533,204,383 (61.8%)\n",
      "Reads written (passing filters):   862,125,010 (100.0%)\n",
      "\n",
      "Total basepairs processed: 130,180,876,510 bp\n",
      "Quality-trimmed:             334,776,299 bp (0.3%)\n",
      "Total written (filtered):  95,709,543,010 bp (73.5%)\n",
      "\n",
      "=== Adapter 1 ===\n",
      "\n",
      "Sequence: CTGTCTCTTATA; Type: regular 3'; Length: 12; Trimmed: 533204383 times.\n",
      "\n",
      "No. of allowed errors:\n",
      "0-9 bp: 0; 10-12 bp: 1\n",
      "\n",
      "Bases preceding removed adapters:\n",
      "  A: 17.6%\n",
      "  C: 35.0%\n",
      "  G: 21.0%\n",
      "  T: 26.2%\n",
      "  none/other: 0.1%\n",
      "\n",
      "Overview of removed sequences\n",
      "length\tcount\texpect\tmax.err\terror counts\n",
      "1\t87080155\t215531252.5\t0\t87080155\n",
      "2\t25292498\t53882813.1\t0\t25292498\n",
      "3\t9084426\t13470703.3\t0\t9084426\n",
      "4\t3966676\t3367675.8\t0\t3966676\n",
      "5\t2555848\t841919.0\t0\t2555848\n",
      "6\t2088422\t210479.7\t0\t2088422\n",
      "7\t2118728\t52619.9\t0\t2118728\n",
      "8\t2261416\t13155.0\t0\t2261416\n",
      "9\t1886696\t3288.7\t0\t1867679 19017\n",
      "10\t2147259\t822.2\t1\t2004833 142426\n",
      "11\t2203344\t205.5\t1\t2075356 127988\n",
      "12\t2108294\t51.4\t1\t1968257 140037\n",
      "13\t2169915\t51.4\t1\t2041590 128325\n",
      "14\t2021464\t51.4\t1\t1897261 124203\n",
      "15\t2393307\t51.4\t1\t2240058 153249\n",
      "16\t2633473\t51.4\t1\t2468212 165261\n",
      "17\t1606055\t51.4\t1\t1514305 91750\n",
      "18\t2244922\t51.4\t1\t2117650 127272\n",
      "19\t2287131\t51.4\t1\t2169631 117500\n",
      "20\t2080994\t51.4\t1\t1956165 124829\n",
      "21\t2250052\t51.4\t1\t2131462 118590\n",
      "22\t2488161\t51.4\t1\t2345857 142304\n",
      "23\t1936361\t51.4\t1\t1824356 112005\n",
      "24\t2320736\t51.4\t1\t2200479 120257\n",
      "25\t2080551\t51.4\t1\t1974444 106107\n",
      "26\t2398126\t51.4\t1\t2261532 136594\n",
      "27\t2282553\t51.4\t1\t2167217 115336\n",
      "28\t2203983\t51.4\t1\t2090981 113002\n",
      "29\t2265578\t51.4\t1\t2158870 106708\n",
      "30\t2459124\t51.4\t1\t2334870 124254\n",
      "31\t2336574\t51.4\t1\t2214330 122244\n",
      "32\t2241450\t51.4\t1\t2137849 103601\n",
      "33\t2300131\t51.4\t1\t2182495 117636\n",
      "34\t2276939\t51.4\t1\t2169620 107319\n",
      "35\t2342900\t51.4\t1\t2222984 119916\n",
      "36\t2497021\t51.4\t1\t2381268 115753\n",
      "37\t2287522\t51.4\t1\t2167544 119978\n",
      "38\t2470408\t51.4\t1\t2352984 117424\n",
      "39\t2425490\t51.4\t1\t2310420 115070\n",
      "40\t2439284\t51.4\t1\t2323810 115474\n",
      "41\t2414344\t51.4\t1\t2297988 116356\n",
      "42\t2448239\t51.4\t1\t2334656 113583\n",
      "43\t2437622\t51.4\t1\t2324780 112842\n",
      "44\t2456618\t51.4\t1\t2341773 114845\n",
      "45\t2462796\t51.4\t1\t2350224 112572\n",
      "46\t2462042\t51.4\t1\t2346814 115228\n",
      "47\t2500349\t51.4\t1\t2387065 113284\n",
      "48\t2496001\t51.4\t1\t2382375 113626\n",
      "49\t2521603\t51.4\t1\t2407901 113702\n",
      "50\t2513138\t51.4\t1\t2401474 111664\n",
      "51\t2534183\t51.4\t1\t2421520 112663\n",
      "52\t2540436\t51.4\t1\t2430240 110196\n",
      "53\t2565999\t51.4\t1\t2455494 110505\n",
      "54\t2555806\t51.4\t1\t2445598 110208\n",
      "55\t2572280\t51.4\t1\t2462609 109671\n",
      "56\t2574683\t51.4\t1\t2465802 108881\n",
      "57\t2594847\t51.4\t1\t2485833 109014\n",
      "58\t2610567\t51.4\t1\t2501646 108921\n",
      "59\t2627272\t51.4\t1\t2517701 109571\n",
      "60\t2635121\t51.4\t1\t2526668 108453\n",
      "61\t2654719\t51.4\t1\t2545287 109432\n",
      "62\t2657073\t51.4\t1\t2549708 107365\n",
      "63\t2667257\t51.4\t1\t2560316 106941\n",
      "64\t2670252\t51.4\t1\t2563309 106943\n",
      "65\t2680079\t51.4\t1\t2573975 106104\n",
      "66\t2688354\t51.4\t1\t2581832 106522\n",
      "67\t2705663\t51.4\t1\t2600297 105366\n",
      "68\t2720740\t51.4\t1\t2614255 106485\n",
      "69\t2733983\t51.4\t1\t2628177 105806\n",
      "70\t2750706\t51.4\t1\t2644823 105883\n",
      "71\t2768728\t51.4\t1\t2662313 106415\n",
      "72\t2763810\t51.4\t1\t2659284 104526\n",
      "73\t2779320\t51.4\t1\t2674617 104703\n",
      "74\t2780824\t51.4\t1\t2676963 103861\n",
      "75\t2790976\t51.4\t1\t2687019 103957\n",
      "76\t2798570\t51.4\t1\t2695245 103325\n",
      "77\t2816231\t51.4\t1\t2713058 103173\n",
      "78\t2829212\t51.4\t1\t2725782 103430\n",
      "79\t2853748\t51.4\t1\t2749834 103914\n",
      "80\t2869560\t51.4\t1\t2766239 103321\n",
      "81\t2890251\t51.4\t1\t2786754 103497\n",
      "82\t2891288\t51.4\t1\t2788604 102684\n",
      "83\t2902158\t51.4\t1\t2799792 102366\n",
      "84\t2896361\t51.4\t1\t2794399 101962\n",
      "85\t2905313\t51.4\t1\t2803625 101688\n",
      "86\t2909616\t51.4\t1\t2809255 100361\n",
      "87\t2923807\t51.4\t1\t2824179 99628\n",
      "88\t2936907\t51.4\t1\t2836420 100487\n",
      "89\t2973776\t51.4\t1\t2873815 99961\n",
      "90\t2990303\t51.4\t1\t2890819 99484\n",
      "91\t3017239\t51.4\t1\t2917249 99990\n",
      "92\t3018303\t51.4\t1\t2919523 98780\n",
      "93\t3014708\t51.4\t1\t2916507 98201\n",
      "94\t3007926\t51.4\t1\t2910368 97558\n",
      "95\t3009371\t51.4\t1\t2913587 95784\n",
      "96\t3014847\t51.4\t1\t2919837 95010\n",
      "97\t3019027\t51.4\t1\t2925475 93552\n",
      "98\t3028470\t51.4\t1\t2935414 93056\n",
      "99\t3062728\t51.4\t1\t2968036 94692\n",
      "100\t3090295\t51.4\t1\t2996430 93865\n",
      "101\t3124072\t51.4\t1\t3029715 94357\n",
      "102\t3134236\t51.4\t1\t3040238 93998\n",
      "103\t3138432\t51.4\t1\t3045501 92931\n",
      "104\t3128789\t51.4\t1\t3036751 92038\n",
      "105\t3133332\t51.4\t1\t3041492 91840\n",
      "106\t3137813\t51.4\t1\t3042358 95455\n",
      "107\t3150614\t51.4\t1\t3056704 93910\n",
      "108\t3161119\t51.4\t1\t3066820 94299\n",
      "109\t3198509\t51.4\t1\t3103458 95051\n",
      "110\t3227579\t51.4\t1\t3132921 94658\n",
      "111\t3260459\t51.4\t1\t3166715 93744\n",
      "112\t3279500\t51.4\t1\t3185425 94075\n",
      "113\t3277782\t51.4\t1\t3184045 93737\n",
      "114\t3261944\t51.4\t1\t3169228 92716\n",
      "115\t3254278\t51.4\t1\t3163286 90992\n",
      "116\t3240928\t51.4\t1\t3150610 90318\n",
      "117\t3242700\t51.4\t1\t3153325 89375\n",
      "118\t3250451\t51.4\t1\t3165383 85068\n",
      "119\t3273347\t51.4\t1\t3188209 85138\n",
      "120\t3318878\t51.4\t1\t3232818 86060\n",
      "121\t3362073\t51.4\t1\t3275743 86330\n",
      "122\t3385722\t51.4\t1\t3299424 86298\n",
      "123\t3391280\t51.4\t1\t3305843 85437\n",
      "124\t3368401\t51.4\t1\t3283712 84689\n",
      "125\t3350159\t51.4\t1\t3266191 83968\n",
      "126\t3337400\t51.4\t1\t3255357 82043\n",
      "127\t3346478\t51.4\t1\t3267373 79105\n",
      "128\t3365912\t51.4\t1\t3290370 75542\n",
      "129\t3392332\t51.4\t1\t3319468 72864\n",
      "130\t3456596\t51.4\t1\t3386811 69785\n",
      "131\t3512135\t51.4\t1\t3443703 68432\n",
      "132\t3534163\t51.4\t1\t3470024 64139\n",
      "133\t3527032\t51.4\t1\t3467795 59237\n",
      "134\t3366546\t51.4\t1\t3313043 53503\n",
      "135\t3314373\t51.4\t1\t3264902 49471\n",
      "136\t3232974\t51.4\t1\t3185864 47110\n",
      "137\t3169126\t51.4\t1\t3125121 44005\n",
      "138\t3051003\t51.4\t1\t3008768 42235\n",
      "139\t3031240\t51.4\t1\t2989648 41592\n",
      "140\t3206007\t51.4\t1\t3161792 44215\n",
      "141\t3388074\t51.4\t1\t3342324 45750\n",
      "142\t3064860\t51.4\t1\t3021913 42947\n",
      "143\t3112107\t51.4\t1\t3069853 42254\n",
      "144\t3282636\t51.4\t1\t3237661 44975\n",
      "145\t3229456\t51.4\t1\t3184052 45404\n",
      "146\t3411642\t51.4\t1\t3362254 49388\n",
      "147\t2895283\t51.4\t1\t2850748 44535\n",
      "148\t2936412\t51.4\t1\t2889423 46989\n",
      "149\t2482511\t51.4\t1\t2437084 45427\n",
      "150\t2322797\t51.4\t1\t2273335 49462\n",
      "151\t642479\t51.4\t1\t558867 83612\n",
      "\n",
      "\n",
      "RUN STATISTICS FOR INPUT FILE: /home/sam/data/geoduck_illumina/NR014_AD014_S5_L001_R1_001.fastq.gz\n",
      "=============================================\n",
      "862125010 sequences processed in total\n",
      "The length threshold of paired-end sequences gets evaluated later on (in the validation step)\n",
      "\n",
      "Writing report to '/home/sam/data/geoduck_illumina/trimmed/NR014_AD014_S5_L001_R2_001.fastq.gz_trimming_report.txt'\n",
      "\n",
      "SUMMARISING RUN PARAMETERS\n",
      "==========================\n",
      "Input filename: /home/sam/data/geoduck_illumina/NR014_AD014_S5_L001_R2_001.fastq.gz\n",
      "Trimming mode: paired-end\n",
      "Trim Galore version: 0.4.4\n",
      "Cutadapt version: 1.9.1\n",
      "Quality Phred score cutoff: 20\n",
      "Quality encoding type selected: ASCII+33\n",
      "Adapter sequence: 'CTGTCTCTTATA' (Nextera Transposase sequence; auto-detected)\n",
      "Maximum trimming error rate: 0.1 (default)\n",
      "Minimum required adapter overlap (stringency): 1 bp\n",
      "Minimum required sequence length for both reads before a sequence pair gets removed: 20 bp\n",
      "Output file(s) will be GZIP compressed\n",
      "\n",
      "Writing final adapter and quality trimmed output to NR014_AD014_S5_L001_R2_001_trimmed.fq.gz\n",
      "\n",
      "\n",
      "  >>> Now performing quality (cutoff 20) and adapter trimming in a single pass for the adapter sequence: 'CTGTCTCTTATA' from file /home/sam/data/geoduck_illumina/NR014_AD014_S5_L001_R2_001.fastq.gz <<< \n",
      "10000000 sequences processed\n",
      "20000000 sequences processed\n",
      "30000000 sequences processed\n",
      "40000000 sequences processed\n",
      "50000000 sequences processed\n",
      "60000000 sequences processed\n",
      "70000000 sequences processed\n",
      "80000000 sequences processed\n",
      "90000000 sequences processed\n",
      "100000000 sequences processed\n",
      "110000000 sequences processed\n",
      "120000000 sequences processed\n",
      "130000000 sequences processed\n",
      "140000000 sequences processed\n",
      "150000000 sequences processed\n",
      "160000000 sequences processed\n",
      "170000000 sequences processed\n",
      "180000000 sequences processed\n",
      "190000000 sequences processed\n",
      "200000000 sequences processed\n",
      "210000000 sequences processed\n",
      "220000000 sequences processed\n",
      "230000000 sequences processed\n",
      "240000000 sequences processed\n",
      "250000000 sequences processed\n",
      "260000000 sequences processed\n",
      "270000000 sequences processed\n",
      "280000000 sequences processed\n",
      "290000000 sequences processed\n",
      "300000000 sequences processed\n",
      "310000000 sequences processed\n",
      "320000000 sequences processed\n",
      "330000000 sequences processed\n",
      "340000000 sequences processed\n",
      "350000000 sequences processed\n",
      "360000000 sequences processed\n",
      "370000000 sequences processed\n",
      "380000000 sequences processed\n",
      "390000000 sequences processed\n",
      "400000000 sequences processed\n",
      "410000000 sequences processed\n",
      "420000000 sequences processed\n",
      "430000000 sequences processed\n",
      "440000000 sequences processed\n",
      "450000000 sequences processed\n",
      "460000000 sequences processed\n",
      "470000000 sequences processed\n",
      "480000000 sequences processed\n",
      "490000000 sequences processed\n",
      "500000000 sequences processed\n",
      "510000000 sequences processed\n",
      "520000000 sequences processed\n",
      "530000000 sequences processed\n",
      "540000000 sequences processed\n",
      "550000000 sequences processed\n",
      "560000000 sequences processed\n",
      "570000000 sequences processed\n",
      "580000000 sequences processed\n",
      "590000000 sequences processed\n",
      "600000000 sequences processed\n",
      "610000000 sequences processed\n",
      "620000000 sequences processed\n",
      "630000000 sequences processed\n",
      "640000000 sequences processed\n",
      "650000000 sequences processed\n",
      "660000000 sequences processed\n",
      "670000000 sequences processed\n",
      "680000000 sequences processed\n",
      "690000000 sequences processed\n",
      "700000000 sequences processed\n",
      "710000000 sequences processed\n",
      "720000000 sequences processed\n",
      "730000000 sequences processed\n",
      "740000000 sequences processed\n",
      "750000000 sequences processed\n",
      "760000000 sequences processed\n",
      "770000000 sequences processed\n",
      "780000000 sequences processed\n",
      "790000000 sequences processed\n",
      "800000000 sequences processed\n",
      "810000000 sequences processed\n",
      "820000000 sequences processed\n",
      "830000000 sequences processed\n",
      "840000000 sequences processed\n",
      "850000000 sequences processed\n",
      "860000000 sequences processed\n",
      "This is cutadapt 1.9.1 with Python 2.7.12\n",
      "Command line parameters: -f fastq -e 0.1 -q 20 -O 1 -a CTGTCTCTTATA /home/sam/data/geoduck_illumina/NR014_AD014_S5_L001_R2_001.fastq.gz\n",
      "Trimming 1 adapter with at most 10.0% errors in single-end mode ...\n",
      "Finished in 20481.41 s (24 us/read; 2.53 M reads/minute).\n",
      "\n",
      "=== Summary ===\n",
      "\n",
      "Total reads processed:             862,125,010\n",
      "Reads with adapters:               487,764,267 (56.6%)\n",
      "Reads written (passing filters):   862,125,010 (100.0%)\n",
      "\n",
      "Total basepairs processed: 130,180,876,510 bp\n",
      "Quality-trimmed:             491,072,754 bp (0.4%)\n",
      "Total written (filtered):  99,751,113,723 bp (76.6%)\n",
      "\n",
      "=== Adapter 1 ===\n",
      "\n",
      "Sequence: CTGTCTCTTATA; Type: regular 3'; Length: 12; Trimmed: 487764267 times.\n",
      "\n",
      "No. of allowed errors:\n",
      "0-9 bp: 0; 10-12 bp: 1\n",
      "\n",
      "Bases preceding removed adapters:\n",
      "  A: 18.2%\n",
      "  C: 34.2%\n",
      "  G: 21.3%\n",
      "  T: 26.2%\n",
      "  none/other: 0.1%\n",
      "\n",
      "Overview of removed sequences\n",
      "length\tcount\texpect\tmax.err\terror counts\n",
      "1\t87028076\t215531252.5\t0\t87028076\n",
      "2\t25005929\t53882813.1\t0\t25005929\n",
      "3\t8757729\t13470703.3\t0\t8757729\n",
      "4\t3927470\t3367675.8\t0\t3927470\n",
      "5\t2400919\t841919.0\t0\t2400919\n",
      "6\t1921020\t210479.7\t0\t1921020\n",
      "7\t1926124\t52619.9\t0\t1926124\n",
      "8\t2100507\t13155.0\t0\t2100507\n",
      "9\t1691346\t3288.7\t0\t1670888 20458\n",
      "10\t1983842\t822.2\t1\t1799270 184572\n",
      "11\t2049425\t205.5\t1\t1872211 177214\n",
      "12\t1908398\t51.4\t1\t1732610 175788\n",
      "13\t2013760\t51.4\t1\t1833479 180281\n",
      "14\t1820806\t51.4\t1\t1662157 158649\n",
      "15\t2236287\t51.4\t1\t2029086 207201\n",
      "16\t2336471\t51.4\t1\t2135777 200694\n",
      "17\t1529610\t51.4\t1\t1404488 125122\n",
      "18\t2075483\t51.4\t1\t1901684 173799\n",
      "19\t2125493\t51.4\t1\t1964773 160720\n",
      "20\t1889054\t51.4\t1\t1738034 151020\n",
      "21\t2065453\t51.4\t1\t1899037 166416\n",
      "22\t2264900\t51.4\t1\t2083297 181603\n",
      "23\t1792766\t51.4\t1\t1648943 143823\n",
      "24\t2116890\t51.4\t1\t1946592 170298\n",
      "25\t1937600\t51.4\t1\t1796850 140750\n",
      "26\t2211197\t51.4\t1\t2022651 188546\n",
      "27\t2119858\t51.4\t1\t1959552 160306\n",
      "28\t2017289\t51.4\t1\t1868078 149211\n",
      "29\t2092915\t51.4\t1\t1941839 151076\n",
      "30\t2230433\t51.4\t1\t2069250 161183\n",
      "31\t2116272\t51.4\t1\t1964321 151951\n",
      "32\t2074805\t51.4\t1\t1926024 148781\n",
      "33\t2112018\t51.4\t1\t1965296 146722\n",
      "34\t2126005\t51.4\t1\t1974863 151142\n",
      "35\t2086790\t51.4\t1\t1941610 145180\n",
      "36\t2349700\t51.4\t1\t2193950 155750\n",
      "37\t2043605\t51.4\t1\t1903729 139876\n",
      "38\t2281250\t51.4\t1\t2127512 153738\n",
      "39\t2232758\t51.4\t1\t2083895 148863\n",
      "40\t2246748\t51.4\t1\t2105177 141571\n",
      "41\t2215186\t51.4\t1\t2075304 139882\n",
      "42\t2256580\t51.4\t1\t2119006 137574\n",
      "43\t2236822\t51.4\t1\t2100470 136352\n",
      "44\t2247188\t51.4\t1\t2109839 137349\n",
      "45\t2264335\t51.4\t1\t2127647 136688\n",
      "46\t2239179\t51.4\t1\t2104091 135088\n",
      "47\t2311919\t51.4\t1\t2173517 138402\n",
      "48\t2282704\t51.4\t1\t2150238 132466\n",
      "49\t2314448\t51.4\t1\t2179692 134756\n",
      "50\t2306588\t51.4\t1\t2174950 131638\n",
      "51\t2334353\t51.4\t1\t2200395 133958\n",
      "52\t2341092\t51.4\t1\t2208567 132525\n",
      "53\t2361761\t51.4\t1\t2228645 133116\n",
      "54\t2353318\t51.4\t1\t2223325 129993\n",
      "55\t2371829\t51.4\t1\t2239474 132355\n",
      "56\t2377716\t51.4\t1\t2245731 131985\n",
      "57\t2382829\t51.4\t1\t2250386 132443\n",
      "58\t2401828\t51.4\t1\t2269156 132672\n",
      "59\t2404005\t51.4\t1\t2272998 131007\n",
      "60\t2411636\t51.4\t1\t2282859 128777\n",
      "61\t2420325\t51.4\t1\t2291354 128971\n",
      "62\t2409011\t51.4\t1\t2282853 126158\n",
      "63\t2412489\t51.4\t1\t2287903 124586\n",
      "64\t2422442\t51.4\t1\t2298046 124396\n",
      "65\t2430842\t51.4\t1\t2306564 124278\n",
      "66\t2441731\t51.4\t1\t2318398 123333\n",
      "67\t2462080\t51.4\t1\t2338880 123200\n",
      "68\t2477727\t51.4\t1\t2358116 119611\n",
      "69\t2494630\t51.4\t1\t2375510 119120\n",
      "70\t2514500\t51.4\t1\t2396752 117748\n",
      "71\t2538019\t51.4\t1\t2419844 118175\n",
      "72\t2535544\t51.4\t1\t2419572 115972\n",
      "73\t2547911\t51.4\t1\t2430295 117616\n",
      "74\t2551576\t51.4\t1\t2436283 115293\n",
      "75\t2564161\t51.4\t1\t2448581 115580\n",
      "76\t2568464\t51.4\t1\t2451468 116996\n",
      "77\t2574669\t51.4\t1\t2459014 115655\n",
      "78\t2576573\t51.4\t1\t2460550 116023\n",
      "79\t2586971\t51.4\t1\t2471587 115384\n",
      "80\t2601956\t51.4\t1\t2487255 114701\n",
      "81\t2604089\t51.4\t1\t2490316 113773\n",
      "82\t2598946\t51.4\t1\t2486257 112689\n",
      "83\t2597111\t51.4\t1\t2484667 112444\n",
      "84\t2596091\t51.4\t1\t2484893 111198\n",
      "85\t2600808\t51.4\t1\t2491003 109805\n",
      "86\t2602017\t51.4\t1\t2492772 109245\n",
      "87\t2617675\t51.4\t1\t2509804 107871\n",
      "88\t2628059\t51.4\t1\t2522908 105151\n",
      "89\t2656556\t51.4\t1\t2550977 105579\n",
      "90\t2678549\t51.4\t1\t2572181 106368\n",
      "91\t2711992\t51.4\t1\t2605720 106272\n",
      "92\t2709747\t51.4\t1\t2604331 105416\n",
      "93\t2720802\t51.4\t1\t2616590 104212\n",
      "94\t2715438\t51.4\t1\t2612666 102772\n",
      "95\t2715878\t51.4\t1\t2613599 102279\n",
      "96\t2717943\t51.4\t1\t2616783 101160\n",
      "97\t2727126\t51.4\t1\t2626685 100441\n",
      "98\t2729717\t51.4\t1\t2630179 99538\n",
      "99\t2750261\t51.4\t1\t2650773 99488\n",
      "100\t2769128\t51.4\t1\t2669279 99849\n",
      "101\t2784074\t51.4\t1\t2685552 98522\n",
      "102\t2779256\t51.4\t1\t2682293 96963\n",
      "103\t2774652\t51.4\t1\t2677750 96902\n",
      "104\t2764015\t51.4\t1\t2668058 95957\n",
      "105\t2763579\t51.4\t1\t2667341 96238\n",
      "106\t2760893\t51.4\t1\t2666635 94258\n",
      "107\t2771229\t51.4\t1\t2676722 94507\n",
      "108\t2778417\t51.4\t1\t2684624 93793\n",
      "109\t2808111\t51.4\t1\t2714040 94071\n",
      "110\t2835428\t51.4\t1\t2741756 93672\n",
      "111\t2859636\t51.4\t1\t2765558 94078\n",
      "112\t2867098\t51.4\t1\t2773415 93683\n",
      "113\t2871085\t51.4\t1\t2778831 92254\n",
      "114\t2848747\t51.4\t1\t2754596 94151\n",
      "115\t2836727\t51.4\t1\t2745064 91663\n",
      "116\t2822508\t51.4\t1\t2733691 88817\n",
      "117\t2822178\t51.4\t1\t2734575 87603\n",
      "118\t2820961\t51.4\t1\t2733877 87084\n",
      "119\t2840487\t51.4\t1\t2755127 85360\n",
      "120\t2872604\t51.4\t1\t2786743 85861\n",
      "121\t2905072\t51.4\t1\t2817025 88047\n",
      "122\t2918926\t51.4\t1\t2831786 87140\n",
      "123\t2918551\t51.4\t1\t2832810 85741\n",
      "124\t2893447\t51.4\t1\t2808747 84700\n",
      "125\t2876293\t51.4\t1\t2792744 83549\n",
      "126\t2860089\t51.4\t1\t2780036 80053\n",
      "127\t2863069\t51.4\t1\t2784561 78508\n",
      "128\t2873015\t51.4\t1\t2795738 77277\n",
      "129\t2897555\t51.4\t1\t2819879 77676\n",
      "130\t2950723\t51.4\t1\t2873723 77000\n",
      "131\t2987708\t51.4\t1\t2910728 76980\n",
      "132\t3006128\t51.4\t1\t2931330 74798\n",
      "133\t2987603\t51.4\t1\t2915914 71689\n",
      "134\t2858781\t51.4\t1\t2791363 67418\n",
      "135\t2810253\t51.4\t1\t2744333 65920\n",
      "136\t2729398\t51.4\t1\t2663657 65741\n",
      "137\t2657428\t51.4\t1\t2594103 63325\n",
      "138\t2554440\t51.4\t1\t2491392 63048\n",
      "139\t2535549\t51.4\t1\t2473600 61949\n",
      "140\t2693635\t51.4\t1\t2628959 64676\n",
      "141\t2807944\t51.4\t1\t2741639 66305\n",
      "142\t2574701\t51.4\t1\t2512666 62035\n",
      "143\t2623685\t51.4\t1\t2562540 61145\n",
      "144\t2760564\t51.4\t1\t2696845 63719\n",
      "145\t2746186\t51.4\t1\t2684250 61936\n",
      "146\t2833928\t51.4\t1\t2770612 63316\n",
      "147\t2428426\t51.4\t1\t2375426 53000\n",
      "148\t2415547\t51.4\t1\t2366272 49275\n",
      "149\t1965341\t51.4\t1\t1923068 42273\n",
      "150\t1809700\t51.4\t1\t1764804 44896\n",
      "151\t497026\t51.4\t1\t420224 76802\n",
      "\n",
      "\n",
      "RUN STATISTICS FOR INPUT FILE: /home/sam/data/geoduck_illumina/NR014_AD014_S5_L001_R2_001.fastq.gz\n",
      "=============================================\n",
      "862125010 sequences processed in total\n",
      "The length threshold of paired-end sequences gets evaluated later on (in the validation step)\n",
      "\n",
      "Validate paired-end files NR014_AD014_S5_L001_R1_001_trimmed.fq.gz and NR014_AD014_S5_L001_R2_001_trimmed.fq.gz\n",
      "file_1: NR014_AD014_S5_L001_R1_001_trimmed.fq.gz, file_2: NR014_AD014_S5_L001_R2_001_trimmed.fq.gz\n",
      "\n",
      "\n",
      ">>>>> Now validing the length of the 2 paired-end infiles: NR014_AD014_S5_L001_R1_001_trimmed.fq.gz and NR014_AD014_S5_L001_R2_001_trimmed.fq.gz <<<<<\n",
      "Writing validated paired-end read 1 reads to NR014_AD014_S5_L001_R1_001_val_1.fq.gz\n",
      "Writing validated paired-end read 2 reads to NR014_AD014_S5_L001_R2_001_val_2.fq.gz\n",
      "\n",
      "Total number of sequences analysed: 862125010\n",
      "\n",
      "Number of sequence pairs removed because at least one read was shorter than the length cutoff (20 bp): 111554567 (12.94%)\n",
      "\n",
      "Deleting both intermediate output files NR014_AD014_S5_L001_R1_001_trimmed.fq.gz and NR014_AD014_S5_L001_R2_001_trimmed.fq.gz\n",
      "\n",
      "====================================================================================================\n",
      "\n",
      "Writing report to '/home/sam/data/geoduck_illumina/trimmed/NR014_AD014_S5_L002_R1_001.fastq.gz_trimming_report.txt'\n",
      "\n",
      "SUMMARISING RUN PARAMETERS\n",
      "==========================\n",
      "Input filename: /home/sam/data/geoduck_illumina/NR014_AD014_S5_L002_R1_001.fastq.gz\n",
      "Trimming mode: paired-end\n",
      "Trim Galore version: 0.4.4\n",
      "Cutadapt version: 1.9.1\n",
      "Quality Phred score cutoff: 20\n",
      "Quality encoding type selected: ASCII+33\n",
      "Adapter sequence: 'CTGTCTCTTATA' (Nextera Transposase sequence; auto-detected)\n",
      "Maximum trimming error rate: 0.1 (default)\n",
      "Minimum required adapter overlap (stringency): 1 bp\n",
      "Minimum required sequence length for both reads before a sequence pair gets removed: 20 bp\n",
      "Output file(s) will be GZIP compressed\n",
      "\n",
      "Writing final adapter and quality trimmed output to NR014_AD014_S5_L002_R1_001_trimmed.fq.gz\n",
      "\n",
      "\n",
      "  >>> Now performing quality (cutoff 20) and adapter trimming in a single pass for the adapter sequence: 'CTGTCTCTTATA' from file /home/sam/data/geoduck_illumina/NR014_AD014_S5_L002_R1_001.fastq.gz <<< \n",
      "10000000 sequences processed\n",
      "20000000 sequences processed\n",
      "30000000 sequences processed\n",
      "40000000 sequences processed\n",
      "50000000 sequences processed\n",
      "60000000 sequences processed\n",
      "70000000 sequences processed\n",
      "80000000 sequences processed\n",
      "90000000 sequences processed\n",
      "100000000 sequences processed\n",
      "110000000 sequences processed\n",
      "120000000 sequences processed\n",
      "130000000 sequences processed\n",
      "140000000 sequences processed\n",
      "150000000 sequences processed\n",
      "160000000 sequences processed\n",
      "170000000 sequences processed\n",
      "180000000 sequences processed\n",
      "190000000 sequences processed\n",
      "200000000 sequences processed\n",
      "210000000 sequences processed\n",
      "220000000 sequences processed\n",
      "230000000 sequences processed\n",
      "240000000 sequences processed\n",
      "250000000 sequences processed\n",
      "260000000 sequences processed\n",
      "270000000 sequences processed\n",
      "280000000 sequences processed\n",
      "290000000 sequences processed\n",
      "300000000 sequences processed\n",
      "310000000 sequences processed\n",
      "320000000 sequences processed\n",
      "330000000 sequences processed\n",
      "340000000 sequences processed\n",
      "350000000 sequences processed\n",
      "360000000 sequences processed\n",
      "370000000 sequences processed\n",
      "380000000 sequences processed\n",
      "390000000 sequences processed\n",
      "400000000 sequences processed\n",
      "410000000 sequences processed\n",
      "420000000 sequences processed\n",
      "430000000 sequences processed\n",
      "440000000 sequences processed\n",
      "450000000 sequences processed\n",
      "460000000 sequences processed\n",
      "470000000 sequences processed\n",
      "480000000 sequences processed\n",
      "490000000 sequences processed\n",
      "500000000 sequences processed\n",
      "510000000 sequences processed\n",
      "520000000 sequences processed\n",
      "530000000 sequences processed\n",
      "540000000 sequences processed\n",
      "550000000 sequences processed\n",
      "560000000 sequences processed\n",
      "570000000 sequences processed\n",
      "580000000 sequences processed\n",
      "590000000 sequences processed\n",
      "600000000 sequences processed\n",
      "610000000 sequences processed\n",
      "620000000 sequences processed\n",
      "630000000 sequences processed\n",
      "640000000 sequences processed\n",
      "650000000 sequences processed\n",
      "660000000 sequences processed\n",
      "670000000 sequences processed\n",
      "680000000 sequences processed\n",
      "690000000 sequences processed\n",
      "700000000 sequences processed\n",
      "710000000 sequences processed\n",
      "720000000 sequences processed\n",
      "730000000 sequences processed\n",
      "740000000 sequences processed\n",
      "750000000 sequences processed\n",
      "760000000 sequences processed\n",
      "770000000 sequences processed\n",
      "780000000 sequences processed\n",
      "790000000 sequences processed\n",
      "800000000 sequences processed\n",
      "810000000 sequences processed\n",
      "820000000 sequences processed\n",
      "830000000 sequences processed\n",
      "840000000 sequences processed\n",
      "850000000 sequences processed\n",
      "860000000 sequences processed\n",
      "870000000 sequences processed\n",
      "880000000 sequences processed\n",
      "890000000 sequences processed\n",
      "900000000 sequences processed\n",
      "910000000 sequences processed\n",
      "920000000 sequences processed\n",
      "930000000 sequences processed\n",
      "940000000 sequences processed\n",
      "950000000 sequences processed\n",
      "960000000 sequences processed\n",
      "970000000 sequences processed\n",
      "980000000 sequences processed\n",
      "990000000 sequences processed\n",
      "1000000000 sequences processed\n",
      "This is cutadapt 1.9.1 with Python 2.7.12\n",
      "Command line parameters: -f fastq -e 0.1 -q 20 -O 1 -a CTGTCTCTTATA /home/sam/data/geoduck_illumina/NR014_AD014_S5_L002_R1_001.fastq.gz\n",
      "Trimming 1 adapter with at most 10.0% errors in single-end mode ...\n",
      "Finished in 23581.69 s (24 us/read; 2.55 M reads/minute).\n",
      "\n",
      "=== Summary ===\n",
      "\n",
      "Total reads processed:           1,001,146,597\n",
      "Reads with adapters:               591,796,332 (59.1%)\n",
      "Reads written (passing filters): 1,001,146,597 (100.0%)\n",
      "\n",
      "Total basepairs processed: 151,173,136,147 bp\n",
      "Quality-trimmed:             401,721,818 bp (0.3%)\n",
      "Total written (filtered):  114,053,730,623 bp (75.4%)\n",
      "\n",
      "=== Adapter 1 ===\n",
      "\n",
      "Sequence: CTGTCTCTTATA; Type: regular 3'; Length: 12; Trimmed: 591796332 times.\n",
      "\n",
      "No. of allowed errors:\n",
      "0-9 bp: 0; 10-12 bp: 1\n",
      "\n",
      "Bases preceding removed adapters:\n",
      "  A: 18.0%\n",
      "  C: 34.6%\n",
      "  G: 21.0%\n",
      "  T: 26.3%\n",
      "  none/other: 0.1%\n",
      "\n",
      "Overview of removed sequences\n",
      "length\tcount\texpect\tmax.err\terror counts\n",
      "1\t106033455\t250286649.2\t0\t106033455\n",
      "2\t30932237\t62571662.3\t0\t30932237\n",
      "3\t10827392\t15642915.6\t0\t10827392\n",
      "4\t4701997\t3910728.9\t0\t4701997\n",
      "5\t2851053\t977682.2\t0\t2851053\n",
      "6\t2372352\t244420.6\t0\t2372352\n",
      "7\t2341225\t61105.1\t0\t2341225\n",
      "8\t2467049\t15276.3\t0\t2467049\n",
      "9\t2128322\t3819.1\t0\t2108774 19548\n",
      "10\t2353039\t954.8\t1\t2243274 109765\n",
      "11\t2416142\t238.7\t1\t2320919 95223\n",
      "12\t2311432\t59.7\t1\t2212214 99218\n",
      "13\t2380798\t59.7\t1\t2289511 91287\n",
      "14\t2252041\t59.7\t1\t2163098 88943\n",
      "15\t2555705\t59.7\t1\t2446445 109260\n",
      "16\t2850726\t59.7\t1\t2733034 117692\n",
      "17\t1817016\t59.7\t1\t1753007 64009\n",
      "18\t2442218\t59.7\t1\t2355110 87108\n",
      "19\t2486262\t59.7\t1\t2405860 80402\n",
      "20\t2298798\t59.7\t1\t2212615 86183\n",
      "21\t2462281\t59.7\t1\t2376944 85337\n",
      "22\t2693256\t59.7\t1\t2591580 101676\n",
      "23\t2117201\t59.7\t1\t2038877 78324\n",
      "24\t2529069\t59.7\t1\t2441475 87594\n",
      "25\t2300578\t59.7\t1\t2226358 74220\n",
      "26\t2596745\t59.7\t1\t2494708 102037\n",
      "27\t2478522\t59.7\t1\t2396674 81848\n",
      "28\t2424775\t59.7\t1\t2344388 80387\n",
      "29\t2468561\t59.7\t1\t2392313 76248\n",
      "30\t2625645\t59.7\t1\t2538195 87450\n",
      "31\t2534062\t59.7\t1\t2448131 85931\n",
      "32\t2463975\t59.7\t1\t2389974 74001\n",
      "33\t2485864\t59.7\t1\t2403092 82772\n",
      "34\t2500312\t59.7\t1\t2422559 77753\n",
      "35\t2541533\t59.7\t1\t2457377 84156\n",
      "36\t2669327\t59.7\t1\t2588129 81198\n",
      "37\t2498773\t59.7\t1\t2414488 84285\n",
      "38\t2672991\t59.7\t1\t2588709 84282\n",
      "39\t2621845\t59.7\t1\t2539402 82443\n",
      "40\t2638822\t59.7\t1\t2557832 80990\n",
      "41\t2612895\t59.7\t1\t2531898 80997\n",
      "42\t2637743\t59.7\t1\t2557749 79994\n",
      "43\t2632806\t59.7\t1\t2553076 79730\n",
      "44\t2647914\t59.7\t1\t2566441 81473\n",
      "45\t2658677\t59.7\t1\t2578375 80302\n",
      "46\t2654614\t59.7\t1\t2572937 81677\n",
      "47\t2696968\t59.7\t1\t2615932 81036\n",
      "48\t2692360\t59.7\t1\t2612005 80355\n",
      "49\t2718504\t59.7\t1\t2638142 80362\n",
      "50\t2713258\t59.7\t1\t2634426 78832\n",
      "51\t2734828\t59.7\t1\t2655686 79142\n",
      "52\t2736621\t59.7\t1\t2659002 77619\n",
      "53\t2761328\t59.7\t1\t2683593 77735\n",
      "54\t2750340\t59.7\t1\t2673319 77021\n",
      "55\t2766771\t59.7\t1\t2689905 76866\n",
      "56\t2771458\t59.7\t1\t2695068 76390\n",
      "57\t2788435\t59.7\t1\t2711265 77170\n",
      "58\t2797577\t59.7\t1\t2720254 77323\n",
      "59\t2818435\t59.7\t1\t2739863 78572\n",
      "60\t2828457\t59.7\t1\t2747028 81429\n",
      "61\t2850147\t59.7\t1\t2769784 80363\n",
      "62\t2849460\t59.7\t1\t2769623 79837\n",
      "63\t2860655\t59.7\t1\t2781562 79093\n",
      "64\t2864719\t59.7\t1\t2785496 79223\n",
      "65\t2874706\t59.7\t1\t2796483 78223\n",
      "66\t2877715\t59.7\t1\t2799029 78686\n",
      "67\t2894761\t59.7\t1\t2815535 79226\n",
      "68\t2916632\t59.7\t1\t2836622 80010\n",
      "69\t2930076\t59.7\t1\t2851903 78173\n",
      "70\t2943178\t59.7\t1\t2864631 78547\n",
      "71\t2960700\t59.7\t1\t2881553 79147\n",
      "72\t2959836\t59.7\t1\t2882947 76889\n",
      "73\t2972005\t59.7\t1\t2895233 76772\n",
      "74\t2972897\t59.7\t1\t2896091 76806\n",
      "75\t2984527\t59.7\t1\t2907091 77436\n",
      "76\t2990507\t59.7\t1\t2913466 77041\n",
      "77\t3009208\t59.7\t1\t2932000 77208\n",
      "78\t3021368\t59.7\t1\t2944757 76611\n",
      "79\t3047385\t59.7\t1\t2970350 77035\n",
      "80\t3064830\t59.7\t1\t2987317 77513\n",
      "81\t3084135\t59.7\t1\t3007723 76412\n",
      "82\t3087131\t59.7\t1\t3009868 77263\n",
      "83\t3098954\t59.7\t1\t3022614 76340\n",
      "84\t3093141\t59.7\t1\t3020085 73056\n",
      "85\t3102656\t59.7\t1\t3029977 72679\n",
      "86\t3111762\t59.7\t1\t3038627 73135\n",
      "87\t3116552\t59.7\t1\t3045030 71522\n",
      "88\t3138573\t59.7\t1\t3066619 71954\n",
      "89\t3171353\t59.7\t1\t3099692 71661\n",
      "90\t3187992\t59.7\t1\t3117359 70633\n",
      "91\t3222803\t59.7\t1\t3151511 71292\n",
      "92\t3227079\t59.7\t1\t3156589 70490\n",
      "93\t3229980\t59.7\t1\t3160347 69633\n",
      "94\t3222379\t59.7\t1\t3154099 68280\n",
      "95\t3220966\t59.7\t1\t3153222 67744\n",
      "96\t3221695\t59.7\t1\t3155035 66660\n",
      "97\t3229784\t59.7\t1\t3163740 66044\n",
      "98\t3242698\t59.7\t1\t3177003 65695\n",
      "99\t3271604\t59.7\t1\t3206202 65402\n",
      "100\t3300569\t59.7\t1\t3234765 65804\n",
      "101\t3334886\t59.7\t1\t3269446 65440\n",
      "102\t3346942\t59.7\t1\t3281672 65270\n",
      "103\t3351934\t59.7\t1\t3287666 64268\n",
      "104\t3343578\t59.7\t1\t3279884 63694\n",
      "105\t3345122\t59.7\t1\t3281513 63609\n",
      "106\t3352183\t59.7\t1\t3288738 63445\n",
      "107\t3367566\t59.7\t1\t3304077 63489\n",
      "108\t3378354\t59.7\t1\t3315190 63164\n",
      "109\t3420686\t59.7\t1\t3357714 62972\n",
      "110\t3449826\t59.7\t1\t3386281 63545\n",
      "111\t3487365\t59.7\t1\t3423872 63493\n",
      "112\t3508368\t59.7\t1\t3445495 62873\n",
      "113\t3507180\t59.7\t1\t3445383 61797\n",
      "114\t3494159\t59.7\t1\t3432618 61541\n",
      "115\t3484932\t59.7\t1\t3424522 60410\n",
      "116\t3476413\t59.7\t1\t3416442 59971\n",
      "117\t3475301\t59.7\t1\t3415502 59799\n",
      "118\t3482407\t59.7\t1\t3423675 58732\n",
      "119\t3508843\t59.7\t1\t3449512 59331\n",
      "120\t3547443\t59.7\t1\t3488043 59400\n",
      "121\t3601402\t59.7\t1\t3542591 58811\n",
      "122\t3631504\t59.7\t1\t3572349 59155\n",
      "123\t3636265\t59.7\t1\t3577504 58761\n",
      "124\t3617049\t59.7\t1\t3558907 58142\n",
      "125\t3595969\t59.7\t1\t3538341 57628\n",
      "126\t3584447\t59.7\t1\t3528276 56171\n",
      "127\t3588428\t59.7\t1\t3534270 54158\n",
      "128\t3609387\t59.7\t1\t3557382 52005\n",
      "129\t3632653\t59.7\t1\t3582960 49693\n",
      "130\t3703568\t59.7\t1\t3655789 47779\n",
      "131\t3760631\t59.7\t1\t3714652 45979\n",
      "132\t3784172\t59.7\t1\t3741312 42860\n",
      "133\t3781183\t59.7\t1\t3741281 39902\n",
      "134\t3607068\t59.7\t1\t3570649 36419\n",
      "135\t3565429\t59.7\t1\t3532081 33348\n",
      "136\t3471060\t59.7\t1\t3439177 31883\n",
      "137\t3403980\t59.7\t1\t3374394 29586\n",
      "138\t3273300\t59.7\t1\t3244828 28472\n",
      "139\t3246174\t59.7\t1\t3218100 28074\n",
      "140\t3438033\t59.7\t1\t3408723 29310\n",
      "141\t3653904\t59.7\t1\t3624269 29635\n",
      "142\t3305687\t59.7\t1\t3277694 27993\n",
      "143\t3365594\t59.7\t1\t3337863 27731\n",
      "144\t3572185\t59.7\t1\t3543480 28705\n",
      "145\t3520268\t59.7\t1\t3492330 27938\n",
      "146\t3747209\t59.7\t1\t3716610 30599\n",
      "147\t3171755\t59.7\t1\t3143829 27926\n",
      "148\t3229065\t59.7\t1\t3199486 29579\n",
      "149\t2726822\t59.7\t1\t2698510 28312\n",
      "150\t2602798\t59.7\t1\t2568461 34337\n",
      "151\t715352\t59.7\t1\t625610 89742\n",
      "\n",
      "\n",
      "RUN STATISTICS FOR INPUT FILE: /home/sam/data/geoduck_illumina/NR014_AD014_S5_L002_R1_001.fastq.gz\n",
      "=============================================\n",
      "1001146597 sequences processed in total\n",
      "The length threshold of paired-end sequences gets evaluated later on (in the validation step)\n",
      "\n",
      "Writing report to '/home/sam/data/geoduck_illumina/trimmed/NR014_AD014_S5_L002_R2_001.fastq.gz_trimming_report.txt'\n",
      "\n",
      "SUMMARISING RUN PARAMETERS\n",
      "==========================\n",
      "Input filename: /home/sam/data/geoduck_illumina/NR014_AD014_S5_L002_R2_001.fastq.gz\n",
      "Trimming mode: paired-end\n",
      "Trim Galore version: 0.4.4\n",
      "Cutadapt version: 1.9.1\n",
      "Quality Phred score cutoff: 20\n",
      "Quality encoding type selected: ASCII+33\n",
      "Adapter sequence: 'CTGTCTCTTATA' (Nextera Transposase sequence; auto-detected)\n",
      "Maximum trimming error rate: 0.1 (default)\n",
      "Minimum required adapter overlap (stringency): 1 bp\n",
      "Minimum required sequence length for both reads before a sequence pair gets removed: 20 bp\n",
      "Output file(s) will be GZIP compressed\n",
      "\n",
      "Writing final adapter and quality trimmed output to NR014_AD014_S5_L002_R2_001_trimmed.fq.gz\n",
      "\n",
      "\n",
      "  >>> Now performing quality (cutoff 20) and adapter trimming in a single pass for the adapter sequence: 'CTGTCTCTTATA' from file /home/sam/data/geoduck_illumina/NR014_AD014_S5_L002_R2_001.fastq.gz <<< \n",
      "10000000 sequences processed\n",
      "20000000 sequences processed\n",
      "30000000 sequences processed\n",
      "40000000 sequences processed\n",
      "50000000 sequences processed\n",
      "60000000 sequences processed\n",
      "70000000 sequences processed\n",
      "80000000 sequences processed\n",
      "90000000 sequences processed\n",
      "100000000 sequences processed\n",
      "110000000 sequences processed\n",
      "120000000 sequences processed\n",
      "130000000 sequences processed\n",
      "140000000 sequences processed\n",
      "150000000 sequences processed\n",
      "160000000 sequences processed\n",
      "170000000 sequences processed\n",
      "180000000 sequences processed\n",
      "190000000 sequences processed\n",
      "200000000 sequences processed\n",
      "210000000 sequences processed\n",
      "220000000 sequences processed\n",
      "230000000 sequences processed\n",
      "240000000 sequences processed\n",
      "250000000 sequences processed\n",
      "260000000 sequences processed\n",
      "270000000 sequences processed\n",
      "280000000 sequences processed\n",
      "290000000 sequences processed\n",
      "300000000 sequences processed\n",
      "310000000 sequences processed\n",
      "320000000 sequences processed\n",
      "330000000 sequences processed\n",
      "340000000 sequences processed\n",
      "350000000 sequences processed\n",
      "360000000 sequences processed\n",
      "370000000 sequences processed\n",
      "380000000 sequences processed\n",
      "390000000 sequences processed\n",
      "400000000 sequences processed\n",
      "410000000 sequences processed\n",
      "420000000 sequences processed\n",
      "430000000 sequences processed\n",
      "440000000 sequences processed\n",
      "450000000 sequences processed\n",
      "460000000 sequences processed\n",
      "470000000 sequences processed\n",
      "480000000 sequences processed\n",
      "490000000 sequences processed\n",
      "500000000 sequences processed\n",
      "510000000 sequences processed\n",
      "520000000 sequences processed\n",
      "530000000 sequences processed\n",
      "540000000 sequences processed\n",
      "550000000 sequences processed\n",
      "560000000 sequences processed\n",
      "570000000 sequences processed\n",
      "580000000 sequences processed\n",
      "590000000 sequences processed\n",
      "600000000 sequences processed\n",
      "610000000 sequences processed\n",
      "620000000 sequences processed\n",
      "630000000 sequences processed\n",
      "640000000 sequences processed\n",
      "650000000 sequences processed\n",
      "660000000 sequences processed\n",
      "670000000 sequences processed\n",
      "680000000 sequences processed\n",
      "690000000 sequences processed\n",
      "700000000 sequences processed\n",
      "710000000 sequences processed\n",
      "720000000 sequences processed\n",
      "730000000 sequences processed\n",
      "740000000 sequences processed\n",
      "750000000 sequences processed\n",
      "760000000 sequences processed\n",
      "770000000 sequences processed\n",
      "780000000 sequences processed\n",
      "790000000 sequences processed\n",
      "800000000 sequences processed\n",
      "810000000 sequences processed\n",
      "820000000 sequences processed\n",
      "830000000 sequences processed\n",
      "840000000 sequences processed\n",
      "850000000 sequences processed\n",
      "860000000 sequences processed\n",
      "870000000 sequences processed\n",
      "880000000 sequences processed\n",
      "890000000 sequences processed\n",
      "900000000 sequences processed\n",
      "910000000 sequences processed\n",
      "920000000 sequences processed\n",
      "930000000 sequences processed\n",
      "940000000 sequences processed\n",
      "950000000 sequences processed\n",
      "960000000 sequences processed\n",
      "970000000 sequences processed\n",
      "980000000 sequences processed\n",
      "990000000 sequences processed\n",
      "1000000000 sequences processed\n",
      "This is cutadapt 1.9.1 with Python 2.7.12\n",
      "Command line parameters: -f fastq -e 0.1 -q 20 -O 1 -a CTGTCTCTTATA /home/sam/data/geoduck_illumina/NR014_AD014_S5_L002_R2_001.fastq.gz\n",
      "Trimming 1 adapter with at most 10.0% errors in single-end mode ...\n",
      "Finished in 23633.95 s (24 us/read; 2.54 M reads/minute).\n",
      "\n",
      "=== Summary ===\n",
      "\n",
      "Total reads processed:           1,001,146,597\n",
      "Reads with adapters:               540,115,126 (53.9%)\n",
      "Reads written (passing filters): 1,001,146,597 (100.0%)\n",
      "\n",
      "Total basepairs processed: 151,173,136,147 bp\n",
      "Quality-trimmed:             591,626,564 bp (0.4%)\n",
      "Total written (filtered):  118,805,814,263 bp (78.6%)\n",
      "\n",
      "=== Adapter 1 ===\n",
      "\n",
      "Sequence: CTGTCTCTTATA; Type: regular 3'; Length: 12; Trimmed: 540115126 times.\n",
      "\n",
      "No. of allowed errors:\n",
      "0-9 bp: 0; 10-12 bp: 1\n",
      "\n",
      "Bases preceding removed adapters:\n",
      "  A: 18.6%\n",
      "  C: 33.8%\n",
      "  G: 21.3%\n",
      "  T: 26.2%\n",
      "  none/other: 0.1%\n",
      "\n",
      "Overview of removed sequences\n",
      "length\tcount\texpect\tmax.err\terror counts\n",
      "1\t106085948\t250286649.2\t0\t106085948\n",
      "2\t30545591\t62571662.3\t0\t30545591\n",
      "3\t10550586\t15642915.6\t0\t10550586\n",
      "4\t4664851\t3910728.9\t0\t4664851\n",
      "5\t2730219\t977682.2\t0\t2730219\n",
      "6\t2212374\t244420.6\t0\t2212374\n",
      "7\t2182387\t61105.1\t0\t2182387\n",
      "8\t2342229\t15276.3\t0\t2342229\n",
      "9\t1941104\t3819.1\t0\t1920265 20839\n",
      "10\t2193387\t954.8\t1\t2050691 142696\n",
      "11\t2286146\t238.7\t1\t2148338 137808\n",
      "12\t2106924\t59.7\t1\t1977364 129560\n",
      "13\t2244772\t59.7\t1\t2109049 135723\n",
      "14\t2045749\t59.7\t1\t1931101 114648\n",
      "15\t2439558\t59.7\t1\t2281814 157744\n",
      "16\t2565286\t59.7\t1\t2417982 147304\n",
      "17\t1703369\t59.7\t1\t1613204 90165\n",
      "18\t2294530\t59.7\t1\t2165029 129501\n",
      "19\t2338286\t59.7\t1\t2222577 115709\n",
      "20\t2066198\t59.7\t1\t1958793 107405\n",
      "21\t2299627\t59.7\t1\t2166870 132757\n",
      "22\t2489525\t59.7\t1\t2352703 136822\n",
      "23\t1931407\t59.7\t1\t1829181 102226\n",
      "24\t2354926\t59.7\t1\t2222838 132088\n",
      "25\t2119895\t59.7\t1\t2018624 101271\n",
      "26\t2446505\t59.7\t1\t2295144 151361\n",
      "27\t2329182\t59.7\t1\t2209792 119390\n",
      "28\t2215355\t59.7\t1\t2106158 109197\n",
      "29\t2288397\t59.7\t1\t2177986 110411\n",
      "30\t2411852\t59.7\t1\t2291226 120626\n",
      "31\t2280500\t59.7\t1\t2173162 107338\n",
      "32\t2315968\t59.7\t1\t2197426 118542\n",
      "33\t2269381\t59.7\t1\t2163038 106343\n",
      "34\t2364664\t59.7\t1\t2241517 123147\n",
      "35\t2283233\t59.7\t1\t2174730 108503\n",
      "36\t2532452\t59.7\t1\t2413990 118462\n",
      "37\t2228870\t59.7\t1\t2127630 101240\n",
      "38\t2492963\t59.7\t1\t2372371 120592\n",
      "39\t2436639\t59.7\t1\t2319409 117230\n",
      "40\t2434599\t59.7\t1\t2325313 109286\n",
      "41\t2397752\t59.7\t1\t2288471 109281\n",
      "42\t2443026\t59.7\t1\t2334096 108930\n",
      "43\t2418021\t59.7\t1\t2313093 104928\n",
      "44\t2426060\t59.7\t1\t2318750 107310\n",
      "45\t2450430\t59.7\t1\t2343658 106772\n",
      "46\t2419646\t59.7\t1\t2314974 104672\n",
      "47\t2495511\t59.7\t1\t2386452 109059\n",
      "48\t2462435\t59.7\t1\t2359933 102502\n",
      "49\t2497544\t59.7\t1\t2390867 106677\n",
      "50\t2486318\t59.7\t1\t2382721 103597\n",
      "51\t2511781\t59.7\t1\t2406493 105288\n",
      "52\t2516552\t59.7\t1\t2413543 103009\n",
      "53\t2536624\t59.7\t1\t2433362 103262\n",
      "54\t2526890\t59.7\t1\t2424609 102281\n",
      "55\t2550150\t59.7\t1\t2447125 103025\n",
      "56\t2554177\t59.7\t1\t2450023 104154\n",
      "57\t2563613\t59.7\t1\t2459640 103973\n",
      "58\t2573415\t59.7\t1\t2470076 103339\n",
      "59\t2576096\t59.7\t1\t2472977 103119\n",
      "60\t2585802\t59.7\t1\t2483898 101904\n",
      "61\t2593413\t59.7\t1\t2492734 100679\n",
      "62\t2580047\t59.7\t1\t2480538 99509\n",
      "63\t2585329\t59.7\t1\t2484750 100579\n",
      "64\t2589823\t59.7\t1\t2489860 99963\n",
      "65\t2598352\t59.7\t1\t2497953 100399\n",
      "66\t2604715\t59.7\t1\t2506764 97951\n",
      "67\t2633277\t59.7\t1\t2534640 98637\n",
      "68\t2646868\t59.7\t1\t2550522 96346\n",
      "69\t2664167\t59.7\t1\t2568169 95998\n",
      "70\t2684490\t59.7\t1\t2589178 95312\n",
      "71\t2711492\t59.7\t1\t2615749 95743\n",
      "72\t2700448\t59.7\t1\t2606191 94257\n",
      "73\t2718039\t59.7\t1\t2623903 94136\n",
      "74\t2724997\t59.7\t1\t2631281 93716\n",
      "75\t2730017\t59.7\t1\t2636608 93409\n",
      "76\t2733638\t59.7\t1\t2640868 92770\n",
      "77\t2741293\t59.7\t1\t2648922 92371\n",
      "78\t2742165\t59.7\t1\t2649675 92490\n",
      "79\t2748939\t59.7\t1\t2656849 92090\n",
      "80\t2762857\t59.7\t1\t2672290 90567\n",
      "81\t2758435\t59.7\t1\t2667298 91137\n",
      "82\t2751060\t59.7\t1\t2661259 89801\n",
      "83\t2749612\t59.7\t1\t2660092 89520\n",
      "84\t2749708\t59.7\t1\t2661184 88524\n",
      "85\t2755076\t59.7\t1\t2667316 87760\n",
      "86\t2760271\t59.7\t1\t2673075 87196\n",
      "87\t2775417\t59.7\t1\t2688980 86437\n",
      "88\t2785855\t59.7\t1\t2700066 85789\n",
      "89\t2816661\t59.7\t1\t2730237 86424\n",
      "90\t2835228\t59.7\t1\t2748725 86503\n",
      "91\t2875858\t59.7\t1\t2789532 86326\n",
      "92\t2875642\t59.7\t1\t2789932 85710\n",
      "93\t2880444\t59.7\t1\t2794772 85672\n",
      "94\t2869719\t59.7\t1\t2785410 84309\n",
      "95\t2877087\t59.7\t1\t2793123 83964\n",
      "96\t2871155\t59.7\t1\t2788195 82960\n",
      "97\t2883832\t59.7\t1\t2800983 82849\n",
      "98\t2881604\t59.7\t1\t2799707 81897\n",
      "99\t2907764\t59.7\t1\t2825768 81996\n",
      "100\t2923615\t59.7\t1\t2842173 81442\n",
      "101\t2937759\t59.7\t1\t2857065 80694\n",
      "102\t2934449\t59.7\t1\t2855607 78842\n",
      "103\t2929161\t59.7\t1\t2851529 77632\n",
      "104\t2916318\t59.7\t1\t2839154 77164\n",
      "105\t2913070\t59.7\t1\t2836950 76120\n",
      "106\t2912618\t59.7\t1\t2837416 75202\n",
      "107\t2922490\t59.7\t1\t2847436 75054\n",
      "108\t2930126\t59.7\t1\t2855625 74501\n",
      "109\t2963369\t59.7\t1\t2888878 74491\n",
      "110\t2989674\t59.7\t1\t2915613 74061\n",
      "111\t3014074\t59.7\t1\t2939976 74098\n",
      "112\t3024627\t59.7\t1\t2950972 73655\n",
      "113\t3020183\t59.7\t1\t2946755 73428\n",
      "114\t2999238\t59.7\t1\t2926933 72305\n",
      "115\t2986885\t59.7\t1\t2914692 72193\n",
      "116\t2972850\t59.7\t1\t2902526 70324\n",
      "117\t2970432\t59.7\t1\t2900649 69783\n",
      "118\t2973151\t59.7\t1\t2904782 68369\n",
      "119\t2991872\t59.7\t1\t2923437 68435\n",
      "120\t3021460\t59.7\t1\t2953417 68043\n",
      "121\t3060676\t59.7\t1\t2992459 68217\n",
      "122\t3070719\t59.7\t1\t3003092 67627\n",
      "123\t3071679\t59.7\t1\t3004456 67223\n",
      "124\t3050603\t59.7\t1\t2984437 66166\n",
      "125\t3030367\t59.7\t1\t2965802 64565\n",
      "126\t3009064\t59.7\t1\t2945744 63320\n",
      "127\t3015770\t59.7\t1\t2953622 62148\n",
      "128\t3033521\t59.7\t1\t2972488 61033\n",
      "129\t3054558\t59.7\t1\t2993596 60962\n",
      "130\t3109911\t59.7\t1\t3049821 60090\n",
      "131\t3152250\t59.7\t1\t3092703 59547\n",
      "132\t3166216\t59.7\t1\t3108855 57361\n",
      "133\t3151380\t59.7\t1\t3096855 54525\n",
      "134\t3013295\t59.7\t1\t2961818 51477\n",
      "135\t2959648\t59.7\t1\t2910572 49076\n",
      "136\t2878218\t59.7\t1\t2831405 46813\n",
      "137\t2804192\t59.7\t1\t2759721 44471\n",
      "138\t2689723\t59.7\t1\t2646720 43003\n",
      "139\t2674202\t59.7\t1\t2631616 42586\n",
      "140\t2833434\t59.7\t1\t2789250 44184\n",
      "141\t2954295\t59.7\t1\t2909824 44471\n",
      "142\t2703519\t59.7\t1\t2661904 41615\n",
      "143\t2758010\t59.7\t1\t2716272 41738\n",
      "144\t2902059\t59.7\t1\t2858855 43204\n",
      "145\t2888745\t59.7\t1\t2846765 41980\n",
      "146\t2988648\t59.7\t1\t2945061 43587\n",
      "147\t2547900\t59.7\t1\t2508323 39577\n",
      "148\t2553590\t59.7\t1\t2515877 37713\n",
      "149\t2071172\t59.7\t1\t2038481 32691\n",
      "150\t1925273\t59.7\t1\t1888267 37006\n",
      "151\t532997\t59.7\t1\t452276 80721\n",
      "\n",
      "\n",
      "RUN STATISTICS FOR INPUT FILE: /home/sam/data/geoduck_illumina/NR014_AD014_S5_L002_R2_001.fastq.gz\n",
      "=============================================\n",
      "1001146597 sequences processed in total\n",
      "The length threshold of paired-end sequences gets evaluated later on (in the validation step)\n",
      "\n",
      "Validate paired-end files NR014_AD014_S5_L002_R1_001_trimmed.fq.gz and NR014_AD014_S5_L002_R2_001_trimmed.fq.gz\n",
      "file_1: NR014_AD014_S5_L002_R1_001_trimmed.fq.gz, file_2: NR014_AD014_S5_L002_R2_001_trimmed.fq.gz\n",
      "\n",
      "\n",
      ">>>>> Now validing the length of the 2 paired-end infiles: NR014_AD014_S5_L002_R1_001_trimmed.fq.gz and NR014_AD014_S5_L002_R2_001_trimmed.fq.gz <<<<<\n",
      "Writing validated paired-end read 1 reads to NR014_AD014_S5_L002_R1_001_val_1.fq.gz\n",
      "Writing validated paired-end read 2 reads to NR014_AD014_S5_L002_R2_001_val_2.fq.gz\n",
      "\n",
      "Total number of sequences analysed: 1001146597\n",
      "\n",
      "Number of sequence pairs removed because at least one read was shorter than the length cutoff (20 bp): 119117389 (11.90%)\n",
      "\n",
      "Deleting both intermediate output files NR014_AD014_S5_L002_R1_001_trimmed.fq.gz and NR014_AD014_S5_L002_R2_001_trimmed.fq.gz\n",
      "\n",
      "====================================================================================================\n",
      "\n",
      "Writing report to '/home/sam/data/geoduck_illumina/trimmed/NR015_AD015_S6_L001_R1_001.fastq.gz_trimming_report.txt'\n",
      "\n",
      "SUMMARISING RUN PARAMETERS\n",
      "==========================\n",
      "Input filename: /home/sam/data/geoduck_illumina/NR015_AD015_S6_L001_R1_001.fastq.gz\n",
      "Trimming mode: paired-end\n",
      "Trim Galore version: 0.4.4\n",
      "Cutadapt version: 1.9.1\n",
      "Quality Phred score cutoff: 20\n",
      "Quality encoding type selected: ASCII+33\n",
      "Adapter sequence: 'CTGTCTCTTATA' (Nextera Transposase sequence; auto-detected)\n",
      "Maximum trimming error rate: 0.1 (default)\n",
      "Minimum required adapter overlap (stringency): 1 bp\n",
      "Minimum required sequence length for both reads before a sequence pair gets removed: 20 bp\n",
      "Output file(s) will be GZIP compressed\n",
      "\n",
      "Writing final adapter and quality trimmed output to NR015_AD015_S6_L001_R1_001_trimmed.fq.gz\n",
      "\n",
      "\n",
      "  >>> Now performing quality (cutoff 20) and adapter trimming in a single pass for the adapter sequence: 'CTGTCTCTTATA' from file /home/sam/data/geoduck_illumina/NR015_AD015_S6_L001_R1_001.fastq.gz <<< \n",
      "10000000 sequences processed\n",
      "20000000 sequences processed\n",
      "30000000 sequences processed\n",
      "40000000 sequences processed\n",
      "50000000 sequences processed\n",
      "60000000 sequences processed\n",
      "70000000 sequences processed\n",
      "80000000 sequences processed\n",
      "90000000 sequences processed\n",
      "100000000 sequences processed\n",
      "110000000 sequences processed\n",
      "120000000 sequences processed\n",
      "130000000 sequences processed\n",
      "140000000 sequences processed\n",
      "150000000 sequences processed\n",
      "160000000 sequences processed\n",
      "170000000 sequences processed\n",
      "180000000 sequences processed\n",
      "190000000 sequences processed\n",
      "200000000 sequences processed\n",
      "210000000 sequences processed\n",
      "220000000 sequences processed\n",
      "230000000 sequences processed\n",
      "240000000 sequences processed\n",
      "250000000 sequences processed\n",
      "260000000 sequences processed\n",
      "270000000 sequences processed\n",
      "280000000 sequences processed\n",
      "This is cutadapt 1.9.1 with Python 2.7.12\n",
      "Command line parameters: -f fastq -e 0.1 -q 20 -O 1 -a CTGTCTCTTATA /home/sam/data/geoduck_illumina/NR015_AD015_S6_L001_R1_001.fastq.gz\n",
      "Trimming 1 adapter with at most 10.0% errors in single-end mode ...\n",
      "Finished in 6775.87 s (24 us/read; 2.53 M reads/minute).\n",
      "\n",
      "=== Summary ===\n",
      "\n",
      "Total reads processed:             285,533,287\n",
      "Reads with adapters:               169,989,862 (59.5%)\n",
      "Reads written (passing filters):   285,533,287 (100.0%)\n",
      "\n",
      "Total basepairs processed: 43,115,526,337 bp\n",
      "Quality-trimmed:             123,865,249 bp (0.3%)\n",
      "Total written (filtered):  32,523,865,323 bp (75.4%)\n",
      "\n",
      "=== Adapter 1 ===\n",
      "\n",
      "Sequence: CTGTCTCTTATA; Type: regular 3'; Length: 12; Trimmed: 169989862 times.\n",
      "\n",
      "No. of allowed errors:\n",
      "0-9 bp: 0; 10-12 bp: 1\n",
      "\n",
      "Bases preceding removed adapters:\n",
      "  A: 18.2%\n",
      "  C: 34.5%\n",
      "  G: 20.8%\n",
      "  T: 26.4%\n",
      "  none/other: 0.1%\n",
      "\n",
      "Overview of removed sequences\n",
      "length\tcount\texpect\tmax.err\terror counts\n",
      "1\t29792576\t71383321.8\t0\t29792576\n",
      "2\t8699292\t17845830.4\t0\t8699292\n",
      "3\t3016802\t4461457.6\t0\t3016802\n",
      "4\t1352424\t1115364.4\t0\t1352424\n",
      "5\t851917\t278841.1\t0\t851917\n",
      "6\t697513\t69710.3\t0\t697513\n",
      "7\t702213\t17427.6\t0\t702213\n",
      "8\t750289\t4356.9\t0\t750289\n",
      "9\t625043\t1089.2\t0\t618654 6389\n",
      "10\t710461\t272.3\t1\t665107 45354\n",
      "11\t725783\t68.1\t1\t686258 39525\n",
      "12\t700592\t17.0\t1\t656441 44151\n",
      "13\t712703\t17.0\t1\t672993 39710\n",
      "14\t670584\t17.0\t1\t631337 39247\n",
      "15\t779929\t17.0\t1\t732943 46986\n",
      "16\t867361\t17.0\t1\t815669 51692\n",
      "17\t530622\t17.0\t1\t501640 28982\n",
      "18\t734140\t17.0\t1\t694490 39650\n",
      "19\t744592\t17.0\t1\t707856 36736\n",
      "20\t696416\t17.0\t1\t656510 39906\n",
      "21\t728429\t17.0\t1\t692715 35714\n",
      "22\t808117\t17.0\t1\t763635 44482\n",
      "23\t647819\t17.0\t1\t612348 35471\n",
      "24\t752409\t17.0\t1\t715569 36840\n",
      "25\t683072\t17.0\t1\t649761 33311\n",
      "26\t774130\t17.0\t1\t732740 41390\n",
      "27\t742658\t17.0\t1\t707216 35442\n",
      "28\t716799\t17.0\t1\t681752 35047\n",
      "29\t736131\t17.0\t1\t703087 33044\n",
      "30\t796547\t17.0\t1\t758064 38483\n",
      "31\t767477\t17.0\t1\t728669 38808\n",
      "32\t719213\t17.0\t1\t687765 31448\n",
      "33\t748944\t17.0\t1\t711909 37035\n",
      "34\t731523\t17.0\t1\t699463 32060\n",
      "35\t757281\t17.0\t1\t720415 36866\n",
      "36\t802097\t17.0\t1\t767134 34963\n",
      "37\t740131\t17.0\t1\t702594 37537\n",
      "38\t796013\t17.0\t1\t760533 35480\n",
      "39\t776981\t17.0\t1\t741938 35043\n",
      "40\t783824\t17.0\t1\t748737 35087\n",
      "41\t775402\t17.0\t1\t739845 35557\n",
      "42\t785913\t17.0\t1\t751294 34619\n",
      "43\t782239\t17.0\t1\t747571 34668\n",
      "44\t786684\t17.0\t1\t751988 34696\n",
      "45\t788745\t17.0\t1\t754358 34387\n",
      "46\t788331\t17.0\t1\t753040 35291\n",
      "47\t798012\t17.0\t1\t763567 34445\n",
      "48\t797453\t17.0\t1\t762868 34585\n",
      "49\t803305\t17.0\t1\t768798 34507\n",
      "50\t799982\t17.0\t1\t766197 33785\n",
      "51\t807707\t17.0\t1\t773712 33995\n",
      "52\t807355\t17.0\t1\t774426 32929\n",
      "53\t815077\t17.0\t1\t781495 33582\n",
      "54\t812082\t17.0\t1\t778651 33431\n",
      "55\t816296\t17.0\t1\t783180 33116\n",
      "56\t817488\t17.0\t1\t784641 32847\n",
      "57\t822263\t17.0\t1\t789225 33038\n",
      "58\t825639\t17.0\t1\t792905 32734\n",
      "59\t828745\t17.0\t1\t796040 32705\n",
      "60\t832287\t17.0\t1\t799618 32669\n",
      "61\t837921\t17.0\t1\t805204 32717\n",
      "62\t838305\t17.0\t1\t806051 32254\n",
      "63\t838900\t17.0\t1\t806855 32045\n",
      "64\t842185\t17.0\t1\t810266 31919\n",
      "65\t844056\t17.0\t1\t811848 32208\n",
      "66\t844356\t17.0\t1\t812113 32243\n",
      "67\t852383\t17.0\t1\t820642 31741\n",
      "68\t859922\t17.0\t1\t827933 31989\n",
      "69\t868210\t17.0\t1\t836120 32090\n",
      "70\t880306\t17.0\t1\t848423 31883\n",
      "71\t897985\t17.0\t1\t865875 32110\n",
      "72\t897151\t17.0\t1\t865344 31807\n",
      "73\t904164\t17.0\t1\t872200 31964\n",
      "74\t903736\t17.0\t1\t871795 31941\n",
      "75\t897498\t17.0\t1\t866324 31174\n",
      "76\t899288\t17.0\t1\t868223 31065\n",
      "77\t894824\t17.0\t1\t863925 30899\n",
      "78\t890599\t17.0\t1\t859591 31008\n",
      "79\t893370\t17.0\t1\t862677 30693\n",
      "80\t893795\t17.0\t1\t863349 30446\n",
      "81\t892345\t17.0\t1\t861685 30660\n",
      "82\t892131\t17.0\t1\t861405 30726\n",
      "83\t893349\t17.0\t1\t863083 30266\n",
      "84\t892497\t17.0\t1\t862117 30380\n",
      "85\t894637\t17.0\t1\t864073 30564\n",
      "86\t897308\t17.0\t1\t867437 29871\n",
      "87\t906289\t17.0\t1\t876604 29685\n",
      "88\t912437\t17.0\t1\t882447 29990\n",
      "89\t922804\t17.0\t1\t892965 29839\n",
      "90\t928225\t17.0\t1\t898763 29462\n",
      "91\t937247\t17.0\t1\t907461 29786\n",
      "92\t935078\t17.0\t1\t905771 29307\n",
      "93\t934982\t17.0\t1\t905742 29240\n",
      "94\t934844\t17.0\t1\t905790 29054\n",
      "95\t933821\t17.0\t1\t905633 28188\n",
      "96\t934157\t17.0\t1\t906041 28116\n",
      "97\t938472\t17.0\t1\t910612 27860\n",
      "98\t940550\t17.0\t1\t912752 27798\n",
      "99\t950446\t17.0\t1\t922495 27951\n",
      "100\t956684\t17.0\t1\t929074 27610\n",
      "101\t967262\t17.0\t1\t938911 28351\n",
      "102\t967365\t17.0\t1\t939406 27959\n",
      "103\t970501\t17.0\t1\t943032 27469\n",
      "104\t964843\t17.0\t1\t937746 27097\n",
      "105\t968357\t17.0\t1\t940882 27475\n",
      "106\t966090\t17.0\t1\t937744 28346\n",
      "107\t969058\t17.0\t1\t940982 28076\n",
      "108\t972508\t17.0\t1\t944372 28136\n",
      "109\t981904\t17.0\t1\t953344 28560\n",
      "110\t990718\t17.0\t1\t962403 28315\n",
      "111\t999198\t17.0\t1\t971471 27727\n",
      "112\t999684\t17.0\t1\t971418 28266\n",
      "113\t1002254\t17.0\t1\t974645 27609\n",
      "114\t995071\t17.0\t1\t967646 27425\n",
      "115\t991423\t17.0\t1\t964462 26961\n",
      "116\t987577\t17.0\t1\t961122 26455\n",
      "117\t986621\t17.0\t1\t960218 26403\n",
      "118\t988469\t17.0\t1\t963274 25195\n",
      "119\t996353\t17.0\t1\t971324 25029\n",
      "120\t1006116\t17.0\t1\t981048 25068\n",
      "121\t1019944\t17.0\t1\t994255 25689\n",
      "122\t1024438\t17.0\t1\t999173 25265\n",
      "123\t1024449\t17.0\t1\t998959 25490\n",
      "124\t1015450\t17.0\t1\t990480 24970\n",
      "125\t1011230\t17.0\t1\t986267 24963\n",
      "126\t1004233\t17.0\t1\t979906 24327\n",
      "127\t1006617\t17.0\t1\t982982 23635\n",
      "128\t1014744\t17.0\t1\t992132 22612\n",
      "129\t1019618\t17.0\t1\t997952 21666\n",
      "130\t1037145\t17.0\t1\t1016488 20657\n",
      "131\t1048705\t17.0\t1\t1028160 20545\n",
      "132\t1052244\t17.0\t1\t1033111 19133\n",
      "133\t1045541\t17.0\t1\t1027553 17988\n",
      "134\t993870\t17.0\t1\t977559 16311\n",
      "135\t972700\t17.0\t1\t958081 14619\n",
      "136\t947762\t17.0\t1\t933670 14092\n",
      "137\t933281\t17.0\t1\t920281 13000\n",
      "138\t896741\t17.0\t1\t884207 12534\n",
      "139\t884462\t17.0\t1\t871784 12678\n",
      "140\t934731\t17.0\t1\t921566 13165\n",
      "141\t987796\t17.0\t1\t974351 13445\n",
      "142\t895519\t17.0\t1\t882888 12631\n",
      "143\t907679\t17.0\t1\t894977 12702\n",
      "144\t954227\t17.0\t1\t940736 13491\n",
      "145\t934549\t17.0\t1\t921308 13241\n",
      "146\t984637\t17.0\t1\t970166 14471\n",
      "147\t863134\t17.0\t1\t849593 13541\n",
      "148\t877344\t17.0\t1\t863127 14217\n",
      "149\t754291\t17.0\t1\t740309 13982\n",
      "150\t703851\t17.0\t1\t688503 15348\n",
      "151\t198449\t17.0\t1\t173946 24503\n",
      "\n",
      "\n",
      "RUN STATISTICS FOR INPUT FILE: /home/sam/data/geoduck_illumina/NR015_AD015_S6_L001_R1_001.fastq.gz\n",
      "=============================================\n",
      "285533287 sequences processed in total\n",
      "The length threshold of paired-end sequences gets evaluated later on (in the validation step)\n",
      "\n",
      "Writing report to '/home/sam/data/geoduck_illumina/trimmed/NR015_AD015_S6_L001_R2_001.fastq.gz_trimming_report.txt'\n",
      "\n",
      "SUMMARISING RUN PARAMETERS\n",
      "==========================\n",
      "Input filename: /home/sam/data/geoduck_illumina/NR015_AD015_S6_L001_R2_001.fastq.gz\n",
      "Trimming mode: paired-end\n",
      "Trim Galore version: 0.4.4\n",
      "Cutadapt version: 1.9.1\n",
      "Quality Phred score cutoff: 20\n",
      "Quality encoding type selected: ASCII+33\n",
      "Adapter sequence: 'CTGTCTCTTATA' (Nextera Transposase sequence; auto-detected)\n",
      "Maximum trimming error rate: 0.1 (default)\n",
      "Minimum required adapter overlap (stringency): 1 bp\n",
      "Minimum required sequence length for both reads before a sequence pair gets removed: 20 bp\n",
      "Output file(s) will be GZIP compressed\n",
      "\n",
      "Writing final adapter and quality trimmed output to NR015_AD015_S6_L001_R2_001_trimmed.fq.gz\n",
      "\n",
      "\n",
      "  >>> Now performing quality (cutoff 20) and adapter trimming in a single pass for the adapter sequence: 'CTGTCTCTTATA' from file /home/sam/data/geoduck_illumina/NR015_AD015_S6_L001_R2_001.fastq.gz <<< \n",
      "10000000 sequences processed\n",
      "20000000 sequences processed\n",
      "30000000 sequences processed\n",
      "40000000 sequences processed\n",
      "50000000 sequences processed\n",
      "60000000 sequences processed\n",
      "70000000 sequences processed\n",
      "80000000 sequences processed\n",
      "90000000 sequences processed\n",
      "100000000 sequences processed\n",
      "110000000 sequences processed\n",
      "120000000 sequences processed\n",
      "130000000 sequences processed\n",
      "140000000 sequences processed\n",
      "150000000 sequences processed\n",
      "160000000 sequences processed\n",
      "170000000 sequences processed\n",
      "180000000 sequences processed\n",
      "190000000 sequences processed\n",
      "200000000 sequences processed\n",
      "210000000 sequences processed\n",
      "220000000 sequences processed\n",
      "230000000 sequences processed\n",
      "240000000 sequences processed\n",
      "250000000 sequences processed\n",
      "260000000 sequences processed\n",
      "270000000 sequences processed\n",
      "280000000 sequences processed\n",
      "This is cutadapt 1.9.1 with Python 2.7.12\n",
      "Command line parameters: -f fastq -e 0.1 -q 20 -O 1 -a CTGTCTCTTATA /home/sam/data/geoduck_illumina/NR015_AD015_S6_L001_R2_001.fastq.gz\n",
      "Trimming 1 adapter with at most 10.0% errors in single-end mode ...\n",
      "Finished in 7088.25 s (25 us/read; 2.42 M reads/minute).\n",
      "\n",
      "=== Summary ===\n",
      "\n",
      "Total reads processed:             285,533,287\n",
      "Reads with adapters:               157,006,761 (55.0%)\n",
      "Reads written (passing filters):   285,533,287 (100.0%)\n",
      "\n",
      "Total basepairs processed: 43,115,526,337 bp\n",
      "Quality-trimmed:             157,457,656 bp (0.4%)\n",
      "Total written (filtered):  33,761,528,513 bp (78.3%)\n",
      "\n",
      "=== Adapter 1 ===\n",
      "\n",
      "Sequence: CTGTCTCTTATA; Type: regular 3'; Length: 12; Trimmed: 157006761 times.\n",
      "\n",
      "No. of allowed errors:\n",
      "0-9 bp: 0; 10-12 bp: 1\n",
      "\n",
      "Bases preceding removed adapters:\n",
      "  A: 18.8%\n",
      "  C: 33.4%\n",
      "  G: 21.1%\n",
      "  T: 26.6%\n",
      "  none/other: 0.1%\n",
      "\n",
      "Overview of removed sequences\n",
      "length\tcount\texpect\tmax.err\terror counts\n",
      "1\t30892934\t71383321.8\t0\t30892934\n",
      "2\t8842653\t17845830.4\t0\t8842653\n",
      "3\t3036633\t4461457.6\t0\t3036633\n",
      "4\t1352553\t1115364.4\t0\t1352553\n",
      "5\t804188\t278841.1\t0\t804188\n",
      "6\t627405\t69710.3\t0\t627405\n",
      "7\t637710\t17427.6\t0\t637710\n",
      "8\t679417\t4356.9\t0\t679417\n",
      "9\t561170\t1089.2\t0\t553764 7406\n",
      "10\t640071\t272.3\t1\t570312 69759\n",
      "11\t684928\t68.1\t1\t614828 70100\n",
      "12\t610934\t17.0\t1\t544462 66472\n",
      "13\t672166\t17.0\t1\t598467 73699\n",
      "14\t580697\t17.0\t1\t520155 60542\n",
      "15\t751250\t17.0\t1\t667039 84211\n",
      "16\t749114\t17.0\t1\t672504 76610\n",
      "17\t488364\t17.0\t1\t439322 49042\n",
      "18\t681948\t17.0\t1\t612562 69386\n",
      "19\t697953\t17.0\t1\t632882 65071\n",
      "20\t595572\t17.0\t1\t537606 57966\n",
      "21\t687290\t17.0\t1\t620331 66959\n",
      "22\t733249\t17.0\t1\t662668 70581\n",
      "23\t560944\t17.0\t1\t505838 55106\n",
      "24\t695222\t17.0\t1\t627539 67683\n",
      "25\t612818\t17.0\t1\t557862 54956\n",
      "26\t723686\t17.0\t1\t649351 74335\n",
      "27\t694756\t17.0\t1\t630424 64332\n",
      "28\t640582\t17.0\t1\t582597 57985\n",
      "29\t677477\t17.0\t1\t617223 60254\n",
      "30\t725168\t17.0\t1\t661387 63781\n",
      "31\t654432\t17.0\t1\t597087 57345\n",
      "32\t681756\t17.0\t1\t621111 60645\n",
      "33\t662584\t17.0\t1\t606062 56522\n",
      "34\t692497\t17.0\t1\t632868 59629\n",
      "35\t660621\t17.0\t1\t604658 55963\n",
      "36\t769018\t17.0\t1\t706367 62651\n",
      "37\t634494\t17.0\t1\t581405 53089\n",
      "38\t733662\t17.0\t1\t673564 60098\n",
      "39\t720096\t17.0\t1\t661670 58426\n",
      "40\t714845\t17.0\t1\t660111 54734\n",
      "41\t706121\t17.0\t1\t651362 54759\n",
      "42\t722037\t17.0\t1\t667239 54798\n",
      "43\t711500\t17.0\t1\t658400 53100\n",
      "44\t712712\t17.0\t1\t658774 53938\n",
      "45\t719585\t17.0\t1\t665912 53673\n",
      "46\t708603\t17.0\t1\t656187 52416\n",
      "47\t738664\t17.0\t1\t683959 54705\n",
      "48\t722752\t17.0\t1\t670419 52333\n",
      "49\t732813\t17.0\t1\t680151 52662\n",
      "50\t730494\t17.0\t1\t679130 51364\n",
      "51\t737398\t17.0\t1\t684817 52581\n",
      "52\t738925\t17.0\t1\t687162 51763\n",
      "53\t743924\t17.0\t1\t691508 52416\n",
      "54\t739480\t17.0\t1\t688344 51136\n",
      "55\t745511\t17.0\t1\t693777 51734\n",
      "56\t746772\t17.0\t1\t695270 51502\n",
      "57\t748412\t17.0\t1\t697011 51401\n",
      "58\t753350\t17.0\t1\t701736 51614\n",
      "59\t755770\t17.0\t1\t704763 51007\n",
      "60\t756077\t17.0\t1\t705375 50702\n",
      "61\t760081\t17.0\t1\t709780 50301\n",
      "62\t757352\t17.0\t1\t708067 49285\n",
      "63\t759930\t17.0\t1\t711067 48863\n",
      "64\t760333\t17.0\t1\t712053 48280\n",
      "65\t760289\t17.0\t1\t711908 48381\n",
      "66\t763657\t17.0\t1\t715196 48461\n",
      "67\t769671\t17.0\t1\t721880 47791\n",
      "68\t772301\t17.0\t1\t725793 46508\n",
      "69\t777565\t17.0\t1\t731201 46364\n",
      "70\t783273\t17.0\t1\t737515 45758\n",
      "71\t787444\t17.0\t1\t741603 45841\n",
      "72\t788184\t17.0\t1\t742881 45303\n",
      "73\t789988\t17.0\t1\t744238 45750\n",
      "74\t791860\t17.0\t1\t746876 44984\n",
      "75\t792960\t17.0\t1\t748415 44545\n",
      "76\t793783\t17.0\t1\t748863 44920\n",
      "77\t795542\t17.0\t1\t750482 45060\n",
      "78\t799402\t17.0\t1\t754179 45223\n",
      "79\t802797\t17.0\t1\t757870 44927\n",
      "80\t805167\t17.0\t1\t760666 44501\n",
      "81\t805524\t17.0\t1\t760846 44678\n",
      "82\t804535\t17.0\t1\t760571 43964\n",
      "83\t804385\t17.0\t1\t760840 43545\n",
      "84\t803261\t17.0\t1\t760237 43024\n",
      "85\t804198\t17.0\t1\t761656 42542\n",
      "86\t804501\t17.0\t1\t762184 42317\n",
      "87\t807766\t17.0\t1\t765356 42410\n",
      "88\t811546\t17.0\t1\t770568 40978\n",
      "89\t820574\t17.0\t1\t779074 41500\n",
      "90\t825523\t17.0\t1\t784488 41035\n",
      "91\t833927\t17.0\t1\t792409 41518\n",
      "92\t832224\t17.0\t1\t791378 40846\n",
      "93\t833467\t17.0\t1\t792752 40715\n",
      "94\t831332\t17.0\t1\t791581 39751\n",
      "95\t832656\t17.0\t1\t793259 39397\n",
      "96\t831820\t17.0\t1\t792267 39553\n",
      "97\t833384\t17.0\t1\t794027 39357\n",
      "98\t835416\t17.0\t1\t796598 38818\n",
      "99\t843521\t17.0\t1\t804367 39154\n",
      "100\t848434\t17.0\t1\t809375 39059\n",
      "101\t852409\t17.0\t1\t813970 38439\n",
      "102\t849342\t17.0\t1\t811821 37521\n",
      "103\t848756\t17.0\t1\t811258 37498\n",
      "104\t843868\t17.0\t1\t806569 37299\n",
      "105\t844315\t17.0\t1\t807043 37272\n",
      "106\t844170\t17.0\t1\t807095 37075\n",
      "107\t846389\t17.0\t1\t809347 37042\n",
      "108\t849244\t17.0\t1\t812412 36832\n",
      "109\t861743\t17.0\t1\t824734 37009\n",
      "110\t866612\t17.0\t1\t830068 36544\n",
      "111\t874546\t17.0\t1\t837664 36882\n",
      "112\t876108\t17.0\t1\t839587 36521\n",
      "113\t875074\t17.0\t1\t838844 36230\n",
      "114\t869212\t17.0\t1\t832999 36213\n",
      "115\t863094\t17.0\t1\t827066 36028\n",
      "116\t859690\t17.0\t1\t824407 35283\n",
      "117\t862067\t17.0\t1\t827437 34630\n",
      "118\t860516\t17.0\t1\t826211 34305\n",
      "119\t867871\t17.0\t1\t833540 34331\n",
      "120\t876210\t17.0\t1\t842324 33886\n",
      "121\t886066\t17.0\t1\t850955 35111\n",
      "122\t886757\t17.0\t1\t851867 34890\n",
      "123\t886389\t17.0\t1\t852753 33636\n",
      "124\t878836\t17.0\t1\t845123 33713\n",
      "125\t874740\t17.0\t1\t841597 33143\n",
      "126\t870295\t17.0\t1\t838143 32152\n",
      "127\t874572\t17.0\t1\t842967 31605\n",
      "128\t874859\t17.0\t1\t843781 31078\n",
      "129\t882170\t17.0\t1\t851081 31089\n",
      "130\t897136\t17.0\t1\t866112 31024\n",
      "131\t907722\t17.0\t1\t876284 31438\n",
      "132\t909859\t17.0\t1\t879421 30438\n",
      "133\t898530\t17.0\t1\t869510 29020\n",
      "134\t860975\t17.0\t1\t833133 27842\n",
      "135\t840928\t17.0\t1\t814542 26386\n",
      "136\t815616\t17.0\t1\t789219 26397\n",
      "137\t796809\t17.0\t1\t771242 25567\n",
      "138\t766737\t17.0\t1\t741069 25668\n",
      "139\t758657\t17.0\t1\t733464 25193\n",
      "140\t802927\t17.0\t1\t776681 26246\n",
      "141\t833980\t17.0\t1\t807836 26144\n",
      "142\t768328\t17.0\t1\t743356 24972\n",
      "143\t781148\t17.0\t1\t756777 24371\n",
      "144\t823246\t17.0\t1\t797509 25737\n",
      "145\t819014\t17.0\t1\t794209 24805\n",
      "146\t840558\t17.0\t1\t815823 24735\n",
      "147\t740025\t17.0\t1\t718959 21066\n",
      "148\t736002\t17.0\t1\t715942 20060\n",
      "149\t602840\t17.0\t1\t585965 16875\n",
      "150\t551937\t17.0\t1\t534926 17011\n",
      "151\t154537\t17.0\t1\t131184 23353\n",
      "\n",
      "\n",
      "RUN STATISTICS FOR INPUT FILE: /home/sam/data/geoduck_illumina/NR015_AD015_S6_L001_R2_001.fastq.gz\n",
      "=============================================\n",
      "285533287 sequences processed in total\n",
      "The length threshold of paired-end sequences gets evaluated later on (in the validation step)\n",
      "\n",
      "Validate paired-end files NR015_AD015_S6_L001_R1_001_trimmed.fq.gz and NR015_AD015_S6_L001_R2_001_trimmed.fq.gz\n",
      "file_1: NR015_AD015_S6_L001_R1_001_trimmed.fq.gz, file_2: NR015_AD015_S6_L001_R2_001_trimmed.fq.gz\n",
      "\n",
      "\n",
      ">>>>> Now validing the length of the 2 paired-end infiles: NR015_AD015_S6_L001_R1_001_trimmed.fq.gz and NR015_AD015_S6_L001_R2_001_trimmed.fq.gz <<<<<\n",
      "Writing validated paired-end read 1 reads to NR015_AD015_S6_L001_R1_001_val_1.fq.gz\n",
      "Writing validated paired-end read 2 reads to NR015_AD015_S6_L001_R2_001_val_2.fq.gz\n",
      "\n",
      "Total number of sequences analysed: 285533287\n",
      "\n",
      "Number of sequence pairs removed because at least one read was shorter than the length cutoff (20 bp): 33572503 (11.76%)\n",
      "\n",
      "Deleting both intermediate output files NR015_AD015_S6_L001_R1_001_trimmed.fq.gz and NR015_AD015_S6_L001_R2_001_trimmed.fq.gz\n",
      "\n",
      "====================================================================================================\n",
      "\n",
      "Writing report to '/home/sam/data/geoduck_illumina/trimmed/NR015_AD015_S6_L002_R1_001.fastq.gz_trimming_report.txt'\n",
      "\n",
      "SUMMARISING RUN PARAMETERS\n",
      "==========================\n",
      "Input filename: /home/sam/data/geoduck_illumina/NR015_AD015_S6_L002_R1_001.fastq.gz\n",
      "Trimming mode: paired-end\n",
      "Trim Galore version: 0.4.4\n",
      "Cutadapt version: 1.9.1\n",
      "Quality Phred score cutoff: 20\n",
      "Quality encoding type selected: ASCII+33\n",
      "Adapter sequence: 'CTGTCTCTTATA' (Nextera Transposase sequence; auto-detected)\n",
      "Maximum trimming error rate: 0.1 (default)\n",
      "Minimum required adapter overlap (stringency): 1 bp\n",
      "Minimum required sequence length for both reads before a sequence pair gets removed: 20 bp\n",
      "Output file(s) will be GZIP compressed\n",
      "\n",
      "Writing final adapter and quality trimmed output to NR015_AD015_S6_L002_R1_001_trimmed.fq.gz\n",
      "\n",
      "\n",
      "  >>> Now performing quality (cutoff 20) and adapter trimming in a single pass for the adapter sequence: 'CTGTCTCTTATA' from file /home/sam/data/geoduck_illumina/NR015_AD015_S6_L002_R1_001.fastq.gz <<< \n",
      "10000000 sequences processed\n",
      "20000000 sequences processed\n",
      "30000000 sequences processed\n",
      "40000000 sequences processed\n",
      "50000000 sequences processed\n",
      "60000000 sequences processed\n",
      "70000000 sequences processed\n",
      "80000000 sequences processed\n",
      "90000000 sequences processed\n",
      "100000000 sequences processed\n",
      "110000000 sequences processed\n",
      "120000000 sequences processed\n",
      "130000000 sequences processed\n",
      "140000000 sequences processed\n",
      "150000000 sequences processed\n",
      "160000000 sequences processed\n",
      "170000000 sequences processed\n",
      "180000000 sequences processed\n",
      "190000000 sequences processed\n",
      "200000000 sequences processed\n",
      "210000000 sequences processed\n",
      "220000000 sequences processed\n",
      "230000000 sequences processed\n",
      "240000000 sequences processed\n",
      "250000000 sequences processed\n",
      "260000000 sequences processed\n",
      "270000000 sequences processed\n",
      "280000000 sequences processed\n",
      "290000000 sequences processed\n",
      "300000000 sequences processed\n",
      "310000000 sequences processed\n",
      "320000000 sequences processed\n",
      "This is cutadapt 1.9.1 with Python 2.7.12\n",
      "Command line parameters: -f fastq -e 0.1 -q 20 -O 1 -a CTGTCTCTTATA /home/sam/data/geoduck_illumina/NR015_AD015_S6_L002_R1_001.fastq.gz\n",
      "Trimming 1 adapter with at most 10.0% errors in single-end mode ...\n",
      "Finished in 7582.12 s (24 us/read; 2.54 M reads/minute).\n",
      "\n",
      "=== Summary ===\n",
      "\n",
      "Total reads processed:             321,498,091\n",
      "Reads with adapters:               184,340,232 (57.3%)\n",
      "Reads written (passing filters):   321,498,091 (100.0%)\n",
      "\n",
      "Total basepairs processed: 48,546,211,741 bp\n",
      "Quality-trimmed:             137,293,336 bp (0.3%)\n",
      "Total written (filtered):  37,358,729,319 bp (77.0%)\n",
      "\n",
      "=== Adapter 1 ===\n",
      "\n",
      "Sequence: CTGTCTCTTATA; Type: regular 3'; Length: 12; Trimmed: 184340232 times.\n",
      "\n",
      "No. of allowed errors:\n",
      "0-9 bp: 0; 10-12 bp: 1\n",
      "\n",
      "Bases preceding removed adapters:\n",
      "  A: 18.4%\n",
      "  C: 34.2%\n",
      "  G: 20.8%\n",
      "  T: 26.4%\n",
      "  none/other: 0.1%\n",
      "\n",
      "Overview of removed sequences\n",
      "length\tcount\texpect\tmax.err\terror counts\n",
      "1\t34790084\t80374522.8\t0\t34790084\n",
      "2\t10150781\t20093630.7\t0\t10150781\n",
      "3\t3455870\t5023407.7\t0\t3455870\n",
      "4\t1550491\t1255851.9\t0\t1550491\n",
      "5\t927725\t313963.0\t0\t927725\n",
      "6\t770794\t78490.7\t0\t770794\n",
      "7\t754425\t19622.7\t0\t754425\n",
      "8\t797832\t4905.7\t0\t797832\n",
      "9\t691341\t1226.4\t0\t684878 6463\n",
      "10\t758798\t306.6\t1\t725268 33530\n",
      "11\t778225\t76.7\t1\t750138 28087\n",
      "12\t748512\t19.2\t1\t718530 29982\n",
      "13\t765827\t19.2\t1\t738763 27064\n",
      "14\t726721\t19.2\t1\t700221 26500\n",
      "15\t815459\t19.2\t1\t783705 31754\n",
      "16\t915524\t19.2\t1\t880213 35311\n",
      "17\t587101\t19.2\t1\t567780 19321\n",
      "18\t781893\t19.2\t1\t755981 25912\n",
      "19\t792509\t19.2\t1\t769002 23507\n",
      "20\t748975\t19.2\t1\t722729 26246\n",
      "21\t782165\t19.2\t1\t757504 24661\n",
      "22\t852780\t19.2\t1\t822725 30055\n",
      "23\t691805\t19.2\t1\t668119 23686\n",
      "24\t805558\t19.2\t1\t779970 25588\n",
      "25\t736868\t19.2\t1\t714887 21981\n",
      "26\t822137\t19.2\t1\t792880 29257\n",
      "27\t789184\t19.2\t1\t764765 24419\n",
      "28\t772352\t19.2\t1\t748395 23957\n",
      "29\t784069\t19.2\t1\t761850 22219\n",
      "30\t833009\t19.2\t1\t807070 25939\n",
      "31\t815747\t19.2\t1\t789965 25782\n",
      "32\t776719\t19.2\t1\t755157 21562\n",
      "33\t789052\t19.2\t1\t764505 24547\n",
      "34\t789460\t19.2\t1\t766973 22487\n",
      "35\t805768\t19.2\t1\t780985 24783\n",
      "36\t842970\t19.2\t1\t819287 23683\n",
      "37\t792247\t19.2\t1\t766918 25329\n",
      "38\t843554\t19.2\t1\t819153 24401\n",
      "39\t824158\t19.2\t1\t800485 23673\n",
      "40\t831040\t19.2\t1\t807572 23468\n",
      "41\t822263\t19.2\t1\t798414 23849\n",
      "42\t831352\t19.2\t1\t808055 23297\n",
      "43\t826580\t19.2\t1\t803164 23416\n",
      "44\t833919\t19.2\t1\t810469 23450\n",
      "45\t835277\t19.2\t1\t811727 23550\n",
      "46\t832389\t19.2\t1\t808710 23679\n",
      "47\t842958\t19.2\t1\t819635 23323\n",
      "48\t842970\t19.2\t1\t819646 23324\n",
      "49\t850579\t19.2\t1\t827435 23144\n",
      "50\t847871\t19.2\t1\t825067 22804\n",
      "51\t852432\t19.2\t1\t829531 22901\n",
      "52\t854212\t19.2\t1\t832043 22169\n",
      "53\t860708\t19.2\t1\t837881 22827\n",
      "54\t856600\t19.2\t1\t834068 22532\n",
      "55\t863103\t19.2\t1\t840716 22387\n",
      "56\t864778\t19.2\t1\t842629 22149\n",
      "57\t867713\t19.2\t1\t845061 22652\n",
      "58\t870823\t19.2\t1\t848482 22341\n",
      "59\t876055\t19.2\t1\t853340 22715\n",
      "60\t876721\t19.2\t1\t853334 23387\n",
      "61\t882393\t19.2\t1\t858988 23405\n",
      "62\t882089\t19.2\t1\t859480 22609\n",
      "63\t884924\t19.2\t1\t862269 22655\n",
      "64\t885825\t19.2\t1\t862985 22840\n",
      "65\t887968\t19.2\t1\t865206 22762\n",
      "66\t890524\t19.2\t1\t867695 22829\n",
      "67\t898610\t19.2\t1\t875470 23140\n",
      "68\t906335\t19.2\t1\t883040 23295\n",
      "69\t913024\t19.2\t1\t889966 23058\n",
      "70\t927209\t19.2\t1\t904215 22994\n",
      "71\t945175\t19.2\t1\t922060 23115\n",
      "72\t943361\t19.2\t1\t920516 22845\n",
      "73\t946895\t19.2\t1\t924212 22683\n",
      "74\t947759\t19.2\t1\t925234 22525\n",
      "75\t942196\t19.2\t1\t919565 22631\n",
      "76\t943335\t19.2\t1\t921008 22327\n",
      "77\t937755\t19.2\t1\t915504 22251\n",
      "78\t934875\t19.2\t1\t912853 22022\n",
      "79\t938131\t19.2\t1\t915986 22145\n",
      "80\t936987\t19.2\t1\t914620 22367\n",
      "81\t938329\t19.2\t1\t916438 21891\n",
      "82\t935782\t19.2\t1\t913692 22090\n",
      "83\t938343\t19.2\t1\t916585 21758\n",
      "84\t935073\t19.2\t1\t914475 20598\n",
      "85\t940035\t19.2\t1\t919229 20806\n",
      "86\t939880\t19.2\t1\t919261 20619\n",
      "87\t951772\t19.2\t1\t930974 20798\n",
      "88\t960541\t19.2\t1\t939850 20691\n",
      "89\t969723\t19.2\t1\t949268 20455\n",
      "90\t977212\t19.2\t1\t956989 20223\n",
      "91\t984873\t19.2\t1\t964532 20341\n",
      "92\t984415\t19.2\t1\t964253 20162\n",
      "93\t984826\t19.2\t1\t964857 19969\n",
      "94\t983629\t19.2\t1\t963927 19702\n",
      "95\t983271\t19.2\t1\t964083 19188\n",
      "96\t984818\t19.2\t1\t965656 19162\n",
      "97\t987744\t19.2\t1\t968976 18768\n",
      "98\t991585\t19.2\t1\t972448 19137\n",
      "99\t998782\t19.2\t1\t980063 18719\n",
      "100\t1006929\t19.2\t1\t988098 18831\n",
      "101\t1016236\t19.2\t1\t997457 18779\n",
      "102\t1015347\t19.2\t1\t996839 18508\n",
      "103\t1015941\t19.2\t1\t997618 18323\n",
      "104\t1016174\t19.2\t1\t998126 18048\n",
      "105\t1016223\t19.2\t1\t998200 18023\n",
      "106\t1015283\t19.2\t1\t997204 18079\n",
      "107\t1020440\t19.2\t1\t1002559 17881\n",
      "108\t1022949\t19.2\t1\t1004987 17962\n",
      "109\t1033100\t19.2\t1\t1014919 18181\n",
      "110\t1040647\t19.2\t1\t1022755 17892\n",
      "111\t1049931\t19.2\t1\t1031961 17970\n",
      "112\t1052388\t19.2\t1\t1034485 17903\n",
      "113\t1053965\t19.2\t1\t1036714 17251\n",
      "114\t1045382\t19.2\t1\t1028126 17256\n",
      "115\t1042613\t19.2\t1\t1025509 17104\n",
      "116\t1039537\t19.2\t1\t1022507 17030\n",
      "117\t1042085\t19.2\t1\t1024968 17117\n",
      "118\t1043803\t19.2\t1\t1027074 16729\n",
      "119\t1048392\t19.2\t1\t1031467 16925\n",
      "120\t1061976\t19.2\t1\t1044853 17123\n",
      "121\t1073203\t19.2\t1\t1056386 16817\n",
      "122\t1080326\t19.2\t1\t1063446 16880\n",
      "123\t1081603\t19.2\t1\t1064743 16860\n",
      "124\t1072872\t19.2\t1\t1056325 16547\n",
      "125\t1067458\t19.2\t1\t1051113 16345\n",
      "126\t1059117\t19.2\t1\t1043275 15842\n",
      "127\t1061594\t19.2\t1\t1045828 15766\n",
      "128\t1066420\t19.2\t1\t1051515 14905\n",
      "129\t1075741\t19.2\t1\t1061136 14605\n",
      "130\t1089148\t19.2\t1\t1075527 13621\n",
      "131\t1102866\t19.2\t1\t1089562 13304\n",
      "132\t1104828\t19.2\t1\t1092300 12528\n",
      "133\t1096746\t19.2\t1\t1085225 11521\n",
      "134\t1045190\t19.2\t1\t1034410 10780\n",
      "135\t1025762\t19.2\t1\t1015987 9775\n",
      "136\t997219\t19.2\t1\t987974 9245\n",
      "137\t981054\t19.2\t1\t972263 8791\n",
      "138\t942063\t19.2\t1\t933525 8538\n",
      "139\t926620\t19.2\t1\t918413 8207\n",
      "140\t980179\t19.2\t1\t971584 8595\n",
      "141\t1041953\t19.2\t1\t1033445 8508\n",
      "142\t945000\t19.2\t1\t936861 8139\n",
      "143\t959076\t19.2\t1\t950894 8182\n",
      "144\t1014659\t19.2\t1\t1006353 8306\n",
      "145\t994434\t19.2\t1\t986281 8153\n",
      "146\t1056592\t19.2\t1\t1047615 8977\n",
      "147\t923683\t19.2\t1\t915307 8376\n",
      "148\t942012\t19.2\t1\t933183 8829\n",
      "149\t808997\t19.2\t1\t800351 8646\n",
      "150\t765587\t19.2\t1\t755202 10385\n",
      "151\t214054\t19.2\t1\t188464 25590\n",
      "\n",
      "\n",
      "RUN STATISTICS FOR INPUT FILE: /home/sam/data/geoduck_illumina/NR015_AD015_S6_L002_R1_001.fastq.gz\n",
      "=============================================\n",
      "321498091 sequences processed in total\n",
      "The length threshold of paired-end sequences gets evaluated later on (in the validation step)\n",
      "\n",
      "Writing report to '/home/sam/data/geoduck_illumina/trimmed/NR015_AD015_S6_L002_R2_001.fastq.gz_trimming_report.txt'\n",
      "\n",
      "SUMMARISING RUN PARAMETERS\n",
      "==========================\n",
      "Input filename: /home/sam/data/geoduck_illumina/NR015_AD015_S6_L002_R2_001.fastq.gz\n",
      "Trimming mode: paired-end\n",
      "Trim Galore version: 0.4.4\n",
      "Cutadapt version: 1.9.1\n",
      "Quality Phred score cutoff: 20\n",
      "Quality encoding type selected: ASCII+33\n",
      "Adapter sequence: 'CTGTCTCTTATA' (Nextera Transposase sequence; auto-detected)\n",
      "Maximum trimming error rate: 0.1 (default)\n",
      "Minimum required adapter overlap (stringency): 1 bp\n",
      "Minimum required sequence length for both reads before a sequence pair gets removed: 20 bp\n",
      "Output file(s) will be GZIP compressed\n",
      "\n",
      "Writing final adapter and quality trimmed output to NR015_AD015_S6_L002_R2_001_trimmed.fq.gz\n",
      "\n",
      "\n",
      "  >>> Now performing quality (cutoff 20) and adapter trimming in a single pass for the adapter sequence: 'CTGTCTCTTATA' from file /home/sam/data/geoduck_illumina/NR015_AD015_S6_L002_R2_001.fastq.gz <<< \n",
      "10000000 sequences processed\n",
      "20000000 sequences processed\n",
      "30000000 sequences processed\n",
      "40000000 sequences processed\n",
      "50000000 sequences processed\n",
      "60000000 sequences processed\n",
      "70000000 sequences processed\n",
      "80000000 sequences processed\n",
      "90000000 sequences processed\n",
      "100000000 sequences processed\n",
      "110000000 sequences processed\n",
      "120000000 sequences processed\n",
      "130000000 sequences processed\n",
      "140000000 sequences processed\n",
      "150000000 sequences processed\n",
      "160000000 sequences processed\n",
      "170000000 sequences processed\n",
      "180000000 sequences processed\n",
      "190000000 sequences processed\n",
      "200000000 sequences processed\n",
      "210000000 sequences processed\n",
      "220000000 sequences processed\n",
      "230000000 sequences processed\n",
      "240000000 sequences processed\n",
      "250000000 sequences processed\n",
      "260000000 sequences processed\n",
      "270000000 sequences processed\n",
      "280000000 sequences processed\n",
      "290000000 sequences processed\n",
      "300000000 sequences processed\n",
      "310000000 sequences processed\n",
      "320000000 sequences processed\n",
      "This is cutadapt 1.9.1 with Python 2.7.12\n",
      "Command line parameters: -f fastq -e 0.1 -q 20 -O 1 -a CTGTCTCTTATA /home/sam/data/geoduck_illumina/NR015_AD015_S6_L002_R2_001.fastq.gz\n",
      "Trimming 1 adapter with at most 10.0% errors in single-end mode ...\n",
      "Finished in 7687.78 s (24 us/read; 2.51 M reads/minute).\n",
      "\n",
      "=== Summary ===\n",
      "\n",
      "Total reads processed:             321,498,091\n",
      "Reads with adapters:               170,243,880 (53.0%)\n",
      "Reads written (passing filters):   321,498,091 (100.0%)\n",
      "\n",
      "Total basepairs processed: 48,546,211,741 bp\n",
      "Quality-trimmed:             172,654,465 bp (0.4%)\n",
      "Total written (filtered):  38,758,510,378 bp (79.8%)\n",
      "\n",
      "=== Adapter 1 ===\n",
      "\n",
      "Sequence: CTGTCTCTTATA; Type: regular 3'; Length: 12; Trimmed: 170243880 times.\n",
      "\n",
      "No. of allowed errors:\n",
      "0-9 bp: 0; 10-12 bp: 1\n",
      "\n",
      "Bases preceding removed adapters:\n",
      "  A: 19.0%\n",
      "  C: 33.2%\n",
      "  G: 21.1%\n",
      "  T: 26.5%\n",
      "  none/other: 0.1%\n",
      "\n",
      "Overview of removed sequences\n",
      "length\tcount\texpect\tmax.err\terror counts\n",
      "1\t35821180\t80374522.8\t0\t35821180\n",
      "2\t10345651\t20093630.7\t0\t10345651\n",
      "3\t3515062\t5023407.7\t0\t3515062\n",
      "4\t1546659\t1255851.9\t0\t1546659\n",
      "5\t892628\t313963.0\t0\t892628\n",
      "6\t713202\t78490.7\t0\t713202\n",
      "7\t707245\t19622.7\t0\t707245\n",
      "8\t750011\t4905.7\t0\t750011\n",
      "9\t632023\t1226.4\t0\t624891 7132\n",
      "10\t701720\t306.6\t1\t651564 50156\n",
      "11\t744375\t76.7\t1\t694721 49654\n",
      "12\t667399\t19.2\t1\t622330 45069\n",
      "13\t732507\t19.2\t1\t681942 50565\n",
      "14\t645906\t19.2\t1\t605175 40731\n",
      "15\t800653\t19.2\t1\t741958 58695\n",
      "16\t814227\t19.2\t1\t762292 51935\n",
      "17\t536472\t19.2\t1\t503877 32595\n",
      "18\t739124\t19.2\t1\t691178 47946\n",
      "19\t755959\t19.2\t1\t713097 42862\n",
      "20\t645905\t19.2\t1\t608312 37593\n",
      "21\t747401\t19.2\t1\t698218 49183\n",
      "22\t798672\t19.2\t1\t749309 49363\n",
      "23\t599746\t19.2\t1\t563392 36354\n",
      "24\t762430\t19.2\t1\t713620 48810\n",
      "25\t663609\t19.2\t1\t627610 35999\n",
      "26\t785312\t19.2\t1\t730745 54567\n",
      "27\t750778\t19.2\t1\t706726 44052\n",
      "28\t696938\t19.2\t1\t657507 39431\n",
      "29\t727087\t19.2\t1\t686935 40152\n",
      "30\t770953\t19.2\t1\t727314 43639\n",
      "31\t702681\t19.2\t1\t664825 37856\n",
      "32\t745054\t19.2\t1\t701250 43804\n",
      "33\t707859\t19.2\t1\t670315 37544\n",
      "34\t753535\t19.2\t1\t709190 44345\n",
      "35\t716629\t19.2\t1\t678511 38118\n",
      "36\t814526\t19.2\t1\t770511 44015\n",
      "37\t685832\t19.2\t1\t650690 35142\n",
      "38\t786295\t19.2\t1\t743333 42962\n",
      "39\t770800\t19.2\t1\t728209 42591\n",
      "40\t764857\t19.2\t1\t725735 39122\n",
      "41\t753964\t19.2\t1\t714760 39204\n",
      "42\t768953\t19.2\t1\t729810 39143\n",
      "43\t758528\t19.2\t1\t720874 37654\n",
      "44\t759904\t19.2\t1\t721620 38284\n",
      "45\t768252\t19.2\t1\t729692 38560\n",
      "46\t755252\t19.2\t1\t717957 37295\n",
      "47\t782949\t19.2\t1\t743701 39248\n",
      "48\t769278\t19.2\t1\t732606 36672\n",
      "49\t780986\t19.2\t1\t742971 38015\n",
      "50\t776196\t19.2\t1\t739070 37126\n",
      "51\t783018\t19.2\t1\t745627 37391\n",
      "52\t784834\t19.2\t1\t747752 37082\n",
      "53\t789080\t19.2\t1\t752048 37032\n",
      "54\t785314\t19.2\t1\t749220 36094\n",
      "55\t791960\t19.2\t1\t755134 36826\n",
      "56\t793510\t19.2\t1\t756398 37112\n",
      "57\t794491\t19.2\t1\t757625 36866\n",
      "58\t797753\t19.2\t1\t760954 36799\n",
      "59\t801924\t19.2\t1\t765216 36708\n",
      "60\t800898\t19.2\t1\t764777 36121\n",
      "61\t803365\t19.2\t1\t767413 35952\n",
      "62\t800610\t19.2\t1\t764766 35844\n",
      "63\t802117\t19.2\t1\t766527 35590\n",
      "64\t804521\t19.2\t1\t768700 35821\n",
      "65\t803126\t19.2\t1\t767444 35682\n",
      "66\t806034\t19.2\t1\t770918 35116\n",
      "67\t811835\t19.2\t1\t776898 34937\n",
      "68\t814695\t19.2\t1\t780709 33986\n",
      "69\t820961\t19.2\t1\t786981 33980\n",
      "70\t824379\t19.2\t1\t790486 33893\n",
      "71\t830072\t19.2\t1\t796500 33572\n",
      "72\t828652\t19.2\t1\t795351 33301\n",
      "73\t829785\t19.2\t1\t796449 33336\n",
      "74\t832733\t19.2\t1\t799938 32795\n",
      "75\t834898\t19.2\t1\t801697 33201\n",
      "76\t835677\t19.2\t1\t802839 32838\n",
      "77\t838829\t19.2\t1\t805994 32835\n",
      "78\t839114\t19.2\t1\t806367 32747\n",
      "79\t842308\t19.2\t1\t809848 32460\n",
      "80\t844389\t19.2\t1\t811913 32476\n",
      "81\t842812\t19.2\t1\t810541 32271\n",
      "82\t841902\t19.2\t1\t809792 32110\n",
      "83\t839941\t19.2\t1\t808430 31511\n",
      "84\t841472\t19.2\t1\t810158 31314\n",
      "85\t842893\t19.2\t1\t811724 31169\n",
      "86\t841634\t19.2\t1\t810482 31152\n",
      "87\t847141\t19.2\t1\t816486 30655\n",
      "88\t847988\t19.2\t1\t817691 30297\n",
      "89\t856838\t19.2\t1\t826335 30503\n",
      "90\t861767\t19.2\t1\t831031 30736\n",
      "91\t870130\t19.2\t1\t839396 30734\n",
      "92\t870519\t19.2\t1\t840098 30421\n",
      "93\t872720\t19.2\t1\t842532 30188\n",
      "94\t867199\t19.2\t1\t837299 29900\n",
      "95\t870431\t19.2\t1\t840294 30137\n",
      "96\t866119\t19.2\t1\t836550 29569\n",
      "97\t871655\t19.2\t1\t842197 29458\n",
      "98\t871196\t19.2\t1\t841756 29440\n",
      "99\t879050\t19.2\t1\t849774 29276\n",
      "100\t882429\t19.2\t1\t852996 29433\n",
      "101\t886488\t19.2\t1\t857813 28675\n",
      "102\t885709\t19.2\t1\t857702 28007\n",
      "103\t883041\t19.2\t1\t855308 27733\n",
      "104\t880615\t19.2\t1\t853348 27267\n",
      "105\t878156\t19.2\t1\t851201 26955\n",
      "106\t876813\t19.2\t1\t849997 26816\n",
      "107\t879870\t19.2\t1\t853114 26756\n",
      "108\t882817\t19.2\t1\t856362 26455\n",
      "109\t893851\t19.2\t1\t866929 26922\n",
      "110\t900868\t19.2\t1\t873583 27285\n",
      "111\t906883\t19.2\t1\t880097 26786\n",
      "112\t908725\t19.2\t1\t882462 26263\n",
      "113\t908170\t19.2\t1\t881831 26339\n",
      "114\t900204\t19.2\t1\t874110 26094\n",
      "115\t896826\t19.2\t1\t870861 25965\n",
      "116\t892341\t19.2\t1\t867266 25075\n",
      "117\t891351\t19.2\t1\t866163 25188\n",
      "118\t893358\t19.2\t1\t868532 24826\n",
      "119\t899627\t19.2\t1\t874913 24714\n",
      "120\t907557\t19.2\t1\t882938 24619\n",
      "121\t917517\t19.2\t1\t892645 24872\n",
      "122\t919984\t19.2\t1\t895291 24693\n",
      "123\t919027\t19.2\t1\t894358 24669\n",
      "124\t912068\t19.2\t1\t887701 24367\n",
      "125\t905685\t19.2\t1\t882228 23457\n",
      "126\t899775\t19.2\t1\t876705 23070\n",
      "127\t905322\t19.2\t1\t882419 22903\n",
      "128\t907214\t19.2\t1\t884900 22314\n",
      "129\t914551\t19.2\t1\t892583 21968\n",
      "130\t931316\t19.2\t1\t908862 22454\n",
      "131\t939671\t19.2\t1\t917402 22269\n",
      "132\t942016\t19.2\t1\t920419 21597\n",
      "133\t932673\t19.2\t1\t912159 20514\n",
      "134\t886972\t19.2\t1\t867900 19072\n",
      "135\t869322\t19.2\t1\t850867 18455\n",
      "136\t845098\t19.2\t1\t827564 17534\n",
      "137\t826006\t19.2\t1\t809330 16676\n",
      "138\t792349\t19.2\t1\t776217 16132\n",
      "139\t784638\t19.2\t1\t768588 16050\n",
      "140\t828483\t19.2\t1\t812017 16466\n",
      "141\t862129\t19.2\t1\t845560 16569\n",
      "142\t793137\t19.2\t1\t777344 15793\n",
      "143\t806747\t19.2\t1\t791137 15610\n",
      "144\t846238\t19.2\t1\t830018 16220\n",
      "145\t844949\t19.2\t1\t828902 16047\n",
      "146\t874514\t19.2\t1\t858356 16158\n",
      "147\t765041\t19.2\t1\t750640 14401\n",
      "148\t762653\t19.2\t1\t748529 14124\n",
      "149\t624227\t19.2\t1\t612024 12203\n",
      "150\t578109\t19.2\t1\t564908 13201\n",
      "151\t163297\t19.2\t1\t139818 23479\n",
      "\n",
      "\n",
      "RUN STATISTICS FOR INPUT FILE: /home/sam/data/geoduck_illumina/NR015_AD015_S6_L002_R2_001.fastq.gz\n",
      "=============================================\n",
      "321498091 sequences processed in total\n",
      "The length threshold of paired-end sequences gets evaluated later on (in the validation step)\n",
      "\n",
      "Validate paired-end files NR015_AD015_S6_L002_R1_001_trimmed.fq.gz and NR015_AD015_S6_L002_R2_001_trimmed.fq.gz\n",
      "file_1: NR015_AD015_S6_L002_R1_001_trimmed.fq.gz, file_2: NR015_AD015_S6_L002_R2_001_trimmed.fq.gz\n",
      "\n",
      "\n",
      ">>>>> Now validing the length of the 2 paired-end infiles: NR015_AD015_S6_L002_R1_001_trimmed.fq.gz and NR015_AD015_S6_L002_R2_001_trimmed.fq.gz <<<<<\n",
      "Writing validated paired-end read 1 reads to NR015_AD015_S6_L002_R1_001_val_1.fq.gz\n",
      "Writing validated paired-end read 2 reads to NR015_AD015_S6_L002_R2_001_val_2.fq.gz\n",
      "\n",
      "Total number of sequences analysed: 321498091\n",
      "\n",
      "Number of sequence pairs removed because at least one read was shorter than the length cutoff (20 bp): 35152577 (10.93%)\n",
      "\n",
      "Deleting both intermediate output files NR015_AD015_S6_L002_R1_001_trimmed.fq.gz and NR015_AD015_S6_L002_R2_001_trimmed.fq.gz\n",
      "\n",
      "====================================================================================================\n",
      "\n",
      "Writing report to '/home/sam/data/geoduck_illumina/trimmed/NR019_S7_L001_R1_001.fastq.gz_trimming_report.txt'\n",
      "\n",
      "SUMMARISING RUN PARAMETERS\n",
      "==========================\n",
      "Input filename: /home/sam/data/geoduck_illumina/NR019_S7_L001_R1_001.fastq.gz\n",
      "Trimming mode: paired-end\n",
      "Trim Galore version: 0.4.4\n",
      "Cutadapt version: 1.9.1\n",
      "Quality Phred score cutoff: 20\n",
      "Quality encoding type selected: ASCII+33\n",
      "Adapter sequence: 'CTGTCTCTTATA' (Nextera Transposase sequence; auto-detected)\n",
      "Maximum trimming error rate: 0.1 (default)\n",
      "Minimum required adapter overlap (stringency): 1 bp\n",
      "Minimum required sequence length for both reads before a sequence pair gets removed: 20 bp\n",
      "Output file(s) will be GZIP compressed\n",
      "\n",
      "Writing final adapter and quality trimmed output to NR019_S7_L001_R1_001_trimmed.fq.gz\n",
      "\n",
      "\n",
      "  >>> Now performing quality (cutoff 20) and adapter trimming in a single pass for the adapter sequence: 'CTGTCTCTTATA' from file /home/sam/data/geoduck_illumina/NR019_S7_L001_R1_001.fastq.gz <<< \n",
      "This is cutadapt 1.9.1 with Python 2.7.12\n",
      "Command line parameters: -f fastq -e 0.1 -q 20 -O 1 -a CTGTCTCTTATA /home/sam/data/geoduck_illumina/NR019_S7_L001_R1_001.fastq.gz\n",
      "Trimming 1 adapter with at most 10.0% errors in single-end mode ...\n",
      "Finished in 1.16 s (26 us/read; 2.28 M reads/minute).\n",
      "\n",
      "=== Summary ===\n",
      "\n",
      "Total reads processed:                  44,316\n",
      "Reads with adapters:                    18,312 (41.3%)\n",
      "Reads written (passing filters):        44,316 (100.0%)\n",
      "\n",
      "Total basepairs processed:     6,691,716 bp\n",
      "Quality-trimmed:                  48,533 bp (0.7%)\n",
      "Total written (filtered):      5,823,503 bp (87.0%)\n",
      "\n",
      "=== Adapter 1 ===\n",
      "\n",
      "Sequence: CTGTCTCTTATA; Type: regular 3'; Length: 12; Trimmed: 18312 times.\n",
      "\n",
      "No. of allowed errors:\n",
      "0-9 bp: 0; 10-12 bp: 1\n",
      "\n",
      "Bases preceding removed adapters:\n",
      "  A: 28.3%\n",
      "  C: 27.4%\n",
      "  G: 16.4%\n",
      "  T: 27.8%\n",
      "  none/other: 0.1%\n",
      "\n",
      "Overview of removed sequences\n",
      "length\tcount\texpect\tmax.err\terror counts\n",
      "1\t5941\t11079.0\t0\t5941\n",
      "2\t1800\t2769.8\t0\t1800\n",
      "3\t410\t692.4\t0\t410\n",
      "4\t164\t173.1\t0\t164\n",
      "5\t87\t43.3\t0\t87\n",
      "6\t54\t10.8\t0\t54\n",
      "7\t64\t2.7\t0\t64\n",
      "8\t55\t0.7\t0\t55\n",
      "9\t42\t0.2\t0\t39 3\n",
      "10\t44\t0.0\t1\t32 12\n",
      "11\t73\t0.0\t1\t44 29\n",
      "12\t68\t0.0\t1\t39 29\n",
      "13\t45\t0.0\t1\t30 15\n",
      "14\t53\t0.0\t1\t33 20\n",
      "15\t60\t0.0\t1\t33 27\n",
      "16\t81\t0.0\t1\t50 31\n",
      "17\t43\t0.0\t1\t27 16\n",
      "18\t60\t0.0\t1\t28 32\n",
      "19\t65\t0.0\t1\t37 28\n",
      "20\t49\t0.0\t1\t29 20\n",
      "21\t40\t0.0\t1\t28 12\n",
      "22\t60\t0.0\t1\t36 24\n",
      "23\t52\t0.0\t1\t29 23\n",
      "24\t72\t0.0\t1\t47 25\n",
      "25\t51\t0.0\t1\t27 24\n",
      "26\t63\t0.0\t1\t41 22\n",
      "27\t72\t0.0\t1\t46 26\n",
      "28\t62\t0.0\t1\t36 26\n",
      "29\t52\t0.0\t1\t35 17\n",
      "30\t48\t0.0\t1\t35 13\n",
      "31\t54\t0.0\t1\t36 18\n",
      "32\t63\t0.0\t1\t40 23\n",
      "33\t73\t0.0\t1\t43 30\n",
      "34\t59\t0.0\t1\t47 12\n",
      "35\t63\t0.0\t1\t40 23\n",
      "36\t63\t0.0\t1\t46 17\n",
      "37\t75\t0.0\t1\t46 29\n",
      "38\t53\t0.0\t1\t32 21\n",
      "39\t70\t0.0\t1\t52 18\n",
      "40\t61\t0.0\t1\t34 27\n",
      "41\t67\t0.0\t1\t43 24\n",
      "42\t57\t0.0\t1\t37 20\n",
      "43\t66\t0.0\t1\t42 24\n",
      "44\t64\t0.0\t1\t38 26\n",
      "45\t59\t0.0\t1\t36 23\n",
      "46\t76\t0.0\t1\t45 31\n",
      "47\t56\t0.0\t1\t47 9\n",
      "48\t61\t0.0\t1\t41 20\n",
      "49\t66\t0.0\t1\t42 24\n",
      "50\t69\t0.0\t1\t39 30\n",
      "51\t50\t0.0\t1\t32 18\n",
      "52\t64\t0.0\t1\t41 23\n",
      "53\t59\t0.0\t1\t39 20\n",
      "54\t73\t0.0\t1\t49 24\n",
      "55\t71\t0.0\t1\t47 24\n",
      "56\t69\t0.0\t1\t40 29\n",
      "57\t65\t0.0\t1\t38 27\n",
      "58\t62\t0.0\t1\t40 22\n",
      "59\t62\t0.0\t1\t31 31\n",
      "60\t64\t0.0\t1\t42 22\n",
      "61\t62\t0.0\t1\t38 24\n",
      "62\t64\t0.0\t1\t43 21\n",
      "63\t61\t0.0\t1\t41 20\n",
      "64\t45\t0.0\t1\t34 11\n",
      "65\t68\t0.0\t1\t48 20\n",
      "66\t76\t0.0\t1\t51 25\n",
      "67\t71\t0.0\t1\t41 30\n",
      "68\t64\t0.0\t1\t43 21\n",
      "69\t54\t0.0\t1\t37 17\n",
      "70\t70\t0.0\t1\t47 23\n",
      "71\t69\t0.0\t1\t45 24\n",
      "72\t73\t0.0\t1\t54 19\n",
      "73\t62\t0.0\t1\t39 23\n",
      "74\t69\t0.0\t1\t45 24\n",
      "75\t63\t0.0\t1\t39 24\n",
      "76\t81\t0.0\t1\t52 29\n",
      "77\t72\t0.0\t1\t44 28\n",
      "78\t58\t0.0\t1\t37 21\n",
      "79\t85\t0.0\t1\t54 31\n",
      "80\t65\t0.0\t1\t43 22\n",
      "81\t73\t0.0\t1\t53 20\n",
      "82\t73\t0.0\t1\t52 21\n",
      "83\t87\t0.0\t1\t64 23\n",
      "84\t67\t0.0\t1\t44 23\n",
      "85\t65\t0.0\t1\t43 22\n",
      "86\t61\t0.0\t1\t42 19\n",
      "87\t82\t0.0\t1\t65 17\n",
      "88\t93\t0.0\t1\t58 35\n",
      "89\t78\t0.0\t1\t55 23\n",
      "90\t74\t0.0\t1\t49 25\n",
      "91\t76\t0.0\t1\t55 21\n",
      "92\t80\t0.0\t1\t51 29\n",
      "93\t69\t0.0\t1\t43 26\n",
      "94\t69\t0.0\t1\t48 21\n",
      "95\t67\t0.0\t1\t48 19\n",
      "96\t76\t0.0\t1\t48 28\n",
      "97\t60\t0.0\t1\t42 18\n",
      "98\t72\t0.0\t1\t44 28\n",
      "99\t82\t0.0\t1\t62 20\n",
      "100\t85\t0.0\t1\t58 27\n",
      "101\t85\t0.0\t1\t58 27\n",
      "102\t78\t0.0\t1\t47 31\n",
      "103\t81\t0.0\t1\t57 24\n",
      "104\t66\t0.0\t1\t41 25\n",
      "105\t86\t0.0\t1\t64 22\n",
      "106\t77\t0.0\t1\t50 27\n",
      "107\t78\t0.0\t1\t52 26\n",
      "108\t96\t0.0\t1\t65 31\n",
      "109\t84\t0.0\t1\t61 23\n",
      "110\t87\t0.0\t1\t66 21\n",
      "111\t76\t0.0\t1\t52 24\n",
      "112\t72\t0.0\t1\t55 17\n",
      "113\t94\t0.0\t1\t71 23\n",
      "114\t62\t0.0\t1\t39 23\n",
      "115\t83\t0.0\t1\t54 29\n",
      "116\t80\t0.0\t1\t54 26\n",
      "117\t77\t0.0\t1\t54 23\n",
      "118\t73\t0.0\t1\t50 23\n",
      "119\t88\t0.0\t1\t63 25\n",
      "120\t66\t0.0\t1\t47 19\n",
      "121\t97\t0.0\t1\t69 28\n",
      "122\t94\t0.0\t1\t57 37\n",
      "123\t81\t0.0\t1\t60 21\n",
      "124\t87\t0.0\t1\t55 32\n",
      "125\t77\t0.0\t1\t53 24\n",
      "126\t64\t0.0\t1\t43 21\n",
      "127\t86\t0.0\t1\t64 22\n",
      "128\t75\t0.0\t1\t50 25\n",
      "129\t68\t0.0\t1\t43 25\n",
      "130\t80\t0.0\t1\t65 15\n",
      "131\t75\t0.0\t1\t57 18\n",
      "132\t76\t0.0\t1\t64 12\n",
      "133\t96\t0.0\t1\t69 27\n",
      "134\t60\t0.0\t1\t44 16\n",
      "135\t76\t0.0\t1\t61 15\n",
      "136\t60\t0.0\t1\t49 11\n",
      "137\t70\t0.0\t1\t57 13\n",
      "138\t56\t0.0\t1\t37 19\n",
      "139\t62\t0.0\t1\t51 11\n",
      "140\t73\t0.0\t1\t55 18\n",
      "141\t75\t0.0\t1\t52 23\n",
      "142\t72\t0.0\t1\t57 15\n",
      "143\t65\t0.0\t1\t54 11\n",
      "144\t68\t0.0\t1\t53 15\n",
      "145\t67\t0.0\t1\t55 12\n",
      "146\t65\t0.0\t1\t51 14\n",
      "147\t53\t0.0\t1\t41 12\n",
      "148\t69\t0.0\t1\t55 14\n",
      "149\t61\t0.0\t1\t52 9\n",
      "150\t38\t0.0\t1\t32 6\n",
      "151\t12\t0.0\t1\t9 3\n",
      "\n",
      "\n",
      "RUN STATISTICS FOR INPUT FILE: /home/sam/data/geoduck_illumina/NR019_S7_L001_R1_001.fastq.gz\n",
      "=============================================\n",
      "44316 sequences processed in total\n",
      "The length threshold of paired-end sequences gets evaluated later on (in the validation step)\n",
      "\n",
      "Writing report to '/home/sam/data/geoduck_illumina/trimmed/NR019_S7_L001_R2_001.fastq.gz_trimming_report.txt'\n",
      "\n",
      "SUMMARISING RUN PARAMETERS\n",
      "==========================\n",
      "Input filename: /home/sam/data/geoduck_illumina/NR019_S7_L001_R2_001.fastq.gz\n",
      "Trimming mode: paired-end\n",
      "Trim Galore version: 0.4.4\n",
      "Cutadapt version: 1.9.1\n",
      "Quality Phred score cutoff: 20\n",
      "Quality encoding type selected: ASCII+33\n",
      "Adapter sequence: 'CTGTCTCTTATA' (Nextera Transposase sequence; auto-detected)\n",
      "Maximum trimming error rate: 0.1 (default)\n",
      "Minimum required adapter overlap (stringency): 1 bp\n",
      "Minimum required sequence length for both reads before a sequence pair gets removed: 20 bp\n",
      "Output file(s) will be GZIP compressed\n",
      "\n",
      "Writing final adapter and quality trimmed output to NR019_S7_L001_R2_001_trimmed.fq.gz\n",
      "\n",
      "\n",
      "  >>> Now performing quality (cutoff 20) and adapter trimming in a single pass for the adapter sequence: 'CTGTCTCTTATA' from file /home/sam/data/geoduck_illumina/NR019_S7_L001_R2_001.fastq.gz <<< \n",
      "This is cutadapt 1.9.1 with Python 2.7.12\n",
      "Command line parameters: -f fastq -e 0.1 -q 20 -O 1 -a CTGTCTCTTATA /home/sam/data/geoduck_illumina/NR019_S7_L001_R2_001.fastq.gz\n",
      "Trimming 1 adapter with at most 10.0% errors in single-end mode ...\n",
      "Finished in 1.16 s (26 us/read; 2.30 M reads/minute).\n",
      "\n",
      "=== Summary ===\n",
      "\n",
      "Total reads processed:                  44,316\n",
      "Reads with adapters:                    18,561 (41.9%)\n",
      "Reads written (passing filters):        44,316 (100.0%)\n",
      "\n",
      "Total basepairs processed:     6,691,716 bp\n",
      "Quality-trimmed:                  61,028 bp (0.9%)\n",
      "Total written (filtered):      5,736,985 bp (85.7%)\n",
      "\n",
      "=== Adapter 1 ===\n",
      "\n",
      "Sequence: CTGTCTCTTATA; Type: regular 3'; Length: 12; Trimmed: 18561 times.\n",
      "\n",
      "No. of allowed errors:\n",
      "0-9 bp: 0; 10-12 bp: 1\n",
      "\n",
      "Bases preceding removed adapters:\n",
      "  A: 26.6%\n",
      "  C: 27.0%\n",
      "  G: 18.1%\n",
      "  T: 28.1%\n",
      "  none/other: 0.2%\n",
      "\n",
      "Overview of removed sequences\n",
      "length\tcount\texpect\tmax.err\terror counts\n",
      "1\t6053\t11079.0\t0\t6053\n",
      "2\t1568\t2769.8\t0\t1568\n",
      "3\t484\t692.4\t0\t484\n",
      "4\t164\t173.1\t0\t164\n",
      "5\t78\t43.3\t0\t78\n",
      "6\t42\t10.8\t0\t42\n",
      "7\t48\t2.7\t0\t48\n",
      "8\t38\t0.7\t0\t38\n",
      "9\t32\t0.2\t0\t28 4\n",
      "10\t56\t0.0\t1\t32 24\n",
      "11\t44\t0.0\t1\t25 19\n",
      "12\t43\t0.0\t1\t24 19\n",
      "13\t62\t0.0\t1\t38 24\n",
      "14\t39\t0.0\t1\t21 18\n",
      "15\t61\t0.0\t1\t35 26\n",
      "16\t55\t0.0\t1\t29 26\n",
      "17\t42\t0.0\t1\t23 19\n",
      "18\t56\t0.0\t1\t28 28\n",
      "19\t44\t0.0\t1\t25 19\n",
      "20\t45\t0.0\t1\t27 18\n",
      "21\t46\t0.0\t1\t30 16\n",
      "22\t58\t0.0\t1\t36 22\n",
      "23\t45\t0.0\t1\t27 18\n",
      "24\t40\t0.0\t1\t29 11\n",
      "25\t51\t0.0\t1\t30 21\n",
      "26\t57\t0.0\t1\t33 24\n",
      "27\t61\t0.0\t1\t40 21\n",
      "28\t45\t0.0\t1\t28 17\n",
      "29\t75\t0.0\t1\t48 27\n",
      "30\t55\t0.0\t1\t26 29\n",
      "31\t66\t0.0\t1\t39 27\n",
      "32\t43\t0.0\t1\t25 18\n",
      "33\t42\t0.0\t1\t29 13\n",
      "34\t53\t0.0\t1\t37 16\n",
      "35\t45\t0.0\t1\t21 24\n",
      "36\t70\t0.0\t1\t40 30\n",
      "37\t61\t0.0\t1\t41 20\n",
      "38\t56\t0.0\t1\t39 17\n",
      "39\t58\t0.0\t1\t36 22\n",
      "40\t55\t0.0\t1\t33 22\n",
      "41\t69\t0.0\t1\t42 27\n",
      "42\t82\t0.0\t1\t57 25\n",
      "43\t57\t0.0\t1\t26 31\n",
      "44\t44\t0.0\t1\t23 21\n",
      "45\t53\t0.0\t1\t35 18\n",
      "46\t55\t0.0\t1\t36 19\n",
      "47\t64\t0.0\t1\t43 21\n",
      "48\t57\t0.0\t1\t34 23\n",
      "49\t58\t0.0\t1\t33 25\n",
      "50\t64\t0.0\t1\t39 25\n",
      "51\t55\t0.0\t1\t37 18\n",
      "52\t59\t0.0\t1\t40 19\n",
      "53\t50\t0.0\t1\t38 12\n",
      "54\t73\t0.0\t1\t46 27\n",
      "55\t61\t0.0\t1\t41 20\n",
      "56\t55\t0.0\t1\t37 18\n",
      "57\t58\t0.0\t1\t37 21\n",
      "58\t76\t0.0\t1\t49 27\n",
      "59\t54\t0.0\t1\t37 17\n",
      "60\t72\t0.0\t1\t47 25\n",
      "61\t64\t0.0\t1\t40 24\n",
      "62\t63\t0.0\t1\t37 26\n",
      "63\t63\t0.0\t1\t31 32\n",
      "64\t55\t0.0\t1\t40 15\n",
      "65\t62\t0.0\t1\t44 18\n",
      "66\t76\t0.0\t1\t48 28\n",
      "67\t76\t0.0\t1\t54 22\n",
      "68\t71\t0.0\t1\t47 24\n",
      "69\t55\t0.0\t1\t37 18\n",
      "70\t74\t0.0\t1\t40 34\n",
      "71\t85\t0.0\t1\t61 24\n",
      "72\t70\t0.0\t1\t49 21\n",
      "73\t67\t0.0\t1\t46 21\n",
      "74\t67\t0.0\t1\t41 26\n",
      "75\t70\t0.0\t1\t48 22\n",
      "76\t73\t0.0\t1\t45 28\n",
      "77\t73\t0.0\t1\t53 20\n",
      "78\t73\t0.0\t1\t53 20\n",
      "79\t78\t0.0\t1\t48 30\n",
      "80\t77\t0.0\t1\t45 32\n",
      "81\t61\t0.0\t1\t47 14\n",
      "82\t61\t0.0\t1\t35 26\n",
      "83\t82\t0.0\t1\t71 11\n",
      "84\t70\t0.0\t1\t48 22\n",
      "85\t78\t0.0\t1\t59 19\n",
      "86\t59\t0.0\t1\t37 22\n",
      "87\t76\t0.0\t1\t47 29\n",
      "88\t55\t0.0\t1\t34 21\n",
      "89\t84\t0.0\t1\t62 22\n",
      "90\t68\t0.0\t1\t44 24\n",
      "91\t69\t0.0\t1\t44 25\n",
      "92\t74\t0.0\t1\t49 25\n",
      "93\t79\t0.0\t1\t60 19\n",
      "94\t87\t0.0\t1\t61 26\n",
      "95\t83\t0.0\t1\t60 23\n",
      "96\t83\t0.0\t1\t58 25\n",
      "97\t77\t0.0\t1\t50 27\n",
      "98\t87\t0.0\t1\t70 17\n",
      "99\t87\t0.0\t1\t57 30\n",
      "100\t85\t0.0\t1\t60 25\n",
      "101\t95\t0.0\t1\t64 31\n",
      "102\t87\t0.0\t1\t61 26\n",
      "103\t65\t0.0\t1\t41 24\n",
      "104\t72\t0.0\t1\t57 15\n",
      "105\t65\t0.0\t1\t48 17\n",
      "106\t70\t0.0\t1\t52 18\n",
      "107\t76\t0.0\t1\t49 27\n",
      "108\t81\t0.0\t1\t53 28\n",
      "109\t69\t0.0\t1\t48 21\n",
      "110\t75\t0.0\t1\t57 18\n",
      "111\t78\t0.0\t1\t55 23\n",
      "112\t93\t0.0\t1\t65 28\n",
      "113\t86\t0.0\t1\t62 24\n",
      "114\t87\t0.0\t1\t50 37\n",
      "115\t91\t0.0\t1\t70 21\n",
      "116\t89\t0.0\t1\t62 27\n",
      "117\t92\t0.0\t1\t70 22\n",
      "118\t100\t0.0\t1\t76 24\n",
      "119\t91\t0.0\t1\t65 26\n",
      "120\t84\t0.0\t1\t57 27\n",
      "121\t93\t0.0\t1\t73 20\n",
      "122\t87\t0.0\t1\t59 28\n",
      "123\t96\t0.0\t1\t74 22\n",
      "124\t106\t0.0\t1\t80 26\n",
      "125\t98\t0.0\t1\t75 23\n",
      "126\t91\t0.0\t1\t65 26\n",
      "127\t88\t0.0\t1\t61 27\n",
      "128\t87\t0.0\t1\t62 25\n",
      "129\t90\t0.0\t1\t61 29\n",
      "130\t95\t0.0\t1\t68 27\n",
      "131\t102\t0.0\t1\t70 32\n",
      "132\t94\t0.0\t1\t71 23\n",
      "133\t104\t0.0\t1\t76 28\n",
      "134\t93\t0.0\t1\t67 26\n",
      "135\t94\t0.0\t1\t75 19\n",
      "136\t83\t0.0\t1\t60 23\n",
      "137\t77\t0.0\t1\t63 14\n",
      "138\t87\t0.0\t1\t59 28\n",
      "139\t81\t0.0\t1\t55 26\n",
      "140\t75\t0.0\t1\t61 14\n",
      "141\t100\t0.0\t1\t81 19\n",
      "142\t88\t0.0\t1\t71 17\n",
      "143\t93\t0.0\t1\t62 31\n",
      "144\t96\t0.0\t1\t75 21\n",
      "145\t95\t0.0\t1\t71 24\n",
      "146\t92\t0.0\t1\t73 19\n",
      "147\t73\t0.0\t1\t64 9\n",
      "148\t71\t0.0\t1\t59 12\n",
      "149\t58\t0.0\t1\t44 14\n",
      "150\t60\t0.0\t1\t42 18\n",
      "151\t19\t0.0\t1\t16 3\n",
      "\n",
      "\n",
      "RUN STATISTICS FOR INPUT FILE: /home/sam/data/geoduck_illumina/NR019_S7_L001_R2_001.fastq.gz\n",
      "=============================================\n",
      "44316 sequences processed in total\n",
      "The length threshold of paired-end sequences gets evaluated later on (in the validation step)\n",
      "\n",
      "Validate paired-end files NR019_S7_L001_R1_001_trimmed.fq.gz and NR019_S7_L001_R2_001_trimmed.fq.gz\n",
      "file_1: NR019_S7_L001_R1_001_trimmed.fq.gz, file_2: NR019_S7_L001_R2_001_trimmed.fq.gz\n",
      "\n",
      "\n",
      ">>>>> Now validing the length of the 2 paired-end infiles: NR019_S7_L001_R1_001_trimmed.fq.gz and NR019_S7_L001_R2_001_trimmed.fq.gz <<<<<\n",
      "Writing validated paired-end read 1 reads to NR019_S7_L001_R1_001_val_1.fq.gz\n",
      "Writing validated paired-end read 2 reads to NR019_S7_L001_R2_001_val_2.fq.gz\n",
      "\n",
      "Total number of sequences analysed: 44316\n",
      "\n",
      "Number of sequence pairs removed because at least one read was shorter than the length cutoff (20 bp): 3072 (6.93%)\n",
      "\n",
      "Deleting both intermediate output files NR019_S7_L001_R1_001_trimmed.fq.gz and NR019_S7_L001_R2_001_trimmed.fq.gz\n",
      "\n",
      "====================================================================================================\n",
      "\n",
      "Writing report to '/home/sam/data/geoduck_illumina/trimmed/NR019_S7_L002_R1_001.fastq.gz_trimming_report.txt'\n",
      "\n",
      "SUMMARISING RUN PARAMETERS\n",
      "==========================\n",
      "Input filename: /home/sam/data/geoduck_illumina/NR019_S7_L002_R1_001.fastq.gz\n",
      "Trimming mode: paired-end\n",
      "Trim Galore version: 0.4.4\n",
      "Cutadapt version: 1.9.1\n",
      "Quality Phred score cutoff: 20\n",
      "Quality encoding type selected: ASCII+33\n",
      "Adapter sequence: 'CTGTCTCTTATA' (Nextera Transposase sequence; auto-detected)\n",
      "Maximum trimming error rate: 0.1 (default)\n",
      "Minimum required adapter overlap (stringency): 1 bp\n",
      "Minimum required sequence length for both reads before a sequence pair gets removed: 20 bp\n",
      "Output file(s) will be GZIP compressed\n",
      "\n",
      "Writing final adapter and quality trimmed output to NR019_S7_L002_R1_001_trimmed.fq.gz\n",
      "\n",
      "\n",
      "  >>> Now performing quality (cutoff 20) and adapter trimming in a single pass for the adapter sequence: 'CTGTCTCTTATA' from file /home/sam/data/geoduck_illumina/NR019_S7_L002_R1_001.fastq.gz <<< \n",
      "This is cutadapt 1.9.1 with Python 2.7.12\n",
      "Command line parameters: -f fastq -e 0.1 -q 20 -O 1 -a CTGTCTCTTATA /home/sam/data/geoduck_illumina/NR019_S7_L002_R1_001.fastq.gz\n",
      "Trimming 1 adapter with at most 10.0% errors in single-end mode ...\n",
      "Finished in 0.63 s (26 us/read; 2.31 M reads/minute).\n",
      "\n",
      "=== Summary ===\n",
      "\n",
      "Total reads processed:                  24,171\n",
      "Reads with adapters:                    10,315 (42.7%)\n",
      "Reads written (passing filters):        24,171 (100.0%)\n",
      "\n",
      "Total basepairs processed:     3,649,821 bp\n",
      "Quality-trimmed:                  26,203 bp (0.7%)\n",
      "Total written (filtered):      3,156,842 bp (86.5%)\n",
      "\n",
      "=== Adapter 1 ===\n",
      "\n",
      "Sequence: CTGTCTCTTATA; Type: regular 3'; Length: 12; Trimmed: 10315 times.\n",
      "\n",
      "No. of allowed errors:\n",
      "0-9 bp: 0; 10-12 bp: 1\n",
      "\n",
      "Bases preceding removed adapters:\n",
      "  A: 26.5%\n",
      "  C: 27.8%\n",
      "  G: 18.1%\n",
      "  T: 27.5%\n",
      "  none/other: 0.1%\n",
      "\n",
      "Overview of removed sequences\n",
      "length\tcount\texpect\tmax.err\terror counts\n",
      "1\t3268\t6042.8\t0\t3268\n",
      "2\t983\t1510.7\t0\t983\n",
      "3\t276\t377.7\t0\t276\n",
      "4\t91\t94.4\t0\t91\n",
      "5\t45\t23.6\t0\t45\n",
      "6\t35\t5.9\t0\t35\n",
      "7\t26\t1.5\t0\t26\n",
      "8\t32\t0.4\t0\t32\n",
      "9\t22\t0.1\t0\t22\n",
      "10\t43\t0.0\t1\t37 6\n",
      "11\t43\t0.0\t1\t30 13\n",
      "12\t35\t0.0\t1\t22 13\n",
      "13\t29\t0.0\t1\t21 8\n",
      "14\t29\t0.0\t1\t23 6\n",
      "15\t43\t0.0\t1\t29 14\n",
      "16\t49\t0.0\t1\t42 7\n",
      "17\t29\t0.0\t1\t18 11\n",
      "18\t37\t0.0\t1\t28 9\n",
      "19\t32\t0.0\t1\t25 7\n",
      "20\t39\t0.0\t1\t29 10\n",
      "21\t31\t0.0\t1\t22 9\n",
      "22\t39\t0.0\t1\t28 11\n",
      "23\t24\t0.0\t1\t15 9\n",
      "24\t26\t0.0\t1\t17 9\n",
      "25\t34\t0.0\t1\t25 9\n",
      "26\t35\t0.0\t1\t29 6\n",
      "27\t40\t0.0\t1\t31 9\n",
      "28\t26\t0.0\t1\t20 6\n",
      "29\t39\t0.0\t1\t24 15\n",
      "30\t27\t0.0\t1\t21 6\n",
      "31\t36\t0.0\t1\t27 9\n",
      "32\t33\t0.0\t1\t23 10\n",
      "33\t33\t0.0\t1\t20 13\n",
      "34\t38\t0.0\t1\t30 8\n",
      "35\t50\t0.0\t1\t33 17\n",
      "36\t45\t0.0\t1\t34 11\n",
      "37\t24\t0.0\t1\t19 5\n",
      "38\t36\t0.0\t1\t21 15\n",
      "39\t37\t0.0\t1\t23 14\n",
      "40\t46\t0.0\t1\t32 14\n",
      "41\t39\t0.0\t1\t22 17\n",
      "42\t30\t0.0\t1\t22 8\n",
      "43\t34\t0.0\t1\t26 8\n",
      "44\t35\t0.0\t1\t19 16\n",
      "45\t40\t0.0\t1\t34 6\n",
      "46\t38\t0.0\t1\t22 16\n",
      "47\t31\t0.0\t1\t23 8\n",
      "48\t35\t0.0\t1\t22 13\n",
      "49\t39\t0.0\t1\t30 9\n",
      "50\t40\t0.0\t1\t29 11\n",
      "51\t29\t0.0\t1\t19 10\n",
      "52\t32\t0.0\t1\t25 7\n",
      "53\t31\t0.0\t1\t24 7\n",
      "54\t32\t0.0\t1\t18 14\n",
      "55\t36\t0.0\t1\t32 4\n",
      "56\t44\t0.0\t1\t34 10\n",
      "57\t37\t0.0\t1\t29 8\n",
      "58\t39\t0.0\t1\t30 9\n",
      "59\t24\t0.0\t1\t19 5\n",
      "60\t37\t0.0\t1\t29 8\n",
      "61\t26\t0.0\t1\t18 8\n",
      "62\t31\t0.0\t1\t29 2\n",
      "63\t38\t0.0\t1\t34 4\n",
      "64\t35\t0.0\t1\t26 9\n",
      "65\t37\t0.0\t1\t30 7\n",
      "66\t37\t0.0\t1\t27 10\n",
      "67\t33\t0.0\t1\t23 10\n",
      "68\t36\t0.0\t1\t28 8\n",
      "69\t45\t0.0\t1\t36 9\n",
      "70\t40\t0.0\t1\t33 7\n",
      "71\t44\t0.0\t1\t36 8\n",
      "72\t42\t0.0\t1\t27 15\n",
      "73\t43\t0.0\t1\t37 6\n",
      "74\t44\t0.0\t1\t38 6\n",
      "75\t43\t0.0\t1\t41 2\n",
      "76\t38\t0.0\t1\t32 6\n",
      "77\t34\t0.0\t1\t25 9\n",
      "78\t38\t0.0\t1\t34 4\n",
      "79\t48\t0.0\t1\t40 8\n",
      "80\t47\t0.0\t1\t33 14\n",
      "81\t45\t0.0\t1\t25 20\n",
      "82\t29\t0.0\t1\t24 5\n",
      "83\t36\t0.0\t1\t30 6\n",
      "84\t26\t0.0\t1\t22 4\n",
      "85\t41\t0.0\t1\t34 7\n",
      "86\t41\t0.0\t1\t30 11\n",
      "87\t37\t0.0\t1\t31 6\n",
      "88\t27\t0.0\t1\t23 4\n",
      "89\t35\t0.0\t1\t30 5\n",
      "90\t48\t0.0\t1\t40 8\n",
      "91\t39\t0.0\t1\t30 9\n",
      "92\t59\t0.0\t1\t46 13\n",
      "93\t39\t0.0\t1\t32 7\n",
      "94\t53\t0.0\t1\t44 9\n",
      "95\t51\t0.0\t1\t42 9\n",
      "96\t46\t0.0\t1\t34 12\n",
      "97\t42\t0.0\t1\t32 10\n",
      "98\t40\t0.0\t1\t29 11\n",
      "99\t45\t0.0\t1\t38 7\n",
      "100\t33\t0.0\t1\t26 7\n",
      "101\t50\t0.0\t1\t43 7\n",
      "102\t48\t0.0\t1\t39 9\n",
      "103\t35\t0.0\t1\t25 10\n",
      "104\t39\t0.0\t1\t28 11\n",
      "105\t33\t0.0\t1\t26 7\n",
      "106\t49\t0.0\t1\t41 8\n",
      "107\t56\t0.0\t1\t47 9\n",
      "108\t55\t0.0\t1\t45 10\n",
      "109\t40\t0.0\t1\t33 7\n",
      "110\t42\t0.0\t1\t33 9\n",
      "111\t39\t0.0\t1\t36 3\n",
      "112\t55\t0.0\t1\t46 9\n",
      "113\t39\t0.0\t1\t30 9\n",
      "114\t49\t0.0\t1\t37 12\n",
      "115\t44\t0.0\t1\t30 14\n",
      "116\t30\t0.0\t1\t25 5\n",
      "117\t45\t0.0\t1\t35 10\n",
      "118\t36\t0.0\t1\t27 9\n",
      "119\t49\t0.0\t1\t35 14\n",
      "120\t44\t0.0\t1\t34 10\n",
      "121\t46\t0.0\t1\t37 9\n",
      "122\t42\t0.0\t1\t32 10\n",
      "123\t43\t0.0\t1\t37 6\n",
      "124\t43\t0.0\t1\t27 16\n",
      "125\t55\t0.0\t1\t42 13\n",
      "126\t36\t0.0\t1\t34 2\n",
      "127\t41\t0.0\t1\t38 3\n",
      "128\t41\t0.0\t1\t32 9\n",
      "129\t44\t0.0\t1\t35 9\n",
      "130\t53\t0.0\t1\t47 6\n",
      "131\t53\t0.0\t1\t47 6\n",
      "132\t56\t0.0\t1\t49 7\n",
      "133\t40\t0.0\t1\t30 10\n",
      "134\t33\t0.0\t1\t29 4\n",
      "135\t43\t0.0\t1\t34 9\n",
      "136\t34\t0.0\t1\t27 7\n",
      "137\t46\t0.0\t1\t40 6\n",
      "138\t47\t0.0\t1\t38 9\n",
      "139\t33\t0.0\t1\t29 4\n",
      "140\t37\t0.0\t1\t31 6\n",
      "141\t32\t0.0\t1\t27 5\n",
      "142\t39\t0.0\t1\t34 5\n",
      "143\t39\t0.0\t1\t33 6\n",
      "144\t51\t0.0\t1\t43 8\n",
      "145\t40\t0.0\t1\t35 5\n",
      "146\t50\t0.0\t1\t43 7\n",
      "147\t38\t0.0\t1\t33 5\n",
      "148\t33\t0.0\t1\t31 2\n",
      "149\t31\t0.0\t1\t30 1\n",
      "150\t35\t0.0\t1\t28 7\n",
      "151\t6\t0.0\t1\t6\n",
      "\n",
      "\n",
      "RUN STATISTICS FOR INPUT FILE: /home/sam/data/geoduck_illumina/NR019_S7_L002_R1_001.fastq.gz\n",
      "=============================================\n",
      "24171 sequences processed in total\n",
      "The length threshold of paired-end sequences gets evaluated later on (in the validation step)\n",
      "\n",
      "Writing report to '/home/sam/data/geoduck_illumina/trimmed/NR019_S7_L002_R2_001.fastq.gz_trimming_report.txt'\n",
      "\n",
      "SUMMARISING RUN PARAMETERS\n",
      "==========================\n",
      "Input filename: /home/sam/data/geoduck_illumina/NR019_S7_L002_R2_001.fastq.gz\n",
      "Trimming mode: paired-end\n",
      "Trim Galore version: 0.4.4\n",
      "Cutadapt version: 1.9.1\n",
      "Quality Phred score cutoff: 20\n",
      "Quality encoding type selected: ASCII+33\n",
      "Adapter sequence: 'CTGTCTCTTATA' (Nextera Transposase sequence; auto-detected)\n",
      "Maximum trimming error rate: 0.1 (default)\n",
      "Minimum required adapter overlap (stringency): 1 bp\n",
      "Minimum required sequence length for both reads before a sequence pair gets removed: 20 bp\n",
      "Output file(s) will be GZIP compressed\n",
      "\n",
      "Writing final adapter and quality trimmed output to NR019_S7_L002_R2_001_trimmed.fq.gz\n",
      "\n",
      "\n",
      "  >>> Now performing quality (cutoff 20) and adapter trimming in a single pass for the adapter sequence: 'CTGTCTCTTATA' from file /home/sam/data/geoduck_illumina/NR019_S7_L002_R2_001.fastq.gz <<< \n",
      "This is cutadapt 1.9.1 with Python 2.7.12\n",
      "Command line parameters: -f fastq -e 0.1 -q 20 -O 1 -a CTGTCTCTTATA /home/sam/data/geoduck_illumina/NR019_S7_L002_R2_001.fastq.gz\n",
      "Trimming 1 adapter with at most 10.0% errors in single-end mode ...\n",
      "Finished in 0.62 s (26 us/read; 2.32 M reads/minute).\n",
      "\n",
      "=== Summary ===\n",
      "\n",
      "Total reads processed:                  24,171\n",
      "Reads with adapters:                    10,641 (44.0%)\n",
      "Reads written (passing filters):        24,171 (100.0%)\n",
      "\n",
      "Total basepairs processed:     3,649,821 bp\n",
      "Quality-trimmed:                  30,593 bp (0.8%)\n",
      "Total written (filtered):      3,064,527 bp (84.0%)\n",
      "\n",
      "=== Adapter 1 ===\n",
      "\n",
      "Sequence: CTGTCTCTTATA; Type: regular 3'; Length: 12; Trimmed: 10641 times.\n",
      "\n",
      "No. of allowed errors:\n",
      "0-9 bp: 0; 10-12 bp: 1\n",
      "\n",
      "Bases preceding removed adapters:\n",
      "  A: 23.8%\n",
      "  C: 29.4%\n",
      "  G: 19.7%\n",
      "  T: 26.9%\n",
      "  none/other: 0.1%\n",
      "\n",
      "Overview of removed sequences\n",
      "length\tcount\texpect\tmax.err\terror counts\n",
      "1\t3115\t6042.8\t0\t3115\n",
      "2\t786\t1510.7\t0\t786\n",
      "3\t254\t377.7\t0\t254\n",
      "4\t83\t94.4\t0\t83\n",
      "5\t45\t23.6\t0\t45\n",
      "6\t39\t5.9\t0\t39\n",
      "7\t25\t1.5\t0\t25\n",
      "8\t41\t0.4\t0\t41\n",
      "9\t21\t0.1\t0\t21\n",
      "10\t28\t0.0\t1\t18 10\n",
      "11\t30\t0.0\t1\t22 8\n",
      "12\t34\t0.0\t1\t26 8\n",
      "13\t32\t0.0\t1\t22 10\n",
      "14\t25\t0.0\t1\t21 4\n",
      "15\t30\t0.0\t1\t19 11\n",
      "16\t32\t0.0\t1\t23 9\n",
      "17\t15\t0.0\t1\t11 4\n",
      "18\t39\t0.0\t1\t23 16\n",
      "19\t29\t0.0\t1\t20 9\n",
      "20\t24\t0.0\t1\t15 9\n",
      "21\t35\t0.0\t1\t20 15\n",
      "22\t42\t0.0\t1\t30 12\n",
      "23\t32\t0.0\t1\t22 10\n",
      "24\t25\t0.0\t1\t16 9\n",
      "25\t33\t0.0\t1\t23 10\n",
      "26\t39\t0.0\t1\t28 11\n",
      "27\t34\t0.0\t1\t28 6\n",
      "28\t28\t0.0\t1\t23 5\n",
      "29\t26\t0.0\t1\t18 8\n",
      "30\t38\t0.0\t1\t28 10\n",
      "31\t33\t0.0\t1\t20 13\n",
      "32\t33\t0.0\t1\t22 11\n",
      "33\t26\t0.0\t1\t18 8\n",
      "34\t29\t0.0\t1\t24 5\n",
      "35\t45\t0.0\t1\t32 13\n",
      "36\t32\t0.0\t1\t22 10\n",
      "37\t40\t0.0\t1\t30 10\n",
      "38\t43\t0.0\t1\t34 9\n",
      "39\t28\t0.0\t1\t24 4\n",
      "40\t40\t0.0\t1\t28 12\n",
      "41\t31\t0.0\t1\t22 9\n",
      "42\t40\t0.0\t1\t25 15\n",
      "43\t36\t0.0\t1\t24 12\n",
      "44\t44\t0.0\t1\t33 11\n",
      "45\t48\t0.0\t1\t37 11\n",
      "46\t28\t0.0\t1\t22 6\n",
      "47\t34\t0.0\t1\t26 8\n",
      "48\t35\t0.0\t1\t22 13\n",
      "49\t39\t0.0\t1\t26 13\n",
      "50\t51\t0.0\t1\t36 15\n",
      "51\t31\t0.0\t1\t21 10\n",
      "52\t39\t0.0\t1\t23 16\n",
      "53\t30\t0.0\t1\t20 10\n",
      "54\t50\t0.0\t1\t36 14\n",
      "55\t41\t0.0\t1\t33 8\n",
      "56\t35\t0.0\t1\t27 8\n",
      "57\t32\t0.0\t1\t24 8\n",
      "58\t34\t0.0\t1\t22 12\n",
      "59\t42\t0.0\t1\t36 6\n",
      "60\t39\t0.0\t1\t30 9\n",
      "61\t39\t0.0\t1\t29 10\n",
      "62\t46\t0.0\t1\t34 12\n",
      "63\t43\t0.0\t1\t31 12\n",
      "64\t37\t0.0\t1\t27 10\n",
      "65\t32\t0.0\t1\t24 8\n",
      "66\t45\t0.0\t1\t36 9\n",
      "67\t34\t0.0\t1\t27 7\n",
      "68\t51\t0.0\t1\t40 11\n",
      "69\t38\t0.0\t1\t29 9\n",
      "70\t56\t0.0\t1\t39 17\n",
      "71\t47\t0.0\t1\t35 12\n",
      "72\t43\t0.0\t1\t36 7\n",
      "73\t40\t0.0\t1\t30 10\n",
      "74\t38\t0.0\t1\t30 8\n",
      "75\t34\t0.0\t1\t24 10\n",
      "76\t42\t0.0\t1\t38 4\n",
      "77\t42\t0.0\t1\t31 11\n",
      "78\t48\t0.0\t1\t39 9\n",
      "79\t33\t0.0\t1\t26 7\n",
      "80\t49\t0.0\t1\t40 9\n",
      "81\t37\t0.0\t1\t26 11\n",
      "82\t43\t0.0\t1\t35 8\n",
      "83\t48\t0.0\t1\t40 8\n",
      "84\t36\t0.0\t1\t28 8\n",
      "85\t50\t0.0\t1\t38 12\n",
      "86\t33\t0.0\t1\t27 6\n",
      "87\t41\t0.0\t1\t34 7\n",
      "88\t59\t0.0\t1\t52 7\n",
      "89\t46\t0.0\t1\t38 8\n",
      "90\t37\t0.0\t1\t35 2\n",
      "91\t35\t0.0\t1\t29 6\n",
      "92\t46\t0.0\t1\t38 8\n",
      "93\t44\t0.0\t1\t33 11\n",
      "94\t46\t0.0\t1\t33 13\n",
      "95\t48\t0.0\t1\t38 10\n",
      "96\t52\t0.0\t1\t42 10\n",
      "97\t52\t0.0\t1\t39 13\n",
      "98\t53\t0.0\t1\t46 7\n",
      "99\t55\t0.0\t1\t46 9\n",
      "100\t48\t0.0\t1\t35 13\n",
      "101\t67\t0.0\t1\t56 11\n",
      "102\t59\t0.0\t1\t44 15\n",
      "103\t41\t0.0\t1\t32 9\n",
      "104\t58\t0.0\t1\t44 14\n",
      "105\t63\t0.0\t1\t44 19\n",
      "106\t46\t0.0\t1\t36 10\n",
      "107\t62\t0.0\t1\t54 8\n",
      "108\t45\t0.0\t1\t35 10\n",
      "109\t51\t0.0\t1\t44 7\n",
      "110\t45\t0.0\t1\t37 8\n",
      "111\t49\t0.0\t1\t44 5\n",
      "112\t47\t0.0\t1\t38 9\n",
      "113\t55\t0.0\t1\t46 9\n",
      "114\t68\t0.0\t1\t53 15\n",
      "115\t49\t0.0\t1\t42 7\n",
      "116\t40\t0.0\t1\t32 8\n",
      "117\t62\t0.0\t1\t52 10\n",
      "118\t51\t0.0\t1\t40 11\n",
      "119\t51\t0.0\t1\t37 14\n",
      "120\t58\t0.0\t1\t47 11\n",
      "121\t52\t0.0\t1\t45 7\n",
      "122\t69\t0.0\t1\t55 14\n",
      "123\t65\t0.0\t1\t48 17\n",
      "124\t55\t0.0\t1\t47 8\n",
      "125\t47\t0.0\t1\t43 4\n",
      "126\t65\t0.0\t1\t57 8\n",
      "127\t60\t0.0\t1\t52 8\n",
      "128\t56\t0.0\t1\t41 15\n",
      "129\t49\t0.0\t1\t45 4\n",
      "130\t63\t0.0\t1\t56 7\n",
      "131\t64\t0.0\t1\t52 12\n",
      "132\t61\t0.0\t1\t56 5\n",
      "133\t60\t0.0\t1\t49 11\n",
      "134\t51\t0.0\t1\t42 9\n",
      "135\t59\t0.0\t1\t51 8\n",
      "136\t42\t0.0\t1\t39 3\n",
      "137\t53\t0.0\t1\t42 11\n",
      "138\t60\t0.0\t1\t48 12\n",
      "139\t58\t0.0\t1\t49 9\n",
      "140\t54\t0.0\t1\t45 9\n",
      "141\t52\t0.0\t1\t47 5\n",
      "142\t60\t0.0\t1\t51 9\n",
      "143\t53\t0.0\t1\t46 7\n",
      "144\t45\t0.0\t1\t40 5\n",
      "145\t54\t0.0\t1\t44 10\n",
      "146\t56\t0.0\t1\t47 9\n",
      "147\t59\t0.0\t1\t51 8\n",
      "148\t46\t0.0\t1\t38 8\n",
      "149\t48\t0.0\t1\t40 8\n",
      "150\t33\t0.0\t1\t28 5\n",
      "151\t12\t0.0\t1\t8 4\n",
      "\n",
      "\n",
      "RUN STATISTICS FOR INPUT FILE: /home/sam/data/geoduck_illumina/NR019_S7_L002_R2_001.fastq.gz\n",
      "=============================================\n",
      "24171 sequences processed in total\n",
      "The length threshold of paired-end sequences gets evaluated later on (in the validation step)\n",
      "\n",
      "Validate paired-end files NR019_S7_L002_R1_001_trimmed.fq.gz and NR019_S7_L002_R2_001_trimmed.fq.gz\n",
      "file_1: NR019_S7_L002_R1_001_trimmed.fq.gz, file_2: NR019_S7_L002_R2_001_trimmed.fq.gz\n",
      "\n",
      "\n",
      ">>>>> Now validing the length of the 2 paired-end infiles: NR019_S7_L002_R1_001_trimmed.fq.gz and NR019_S7_L002_R2_001_trimmed.fq.gz <<<<<\n",
      "Writing validated paired-end read 1 reads to NR019_S7_L002_R1_001_val_1.fq.gz\n",
      "Writing validated paired-end read 2 reads to NR019_S7_L002_R2_001_val_2.fq.gz\n",
      "\n",
      "Total number of sequences analysed: 24171\n",
      "\n",
      "Number of sequence pairs removed because at least one read was shorter than the length cutoff (20 bp): 1861 (7.70%)\n",
      "\n",
      "Deleting both intermediate output files NR019_S7_L002_R1_001_trimmed.fq.gz and NR019_S7_L002_R2_001_trimmed.fq.gz\n",
      "\n",
      "====================================================================================================\n",
      "\n",
      "Writing report to '/home/sam/data/geoduck_illumina/trimmed/NR021_S8_L001_R1_001.fastq.gz_trimming_report.txt'\n",
      "\n",
      "SUMMARISING RUN PARAMETERS\n",
      "==========================\n",
      "Input filename: /home/sam/data/geoduck_illumina/NR021_S8_L001_R1_001.fastq.gz\n",
      "Trimming mode: paired-end\n",
      "Trim Galore version: 0.4.4\n",
      "Cutadapt version: 1.9.1\n",
      "Quality Phred score cutoff: 20\n",
      "Quality encoding type selected: ASCII+33\n",
      "Adapter sequence: 'CTGTCTCTTATA' (Nextera Transposase sequence; auto-detected)\n",
      "Maximum trimming error rate: 0.1 (default)\n",
      "Minimum required adapter overlap (stringency): 1 bp\n",
      "Minimum required sequence length for both reads before a sequence pair gets removed: 20 bp\n",
      "Output file(s) will be GZIP compressed\n",
      "\n",
      "Writing final adapter and quality trimmed output to NR021_S8_L001_R1_001_trimmed.fq.gz\n",
      "\n",
      "\n",
      "  >>> Now performing quality (cutoff 20) and adapter trimming in a single pass for the adapter sequence: 'CTGTCTCTTATA' from file /home/sam/data/geoduck_illumina/NR021_S8_L001_R1_001.fastq.gz <<< \n",
      "This is cutadapt 1.9.1 with Python 2.7.12\n",
      "Command line parameters: -f fastq -e 0.1 -q 20 -O 1 -a CTGTCTCTTATA /home/sam/data/geoduck_illumina/NR021_S8_L001_R1_001.fastq.gz\n",
      "Trimming 1 adapter with at most 10.0% errors in single-end mode ...\n",
      "Finished in 103.81 s (26 us/read; 2.33 M reads/minute).\n",
      "\n",
      "=== Summary ===\n",
      "\n",
      "Total reads processed:               4,037,578\n",
      "Reads with adapters:                 1,275,359 (31.6%)\n",
      "Reads written (passing filters):     4,037,578 (100.0%)\n",
      "\n",
      "Total basepairs processed:   609,674,278 bp\n",
      "Quality-trimmed:               1,279,284 bp (0.2%)\n",
      "Total written (filtered):    596,297,509 bp (97.8%)\n",
      "\n",
      "=== Adapter 1 ===\n",
      "\n",
      "Sequence: CTGTCTCTTATA; Type: regular 3'; Length: 12; Trimmed: 1275359 times.\n",
      "\n",
      "No. of allowed errors:\n",
      "0-9 bp: 0; 10-12 bp: 1\n",
      "\n",
      "Bases preceding removed adapters:\n",
      "  A: 29.7%\n",
      "  C: 18.6%\n",
      "  G: 15.4%\n",
      "  T: 36.3%\n",
      "  none/other: 0.0%\n",
      "\n",
      "Overview of removed sequences\n",
      "length\tcount\texpect\tmax.err\terror counts\n",
      "1\t848241\t1009394.5\t0\t848241\n",
      "2\t210630\t252348.6\t0\t210630\n",
      "3\t69160\t63087.2\t0\t69160\n",
      "4\t15786\t15771.8\t0\t15786\n",
      "5\t4086\t3942.9\t0\t4086\n",
      "6\t1638\t985.7\t0\t1638\n",
      "7\t852\t246.4\t0\t852\n",
      "8\t835\t61.6\t0\t835\n",
      "9\t658\t15.4\t0\t583 75\n",
      "10\t720\t3.9\t1\t588 132\n",
      "11\t722\t1.0\t1\t621 101\n",
      "12\t620\t0.2\t1\t553 67\n",
      "13\t682\t0.2\t1\t628 54\n",
      "14\t624\t0.2\t1\t572 52\n",
      "15\t741\t0.2\t1\t680 61\n",
      "16\t812\t0.2\t1\t729 83\n",
      "17\t502\t0.2\t1\t459 43\n",
      "18\t731\t0.2\t1\t681 50\n",
      "19\t700\t0.2\t1\t628 72\n",
      "20\t669\t0.2\t1\t607 62\n",
      "21\t662\t0.2\t1\t601 61\n",
      "22\t783\t0.2\t1\t708 75\n",
      "23\t555\t0.2\t1\t504 51\n",
      "24\t766\t0.2\t1\t706 60\n",
      "25\t699\t0.2\t1\t631 68\n",
      "26\t725\t0.2\t1\t665 60\n",
      "27\t673\t0.2\t1\t605 68\n",
      "28\t751\t0.2\t1\t676 75\n",
      "29\t713\t0.2\t1\t658 55\n",
      "30\t764\t0.2\t1\t695 69\n",
      "31\t799\t0.2\t1\t717 82\n",
      "32\t708\t0.2\t1\t643 65\n",
      "33\t733\t0.2\t1\t652 81\n",
      "34\t678\t0.2\t1\t624 54\n",
      "35\t708\t0.2\t1\t631 77\n",
      "36\t761\t0.2\t1\t682 79\n",
      "37\t713\t0.2\t1\t658 55\n",
      "38\t762\t0.2\t1\t696 66\n",
      "39\t707\t0.2\t1\t640 67\n",
      "40\t743\t0.2\t1\t689 54\n",
      "41\t709\t0.2\t1\t644 65\n",
      "42\t773\t0.2\t1\t726 47\n",
      "43\t800\t0.2\t1\t744 56\n",
      "44\t783\t0.2\t1\t704 79\n",
      "45\t799\t0.2\t1\t694 105\n",
      "46\t797\t0.2\t1\t726 71\n",
      "47\t830\t0.2\t1\t755 75\n",
      "48\t819\t0.2\t1\t747 72\n",
      "49\t806\t0.2\t1\t733 73\n",
      "50\t780\t0.2\t1\t710 70\n",
      "51\t815\t0.2\t1\t755 60\n",
      "52\t747\t0.2\t1\t694 53\n",
      "53\t780\t0.2\t1\t723 57\n",
      "54\t759\t0.2\t1\t708 51\n",
      "55\t767\t0.2\t1\t719 48\n",
      "56\t840\t0.2\t1\t768 72\n",
      "57\t792\t0.2\t1\t723 69\n",
      "58\t806\t0.2\t1\t749 57\n",
      "59\t772\t0.2\t1\t703 69\n",
      "60\t832\t0.2\t1\t771 61\n",
      "61\t871\t0.2\t1\t799 72\n",
      "62\t883\t0.2\t1\t821 62\n",
      "63\t812\t0.2\t1\t753 59\n",
      "64\t787\t0.2\t1\t719 68\n",
      "65\t889\t0.2\t1\t815 74\n",
      "66\t879\t0.2\t1\t815 64\n",
      "67\t850\t0.2\t1\t767 83\n",
      "68\t877\t0.2\t1\t828 49\n",
      "69\t837\t0.2\t1\t775 62\n",
      "70\t896\t0.2\t1\t823 73\n",
      "71\t834\t0.2\t1\t770 64\n",
      "72\t891\t0.2\t1\t823 68\n",
      "73\t879\t0.2\t1\t816 63\n",
      "74\t885\t0.2\t1\t817 68\n",
      "75\t797\t0.2\t1\t734 63\n",
      "76\t823\t0.2\t1\t762 61\n",
      "77\t893\t0.2\t1\t844 49\n",
      "78\t872\t0.2\t1\t814 58\n",
      "79\t895\t0.2\t1\t840 55\n",
      "80\t903\t0.2\t1\t821 82\n",
      "81\t952\t0.2\t1\t897 55\n",
      "82\t923\t0.2\t1\t857 66\n",
      "83\t897\t0.2\t1\t821 76\n",
      "84\t882\t0.2\t1\t840 42\n",
      "85\t901\t0.2\t1\t836 65\n",
      "86\t910\t0.2\t1\t851 59\n",
      "87\t947\t0.2\t1\t884 63\n",
      "88\t918\t0.2\t1\t859 59\n",
      "89\t922\t0.2\t1\t862 60\n",
      "90\t940\t0.2\t1\t875 65\n",
      "91\t938\t0.2\t1\t884 54\n",
      "92\t934\t0.2\t1\t870 64\n",
      "93\t982\t0.2\t1\t929 53\n",
      "94\t916\t0.2\t1\t841 75\n",
      "95\t884\t0.2\t1\t834 50\n",
      "96\t1010\t0.2\t1\t930 80\n",
      "97\t1002\t0.2\t1\t939 63\n",
      "98\t990\t0.2\t1\t946 44\n",
      "99\t924\t0.2\t1\t872 52\n",
      "100\t988\t0.2\t1\t937 51\n",
      "101\t939\t0.2\t1\t880 59\n",
      "102\t953\t0.2\t1\t897 56\n",
      "103\t979\t0.2\t1\t920 59\n",
      "104\t959\t0.2\t1\t896 63\n",
      "105\t997\t0.2\t1\t951 46\n",
      "106\t938\t0.2\t1\t839 99\n",
      "107\t979\t0.2\t1\t918 61\n",
      "108\t952\t0.2\t1\t866 86\n",
      "109\t957\t0.2\t1\t891 66\n",
      "110\t931\t0.2\t1\t879 52\n",
      "111\t1011\t0.2\t1\t942 69\n",
      "112\t1000\t0.2\t1\t932 68\n",
      "113\t1038\t0.2\t1\t977 61\n",
      "114\t1019\t0.2\t1\t963 56\n",
      "115\t985\t0.2\t1\t924 61\n",
      "116\t1024\t0.2\t1\t957 67\n",
      "117\t1075\t0.2\t1\t1011 64\n",
      "118\t956\t0.2\t1\t896 60\n",
      "119\t999\t0.2\t1\t942 57\n",
      "120\t1057\t0.2\t1\t988 69\n",
      "121\t1001\t0.2\t1\t948 53\n",
      "122\t1048\t0.2\t1\t994 54\n",
      "123\t1019\t0.2\t1\t960 59\n",
      "124\t1008\t0.2\t1\t941 67\n",
      "125\t1009\t0.2\t1\t938 71\n",
      "126\t984\t0.2\t1\t930 54\n",
      "127\t1007\t0.2\t1\t955 52\n",
      "128\t1017\t0.2\t1\t964 53\n",
      "129\t1075\t0.2\t1\t1014 61\n",
      "130\t1087\t0.2\t1\t1045 42\n",
      "131\t1054\t0.2\t1\t1014 40\n",
      "132\t1091\t0.2\t1\t1036 55\n",
      "133\t1049\t0.2\t1\t1000 49\n",
      "134\t1055\t0.2\t1\t1011 44\n",
      "135\t1071\t0.2\t1\t1021 50\n",
      "136\t1057\t0.2\t1\t1006 51\n",
      "137\t905\t0.2\t1\t867 38\n",
      "138\t919\t0.2\t1\t886 33\n",
      "139\t937\t0.2\t1\t889 48\n",
      "140\t1009\t0.2\t1\t951 58\n",
      "141\t1090\t0.2\t1\t1040 50\n",
      "142\t952\t0.2\t1\t904 48\n",
      "143\t992\t0.2\t1\t915 77\n",
      "144\t979\t0.2\t1\t921 58\n",
      "145\t975\t0.2\t1\t918 57\n",
      "146\t1085\t0.2\t1\t1001 84\n",
      "147\t886\t0.2\t1\t851 35\n",
      "148\t908\t0.2\t1\t865 43\n",
      "149\t751\t0.2\t1\t717 34\n",
      "150\t751\t0.2\t1\t718 33\n",
      "151\t209\t0.2\t1\t166 43\n",
      "\n",
      "\n",
      "RUN STATISTICS FOR INPUT FILE: /home/sam/data/geoduck_illumina/NR021_S8_L001_R1_001.fastq.gz\n",
      "=============================================\n",
      "4037578 sequences processed in total\n",
      "The length threshold of paired-end sequences gets evaluated later on (in the validation step)\n",
      "\n",
      "Writing report to '/home/sam/data/geoduck_illumina/trimmed/NR021_S8_L001_R2_001.fastq.gz_trimming_report.txt'\n",
      "\n",
      "SUMMARISING RUN PARAMETERS\n",
      "==========================\n",
      "Input filename: /home/sam/data/geoduck_illumina/NR021_S8_L001_R2_001.fastq.gz\n",
      "Trimming mode: paired-end\n",
      "Trim Galore version: 0.4.4\n",
      "Cutadapt version: 1.9.1\n",
      "Quality Phred score cutoff: 20\n",
      "Quality encoding type selected: ASCII+33\n",
      "Adapter sequence: 'CTGTCTCTTATA' (Nextera Transposase sequence; auto-detected)\n",
      "Maximum trimming error rate: 0.1 (default)\n",
      "Minimum required adapter overlap (stringency): 1 bp\n",
      "Minimum required sequence length for both reads before a sequence pair gets removed: 20 bp\n",
      "Output file(s) will be GZIP compressed\n",
      "\n",
      "Writing final adapter and quality trimmed output to NR021_S8_L001_R2_001_trimmed.fq.gz\n",
      "\n",
      "\n",
      "  >>> Now performing quality (cutoff 20) and adapter trimming in a single pass for the adapter sequence: 'CTGTCTCTTATA' from file /home/sam/data/geoduck_illumina/NR021_S8_L001_R2_001.fastq.gz <<< \n",
      "This is cutadapt 1.9.1 with Python 2.7.12\n",
      "Command line parameters: -f fastq -e 0.1 -q 20 -O 1 -a CTGTCTCTTATA /home/sam/data/geoduck_illumina/NR021_S8_L001_R2_001.fastq.gz\n",
      "Trimming 1 adapter with at most 10.0% errors in single-end mode ...\n",
      "Finished in 100.95 s (25 us/read; 2.40 M reads/minute).\n",
      "\n",
      "=== Summary ===\n",
      "\n",
      "Total reads processed:               4,037,578\n",
      "Reads with adapters:                   905,282 (22.4%)\n",
      "Reads written (passing filters):     4,037,578 (100.0%)\n",
      "\n",
      "Total basepairs processed:   609,674,278 bp\n",
      "Quality-trimmed:               3,103,799 bp (0.5%)\n",
      "Total written (filtered):    596,405,354 bp (97.8%)\n",
      "\n",
      "=== Adapter 1 ===\n",
      "\n",
      "Sequence: CTGTCTCTTATA; Type: regular 3'; Length: 12; Trimmed: 905282 times.\n",
      "\n",
      "No. of allowed errors:\n",
      "0-9 bp: 0; 10-12 bp: 1\n",
      "\n",
      "Bases preceding removed adapters:\n",
      "  A: 26.9%\n",
      "  C: 19.4%\n",
      "  G: 20.9%\n",
      "  T: 32.7%\n",
      "  none/other: 0.0%\n",
      "\n",
      "Overview of removed sequences\n",
      "length\tcount\texpect\tmax.err\terror counts\n",
      "1\t591315\t1009394.5\t0\t591315\n",
      "2\t141381\t252348.6\t0\t141381\n",
      "3\t44243\t63087.2\t0\t44243\n",
      "4\t13764\t15771.8\t0\t13764\n",
      "5\t3067\t3942.9\t0\t3067\n",
      "6\t1349\t985.7\t0\t1349\n",
      "7\t737\t246.4\t0\t737\n",
      "8\t666\t61.6\t0\t666\n",
      "9\t608\t15.4\t0\t527 81\n",
      "10\t706\t3.9\t1\t544 162\n",
      "11\t657\t1.0\t1\t574 83\n",
      "12\t586\t0.2\t1\t521 65\n",
      "13\t623\t0.2\t1\t557 66\n",
      "14\t563\t0.2\t1\t510 53\n",
      "15\t682\t0.2\t1\t600 82\n",
      "16\t693\t0.2\t1\t636 57\n",
      "17\t498\t0.2\t1\t451 47\n",
      "18\t614\t0.2\t1\t541 73\n",
      "19\t633\t0.2\t1\t566 67\n",
      "20\t611\t0.2\t1\t543 68\n",
      "21\t646\t0.2\t1\t586 60\n",
      "22\t692\t0.2\t1\t629 63\n",
      "23\t532\t0.2\t1\t480 52\n",
      "24\t677\t0.2\t1\t599 78\n",
      "25\t595\t0.2\t1\t545 50\n",
      "26\t684\t0.2\t1\t615 69\n",
      "27\t634\t0.2\t1\t570 64\n",
      "28\t668\t0.2\t1\t608 60\n",
      "29\t626\t0.2\t1\t568 58\n",
      "30\t699\t0.2\t1\t647 52\n",
      "31\t688\t0.2\t1\t616 72\n",
      "32\t622\t0.2\t1\t568 54\n",
      "33\t654\t0.2\t1\t592 62\n",
      "34\t659\t0.2\t1\t593 66\n",
      "35\t649\t0.2\t1\t589 60\n",
      "36\t725\t0.2\t1\t665 60\n",
      "37\t587\t0.2\t1\t526 61\n",
      "38\t665\t0.2\t1\t607 58\n",
      "39\t689\t0.2\t1\t636 53\n",
      "40\t714\t0.2\t1\t646 68\n",
      "41\t683\t0.2\t1\t621 62\n",
      "42\t683\t0.2\t1\t628 55\n",
      "43\t672\t0.2\t1\t626 46\n",
      "44\t702\t0.2\t1\t637 65\n",
      "45\t716\t0.2\t1\t655 61\n",
      "46\t722\t0.2\t1\t665 57\n",
      "47\t696\t0.2\t1\t647 49\n",
      "48\t717\t0.2\t1\t653 64\n",
      "49\t681\t0.2\t1\t614 67\n",
      "50\t739\t0.2\t1\t681 58\n",
      "51\t751\t0.2\t1\t690 61\n",
      "52\t732\t0.2\t1\t666 66\n",
      "53\t677\t0.2\t1\t623 54\n",
      "54\t732\t0.2\t1\t678 54\n",
      "55\t717\t0.2\t1\t645 72\n",
      "56\t761\t0.2\t1\t699 62\n",
      "57\t675\t0.2\t1\t612 63\n",
      "58\t685\t0.2\t1\t634 51\n",
      "59\t757\t0.2\t1\t699 58\n",
      "60\t794\t0.2\t1\t736 58\n",
      "61\t744\t0.2\t1\t676 68\n",
      "62\t729\t0.2\t1\t674 55\n",
      "63\t703\t0.2\t1\t641 62\n",
      "64\t754\t0.2\t1\t713 41\n",
      "65\t724\t0.2\t1\t667 57\n",
      "66\t788\t0.2\t1\t726 62\n",
      "67\t801\t0.2\t1\t735 66\n",
      "68\t762\t0.2\t1\t706 56\n",
      "69\t788\t0.2\t1\t733 55\n",
      "70\t791\t0.2\t1\t730 61\n",
      "71\t783\t0.2\t1\t729 54\n",
      "72\t793\t0.2\t1\t745 48\n",
      "73\t805\t0.2\t1\t752 53\n",
      "74\t823\t0.2\t1\t753 70\n",
      "75\t771\t0.2\t1\t720 51\n",
      "76\t774\t0.2\t1\t743 31\n",
      "77\t768\t0.2\t1\t737 31\n",
      "78\t755\t0.2\t1\t698 57\n",
      "79\t787\t0.2\t1\t744 43\n",
      "80\t738\t0.2\t1\t688 50\n",
      "81\t747\t0.2\t1\t703 44\n",
      "82\t764\t0.2\t1\t720 44\n",
      "83\t790\t0.2\t1\t737 53\n",
      "84\t789\t0.2\t1\t736 53\n",
      "85\t734\t0.2\t1\t681 53\n",
      "86\t835\t0.2\t1\t785 50\n",
      "87\t835\t0.2\t1\t781 54\n",
      "88\t789\t0.2\t1\t748 41\n",
      "89\t822\t0.2\t1\t776 46\n",
      "90\t871\t0.2\t1\t815 56\n",
      "91\t849\t0.2\t1\t794 55\n",
      "92\t817\t0.2\t1\t768 49\n",
      "93\t829\t0.2\t1\t779 50\n",
      "94\t850\t0.2\t1\t801 49\n",
      "95\t883\t0.2\t1\t826 57\n",
      "96\t853\t0.2\t1\t798 55\n",
      "97\t851\t0.2\t1\t798 53\n",
      "98\t838\t0.2\t1\t783 55\n",
      "99\t882\t0.2\t1\t828 54\n",
      "100\t859\t0.2\t1\t813 46\n",
      "101\t910\t0.2\t1\t876 34\n",
      "102\t818\t0.2\t1\t769 49\n",
      "103\t850\t0.2\t1\t807 43\n",
      "104\t818\t0.2\t1\t776 42\n",
      "105\t816\t0.2\t1\t759 57\n",
      "106\t856\t0.2\t1\t820 36\n",
      "107\t773\t0.2\t1\t716 57\n",
      "108\t861\t0.2\t1\t804 57\n",
      "109\t867\t0.2\t1\t822 45\n",
      "110\t867\t0.2\t1\t818 49\n",
      "111\t881\t0.2\t1\t833 48\n",
      "112\t864\t0.2\t1\t828 36\n",
      "113\t839\t0.2\t1\t807 32\n",
      "114\t888\t0.2\t1\t832 56\n",
      "115\t890\t0.2\t1\t846 44\n",
      "116\t900\t0.2\t1\t865 35\n",
      "117\t854\t0.2\t1\t810 44\n",
      "118\t839\t0.2\t1\t803 36\n",
      "119\t836\t0.2\t1\t804 32\n",
      "120\t898\t0.2\t1\t843 55\n",
      "121\t869\t0.2\t1\t833 36\n",
      "122\t885\t0.2\t1\t832 53\n",
      "123\t883\t0.2\t1\t838 45\n",
      "124\t863\t0.2\t1\t813 50\n",
      "125\t847\t0.2\t1\t817 30\n",
      "126\t802\t0.2\t1\t771 31\n",
      "127\t875\t0.2\t1\t827 48\n",
      "128\t860\t0.2\t1\t814 46\n",
      "129\t893\t0.2\t1\t850 43\n",
      "130\t847\t0.2\t1\t802 45\n",
      "131\t926\t0.2\t1\t890 36\n",
      "132\t938\t0.2\t1\t893 45\n",
      "133\t928\t0.2\t1\t874 54\n",
      "134\t829\t0.2\t1\t799 30\n",
      "135\t858\t0.2\t1\t824 34\n",
      "136\t867\t0.2\t1\t819 48\n",
      "137\t789\t0.2\t1\t758 31\n",
      "138\t831\t0.2\t1\t801 30\n",
      "139\t759\t0.2\t1\t730 29\n",
      "140\t853\t0.2\t1\t809 44\n",
      "141\t824\t0.2\t1\t790 34\n",
      "142\t789\t0.2\t1\t744 45\n",
      "143\t740\t0.2\t1\t702 38\n",
      "144\t886\t0.2\t1\t832 54\n",
      "145\t841\t0.2\t1\t794 47\n",
      "146\t858\t0.2\t1\t809 49\n",
      "147\t710\t0.2\t1\t686 24\n",
      "148\t734\t0.2\t1\t705 29\n",
      "149\t618\t0.2\t1\t589 29\n",
      "150\t529\t0.2\t1\t487 42\n",
      "151\t165\t0.2\t1\t129 36\n",
      "\n",
      "\n",
      "RUN STATISTICS FOR INPUT FILE: /home/sam/data/geoduck_illumina/NR021_S8_L001_R2_001.fastq.gz\n",
      "=============================================\n",
      "4037578 sequences processed in total\n",
      "The length threshold of paired-end sequences gets evaluated later on (in the validation step)\n",
      "\n",
      "Validate paired-end files NR021_S8_L001_R1_001_trimmed.fq.gz and NR021_S8_L001_R2_001_trimmed.fq.gz\n",
      "file_1: NR021_S8_L001_R1_001_trimmed.fq.gz, file_2: NR021_S8_L001_R2_001_trimmed.fq.gz\n",
      "\n",
      "\n",
      ">>>>> Now validing the length of the 2 paired-end infiles: NR021_S8_L001_R1_001_trimmed.fq.gz and NR021_S8_L001_R2_001_trimmed.fq.gz <<<<<\n",
      "Writing validated paired-end read 1 reads to NR021_S8_L001_R1_001_val_1.fq.gz\n",
      "Writing validated paired-end read 2 reads to NR021_S8_L001_R2_001_val_2.fq.gz\n",
      "\n",
      "Total number of sequences analysed: 4037578\n",
      "\n",
      "Number of sequence pairs removed because at least one read was shorter than the length cutoff (20 bp): 41263 (1.02%)\n",
      "\n",
      "Deleting both intermediate output files NR021_S8_L001_R1_001_trimmed.fq.gz and NR021_S8_L001_R2_001_trimmed.fq.gz\n",
      "\n",
      "====================================================================================================\n",
      "\n",
      "Writing report to '/home/sam/data/geoduck_illumina/trimmed/NR021_S8_L002_R1_001.fastq.gz_trimming_report.txt'\n",
      "\n",
      "SUMMARISING RUN PARAMETERS\n",
      "==========================\n",
      "Input filename: /home/sam/data/geoduck_illumina/NR021_S8_L002_R1_001.fastq.gz\n",
      "Trimming mode: paired-end\n",
      "Trim Galore version: 0.4.4\n",
      "Cutadapt version: 1.9.1\n",
      "Quality Phred score cutoff: 20\n",
      "Quality encoding type selected: ASCII+33\n",
      "Adapter sequence: 'CTGTCTCTTATA' (Nextera Transposase sequence; auto-detected)\n",
      "Maximum trimming error rate: 0.1 (default)\n",
      "Minimum required adapter overlap (stringency): 1 bp\n",
      "Minimum required sequence length for both reads before a sequence pair gets removed: 20 bp\n",
      "Output file(s) will be GZIP compressed\n",
      "\n",
      "Writing final adapter and quality trimmed output to NR021_S8_L002_R1_001_trimmed.fq.gz\n",
      "\n",
      "\n",
      "  >>> Now performing quality (cutoff 20) and adapter trimming in a single pass for the adapter sequence: 'CTGTCTCTTATA' from file /home/sam/data/geoduck_illumina/NR021_S8_L002_R1_001.fastq.gz <<< \n",
      "This is cutadapt 1.9.1 with Python 2.7.12\n",
      "Command line parameters: -f fastq -e 0.1 -q 20 -O 1 -a CTGTCTCTTATA /home/sam/data/geoduck_illumina/NR021_S8_L002_R1_001.fastq.gz\n",
      "Trimming 1 adapter with at most 10.0% errors in single-end mode ...\n",
      "Finished in 91.25 s (26 us/read; 2.35 M reads/minute).\n",
      "\n",
      "=== Summary ===\n",
      "\n",
      "Total reads processed:               3,575,592\n",
      "Reads with adapters:                 1,134,870 (31.7%)\n",
      "Reads written (passing filters):     3,575,592 (100.0%)\n",
      "\n",
      "Total basepairs processed:   539,914,392 bp\n",
      "Quality-trimmed:               1,205,207 bp (0.2%)\n",
      "Total written (filtered):    526,158,055 bp (97.5%)\n",
      "\n",
      "=== Adapter 1 ===\n",
      "\n",
      "Sequence: CTGTCTCTTATA; Type: regular 3'; Length: 12; Trimmed: 1134870 times.\n",
      "\n",
      "No. of allowed errors:\n",
      "0-9 bp: 0; 10-12 bp: 1\n",
      "\n",
      "Bases preceding removed adapters:\n",
      "  A: 29.0%\n",
      "  C: 19.1%\n",
      "  G: 15.7%\n",
      "  T: 36.2%\n",
      "  none/other: 0.0%\n",
      "\n",
      "Overview of removed sequences\n",
      "length\tcount\texpect\tmax.err\terror counts\n",
      "1\t737588\t893898.0\t0\t737588\n",
      "2\t183929\t223474.5\t0\t183929\n",
      "3\t60993\t55868.6\t0\t60993\n",
      "4\t13612\t13967.2\t0\t13612\n",
      "5\t3694\t3491.8\t0\t3694\n",
      "6\t1512\t872.9\t0\t1512\n",
      "7\t839\t218.2\t0\t839\n",
      "8\t820\t54.6\t0\t820\n",
      "9\t669\t13.6\t0\t597 72\n",
      "10\t771\t3.4\t1\t639 132\n",
      "11\t767\t0.9\t1\t709 58\n",
      "12\t639\t0.2\t1\t594 45\n",
      "13\t745\t0.2\t1\t709 36\n",
      "14\t660\t0.2\t1\t622 38\n",
      "15\t820\t0.2\t1\t754 66\n",
      "16\t888\t0.2\t1\t819 69\n",
      "17\t579\t0.2\t1\t541 38\n",
      "18\t703\t0.2\t1\t666 37\n",
      "19\t712\t0.2\t1\t665 47\n",
      "20\t723\t0.2\t1\t676 47\n",
      "21\t803\t0.2\t1\t750 53\n",
      "22\t839\t0.2\t1\t796 43\n",
      "23\t669\t0.2\t1\t620 49\n",
      "24\t792\t0.2\t1\t743 49\n",
      "25\t677\t0.2\t1\t622 55\n",
      "26\t809\t0.2\t1\t748 61\n",
      "27\t824\t0.2\t1\t748 76\n",
      "28\t792\t0.2\t1\t730 62\n",
      "29\t732\t0.2\t1\t681 51\n",
      "30\t858\t0.2\t1\t787 71\n",
      "31\t801\t0.2\t1\t732 69\n",
      "32\t803\t0.2\t1\t749 54\n",
      "33\t804\t0.2\t1\t746 58\n",
      "34\t792\t0.2\t1\t741 51\n",
      "35\t753\t0.2\t1\t700 53\n",
      "36\t815\t0.2\t1\t765 50\n",
      "37\t765\t0.2\t1\t713 52\n",
      "38\t787\t0.2\t1\t745 42\n",
      "39\t819\t0.2\t1\t780 39\n",
      "40\t803\t0.2\t1\t761 42\n",
      "41\t773\t0.2\t1\t731 42\n",
      "42\t790\t0.2\t1\t751 39\n",
      "43\t824\t0.2\t1\t764 60\n",
      "44\t779\t0.2\t1\t734 45\n",
      "45\t864\t0.2\t1\t786 78\n",
      "46\t848\t0.2\t1\t791 57\n",
      "47\t771\t0.2\t1\t726 45\n",
      "48\t867\t0.2\t1\t815 52\n",
      "49\t822\t0.2\t1\t777 45\n",
      "50\t777\t0.2\t1\t725 52\n",
      "51\t801\t0.2\t1\t751 50\n",
      "52\t794\t0.2\t1\t740 54\n",
      "53\t880\t0.2\t1\t841 39\n",
      "54\t891\t0.2\t1\t842 49\n",
      "55\t771\t0.2\t1\t730 41\n",
      "56\t822\t0.2\t1\t767 55\n",
      "57\t865\t0.2\t1\t802 63\n",
      "58\t870\t0.2\t1\t820 50\n",
      "59\t898\t0.2\t1\t848 50\n",
      "60\t905\t0.2\t1\t864 41\n",
      "61\t937\t0.2\t1\t890 47\n",
      "62\t900\t0.2\t1\t844 56\n",
      "63\t844\t0.2\t1\t795 49\n",
      "64\t858\t0.2\t1\t807 51\n",
      "65\t871\t0.2\t1\t808 63\n",
      "66\t890\t0.2\t1\t835 55\n",
      "67\t825\t0.2\t1\t777 48\n",
      "68\t856\t0.2\t1\t813 43\n",
      "69\t857\t0.2\t1\t814 43\n",
      "70\t954\t0.2\t1\t910 44\n",
      "71\t937\t0.2\t1\t890 47\n",
      "72\t914\t0.2\t1\t862 52\n",
      "73\t889\t0.2\t1\t839 50\n",
      "74\t943\t0.2\t1\t891 52\n",
      "75\t882\t0.2\t1\t829 53\n",
      "76\t891\t0.2\t1\t850 41\n",
      "77\t877\t0.2\t1\t834 43\n",
      "78\t930\t0.2\t1\t883 47\n",
      "79\t933\t0.2\t1\t896 37\n",
      "80\t976\t0.2\t1\t914 62\n",
      "81\t949\t0.2\t1\t898 51\n",
      "82\t941\t0.2\t1\t885 56\n",
      "83\t988\t0.2\t1\t926 62\n",
      "84\t1007\t0.2\t1\t963 44\n",
      "85\t947\t0.2\t1\t885 62\n",
      "86\t965\t0.2\t1\t913 52\n",
      "87\t953\t0.2\t1\t906 47\n",
      "88\t1015\t0.2\t1\t967 48\n",
      "89\t1010\t0.2\t1\t946 64\n",
      "90\t955\t0.2\t1\t895 60\n",
      "91\t1034\t0.2\t1\t996 38\n",
      "92\t1010\t0.2\t1\t980 30\n",
      "93\t971\t0.2\t1\t921 50\n",
      "94\t999\t0.2\t1\t947 52\n",
      "95\t966\t0.2\t1\t918 48\n",
      "96\t947\t0.2\t1\t884 63\n",
      "97\t1009\t0.2\t1\t973 36\n",
      "98\t1061\t0.2\t1\t1008 53\n",
      "99\t1027\t0.2\t1\t981 46\n",
      "100\t1076\t0.2\t1\t1034 42\n",
      "101\t1016\t0.2\t1\t965 51\n",
      "102\t1009\t0.2\t1\t953 56\n",
      "103\t978\t0.2\t1\t946 32\n",
      "104\t1026\t0.2\t1\t997 29\n",
      "105\t1086\t0.2\t1\t1048 38\n",
      "106\t976\t0.2\t1\t899 77\n",
      "107\t966\t0.2\t1\t931 35\n",
      "108\t1087\t0.2\t1\t1029 58\n",
      "109\t1087\t0.2\t1\t1030 57\n",
      "110\t1058\t0.2\t1\t1018 40\n",
      "111\t1092\t0.2\t1\t1055 37\n",
      "112\t1090\t0.2\t1\t1040 50\n",
      "113\t1034\t0.2\t1\t988 46\n",
      "114\t1057\t0.2\t1\t1021 36\n",
      "115\t1027\t0.2\t1\t985 42\n",
      "116\t1067\t0.2\t1\t1025 42\n",
      "117\t1129\t0.2\t1\t1071 58\n",
      "118\t1083\t0.2\t1\t1033 50\n",
      "119\t1097\t0.2\t1\t1040 57\n",
      "120\t1119\t0.2\t1\t1074 45\n",
      "121\t1090\t0.2\t1\t1051 39\n",
      "122\t1096\t0.2\t1\t1059 37\n",
      "123\t1168\t0.2\t1\t1117 51\n",
      "124\t1147\t0.2\t1\t1074 73\n",
      "125\t1113\t0.2\t1\t1058 55\n",
      "126\t1099\t0.2\t1\t1065 34\n",
      "127\t1092\t0.2\t1\t1052 40\n",
      "128\t1091\t0.2\t1\t1054 37\n",
      "129\t1107\t0.2\t1\t1071 36\n",
      "130\t1090\t0.2\t1\t1043 47\n",
      "131\t1141\t0.2\t1\t1101 40\n",
      "132\t1145\t0.2\t1\t1108 37\n",
      "133\t1159\t0.2\t1\t1135 24\n",
      "134\t1086\t0.2\t1\t1054 32\n",
      "135\t1084\t0.2\t1\t1042 42\n",
      "136\t977\t0.2\t1\t957 20\n",
      "137\t1030\t0.2\t1\t1003 27\n",
      "138\t1018\t0.2\t1\t977 41\n",
      "139\t1007\t0.2\t1\t978 29\n",
      "140\t1075\t0.2\t1\t1032 43\n",
      "141\t1106\t0.2\t1\t1075 31\n",
      "142\t1113\t0.2\t1\t1086 27\n",
      "143\t1085\t0.2\t1\t1022 63\n",
      "144\t1030\t0.2\t1\t984 46\n",
      "145\t1059\t0.2\t1\t1022 37\n",
      "146\t1123\t0.2\t1\t1066 57\n",
      "147\t925\t0.2\t1\t899 26\n",
      "148\t918\t0.2\t1\t883 35\n",
      "149\t805\t0.2\t1\t784 21\n",
      "150\t758\t0.2\t1\t733 25\n",
      "151\t243\t0.2\t1\t195 48\n",
      "\n",
      "\n",
      "RUN STATISTICS FOR INPUT FILE: /home/sam/data/geoduck_illumina/NR021_S8_L002_R1_001.fastq.gz\n",
      "=============================================\n",
      "3575592 sequences processed in total\n",
      "The length threshold of paired-end sequences gets evaluated later on (in the validation step)\n",
      "\n",
      "Writing report to '/home/sam/data/geoduck_illumina/trimmed/NR021_S8_L002_R2_001.fastq.gz_trimming_report.txt'\n",
      "\n",
      "SUMMARISING RUN PARAMETERS\n",
      "==========================\n",
      "Input filename: /home/sam/data/geoduck_illumina/NR021_S8_L002_R2_001.fastq.gz\n",
      "Trimming mode: paired-end\n",
      "Trim Galore version: 0.4.4\n",
      "Cutadapt version: 1.9.1\n",
      "Quality Phred score cutoff: 20\n",
      "Quality encoding type selected: ASCII+33\n",
      "Adapter sequence: 'CTGTCTCTTATA' (Nextera Transposase sequence; auto-detected)\n",
      "Maximum trimming error rate: 0.1 (default)\n",
      "Minimum required adapter overlap (stringency): 1 bp\n",
      "Minimum required sequence length for both reads before a sequence pair gets removed: 20 bp\n",
      "Output file(s) will be GZIP compressed\n",
      "\n",
      "Writing final adapter and quality trimmed output to NR021_S8_L002_R2_001_trimmed.fq.gz\n",
      "\n",
      "\n",
      "  >>> Now performing quality (cutoff 20) and adapter trimming in a single pass for the adapter sequence: 'CTGTCTCTTATA' from file /home/sam/data/geoduck_illumina/NR021_S8_L002_R2_001.fastq.gz <<< \n",
      "This is cutadapt 1.9.1 with Python 2.7.12\n",
      "Command line parameters: -f fastq -e 0.1 -q 20 -O 1 -a CTGTCTCTTATA /home/sam/data/geoduck_illumina/NR021_S8_L002_R2_001.fastq.gz\n",
      "Trimming 1 adapter with at most 10.0% errors in single-end mode ...\n",
      "Finished in 89.37 s (25 us/read; 2.40 M reads/minute).\n",
      "\n",
      "=== Summary ===\n",
      "\n",
      "Total reads processed:               3,575,592\n",
      "Reads with adapters:                   806,492 (22.6%)\n",
      "Reads written (passing filters):     3,575,592 (100.0%)\n",
      "\n",
      "Total basepairs processed:   539,914,392 bp\n",
      "Quality-trimmed:               2,894,800 bp (0.5%)\n",
      "Total written (filtered):    526,617,946 bp (97.5%)\n",
      "\n",
      "=== Adapter 1 ===\n",
      "\n",
      "Sequence: CTGTCTCTTATA; Type: regular 3'; Length: 12; Trimmed: 806492 times.\n",
      "\n",
      "No. of allowed errors:\n",
      "0-9 bp: 0; 10-12 bp: 1\n",
      "\n",
      "Bases preceding removed adapters:\n",
      "  A: 26.3%\n",
      "  C: 20.1%\n",
      "  G: 21.1%\n",
      "  T: 32.5%\n",
      "  none/other: 0.0%\n",
      "\n",
      "Overview of removed sequences\n",
      "length\tcount\texpect\tmax.err\terror counts\n",
      "1\t513615\t893898.0\t0\t513615\n",
      "2\t123311\t223474.5\t0\t123311\n",
      "3\t38272\t55868.6\t0\t38272\n",
      "4\t11752\t13967.2\t0\t11752\n",
      "5\t2698\t3491.8\t0\t2698\n",
      "6\t1276\t872.9\t0\t1276\n",
      "7\t810\t218.2\t0\t810\n",
      "8\t687\t54.6\t0\t687\n",
      "9\t642\t13.6\t0\t569 73\n",
      "10\t713\t3.4\t1\t589 124\n",
      "11\t731\t0.9\t1\t650 81\n",
      "12\t657\t0.2\t1\t603 54\n",
      "13\t641\t0.2\t1\t592 49\n",
      "14\t613\t0.2\t1\t571 42\n",
      "15\t760\t0.2\t1\t694 66\n",
      "16\t804\t0.2\t1\t752 52\n",
      "17\t494\t0.2\t1\t454 40\n",
      "18\t688\t0.2\t1\t627 61\n",
      "19\t692\t0.2\t1\t637 55\n",
      "20\t636\t0.2\t1\t603 33\n",
      "21\t681\t0.2\t1\t633 48\n",
      "22\t703\t0.2\t1\t650 53\n",
      "23\t609\t0.2\t1\t572 37\n",
      "24\t715\t0.2\t1\t665 50\n",
      "25\t649\t0.2\t1\t601 48\n",
      "26\t759\t0.2\t1\t698 61\n",
      "27\t730\t0.2\t1\t686 44\n",
      "28\t689\t0.2\t1\t646 43\n",
      "29\t697\t0.2\t1\t660 37\n",
      "30\t720\t0.2\t1\t688 32\n",
      "31\t731\t0.2\t1\t678 53\n",
      "32\t701\t0.2\t1\t648 53\n",
      "33\t683\t0.2\t1\t645 38\n",
      "34\t699\t0.2\t1\t644 55\n",
      "35\t660\t0.2\t1\t624 36\n",
      "36\t748\t0.2\t1\t697 51\n",
      "37\t646\t0.2\t1\t605 41\n",
      "38\t772\t0.2\t1\t721 51\n",
      "39\t744\t0.2\t1\t688 56\n",
      "40\t734\t0.2\t1\t688 46\n",
      "41\t733\t0.2\t1\t682 51\n",
      "42\t731\t0.2\t1\t683 48\n",
      "43\t723\t0.2\t1\t682 41\n",
      "44\t694\t0.2\t1\t648 46\n",
      "45\t738\t0.2\t1\t696 42\n",
      "46\t761\t0.2\t1\t707 54\n",
      "47\t786\t0.2\t1\t745 41\n",
      "48\t821\t0.2\t1\t775 46\n",
      "49\t774\t0.2\t1\t735 39\n",
      "50\t803\t0.2\t1\t751 52\n",
      "51\t739\t0.2\t1\t700 39\n",
      "52\t741\t0.2\t1\t706 35\n",
      "53\t788\t0.2\t1\t748 40\n",
      "54\t731\t0.2\t1\t682 49\n",
      "55\t751\t0.2\t1\t704 47\n",
      "56\t761\t0.2\t1\t704 57\n",
      "57\t795\t0.2\t1\t752 43\n",
      "58\t779\t0.2\t1\t726 53\n",
      "59\t787\t0.2\t1\t739 48\n",
      "60\t796\t0.2\t1\t735 61\n",
      "61\t795\t0.2\t1\t766 29\n",
      "62\t736\t0.2\t1\t695 41\n",
      "63\t783\t0.2\t1\t741 42\n",
      "64\t726\t0.2\t1\t684 42\n",
      "65\t774\t0.2\t1\t728 46\n",
      "66\t795\t0.2\t1\t749 46\n",
      "67\t748\t0.2\t1\t703 45\n",
      "68\t810\t0.2\t1\t777 33\n",
      "69\t810\t0.2\t1\t769 41\n",
      "70\t820\t0.2\t1\t771 49\n",
      "71\t802\t0.2\t1\t765 37\n",
      "72\t867\t0.2\t1\t827 40\n",
      "73\t830\t0.2\t1\t794 36\n",
      "74\t822\t0.2\t1\t775 47\n",
      "75\t812\t0.2\t1\t768 44\n",
      "76\t843\t0.2\t1\t795 48\n",
      "77\t780\t0.2\t1\t757 23\n",
      "78\t820\t0.2\t1\t769 51\n",
      "79\t866\t0.2\t1\t819 47\n",
      "80\t883\t0.2\t1\t846 37\n",
      "81\t766\t0.2\t1\t731 35\n",
      "82\t820\t0.2\t1\t770 50\n",
      "83\t794\t0.2\t1\t753 41\n",
      "84\t822\t0.2\t1\t788 34\n",
      "85\t843\t0.2\t1\t801 42\n",
      "86\t873\t0.2\t1\t840 33\n",
      "87\t842\t0.2\t1\t810 32\n",
      "88\t816\t0.2\t1\t781 35\n",
      "89\t838\t0.2\t1\t807 31\n",
      "90\t857\t0.2\t1\t819 38\n",
      "91\t883\t0.2\t1\t841 42\n",
      "92\t850\t0.2\t1\t805 45\n",
      "93\t843\t0.2\t1\t794 49\n",
      "94\t859\t0.2\t1\t812 47\n",
      "95\t893\t0.2\t1\t851 42\n",
      "96\t863\t0.2\t1\t819 44\n",
      "97\t875\t0.2\t1\t840 35\n",
      "98\t863\t0.2\t1\t819 44\n",
      "99\t902\t0.2\t1\t867 35\n",
      "100\t871\t0.2\t1\t828 43\n",
      "101\t830\t0.2\t1\t797 33\n",
      "102\t858\t0.2\t1\t828 30\n",
      "103\t860\t0.2\t1\t819 41\n",
      "104\t887\t0.2\t1\t853 34\n",
      "105\t842\t0.2\t1\t805 37\n",
      "106\t932\t0.2\t1\t899 33\n",
      "107\t844\t0.2\t1\t810 34\n",
      "108\t882\t0.2\t1\t840 42\n",
      "109\t903\t0.2\t1\t869 34\n",
      "110\t866\t0.2\t1\t819 47\n",
      "111\t900\t0.2\t1\t864 36\n",
      "112\t927\t0.2\t1\t896 31\n",
      "113\t947\t0.2\t1\t910 37\n",
      "114\t982\t0.2\t1\t936 46\n",
      "115\t892\t0.2\t1\t859 33\n",
      "116\t923\t0.2\t1\t890 33\n",
      "117\t914\t0.2\t1\t877 37\n",
      "118\t857\t0.2\t1\t827 30\n",
      "119\t881\t0.2\t1\t849 32\n",
      "120\t970\t0.2\t1\t937 33\n",
      "121\t906\t0.2\t1\t868 38\n",
      "122\t917\t0.2\t1\t869 48\n",
      "123\t896\t0.2\t1\t858 38\n",
      "124\t850\t0.2\t1\t815 35\n",
      "125\t946\t0.2\t1\t912 34\n",
      "126\t898\t0.2\t1\t853 45\n",
      "127\t901\t0.2\t1\t866 35\n",
      "128\t935\t0.2\t1\t906 29\n",
      "129\t884\t0.2\t1\t855 29\n",
      "130\t913\t0.2\t1\t885 28\n",
      "131\t962\t0.2\t1\t930 32\n",
      "132\t945\t0.2\t1\t909 36\n",
      "133\t980\t0.2\t1\t959 21\n",
      "134\t857\t0.2\t1\t824 33\n",
      "135\t828\t0.2\t1\t811 17\n",
      "136\t884\t0.2\t1\t861 23\n",
      "137\t802\t0.2\t1\t777 25\n",
      "138\t801\t0.2\t1\t781 20\n",
      "139\t777\t0.2\t1\t749 28\n",
      "140\t850\t0.2\t1\t830 20\n",
      "141\t856\t0.2\t1\t822 34\n",
      "142\t792\t0.2\t1\t766 26\n",
      "143\t843\t0.2\t1\t812 31\n",
      "144\t860\t0.2\t1\t834 26\n",
      "145\t878\t0.2\t1\t848 30\n",
      "146\t887\t0.2\t1\t852 35\n",
      "147\t764\t0.2\t1\t737 27\n",
      "148\t737\t0.2\t1\t705 32\n",
      "149\t614\t0.2\t1\t587 27\n",
      "150\t595\t0.2\t1\t565 30\n",
      "151\t148\t0.2\t1\t106 42\n",
      "\n",
      "\n",
      "RUN STATISTICS FOR INPUT FILE: /home/sam/data/geoduck_illumina/NR021_S8_L002_R2_001.fastq.gz\n",
      "=============================================\n",
      "3575592 sequences processed in total\n",
      "The length threshold of paired-end sequences gets evaluated later on (in the validation step)\n",
      "\n",
      "Validate paired-end files NR021_S8_L002_R1_001_trimmed.fq.gz and NR021_S8_L002_R2_001_trimmed.fq.gz\n",
      "file_1: NR021_S8_L002_R1_001_trimmed.fq.gz, file_2: NR021_S8_L002_R2_001_trimmed.fq.gz\n",
      "\n",
      "\n",
      ">>>>> Now validing the length of the 2 paired-end infiles: NR021_S8_L002_R1_001_trimmed.fq.gz and NR021_S8_L002_R2_001_trimmed.fq.gz <<<<<\n",
      "Writing validated paired-end read 1 reads to NR021_S8_L002_R1_001_val_1.fq.gz\n",
      "Writing validated paired-end read 2 reads to NR021_S8_L002_R2_001_val_2.fq.gz\n",
      "\n",
      "Total number of sequences analysed: 3575592\n",
      "\n",
      "Number of sequence pairs removed because at least one read was shorter than the length cutoff (20 bp): 43175 (1.21%)\n",
      "\n",
      "Deleting both intermediate output files NR021_S8_L002_R1_001_trimmed.fq.gz and NR021_S8_L002_R2_001_trimmed.fq.gz\n",
      "\n",
      "====================================================================================================\n",
      "\n",
      "\n",
      "real\t5246m24.934s\n",
      "user\t11047m42.956s\n",
      "sys\t242m23.672s\n"
     ]
    }
   ],
   "source": [
    "%%bash\n",
    "time /home/shared/trimgalore/trim_galore \\\n",
    "--paired \\\n",
    "--output_dir /home/sam/data/geoduck_illumina/trimmed/ \\\n",
    "/home/sam/data/geoduck_illumina/AD002_S9_L001_R1_001.fastq.gz \\\n",
    "/home/sam/data/geoduck_illumina/AD002_S9_L001_R2_001.fastq.gz \\\n",
    "/home/sam/data/geoduck_illumina/AD002_S9_L002_R1_001.fastq.gz \\\n",
    "/home/sam/data/geoduck_illumina/AD002_S9_L002_R2_001.fastq.gz \\\n",
    "/home/sam/data/geoduck_illumina/NR005_S4_L001_R1_001.fastq.gz \\\n",
    "/home/sam/data/geoduck_illumina/NR005_S4_L001_R2_001.fastq.gz \\\n",
    "/home/sam/data/geoduck_illumina/NR005_S4_L002_R1_001.fastq.gz \\\n",
    "/home/sam/data/geoduck_illumina/NR005_S4_L002_R2_001.fastq.gz \\\n",
    "/home/sam/data/geoduck_illumina/NR006_S3_L001_R1_001.fastq.gz \\\n",
    "/home/sam/data/geoduck_illumina/NR006_S3_L001_R2_001.fastq.gz \\\n",
    "/home/sam/data/geoduck_illumina/NR006_S3_L002_R1_001.fastq.gz \\\n",
    "/home/sam/data/geoduck_illumina/NR006_S3_L002_R2_001.fastq.gz \\\n",
    "/home/sam/data/geoduck_illumina/NR012_S1_L001_R1_001.fastq.gz \\\n",
    "/home/sam/data/geoduck_illumina/NR012_S1_L001_R2_001.fastq.gz \\\n",
    "/home/sam/data/geoduck_illumina/NR012_S1_L002_R1_001.fastq.gz \\\n",
    "/home/sam/data/geoduck_illumina/NR012_S1_L002_R2_001.fastq.gz \\\n",
    "/home/sam/data/geoduck_illumina/NR013_AD013_S2_L001_R1_001.fastq.gz \\\n",
    "/home/sam/data/geoduck_illumina/NR013_AD013_S2_L001_R2_001.fastq.gz \\\n",
    "/home/sam/data/geoduck_illumina/NR013_AD013_S2_L002_R1_001.fastq.gz \\\n",
    "/home/sam/data/geoduck_illumina/NR013_AD013_S2_L002_R2_001.fastq.gz \\\n",
    "/home/sam/data/geoduck_illumina/NR014_AD014_S5_L001_R1_001.fastq.gz \\\n",
    "/home/sam/data/geoduck_illumina/NR014_AD014_S5_L001_R2_001.fastq.gz \\\n",
    "/home/sam/data/geoduck_illumina/NR014_AD014_S5_L002_R1_001.fastq.gz \\\n",
    "/home/sam/data/geoduck_illumina/NR014_AD014_S5_L002_R2_001.fastq.gz \\\n",
    "/home/sam/data/geoduck_illumina/NR015_AD015_S6_L001_R1_001.fastq.gz \\\n",
    "/home/sam/data/geoduck_illumina/NR015_AD015_S6_L001_R2_001.fastq.gz \\\n",
    "/home/sam/data/geoduck_illumina/NR015_AD015_S6_L002_R1_001.fastq.gz \\\n",
    "/home/sam/data/geoduck_illumina/NR015_AD015_S6_L002_R2_001.fastq.gz \\\n",
    "/home/sam/data/geoduck_illumina/NR019_S7_L001_R1_001.fastq.gz \\\n",
    "/home/sam/data/geoduck_illumina/NR019_S7_L001_R2_001.fastq.gz \\\n",
    "/home/sam/data/geoduck_illumina/NR019_S7_L002_R1_001.fastq.gz \\\n",
    "/home/sam/data/geoduck_illumina/NR019_S7_L002_R2_001.fastq.gz \\\n",
    "/home/sam/data/geoduck_illumina/NR021_S8_L001_R1_001.fastq.gz \\\n",
    "/home/sam/data/geoduck_illumina/NR021_S8_L001_R2_001.fastq.gz \\\n",
    "/home/sam/data/geoduck_illumina/NR021_S8_L002_R1_001.fastq.gz \\\n",
    "/home/sam/data/geoduck_illumina/NR021_S8_L002_R2_001.fastq.gz"
   ]
  },
  {
   "cell_type": "code",
   "execution_count": 20,
   "metadata": {
    "collapsed": true
   },
   "outputs": [],
   "source": [
    "cat ~/.default-subject.mail | msmtp \"$EMAIL\""
   ]
  },
  {
   "cell_type": "code",
   "execution_count": 21,
   "metadata": {},
   "outputs": [
    {
     "name": "stdout",
     "output_type": "stream",
     "text": [
      "total 381G\n",
      "-rw-rw-r-- 1 sam sam 6.2K Jan 25 14:07 AD002_S9_L001_R1_001.fastq.gz_trimming_report.txt\n",
      "-rw-rw-r-- 1 sam sam 9.9G Jan 25 17:19 AD002_S9_L001_R1_001_val_1.fq.gz\n",
      "-rw-rw-r-- 1 sam sam 6.4K Jan 25 17:19 AD002_S9_L001_R2_001.fastq.gz_trimming_report.txt\n",
      "-rw-rw-r-- 1 sam sam  11G Jan 25 17:19 AD002_S9_L001_R2_001_val_2.fq.gz\n",
      "-rw-rw-r-- 1 sam sam 6.2K Jan 25 19:11 AD002_S9_L002_R1_001.fastq.gz_trimming_report.txt\n",
      "-rw-rw-r-- 1 sam sam  13G Jan 25 23:04 AD002_S9_L002_R1_001_val_1.fq.gz\n",
      "-rw-rw-r-- 1 sam sam 6.4K Jan 25 23:04 AD002_S9_L002_R2_001.fastq.gz_trimming_report.txt\n",
      "-rw-rw-r-- 1 sam sam  13G Jan 25 23:04 AD002_S9_L002_R2_001_val_2.fq.gz\n",
      "-rw-rw-r-- 1 sam sam 3.0K Jan 25 23:04 NR005_S4_L001_R1_001.fastq.gz_trimming_report.txt\n",
      "-rw-rw-r-- 1 sam sam 3.0M Jan 25 23:04 NR005_S4_L001_R1_001_val_1.fq.gz\n",
      "-rw-rw-r-- 1 sam sam 3.0K Jan 25 23:04 NR005_S4_L001_R2_001.fastq.gz_trimming_report.txt\n",
      "-rw-rw-r-- 1 sam sam 3.1M Jan 25 23:04 NR005_S4_L001_R2_001_val_2.fq.gz\n",
      "-rw-rw-r-- 1 sam sam 3.0K Jan 25 23:04 NR005_S4_L002_R1_001.fastq.gz_trimming_report.txt\n",
      "-rw-rw-r-- 1 sam sam 2.5M Jan 25 23:04 NR005_S4_L002_R1_001_val_1.fq.gz\n",
      "-rw-rw-r-- 1 sam sam 2.9K Jan 25 23:04 NR005_S4_L002_R2_001.fastq.gz_trimming_report.txt\n",
      "-rw-rw-r-- 1 sam sam 2.6M Jan 25 23:04 NR005_S4_L002_R2_001_val_2.fq.gz\n",
      "-rw-rw-r-- 1 sam sam 4.4K Jan 25 23:06 NR006_S3_L001_R1_001.fastq.gz_trimming_report.txt\n",
      "-rw-rw-r-- 1 sam sam 234M Jan 25 23:10 NR006_S3_L001_R1_001_val_1.fq.gz\n",
      "-rw-rw-r-- 1 sam sam 4.6K Jan 25 23:10 NR006_S3_L001_R2_001.fastq.gz_trimming_report.txt\n",
      "-rw-rw-r-- 1 sam sam 242M Jan 25 23:10 NR006_S3_L001_R2_001_val_2.fq.gz\n",
      "-rw-rw-r-- 1 sam sam 4.4K Jan 25 23:12 NR006_S3_L002_R1_001.fastq.gz_trimming_report.txt\n",
      "-rw-rw-r-- 1 sam sam 192M Jan 25 23:15 NR006_S3_L002_R1_001_val_1.fq.gz\n",
      "-rw-rw-r-- 1 sam sam 4.6K Jan 25 23:15 NR006_S3_L002_R2_001.fastq.gz_trimming_report.txt\n",
      "-rw-rw-r-- 1 sam sam 201M Jan 25 23:15 NR006_S3_L002_R2_001_val_2.fq.gz\n",
      "-rw-rw-r-- 1 sam sam 4.7K Jan 25 23:15 NR012_S1_L001_R1_001.fastq.gz_trimming_report.txt\n",
      "-rw-rw-r-- 1 sam sam  19M Jan 25 23:16 NR012_S1_L001_R1_001_val_1.fq.gz\n",
      "-rw-rw-r-- 1 sam sam 4.9K Jan 25 23:16 NR012_S1_L001_R2_001.fastq.gz_trimming_report.txt\n",
      "-rw-rw-r-- 1 sam sam  20M Jan 25 23:16 NR012_S1_L001_R2_001_val_2.fq.gz\n",
      "-rw-rw-r-- 1 sam sam 4.7K Jan 25 23:16 NR012_S1_L002_R1_001.fastq.gz_trimming_report.txt\n",
      "-rw-rw-r-- 1 sam sam  21M Jan 25 23:17 NR012_S1_L002_R1_001_val_1.fq.gz\n",
      "-rw-rw-r-- 1 sam sam 4.9K Jan 25 23:17 NR012_S1_L002_R2_001.fastq.gz_trimming_report.txt\n",
      "-rw-rw-r-- 1 sam sam  22M Jan 25 23:17 NR012_S1_L002_R2_001_val_2.fq.gz\n",
      "-rw-rw-r-- 1 sam sam 6.2K Jan 26 02:10 NR013_AD013_S2_L001_R1_001.fastq.gz_trimming_report.txt\n",
      "-rw-rw-r-- 1 sam sam  19G Jan 26 08:25 NR013_AD013_S2_L001_R1_001_val_1.fq.gz\n",
      "-rw-rw-r-- 1 sam sam 6.4K Jan 26 08:25 NR013_AD013_S2_L001_R2_001.fastq.gz_trimming_report.txt\n",
      "-rw-rw-r-- 1 sam sam  21G Jan 26 08:25 NR013_AD013_S2_L001_R2_001_val_2.fq.gz\n",
      "-rw-rw-r-- 1 sam sam 6.3K Jan 26 11:46 NR013_AD013_S2_L002_R1_001.fastq.gz_trimming_report.txt\n",
      "-rw-rw-r-- 1 sam sam  23G Jan 26 18:50 NR013_AD013_S2_L002_R1_001_val_1.fq.gz\n",
      "-rw-rw-r-- 1 sam sam 6.4K Jan 26 18:50 NR013_AD013_S2_L002_R2_001.fastq.gz_trimming_report.txt\n",
      "-rw-rw-r-- 1 sam sam  24G Jan 26 18:50 NR013_AD013_S2_L002_R2_001_val_2.fq.gz\n",
      "-rw-rw-r-- 1 sam sam 6.6K Jan 27 01:21 NR014_AD014_S5_L001_R1_001.fastq.gz_trimming_report.txt\n",
      "-rw-rw-r-- 1 sam sam  43G Jan 27 14:53 NR014_AD014_S5_L001_R1_001_val_1.fq.gz\n",
      "-rw-rw-r-- 1 sam sam 6.8K Jan 27 14:53 NR014_AD014_S5_L001_R2_001.fastq.gz_trimming_report.txt\n",
      "-rw-rw-r-- 1 sam sam  45G Jan 27 14:53 NR014_AD014_S5_L001_R2_001_val_2.fq.gz\n",
      "-rw-rw-r-- 1 sam sam 6.5K Jan 27 22:11 NR014_AD014_S5_L002_R1_001.fastq.gz_trimming_report.txt\n",
      "-rw-rw-r-- 1 sam sam  48G Jan 28 13:18 NR014_AD014_S5_L002_R1_001_val_1.fq.gz\n",
      "-rw-rw-r-- 1 sam sam 6.8K Jan 28 13:18 NR014_AD014_S5_L002_R2_001.fastq.gz_trimming_report.txt\n",
      "-rw-rw-r-- 1 sam sam  51G Jan 28 13:18 NR014_AD014_S5_L002_R2_001_val_2.fq.gz\n",
      "-rw-rw-r-- 1 sam sam 6.2K Jan 28 15:32 NR015_AD015_S6_L001_R1_001.fastq.gz_trimming_report.txt\n",
      "-rw-rw-r-- 1 sam sam  15G Jan 28 20:19 NR015_AD015_S6_L001_R1_001_val_1.fq.gz\n",
      "-rw-rw-r-- 1 sam sam 6.4K Jan 28 20:19 NR015_AD015_S6_L001_R2_001.fastq.gz_trimming_report.txt\n",
      "-rw-rw-r-- 1 sam sam  16G Jan 28 20:19 NR015_AD015_S6_L001_R2_001_val_2.fq.gz\n",
      "-rw-rw-r-- 1 sam sam 6.3K Jan 28 22:43 NR015_AD015_S6_L002_R1_001.fastq.gz_trimming_report.txt\n",
      "-rw-rw-r-- 1 sam sam  16G Jan 29 03:47 NR015_AD015_S6_L002_R1_001_val_1.fq.gz\n",
      "-rw-rw-r-- 1 sam sam 6.4K Jan 29 03:48 NR015_AD015_S6_L002_R2_001.fastq.gz_trimming_report.txt\n",
      "-rw-rw-r-- 1 sam sam  18G Jan 29 03:47 NR015_AD015_S6_L002_R2_001_val_2.fq.gz\n",
      "-rw-rw-r-- 1 sam sam 4.4K Jan 29 03:48 NR019_S7_L001_R1_001.fastq.gz_trimming_report.txt\n",
      "-rw-rw-r-- 1 sam sam 3.6M Jan 29 03:48 NR019_S7_L001_R1_001_val_1.fq.gz\n",
      "-rw-rw-r-- 1 sam sam 4.6K Jan 29 03:48 NR019_S7_L001_R2_001.fastq.gz_trimming_report.txt\n",
      "-rw-rw-r-- 1 sam sam 3.6M Jan 29 03:48 NR019_S7_L001_R2_001_val_2.fq.gz\n",
      "-rw-rw-r-- 1 sam sam 4.3K Jan 29 03:48 NR019_S7_L002_R1_001.fastq.gz_trimming_report.txt\n",
      "-rw-rw-r-- 1 sam sam 1.9M Jan 29 03:48 NR019_S7_L002_R1_001_val_1.fq.gz\n",
      "-rw-rw-r-- 1 sam sam 4.5K Jan 29 03:48 NR019_S7_L002_R2_001.fastq.gz_trimming_report.txt\n",
      "-rw-rw-r-- 1 sam sam 1.9M Jan 29 03:48 NR019_S7_L002_R2_001_val_2.fq.gz\n",
      "-rw-rw-r-- 1 sam sam 4.8K Jan 29 03:50 NR021_S8_L001_R1_001.fastq.gz_trimming_report.txt\n",
      "-rw-rw-r-- 1 sam sam 280M Jan 29 03:55 NR021_S8_L001_R1_001_val_1.fq.gz\n",
      "-rw-rw-r-- 1 sam sam 4.9K Jan 29 03:55 NR021_S8_L001_R2_001.fastq.gz_trimming_report.txt\n",
      "-rw-rw-r-- 1 sam sam 295M Jan 29 03:55 NR021_S8_L001_R2_001_val_2.fq.gz\n",
      "-rw-rw-r-- 1 sam sam 4.8K Jan 29 03:57 NR021_S8_L002_R1_001.fastq.gz_trimming_report.txt\n",
      "-rw-rw-r-- 1 sam sam 231M Jan 29 04:01 NR021_S8_L002_R1_001_val_1.fq.gz\n",
      "-rw-rw-r-- 1 sam sam 4.9K Jan 29 04:02 NR021_S8_L002_R2_001.fastq.gz_trimming_report.txt\n",
      "-rw-rw-r-- 1 sam sam 242M Jan 29 04:01 NR021_S8_L002_R2_001_val_2.fq.gz\n"
     ]
    }
   ],
   "source": [
    "%%bash\n",
    "ls -lh /home/sam/data/geoduck_illumina/trimmed/"
   ]
  },
  {
   "cell_type": "code",
   "execution_count": 22,
   "metadata": {},
   "outputs": [
    {
     "name": "stderr",
     "output_type": "stream",
     "text": [
      "bash: line 1: /home/shared/MultiQC/multiqc/multiqc: No such file or directory\n",
      "\n",
      "real\t0m0.001s\n",
      "user\t0m0.000s\n",
      "sys\t0m0.000s\n"
     ]
    }
   ],
   "source": [
    "%%bash\n",
    "time /home/shared/MultiQC/multiqc/multiqc /home/sam/data/geoduck_illumina/trimmed/\n",
    "cat ~/.default-subject.mail | msmtp \"$EMAIL\""
   ]
  },
  {
   "cell_type": "code",
   "execution_count": 23,
   "metadata": {},
   "outputs": [
    {
     "name": "stdout",
     "output_type": "stream",
     "text": [
      "Searching 72 files..\n"
     ]
    },
    {
     "name": "stderr",
     "output_type": "stream",
     "text": [
      "[INFO   ]         multiqc : This is MultiQC v1.5.dev0\n",
      "[INFO   ]         multiqc : Template    : default\n",
      "[INFO   ]         multiqc : Searching '/home/sam/data/geoduck_illumina/trimmed/'\n",
      "/home/shared/anaconda3/lib/python3.6/site-packages/matplotlib/font_manager.py:279: UserWarning: Matplotlib is building the font cache using fc-list. This may take a moment.\n",
      "  'Matplotlib is building the font cache using fc-list. '\n",
      "[INFO   ]        cutadapt : Found 36 reports\n",
      "[INFO   ]         multiqc : Compressing plot data\n",
      "[INFO   ]         multiqc : Report      : multiqc_report.html\n",
      "[INFO   ]         multiqc : Data        : multiqc_data\n",
      "[INFO   ]         multiqc : MultiQC complete\n",
      "\n",
      "real\t0m44.957s\n",
      "user\t0m32.260s\n",
      "sys\t0m0.656s\n"
     ]
    }
   ],
   "source": [
    "%%bash\n",
    "time /home/shared/anaconda3/bin/multiqc /home/sam/data/geoduck_illumina/trimmed/\n",
    "cat ~/.default-subject.mail | msmtp \"$EMAIL\""
   ]
  },
  {
   "cell_type": "code",
   "execution_count": 24,
   "metadata": {},
   "outputs": [
    {
     "name": "stdout",
     "output_type": "stream",
     "text": [
      "total 398753364\n",
      "-rw-rw-r-- 1 sam sam        6262 Jan 25 14:07 AD002_S9_L001_R1_001.fastq.gz_trimming_report.txt\n",
      "-rw-rw-r-- 1 sam sam 10588237401 Jan 25 17:19 AD002_S9_L001_R1_001_val_1.fq.gz\n",
      "-rw-rw-r-- 1 sam sam 11222294155 Jan 25 17:19 AD002_S9_L001_R2_001_val_2.fq.gz\n",
      "-rw-rw-r-- 1 sam sam        6479 Jan 25 17:19 AD002_S9_L001_R2_001.fastq.gz_trimming_report.txt\n",
      "-rw-rw-r-- 1 sam sam        6256 Jan 25 19:11 AD002_S9_L002_R1_001.fastq.gz_trimming_report.txt\n",
      "-rw-rw-r-- 1 sam sam 13062478451 Jan 25 23:04 AD002_S9_L002_R1_001_val_1.fq.gz\n",
      "-rw-rw-r-- 1 sam sam 13843776887 Jan 25 23:04 AD002_S9_L002_R2_001_val_2.fq.gz\n",
      "-rw-rw-r-- 1 sam sam        6480 Jan 25 23:04 AD002_S9_L002_R2_001.fastq.gz_trimming_report.txt\n",
      "-rw-rw-r-- 1 sam sam        3027 Jan 25 23:04 NR005_S4_L001_R1_001.fastq.gz_trimming_report.txt\n",
      "-rw-rw-r-- 1 sam sam     3141496 Jan 25 23:04 NR005_S4_L001_R1_001_val_1.fq.gz\n",
      "-rw-rw-r-- 1 sam sam     3245747 Jan 25 23:04 NR005_S4_L001_R2_001_val_2.fq.gz\n",
      "-rw-rw-r-- 1 sam sam        2992 Jan 25 23:04 NR005_S4_L001_R2_001.fastq.gz_trimming_report.txt\n",
      "-rw-rw-r-- 1 sam sam        2970 Jan 25 23:04 NR005_S4_L002_R1_001.fastq.gz_trimming_report.txt\n",
      "-rw-rw-r-- 1 sam sam     2574088 Jan 25 23:04 NR005_S4_L002_R1_001_val_1.fq.gz\n",
      "-rw-rw-r-- 1 sam sam     2660195 Jan 25 23:04 NR005_S4_L002_R2_001_val_2.fq.gz\n",
      "-rw-rw-r-- 1 sam sam        2925 Jan 25 23:04 NR005_S4_L002_R2_001.fastq.gz_trimming_report.txt\n",
      "-rw-rw-r-- 1 sam sam        4473 Jan 25 23:06 NR006_S3_L001_R1_001.fastq.gz_trimming_report.txt\n",
      "-rw-rw-r-- 1 sam sam   244698326 Jan 25 23:10 NR006_S3_L001_R1_001_val_1.fq.gz\n",
      "-rw-rw-r-- 1 sam sam   253711980 Jan 25 23:10 NR006_S3_L001_R2_001_val_2.fq.gz\n",
      "-rw-rw-r-- 1 sam sam        4663 Jan 25 23:10 NR006_S3_L001_R2_001.fastq.gz_trimming_report.txt\n",
      "-rw-rw-r-- 1 sam sam        4491 Jan 25 23:12 NR006_S3_L002_R1_001.fastq.gz_trimming_report.txt\n",
      "-rw-rw-r-- 1 sam sam   201301350 Jan 25 23:15 NR006_S3_L002_R1_001_val_1.fq.gz\n",
      "-rw-rw-r-- 1 sam sam   209991311 Jan 25 23:15 NR006_S3_L002_R2_001_val_2.fq.gz\n",
      "-rw-rw-r-- 1 sam sam        4673 Jan 25 23:15 NR006_S3_L002_R2_001.fastq.gz_trimming_report.txt\n",
      "-rw-rw-r-- 1 sam sam        4759 Jan 25 23:15 NR012_S1_L001_R1_001.fastq.gz_trimming_report.txt\n",
      "-rw-rw-r-- 1 sam sam    19050757 Jan 25 23:16 NR012_S1_L001_R1_001_val_1.fq.gz\n",
      "-rw-rw-r-- 1 sam sam    20357336 Jan 25 23:16 NR012_S1_L001_R2_001_val_2.fq.gz\n",
      "-rw-rw-r-- 1 sam sam        4961 Jan 25 23:16 NR012_S1_L001_R2_001.fastq.gz_trimming_report.txt\n",
      "-rw-rw-r-- 1 sam sam        4761 Jan 25 23:16 NR012_S1_L002_R1_001.fastq.gz_trimming_report.txt\n",
      "-rw-rw-r-- 1 sam sam    21014972 Jan 25 23:17 NR012_S1_L002_R1_001_val_1.fq.gz\n",
      "-rw-rw-r-- 1 sam sam    22380350 Jan 25 23:17 NR012_S1_L002_R2_001_val_2.fq.gz\n",
      "-rw-rw-r-- 1 sam sam        4965 Jan 25 23:17 NR012_S1_L002_R2_001.fastq.gz_trimming_report.txt\n",
      "-rw-rw-r-- 1 sam sam        6300 Jan 26 02:10 NR013_AD013_S2_L001_R1_001.fastq.gz_trimming_report.txt\n",
      "-rw-rw-r-- 1 sam sam 20147543284 Jan 26 08:25 NR013_AD013_S2_L001_R1_001_val_1.fq.gz\n",
      "-rw-rw-r-- 1 sam sam 21774947842 Jan 26 08:25 NR013_AD013_S2_L001_R2_001_val_2.fq.gz\n",
      "-rw-rw-r-- 1 sam sam        6504 Jan 26 08:25 NR013_AD013_S2_L001_R2_001.fastq.gz_trimming_report.txt\n",
      "-rw-rw-r-- 1 sam sam        6395 Jan 26 11:46 NR013_AD013_S2_L002_R1_001.fastq.gz_trimming_report.txt\n",
      "-rw-rw-r-- 1 sam sam 23696944504 Jan 26 18:50 NR013_AD013_S2_L002_R1_001_val_1.fq.gz\n",
      "-rw-rw-r-- 1 sam sam 25437014741 Jan 26 18:50 NR013_AD013_S2_L002_R2_001_val_2.fq.gz\n",
      "-rw-rw-r-- 1 sam sam        6509 Jan 26 18:50 NR013_AD013_S2_L002_R2_001.fastq.gz_trimming_report.txt\n",
      "-rw-rw-r-- 1 sam sam        6674 Jan 27 01:21 NR014_AD014_S5_L001_R1_001.fastq.gz_trimming_report.txt\n",
      "-rw-rw-r-- 1 sam sam 45117073184 Jan 27 14:53 NR014_AD014_S5_L001_R1_001_val_1.fq.gz\n",
      "-rw-rw-r-- 1 sam sam 48017176928 Jan 27 14:53 NR014_AD014_S5_L001_R2_001_val_2.fq.gz\n",
      "-rw-rw-r-- 1 sam sam        6894 Jan 27 14:53 NR014_AD014_S5_L001_R2_001.fastq.gz_trimming_report.txt\n",
      "-rw-rw-r-- 1 sam sam        6608 Jan 27 22:11 NR014_AD014_S5_L002_R1_001.fastq.gz_trimming_report.txt\n",
      "-rw-rw-r-- 1 sam sam 51252846392 Jan 28 13:18 NR014_AD014_S5_L002_R1_001_val_1.fq.gz\n",
      "-rw-rw-r-- 1 sam sam 54405899981 Jan 28 13:18 NR014_AD014_S5_L002_R2_001_val_2.fq.gz\n",
      "-rw-rw-r-- 1 sam sam        6866 Jan 28 13:18 NR014_AD014_S5_L002_R2_001.fastq.gz_trimming_report.txt\n",
      "-rw-rw-r-- 1 sam sam        6314 Jan 28 15:32 NR015_AD015_S6_L001_R1_001.fastq.gz_trimming_report.txt\n",
      "-rw-rw-r-- 1 sam sam 15496411838 Jan 28 20:19 NR015_AD015_S6_L001_R1_001_val_1.fq.gz\n",
      "-rw-rw-r-- 1 sam sam 16932572896 Jan 28 20:19 NR015_AD015_S6_L001_R2_001_val_2.fq.gz\n",
      "-rw-rw-r-- 1 sam sam        6503 Jan 28 20:19 NR015_AD015_S6_L001_R2_001.fastq.gz_trimming_report.txt\n",
      "-rw-rw-r-- 1 sam sam        6353 Jan 28 22:43 NR015_AD015_S6_L002_R1_001.fastq.gz_trimming_report.txt\n",
      "-rw-rw-r-- 1 sam sam 16904415137 Jan 29 03:47 NR015_AD015_S6_L002_R1_001_val_1.fq.gz\n",
      "-rw-rw-r-- 1 sam sam 18311321441 Jan 29 03:47 NR015_AD015_S6_L002_R2_001_val_2.fq.gz\n",
      "-rw-rw-r-- 1 sam sam        6505 Jan 29 03:48 NR015_AD015_S6_L002_R2_001.fastq.gz_trimming_report.txt\n",
      "-rw-rw-r-- 1 sam sam        4449 Jan 29 03:48 NR019_S7_L001_R1_001.fastq.gz_trimming_report.txt\n",
      "-rw-rw-r-- 1 sam sam     3676920 Jan 29 03:48 NR019_S7_L001_R1_001_val_1.fq.gz\n",
      "-rw-rw-r-- 1 sam sam     3697878 Jan 29 03:48 NR019_S7_L001_R2_001_val_2.fq.gz\n",
      "-rw-rw-r-- 1 sam sam        4656 Jan 29 03:48 NR019_S7_L001_R2_001.fastq.gz_trimming_report.txt\n",
      "-rw-rw-r-- 1 sam sam        4343 Jan 29 03:48 NR019_S7_L002_R1_001.fastq.gz_trimming_report.txt\n",
      "-rw-rw-r-- 1 sam sam     1930743 Jan 29 03:48 NR019_S7_L002_R1_001_val_1.fq.gz\n",
      "-rw-rw-r-- 1 sam sam     1921192 Jan 29 03:48 NR019_S7_L002_R2_001_val_2.fq.gz\n",
      "-rw-rw-r-- 1 sam sam        4566 Jan 29 03:48 NR019_S7_L002_R2_001.fastq.gz_trimming_report.txt\n",
      "-rw-rw-r-- 1 sam sam        4831 Jan 29 03:50 NR021_S8_L001_R1_001.fastq.gz_trimming_report.txt\n",
      "-rw-rw-r-- 1 sam sam   293401963 Jan 29 03:55 NR021_S8_L001_R1_001_val_1.fq.gz\n",
      "-rw-rw-r-- 1 sam sam   308744383 Jan 29 03:55 NR021_S8_L001_R2_001_val_2.fq.gz\n",
      "-rw-rw-r-- 1 sam sam        4992 Jan 29 03:55 NR021_S8_L001_R2_001.fastq.gz_trimming_report.txt\n",
      "-rw-rw-r-- 1 sam sam        4876 Jan 29 03:57 NR021_S8_L002_R1_001.fastq.gz_trimming_report.txt\n",
      "-rw-rw-r-- 1 sam sam   241735249 Jan 29 04:01 NR021_S8_L002_R1_001_val_1.fq.gz\n",
      "-rw-rw-r-- 1 sam sam   252776822 Jan 29 04:01 NR021_S8_L002_R2_001_val_2.fq.gz\n",
      "-rw-rw-r-- 1 sam sam        4990 Jan 29 04:02 NR021_S8_L002_R2_001.fastq.gz_trimming_report.txt\n"
     ]
    }
   ],
   "source": [
    "%%bash\n",
    "ls -ltr /home/sam/data/geoduck_illumina/trimmed/"
   ]
  },
  {
   "cell_type": "code",
   "execution_count": 25,
   "metadata": {},
   "outputs": [
    {
     "name": "stdout",
     "output_type": "stream",
     "text": [
      "total 49704\n",
      "-rw-rw-r-- 1 sam sam    28693 Mar  7  2017 20160203_Olurida_Zymo_Data_Handling.ipynb\n",
      "-rw-rw-r-- 1 sam sam    24004 Mar  7  2017 20160126_Pgenerosa_BGI_data_handling.ipynb\n",
      "-rw-rw-r-- 1 sam sam    24090 Mar  7  2017 20160126_Olurida_BGI_data_handling.ipynb\n",
      "-rw-rw-r-- 1 sam sam    13392 Mar  7  2017 20160114_wasted_space_synologies.ipynb\n",
      "-rw-rw-r-- 1 sam sam    19015 Mar  7  2017 20150521_Cgigas_larvae_OA_Trimmomatic_FASTQC.ipynb\n",
      "-rw-rw-r-- 1 sam sam     9497 Mar  7  2017 20150506_Cgigas_larvae_OA_trimmomatic_FASTQC.ipynb\n",
      "-rw-rw-r-- 1 sam sam    23188 Mar  7  2017 20150501_Cgigas_larvae_OA_BLASTn_nt.ipynb\n",
      "-rw-rw-r-- 1 sam sam    14730 Mar  7  2017 20150429_Gigas_larvae_OA_BLASTn.ipynb\n",
      "-rw-rw-r-- 1 sam sam    74171 Mar  7  2017 20150414_C_virginica_LSU_Oil_Spill_Trimmomatic_FASTQC.ipynb\n",
      "-rw-rw-r-- 1 sam sam    62599 Mar  7  2017 20150414_C_gigas_Larvae_OA_Trimmomatic_FASTQC.ipynb\n",
      "-rw-rw-r-- 1 sam sam     5147 Mar  7  2017 20150408_Install_Bismark_bisulfite_mapper.ipynb\n",
      "-rw-rw-r-- 1 sam sam     8239 Mar  7  2017 20150317_LSU_OilSpill_EpinextAdaptor1_ID.ipynb\n",
      "-rw-rw-r-- 1 sam sam    14225 Mar  7  2017 20150316_LSU_OilSpill_Adapter_ID.ipynb\n",
      "-rw-rw-r-- 1 sam sam    30320 Mar  7  2017 20150313_LSU_Oil_Spill_IndexID_Comparisons.ipynb\n",
      "-rw-rw-r-- 1 sam sam 31709669 Mar  7  2017 20160308_find_rename_2bRAD_undetermined_fastqs.ipynb\n",
      "-rw-rw-r-- 1 sam sam   167492 Mar  7  2017 20160406_STACKS_install.ipynb\n",
      "-rw-rw-r-- 1 sam sam  1402108 Mar  7  2017 20160406_Oly_GBS_STACKS.ipynb\n",
      "-rw-rw-r-- 1 sam sam    20652 Mar  7  2017 20160314_Olurida_GBS_data_management.ipynb\n",
      "-rw-rw-r-- 1 sam sam     7461 Mar  7  2017 20160516_Oly_Small_Insert_Library_Genome_Read_Counts.ipynb\n",
      "-rw-rw-r-- 1 sam sam    12166 Mar  7  2017 20160502_Oly_GBS_barcode_repair.ipynb\n",
      "-rw-rw-r-- 1 sam sam    32920 Mar  7  2017 20160428_Oly_GBS_STACKS.ipynb\n",
      "-rw-rw-r-- 1 sam sam     9705 Mar  7  2017 20160427_speed_comparison.ipynb\n",
      "-rw-rw-r-- 1 sam sam     5380 Mar  7  2017 20160427_Oly_GBS_data_management.ipynb\n",
      "-rw-rw-r-- 1 sam sam   138177 Mar  7  2017 20160418_pyrad_oly_PE-GBS.ipynb\n",
      "-rw-rw-r-- 1 sam sam   116180 Mar  7  2017 20160418_Oly_GBS_PE-Pyrad_populations.ipynb\n",
      "-rw-rw-r-- 1 sam sam    16182 Mar  7  2017 20160411_Concatenate_Oly_MBDseq.ipynb\n",
      "-rw-rw-r-- 1 sam sam    17959 Mar  7  2017 20160525_pyrad_oly_gbs_bgi.ipynb\n",
      "-rw-rw-r-- 1 sam sam  2949276 Mar  7  2017 20160523_Oly_GBS_Stacks.ipynb\n",
      "-rw-rw-r-- 1 sam sam   334644 Mar  7  2017 20160715_ec2_oly_gbs_pyrad.ipynb\n",
      "-rw-rw-r-- 1 sam sam    96903 Mar  7  2017 20160714_EC2_Oly_GBS_stacks_analysis.ipynb\n",
      "-rw-rw-r-- 1 sam sam   431314 Mar  7  2017 20160609_pyrad_oly_gbs_bgi.ipynb\n",
      "-rw-rw-r-- 1 sam sam    24394 Mar  7  2017 20170104_docker_oly_BGI_genome_corruption_solved.ipynb\n",
      "-rw-rw-r-- 1 sam sam    37578 Mar  7  2017 20161230_docker_geoduck_RRBS_md5_checks.ipynb\n",
      "-rw-rw-r-- 1 sam sam    22311 Mar  7  2017 20161229_docker_genewiz_geoduck_RRBS_data.ipynb\n",
      "-rw-rw-r-- 1 sam sam    18693 Mar  7  2017 20161214_docker_notebook_trimming.ipynb\n",
      "-rw-rw-r-- 1 sam sam    82776 Mar  7  2017 20161214_docker_BGI_data_integrity_check.ipynb\n",
      "-rw-rw-r-- 1 sam sam     6509 Mar  7  2017 20161206_docker_BGI_genome_downloads.ipynb\n",
      "-rw-rw-r-- 1 sam sam   395673 Mar  7  2017 20161129_docker_R_magics_failure.ipynb\n",
      "-rw-rw-r-- 1 sam sam    24079 Mar  7  2017 20161117_docker_oly_vcf_analysis.ipynb\n",
      "-rw-rw-r-- 1 sam sam    22623 Mar  7  2017 20161117_docker_oly_genome_fastq_corruption.ipynb\n",
      "-rw-rw-r-- 1 sam sam     7544 Mar  7  2017 20161025_Pgenerosa_Small_Library_Genome_Read_Counts.ipynb\n",
      "-rw-rw-r-- 1 sam sam   693032 Mar  7  2017 20160816_oly_gbs_fst_calcs.ipynb\n",
      "-rwxrwxr-x 1 sam sam    17069 Mar  7  2017 PhageNGS.ipynb\n",
      "-rwxrwxr-x 1 sam sam   114364 Mar  7  2017 PhageNGS_ID.ipynb\n",
      "-rw-rw-r-- 1 sam sam    98935 Mar  7  2017 PE-GBS_empirical.ipynb\n",
      "-rwxrwxr-x 1 sam sam    21432 Mar  7  2017 InstallingBLAST.ipynb\n",
      "-rw-rw-r-- 1 sam sam   345694 Mar  7  2017 20170306_docker_fastqc_demultiplexed_bgi_oly_gbs.ipynb\n",
      "-rw-rw-r-- 1 sam sam   118825 Mar  7  2017 20170227_docker_jay_ngs_data_retrieval.ipynb\n",
      "-rw-rw-r-- 1 sam sam    30324 Jan 25 08:15 20170320_docker_Oly_BGI_GBS_demultiplexing_reproducibility.ipynb\n",
      "-rw-rw-r-- 1 sam sam    94306 Jan 25 08:15 20170314_docker_Oly_BGI_GBS_demultiplexing_reproducibility.ipynb\n",
      "-rw-rw-r-- 1 sam sam    50140 Jan 25 08:15 20170301_docker_fastqc_nondemultiplexed_bgi_oly_gbs.ipynb\n",
      "-rw-rw-r-- 1 sam sam    45963 Jan 25 08:15 20170918_docker_pacbio_oly_miniasm0.2.ipynb\n",
      "-rw-rw-r-- 1 sam sam    44545 Jan 25 08:15 20170907_docker_pacbio_oly_minimap2.ipynb\n",
      "-rw-rw-r-- 1 sam sam    79744 Jan 25 08:15 20170622_oly_pacbio_data_management.ipynb\n",
      "-rw-rw-r-- 1 sam sam  1332923 Jan 25 08:15 20170918_docker_pacbio_oly_racon0.5.0.ipynb\n",
      "-rw-rw-r-- 1 sam sam  1345133 Jan 25 08:15 20171003_docker_oly_assembly_comparisons.ipynb\n",
      "-rw-rw-r-- 1 sam sam  3211263 Jan 25 08:15 20171004_docker_oly_redundans.ipynb\n",
      "-rw-rw-r-- 1 sam sam   297913 Jan 25 08:15 20171018_docker_oly_canu.ipynb\n",
      "-rw-rw-r-- 1 sam sam  1596545 Jan 25 08:15 20171005_docker_oly_redundans.ipynb\n",
      "-rw-rw-r-- 1 sam sam      240 Jan 25 08:15 README.md\n",
      "-rw-rw-r-- 1 sam sam    31429 Jan 25 08:15 20180103_emu_pbjelly.ipynb\n",
      "-rw-rw-r-- 1 sam sam    20436 Jan 25 08:15 20171130_emu_pbjelly.ipynb\n",
      "-rw-rw-r-- 1 sam sam    26697 Jan 25 08:15 20171114_swoose_oly_assembly_comparisons_quast.ipynb\n",
      "-rw-rw-r-- 1 sam sam    28575 Jan 25 08:15 20171114_emu_pbjelly_BGI_scaffold.ipynb\n",
      "-rw-rw-r-- 1 sam sam    34367 Jan 25 08:15 20171113_emu_pbjelly_22mer_plat.ipynb\n",
      "-rw-rw-r-- 1 sam sam    16623 Jan 25 08:15 20171023_restore_scaphapoda_data.ipynb\n",
      "-rw-rw-r-- 1 sam sam    34879 Jan 25 08:15 20171023_docker_oly_redundans.ipynb\n",
      "-rw-rw-r-- 1 sam sam   983417 Jan 25 08:15 20171023_docker_oly_pacbio_canu_comparisons.ipynb\n",
      "-rw-rw-r-- 1 sam sam    14756 Jan 25 08:55 20180116_swoose_oly_assembly_comparisons_quast.ipynb\n",
      "-rw-rw-r-- 1 sam sam     1069 Jan 25 08:58 template_linux.ipynb\n",
      "-rw-rw-r-- 1 sam sam  1151178 Jan 29 07:25 multiqc_report.html\n",
      "drwxrwxr-x 2 sam sam     4096 Jan 29 07:25 multiqc_data\n",
      "-rw-rw-r-- 1 sam sam   390808 Jan 29 07:27 20180125_roadrunner_bwa_geoduck_novaseq.ipynb\n"
     ]
    }
   ],
   "source": [
    "%%bash\n",
    "ls -ltr /home/sam/gitrepos/LabDocs/jupyter_nbs/sam"
   ]
  },
  {
   "cell_type": "code",
   "execution_count": 26,
   "metadata": {
    "collapsed": true
   },
   "outputs": [],
   "source": [
    "%%bash\n",
    "mv /home/sam/gitrepos/LabDocs/jupyter_nbs/sam/multiqc* /home/sam/data/geoduck_illumina/trimmed/"
   ]
  },
  {
   "cell_type": "code",
   "execution_count": 27,
   "metadata": {},
   "outputs": [
    {
     "name": "stdout",
     "output_type": "stream",
     "text": [
      "-rw-rw-r-- 1 sam sam 1151178 Jan 29 07:25 /home/sam/data/geoduck_illumina/trimmed/multiqc_report.html\n",
      "\n",
      "/home/sam/data/geoduck_illumina/trimmed/multiqc_data:\n",
      "total 1472\n",
      "-rw-rw-r-- 1 sam sam    3231 Jan 29 07:25 multiqc_cutadapt.txt\n",
      "-rw-rw-r-- 1 sam sam    4966 Jan 29 07:25 multiqc_sources.txt\n",
      "-rw-rw-r-- 1 sam sam    1552 Jan 29 07:25 multiqc_general_stats.txt\n",
      "-rw-rw-r-- 1 sam sam 1469611 Jan 29 07:25 multiqc_data.json\n",
      "-rw-rw-r-- 1 sam sam   16427 Jan 29 07:25 multiqc.log\n"
     ]
    }
   ],
   "source": [
    "%%bash\n",
    "ls -ltr /home/sam/data/geoduck_illumina/trimmed/multiqc*"
   ]
  },
  {
   "cell_type": "code",
   "execution_count": 28,
   "metadata": {},
   "outputs": [
    {
     "name": "stdout",
     "output_type": "stream",
     "text": [
      "total 398753368\n",
      "-rw-rw-r-- 1 sam sam        6262 Jan 25 14:07 AD002_S9_L001_R1_001.fastq.gz_trimming_report.txt\n",
      "-rw-rw-r-- 1 sam sam 10588237401 Jan 25 17:19 AD002_S9_L001_R1_001_val_1.fq.gz\n",
      "-rw-rw-r-- 1 sam sam 11222294155 Jan 25 17:19 AD002_S9_L001_R2_001_val_2.fq.gz\n",
      "-rw-rw-r-- 1 sam sam        6479 Jan 25 17:19 AD002_S9_L001_R2_001.fastq.gz_trimming_report.txt\n",
      "-rw-rw-r-- 1 sam sam        6256 Jan 25 19:11 AD002_S9_L002_R1_001.fastq.gz_trimming_report.txt\n",
      "-rw-rw-r-- 1 sam sam 13062478451 Jan 25 23:04 AD002_S9_L002_R1_001_val_1.fq.gz\n",
      "-rw-rw-r-- 1 sam sam 13843776887 Jan 25 23:04 AD002_S9_L002_R2_001_val_2.fq.gz\n",
      "-rw-rw-r-- 1 sam sam        6480 Jan 25 23:04 AD002_S9_L002_R2_001.fastq.gz_trimming_report.txt\n",
      "-rw-rw-r-- 1 sam sam        3027 Jan 25 23:04 NR005_S4_L001_R1_001.fastq.gz_trimming_report.txt\n",
      "-rw-rw-r-- 1 sam sam     3141496 Jan 25 23:04 NR005_S4_L001_R1_001_val_1.fq.gz\n",
      "-rw-rw-r-- 1 sam sam     3245747 Jan 25 23:04 NR005_S4_L001_R2_001_val_2.fq.gz\n",
      "-rw-rw-r-- 1 sam sam        2992 Jan 25 23:04 NR005_S4_L001_R2_001.fastq.gz_trimming_report.txt\n",
      "-rw-rw-r-- 1 sam sam        2970 Jan 25 23:04 NR005_S4_L002_R1_001.fastq.gz_trimming_report.txt\n",
      "-rw-rw-r-- 1 sam sam     2574088 Jan 25 23:04 NR005_S4_L002_R1_001_val_1.fq.gz\n",
      "-rw-rw-r-- 1 sam sam     2660195 Jan 25 23:04 NR005_S4_L002_R2_001_val_2.fq.gz\n",
      "-rw-rw-r-- 1 sam sam        2925 Jan 25 23:04 NR005_S4_L002_R2_001.fastq.gz_trimming_report.txt\n",
      "-rw-rw-r-- 1 sam sam        4473 Jan 25 23:06 NR006_S3_L001_R1_001.fastq.gz_trimming_report.txt\n",
      "-rw-rw-r-- 1 sam sam   244698326 Jan 25 23:10 NR006_S3_L001_R1_001_val_1.fq.gz\n",
      "-rw-rw-r-- 1 sam sam   253711980 Jan 25 23:10 NR006_S3_L001_R2_001_val_2.fq.gz\n",
      "-rw-rw-r-- 1 sam sam        4663 Jan 25 23:10 NR006_S3_L001_R2_001.fastq.gz_trimming_report.txt\n",
      "-rw-rw-r-- 1 sam sam        4491 Jan 25 23:12 NR006_S3_L002_R1_001.fastq.gz_trimming_report.txt\n",
      "-rw-rw-r-- 1 sam sam   201301350 Jan 25 23:15 NR006_S3_L002_R1_001_val_1.fq.gz\n",
      "-rw-rw-r-- 1 sam sam   209991311 Jan 25 23:15 NR006_S3_L002_R2_001_val_2.fq.gz\n",
      "-rw-rw-r-- 1 sam sam        4673 Jan 25 23:15 NR006_S3_L002_R2_001.fastq.gz_trimming_report.txt\n",
      "-rw-rw-r-- 1 sam sam        4759 Jan 25 23:15 NR012_S1_L001_R1_001.fastq.gz_trimming_report.txt\n",
      "-rw-rw-r-- 1 sam sam    19050757 Jan 25 23:16 NR012_S1_L001_R1_001_val_1.fq.gz\n",
      "-rw-rw-r-- 1 sam sam    20357336 Jan 25 23:16 NR012_S1_L001_R2_001_val_2.fq.gz\n",
      "-rw-rw-r-- 1 sam sam        4961 Jan 25 23:16 NR012_S1_L001_R2_001.fastq.gz_trimming_report.txt\n",
      "-rw-rw-r-- 1 sam sam        4761 Jan 25 23:16 NR012_S1_L002_R1_001.fastq.gz_trimming_report.txt\n",
      "-rw-rw-r-- 1 sam sam    21014972 Jan 25 23:17 NR012_S1_L002_R1_001_val_1.fq.gz\n",
      "-rw-rw-r-- 1 sam sam    22380350 Jan 25 23:17 NR012_S1_L002_R2_001_val_2.fq.gz\n",
      "-rw-rw-r-- 1 sam sam        4965 Jan 25 23:17 NR012_S1_L002_R2_001.fastq.gz_trimming_report.txt\n",
      "-rw-rw-r-- 1 sam sam        6300 Jan 26 02:10 NR013_AD013_S2_L001_R1_001.fastq.gz_trimming_report.txt\n",
      "-rw-rw-r-- 1 sam sam 20147543284 Jan 26 08:25 NR013_AD013_S2_L001_R1_001_val_1.fq.gz\n",
      "-rw-rw-r-- 1 sam sam 21774947842 Jan 26 08:25 NR013_AD013_S2_L001_R2_001_val_2.fq.gz\n",
      "-rw-rw-r-- 1 sam sam        6504 Jan 26 08:25 NR013_AD013_S2_L001_R2_001.fastq.gz_trimming_report.txt\n",
      "-rw-rw-r-- 1 sam sam        6395 Jan 26 11:46 NR013_AD013_S2_L002_R1_001.fastq.gz_trimming_report.txt\n",
      "-rw-rw-r-- 1 sam sam 23696944504 Jan 26 18:50 NR013_AD013_S2_L002_R1_001_val_1.fq.gz\n",
      "-rw-rw-r-- 1 sam sam 25437014741 Jan 26 18:50 NR013_AD013_S2_L002_R2_001_val_2.fq.gz\n",
      "-rw-rw-r-- 1 sam sam        6509 Jan 26 18:50 NR013_AD013_S2_L002_R2_001.fastq.gz_trimming_report.txt\n",
      "-rw-rw-r-- 1 sam sam        6674 Jan 27 01:21 NR014_AD014_S5_L001_R1_001.fastq.gz_trimming_report.txt\n",
      "-rw-rw-r-- 1 sam sam 45117073184 Jan 27 14:53 NR014_AD014_S5_L001_R1_001_val_1.fq.gz\n",
      "-rw-rw-r-- 1 sam sam 48017176928 Jan 27 14:53 NR014_AD014_S5_L001_R2_001_val_2.fq.gz\n",
      "-rw-rw-r-- 1 sam sam        6894 Jan 27 14:53 NR014_AD014_S5_L001_R2_001.fastq.gz_trimming_report.txt\n",
      "-rw-rw-r-- 1 sam sam        6608 Jan 27 22:11 NR014_AD014_S5_L002_R1_001.fastq.gz_trimming_report.txt\n",
      "-rw-rw-r-- 1 sam sam 51252846392 Jan 28 13:18 NR014_AD014_S5_L002_R1_001_val_1.fq.gz\n",
      "-rw-rw-r-- 1 sam sam 54405899981 Jan 28 13:18 NR014_AD014_S5_L002_R2_001_val_2.fq.gz\n",
      "-rw-rw-r-- 1 sam sam        6866 Jan 28 13:18 NR014_AD014_S5_L002_R2_001.fastq.gz_trimming_report.txt\n",
      "-rw-rw-r-- 1 sam sam        6314 Jan 28 15:32 NR015_AD015_S6_L001_R1_001.fastq.gz_trimming_report.txt\n",
      "-rw-rw-r-- 1 sam sam 15496411838 Jan 28 20:19 NR015_AD015_S6_L001_R1_001_val_1.fq.gz\n",
      "-rw-rw-r-- 1 sam sam 16932572896 Jan 28 20:19 NR015_AD015_S6_L001_R2_001_val_2.fq.gz\n",
      "-rw-rw-r-- 1 sam sam        6503 Jan 28 20:19 NR015_AD015_S6_L001_R2_001.fastq.gz_trimming_report.txt\n",
      "-rw-rw-r-- 1 sam sam        6353 Jan 28 22:43 NR015_AD015_S6_L002_R1_001.fastq.gz_trimming_report.txt\n",
      "-rw-rw-r-- 1 sam sam 16904415137 Jan 29 03:47 NR015_AD015_S6_L002_R1_001_val_1.fq.gz\n",
      "-rw-rw-r-- 1 sam sam 18311321441 Jan 29 03:47 NR015_AD015_S6_L002_R2_001_val_2.fq.gz\n",
      "-rw-rw-r-- 1 sam sam        6505 Jan 29 03:48 NR015_AD015_S6_L002_R2_001.fastq.gz_trimming_report.txt\n",
      "-rw-rw-r-- 1 sam sam        4449 Jan 29 03:48 NR019_S7_L001_R1_001.fastq.gz_trimming_report.txt\n",
      "-rw-rw-r-- 1 sam sam     3676920 Jan 29 03:48 NR019_S7_L001_R1_001_val_1.fq.gz\n",
      "-rw-rw-r-- 1 sam sam     3697878 Jan 29 03:48 NR019_S7_L001_R2_001_val_2.fq.gz\n",
      "-rw-rw-r-- 1 sam sam        4656 Jan 29 03:48 NR019_S7_L001_R2_001.fastq.gz_trimming_report.txt\n",
      "-rw-rw-r-- 1 sam sam        4343 Jan 29 03:48 NR019_S7_L002_R1_001.fastq.gz_trimming_report.txt\n",
      "-rw-rw-r-- 1 sam sam     1930743 Jan 29 03:48 NR019_S7_L002_R1_001_val_1.fq.gz\n",
      "-rw-rw-r-- 1 sam sam     1921192 Jan 29 03:48 NR019_S7_L002_R2_001_val_2.fq.gz\n",
      "-rw-rw-r-- 1 sam sam        4566 Jan 29 03:48 NR019_S7_L002_R2_001.fastq.gz_trimming_report.txt\n",
      "-rw-rw-r-- 1 sam sam        4831 Jan 29 03:50 NR021_S8_L001_R1_001.fastq.gz_trimming_report.txt\n",
      "-rw-rw-r-- 1 sam sam   293401963 Jan 29 03:55 NR021_S8_L001_R1_001_val_1.fq.gz\n",
      "-rw-rw-r-- 1 sam sam   308744383 Jan 29 03:55 NR021_S8_L001_R2_001_val_2.fq.gz\n",
      "-rw-rw-r-- 1 sam sam        4992 Jan 29 03:55 NR021_S8_L001_R2_001.fastq.gz_trimming_report.txt\n",
      "-rw-rw-r-- 1 sam sam        4876 Jan 29 03:57 NR021_S8_L002_R1_001.fastq.gz_trimming_report.txt\n",
      "-rw-rw-r-- 1 sam sam   241735249 Jan 29 04:01 NR021_S8_L002_R1_001_val_1.fq.gz\n",
      "-rw-rw-r-- 1 sam sam   252776822 Jan 29 04:01 NR021_S8_L002_R2_001_val_2.fq.gz\n",
      "-rw-rw-r-- 1 sam sam        4990 Jan 29 04:02 NR021_S8_L002_R2_001.fastq.gz_trimming_report.txt\n",
      "drwxrwxr-x 2 sam sam        4096 Jan 29 07:39 20180129_multiqc_trim01\n"
     ]
    }
   ],
   "source": [
    "%%bash\n",
    "mv /home/sam/data/geoduck_illumina/trimmed/multiqc_report.html /home/sam/data/geoduck_illumina/trimmed/multiqc_data/\n",
    "mv /home/sam/data/geoduck_illumina/trimmed/multiqc_data/ /home/sam/data/geoduck_illumina/trimmed/20180129_multiqc_trim01\n",
    "ls -ltr /home/sam/data/geoduck_illumina/trimmed/"
   ]
  },
  {
   "cell_type": "code",
   "execution_count": 29,
   "metadata": {},
   "outputs": [
    {
     "name": "stdout",
     "output_type": "stream",
     "text": [
      "Searching 78 files..\n"
     ]
    },
    {
     "name": "stderr",
     "output_type": "stream",
     "text": [
      "[INFO   ]         multiqc : This is MultiQC v1.5.dev0\n",
      "[INFO   ]         multiqc : Template    : default\n",
      "[INFO   ]         multiqc : Searching '/home/sam/data/geoduck_illumina/trimmed/'\n",
      "[INFO   ]        cutadapt : Found 36 reports\n",
      "[INFO   ]         multiqc : Compressing plot data\n",
      "[INFO   ]         multiqc : Report      : multiqc_report.html\n",
      "[INFO   ]         multiqc : Data        : multiqc_data\n",
      "[INFO   ]         multiqc : MultiQC complete\n",
      "\n",
      "real\t0m4.994s\n",
      "user\t0m4.332s\n",
      "sys\t0m0.204s\n"
     ]
    }
   ],
   "source": [
    "%%bash\n",
    "time /home/shared/anaconda3/bin/multiqc --ignore '.txt' /home/sam/data/geoduck_illumina/trimmed/\n",
    "cat ~/.default-subject.mail | msmtp \"$EMAIL\""
   ]
  },
  {
   "cell_type": "code",
   "execution_count": 30,
   "metadata": {},
   "outputs": [
    {
     "name": "stdout",
     "output_type": "stream",
     "text": [
      "/home/sam/data/geoduck_illumina/trimmed\n",
      "Searching 78 files..\n"
     ]
    },
    {
     "name": "stderr",
     "output_type": "stream",
     "text": [
      "rm: cannot remove 'multiqc_report.html': No such file or directory\n",
      "[INFO   ]         multiqc : This is MultiQC v1.5.dev0\n",
      "[INFO   ]         multiqc : Template    : default\n",
      "[INFO   ]         multiqc : Searching '/home/sam/data/geoduck_illumina/trimmed/'\n",
      "[WARNING]         multiqc : No analysis results found. Cleaning up..\n",
      "[INFO   ]         multiqc : MultiQC complete\n",
      "\n",
      "real\t0m4.253s\n",
      "user\t0m3.184s\n",
      "sys\t0m0.136s\n"
     ]
    }
   ],
   "source": [
    "%%bash\n",
    "cd /home/sam/data/geoduck_illumina/trimmed/\n",
    "pwd\n",
    "rm multiqc_report.html\n",
    "rm -rf multiqc_data/\n",
    "time /home/shared/anaconda3/bin/multiqc --ignore '*.txt' /home/sam/data/geoduck_illumina/trimmed/\n",
    "cat ~/.default-subject.mail | msmtp \"$EMAIL\""
   ]
  },
  {
   "cell_type": "code",
   "execution_count": 31,
   "metadata": {},
   "outputs": [
    {
     "name": "stderr",
     "output_type": "stream",
     "text": [
      "Skipping '*.fq.gz' which didn't exist, or couldn't be read\n",
      "\n",
      "real\t0m1.469s\n",
      "user\t0m0.364s\n",
      "sys\t0m0.084s\n"
     ]
    }
   ],
   "source": [
    "%%bash\n",
    "time fastqc *.fq.gz\n",
    "cat ~/.default-subject.mail | msmtp \"$EMAIL\""
   ]
  },
  {
   "cell_type": "code",
   "execution_count": 32,
   "metadata": {},
   "outputs": [
    {
     "name": "stderr",
     "output_type": "stream",
     "text": [
      "Exception in thread \"main\" java.awt.HeadlessException: \n",
      "No X11 DISPLAY variable was set, but this program performed an operation which requires it.\n",
      "\tat java.awt.GraphicsEnvironment.checkHeadless(GraphicsEnvironment.java:204)\n",
      "\tat java.awt.Window.<init>(Window.java:536)\n",
      "\tat java.awt.Frame.<init>(Frame.java:420)\n",
      "\tat java.awt.Frame.<init>(Frame.java:385)\n",
      "\tat javax.swing.JFrame.<init>(JFrame.java:189)\n",
      "\tat uk.ac.babraham.FastQC.FastQCApplication.<init>(FastQCApplication.java:63)\n",
      "\tat uk.ac.babraham.FastQC.FastQCApplication.main(FastQCApplication.java:332)\n",
      "\n",
      "real\t0m0.368s\n",
      "user\t0m0.392s\n",
      "sys\t0m0.060s\n"
     ]
    }
   ],
   "source": [
    "%%bash\n",
    "time find -name '*.fq.gz' | xargs fastqc\n",
    "cat ~/.default-subject.mail | msmtp \"$EMAIL\""
   ]
  },
  {
   "cell_type": "code",
   "execution_count": 33,
   "metadata": {},
   "outputs": [
    {
     "name": "stdout",
     "output_type": "stream",
     "text": [
      "AD002_S9_L001_R1_001_val_1.fq.gz \\\n",
      "AD002_S9_L001_R2_001_val_2.fq.gz \\\n",
      "AD002_S9_L002_R1_001_val_1.fq.gz \\\n",
      "AD002_S9_L002_R2_001_val_2.fq.gz \\\n",
      "NR005_S4_L001_R1_001_val_1.fq.gz \\\n",
      "NR005_S4_L001_R2_001_val_2.fq.gz \\\n",
      "NR005_S4_L002_R1_001_val_1.fq.gz \\\n",
      "NR005_S4_L002_R2_001_val_2.fq.gz \\\n",
      "NR006_S3_L001_R1_001_val_1.fq.gz \\\n",
      "NR006_S3_L001_R2_001_val_2.fq.gz \\\n",
      "NR006_S3_L002_R1_001_val_1.fq.gz \\\n",
      "NR006_S3_L002_R2_001_val_2.fq.gz \\\n",
      "NR012_S1_L001_R1_001_val_1.fq.gz \\\n",
      "NR012_S1_L001_R2_001_val_2.fq.gz \\\n",
      "NR012_S1_L002_R1_001_val_1.fq.gz \\\n",
      "NR012_S1_L002_R2_001_val_2.fq.gz \\\n",
      "NR013_AD013_S2_L001_R1_001_val_1.fq.gz \\\n",
      "NR013_AD013_S2_L001_R2_001_val_2.fq.gz \\\n",
      "NR013_AD013_S2_L002_R1_001_val_1.fq.gz \\\n",
      "NR013_AD013_S2_L002_R2_001_val_2.fq.gz \\\n",
      "NR014_AD014_S5_L001_R1_001_val_1.fq.gz \\\n",
      "NR014_AD014_S5_L001_R2_001_val_2.fq.gz \\\n",
      "NR014_AD014_S5_L002_R1_001_val_1.fq.gz \\\n",
      "NR014_AD014_S5_L002_R2_001_val_2.fq.gz \\\n",
      "NR015_AD015_S6_L001_R1_001_val_1.fq.gz \\\n",
      "NR015_AD015_S6_L001_R2_001_val_2.fq.gz \\\n",
      "NR015_AD015_S6_L002_R1_001_val_1.fq.gz \\\n",
      "NR015_AD015_S6_L002_R2_001_val_2.fq.gz \\\n",
      "NR019_S7_L001_R1_001_val_1.fq.gz \\\n",
      "NR019_S7_L001_R2_001_val_2.fq.gz \\\n",
      "NR019_S7_L002_R1_001_val_1.fq.gz \\\n",
      "NR019_S7_L002_R2_001_val_2.fq.gz \\\n",
      "NR021_S8_L001_R1_001_val_1.fq.gz \\\n",
      "NR021_S8_L001_R2_001_val_2.fq.gz \\\n",
      "NR021_S8_L002_R1_001_val_1.fq.gz \\\n",
      "NR021_S8_L002_R2_001_val_2.fq.gz \\\n"
     ]
    }
   ],
   "source": [
    "%%bash\n",
    "for file in /home/sam/data/geoduck_illumina/trimmed/*.gz\n",
    "do\n",
    "filename=${file##*/}\n",
    "printf \"%s\\n\" \"$filename \\\\\"\n",
    "done"
   ]
  },
  {
   "cell_type": "code",
   "execution_count": 34,
   "metadata": {},
   "outputs": [
    {
     "name": "stderr",
     "output_type": "stream",
     "text": [
      "Skipping 'AD002_S9_L001_R1_001_val_1.fq.gz' which didn't exist, or couldn't be read\n",
      "Skipping 'AD002_S9_L001_R2_001_val_2.fq.gz' which didn't exist, or couldn't be read\n",
      "Skipping 'AD002_S9_L002_R1_001_val_1.fq.gz' which didn't exist, or couldn't be read\n",
      "Skipping 'AD002_S9_L002_R2_001_val_2.fq.gz' which didn't exist, or couldn't be read\n",
      "Skipping 'NR005_S4_L001_R1_001_val_1.fq.gz' which didn't exist, or couldn't be read\n",
      "Skipping 'NR005_S4_L001_R2_001_val_2.fq.gz' which didn't exist, or couldn't be read\n",
      "Skipping 'NR005_S4_L002_R1_001_val_1.fq.gz' which didn't exist, or couldn't be read\n",
      "Skipping 'NR005_S4_L002_R2_001_val_2.fq.gz' which didn't exist, or couldn't be read\n",
      "Skipping 'NR006_S3_L001_R1_001_val_1.fq.gz' which didn't exist, or couldn't be read\n",
      "Skipping 'NR006_S3_L001_R2_001_val_2.fq.gz' which didn't exist, or couldn't be read\n",
      "Skipping 'NR006_S3_L002_R1_001_val_1.fq.gz' which didn't exist, or couldn't be read\n",
      "Skipping 'NR006_S3_L002_R2_001_val_2.fq.gz' which didn't exist, or couldn't be read\n",
      "Skipping 'NR012_S1_L001_R1_001_val_1.fq.gz' which didn't exist, or couldn't be read\n",
      "Skipping 'NR012_S1_L001_R2_001_val_2.fq.gz' which didn't exist, or couldn't be read\n",
      "Skipping 'NR012_S1_L002_R1_001_val_1.fq.gz' which didn't exist, or couldn't be read\n",
      "Skipping 'NR012_S1_L002_R2_001_val_2.fq.gz' which didn't exist, or couldn't be read\n",
      "Skipping 'NR013_AD013_S2_L001_R1_001_val_1.fq.gz' which didn't exist, or couldn't be read\n",
      "Skipping 'NR013_AD013_S2_L001_R2_001_val_2.fq.gz' which didn't exist, or couldn't be read\n",
      "Skipping 'NR013_AD013_S2_L002_R1_001_val_1.fq.gz' which didn't exist, or couldn't be read\n",
      "Skipping 'NR013_AD013_S2_L002_R2_001_val_2.fq.gz' which didn't exist, or couldn't be read\n",
      "Skipping 'NR014_AD014_S5_L001_R1_001_val_1.fq.gz' which didn't exist, or couldn't be read\n",
      "Skipping 'NR014_AD014_S5_L001_R2_001_val_2.fq.gz' which didn't exist, or couldn't be read\n",
      "Skipping 'NR014_AD014_S5_L002_R1_001_val_1.fq.gz' which didn't exist, or couldn't be read\n",
      "Skipping 'NR014_AD014_S5_L002_R2_001_val_2.fq.gz' which didn't exist, or couldn't be read\n",
      "Skipping 'NR015_AD015_S6_L001_R1_001_val_1.fq.gz' which didn't exist, or couldn't be read\n",
      "Skipping 'NR015_AD015_S6_L001_R2_001_val_2.fq.gz' which didn't exist, or couldn't be read\n",
      "Skipping 'NR015_AD015_S6_L002_R1_001_val_1.fq.gz' which didn't exist, or couldn't be read\n",
      "Skipping 'NR015_AD015_S6_L002_R2_001_val_2.fq.gz' which didn't exist, or couldn't be read\n",
      "Skipping 'NR019_S7_L001_R1_001_val_1.fq.gz' which didn't exist, or couldn't be read\n",
      "Skipping 'NR019_S7_L001_R2_001_val_2.fq.gz' which didn't exist, or couldn't be read\n",
      "Skipping 'NR019_S7_L002_R1_001_val_1.fq.gz' which didn't exist, or couldn't be read\n",
      "Skipping 'NR019_S7_L002_R2_001_val_2.fq.gz' which didn't exist, or couldn't be read\n",
      "Skipping 'NR021_S8_L001_R1_001_val_1.fq.gz' which didn't exist, or couldn't be read\n",
      "Skipping 'NR021_S8_L001_R2_001_val_2.fq.gz' which didn't exist, or couldn't be read\n",
      "Skipping 'NR021_S8_L002_R1_001_val_1.fq.gz' which didn't exist, or couldn't be read\n",
      "Skipping 'NR021_S8_L002_R2_001_val_2.fq.gz' which didn't exist, or couldn't be read\n",
      "\n",
      "real\t0m0.237s\n",
      "user\t0m0.364s\n",
      "sys\t0m0.036s\n"
     ]
    }
   ],
   "source": [
    "%%bash\n",
    "time fastqc \\\n",
    "AD002_S9_L001_R1_001_val_1.fq.gz \\\n",
    "AD002_S9_L001_R2_001_val_2.fq.gz \\\n",
    "AD002_S9_L002_R1_001_val_1.fq.gz \\\n",
    "AD002_S9_L002_R2_001_val_2.fq.gz \\\n",
    "NR005_S4_L001_R1_001_val_1.fq.gz \\\n",
    "NR005_S4_L001_R2_001_val_2.fq.gz \\\n",
    "NR005_S4_L002_R1_001_val_1.fq.gz \\\n",
    "NR005_S4_L002_R2_001_val_2.fq.gz \\\n",
    "NR006_S3_L001_R1_001_val_1.fq.gz \\\n",
    "NR006_S3_L001_R2_001_val_2.fq.gz \\\n",
    "NR006_S3_L002_R1_001_val_1.fq.gz \\\n",
    "NR006_S3_L002_R2_001_val_2.fq.gz \\\n",
    "NR012_S1_L001_R1_001_val_1.fq.gz \\\n",
    "NR012_S1_L001_R2_001_val_2.fq.gz \\\n",
    "NR012_S1_L002_R1_001_val_1.fq.gz \\\n",
    "NR012_S1_L002_R2_001_val_2.fq.gz \\\n",
    "NR013_AD013_S2_L001_R1_001_val_1.fq.gz \\\n",
    "NR013_AD013_S2_L001_R2_001_val_2.fq.gz \\\n",
    "NR013_AD013_S2_L002_R1_001_val_1.fq.gz \\\n",
    "NR013_AD013_S2_L002_R2_001_val_2.fq.gz \\\n",
    "NR014_AD014_S5_L001_R1_001_val_1.fq.gz \\\n",
    "NR014_AD014_S5_L001_R2_001_val_2.fq.gz \\\n",
    "NR014_AD014_S5_L002_R1_001_val_1.fq.gz \\\n",
    "NR014_AD014_S5_L002_R2_001_val_2.fq.gz \\\n",
    "NR015_AD015_S6_L001_R1_001_val_1.fq.gz \\\n",
    "NR015_AD015_S6_L001_R2_001_val_2.fq.gz \\\n",
    "NR015_AD015_S6_L002_R1_001_val_1.fq.gz \\\n",
    "NR015_AD015_S6_L002_R2_001_val_2.fq.gz \\\n",
    "NR019_S7_L001_R1_001_val_1.fq.gz \\\n",
    "NR019_S7_L001_R2_001_val_2.fq.gz \\\n",
    "NR019_S7_L002_R1_001_val_1.fq.gz \\\n",
    "NR019_S7_L002_R2_001_val_2.fq.gz \\\n",
    "NR021_S8_L001_R1_001_val_1.fq.gz \\\n",
    "NR021_S8_L001_R2_001_val_2.fq.gz \\\n",
    "NR021_S8_L002_R1_001_val_1.fq.gz \\\n",
    "NR021_S8_L002_R2_001_val_2.fq.gz\n",
    "cat ~/.default-subject.mail | msmtp \"$EMAIL\""
   ]
  },
  {
   "cell_type": "code",
   "execution_count": 35,
   "metadata": {},
   "outputs": [
    {
     "name": "stdout",
     "output_type": "stream",
     "text": [
      "Analysis complete for stdin\n"
     ]
    },
    {
     "name": "stderr",
     "output_type": "stream",
     "text": [
      "gzip: *.gz: No such file or directory\n",
      "java.io.FileNotFoundException: /etc/fastqc/Configuration/adapter_list.txt (No such file or directory)\n",
      "\tat java.io.FileInputStream.open0(Native Method)\n",
      "\tat java.io.FileInputStream.open(FileInputStream.java:195)\n",
      "\tat java.io.FileInputStream.<init>(FileInputStream.java:138)\n",
      "\tat java.io.FileInputStream.<init>(FileInputStream.java:93)\n",
      "\tat uk.ac.babraham.FastQC.Modules.AdapterContent.<init>(AdapterContent.java:75)\n",
      "\tat uk.ac.babraham.FastQC.Modules.ModuleFactory.getStandardModuleList(ModuleFactory.java:37)\n",
      "\tat uk.ac.babraham.FastQC.Analysis.OfflineRunner.processFile(OfflineRunner.java:134)\n",
      "\tat uk.ac.babraham.FastQC.Analysis.OfflineRunner.<init>(OfflineRunner.java:102)\n",
      "\tat uk.ac.babraham.FastQC.FastQCApplication.main(FastQCApplication.java:316)\n",
      "Started analysis of stdin\n",
      "java.io.FileNotFoundException: /etc/fastqc/Configuration/limits.txt (No such file or directory)\n",
      "\tat java.io.FileInputStream.open0(Native Method)\n",
      "\tat java.io.FileInputStream.open(FileInputStream.java:195)\n",
      "\tat java.io.FileInputStream.<init>(FileInputStream.java:138)\n",
      "\tat java.io.FileInputStream.<init>(FileInputStream.java:93)\n",
      "\tat uk.ac.babraham.FastQC.Modules.ModuleConfig.readParams(ModuleConfig.java:87)\n",
      "\tat uk.ac.babraham.FastQC.Modules.ModuleConfig.<clinit>(ModuleConfig.java:35)\n",
      "\tat uk.ac.babraham.FastQC.Modules.PerBaseQualityScores.ignoreInReport(PerBaseQualityScores.java:62)\n",
      "\tat uk.ac.babraham.FastQC.Report.HTMLReportArchive.startDocument(HTMLReportArchive.java:331)\n",
      "\tat uk.ac.babraham.FastQC.Report.HTMLReportArchive.<init>(HTMLReportArchive.java:84)\n",
      "\tat uk.ac.babraham.FastQC.Analysis.OfflineRunner.analysisComplete(OfflineRunner.java:155)\n",
      "\tat uk.ac.babraham.FastQC.Analysis.AnalysisRunner.run(AnalysisRunner.java:110)\n",
      "\tat java.lang.Thread.run(Thread.java:748)\n",
      "Failed to process file stdin\n",
      "java.lang.IllegalArgumentException: No key called gc_sequence:ignore in the config data\n",
      "\tat uk.ac.babraham.FastQC.Modules.ModuleConfig.getParam(ModuleConfig.java:148)\n",
      "\tat uk.ac.babraham.FastQC.Modules.PerSequenceGCContent.ignoreInReport(PerSequenceGCContent.java:57)\n",
      "\tat uk.ac.babraham.FastQC.Report.HTMLReportArchive.startDocument(HTMLReportArchive.java:331)\n",
      "\tat uk.ac.babraham.FastQC.Report.HTMLReportArchive.<init>(HTMLReportArchive.java:84)\n",
      "\tat uk.ac.babraham.FastQC.Analysis.OfflineRunner.analysisComplete(OfflineRunner.java:155)\n",
      "\tat uk.ac.babraham.FastQC.Analysis.AnalysisRunner.run(AnalysisRunner.java:110)\n",
      "\tat java.lang.Thread.run(Thread.java:748)\n",
      "\n",
      "real\t0m1.430s\n",
      "user\t0m0.620s\n",
      "sys\t0m0.064s\n"
     ]
    }
   ],
   "source": [
    "%%bash\n",
    "time \\\n",
    "for fq in *.gz\n",
    "do\n",
    "gunzip -c \"$fq\" | fastqc stdin\n",
    "done\n",
    "cat ~/.default-subject.mail | msmtp \"$EMAIL\""
   ]
  },
  {
   "cell_type": "code",
   "execution_count": 36,
   "metadata": {},
   "outputs": [
    {
     "name": "stdout",
     "output_type": "stream",
     "text": [
      "Analysis complete for stdin\n"
     ]
    },
    {
     "name": "stderr",
     "output_type": "stream",
     "text": [
      "java.io.FileNotFoundException: /etc/fastqc/Configuration/adapter_list.txt (No such file or directory)\n",
      "\tat java.io.FileInputStream.open0(Native Method)\n",
      "\tat java.io.FileInputStream.open(FileInputStream.java:195)\n",
      "\tat java.io.FileInputStream.<init>(FileInputStream.java:138)\n",
      "\tat java.io.FileInputStream.<init>(FileInputStream.java:93)\n",
      "\tat uk.ac.babraham.FastQC.Modules.AdapterContent.<init>(AdapterContent.java:75)\n",
      "\tat uk.ac.babraham.FastQC.Modules.ModuleFactory.getStandardModuleList(ModuleFactory.java:37)\n",
      "\tat uk.ac.babraham.FastQC.Analysis.OfflineRunner.processFile(OfflineRunner.java:134)\n",
      "\tat uk.ac.babraham.FastQC.Analysis.OfflineRunner.<init>(OfflineRunner.java:102)\n",
      "\tat uk.ac.babraham.FastQC.FastQCApplication.main(FastQCApplication.java:316)\n",
      "Started analysis of stdin\n",
      "Failed to process file stdin\n",
      "java.io.FileNotFoundException: /dev/stdin_fastqc.zip (Permission denied)\n",
      "\tat java.io.FileOutputStream.open0(Native Method)\n",
      "\tat java.io.FileOutputStream.open(FileOutputStream.java:270)\n",
      "\tat java.io.FileOutputStream.<init>(FileOutputStream.java:213)\n",
      "\tat java.io.FileOutputStream.<init>(FileOutputStream.java:162)\n",
      "\tat uk.ac.babraham.FastQC.Report.HTMLReportArchive.<init>(HTMLReportArchive.java:80)\n",
      "\tat uk.ac.babraham.FastQC.Analysis.OfflineRunner.analysisComplete(OfflineRunner.java:155)\n",
      "\tat uk.ac.babraham.FastQC.Analysis.AnalysisRunner.run(AnalysisRunner.java:110)\n",
      "\tat java.lang.Thread.run(Thread.java:748)\n"
     ]
    }
   ],
   "source": [
    "%%bash\n",
    "find . -name \"*.fq.gz\"  -exec gunzip -c  '{}' ';' | fastqc  /dev/stdin\n",
    "cat ~/.default-subject.mail | msmtp \"$EMAIL\""
   ]
  },
  {
   "cell_type": "markdown",
   "metadata": {},
   "source": [
    "#### Performed fresh install of FastQC - needed sudo so ran outside of notebook."
   ]
  },
  {
   "cell_type": "code",
   "execution_count": 37,
   "metadata": {},
   "outputs": [
    {
     "name": "stderr",
     "output_type": "stream",
     "text": [
      "Skipping 'AD002_S9_L001_R1_001_val_1.fq.gz' which didn't exist, or couldn't be read\n",
      "Skipping 'AD002_S9_L001_R2_001_val_2.fq.gz' which didn't exist, or couldn't be read\n",
      "Skipping 'AD002_S9_L002_R1_001_val_1.fq.gz' which didn't exist, or couldn't be read\n",
      "Skipping 'AD002_S9_L002_R2_001_val_2.fq.gz' which didn't exist, or couldn't be read\n",
      "Skipping 'NR005_S4_L001_R1_001_val_1.fq.gz' which didn't exist, or couldn't be read\n",
      "Skipping 'NR005_S4_L001_R2_001_val_2.fq.gz' which didn't exist, or couldn't be read\n",
      "Skipping 'NR005_S4_L002_R1_001_val_1.fq.gz' which didn't exist, or couldn't be read\n",
      "Skipping 'NR005_S4_L002_R2_001_val_2.fq.gz' which didn't exist, or couldn't be read\n",
      "Skipping 'NR006_S3_L001_R1_001_val_1.fq.gz' which didn't exist, or couldn't be read\n",
      "Skipping 'NR006_S3_L001_R2_001_val_2.fq.gz' which didn't exist, or couldn't be read\n",
      "Skipping 'NR006_S3_L002_R1_001_val_1.fq.gz' which didn't exist, or couldn't be read\n",
      "Skipping 'NR006_S3_L002_R2_001_val_2.fq.gz' which didn't exist, or couldn't be read\n",
      "Skipping 'NR012_S1_L001_R1_001_val_1.fq.gz' which didn't exist, or couldn't be read\n",
      "Skipping 'NR012_S1_L001_R2_001_val_2.fq.gz' which didn't exist, or couldn't be read\n",
      "Skipping 'NR012_S1_L002_R1_001_val_1.fq.gz' which didn't exist, or couldn't be read\n",
      "Skipping 'NR012_S1_L002_R2_001_val_2.fq.gz' which didn't exist, or couldn't be read\n",
      "Skipping 'NR013_AD013_S2_L001_R1_001_val_1.fq.gz' which didn't exist, or couldn't be read\n",
      "Skipping 'NR013_AD013_S2_L001_R2_001_val_2.fq.gz' which didn't exist, or couldn't be read\n",
      "Skipping 'NR013_AD013_S2_L002_R1_001_val_1.fq.gz' which didn't exist, or couldn't be read\n",
      "Skipping 'NR013_AD013_S2_L002_R2_001_val_2.fq.gz' which didn't exist, or couldn't be read\n",
      "Skipping 'NR014_AD014_S5_L001_R1_001_val_1.fq.gz' which didn't exist, or couldn't be read\n",
      "Skipping 'NR014_AD014_S5_L001_R2_001_val_2.fq.gz' which didn't exist, or couldn't be read\n",
      "Skipping 'NR014_AD014_S5_L002_R1_001_val_1.fq.gz' which didn't exist, or couldn't be read\n",
      "Skipping 'NR014_AD014_S5_L002_R2_001_val_2.fq.gz' which didn't exist, or couldn't be read\n",
      "Skipping 'NR015_AD015_S6_L001_R1_001_val_1.fq.gz' which didn't exist, or couldn't be read\n",
      "Skipping 'NR015_AD015_S6_L001_R2_001_val_2.fq.gz' which didn't exist, or couldn't be read\n",
      "Skipping 'NR015_AD015_S6_L002_R1_001_val_1.fq.gz' which didn't exist, or couldn't be read\n",
      "Skipping 'NR015_AD015_S6_L002_R2_001_val_2.fq.gz' which didn't exist, or couldn't be read\n",
      "Skipping 'NR019_S7_L001_R1_001_val_1.fq.gz' which didn't exist, or couldn't be read\n",
      "Skipping 'NR019_S7_L001_R2_001_val_2.fq.gz' which didn't exist, or couldn't be read\n",
      "Skipping 'NR019_S7_L002_R1_001_val_1.fq.gz' which didn't exist, or couldn't be read\n",
      "Skipping 'NR019_S7_L002_R2_001_val_2.fq.gz' which didn't exist, or couldn't be read\n",
      "Skipping 'NR021_S8_L001_R1_001_val_1.fq.gz' which didn't exist, or couldn't be read\n",
      "Skipping 'NR021_S8_L001_R2_001_val_2.fq.gz' which didn't exist, or couldn't be read\n",
      "Skipping 'NR021_S8_L002_R1_001_val_1.fq.gz' which didn't exist, or couldn't be read\n",
      "Skipping 'NR021_S8_L002_R2_001_val_2.fq.gz' which didn't exist, or couldn't be read\n",
      "\n",
      "real\t0m0.275s\n",
      "user\t0m0.328s\n",
      "sys\t0m0.052s\n"
     ]
    }
   ],
   "source": [
    "%%bash\n",
    "time fastqc \\\n",
    "AD002_S9_L001_R1_001_val_1.fq.gz \\\n",
    "AD002_S9_L001_R2_001_val_2.fq.gz \\\n",
    "AD002_S9_L002_R1_001_val_1.fq.gz \\\n",
    "AD002_S9_L002_R2_001_val_2.fq.gz \\\n",
    "NR005_S4_L001_R1_001_val_1.fq.gz \\\n",
    "NR005_S4_L001_R2_001_val_2.fq.gz \\\n",
    "NR005_S4_L002_R1_001_val_1.fq.gz \\\n",
    "NR005_S4_L002_R2_001_val_2.fq.gz \\\n",
    "NR006_S3_L001_R1_001_val_1.fq.gz \\\n",
    "NR006_S3_L001_R2_001_val_2.fq.gz \\\n",
    "NR006_S3_L002_R1_001_val_1.fq.gz \\\n",
    "NR006_S3_L002_R2_001_val_2.fq.gz \\\n",
    "NR012_S1_L001_R1_001_val_1.fq.gz \\\n",
    "NR012_S1_L001_R2_001_val_2.fq.gz \\\n",
    "NR012_S1_L002_R1_001_val_1.fq.gz \\\n",
    "NR012_S1_L002_R2_001_val_2.fq.gz \\\n",
    "NR013_AD013_S2_L001_R1_001_val_1.fq.gz \\\n",
    "NR013_AD013_S2_L001_R2_001_val_2.fq.gz \\\n",
    "NR013_AD013_S2_L002_R1_001_val_1.fq.gz \\\n",
    "NR013_AD013_S2_L002_R2_001_val_2.fq.gz \\\n",
    "NR014_AD014_S5_L001_R1_001_val_1.fq.gz \\\n",
    "NR014_AD014_S5_L001_R2_001_val_2.fq.gz \\\n",
    "NR014_AD014_S5_L002_R1_001_val_1.fq.gz \\\n",
    "NR014_AD014_S5_L002_R2_001_val_2.fq.gz \\\n",
    "NR015_AD015_S6_L001_R1_001_val_1.fq.gz \\\n",
    "NR015_AD015_S6_L001_R2_001_val_2.fq.gz \\\n",
    "NR015_AD015_S6_L002_R1_001_val_1.fq.gz \\\n",
    "NR015_AD015_S6_L002_R2_001_val_2.fq.gz \\\n",
    "NR019_S7_L001_R1_001_val_1.fq.gz \\\n",
    "NR019_S7_L001_R2_001_val_2.fq.gz \\\n",
    "NR019_S7_L002_R1_001_val_1.fq.gz \\\n",
    "NR019_S7_L002_R2_001_val_2.fq.gz \\\n",
    "NR021_S8_L001_R1_001_val_1.fq.gz \\\n",
    "NR021_S8_L001_R2_001_val_2.fq.gz \\\n",
    "NR021_S8_L002_R1_001_val_1.fq.gz \\\n",
    "NR021_S8_L002_R2_001_val_2.fq.gz\n",
    "cat ~/.default-subject.mail | msmtp \"$EMAIL\""
   ]
  },
  {
   "cell_type": "code",
   "execution_count": 38,
   "metadata": {},
   "outputs": [
    {
     "name": "stdout",
     "output_type": "stream",
     "text": [
      "/usr/local/bin/fastqc\n"
     ]
    }
   ],
   "source": [
    "%%bash\n",
    "which fastqc"
   ]
  },
  {
   "cell_type": "code",
   "execution_count": 39,
   "metadata": {},
   "outputs": [
    {
     "name": "stderr",
     "output_type": "stream",
     "text": [
      "Skipping '*.gz' which didn't exist, or couldn't be read\n",
      "\n",
      "real\t0m0.234s\n",
      "user\t0m0.320s\n",
      "sys\t0m0.080s\n"
     ]
    }
   ],
   "source": [
    "%%bash\n",
    "time \\\n",
    "for fq in *.gz\n",
    "do\n",
    "fastqc \"$fq\"\n",
    "done\n",
    "cat ~/.default-subject.mail | msmtp \"$EMAIL\""
   ]
  },
  {
   "cell_type": "code",
   "execution_count": 40,
   "metadata": {},
   "outputs": [
    {
     "name": "stdout",
     "output_type": "stream",
     "text": [
      "/home/sam/gitrepos/LabDocs/jupyter_nbs/sam\n"
     ]
    }
   ],
   "source": [
    "%%bash\n",
    "pwd"
   ]
  },
  {
   "cell_type": "code",
   "execution_count": 41,
   "metadata": {},
   "outputs": [
    {
     "name": "stdout",
     "output_type": "stream",
     "text": [
      "/home/sam/data/geoduck_illumina/trimmed/AD002_S9_L001_R1_001_val_1.fq.gz \\\n",
      "/home/sam/data/geoduck_illumina/trimmed/AD002_S9_L001_R2_001_val_2.fq.gz \\\n",
      "/home/sam/data/geoduck_illumina/trimmed/AD002_S9_L002_R1_001_val_1.fq.gz \\\n",
      "/home/sam/data/geoduck_illumina/trimmed/AD002_S9_L002_R2_001_val_2.fq.gz \\\n",
      "/home/sam/data/geoduck_illumina/trimmed/NR005_S4_L001_R1_001_val_1.fq.gz \\\n",
      "/home/sam/data/geoduck_illumina/trimmed/NR005_S4_L001_R2_001_val_2.fq.gz \\\n",
      "/home/sam/data/geoduck_illumina/trimmed/NR005_S4_L002_R1_001_val_1.fq.gz \\\n",
      "/home/sam/data/geoduck_illumina/trimmed/NR005_S4_L002_R2_001_val_2.fq.gz \\\n",
      "/home/sam/data/geoduck_illumina/trimmed/NR006_S3_L001_R1_001_val_1.fq.gz \\\n",
      "/home/sam/data/geoduck_illumina/trimmed/NR006_S3_L001_R2_001_val_2.fq.gz \\\n",
      "/home/sam/data/geoduck_illumina/trimmed/NR006_S3_L002_R1_001_val_1.fq.gz \\\n",
      "/home/sam/data/geoduck_illumina/trimmed/NR006_S3_L002_R2_001_val_2.fq.gz \\\n",
      "/home/sam/data/geoduck_illumina/trimmed/NR012_S1_L001_R1_001_val_1.fq.gz \\\n",
      "/home/sam/data/geoduck_illumina/trimmed/NR012_S1_L001_R2_001_val_2.fq.gz \\\n",
      "/home/sam/data/geoduck_illumina/trimmed/NR012_S1_L002_R1_001_val_1.fq.gz \\\n",
      "/home/sam/data/geoduck_illumina/trimmed/NR012_S1_L002_R2_001_val_2.fq.gz \\\n",
      "/home/sam/data/geoduck_illumina/trimmed/NR013_AD013_S2_L001_R1_001_val_1.fq.gz \\\n",
      "/home/sam/data/geoduck_illumina/trimmed/NR013_AD013_S2_L001_R2_001_val_2.fq.gz \\\n",
      "/home/sam/data/geoduck_illumina/trimmed/NR013_AD013_S2_L002_R1_001_val_1.fq.gz \\\n",
      "/home/sam/data/geoduck_illumina/trimmed/NR013_AD013_S2_L002_R2_001_val_2.fq.gz \\\n",
      "/home/sam/data/geoduck_illumina/trimmed/NR014_AD014_S5_L001_R1_001_val_1.fq.gz \\\n",
      "/home/sam/data/geoduck_illumina/trimmed/NR014_AD014_S5_L001_R2_001_val_2.fq.gz \\\n",
      "/home/sam/data/geoduck_illumina/trimmed/NR014_AD014_S5_L002_R1_001_val_1.fq.gz \\\n",
      "/home/sam/data/geoduck_illumina/trimmed/NR014_AD014_S5_L002_R2_001_val_2.fq.gz \\\n",
      "/home/sam/data/geoduck_illumina/trimmed/NR015_AD015_S6_L001_R1_001_val_1.fq.gz \\\n",
      "/home/sam/data/geoduck_illumina/trimmed/NR015_AD015_S6_L001_R2_001_val_2.fq.gz \\\n",
      "/home/sam/data/geoduck_illumina/trimmed/NR015_AD015_S6_L002_R1_001_val_1.fq.gz \\\n",
      "/home/sam/data/geoduck_illumina/trimmed/NR015_AD015_S6_L002_R2_001_val_2.fq.gz \\\n",
      "/home/sam/data/geoduck_illumina/trimmed/NR019_S7_L001_R1_001_val_1.fq.gz \\\n",
      "/home/sam/data/geoduck_illumina/trimmed/NR019_S7_L001_R2_001_val_2.fq.gz \\\n",
      "/home/sam/data/geoduck_illumina/trimmed/NR019_S7_L002_R1_001_val_1.fq.gz \\\n",
      "/home/sam/data/geoduck_illumina/trimmed/NR019_S7_L002_R2_001_val_2.fq.gz \\\n",
      "/home/sam/data/geoduck_illumina/trimmed/NR021_S8_L001_R1_001_val_1.fq.gz \\\n",
      "/home/sam/data/geoduck_illumina/trimmed/NR021_S8_L001_R2_001_val_2.fq.gz \\\n",
      "/home/sam/data/geoduck_illumina/trimmed/NR021_S8_L002_R1_001_val_1.fq.gz \\\n",
      "/home/sam/data/geoduck_illumina/trimmed/NR021_S8_L002_R2_001_val_2.fq.gz \\\n"
     ]
    }
   ],
   "source": [
    "%%bash\n",
    "for file in /home/sam/data/geoduck_illumina/trimmed/*.gz\n",
    "do\n",
    "printf \"%s\\n\" \"$file \\\\\"\n",
    "done"
   ]
  },
  {
   "cell_type": "code",
   "execution_count": 42,
   "metadata": {},
   "outputs": [
    {
     "name": "stdout",
     "output_type": "stream",
     "text": [
      "Analysis complete for NR005_S4_L001_R1_001_val_1.fq.gz\n",
      "Analysis complete for NR005_S4_L001_R2_001_val_2.fq.gz\n",
      "Analysis complete for NR005_S4_L002_R1_001_val_1.fq.gz\n",
      "Analysis complete for NR005_S4_L002_R2_001_val_2.fq.gz\n",
      "Analysis complete for NR012_S1_L001_R1_001_val_1.fq.gz\n",
      "Analysis complete for NR012_S1_L001_R2_001_val_2.fq.gz\n",
      "Analysis complete for NR012_S1_L002_R1_001_val_1.fq.gz\n",
      "Analysis complete for NR012_S1_L002_R2_001_val_2.fq.gz\n",
      "Analysis complete for NR006_S3_L002_R1_001_val_1.fq.gz\n",
      "Analysis complete for NR006_S3_L002_R2_001_val_2.fq.gz\n",
      "Analysis complete for NR006_S3_L001_R1_001_val_1.fq.gz\n",
      "Analysis complete for NR006_S3_L001_R2_001_val_2.fq.gz\n",
      "Analysis complete for AD002_S9_L001_R1_001_val_1.fq.gz\n",
      "Analysis complete for AD002_S9_L001_R2_001_val_2.fq.gz\n",
      "Analysis complete for AD002_S9_L002_R1_001_val_1.fq.gz\n",
      "Analysis complete for AD002_S9_L002_R2_001_val_2.fq.gz\n",
      "Analysis complete for NR013_AD013_S2_L002_R1_001_val_1.fq.gz\n",
      "Analysis complete for NR019_S7_L001_R1_001_val_1.fq.gz\n",
      "Analysis complete for NR019_S7_L001_R2_001_val_2.fq.gz\n",
      "Analysis complete for NR019_S7_L002_R1_001_val_1.fq.gz\n",
      "Analysis complete for NR019_S7_L002_R2_001_val_2.fq.gz\n",
      "Analysis complete for NR021_S8_L001_R1_001_val_1.fq.gz\n",
      "Analysis complete for NR015_AD015_S6_L001_R1_001_val_1.fq.gz\n",
      "Analysis complete for NR021_S8_L001_R2_001_val_2.fq.gz\n",
      "Analysis complete for NR021_S8_L002_R1_001_val_1.fq.gz\n",
      "Analysis complete for NR021_S8_L002_R2_001_val_2.fq.gz\n",
      "Analysis complete for NR013_AD013_S2_L002_R2_001_val_2.fq.gz\n",
      "Analysis complete for NR015_AD015_S6_L001_R2_001_val_2.fq.gz\n",
      "Analysis complete for NR015_AD015_S6_L002_R1_001_val_1.fq.gz\n",
      "Analysis complete for NR013_AD013_S2_L001_R1_001_val_1.fq.gz\n",
      "Analysis complete for NR013_AD013_S2_L001_R2_001_val_2.fq.gz\n",
      "Analysis complete for NR015_AD015_S6_L002_R2_001_val_2.fq.gz\n",
      "Analysis complete for NR014_AD014_S5_L001_R1_001_val_1.fq.gz\n",
      "Analysis complete for NR014_AD014_S5_L001_R2_001_val_2.fq.gz\n",
      "Analysis complete for NR014_AD014_S5_L002_R1_001_val_1.fq.gz\n",
      "Analysis complete for NR014_AD014_S5_L002_R2_001_val_2.fq.gz\n"
     ]
    },
    {
     "name": "stderr",
     "output_type": "stream",
     "text": [
      "Started analysis of AD002_S9_L001_R1_001_val_1.fq.gz\n",
      "Started analysis of AD002_S9_L001_R2_001_val_2.fq.gz\n",
      "Started analysis of AD002_S9_L002_R1_001_val_1.fq.gz\n",
      "Started analysis of AD002_S9_L002_R2_001_val_2.fq.gz\n",
      "Started analysis of NR005_S4_L001_R1_001_val_1.fq.gz\n",
      "Approx 5% complete for NR005_S4_L001_R1_001_val_1.fq.gz\n",
      "Approx 10% complete for NR005_S4_L001_R1_001_val_1.fq.gz\n",
      "Approx 15% complete for NR005_S4_L001_R1_001_val_1.fq.gz\n",
      "Approx 20% complete for NR005_S4_L001_R1_001_val_1.fq.gz\n",
      "Approx 25% complete for NR005_S4_L001_R1_001_val_1.fq.gz\n",
      "Approx 30% complete for NR005_S4_L001_R1_001_val_1.fq.gz\n",
      "Started analysis of NR005_S4_L001_R2_001_val_2.fq.gz\n",
      "Approx 35% complete for NR005_S4_L001_R1_001_val_1.fq.gz\n",
      "Approx 40% complete for NR005_S4_L001_R1_001_val_1.fq.gz\n",
      "Approx 5% complete for NR005_S4_L001_R2_001_val_2.fq.gz\n",
      "Approx 45% complete for NR005_S4_L001_R1_001_val_1.fq.gz\n",
      "Approx 10% complete for NR005_S4_L001_R2_001_val_2.fq.gz\n",
      "Approx 50% complete for NR005_S4_L001_R1_001_val_1.fq.gz\n",
      "Approx 15% complete for NR005_S4_L001_R2_001_val_2.fq.gz\n",
      "Approx 55% complete for NR005_S4_L001_R1_001_val_1.fq.gz\n",
      "Approx 20% complete for NR005_S4_L001_R2_001_val_2.fq.gz\n",
      "Approx 25% complete for NR005_S4_L001_R2_001_val_2.fq.gz\n",
      "Approx 60% complete for NR005_S4_L001_R1_001_val_1.fq.gz\n",
      "Approx 30% complete for NR005_S4_L001_R2_001_val_2.fq.gz\n",
      "Approx 65% complete for NR005_S4_L001_R1_001_val_1.fq.gz\n",
      "Approx 35% complete for NR005_S4_L001_R2_001_val_2.fq.gz\n",
      "Approx 70% complete for NR005_S4_L001_R1_001_val_1.fq.gz\n",
      "Approx 40% complete for NR005_S4_L001_R2_001_val_2.fq.gz\n",
      "Started analysis of NR005_S4_L002_R1_001_val_1.fq.gz\n",
      "Approx 75% complete for NR005_S4_L001_R1_001_val_1.fq.gz\n",
      "Approx 45% complete for NR005_S4_L001_R2_001_val_2.fq.gz\n",
      "Approx 5% complete for NR005_S4_L002_R1_001_val_1.fq.gz\n",
      "Approx 80% complete for NR005_S4_L001_R1_001_val_1.fq.gz\n",
      "Approx 50% complete for NR005_S4_L001_R2_001_val_2.fq.gz\n",
      "Approx 10% complete for NR005_S4_L002_R1_001_val_1.fq.gz\n",
      "Approx 85% complete for NR005_S4_L001_R1_001_val_1.fq.gz\n",
      "Approx 55% complete for NR005_S4_L001_R2_001_val_2.fq.gz\n",
      "Approx 15% complete for NR005_S4_L002_R1_001_val_1.fq.gz\n",
      "Approx 90% complete for NR005_S4_L001_R1_001_val_1.fq.gz\n",
      "Approx 20% complete for NR005_S4_L002_R1_001_val_1.fq.gz\n",
      "Approx 60% complete for NR005_S4_L001_R2_001_val_2.fq.gz\n",
      "Approx 95% complete for NR005_S4_L001_R1_001_val_1.fq.gz\n",
      "Approx 25% complete for NR005_S4_L002_R1_001_val_1.fq.gz\n",
      "Approx 65% complete for NR005_S4_L001_R2_001_val_2.fq.gz\n",
      "Approx 30% complete for NR005_S4_L002_R1_001_val_1.fq.gz\n",
      "Started analysis of NR005_S4_L002_R2_001_val_2.fq.gz\n",
      "Approx 70% complete for NR005_S4_L001_R2_001_val_2.fq.gz\n",
      "Approx 35% complete for NR005_S4_L002_R1_001_val_1.fq.gz\n",
      "Approx 5% complete for NR005_S4_L002_R2_001_val_2.fq.gz\n",
      "Approx 75% complete for NR005_S4_L001_R2_001_val_2.fq.gz\n",
      "Approx 40% complete for NR005_S4_L002_R1_001_val_1.fq.gz\n",
      "Approx 10% complete for NR005_S4_L002_R2_001_val_2.fq.gz\n",
      "Approx 80% complete for NR005_S4_L001_R2_001_val_2.fq.gz\n",
      "Approx 45% complete for NR005_S4_L002_R1_001_val_1.fq.gz\n",
      "Approx 15% complete for NR005_S4_L002_R2_001_val_2.fq.gz\n",
      "Approx 85% complete for NR005_S4_L001_R2_001_val_2.fq.gz\n",
      "Approx 50% complete for NR005_S4_L002_R1_001_val_1.fq.gz\n",
      "Approx 20% complete for NR005_S4_L002_R2_001_val_2.fq.gz\n",
      "Approx 90% complete for NR005_S4_L001_R2_001_val_2.fq.gz\n",
      "Approx 55% complete for NR005_S4_L002_R1_001_val_1.fq.gz\n",
      "Approx 95% complete for NR005_S4_L001_R2_001_val_2.fq.gz\n",
      "Approx 25% complete for NR005_S4_L002_R2_001_val_2.fq.gz\n",
      "Started analysis of NR006_S3_L001_R1_001_val_1.fq.gz\n",
      "Approx 30% complete for NR005_S4_L002_R2_001_val_2.fq.gz\n",
      "Approx 60% complete for NR005_S4_L002_R1_001_val_1.fq.gz\n",
      "Approx 65% complete for NR005_S4_L002_R1_001_val_1.fq.gz\n",
      "Approx 35% complete for NR005_S4_L002_R2_001_val_2.fq.gz\n",
      "Approx 40% complete for NR005_S4_L002_R2_001_val_2.fq.gz\n",
      "Approx 70% complete for NR005_S4_L002_R1_001_val_1.fq.gz\n",
      "Approx 45% complete for NR005_S4_L002_R2_001_val_2.fq.gz\n",
      "Approx 75% complete for NR005_S4_L002_R1_001_val_1.fq.gz\n",
      "Approx 50% complete for NR005_S4_L002_R2_001_val_2.fq.gz\n",
      "Approx 80% complete for NR005_S4_L002_R1_001_val_1.fq.gz\n",
      "Approx 85% complete for NR005_S4_L002_R1_001_val_1.fq.gz\n",
      "Approx 55% complete for NR005_S4_L002_R2_001_val_2.fq.gz\n",
      "Started analysis of NR006_S3_L001_R2_001_val_2.fq.gz\n",
      "Approx 90% complete for NR005_S4_L002_R1_001_val_1.fq.gz\n",
      "Approx 60% complete for NR005_S4_L002_R2_001_val_2.fq.gz\n",
      "Approx 95% complete for NR005_S4_L002_R1_001_val_1.fq.gz\n",
      "Approx 65% complete for NR005_S4_L002_R2_001_val_2.fq.gz\n",
      "Approx 70% complete for NR005_S4_L002_R2_001_val_2.fq.gz\n",
      "Approx 75% complete for NR005_S4_L002_R2_001_val_2.fq.gz\n",
      "Started analysis of NR006_S3_L002_R1_001_val_1.fq.gz\n",
      "Approx 80% complete for NR005_S4_L002_R2_001_val_2.fq.gz\n",
      "Approx 85% complete for NR005_S4_L002_R2_001_val_2.fq.gz\n",
      "Approx 90% complete for NR005_S4_L002_R2_001_val_2.fq.gz\n",
      "Approx 95% complete for NR005_S4_L002_R2_001_val_2.fq.gz\n",
      "Started analysis of NR006_S3_L002_R2_001_val_2.fq.gz\n",
      "Approx 5% complete for NR006_S3_L001_R1_001_val_1.fq.gz\n",
      "Approx 5% complete for NR006_S3_L002_R1_001_val_1.fq.gz\n",
      "Approx 5% complete for NR006_S3_L001_R2_001_val_2.fq.gz\n",
      "Approx 5% complete for NR006_S3_L002_R2_001_val_2.fq.gz\n",
      "Approx 10% complete for NR006_S3_L002_R1_001_val_1.fq.gz\n",
      "Approx 10% complete for NR006_S3_L001_R1_001_val_1.fq.gz\n",
      "Approx 10% complete for NR006_S3_L002_R2_001_val_2.fq.gz\n",
      "Approx 10% complete for NR006_S3_L001_R2_001_val_2.fq.gz\n",
      "Started analysis of NR012_S1_L001_R1_001_val_1.fq.gz\n",
      "Approx 5% complete for NR012_S1_L001_R1_001_val_1.fq.gz\n",
      "Started analysis of NR012_S1_L001_R2_001_val_2.fq.gz\n",
      "Approx 10% complete for NR012_S1_L001_R1_001_val_1.fq.gz\n",
      "Approx 15% complete for NR006_S3_L002_R1_001_val_1.fq.gz\n",
      "Approx 5% complete for NR012_S1_L001_R2_001_val_2.fq.gz\n",
      "Started analysis of NR012_S1_L002_R1_001_val_1.fq.gz\n",
      "Approx 15% complete for NR012_S1_L001_R1_001_val_1.fq.gz\n",
      "Approx 10% complete for NR012_S1_L001_R2_001_val_2.fq.gz\n",
      "Approx 15% complete for NR006_S3_L002_R2_001_val_2.fq.gz\n",
      "Approx 20% complete for NR012_S1_L001_R1_001_val_1.fq.gz\n",
      "Started analysis of NR012_S1_L002_R2_001_val_2.fq.gz\n",
      "Approx 5% complete for NR012_S1_L002_R1_001_val_1.fq.gz\n",
      "Approx 15% complete for NR012_S1_L001_R2_001_val_2.fq.gz\n",
      "Approx 15% complete for NR006_S3_L001_R1_001_val_1.fq.gz\n",
      "Approx 25% complete for NR012_S1_L001_R1_001_val_1.fq.gz\n",
      "Approx 10% complete for NR012_S1_L002_R1_001_val_1.fq.gz\n",
      "Approx 5% complete for NR012_S1_L002_R2_001_val_2.fq.gz\n",
      "Approx 20% complete for NR012_S1_L001_R2_001_val_2.fq.gz\n",
      "Approx 30% complete for NR012_S1_L001_R1_001_val_1.fq.gz\n",
      "Approx 15% complete for NR012_S1_L002_R1_001_val_1.fq.gz\n",
      "Approx 25% complete for NR012_S1_L001_R2_001_val_2.fq.gz\n",
      "Approx 10% complete for NR012_S1_L002_R2_001_val_2.fq.gz\n",
      "Approx 15% complete for NR006_S3_L001_R2_001_val_2.fq.gz\n",
      "Approx 35% complete for NR012_S1_L001_R1_001_val_1.fq.gz\n",
      "Approx 20% complete for NR012_S1_L002_R1_001_val_1.fq.gz\n",
      "Approx 30% complete for NR012_S1_L001_R2_001_val_2.fq.gz\n",
      "Approx 15% complete for NR012_S1_L002_R2_001_val_2.fq.gz\n",
      "Approx 40% complete for NR012_S1_L001_R1_001_val_1.fq.gz\n",
      "Approx 25% complete for NR012_S1_L002_R1_001_val_1.fq.gz\n",
      "Approx 35% complete for NR012_S1_L001_R2_001_val_2.fq.gz\n",
      "Approx 45% complete for NR012_S1_L001_R1_001_val_1.fq.gz\n",
      "Approx 20% complete for NR012_S1_L002_R2_001_val_2.fq.gz\n",
      "Approx 50% complete for NR012_S1_L001_R1_001_val_1.fq.gz\n",
      "Approx 30% complete for NR012_S1_L002_R1_001_val_1.fq.gz\n",
      "Approx 40% complete for NR012_S1_L001_R2_001_val_2.fq.gz\n",
      "Approx 25% complete for NR012_S1_L002_R2_001_val_2.fq.gz\n",
      "Approx 55% complete for NR012_S1_L001_R1_001_val_1.fq.gz\n",
      "Approx 35% complete for NR012_S1_L002_R1_001_val_1.fq.gz\n",
      "Approx 45% complete for NR012_S1_L001_R2_001_val_2.fq.gz\n",
      "Approx 60% complete for NR012_S1_L001_R1_001_val_1.fq.gz\n",
      "Approx 30% complete for NR012_S1_L002_R2_001_val_2.fq.gz\n",
      "Approx 40% complete for NR012_S1_L002_R1_001_val_1.fq.gz\n",
      "Approx 50% complete for NR012_S1_L001_R2_001_val_2.fq.gz\n",
      "Approx 65% complete for NR012_S1_L001_R1_001_val_1.fq.gz\n",
      "Approx 45% complete for NR012_S1_L002_R1_001_val_1.fq.gz\n",
      "Approx 35% complete for NR012_S1_L002_R2_001_val_2.fq.gz\n",
      "Approx 20% complete for NR006_S3_L002_R1_001_val_1.fq.gz\n",
      "Approx 55% complete for NR012_S1_L001_R2_001_val_2.fq.gz\n",
      "Approx 70% complete for NR012_S1_L001_R1_001_val_1.fq.gz\n",
      "Approx 50% complete for NR012_S1_L002_R1_001_val_1.fq.gz\n",
      "Approx 60% complete for NR012_S1_L001_R2_001_val_2.fq.gz\n",
      "Approx 40% complete for NR012_S1_L002_R2_001_val_2.fq.gz\n",
      "Approx 75% complete for NR012_S1_L001_R1_001_val_1.fq.gz\n",
      "Approx 20% complete for NR006_S3_L002_R2_001_val_2.fq.gz\n",
      "Approx 80% complete for NR012_S1_L001_R1_001_val_1.fq.gz\n",
      "Approx 65% complete for NR012_S1_L001_R2_001_val_2.fq.gz\n",
      "Approx 55% complete for NR012_S1_L002_R1_001_val_1.fq.gz\n",
      "Approx 45% complete for NR012_S1_L002_R2_001_val_2.fq.gz\n",
      "Approx 85% complete for NR012_S1_L001_R1_001_val_1.fq.gz\n",
      "Approx 70% complete for NR012_S1_L001_R2_001_val_2.fq.gz\n",
      "Approx 60% complete for NR012_S1_L002_R1_001_val_1.fq.gz\n",
      "Approx 50% complete for NR012_S1_L002_R2_001_val_2.fq.gz\n",
      "Approx 90% complete for NR012_S1_L001_R1_001_val_1.fq.gz\n",
      "Approx 75% complete for NR012_S1_L001_R2_001_val_2.fq.gz\n",
      "Approx 65% complete for NR012_S1_L002_R1_001_val_1.fq.gz\n",
      "Approx 20% complete for NR006_S3_L001_R1_001_val_1.fq.gz\n",
      "Approx 55% complete for NR012_S1_L002_R2_001_val_2.fq.gz\n",
      "Approx 95% complete for NR012_S1_L001_R1_001_val_1.fq.gz\n",
      "Approx 80% complete for NR012_S1_L001_R2_001_val_2.fq.gz\n",
      "Approx 70% complete for NR012_S1_L002_R1_001_val_1.fq.gz\n",
      "Approx 60% complete for NR012_S1_L002_R2_001_val_2.fq.gz\n",
      "Approx 85% complete for NR012_S1_L001_R2_001_val_2.fq.gz\n",
      "Approx 75% complete for NR012_S1_L002_R1_001_val_1.fq.gz\n",
      "Approx 65% complete for NR012_S1_L002_R2_001_val_2.fq.gz\n",
      "Approx 20% complete for NR006_S3_L001_R2_001_val_2.fq.gz\n",
      "Approx 90% complete for NR012_S1_L001_R2_001_val_2.fq.gz\n",
      "Approx 95% complete for NR012_S1_L001_R2_001_val_2.fq.gz\n",
      "Approx 80% complete for NR012_S1_L002_R1_001_val_1.fq.gz\n",
      "Approx 70% complete for NR012_S1_L002_R2_001_val_2.fq.gz\n",
      "Approx 85% complete for NR012_S1_L002_R1_001_val_1.fq.gz\n",
      "Approx 75% complete for NR012_S1_L002_R2_001_val_2.fq.gz\n",
      "Approx 90% complete for NR012_S1_L002_R1_001_val_1.fq.gz\n",
      "Approx 80% complete for NR012_S1_L002_R2_001_val_2.fq.gz\n",
      "Approx 25% complete for NR006_S3_L002_R1_001_val_1.fq.gz\n",
      "Approx 95% complete for NR012_S1_L002_R1_001_val_1.fq.gz\n",
      "Approx 85% complete for NR012_S1_L002_R2_001_val_2.fq.gz\n",
      "Approx 25% complete for NR006_S3_L002_R2_001_val_2.fq.gz\n",
      "Approx 90% complete for NR012_S1_L002_R2_001_val_2.fq.gz\n",
      "Approx 95% complete for NR012_S1_L002_R2_001_val_2.fq.gz\n",
      "Started analysis of NR013_AD013_S2_L001_R1_001_val_1.fq.gz\n",
      "Started analysis of NR013_AD013_S2_L001_R2_001_val_2.fq.gz\n",
      "Approx 25% complete for NR006_S3_L001_R1_001_val_1.fq.gz\n",
      "Approx 25% complete for NR006_S3_L001_R2_001_val_2.fq.gz\n",
      "Started analysis of NR013_AD013_S2_L002_R1_001_val_1.fq.gz\n",
      "Approx 30% complete for NR006_S3_L002_R1_001_val_1.fq.gz\n",
      "Approx 30% complete for NR006_S3_L002_R2_001_val_2.fq.gz\n",
      "Started analysis of NR013_AD013_S2_L002_R2_001_val_2.fq.gz\n",
      "Approx 30% complete for NR006_S3_L001_R1_001_val_1.fq.gz\n",
      "Approx 30% complete for NR006_S3_L001_R2_001_val_2.fq.gz\n",
      "Approx 35% complete for NR006_S3_L002_R1_001_val_1.fq.gz\n",
      "Approx 35% complete for NR006_S3_L002_R2_001_val_2.fq.gz\n",
      "Approx 35% complete for NR006_S3_L001_R1_001_val_1.fq.gz\n",
      "Approx 40% complete for NR006_S3_L002_R1_001_val_1.fq.gz\n",
      "Approx 40% complete for NR006_S3_L002_R2_001_val_2.fq.gz\n",
      "Approx 35% complete for NR006_S3_L001_R2_001_val_2.fq.gz\n",
      "Approx 45% complete for NR006_S3_L002_R1_001_val_1.fq.gz\n",
      "Approx 45% complete for NR006_S3_L002_R2_001_val_2.fq.gz\n",
      "Approx 40% complete for NR006_S3_L001_R1_001_val_1.fq.gz\n",
      "Approx 40% complete for NR006_S3_L001_R2_001_val_2.fq.gz\n",
      "Approx 50% complete for NR006_S3_L002_R1_001_val_1.fq.gz\n",
      "Approx 50% complete for NR006_S3_L002_R2_001_val_2.fq.gz\n",
      "Approx 45% complete for NR006_S3_L001_R1_001_val_1.fq.gz\n",
      "Approx 45% complete for NR006_S3_L001_R2_001_val_2.fq.gz\n",
      "Approx 55% complete for NR006_S3_L002_R1_001_val_1.fq.gz\n",
      "Approx 55% complete for NR006_S3_L002_R2_001_val_2.fq.gz\n",
      "Approx 50% complete for NR006_S3_L001_R1_001_val_1.fq.gz\n",
      "Approx 50% complete for NR006_S3_L001_R2_001_val_2.fq.gz\n",
      "Approx 60% complete for NR006_S3_L002_R1_001_val_1.fq.gz\n",
      "Approx 60% complete for NR006_S3_L002_R2_001_val_2.fq.gz\n",
      "Approx 55% complete for NR006_S3_L001_R1_001_val_1.fq.gz\n",
      "Approx 65% complete for NR006_S3_L002_R1_001_val_1.fq.gz\n",
      "Approx 55% complete for NR006_S3_L001_R2_001_val_2.fq.gz\n",
      "Approx 65% complete for NR006_S3_L002_R2_001_val_2.fq.gz\n",
      "Approx 60% complete for NR006_S3_L001_R1_001_val_1.fq.gz\n",
      "Approx 70% complete for NR006_S3_L002_R1_001_val_1.fq.gz\n",
      "Approx 60% complete for NR006_S3_L001_R2_001_val_2.fq.gz\n",
      "Approx 70% complete for NR006_S3_L002_R2_001_val_2.fq.gz\n",
      "Approx 65% complete for NR006_S3_L001_R1_001_val_1.fq.gz\n",
      "Approx 75% complete for NR006_S3_L002_R1_001_val_1.fq.gz\n",
      "Approx 75% complete for NR006_S3_L002_R2_001_val_2.fq.gz\n",
      "Approx 65% complete for NR006_S3_L001_R2_001_val_2.fq.gz\n",
      "Approx 70% complete for NR006_S3_L001_R1_001_val_1.fq.gz\n",
      "Approx 80% complete for NR006_S3_L002_R1_001_val_1.fq.gz\n",
      "Approx 80% complete for NR006_S3_L002_R2_001_val_2.fq.gz\n",
      "Approx 70% complete for NR006_S3_L001_R2_001_val_2.fq.gz\n",
      "Approx 85% complete for NR006_S3_L002_R1_001_val_1.fq.gz\n",
      "Approx 75% complete for NR006_S3_L001_R1_001_val_1.fq.gz\n",
      "Approx 85% complete for NR006_S3_L002_R2_001_val_2.fq.gz\n",
      "Approx 75% complete for NR006_S3_L001_R2_001_val_2.fq.gz\n",
      "Approx 90% complete for NR006_S3_L002_R1_001_val_1.fq.gz\n",
      "Approx 80% complete for NR006_S3_L001_R1_001_val_1.fq.gz\n",
      "Approx 90% complete for NR006_S3_L002_R2_001_val_2.fq.gz\n",
      "Approx 95% complete for NR006_S3_L002_R1_001_val_1.fq.gz\n",
      "Approx 80% complete for NR006_S3_L001_R2_001_val_2.fq.gz\n",
      "Approx 95% complete for NR006_S3_L002_R2_001_val_2.fq.gz\n",
      "Approx 85% complete for NR006_S3_L001_R1_001_val_1.fq.gz\n",
      "Approx 85% complete for NR006_S3_L001_R2_001_val_2.fq.gz\n",
      "Started analysis of NR014_AD014_S5_L001_R1_001_val_1.fq.gz\n",
      "Approx 90% complete for NR006_S3_L001_R1_001_val_1.fq.gz\n",
      "Approx 90% complete for NR006_S3_L001_R2_001_val_2.fq.gz\n",
      "Started analysis of NR014_AD014_S5_L001_R2_001_val_2.fq.gz\n",
      "Approx 95% complete for NR006_S3_L001_R1_001_val_1.fq.gz\n",
      "Approx 95% complete for NR006_S3_L001_R2_001_val_2.fq.gz\n",
      "Started analysis of NR014_AD014_S5_L002_R1_001_val_1.fq.gz\n",
      "Started analysis of NR014_AD014_S5_L002_R2_001_val_2.fq.gz\n",
      "Approx 5% complete for AD002_S9_L001_R1_001_val_1.fq.gz\n",
      "Approx 5% complete for AD002_S9_L001_R2_001_val_2.fq.gz\n",
      "Approx 5% complete for AD002_S9_L002_R1_001_val_1.fq.gz\n",
      "Approx 5% complete for AD002_S9_L002_R2_001_val_2.fq.gz\n",
      "Approx 10% complete for AD002_S9_L001_R1_001_val_1.fq.gz\n",
      "Approx 10% complete for AD002_S9_L001_R2_001_val_2.fq.gz\n",
      "Approx 5% complete for NR013_AD013_S2_L001_R1_001_val_1.fq.gz\n",
      "Approx 5% complete for NR013_AD013_S2_L001_R2_001_val_2.fq.gz\n",
      "Approx 10% complete for AD002_S9_L002_R1_001_val_1.fq.gz\n",
      "Approx 10% complete for AD002_S9_L002_R2_001_val_2.fq.gz\n",
      "Approx 5% complete for NR013_AD013_S2_L002_R1_001_val_1.fq.gz\n",
      "Approx 5% complete for NR013_AD013_S2_L002_R2_001_val_2.fq.gz\n",
      "Approx 15% complete for AD002_S9_L001_R1_001_val_1.fq.gz\n",
      "Approx 15% complete for AD002_S9_L001_R2_001_val_2.fq.gz\n",
      "Approx 15% complete for AD002_S9_L002_R1_001_val_1.fq.gz\n",
      "Approx 15% complete for AD002_S9_L002_R2_001_val_2.fq.gz\n",
      "Approx 20% complete for AD002_S9_L001_R1_001_val_1.fq.gz\n",
      "Approx 10% complete for NR013_AD013_S2_L001_R1_001_val_1.fq.gz\n",
      "Approx 20% complete for AD002_S9_L001_R2_001_val_2.fq.gz\n",
      "Approx 10% complete for NR013_AD013_S2_L001_R2_001_val_2.fq.gz\n",
      "Approx 20% complete for AD002_S9_L002_R1_001_val_1.fq.gz\n",
      "Approx 25% complete for AD002_S9_L001_R1_001_val_1.fq.gz\n",
      "Approx 25% complete for AD002_S9_L001_R2_001_val_2.fq.gz\n",
      "Approx 20% complete for AD002_S9_L002_R2_001_val_2.fq.gz\n",
      "Approx 10% complete for NR013_AD013_S2_L002_R1_001_val_1.fq.gz\n",
      "Approx 10% complete for NR013_AD013_S2_L002_R2_001_val_2.fq.gz\n",
      "Approx 15% complete for NR013_AD013_S2_L001_R1_001_val_1.fq.gz\n",
      "Approx 30% complete for AD002_S9_L001_R1_001_val_1.fq.gz\n",
      "Approx 25% complete for AD002_S9_L002_R1_001_val_1.fq.gz\n",
      "Approx 30% complete for AD002_S9_L001_R2_001_val_2.fq.gz\n",
      "Approx 15% complete for NR013_AD013_S2_L001_R2_001_val_2.fq.gz\n",
      "Approx 25% complete for AD002_S9_L002_R2_001_val_2.fq.gz\n",
      "Approx 5% complete for NR014_AD014_S5_L001_R1_001_val_1.fq.gz\n",
      "Approx 35% complete for AD002_S9_L001_R1_001_val_1.fq.gz\n",
      "Approx 5% complete for NR014_AD014_S5_L001_R2_001_val_2.fq.gz\n",
      "Approx 35% complete for AD002_S9_L001_R2_001_val_2.fq.gz\n",
      "Approx 30% complete for AD002_S9_L002_R1_001_val_1.fq.gz\n",
      "Approx 5% complete for NR014_AD014_S5_L002_R1_001_val_1.fq.gz\n",
      "Approx 15% complete for NR013_AD013_S2_L002_R1_001_val_1.fq.gz\n",
      "Approx 30% complete for AD002_S9_L002_R2_001_val_2.fq.gz\n",
      "Approx 20% complete for NR013_AD013_S2_L001_R1_001_val_1.fq.gz\n",
      "Approx 40% complete for AD002_S9_L001_R1_001_val_1.fq.gz\n",
      "Approx 5% complete for NR014_AD014_S5_L002_R2_001_val_2.fq.gz\n",
      "Approx 15% complete for NR013_AD013_S2_L002_R2_001_val_2.fq.gz\n",
      "Approx 40% complete for AD002_S9_L001_R2_001_val_2.fq.gz\n",
      "Approx 20% complete for NR013_AD013_S2_L001_R2_001_val_2.fq.gz\n",
      "Approx 35% complete for AD002_S9_L002_R1_001_val_1.fq.gz\n",
      "Approx 45% complete for AD002_S9_L001_R1_001_val_1.fq.gz\n",
      "Approx 35% complete for AD002_S9_L002_R2_001_val_2.fq.gz\n",
      "Approx 45% complete for AD002_S9_L001_R2_001_val_2.fq.gz\n",
      "Approx 25% complete for NR013_AD013_S2_L001_R1_001_val_1.fq.gz\n",
      "Approx 40% complete for AD002_S9_L002_R1_001_val_1.fq.gz\n",
      "Approx 50% complete for AD002_S9_L001_R1_001_val_1.fq.gz\n",
      "Approx 20% complete for NR013_AD013_S2_L002_R1_001_val_1.fq.gz\n",
      "Approx 50% complete for AD002_S9_L001_R2_001_val_2.fq.gz\n",
      "Approx 40% complete for AD002_S9_L002_R2_001_val_2.fq.gz\n",
      "Approx 25% complete for NR013_AD013_S2_L001_R2_001_val_2.fq.gz\n",
      "Approx 20% complete for NR013_AD013_S2_L002_R2_001_val_2.fq.gz\n",
      "Approx 55% complete for AD002_S9_L001_R1_001_val_1.fq.gz\n",
      "Approx 45% complete for AD002_S9_L002_R1_001_val_1.fq.gz\n",
      "Approx 10% complete for NR014_AD014_S5_L001_R1_001_val_1.fq.gz\n",
      "Approx 55% complete for AD002_S9_L001_R2_001_val_2.fq.gz\n",
      "Approx 45% complete for AD002_S9_L002_R2_001_val_2.fq.gz\n",
      "Approx 30% complete for NR013_AD013_S2_L001_R1_001_val_1.fq.gz\n",
      "Approx 10% complete for NR014_AD014_S5_L001_R2_001_val_2.fq.gz\n",
      "Approx 60% complete for AD002_S9_L001_R1_001_val_1.fq.gz\n",
      "Approx 50% complete for AD002_S9_L002_R1_001_val_1.fq.gz\n",
      "Approx 25% complete for NR013_AD013_S2_L002_R1_001_val_1.fq.gz\n",
      "Approx 10% complete for NR014_AD014_S5_L002_R1_001_val_1.fq.gz\n",
      "Approx 60% complete for AD002_S9_L001_R2_001_val_2.fq.gz\n",
      "Approx 30% complete for NR013_AD013_S2_L001_R2_001_val_2.fq.gz\n",
      "Approx 65% complete for AD002_S9_L001_R1_001_val_1.fq.gz\n",
      "Approx 25% complete for NR013_AD013_S2_L002_R2_001_val_2.fq.gz\n",
      "Approx 10% complete for NR014_AD014_S5_L002_R2_001_val_2.fq.gz\n",
      "Approx 50% complete for AD002_S9_L002_R2_001_val_2.fq.gz\n",
      "Approx 65% complete for AD002_S9_L001_R2_001_val_2.fq.gz\n",
      "Approx 55% complete for AD002_S9_L002_R1_001_val_1.fq.gz\n",
      "Approx 35% complete for NR013_AD013_S2_L001_R1_001_val_1.fq.gz\n",
      "Approx 70% complete for AD002_S9_L001_R1_001_val_1.fq.gz\n",
      "Approx 55% complete for AD002_S9_L002_R2_001_val_2.fq.gz\n",
      "Approx 70% complete for AD002_S9_L001_R2_001_val_2.fq.gz\n",
      "Approx 30% complete for NR013_AD013_S2_L002_R1_001_val_1.fq.gz\n",
      "Approx 35% complete for NR013_AD013_S2_L001_R2_001_val_2.fq.gz\n",
      "Approx 60% complete for AD002_S9_L002_R1_001_val_1.fq.gz\n",
      "Approx 75% complete for AD002_S9_L001_R1_001_val_1.fq.gz\n",
      "Approx 30% complete for NR013_AD013_S2_L002_R2_001_val_2.fq.gz\n",
      "Approx 15% complete for NR014_AD014_S5_L001_R1_001_val_1.fq.gz\n",
      "Approx 75% complete for AD002_S9_L001_R2_001_val_2.fq.gz\n",
      "Approx 60% complete for AD002_S9_L002_R2_001_val_2.fq.gz\n",
      "Approx 40% complete for NR013_AD013_S2_L001_R1_001_val_1.fq.gz\n",
      "Approx 80% complete for AD002_S9_L001_R1_001_val_1.fq.gz\n",
      "Approx 65% complete for AD002_S9_L002_R1_001_val_1.fq.gz\n",
      "Approx 15% complete for NR014_AD014_S5_L001_R2_001_val_2.fq.gz\n",
      "Approx 80% complete for AD002_S9_L001_R2_001_val_2.fq.gz\n",
      "Approx 40% complete for NR013_AD013_S2_L001_R2_001_val_2.fq.gz\n",
      "Approx 85% complete for AD002_S9_L001_R1_001_val_1.fq.gz\n",
      "Approx 35% complete for NR013_AD013_S2_L002_R1_001_val_1.fq.gz\n",
      "Approx 65% complete for AD002_S9_L002_R2_001_val_2.fq.gz\n",
      "Approx 15% complete for NR014_AD014_S5_L002_R1_001_val_1.fq.gz\n",
      "Approx 70% complete for AD002_S9_L002_R1_001_val_1.fq.gz\n",
      "Approx 45% complete for NR013_AD013_S2_L001_R1_001_val_1.fq.gz\n",
      "Approx 85% complete for AD002_S9_L001_R2_001_val_2.fq.gz\n",
      "Approx 90% complete for AD002_S9_L001_R1_001_val_1.fq.gz\n",
      "Approx 35% complete for NR013_AD013_S2_L002_R2_001_val_2.fq.gz\n",
      "Approx 15% complete for NR014_AD014_S5_L002_R2_001_val_2.fq.gz\n",
      "Approx 70% complete for AD002_S9_L002_R2_001_val_2.fq.gz\n",
      "Approx 75% complete for AD002_S9_L002_R1_001_val_1.fq.gz\n",
      "Approx 90% complete for AD002_S9_L001_R2_001_val_2.fq.gz\n",
      "Approx 95% complete for AD002_S9_L001_R1_001_val_1.fq.gz\n",
      "Approx 45% complete for NR013_AD013_S2_L001_R2_001_val_2.fq.gz\n",
      "Approx 40% complete for NR013_AD013_S2_L002_R1_001_val_1.fq.gz\n",
      "Approx 50% complete for NR013_AD013_S2_L001_R1_001_val_1.fq.gz\n",
      "Approx 75% complete for AD002_S9_L002_R2_001_val_2.fq.gz\n",
      "Approx 20% complete for NR014_AD014_S5_L001_R1_001_val_1.fq.gz\n",
      "Approx 95% complete for AD002_S9_L001_R2_001_val_2.fq.gz\n",
      "Approx 80% complete for AD002_S9_L002_R1_001_val_1.fq.gz\n",
      "Started analysis of NR015_AD015_S6_L001_R1_001_val_1.fq.gz\n",
      "Approx 40% complete for NR013_AD013_S2_L002_R2_001_val_2.fq.gz\n",
      "Approx 20% complete for NR014_AD014_S5_L001_R2_001_val_2.fq.gz\n",
      "Started analysis of NR015_AD015_S6_L001_R2_001_val_2.fq.gz\n",
      "Approx 80% complete for AD002_S9_L002_R2_001_val_2.fq.gz\n",
      "Approx 50% complete for NR013_AD013_S2_L001_R2_001_val_2.fq.gz\n",
      "Approx 85% complete for AD002_S9_L002_R1_001_val_1.fq.gz\n",
      "Approx 5% complete for NR015_AD015_S6_L001_R1_001_val_1.fq.gz\n",
      "Approx 45% complete for NR013_AD013_S2_L002_R1_001_val_1.fq.gz\n",
      "Approx 55% complete for NR013_AD013_S2_L001_R1_001_val_1.fq.gz\n",
      "Approx 20% complete for NR014_AD014_S5_L002_R1_001_val_1.fq.gz\n",
      "Approx 5% complete for NR015_AD015_S6_L001_R2_001_val_2.fq.gz\n",
      "Approx 85% complete for AD002_S9_L002_R2_001_val_2.fq.gz\n",
      "Approx 90% complete for AD002_S9_L002_R1_001_val_1.fq.gz\n",
      "Approx 45% complete for NR013_AD013_S2_L002_R2_001_val_2.fq.gz\n",
      "Approx 10% complete for NR015_AD015_S6_L001_R1_001_val_1.fq.gz\n",
      "Approx 20% complete for NR014_AD014_S5_L002_R2_001_val_2.fq.gz\n",
      "Approx 55% complete for NR013_AD013_S2_L001_R2_001_val_2.fq.gz\n",
      "Approx 50% complete for NR013_AD013_S2_L002_R1_001_val_1.fq.gz\n",
      "Approx 25% complete for NR014_AD014_S5_L001_R1_001_val_1.fq.gz\n",
      "Approx 10% complete for NR015_AD015_S6_L001_R2_001_val_2.fq.gz\n",
      "Approx 60% complete for NR013_AD013_S2_L001_R1_001_val_1.fq.gz\n",
      "Approx 15% complete for NR015_AD015_S6_L001_R1_001_val_1.fq.gz\n",
      "Approx 95% complete for AD002_S9_L002_R1_001_val_1.fq.gz\n",
      "Approx 90% complete for AD002_S9_L002_R2_001_val_2.fq.gz\n",
      "Approx 50% complete for NR013_AD013_S2_L002_R2_001_val_2.fq.gz\n",
      "Approx 25% complete for NR014_AD014_S5_L001_R2_001_val_2.fq.gz\n",
      "Approx 15% complete for NR015_AD015_S6_L001_R2_001_val_2.fq.gz\n",
      "Approx 20% complete for NR015_AD015_S6_L001_R1_001_val_1.fq.gz\n",
      "Approx 100% complete for AD002_S9_L002_R1_001_val_1.fq.gz\n",
      "Started analysis of NR015_AD015_S6_L002_R1_001_val_1.fq.gz\n",
      "Approx 95% complete for AD002_S9_L002_R2_001_val_2.fq.gz\n",
      "Approx 55% complete for NR013_AD013_S2_L002_R1_001_val_1.fq.gz\n",
      "Approx 60% complete for NR013_AD013_S2_L001_R2_001_val_2.fq.gz\n",
      "Approx 25% complete for NR014_AD014_S5_L002_R1_001_val_1.fq.gz\n",
      "Approx 25% complete for NR015_AD015_S6_L001_R1_001_val_1.fq.gz\n",
      "Approx 65% complete for NR013_AD013_S2_L001_R1_001_val_1.fq.gz\n",
      "Approx 20% complete for NR015_AD015_S6_L001_R2_001_val_2.fq.gz\n",
      "Approx 5% complete for NR015_AD015_S6_L002_R1_001_val_1.fq.gz\n",
      "Approx 55% complete for NR013_AD013_S2_L002_R2_001_val_2.fq.gz\n",
      "Approx 30% complete for NR014_AD014_S5_L001_R1_001_val_1.fq.gz\n",
      "Approx 25% complete for NR014_AD014_S5_L002_R2_001_val_2.fq.gz\n",
      "Approx 60% complete for NR013_AD013_S2_L002_R1_001_val_1.fq.gz\n",
      "Approx 30% complete for NR015_AD015_S6_L001_R1_001_val_1.fq.gz\n",
      "Approx 25% complete for NR015_AD015_S6_L001_R2_001_val_2.fq.gz\n",
      "Approx 10% complete for NR015_AD015_S6_L002_R1_001_val_1.fq.gz\n",
      "Approx 100% complete for AD002_S9_L002_R2_001_val_2.fq.gz\n",
      "Started analysis of NR015_AD015_S6_L002_R2_001_val_2.fq.gz\n",
      "Approx 30% complete for NR014_AD014_S5_L001_R2_001_val_2.fq.gz\n",
      "Approx 35% complete for NR015_AD015_S6_L001_R1_001_val_1.fq.gz\n",
      "Approx 60% complete for NR013_AD013_S2_L002_R2_001_val_2.fq.gz\n",
      "Approx 30% complete for NR015_AD015_S6_L001_R2_001_val_2.fq.gz\n",
      "Approx 15% complete for NR015_AD015_S6_L002_R1_001_val_1.fq.gz\n",
      "Approx 65% complete for NR013_AD013_S2_L002_R1_001_val_1.fq.gz\n",
      "Approx 5% complete for NR015_AD015_S6_L002_R2_001_val_2.fq.gz\n",
      "Approx 40% complete for NR015_AD015_S6_L001_R1_001_val_1.fq.gz\n",
      "Approx 30% complete for NR014_AD014_S5_L002_R1_001_val_1.fq.gz\n",
      "Approx 35% complete for NR015_AD015_S6_L001_R2_001_val_2.fq.gz\n",
      "Approx 20% complete for NR015_AD015_S6_L002_R1_001_val_1.fq.gz\n",
      "Approx 35% complete for NR014_AD014_S5_L001_R1_001_val_1.fq.gz\n",
      "Approx 65% complete for NR013_AD013_S2_L002_R2_001_val_2.fq.gz\n",
      "Approx 10% complete for NR015_AD015_S6_L002_R2_001_val_2.fq.gz\n",
      "Approx 45% complete for NR015_AD015_S6_L001_R1_001_val_1.fq.gz\n",
      "Approx 70% complete for NR013_AD013_S2_L002_R1_001_val_1.fq.gz\n",
      "Approx 40% complete for NR015_AD015_S6_L001_R2_001_val_2.fq.gz\n",
      "Approx 30% complete for NR014_AD014_S5_L002_R2_001_val_2.fq.gz\n",
      "Approx 25% complete for NR015_AD015_S6_L002_R1_001_val_1.fq.gz\n",
      "Approx 50% complete for NR015_AD015_S6_L001_R1_001_val_1.fq.gz\n",
      "Approx 15% complete for NR015_AD015_S6_L002_R2_001_val_2.fq.gz\n",
      "Approx 35% complete for NR014_AD014_S5_L001_R2_001_val_2.fq.gz\n",
      "Approx 65% complete for NR013_AD013_S2_L001_R2_001_val_2.fq.gz\n",
      "Approx 45% complete for NR015_AD015_S6_L001_R2_001_val_2.fq.gz\n",
      "Approx 70% complete for NR013_AD013_S2_L002_R2_001_val_2.fq.gz\n",
      "Approx 75% complete for NR013_AD013_S2_L002_R1_001_val_1.fq.gz\n",
      "Approx 70% complete for NR013_AD013_S2_L001_R1_001_val_1.fq.gz\n",
      "Approx 30% complete for NR015_AD015_S6_L002_R1_001_val_1.fq.gz\n",
      "Approx 55% complete for NR015_AD015_S6_L001_R1_001_val_1.fq.gz\n",
      "Approx 20% complete for NR015_AD015_S6_L002_R2_001_val_2.fq.gz\n",
      "Approx 50% complete for NR015_AD015_S6_L001_R2_001_val_2.fq.gz\n",
      "Approx 35% complete for NR014_AD014_S5_L002_R1_001_val_1.fq.gz\n",
      "Approx 40% complete for NR014_AD014_S5_L001_R1_001_val_1.fq.gz\n",
      "Approx 60% complete for NR015_AD015_S6_L001_R1_001_val_1.fq.gz\n",
      "Approx 35% complete for NR015_AD015_S6_L002_R1_001_val_1.fq.gz\n",
      "Approx 75% complete for NR013_AD013_S2_L002_R2_001_val_2.fq.gz\n",
      "Approx 25% complete for NR015_AD015_S6_L002_R2_001_val_2.fq.gz\n",
      "Approx 55% complete for NR015_AD015_S6_L001_R2_001_val_2.fq.gz\n",
      "Approx 80% complete for NR013_AD013_S2_L002_R1_001_val_1.fq.gz\n",
      "Approx 35% complete for NR014_AD014_S5_L002_R2_001_val_2.fq.gz\n",
      "Approx 65% complete for NR015_AD015_S6_L001_R1_001_val_1.fq.gz\n",
      "Approx 40% complete for NR015_AD015_S6_L002_R1_001_val_1.fq.gz\n",
      "Approx 40% complete for NR014_AD014_S5_L001_R2_001_val_2.fq.gz\n",
      "Approx 60% complete for NR015_AD015_S6_L001_R2_001_val_2.fq.gz\n",
      "Approx 30% complete for NR015_AD015_S6_L002_R2_001_val_2.fq.gz\n",
      "Approx 70% complete for NR015_AD015_S6_L001_R1_001_val_1.fq.gz\n",
      "Approx 80% complete for NR013_AD013_S2_L002_R2_001_val_2.fq.gz\n",
      "Approx 85% complete for NR013_AD013_S2_L002_R1_001_val_1.fq.gz\n",
      "Approx 45% complete for NR015_AD015_S6_L002_R1_001_val_1.fq.gz\n",
      "Approx 65% complete for NR015_AD015_S6_L001_R2_001_val_2.fq.gz\n",
      "Approx 45% complete for NR014_AD014_S5_L001_R1_001_val_1.fq.gz\n",
      "Approx 40% complete for NR014_AD014_S5_L002_R1_001_val_1.fq.gz\n",
      "Approx 35% complete for NR015_AD015_S6_L002_R2_001_val_2.fq.gz\n",
      "Approx 75% complete for NR015_AD015_S6_L001_R1_001_val_1.fq.gz\n",
      "Approx 50% complete for NR015_AD015_S6_L002_R1_001_val_1.fq.gz\n",
      "Approx 70% complete for NR015_AD015_S6_L001_R2_001_val_2.fq.gz\n",
      "Approx 90% complete for NR013_AD013_S2_L002_R1_001_val_1.fq.gz\n",
      "Approx 85% complete for NR013_AD013_S2_L002_R2_001_val_2.fq.gz\n",
      "Approx 80% complete for NR015_AD015_S6_L001_R1_001_val_1.fq.gz\n",
      "Approx 40% complete for NR015_AD015_S6_L002_R2_001_val_2.fq.gz\n",
      "Approx 40% complete for NR014_AD014_S5_L002_R2_001_val_2.fq.gz\n",
      "Approx 45% complete for NR014_AD014_S5_L001_R2_001_val_2.fq.gz\n",
      "Approx 55% complete for NR015_AD015_S6_L002_R1_001_val_1.fq.gz\n",
      "Approx 75% complete for NR015_AD015_S6_L001_R2_001_val_2.fq.gz\n",
      "Approx 85% complete for NR015_AD015_S6_L001_R1_001_val_1.fq.gz\n",
      "Approx 45% complete for NR015_AD015_S6_L002_R2_001_val_2.fq.gz\n",
      "Approx 95% complete for NR013_AD013_S2_L002_R1_001_val_1.fq.gz\n",
      "Approx 90% complete for NR013_AD013_S2_L002_R2_001_val_2.fq.gz\n",
      "Approx 60% complete for NR015_AD015_S6_L002_R1_001_val_1.fq.gz\n",
      "Approx 50% complete for NR014_AD014_S5_L001_R1_001_val_1.fq.gz\n",
      "Approx 80% complete for NR015_AD015_S6_L001_R2_001_val_2.fq.gz\n",
      "Approx 90% complete for NR015_AD015_S6_L001_R1_001_val_1.fq.gz\n",
      "Approx 45% complete for NR014_AD014_S5_L002_R1_001_val_1.fq.gz\n",
      "Approx 50% complete for NR015_AD015_S6_L002_R2_001_val_2.fq.gz\n",
      "Approx 65% complete for NR015_AD015_S6_L002_R1_001_val_1.fq.gz\n",
      "Approx 85% complete for NR015_AD015_S6_L001_R2_001_val_2.fq.gz\n",
      "Approx 95% complete for NR015_AD015_S6_L001_R1_001_val_1.fq.gz\n",
      "Started analysis of NR019_S7_L001_R1_001_val_1.fq.gz\n",
      "Approx 5% complete for NR019_S7_L001_R1_001_val_1.fq.gz\n",
      "Approx 10% complete for NR019_S7_L001_R1_001_val_1.fq.gz\n",
      "Approx 15% complete for NR019_S7_L001_R1_001_val_1.fq.gz\n",
      "Approx 20% complete for NR019_S7_L001_R1_001_val_1.fq.gz\n",
      "Approx 25% complete for NR019_S7_L001_R1_001_val_1.fq.gz\n",
      "Approx 30% complete for NR019_S7_L001_R1_001_val_1.fq.gz\n",
      "Approx 35% complete for NR019_S7_L001_R1_001_val_1.fq.gz\n",
      "Approx 40% complete for NR019_S7_L001_R1_001_val_1.fq.gz\n",
      "Approx 45% complete for NR019_S7_L001_R1_001_val_1.fq.gz\n",
      "Approx 50% complete for NR019_S7_L001_R1_001_val_1.fq.gz\n",
      "Approx 55% complete for NR019_S7_L001_R1_001_val_1.fq.gz\n",
      "Approx 60% complete for NR019_S7_L001_R1_001_val_1.fq.gz\n",
      "Approx 65% complete for NR019_S7_L001_R1_001_val_1.fq.gz\n",
      "Approx 70% complete for NR019_S7_L001_R1_001_val_1.fq.gz\n",
      "Approx 75% complete for NR019_S7_L001_R1_001_val_1.fq.gz\n",
      "Approx 80% complete for NR019_S7_L001_R1_001_val_1.fq.gz\n",
      "Approx 85% complete for NR019_S7_L001_R1_001_val_1.fq.gz\n",
      "Approx 90% complete for NR019_S7_L001_R1_001_val_1.fq.gz\n",
      "Approx 95% complete for NR019_S7_L001_R1_001_val_1.fq.gz\n",
      "Started analysis of NR019_S7_L001_R2_001_val_2.fq.gz\n",
      "Approx 5% complete for NR019_S7_L001_R2_001_val_2.fq.gz\n",
      "Approx 10% complete for NR019_S7_L001_R2_001_val_2.fq.gz\n",
      "Approx 15% complete for NR019_S7_L001_R2_001_val_2.fq.gz\n",
      "Approx 20% complete for NR019_S7_L001_R2_001_val_2.fq.gz\n",
      "Approx 25% complete for NR019_S7_L001_R2_001_val_2.fq.gz\n",
      "Approx 30% complete for NR019_S7_L001_R2_001_val_2.fq.gz\n",
      "Approx 35% complete for NR019_S7_L001_R2_001_val_2.fq.gz\n",
      "Approx 40% complete for NR019_S7_L001_R2_001_val_2.fq.gz\n",
      "Approx 45% complete for NR019_S7_L001_R2_001_val_2.fq.gz\n",
      "Approx 50% complete for NR019_S7_L001_R2_001_val_2.fq.gz\n",
      "Approx 55% complete for NR019_S7_L001_R2_001_val_2.fq.gz\n",
      "Approx 60% complete for NR019_S7_L001_R2_001_val_2.fq.gz\n",
      "Approx 65% complete for NR019_S7_L001_R2_001_val_2.fq.gz\n",
      "Approx 70% complete for NR019_S7_L001_R2_001_val_2.fq.gz\n",
      "Approx 75% complete for NR019_S7_L001_R2_001_val_2.fq.gz\n",
      "Approx 80% complete for NR019_S7_L001_R2_001_val_2.fq.gz\n",
      "Approx 85% complete for NR019_S7_L001_R2_001_val_2.fq.gz\n",
      "Approx 90% complete for NR019_S7_L001_R2_001_val_2.fq.gz\n",
      "Approx 95% complete for NR019_S7_L001_R2_001_val_2.fq.gz\n",
      "Started analysis of NR019_S7_L002_R1_001_val_1.fq.gz\n",
      "Approx 5% complete for NR019_S7_L002_R1_001_val_1.fq.gz\n",
      "Approx 10% complete for NR019_S7_L002_R1_001_val_1.fq.gz\n",
      "Approx 15% complete for NR019_S7_L002_R1_001_val_1.fq.gz\n",
      "Approx 20% complete for NR019_S7_L002_R1_001_val_1.fq.gz\n",
      "Approx 25% complete for NR019_S7_L002_R1_001_val_1.fq.gz\n",
      "Approx 30% complete for NR019_S7_L002_R1_001_val_1.fq.gz\n",
      "Approx 35% complete for NR019_S7_L002_R1_001_val_1.fq.gz\n",
      "Approx 40% complete for NR019_S7_L002_R1_001_val_1.fq.gz\n",
      "Approx 45% complete for NR019_S7_L002_R1_001_val_1.fq.gz\n",
      "Approx 50% complete for NR019_S7_L002_R1_001_val_1.fq.gz\n",
      "Approx 55% complete for NR019_S7_L002_R1_001_val_1.fq.gz\n",
      "Approx 60% complete for NR019_S7_L002_R1_001_val_1.fq.gz\n",
      "Approx 65% complete for NR019_S7_L002_R1_001_val_1.fq.gz\n",
      "Approx 70% complete for NR019_S7_L002_R1_001_val_1.fq.gz\n",
      "Approx 75% complete for NR019_S7_L002_R1_001_val_1.fq.gz\n",
      "Approx 80% complete for NR019_S7_L002_R1_001_val_1.fq.gz\n",
      "Approx 85% complete for NR019_S7_L002_R1_001_val_1.fq.gz\n",
      "Approx 90% complete for NR019_S7_L002_R1_001_val_1.fq.gz\n",
      "Approx 95% complete for NR019_S7_L002_R1_001_val_1.fq.gz\n",
      "Started analysis of NR019_S7_L002_R2_001_val_2.fq.gz\n",
      "Approx 5% complete for NR019_S7_L002_R2_001_val_2.fq.gz\n",
      "Approx 10% complete for NR019_S7_L002_R2_001_val_2.fq.gz\n",
      "Approx 15% complete for NR019_S7_L002_R2_001_val_2.fq.gz\n",
      "Approx 20% complete for NR019_S7_L002_R2_001_val_2.fq.gz\n",
      "Approx 25% complete for NR019_S7_L002_R2_001_val_2.fq.gz\n",
      "Approx 30% complete for NR019_S7_L002_R2_001_val_2.fq.gz\n",
      "Approx 35% complete for NR019_S7_L002_R2_001_val_2.fq.gz\n",
      "Approx 40% complete for NR019_S7_L002_R2_001_val_2.fq.gz\n",
      "Approx 45% complete for NR019_S7_L002_R2_001_val_2.fq.gz\n",
      "Approx 50% complete for NR019_S7_L002_R2_001_val_2.fq.gz\n",
      "Approx 55% complete for NR019_S7_L002_R2_001_val_2.fq.gz\n",
      "Approx 60% complete for NR019_S7_L002_R2_001_val_2.fq.gz\n",
      "Approx 65% complete for NR019_S7_L002_R2_001_val_2.fq.gz\n",
      "Approx 70% complete for NR019_S7_L002_R2_001_val_2.fq.gz\n",
      "Approx 75% complete for NR019_S7_L002_R2_001_val_2.fq.gz\n",
      "Approx 80% complete for NR019_S7_L002_R2_001_val_2.fq.gz\n",
      "Approx 85% complete for NR019_S7_L002_R2_001_val_2.fq.gz\n",
      "Approx 90% complete for NR019_S7_L002_R2_001_val_2.fq.gz\n",
      "Approx 95% complete for NR019_S7_L002_R2_001_val_2.fq.gz\n",
      "Started analysis of NR021_S8_L001_R1_001_val_1.fq.gz\n",
      "Approx 5% complete for NR021_S8_L001_R1_001_val_1.fq.gz\n",
      "Approx 10% complete for NR021_S8_L001_R1_001_val_1.fq.gz\n",
      "Approx 15% complete for NR021_S8_L001_R1_001_val_1.fq.gz\n",
      "Approx 20% complete for NR021_S8_L001_R1_001_val_1.fq.gz\n",
      "Approx 25% complete for NR021_S8_L001_R1_001_val_1.fq.gz\n",
      "Approx 30% complete for NR021_S8_L001_R1_001_val_1.fq.gz\n",
      "Approx 35% complete for NR021_S8_L001_R1_001_val_1.fq.gz\n",
      "Approx 40% complete for NR021_S8_L001_R1_001_val_1.fq.gz\n",
      "Approx 45% complete for NR021_S8_L001_R1_001_val_1.fq.gz\n",
      "Approx 50% complete for NR021_S8_L001_R1_001_val_1.fq.gz\n",
      "Approx 70% complete for NR013_AD013_S2_L001_R2_001_val_2.fq.gz\n",
      "Approx 55% complete for NR021_S8_L001_R1_001_val_1.fq.gz\n",
      "Approx 75% complete for NR013_AD013_S2_L001_R1_001_val_1.fq.gz\n",
      "Approx 60% complete for NR021_S8_L001_R1_001_val_1.fq.gz\n",
      "Approx 95% complete for NR013_AD013_S2_L002_R2_001_val_2.fq.gz\n",
      "Approx 65% complete for NR021_S8_L001_R1_001_val_1.fq.gz\n",
      "Approx 70% complete for NR021_S8_L001_R1_001_val_1.fq.gz\n",
      "Approx 75% complete for NR021_S8_L001_R1_001_val_1.fq.gz\n",
      "Approx 80% complete for NR021_S8_L001_R1_001_val_1.fq.gz\n",
      "Approx 85% complete for NR021_S8_L001_R1_001_val_1.fq.gz\n",
      "Approx 90% complete for NR021_S8_L001_R1_001_val_1.fq.gz\n",
      "Approx 95% complete for NR021_S8_L001_R1_001_val_1.fq.gz\n",
      "Started analysis of NR021_S8_L001_R2_001_val_2.fq.gz\n",
      "Approx 50% complete for NR014_AD014_S5_L001_R2_001_val_2.fq.gz\n",
      "Approx 5% complete for NR021_S8_L001_R2_001_val_2.fq.gz\n",
      "Approx 10% complete for NR021_S8_L001_R2_001_val_2.fq.gz\n",
      "Approx 15% complete for NR021_S8_L001_R2_001_val_2.fq.gz\n",
      "Approx 20% complete for NR021_S8_L001_R2_001_val_2.fq.gz\n",
      "Approx 25% complete for NR021_S8_L001_R2_001_val_2.fq.gz\n",
      "Approx 30% complete for NR021_S8_L001_R2_001_val_2.fq.gz\n",
      "Approx 35% complete for NR021_S8_L001_R2_001_val_2.fq.gz\n",
      "Approx 40% complete for NR021_S8_L001_R2_001_val_2.fq.gz\n",
      "Approx 45% complete for NR021_S8_L001_R2_001_val_2.fq.gz\n",
      "Approx 55% complete for NR015_AD015_S6_L002_R2_001_val_2.fq.gz\n",
      "Approx 50% complete for NR021_S8_L001_R2_001_val_2.fq.gz\n",
      "Approx 55% complete for NR021_S8_L001_R2_001_val_2.fq.gz\n",
      "Approx 60% complete for NR021_S8_L001_R2_001_val_2.fq.gz\n",
      "Approx 65% complete for NR021_S8_L001_R2_001_val_2.fq.gz\n",
      "Approx 70% complete for NR021_S8_L001_R2_001_val_2.fq.gz\n",
      "Approx 75% complete for NR021_S8_L001_R2_001_val_2.fq.gz\n",
      "Approx 80% complete for NR021_S8_L001_R2_001_val_2.fq.gz\n",
      "Approx 85% complete for NR021_S8_L001_R2_001_val_2.fq.gz\n",
      "Approx 70% complete for NR015_AD015_S6_L002_R1_001_val_1.fq.gz\n",
      "Approx 90% complete for NR021_S8_L001_R2_001_val_2.fq.gz\n",
      "Approx 45% complete for NR014_AD014_S5_L002_R2_001_val_2.fq.gz\n",
      "Approx 95% complete for NR021_S8_L001_R2_001_val_2.fq.gz\n",
      "Approx 90% complete for NR015_AD015_S6_L001_R2_001_val_2.fq.gz\n",
      "Started analysis of NR021_S8_L002_R1_001_val_1.fq.gz\n",
      "Started analysis of NR021_S8_L002_R2_001_val_2.fq.gz\n",
      "Approx 5% complete for NR021_S8_L002_R1_001_val_1.fq.gz\n",
      "Approx 10% complete for NR021_S8_L002_R1_001_val_1.fq.gz\n",
      "Approx 5% complete for NR021_S8_L002_R2_001_val_2.fq.gz\n",
      "Approx 15% complete for NR021_S8_L002_R1_001_val_1.fq.gz\n",
      "Approx 10% complete for NR021_S8_L002_R2_001_val_2.fq.gz\n",
      "Approx 20% complete for NR021_S8_L002_R1_001_val_1.fq.gz\n",
      "Approx 15% complete for NR021_S8_L002_R2_001_val_2.fq.gz\n",
      "Approx 25% complete for NR021_S8_L002_R1_001_val_1.fq.gz\n",
      "Approx 20% complete for NR021_S8_L002_R2_001_val_2.fq.gz\n",
      "Approx 30% complete for NR021_S8_L002_R1_001_val_1.fq.gz\n",
      "Approx 25% complete for NR021_S8_L002_R2_001_val_2.fq.gz\n",
      "Approx 35% complete for NR021_S8_L002_R1_001_val_1.fq.gz\n",
      "Approx 30% complete for NR021_S8_L002_R2_001_val_2.fq.gz\n",
      "Approx 40% complete for NR021_S8_L002_R1_001_val_1.fq.gz\n",
      "Approx 35% complete for NR021_S8_L002_R2_001_val_2.fq.gz\n",
      "Approx 45% complete for NR021_S8_L002_R1_001_val_1.fq.gz\n",
      "Approx 40% complete for NR021_S8_L002_R2_001_val_2.fq.gz\n",
      "Approx 50% complete for NR021_S8_L002_R1_001_val_1.fq.gz\n",
      "Approx 45% complete for NR021_S8_L002_R2_001_val_2.fq.gz\n",
      "Approx 55% complete for NR021_S8_L002_R1_001_val_1.fq.gz\n",
      "Approx 50% complete for NR021_S8_L002_R2_001_val_2.fq.gz\n",
      "Approx 60% complete for NR021_S8_L002_R1_001_val_1.fq.gz\n",
      "Approx 55% complete for NR021_S8_L002_R2_001_val_2.fq.gz\n",
      "Approx 65% complete for NR021_S8_L002_R1_001_val_1.fq.gz\n",
      "Approx 60% complete for NR021_S8_L002_R2_001_val_2.fq.gz\n",
      "Approx 70% complete for NR021_S8_L002_R1_001_val_1.fq.gz\n",
      "Approx 65% complete for NR021_S8_L002_R2_001_val_2.fq.gz\n",
      "Approx 75% complete for NR021_S8_L002_R1_001_val_1.fq.gz\n",
      "Approx 70% complete for NR021_S8_L002_R2_001_val_2.fq.gz\n",
      "Approx 80% complete for NR021_S8_L002_R1_001_val_1.fq.gz\n",
      "Approx 75% complete for NR021_S8_L002_R2_001_val_2.fq.gz\n",
      "Approx 85% complete for NR021_S8_L002_R1_001_val_1.fq.gz\n",
      "Approx 80% complete for NR021_S8_L002_R2_001_val_2.fq.gz\n",
      "Approx 90% complete for NR021_S8_L002_R1_001_val_1.fq.gz\n",
      "Approx 85% complete for NR021_S8_L002_R2_001_val_2.fq.gz\n",
      "Approx 95% complete for NR021_S8_L002_R1_001_val_1.fq.gz\n",
      "Approx 90% complete for NR021_S8_L002_R2_001_val_2.fq.gz\n",
      "Approx 95% complete for NR021_S8_L002_R2_001_val_2.fq.gz\n",
      "Approx 55% complete for NR014_AD014_S5_L001_R1_001_val_1.fq.gz\n",
      "Approx 60% complete for NR015_AD015_S6_L002_R2_001_val_2.fq.gz\n",
      "Approx 95% complete for NR015_AD015_S6_L001_R2_001_val_2.fq.gz\n",
      "Approx 75% complete for NR015_AD015_S6_L002_R1_001_val_1.fq.gz\n",
      "Approx 50% complete for NR014_AD014_S5_L002_R1_001_val_1.fq.gz\n",
      "Approx 75% complete for NR013_AD013_S2_L001_R2_001_val_2.fq.gz\n",
      "Approx 80% complete for NR013_AD013_S2_L001_R1_001_val_1.fq.gz\n",
      "Approx 65% complete for NR015_AD015_S6_L002_R2_001_val_2.fq.gz\n",
      "Approx 80% complete for NR015_AD015_S6_L002_R1_001_val_1.fq.gz\n",
      "Approx 55% complete for NR014_AD014_S5_L001_R2_001_val_2.fq.gz\n",
      "Approx 70% complete for NR015_AD015_S6_L002_R2_001_val_2.fq.gz\n",
      "Approx 85% complete for NR013_AD013_S2_L001_R1_001_val_1.fq.gz\n",
      "Approx 80% complete for NR013_AD013_S2_L001_R2_001_val_2.fq.gz\n",
      "Approx 85% complete for NR015_AD015_S6_L002_R1_001_val_1.fq.gz\n",
      "Approx 50% complete for NR014_AD014_S5_L002_R2_001_val_2.fq.gz\n",
      "Approx 60% complete for NR014_AD014_S5_L001_R1_001_val_1.fq.gz\n",
      "Approx 75% complete for NR015_AD015_S6_L002_R2_001_val_2.fq.gz\n",
      "Approx 90% complete for NR015_AD015_S6_L002_R1_001_val_1.fq.gz\n",
      "Approx 90% complete for NR013_AD013_S2_L001_R1_001_val_1.fq.gz\n",
      "Approx 85% complete for NR013_AD013_S2_L001_R2_001_val_2.fq.gz\n",
      "Approx 55% complete for NR014_AD014_S5_L002_R1_001_val_1.fq.gz\n",
      "Approx 80% complete for NR015_AD015_S6_L002_R2_001_val_2.fq.gz\n",
      "Approx 95% complete for NR015_AD015_S6_L002_R1_001_val_1.fq.gz\n",
      "Approx 60% complete for NR014_AD014_S5_L001_R2_001_val_2.fq.gz\n",
      "Approx 95% complete for NR013_AD013_S2_L001_R1_001_val_1.fq.gz\n",
      "Approx 90% complete for NR013_AD013_S2_L001_R2_001_val_2.fq.gz\n",
      "Approx 65% complete for NR014_AD014_S5_L001_R1_001_val_1.fq.gz\n",
      "Approx 85% complete for NR015_AD015_S6_L002_R2_001_val_2.fq.gz\n",
      "Approx 55% complete for NR014_AD014_S5_L002_R2_001_val_2.fq.gz\n",
      "Approx 95% complete for NR013_AD013_S2_L001_R2_001_val_2.fq.gz\n",
      "Approx 90% complete for NR015_AD015_S6_L002_R2_001_val_2.fq.gz\n",
      "Approx 60% complete for NR014_AD014_S5_L002_R1_001_val_1.fq.gz\n",
      "Approx 65% complete for NR014_AD014_S5_L001_R2_001_val_2.fq.gz\n",
      "Approx 95% complete for NR015_AD015_S6_L002_R2_001_val_2.fq.gz\n",
      "Approx 70% complete for NR014_AD014_S5_L001_R1_001_val_1.fq.gz\n",
      "Approx 60% complete for NR014_AD014_S5_L002_R2_001_val_2.fq.gz\n",
      "Approx 70% complete for NR014_AD014_S5_L001_R2_001_val_2.fq.gz\n",
      "Approx 65% complete for NR014_AD014_S5_L002_R1_001_val_1.fq.gz\n",
      "Approx 75% complete for NR014_AD014_S5_L001_R1_001_val_1.fq.gz\n",
      "Approx 65% complete for NR014_AD014_S5_L002_R2_001_val_2.fq.gz\n",
      "Approx 75% complete for NR014_AD014_S5_L001_R2_001_val_2.fq.gz\n",
      "Approx 80% complete for NR014_AD014_S5_L001_R1_001_val_1.fq.gz\n",
      "Approx 70% complete for NR014_AD014_S5_L002_R1_001_val_1.fq.gz\n",
      "Approx 70% complete for NR014_AD014_S5_L002_R2_001_val_2.fq.gz\n",
      "Approx 80% complete for NR014_AD014_S5_L001_R2_001_val_2.fq.gz\n",
      "Approx 85% complete for NR014_AD014_S5_L001_R1_001_val_1.fq.gz\n",
      "Approx 75% complete for NR014_AD014_S5_L002_R1_001_val_1.fq.gz\n",
      "Approx 85% complete for NR014_AD014_S5_L001_R2_001_val_2.fq.gz\n",
      "Approx 90% complete for NR014_AD014_S5_L001_R1_001_val_1.fq.gz\n",
      "Approx 75% complete for NR014_AD014_S5_L002_R2_001_val_2.fq.gz\n",
      "Approx 80% complete for NR014_AD014_S5_L002_R1_001_val_1.fq.gz\n",
      "Approx 90% complete for NR014_AD014_S5_L001_R2_001_val_2.fq.gz\n",
      "Approx 95% complete for NR014_AD014_S5_L001_R1_001_val_1.fq.gz\n",
      "Approx 80% complete for NR014_AD014_S5_L002_R2_001_val_2.fq.gz\n",
      "Approx 85% complete for NR014_AD014_S5_L002_R1_001_val_1.fq.gz\n",
      "Approx 95% complete for NR014_AD014_S5_L001_R2_001_val_2.fq.gz\n",
      "Approx 85% complete for NR014_AD014_S5_L002_R2_001_val_2.fq.gz\n",
      "Approx 90% complete for NR014_AD014_S5_L002_R1_001_val_1.fq.gz\n",
      "Approx 90% complete for NR014_AD014_S5_L002_R2_001_val_2.fq.gz\n",
      "Approx 95% complete for NR014_AD014_S5_L002_R1_001_val_1.fq.gz\n",
      "Approx 95% complete for NR014_AD014_S5_L002_R2_001_val_2.fq.gz\n",
      "\n",
      "real\t282m31.564s\n",
      "user\t1338m9.276s\n",
      "sys\t25m38.300s\n"
     ]
    }
   ],
   "source": [
    "%%bash\n",
    "time fastqc --threads 12 \\\n",
    "/home/sam/data/geoduck_illumina/trimmed/AD002_S9_L001_R1_001_val_1.fq.gz \\\n",
    "/home/sam/data/geoduck_illumina/trimmed/AD002_S9_L001_R2_001_val_2.fq.gz \\\n",
    "/home/sam/data/geoduck_illumina/trimmed/AD002_S9_L002_R1_001_val_1.fq.gz \\\n",
    "/home/sam/data/geoduck_illumina/trimmed/AD002_S9_L002_R2_001_val_2.fq.gz \\\n",
    "/home/sam/data/geoduck_illumina/trimmed/NR005_S4_L001_R1_001_val_1.fq.gz \\\n",
    "/home/sam/data/geoduck_illumina/trimmed/NR005_S4_L001_R2_001_val_2.fq.gz \\\n",
    "/home/sam/data/geoduck_illumina/trimmed/NR005_S4_L002_R1_001_val_1.fq.gz \\\n",
    "/home/sam/data/geoduck_illumina/trimmed/NR005_S4_L002_R2_001_val_2.fq.gz \\\n",
    "/home/sam/data/geoduck_illumina/trimmed/NR006_S3_L001_R1_001_val_1.fq.gz \\\n",
    "/home/sam/data/geoduck_illumina/trimmed/NR006_S3_L001_R2_001_val_2.fq.gz \\\n",
    "/home/sam/data/geoduck_illumina/trimmed/NR006_S3_L002_R1_001_val_1.fq.gz \\\n",
    "/home/sam/data/geoduck_illumina/trimmed/NR006_S3_L002_R2_001_val_2.fq.gz \\\n",
    "/home/sam/data/geoduck_illumina/trimmed/NR012_S1_L001_R1_001_val_1.fq.gz \\\n",
    "/home/sam/data/geoduck_illumina/trimmed/NR012_S1_L001_R2_001_val_2.fq.gz \\\n",
    "/home/sam/data/geoduck_illumina/trimmed/NR012_S1_L002_R1_001_val_1.fq.gz \\\n",
    "/home/sam/data/geoduck_illumina/trimmed/NR012_S1_L002_R2_001_val_2.fq.gz \\\n",
    "/home/sam/data/geoduck_illumina/trimmed/NR013_AD013_S2_L001_R1_001_val_1.fq.gz \\\n",
    "/home/sam/data/geoduck_illumina/trimmed/NR013_AD013_S2_L001_R2_001_val_2.fq.gz \\\n",
    "/home/sam/data/geoduck_illumina/trimmed/NR013_AD013_S2_L002_R1_001_val_1.fq.gz \\\n",
    "/home/sam/data/geoduck_illumina/trimmed/NR013_AD013_S2_L002_R2_001_val_2.fq.gz \\\n",
    "/home/sam/data/geoduck_illumina/trimmed/NR014_AD014_S5_L001_R1_001_val_1.fq.gz \\\n",
    "/home/sam/data/geoduck_illumina/trimmed/NR014_AD014_S5_L001_R2_001_val_2.fq.gz \\\n",
    "/home/sam/data/geoduck_illumina/trimmed/NR014_AD014_S5_L002_R1_001_val_1.fq.gz \\\n",
    "/home/sam/data/geoduck_illumina/trimmed/NR014_AD014_S5_L002_R2_001_val_2.fq.gz \\\n",
    "/home/sam/data/geoduck_illumina/trimmed/NR015_AD015_S6_L001_R1_001_val_1.fq.gz \\\n",
    "/home/sam/data/geoduck_illumina/trimmed/NR015_AD015_S6_L001_R2_001_val_2.fq.gz \\\n",
    "/home/sam/data/geoduck_illumina/trimmed/NR015_AD015_S6_L002_R1_001_val_1.fq.gz \\\n",
    "/home/sam/data/geoduck_illumina/trimmed/NR015_AD015_S6_L002_R2_001_val_2.fq.gz \\\n",
    "/home/sam/data/geoduck_illumina/trimmed/NR019_S7_L001_R1_001_val_1.fq.gz \\\n",
    "/home/sam/data/geoduck_illumina/trimmed/NR019_S7_L001_R2_001_val_2.fq.gz \\\n",
    "/home/sam/data/geoduck_illumina/trimmed/NR019_S7_L002_R1_001_val_1.fq.gz \\\n",
    "/home/sam/data/geoduck_illumina/trimmed/NR019_S7_L002_R2_001_val_2.fq.gz \\\n",
    "/home/sam/data/geoduck_illumina/trimmed/NR021_S8_L001_R1_001_val_1.fq.gz \\\n",
    "/home/sam/data/geoduck_illumina/trimmed/NR021_S8_L001_R2_001_val_2.fq.gz \\\n",
    "/home/sam/data/geoduck_illumina/trimmed/NR021_S8_L002_R1_001_val_1.fq.gz \\\n",
    "/home/sam/data/geoduck_illumina/trimmed/NR021_S8_L002_R2_001_val_2.fq.gz\n",
    "cat ~/.default-subject.mail | msmtp \"$EMAIL\""
   ]
  },
  {
   "cell_type": "code",
   "execution_count": 43,
   "metadata": {},
   "outputs": [
    {
     "name": "stdout",
     "output_type": "stream",
     "text": [
      "total 28324\n",
      "-rw-rw-r-- 1 sam sam 456483 Jan 29 08:25 NR005_S4_L002_R2_001_val_2_fastqc.zip\n",
      "-rw-rw-r-- 1 sam sam 480861 Jan 29 08:25 NR005_S4_L001_R2_001_val_2_fastqc.zip\n",
      "-rw-rw-r-- 1 sam sam 479225 Jan 29 08:25 NR005_S4_L001_R1_001_val_1_fastqc.zip\n",
      "-rw-rw-r-- 1 sam sam 446955 Jan 29 08:25 NR005_S4_L002_R1_001_val_1_fastqc.zip\n",
      "-rw-rw-r-- 1 sam sam 267316 Jan 29 08:25 NR005_S4_L002_R2_001_val_2_fastqc.html\n",
      "-rw-rw-r-- 1 sam sam 268028 Jan 29 08:25 NR005_S4_L002_R1_001_val_1_fastqc.html\n",
      "-rw-rw-r-- 1 sam sam 268463 Jan 29 08:25 NR005_S4_L001_R2_001_val_2_fastqc.html\n",
      "-rw-rw-r-- 1 sam sam 271980 Jan 29 08:25 NR005_S4_L001_R1_001_val_1_fastqc.html\n",
      "-rw-rw-r-- 1 sam sam 569702 Jan 29 08:25 NR012_S1_L001_R1_001_val_1_fastqc.zip\n",
      "-rw-rw-r-- 1 sam sam 250779 Jan 29 08:25 NR012_S1_L001_R1_001_val_1_fastqc.html\n",
      "-rw-rw-r-- 1 sam sam 573531 Jan 29 08:25 NR012_S1_L001_R2_001_val_2_fastqc.zip\n",
      "-rw-rw-r-- 1 sam sam 248627 Jan 29 08:25 NR012_S1_L001_R2_001_val_2_fastqc.html\n",
      "-rw-rw-r-- 1 sam sam 578221 Jan 29 08:25 NR012_S1_L002_R1_001_val_1_fastqc.zip\n",
      "-rw-rw-r-- 1 sam sam 249959 Jan 29 08:25 NR012_S1_L002_R1_001_val_1_fastqc.html\n",
      "-rw-rw-r-- 1 sam sam 574996 Jan 29 08:25 NR012_S1_L002_R2_001_val_2_fastqc.zip\n",
      "-rw-rw-r-- 1 sam sam 245972 Jan 29 08:25 NR012_S1_L002_R2_001_val_2_fastqc.html\n",
      "-rw-rw-r-- 1 sam sam 612551 Jan 29 08:26 NR006_S3_L002_R1_001_val_1_fastqc.zip\n",
      "-rw-rw-r-- 1 sam sam 259588 Jan 29 08:26 NR006_S3_L002_R1_001_val_1_fastqc.html\n",
      "-rw-rw-r-- 1 sam sam 609942 Jan 29 08:26 NR006_S3_L002_R2_001_val_2_fastqc.zip\n",
      "-rw-rw-r-- 1 sam sam 254479 Jan 29 08:26 NR006_S3_L002_R2_001_val_2_fastqc.html\n",
      "-rw-rw-r-- 1 sam sam 610803 Jan 29 08:26 NR006_S3_L001_R1_001_val_1_fastqc.zip\n",
      "-rw-rw-r-- 1 sam sam 259540 Jan 29 08:26 NR006_S3_L001_R1_001_val_1_fastqc.html\n",
      "-rw-rw-r-- 1 sam sam 603447 Jan 29 08:26 NR006_S3_L001_R2_001_val_2_fastqc.zip\n",
      "-rw-rw-r-- 1 sam sam 254689 Jan 29 08:26 NR006_S3_L001_R2_001_val_2_fastqc.html\n",
      "-rw-rw-r-- 1 sam sam 562157 Jan 29 09:42 AD002_S9_L001_R1_001_val_1_fastqc.zip\n",
      "-rw-rw-r-- 1 sam sam 232835 Jan 29 09:42 AD002_S9_L001_R1_001_val_1_fastqc.html\n",
      "-rw-rw-r-- 1 sam sam 560796 Jan 29 09:47 AD002_S9_L001_R2_001_val_2_fastqc.zip\n",
      "-rw-rw-r-- 1 sam sam 234363 Jan 29 09:47 AD002_S9_L001_R2_001_val_2_fastqc.html\n",
      "-rw-rw-r-- 1 sam sam 564514 Jan 29 10:06 AD002_S9_L002_R1_001_val_1_fastqc.zip\n",
      "-rw-rw-r-- 1 sam sam 229772 Jan 29 10:06 AD002_S9_L002_R1_001_val_1_fastqc.html\n",
      "-rw-rw-r-- 1 sam sam 566594 Jan 29 10:18 AD002_S9_L002_R2_001_val_2_fastqc.zip\n",
      "-rw-rw-r-- 1 sam sam 231792 Jan 29 10:18 AD002_S9_L002_R2_001_val_2_fastqc.html\n",
      "-rw-rw-r-- 1 sam sam 570162 Jan 29 11:24 NR013_AD013_S2_L002_R1_001_val_1_fastqc.zip\n",
      "-rw-rw-r-- 1 sam sam 233346 Jan 29 11:24 NR013_AD013_S2_L002_R1_001_val_1_fastqc.html\n",
      "-rw-rw-r-- 1 sam sam 489645 Jan 29 11:24 NR019_S7_L001_R1_001_val_1_fastqc.zip\n",
      "-rw-rw-r-- 1 sam sam 259758 Jan 29 11:24 NR019_S7_L001_R1_001_val_1_fastqc.html\n",
      "-rw-rw-r-- 1 sam sam 488701 Jan 29 11:24 NR019_S7_L001_R2_001_val_2_fastqc.zip\n",
      "-rw-rw-r-- 1 sam sam 258169 Jan 29 11:24 NR019_S7_L001_R2_001_val_2_fastqc.html\n",
      "-rw-rw-r-- 1 sam sam 460967 Jan 29 11:24 NR019_S7_L002_R1_001_val_1_fastqc.zip\n",
      "-rw-rw-r-- 1 sam sam 263389 Jan 29 11:24 NR019_S7_L002_R1_001_val_1_fastqc.html\n",
      "-rw-rw-r-- 1 sam sam 462735 Jan 29 11:24 NR019_S7_L002_R2_001_val_2_fastqc.zip\n",
      "-rw-rw-r-- 1 sam sam 264526 Jan 29 11:24 NR019_S7_L002_R2_001_val_2_fastqc.html\n",
      "-rw-rw-r-- 1 sam sam 603205 Jan 29 11:26 NR021_S8_L001_R1_001_val_1_fastqc.zip\n",
      "-rw-rw-r-- 1 sam sam 254836 Jan 29 11:26 NR021_S8_L001_R1_001_val_1_fastqc.html\n",
      "-rw-rw-r-- 1 sam sam 560445 Jan 29 11:28 NR015_AD015_S6_L001_R1_001_val_1_fastqc.zip\n",
      "-rw-rw-r-- 1 sam sam 231789 Jan 29 11:28 NR015_AD015_S6_L001_R1_001_val_1_fastqc.html\n",
      "-rw-rw-r-- 1 sam sam 602815 Jan 29 11:28 NR021_S8_L001_R2_001_val_2_fastqc.zip\n",
      "-rw-rw-r-- 1 sam sam 256338 Jan 29 11:28 NR021_S8_L001_R2_001_val_2_fastqc.html\n",
      "-rw-rw-r-- 1 sam sam 604313 Jan 29 11:30 NR021_S8_L002_R1_001_val_1_fastqc.zip\n",
      "-rw-rw-r-- 1 sam sam 253109 Jan 29 11:30 NR021_S8_L002_R1_001_val_1_fastqc.html\n",
      "-rw-rw-r-- 1 sam sam 597673 Jan 29 11:30 NR021_S8_L002_R2_001_val_2_fastqc.zip\n",
      "-rw-rw-r-- 1 sam sam 249060 Jan 29 11:30 NR021_S8_L002_R2_001_val_2_fastqc.html\n",
      "-rw-rw-r-- 1 sam sam 570723 Jan 29 11:34 NR013_AD013_S2_L002_R2_001_val_2_fastqc.zip\n",
      "-rw-rw-r-- 1 sam sam 233618 Jan 29 11:34 NR013_AD013_S2_L002_R2_001_val_2_fastqc.html\n",
      "-rw-rw-r-- 1 sam sam 564535 Jan 29 11:36 NR015_AD015_S6_L001_R2_001_val_2_fastqc.zip\n",
      "-rw-rw-r-- 1 sam sam 234613 Jan 29 11:36 NR015_AD015_S6_L001_R2_001_val_2_fastqc.html\n",
      "-rw-rw-r-- 1 sam sam 563160 Jan 29 11:49 NR015_AD015_S6_L002_R1_001_val_1_fastqc.zip\n",
      "-rw-rw-r-- 1 sam sam 227702 Jan 29 11:49 NR015_AD015_S6_L002_R1_001_val_1_fastqc.html\n",
      "-rw-rw-r-- 1 sam sam 563756 Jan 29 11:50 NR013_AD013_S2_L001_R1_001_val_1_fastqc.zip\n",
      "-rw-rw-r-- 1 sam sam 233509 Jan 29 11:50 NR013_AD013_S2_L001_R1_001_val_1_fastqc.html\n",
      "-rw-rw-r-- 1 sam sam 567421 Jan 29 11:53 NR013_AD013_S2_L001_R2_001_val_2_fastqc.zip\n",
      "-rw-rw-r-- 1 sam sam 236469 Jan 29 11:53 NR013_AD013_S2_L001_R2_001_val_2_fastqc.html\n",
      "-rw-rw-r-- 1 sam sam 568125 Jan 29 11:58 NR015_AD015_S6_L002_R2_001_val_2_fastqc.zip\n",
      "-rw-rw-r-- 1 sam sam 232282 Jan 29 11:58 NR015_AD015_S6_L002_R2_001_val_2_fastqc.html\n",
      "-rw-rw-r-- 1 sam sam 562916 Jan 29 12:38 NR014_AD014_S5_L001_R1_001_val_1_fastqc.zip\n",
      "-rw-rw-r-- 1 sam sam 232937 Jan 29 12:38 NR014_AD014_S5_L001_R1_001_val_1_fastqc.html\n",
      "-rw-rw-r-- 1 sam sam 565340 Jan 29 12:45 NR014_AD014_S5_L001_R2_001_val_2_fastqc.zip\n",
      "-rw-rw-r-- 1 sam sam 234534 Jan 29 12:45 NR014_AD014_S5_L001_R2_001_val_2_fastqc.html\n",
      "-rw-rw-r-- 1 sam sam 569756 Jan 29 13:00 NR014_AD014_S5_L002_R1_001_val_1_fastqc.zip\n",
      "-rw-rw-r-- 1 sam sam 230261 Jan 29 13:00 NR014_AD014_S5_L002_R1_001_val_1_fastqc.html\n",
      "-rw-rw-r-- 1 sam sam 569447 Jan 29 13:07 NR014_AD014_S5_L002_R2_001_val_2_fastqc.zip\n",
      "-rw-rw-r-- 1 sam sam 233175 Jan 29 13:07 NR014_AD014_S5_L002_R2_001_val_2_fastqc.html\n"
     ]
    },
    {
     "name": "stderr",
     "output_type": "stream",
     "text": [
      "mv: cannot move '/home/sam/data/geoduck_illumina/trimmed/20180129_trimmed_fastqc_01' to a subdirectory of itself, '/home/sam/data/geoduck_illumina/trimmed/20180129_trimmed_fastqc_01/20180129_trimmed_fastqc_01'\n"
     ]
    }
   ],
   "source": [
    "%%bash\n",
    "mkdir /home/sam/data/geoduck_illumina/trimmed/20180129_trimmed_fastqc_01\n",
    "for fastqc in /home/sam/data/geoduck_illumina/trimmed/*fastqc*\n",
    "do\n",
    "mv \"$fastqc\" /home/sam/data/geoduck_illumina/trimmed/20180129_trimmed_fastqc_01\n",
    "done\n",
    "ls -ltr /home/sam/data/geoduck_illumina/trimmed/20180129_trimmed_fastqc_01"
   ]
  },
  {
   "cell_type": "code",
   "execution_count": 44,
   "metadata": {},
   "outputs": [
    {
     "name": "stdout",
     "output_type": "stream",
     "text": [
      "Searching 72 files..\n"
     ]
    },
    {
     "name": "stderr",
     "output_type": "stream",
     "text": [
      "[INFO   ]         multiqc : This is MultiQC v1.5.dev0\n",
      "[INFO   ]         multiqc : Template    : default\n",
      "[INFO   ]         multiqc : Searching '/home/sam/data/geoduck_illumina/trimmed/20180129_trimmed_fastqc_01/'\n",
      "[INFO   ]          fastqc : Found 36 reports\n",
      "[INFO   ]         multiqc : Compressing plot data\n",
      "[INFO   ]         multiqc : Report      : multiqc_report.html\n",
      "[INFO   ]         multiqc : Data        : multiqc_data\n",
      "[INFO   ]         multiqc : MultiQC complete\n"
     ]
    }
   ],
   "source": [
    "%%bash\n",
    "cd /home/sam/data/geoduck_illumina/trimmed/\n",
    "/home/shared/anaconda3/bin/multiqc /home/sam/data/geoduck_illumina/trimmed/20180129_trimmed_fastqc_01/"
   ]
  },
  {
   "cell_type": "code",
   "execution_count": 45,
   "metadata": {},
   "outputs": [
    {
     "name": "stdout",
     "output_type": "stream",
     "text": [
      "total 1460\n",
      "-rw-rw-r-- 1 sam sam 1490524 Jan 29 13:42 multiqc_report.html\n",
      "drwxrwxr-x 2 sam sam    4096 Jan 29 13:42 multiqc_data\n"
     ]
    }
   ],
   "source": [
    "%%bash\n",
    "mkdir /home/sam/data/geoduck_illumina/trimmed/20180129_trimmed_multiqc_fastqc_01\n",
    "mv /home/sam/data/geoduck_illumina/trimmed/multiqc* /home/sam/data/geoduck_illumina/trimmed/20180129_trimmed_multiqc_fastqc_01\n",
    "ls -ltr /home/sam/data/geoduck_illumina/trimmed/20180129_trimmed_multiqc_fastqc_01"
   ]
  },
  {
   "cell_type": "code",
   "execution_count": 46,
   "metadata": {},
   "outputs": [
    {
     "name": "stdout",
     "output_type": "stream",
     "text": [
      "1.9.1\n"
     ]
    },
    {
     "name": "stderr",
     "output_type": "stream",
     "text": [
      "No quality encoding type selected. Assuming that the data provided uses Sanger encoded Phred scores (default)\n",
      "\n",
      "Path to Cutadapt set as: 'cutadapt' (default)\n",
      "Cutadapt seems to be working fine (tested command 'cutadapt --version')\n",
      "\n",
      "\n",
      "AUTO-DETECTING ADAPTER TYPE\n",
      "===========================\n",
      "Attempting to auto-detect adapter type from the first 1 million sequences of the first file (>> /home/sam/data/geoduck_illumina/trimmed/AD002_S9_L001_R1_001_val_1.fq.gz <<)\n",
      "\n",
      "Found perfect matches for the following adapter sequences:\n",
      "Adapter type\tCount\tSequence\tSequences analysed\tPercentage\n",
      "Illumina\t6622\tAGATCGGAAGAGC\t1000000\t0.66\n",
      "smallRNA\t4\tTGGAATTCTCGG\t1000000\t0.00\n",
      "Nextera\t0\tCTGTCTCTTATA\t1000000\t0.00\n",
      "Using Illumina adapter for trimming (count: 6622). Second best hit was smallRNA (count: 4)\n",
      "\n",
      "Failed to write to file 'AD002_S9_L001_R1_001_val_1.fq.gz_trimming_report.txt': No such file or directory\n",
      "\n",
      "real\t0m4.498s\n",
      "user\t0m4.668s\n",
      "sys\t0m0.216s\n"
     ]
    }
   ],
   "source": [
    "%%bash\n",
    "mkdir /home/sam/data/geoduck_illumina/trimmed/trimmed_again\n",
    "time /home/shared/trimgalore/trim_galore \\\n",
    "--paired \\\n",
    "--output_dir /home/sam/data/geoduck_illumina/trimmed/20180129_trimmed_again \\\n",
    "/home/sam/data/geoduck_illumina/trimmed/AD002_S9_L001_R1_001_val_1.fq.gz \\\n",
    "/home/sam/data/geoduck_illumina/trimmed/AD002_S9_L001_R2_001_val_2.fq.gz \\\n",
    "/home/sam/data/geoduck_illumina/trimmed/AD002_S9_L002_R1_001_val_1.fq.gz \\\n",
    "/home/sam/data/geoduck_illumina/trimmed/AD002_S9_L002_R2_001_val_2.fq.gz \\\n",
    "/home/sam/data/geoduck_illumina/trimmed/NR005_S4_L001_R1_001_val_1.fq.gz \\\n",
    "/home/sam/data/geoduck_illumina/trimmed/NR005_S4_L001_R2_001_val_2.fq.gz \\\n",
    "/home/sam/data/geoduck_illumina/trimmed/NR005_S4_L002_R1_001_val_1.fq.gz \\\n",
    "/home/sam/data/geoduck_illumina/trimmed/NR005_S4_L002_R2_001_val_2.fq.gz \\\n",
    "/home/sam/data/geoduck_illumina/trimmed/NR006_S3_L001_R1_001_val_1.fq.gz \\\n",
    "/home/sam/data/geoduck_illumina/trimmed/NR006_S3_L001_R2_001_val_2.fq.gz \\\n",
    "/home/sam/data/geoduck_illumina/trimmed/NR006_S3_L002_R1_001_val_1.fq.gz \\\n",
    "/home/sam/data/geoduck_illumina/trimmed/NR006_S3_L002_R2_001_val_2.fq.gz \\\n",
    "/home/sam/data/geoduck_illumina/trimmed/NR012_S1_L001_R1_001_val_1.fq.gz \\\n",
    "/home/sam/data/geoduck_illumina/trimmed/NR012_S1_L001_R2_001_val_2.fq.gz \\\n",
    "/home/sam/data/geoduck_illumina/trimmed/NR012_S1_L002_R1_001_val_1.fq.gz \\\n",
    "/home/sam/data/geoduck_illumina/trimmed/NR012_S1_L002_R2_001_val_2.fq.gz \\\n",
    "/home/sam/data/geoduck_illumina/trimmed/NR013_AD013_S2_L001_R1_001_val_1.fq.gz \\\n",
    "/home/sam/data/geoduck_illumina/trimmed/NR013_AD013_S2_L001_R2_001_val_2.fq.gz \\\n",
    "/home/sam/data/geoduck_illumina/trimmed/NR013_AD013_S2_L002_R1_001_val_1.fq.gz \\\n",
    "/home/sam/data/geoduck_illumina/trimmed/NR013_AD013_S2_L002_R2_001_val_2.fq.gz \\\n",
    "/home/sam/data/geoduck_illumina/trimmed/NR014_AD014_S5_L001_R1_001_val_1.fq.gz \\\n",
    "/home/sam/data/geoduck_illumina/trimmed/NR014_AD014_S5_L001_R2_001_val_2.fq.gz \\\n",
    "/home/sam/data/geoduck_illumina/trimmed/NR014_AD014_S5_L002_R1_001_val_1.fq.gz \\\n",
    "/home/sam/data/geoduck_illumina/trimmed/NR014_AD014_S5_L002_R2_001_val_2.fq.gz \\\n",
    "/home/sam/data/geoduck_illumina/trimmed/NR015_AD015_S6_L001_R1_001_val_1.fq.gz \\\n",
    "/home/sam/data/geoduck_illumina/trimmed/NR015_AD015_S6_L001_R2_001_val_2.fq.gz \\\n",
    "/home/sam/data/geoduck_illumina/trimmed/NR015_AD015_S6_L002_R1_001_val_1.fq.gz \\\n",
    "/home/sam/data/geoduck_illumina/trimmed/NR015_AD015_S6_L002_R2_001_val_2.fq.gz \\\n",
    "/home/sam/data/geoduck_illumina/trimmed/NR019_S7_L001_R1_001_val_1.fq.gz \\\n",
    "/home/sam/data/geoduck_illumina/trimmed/NR019_S7_L001_R2_001_val_2.fq.gz \\\n",
    "/home/sam/data/geoduck_illumina/trimmed/NR019_S7_L002_R1_001_val_1.fq.gz \\\n",
    "/home/sam/data/geoduck_illumina/trimmed/NR019_S7_L002_R2_001_val_2.fq.gz \\\n",
    "/home/sam/data/geoduck_illumina/trimmed/NR021_S8_L001_R1_001_val_1.fq.gz \\\n",
    "/home/sam/data/geoduck_illumina/trimmed/NR021_S8_L001_R2_001_val_2.fq.gz \\\n",
    "/home/sam/data/geoduck_illumina/trimmed/NR021_S8_L002_R1_001_val_1.fq.gz \\\n",
    "/home/sam/data/geoduck_illumina/trimmed/NR021_S8_L002_R2_001_val_2.fq.gz\n",
    "cat ~/.default-subject.mail | msmtp \"$EMAIL\""
   ]
  },
  {
   "cell_type": "code",
   "execution_count": 47,
   "metadata": {},
   "outputs": [
    {
     "name": "stdout",
     "output_type": "stream",
     "text": [
      "1.9.1\n"
     ]
    },
    {
     "name": "stderr",
     "output_type": "stream",
     "text": [
      "No quality encoding type selected. Assuming that the data provided uses Sanger encoded Phred scores (default)\n",
      "\n",
      "Path to Cutadapt set as: 'cutadapt' (default)\n",
      "Cutadapt seems to be working fine (tested command 'cutadapt --version')\n",
      "\n",
      "\n",
      "AUTO-DETECTING ADAPTER TYPE\n",
      "===========================\n",
      "Attempting to auto-detect adapter type from the first 1 million sequences of the first file (>> /home/sam/data/geoduck_illumina/trimmed/AD002_S9_L001_R1_001_val_1.fq.gz <<)\n",
      "\n",
      "Found perfect matches for the following adapter sequences:\n",
      "Adapter type\tCount\tSequence\tSequences analysed\tPercentage\n",
      "Illumina\t6622\tAGATCGGAAGAGC\t1000000\t0.66\n",
      "smallRNA\t4\tTGGAATTCTCGG\t1000000\t0.00\n",
      "Nextera\t0\tCTGTCTCTTATA\t1000000\t0.00\n",
      "Using Illumina adapter for trimming (count: 6622). Second best hit was smallRNA (count: 4)\n",
      "\n",
      "Writing report to '/home/sam/data/geoduck_illumina/trimmed/20180129_trimmed_again/AD002_S9_L001_R1_001_val_1.fq.gz_trimming_report.txt'\n",
      "\n",
      "SUMMARISING RUN PARAMETERS\n",
      "==========================\n",
      "Input filename: /home/sam/data/geoduck_illumina/trimmed/AD002_S9_L001_R1_001_val_1.fq.gz\n",
      "Trimming mode: paired-end\n",
      "Trim Galore version: 0.4.4\n",
      "Cutadapt version: 1.9.1\n",
      "Quality Phred score cutoff: 20\n",
      "Quality encoding type selected: ASCII+33\n",
      "Adapter sequence: 'AGATCGGAAGAGC' (Illumina TruSeq, Sanger iPCR; auto-detected)\n",
      "Maximum trimming error rate: 0.1 (default)\n",
      "Minimum required adapter overlap (stringency): 1 bp\n",
      "Minimum required sequence length for both reads before a sequence pair gets removed: 20 bp\n",
      "Output file(s) will be GZIP compressed\n",
      "\n",
      "Writing final adapter and quality trimmed output to AD002_S9_L001_R1_001_val_1_trimmed.fq.gz\n",
      "\n",
      "\n",
      "  >>> Now performing quality (cutoff 20) and adapter trimming in a single pass for the adapter sequence: 'AGATCGGAAGAGC' from file /home/sam/data/geoduck_illumina/trimmed/AD002_S9_L001_R1_001_val_1.fq.gz <<< \n",
      "10000000 sequences processed\n",
      "20000000 sequences processed\n",
      "30000000 sequences processed\n",
      "40000000 sequences processed\n",
      "50000000 sequences processed\n",
      "60000000 sequences processed\n",
      "70000000 sequences processed\n",
      "80000000 sequences processed\n",
      "90000000 sequences processed\n",
      "100000000 sequences processed\n",
      "110000000 sequences processed\n",
      "120000000 sequences processed\n",
      "130000000 sequences processed\n",
      "140000000 sequences processed\n",
      "150000000 sequences processed\n",
      "160000000 sequences processed\n",
      "This is cutadapt 1.9.1 with Python 2.7.12\n",
      "Command line parameters: -f fastq -e 0.1 -q 20 -O 1 -a AGATCGGAAGAGC /home/sam/data/geoduck_illumina/trimmed/AD002_S9_L001_R1_001_val_1.fq.gz\n",
      "Trimming 1 adapter with at most 10.0% errors in single-end mode ...\n",
      "Finished in 4009.23 s (24 us/read; 2.49 M reads/minute).\n",
      "\n",
      "=== Summary ===\n",
      "\n",
      "Total reads processed:             166,519,549\n",
      "Reads with adapters:                62,950,909 (37.8%)\n",
      "Reads written (passing filters):   166,519,549 (100.0%)\n",
      "\n",
      "Total basepairs processed: 20,263,772,384 bp\n",
      "Quality-trimmed:              10,658,407 bp (0.1%)\n",
      "Total written (filtered):  20,075,819,474 bp (99.1%)\n",
      "\n",
      "=== Adapter 1 ===\n",
      "\n",
      "Sequence: AGATCGGAAGAGC; Type: regular 3'; Length: 13; Trimmed: 62950909 times.\n",
      "\n",
      "No. of allowed errors:\n",
      "0-9 bp: 0; 10-13 bp: 1\n",
      "\n",
      "Bases preceding removed adapters:\n",
      "  A: 32.6%\n",
      "  C: 22.7%\n",
      "  G: 14.4%\n",
      "  T: 30.1%\n",
      "  none/other: 0.2%\n",
      "\n",
      "Overview of removed sequences\n",
      "length\tcount\texpect\tmax.err\terror counts\n",
      "1\t45191016\t41629887.2\t0\t45191016\n",
      "2\t11892878\t10407471.8\t0\t11892878\n",
      "3\t3014795\t2601868.0\t0\t3014795\n",
      "4\t1184885\t650467.0\t0\t1184885\n",
      "5\t136203\t162616.7\t0\t136203\n",
      "6\t26162\t40654.2\t0\t26162\n",
      "7\t15017\t10163.5\t0\t15017\n",
      "8\t16951\t2540.9\t0\t16951\n",
      "9\t13393\t635.2\t0\t11409 1984\n",
      "10\t20526\t158.8\t1\t12168 8358\n",
      "11\t18484\t39.7\t1\t12657 5827\n",
      "12\t30871\t9.9\t1\t23904 6967\n",
      "13\t1883\t2.5\t1\t977 906\n",
      "14\t28295\t2.5\t1\t22411 5884\n",
      "15\t1728\t2.5\t1\t845 883\n",
      "16\t31609\t2.5\t1\t24109 7500\n",
      "17\t2352\t2.5\t1\t1197 1155\n",
      "18\t22811\t2.5\t1\t17377 5434\n",
      "19\t44221\t2.5\t1\t36321 7900\n",
      "20\t476\t2.5\t1\t204 272\n",
      "21\t735\t2.5\t1\t260 475\n",
      "22\t1509\t2.5\t1\t663 846\n",
      "23\t19790\t2.5\t1\t14697 5093\n",
      "24\t38449\t2.5\t1\t30670 7779\n",
      "25\t2192\t2.5\t1\t1125 1067\n",
      "26\t16807\t2.5\t1\t12495 4312\n",
      "27\t13827\t2.5\t1\t11153 2674\n",
      "28\t810\t2.5\t1\t317 493\n",
      "29\t16333\t2.5\t1\t12128 4205\n",
      "30\t20883\t2.5\t1\t16077 4806\n",
      "31\t24181\t2.5\t1\t19042 5139\n",
      "32\t3089\t2.5\t1\t1832 1257\n",
      "33\t29110\t2.5\t1\t22048 7062\n",
      "34\t14162\t2.5\t1\t11204 2958\n",
      "35\t817\t2.5\t1\t349 468\n",
      "36\t16927\t2.5\t1\t12775 4152\n",
      "37\t19935\t2.5\t1\t15059 4876\n",
      "38\t12063\t2.5\t1\t8977 3086\n",
      "39\t18055\t2.5\t1\t13767 4288\n",
      "40\t16204\t2.5\t1\t12318 3886\n",
      "41\t19447\t2.5\t1\t14427 5020\n",
      "42\t35132\t2.5\t1\t27593 7539\n",
      "43\t6650\t2.5\t1\t5421 1229\n",
      "44\t9775\t2.5\t1\t6973 2802\n",
      "45\t796\t2.5\t1\t368 428\n",
      "46\t19159\t2.5\t1\t14642 4517\n",
      "47\t14576\t2.5\t1\t11098 3478\n",
      "48\t13926\t2.5\t1\t10450 3476\n",
      "49\t19134\t2.5\t1\t13988 5146\n",
      "50\t29774\t2.5\t1\t23411 6363\n",
      "51\t10816\t2.5\t1\t8626 2190\n",
      "52\t2049\t2.5\t1\t1278 771\n",
      "53\t28745\t2.5\t1\t22950 5795\n",
      "54\t35400\t2.5\t1\t28635 6765\n",
      "55\t3293\t2.5\t1\t2440 853\n",
      "56\t8907\t2.5\t1\t6800 2107\n",
      "57\t29998\t2.5\t1\t24644 5354\n",
      "58\t1367\t2.5\t1\t744 623\n",
      "59\t3980\t2.5\t1\t2473 1507\n",
      "60\t25914\t2.5\t1\t21592 4322\n",
      "61\t732\t2.5\t1\t362 370\n",
      "62\t2084\t2.5\t1\t1131 953\n",
      "63\t7543\t2.5\t1\t5611 1932\n",
      "64\t19408\t2.5\t1\t15793 3615\n",
      "65\t16686\t2.5\t1\t13541 3145\n",
      "66\t16846\t2.5\t1\t13865 2981\n",
      "67\t9203\t2.5\t1\t7168 2035\n",
      "68\t7795\t2.5\t1\t5744 2051\n",
      "69\t5135\t2.5\t1\t3682 1453\n",
      "70\t5891\t2.5\t1\t4597 1294\n",
      "71\t8634\t2.5\t1\t6924 1710\n",
      "72\t9794\t2.5\t1\t7862 1932\n",
      "73\t10352\t2.5\t1\t8360 1992\n",
      "74\t10759\t2.5\t1\t8560 2199\n",
      "75\t11037\t2.5\t1\t8815 2222\n",
      "76\t11355\t2.5\t1\t9027 2328\n",
      "77\t11168\t2.5\t1\t8869 2299\n",
      "78\t11167\t2.5\t1\t8809 2358\n",
      "79\t11221\t2.5\t1\t8924 2297\n",
      "80\t11098\t2.5\t1\t8767 2331\n",
      "81\t10888\t2.5\t1\t8548 2340\n",
      "82\t10994\t2.5\t1\t8677 2317\n",
      "83\t10736\t2.5\t1\t8441 2295\n",
      "84\t10842\t2.5\t1\t8489 2353\n",
      "85\t10728\t2.5\t1\t8418 2310\n",
      "86\t10495\t2.5\t1\t8215 2280\n",
      "87\t10484\t2.5\t1\t8225 2259\n",
      "88\t10007\t2.5\t1\t7802 2205\n",
      "89\t10091\t2.5\t1\t7857 2234\n",
      "90\t9940\t2.5\t1\t7828 2112\n",
      "91\t9628\t2.5\t1\t7583 2045\n",
      "92\t9575\t2.5\t1\t7475 2100\n",
      "93\t9431\t2.5\t1\t7431 2000\n",
      "94\t9329\t2.5\t1\t7319 2010\n",
      "95\t8980\t2.5\t1\t6971 2009\n",
      "96\t8950\t2.5\t1\t7007 1943\n",
      "97\t8734\t2.5\t1\t6774 1960\n",
      "98\t8538\t2.5\t1\t6692 1846\n",
      "99\t8250\t2.5\t1\t6355 1895\n",
      "100\t8216\t2.5\t1\t6456 1760\n",
      "101\t7764\t2.5\t1\t6063 1701\n",
      "102\t7618\t2.5\t1\t5928 1690\n",
      "103\t7234\t2.5\t1\t5651 1583\n",
      "104\t6971\t2.5\t1\t5450 1521\n",
      "105\t6699\t2.5\t1\t5123 1576\n",
      "106\t6627\t2.5\t1\t5073 1554\n",
      "107\t6023\t2.5\t1\t4625 1398\n",
      "108\t5843\t2.5\t1\t4473 1370\n",
      "109\t5608\t2.5\t1\t4271 1337\n",
      "110\t5329\t2.5\t1\t4030 1299\n",
      "111\t5128\t2.5\t1\t3929 1199\n",
      "112\t4870\t2.5\t1\t3694 1176\n",
      "113\t4622\t2.5\t1\t3450 1172\n",
      "114\t4528\t2.5\t1\t3397 1131\n",
      "115\t4255\t2.5\t1\t3226 1029\n",
      "116\t4100\t2.5\t1\t3085 1015\n",
      "117\t3886\t2.5\t1\t2902 984\n",
      "118\t3780\t2.5\t1\t2870 910\n",
      "119\t3676\t2.5\t1\t2785 891\n",
      "120\t3372\t2.5\t1\t2520 852\n",
      "121\t3199\t2.5\t1\t2440 759\n",
      "122\t2979\t2.5\t1\t2269 710\n",
      "123\t2811\t2.5\t1\t2107 704\n",
      "124\t2615\t2.5\t1\t1971 644\n",
      "125\t2537\t2.5\t1\t1908 629\n",
      "126\t2335\t2.5\t1\t1795 540\n",
      "127\t2138\t2.5\t1\t1639 499\n",
      "128\t2019\t2.5\t1\t1567 452\n",
      "129\t1900\t2.5\t1\t1514 386\n",
      "130\t1584\t2.5\t1\t1268 316\n",
      "131\t1574\t2.5\t1\t1292 282\n",
      "132\t1236\t2.5\t1\t1037 199\n",
      "133\t1139\t2.5\t1\t979 160\n",
      "134\t950\t2.5\t1\t804 146\n",
      "135\t914\t2.5\t1\t788 126\n",
      "136\t764\t2.5\t1\t646 118\n",
      "137\t685\t2.5\t1\t601 84\n",
      "138\t556\t2.5\t1\t487 69\n",
      "139\t495\t2.5\t1\t442 53\n",
      "140\t421\t2.5\t1\t358 63\n",
      "141\t343\t2.5\t1\t295 48\n",
      "142\t266\t2.5\t1\t225 41\n",
      "143\t305\t2.5\t1\t243 62\n",
      "144\t238\t2.5\t1\t195 43\n",
      "145\t269\t2.5\t1\t214 55\n",
      "146\t219\t2.5\t1\t174 45\n",
      "147\t245\t2.5\t1\t185 60\n",
      "148\t203\t2.5\t1\t128 75\n",
      "149\t310\t2.5\t1\t192 118\n",
      "150\t1172\t2.5\t1\t461 711\n",
      "151\t114539\t2.5\t1\t42001 72538\n",
      "\n",
      "\n",
      "RUN STATISTICS FOR INPUT FILE: /home/sam/data/geoduck_illumina/trimmed/AD002_S9_L001_R1_001_val_1.fq.gz\n",
      "=============================================\n",
      "166519549 sequences processed in total\n",
      "The length threshold of paired-end sequences gets evaluated later on (in the validation step)\n",
      "\n",
      "Writing report to '/home/sam/data/geoduck_illumina/trimmed/20180129_trimmed_again/AD002_S9_L001_R2_001_val_2.fq.gz_trimming_report.txt'\n",
      "\n",
      "SUMMARISING RUN PARAMETERS\n",
      "==========================\n",
      "Input filename: /home/sam/data/geoduck_illumina/trimmed/AD002_S9_L001_R2_001_val_2.fq.gz\n",
      "Trimming mode: paired-end\n",
      "Trim Galore version: 0.4.4\n",
      "Cutadapt version: 1.9.1\n",
      "Quality Phred score cutoff: 20\n",
      "Quality encoding type selected: ASCII+33\n",
      "Adapter sequence: 'AGATCGGAAGAGC' (Illumina TruSeq, Sanger iPCR; auto-detected)\n",
      "Maximum trimming error rate: 0.1 (default)\n",
      "Minimum required adapter overlap (stringency): 1 bp\n",
      "Minimum required sequence length for both reads before a sequence pair gets removed: 20 bp\n",
      "Output file(s) will be GZIP compressed\n",
      "\n",
      "Writing final adapter and quality trimmed output to AD002_S9_L001_R2_001_val_2_trimmed.fq.gz\n",
      "\n",
      "\n",
      "  >>> Now performing quality (cutoff 20) and adapter trimming in a single pass for the adapter sequence: 'AGATCGGAAGAGC' from file /home/sam/data/geoduck_illumina/trimmed/AD002_S9_L001_R2_001_val_2.fq.gz <<< \n",
      "10000000 sequences processed\n",
      "20000000 sequences processed\n",
      "30000000 sequences processed\n",
      "40000000 sequences processed\n",
      "50000000 sequences processed\n",
      "60000000 sequences processed\n",
      "70000000 sequences processed\n",
      "80000000 sequences processed\n",
      "90000000 sequences processed\n",
      "100000000 sequences processed\n",
      "110000000 sequences processed\n",
      "120000000 sequences processed\n",
      "130000000 sequences processed\n",
      "140000000 sequences processed\n",
      "150000000 sequences processed\n",
      "160000000 sequences processed\n",
      "This is cutadapt 1.9.1 with Python 2.7.12\n",
      "Command line parameters: -f fastq -e 0.1 -q 20 -O 1 -a AGATCGGAAGAGC /home/sam/data/geoduck_illumina/trimmed/AD002_S9_L001_R2_001_val_2.fq.gz\n",
      "Trimming 1 adapter with at most 10.0% errors in single-end mode ...\n",
      "Finished in 4136.23 s (25 us/read; 2.42 M reads/minute).\n",
      "\n",
      "=== Summary ===\n",
      "\n",
      "Total reads processed:             166,519,549\n",
      "Reads with adapters:                64,336,761 (38.6%)\n",
      "Reads written (passing filters):   166,519,549 (100.0%)\n",
      "\n",
      "Total basepairs processed: 20,806,009,128 bp\n",
      "Quality-trimmed:              12,672,889 bp (0.1%)\n",
      "Total written (filtered):  20,606,066,844 bp (99.0%)\n",
      "\n",
      "=== Adapter 1 ===\n",
      "\n",
      "Sequence: AGATCGGAAGAGC; Type: regular 3'; Length: 13; Trimmed: 64336761 times.\n",
      "\n",
      "No. of allowed errors:\n",
      "0-9 bp: 0; 10-13 bp: 1\n",
      "\n",
      "Bases preceding removed adapters:\n",
      "  A: 33.0%\n",
      "  C: 22.5%\n",
      "  G: 14.6%\n",
      "  T: 29.8%\n",
      "  none/other: 0.1%\n",
      "\n",
      "Overview of removed sequences\n",
      "length\tcount\texpect\tmax.err\terror counts\n",
      "1\t46320342\t41629887.2\t0\t46320342\n",
      "2\t11729062\t10407471.8\t0\t11729062\n",
      "3\t3131394\t2601868.0\t0\t3131394\n",
      "4\t1193082\t650467.0\t0\t1193082\n",
      "5\t125066\t162616.7\t0\t125066\n",
      "6\t31913\t40654.2\t0\t31913\n",
      "7\t21845\t10163.5\t0\t21845\n",
      "8\t22466\t2540.9\t0\t22466\n",
      "9\t16214\t635.2\t0\t14145 2069\n",
      "10\t28137\t158.8\t1\t17439 10698\n",
      "11\t22555\t39.7\t1\t15567 6988\n",
      "12\t42264\t9.9\t1\t32388 9876\n",
      "13\t661\t2.5\t1\t252 409\n",
      "14\t28072\t2.5\t1\t20831 7241\n",
      "15\t31160\t2.5\t1\t24417 6743\n",
      "16\t383\t2.5\t1\t115 268\n",
      "17\t24954\t2.5\t1\t18991 5963\n",
      "18\t13631\t2.5\t1\t10359 3272\n",
      "19\t28189\t2.5\t1\t21318 6871\n",
      "20\t18450\t2.5\t1\t13935 4515\n",
      "21\t15196\t2.5\t1\t11701 3495\n",
      "22\t19651\t2.5\t1\t14996 4655\n",
      "23\t21752\t2.5\t1\t16404 5348\n",
      "24\t40107\t2.5\t1\t29978 10129\n",
      "25\t18750\t2.5\t1\t14580 4170\n",
      "26\t18519\t2.5\t1\t14350 4169\n",
      "27\t14830\t2.5\t1\t10978 3852\n",
      "28\t29641\t2.5\t1\t23843 5798\n",
      "29\t9716\t2.5\t1\t6627 3089\n",
      "30\t29481\t2.5\t1\t23634 5847\n",
      "31\t13670\t2.5\t1\t9881 3789\n",
      "32\t51895\t2.5\t1\t43576 8319\n",
      "33\t24414\t2.5\t1\t18797 5617\n",
      "34\t8902\t2.5\t1\t6076 2826\n",
      "35\t29279\t2.5\t1\t23981 5298\n",
      "36\t10624\t2.5\t1\t7727 2897\n",
      "37\t15397\t2.5\t1\t11525 3872\n",
      "38\t8278\t2.5\t1\t6292 1986\n",
      "39\t11562\t2.5\t1\t8638 2924\n",
      "40\t5567\t2.5\t1\t4069 1498\n",
      "41\t17152\t2.5\t1\t13742 3410\n",
      "42\t23546\t2.5\t1\t19042 4504\n",
      "43\t12652\t2.5\t1\t9789 2863\n",
      "44\t19653\t2.5\t1\t15410 4243\n",
      "45\t36962\t2.5\t1\t30323 6639\n",
      "46\t18823\t2.5\t1\t14422 4401\n",
      "47\t11178\t2.5\t1\t8633 2545\n",
      "48\t36958\t2.5\t1\t30588 6370\n",
      "49\t14327\t2.5\t1\t11252 3075\n",
      "50\t5729\t2.5\t1\t3947 1782\n",
      "51\t29652\t2.5\t1\t24002 5650\n",
      "52\t44916\t2.5\t1\t38911 6005\n",
      "53\t15569\t2.5\t1\t12618 2951\n",
      "54\t20470\t2.5\t1\t16407 4063\n",
      "55\t12026\t2.5\t1\t9700 2326\n",
      "56\t6551\t2.5\t1\t4773 1778\n",
      "57\t11696\t2.5\t1\t9080 2616\n",
      "58\t13830\t2.5\t1\t10710 3120\n",
      "59\t7683\t2.5\t1\t5618 2065\n",
      "60\t5817\t2.5\t1\t3898 1919\n",
      "61\t4974\t2.5\t1\t3438 1536\n",
      "62\t5588\t2.5\t1\t3870 1718\n",
      "63\t6329\t2.5\t1\t4654 1675\n",
      "64\t9893\t2.5\t1\t7791 2102\n",
      "65\t13175\t2.5\t1\t10559 2616\n",
      "66\t14511\t2.5\t1\t11692 2819\n",
      "67\t15084\t2.5\t1\t12069 3015\n",
      "68\t15405\t2.5\t1\t12128 3277\n",
      "69\t15754\t2.5\t1\t12484 3270\n",
      "70\t15554\t2.5\t1\t12293 3261\n",
      "71\t15695\t2.5\t1\t12342 3353\n",
      "72\t15542\t2.5\t1\t12261 3281\n",
      "73\t15911\t2.5\t1\t12488 3423\n",
      "74\t15564\t2.5\t1\t12295 3269\n",
      "75\t15370\t2.5\t1\t12091 3279\n",
      "76\t15235\t2.5\t1\t12023 3212\n",
      "77\t15186\t2.5\t1\t11947 3239\n",
      "78\t14886\t2.5\t1\t11638 3248\n",
      "79\t14938\t2.5\t1\t11741 3197\n",
      "80\t14749\t2.5\t1\t11564 3185\n",
      "81\t14257\t2.5\t1\t11136 3121\n",
      "82\t14380\t2.5\t1\t11319 3061\n",
      "83\t14226\t2.5\t1\t11240 2986\n",
      "84\t14050\t2.5\t1\t11075 2975\n",
      "85\t14041\t2.5\t1\t11082 2959\n",
      "86\t13664\t2.5\t1\t10701 2963\n",
      "87\t13612\t2.5\t1\t10731 2881\n",
      "88\t13323\t2.5\t1\t10438 2885\n",
      "89\t13177\t2.5\t1\t10433 2744\n",
      "90\t12972\t2.5\t1\t10190 2782\n",
      "91\t12706\t2.5\t1\t9910 2796\n",
      "92\t12299\t2.5\t1\t9703 2596\n",
      "93\t12161\t2.5\t1\t9545 2616\n",
      "94\t11979\t2.5\t1\t9442 2537\n",
      "95\t11751\t2.5\t1\t9172 2579\n",
      "96\t11612\t2.5\t1\t9161 2451\n",
      "97\t11434\t2.5\t1\t9052 2382\n",
      "98\t10985\t2.5\t1\t8714 2271\n",
      "99\t10710\t2.5\t1\t8346 2364\n",
      "100\t10595\t2.5\t1\t8333 2262\n",
      "101\t10225\t2.5\t1\t8010 2215\n",
      "102\t9958\t2.5\t1\t7869 2089\n",
      "103\t9684\t2.5\t1\t7551 2133\n",
      "104\t9139\t2.5\t1\t7086 2053\n",
      "105\t8751\t2.5\t1\t6828 1923\n",
      "106\t8325\t2.5\t1\t6414 1911\n",
      "107\t7751\t2.5\t1\t5910 1841\n",
      "108\t7293\t2.5\t1\t5567 1726\n",
      "109\t6982\t2.5\t1\t5289 1693\n",
      "110\t6537\t2.5\t1\t4931 1606\n",
      "111\t6361\t2.5\t1\t4812 1549\n",
      "112\t6064\t2.5\t1\t4553 1511\n",
      "113\t5782\t2.5\t1\t4332 1450\n",
      "114\t5544\t2.5\t1\t4201 1343\n",
      "115\t5196\t2.5\t1\t3928 1268\n",
      "116\t5079\t2.5\t1\t3833 1246\n",
      "117\t4888\t2.5\t1\t3754 1134\n",
      "118\t4690\t2.5\t1\t3571 1119\n",
      "119\t4561\t2.5\t1\t3506 1055\n",
      "120\t4166\t2.5\t1\t3208 958\n",
      "121\t3912\t2.5\t1\t2949 963\n",
      "122\t3756\t2.5\t1\t2849 907\n",
      "123\t3499\t2.5\t1\t2677 822\n",
      "124\t3279\t2.5\t1\t2478 801\n",
      "125\t3078\t2.5\t1\t2378 700\n",
      "126\t2873\t2.5\t1\t2201 672\n",
      "127\t2645\t2.5\t1\t2035 610\n",
      "128\t2353\t2.5\t1\t1889 464\n",
      "129\t2192\t2.5\t1\t1820 372\n",
      "130\t1761\t2.5\t1\t1460 301\n",
      "131\t1690\t2.5\t1\t1438 252\n",
      "132\t1293\t2.5\t1\t1114 179\n",
      "133\t1195\t2.5\t1\t1033 162\n",
      "134\t981\t2.5\t1\t873 108\n",
      "135\t940\t2.5\t1\t841 99\n",
      "136\t772\t2.5\t1\t667 105\n",
      "137\t701\t2.5\t1\t627 74\n",
      "138\t560\t2.5\t1\t501 59\n",
      "139\t486\t2.5\t1\t422 64\n",
      "140\t410\t2.5\t1\t356 54\n",
      "141\t340\t2.5\t1\t284 56\n",
      "142\t259\t2.5\t1\t203 56\n",
      "143\t273\t2.5\t1\t211 62\n",
      "144\t226\t2.5\t1\t161 65\n",
      "145\t247\t2.5\t1\t183 64\n",
      "146\t185\t2.5\t1\t128 57\n",
      "147\t178\t2.5\t1\t123 55\n",
      "148\t127\t2.5\t1\t68 59\n",
      "149\t178\t2.5\t1\t114 64\n",
      "150\t712\t2.5\t1\t592 120\n",
      "151\t43146\t2.5\t1\t42014 1132\n",
      "\n",
      "\n",
      "RUN STATISTICS FOR INPUT FILE: /home/sam/data/geoduck_illumina/trimmed/AD002_S9_L001_R2_001_val_2.fq.gz\n",
      "=============================================\n",
      "166519549 sequences processed in total\n",
      "The length threshold of paired-end sequences gets evaluated later on (in the validation step)\n",
      "\n",
      "Validate paired-end files AD002_S9_L001_R1_001_val_1_trimmed.fq.gz and AD002_S9_L001_R2_001_val_2_trimmed.fq.gz\n",
      "file_1: AD002_S9_L001_R1_001_val_1_trimmed.fq.gz, file_2: AD002_S9_L001_R2_001_val_2_trimmed.fq.gz\n",
      "\n",
      "\n",
      ">>>>> Now validing the length of the 2 paired-end infiles: AD002_S9_L001_R1_001_val_1_trimmed.fq.gz and AD002_S9_L001_R2_001_val_2_trimmed.fq.gz <<<<<\n",
      "Writing validated paired-end read 1 reads to AD002_S9_L001_R1_001_val_1_val_1.fq.gz\n",
      "Writing validated paired-end read 2 reads to AD002_S9_L001_R2_001_val_2_val_2.fq.gz\n",
      "\n",
      "Total number of sequences analysed: 166519549\n",
      "\n",
      "Number of sequence pairs removed because at least one read was shorter than the length cutoff (20 bp): 577240 (0.35%)\n",
      "\n",
      "Deleting both intermediate output files AD002_S9_L001_R1_001_val_1_trimmed.fq.gz and AD002_S9_L001_R2_001_val_2_trimmed.fq.gz\n",
      "\n",
      "====================================================================================================\n",
      "\n",
      "Writing report to '/home/sam/data/geoduck_illumina/trimmed/20180129_trimmed_again/AD002_S9_L002_R1_001_val_1.fq.gz_trimming_report.txt'\n",
      "\n",
      "SUMMARISING RUN PARAMETERS\n",
      "==========================\n",
      "Input filename: /home/sam/data/geoduck_illumina/trimmed/AD002_S9_L002_R1_001_val_1.fq.gz\n",
      "Trimming mode: paired-end\n",
      "Trim Galore version: 0.4.4\n",
      "Cutadapt version: 1.9.1\n",
      "Quality Phred score cutoff: 20\n",
      "Quality encoding type selected: ASCII+33\n",
      "Adapter sequence: 'AGATCGGAAGAGC' (Illumina TruSeq, Sanger iPCR; auto-detected)\n",
      "Maximum trimming error rate: 0.1 (default)\n",
      "Minimum required adapter overlap (stringency): 1 bp\n",
      "Minimum required sequence length for both reads before a sequence pair gets removed: 20 bp\n",
      "Output file(s) will be GZIP compressed\n",
      "\n",
      "Writing final adapter and quality trimmed output to AD002_S9_L002_R1_001_val_1_trimmed.fq.gz\n",
      "\n",
      "\n",
      "  >>> Now performing quality (cutoff 20) and adapter trimming in a single pass for the adapter sequence: 'AGATCGGAAGAGC' from file /home/sam/data/geoduck_illumina/trimmed/AD002_S9_L002_R1_001_val_1.fq.gz <<< \n",
      "10000000 sequences processed\n",
      "20000000 sequences processed\n",
      "30000000 sequences processed\n",
      "40000000 sequences processed\n",
      "50000000 sequences processed\n",
      "60000000 sequences processed\n",
      "70000000 sequences processed\n",
      "80000000 sequences processed\n",
      "90000000 sequences processed\n",
      "100000000 sequences processed\n",
      "110000000 sequences processed\n",
      "120000000 sequences processed\n",
      "130000000 sequences processed\n",
      "140000000 sequences processed\n",
      "150000000 sequences processed\n",
      "160000000 sequences processed\n",
      "170000000 sequences processed\n",
      "180000000 sequences processed\n",
      "190000000 sequences processed\n",
      "200000000 sequences processed\n",
      "210000000 sequences processed\n",
      "This is cutadapt 1.9.1 with Python 2.7.12\n",
      "Command line parameters: -f fastq -e 0.1 -q 20 -O 1 -a AGATCGGAAGAGC /home/sam/data/geoduck_illumina/trimmed/AD002_S9_L002_R1_001_val_1.fq.gz\n",
      "Trimming 1 adapter with at most 10.0% errors in single-end mode ...\n",
      "Finished in 5175.37 s (24 us/read; 2.46 M reads/minute).\n",
      "\n",
      "=== Summary ===\n",
      "\n",
      "Total reads processed:             212,571,014\n",
      "Reads with adapters:                82,462,079 (38.8%)\n",
      "Reads written (passing filters):   212,571,014 (100.0%)\n",
      "\n",
      "Total basepairs processed: 26,380,336,744 bp\n",
      "Quality-trimmed:              11,282,411 bp (0.0%)\n",
      "Total written (filtered):  26,072,320,126 bp (98.8%)\n",
      "\n",
      "=== Adapter 1 ===\n",
      "\n",
      "Sequence: AGATCGGAAGAGC; Type: regular 3'; Length: 13; Trimmed: 82462079 times.\n",
      "\n",
      "No. of allowed errors:\n",
      "0-9 bp: 0; 10-13 bp: 1\n",
      "\n",
      "Bases preceding removed adapters:\n",
      "  A: 32.8%\n",
      "  C: 22.7%\n",
      "  G: 14.3%\n",
      "  T: 30.1%\n",
      "  none/other: 0.2%\n",
      "\n",
      "Overview of removed sequences\n",
      "length\tcount\texpect\tmax.err\terror counts\n",
      "1\t59039996\t53142753.5\t0\t59039996\n",
      "2\t14966438\t13285688.4\t0\t14966438\n",
      "3\t3831679\t3321422.1\t0\t3831679\n",
      "4\t1480847\t830355.5\t0\t1480847\n",
      "5\t163128\t207588.9\t0\t163128\n",
      "6\t38541\t51897.2\t0\t38541\n",
      "7\t22951\t12974.3\t0\t22951\n",
      "8\t27327\t3243.6\t0\t27327\n",
      "9\t22259\t810.9\t0\t19747 2512\n",
      "10\t31473\t202.7\t1\t19843 11630\n",
      "11\t30840\t50.7\t1\t21362 9478\n",
      "12\t50070\t12.7\t1\t39221 10849\n",
      "13\t2844\t3.2\t1\t1464 1380\n",
      "14\t47563\t3.2\t1\t37749 9814\n",
      "15\t2742\t3.2\t1\t1398 1344\n",
      "16\t55622\t3.2\t1\t42339 13283\n",
      "17\t4406\t3.2\t1\t2343 2063\n",
      "18\t38657\t3.2\t1\t30067 8590\n",
      "19\t76638\t3.2\t1\t63646 12992\n",
      "20\t785\t3.2\t1\t335 450\n",
      "21\t1199\t3.2\t1\t405 794\n",
      "22\t2135\t3.2\t1\t907 1228\n",
      "23\t33554\t3.2\t1\t25117 8437\n",
      "24\t69371\t3.2\t1\t55342 14029\n",
      "25\t3521\t3.2\t1\t1864 1657\n",
      "26\t30185\t3.2\t1\t22674 7511\n",
      "27\t23530\t3.2\t1\t19348 4182\n",
      "28\t1332\t3.2\t1\t573 759\n",
      "29\t30080\t3.2\t1\t22282 7798\n",
      "30\t36733\t3.2\t1\t28722 8011\n",
      "31\t43072\t3.2\t1\t34257 8815\n",
      "32\t5026\t3.2\t1\t2892 2134\n",
      "33\t54705\t3.2\t1\t41452 13253\n",
      "34\t25607\t3.2\t1\t20697 4910\n",
      "35\t1315\t3.2\t1\t598 717\n",
      "36\t29363\t3.2\t1\t22625 6738\n",
      "37\t37853\t3.2\t1\t28899 8954\n",
      "38\t22367\t3.2\t1\t16611 5756\n",
      "39\t31698\t3.2\t1\t24957 6741\n",
      "40\t28603\t3.2\t1\t22260 6343\n",
      "41\t36993\t3.2\t1\t27714 9279\n",
      "42\t67000\t3.2\t1\t53293 13707\n",
      "43\t12117\t3.2\t1\t10136 1981\n",
      "44\t17331\t3.2\t1\t12768 4563\n",
      "45\t1153\t3.2\t1\t533 620\n",
      "46\t32065\t3.2\t1\t25213 6852\n",
      "47\t25756\t3.2\t1\t20118 5638\n",
      "48\t28466\t3.2\t1\t21563 6903\n",
      "49\t37422\t3.2\t1\t27270 10152\n",
      "50\t55745\t3.2\t1\t44380 11365\n",
      "51\t21907\t3.2\t1\t17848 4059\n",
      "52\t3678\t3.2\t1\t2299 1379\n",
      "53\t50349\t3.2\t1\t40043 10306\n",
      "54\t73367\t3.2\t1\t59796 13571\n",
      "55\t6120\t3.2\t1\t4526 1594\n",
      "56\t18476\t3.2\t1\t14265 4211\n",
      "57\t66431\t3.2\t1\t54830 11601\n",
      "58\t2492\t3.2\t1\t1488 1004\n",
      "59\t7423\t3.2\t1\t4507 2916\n",
      "60\t51321\t3.2\t1\t43286 8035\n",
      "61\t1057\t3.2\t1\t524 533\n",
      "62\t2916\t3.2\t1\t1514 1402\n",
      "63\t16873\t3.2\t1\t12893 3980\n",
      "64\t34310\t3.2\t1\t28027 6283\n",
      "65\t36670\t3.2\t1\t29768 6902\n",
      "66\t38998\t3.2\t1\t32040 6958\n",
      "67\t27680\t3.2\t1\t21977 5703\n",
      "68\t30139\t3.2\t1\t23969 6170\n",
      "69\t9433\t3.2\t1\t6500 2933\n",
      "70\t10154\t3.2\t1\t7699 2455\n",
      "71\t16385\t3.2\t1\t13467 2918\n",
      "72\t19640\t3.2\t1\t16226 3414\n",
      "73\t21512\t3.2\t1\t17701 3811\n",
      "74\t22493\t3.2\t1\t18430 4063\n",
      "75\t22985\t3.2\t1\t18770 4215\n",
      "76\t23564\t3.2\t1\t19042 4522\n",
      "77\t23835\t3.2\t1\t19293 4542\n",
      "78\t23686\t3.2\t1\t19111 4575\n",
      "79\t23946\t3.2\t1\t19361 4585\n",
      "80\t23986\t3.2\t1\t19298 4688\n",
      "81\t23804\t3.2\t1\t19288 4516\n",
      "82\t23811\t3.2\t1\t19206 4605\n",
      "83\t23556\t3.2\t1\t18944 4612\n",
      "84\t23072\t3.2\t1\t18538 4534\n",
      "85\t23578\t3.2\t1\t18987 4591\n",
      "86\t23342\t3.2\t1\t18729 4613\n",
      "87\t23024\t3.2\t1\t18465 4559\n",
      "88\t22665\t3.2\t1\t18278 4387\n",
      "89\t22267\t3.2\t1\t17848 4419\n",
      "90\t22426\t3.2\t1\t18024 4402\n",
      "91\t22119\t3.2\t1\t17670 4449\n",
      "92\t21903\t3.2\t1\t17527 4376\n",
      "93\t21581\t3.2\t1\t17328 4253\n",
      "94\t20982\t3.2\t1\t16814 4168\n",
      "95\t20719\t3.2\t1\t16547 4172\n",
      "96\t20401\t3.2\t1\t16240 4161\n",
      "97\t20127\t3.2\t1\t16014 4113\n",
      "98\t19866\t3.2\t1\t15899 3967\n",
      "99\t19812\t3.2\t1\t15778 4034\n",
      "100\t19194\t3.2\t1\t15304 3890\n",
      "101\t18736\t3.2\t1\t14893 3843\n",
      "102\t18130\t3.2\t1\t14414 3716\n",
      "103\t17646\t3.2\t1\t14103 3543\n",
      "104\t17379\t3.2\t1\t13812 3567\n",
      "105\t16898\t3.2\t1\t13401 3497\n",
      "106\t16141\t3.2\t1\t12757 3384\n",
      "107\t15731\t3.2\t1\t12435 3296\n",
      "108\t15140\t3.2\t1\t11901 3239\n",
      "109\t14543\t3.2\t1\t11434 3109\n",
      "110\t14054\t3.2\t1\t10931 3123\n",
      "111\t13746\t3.2\t1\t10800 2946\n",
      "112\t13320\t3.2\t1\t10353 2967\n",
      "113\t12601\t3.2\t1\t9841 2760\n",
      "114\t12035\t3.2\t1\t9427 2608\n",
      "115\t11563\t3.2\t1\t9102 2461\n",
      "116\t11244\t3.2\t1\t8830 2414\n",
      "117\t10939\t3.2\t1\t8547 2392\n",
      "118\t10488\t3.2\t1\t8214 2274\n",
      "119\t9944\t3.2\t1\t7746 2198\n",
      "120\t9315\t3.2\t1\t7136 2179\n",
      "121\t9004\t3.2\t1\t7044 1960\n",
      "122\t8319\t3.2\t1\t6468 1851\n",
      "123\t7911\t3.2\t1\t6134 1777\n",
      "124\t7343\t3.2\t1\t5701 1642\n",
      "125\t6984\t3.2\t1\t5463 1521\n",
      "126\t6625\t3.2\t1\t5163 1462\n",
      "127\t6127\t3.2\t1\t4776 1351\n",
      "128\t5503\t3.2\t1\t4353 1150\n",
      "129\t5386\t3.2\t1\t4334 1052\n",
      "130\t4455\t3.2\t1\t3636 819\n",
      "131\t4178\t3.2\t1\t3487 691\n",
      "132\t3362\t3.2\t1\t2811 551\n",
      "133\t3044\t3.2\t1\t2598 446\n",
      "134\t2477\t3.2\t1\t2143 334\n",
      "135\t2320\t3.2\t1\t2028 292\n",
      "136\t1849\t3.2\t1\t1616 233\n",
      "137\t1640\t3.2\t1\t1429 211\n",
      "138\t1271\t3.2\t1\t1112 159\n",
      "139\t1045\t3.2\t1\t941 104\n",
      "140\t761\t3.2\t1\t666 95\n",
      "141\t600\t3.2\t1\t528 72\n",
      "142\t478\t3.2\t1\t401 77\n",
      "143\t396\t3.2\t1\t338 58\n",
      "144\t395\t3.2\t1\t338 57\n",
      "145\t377\t3.2\t1\t304 73\n",
      "146\t314\t3.2\t1\t248 66\n",
      "147\t287\t3.2\t1\t225 62\n",
      "148\t291\t3.2\t1\t201 90\n",
      "149\t349\t3.2\t1\t241 108\n",
      "150\t1019\t3.2\t1\t426 593\n",
      "151\t158198\t3.2\t1\t58633 99565\n",
      "\n",
      "\n",
      "RUN STATISTICS FOR INPUT FILE: /home/sam/data/geoduck_illumina/trimmed/AD002_S9_L002_R1_001_val_1.fq.gz\n",
      "=============================================\n",
      "212571014 sequences processed in total\n",
      "The length threshold of paired-end sequences gets evaluated later on (in the validation step)\n",
      "\n",
      "Writing report to '/home/sam/data/geoduck_illumina/trimmed/20180129_trimmed_again/AD002_S9_L002_R2_001_val_2.fq.gz_trimming_report.txt'\n",
      "\n",
      "SUMMARISING RUN PARAMETERS\n",
      "==========================\n",
      "Input filename: /home/sam/data/geoduck_illumina/trimmed/AD002_S9_L002_R2_001_val_2.fq.gz\n",
      "Trimming mode: paired-end\n",
      "Trim Galore version: 0.4.4\n",
      "Cutadapt version: 1.9.1\n",
      "Quality Phred score cutoff: 20\n",
      "Quality encoding type selected: ASCII+33\n",
      "Adapter sequence: 'AGATCGGAAGAGC' (Illumina TruSeq, Sanger iPCR; auto-detected)\n",
      "Maximum trimming error rate: 0.1 (default)\n",
      "Minimum required adapter overlap (stringency): 1 bp\n",
      "Minimum required sequence length for both reads before a sequence pair gets removed: 20 bp\n",
      "Output file(s) will be GZIP compressed\n",
      "\n",
      "Writing final adapter and quality trimmed output to AD002_S9_L002_R2_001_val_2_trimmed.fq.gz\n",
      "\n",
      "\n",
      "  >>> Now performing quality (cutoff 20) and adapter trimming in a single pass for the adapter sequence: 'AGATCGGAAGAGC' from file /home/sam/data/geoduck_illumina/trimmed/AD002_S9_L002_R2_001_val_2.fq.gz <<< \n",
      "10000000 sequences processed\n",
      "20000000 sequences processed\n",
      "30000000 sequences processed\n",
      "40000000 sequences processed\n",
      "50000000 sequences processed\n",
      "60000000 sequences processed\n",
      "70000000 sequences processed\n",
      "80000000 sequences processed\n",
      "90000000 sequences processed\n",
      "100000000 sequences processed\n",
      "110000000 sequences processed\n",
      "120000000 sequences processed\n",
      "130000000 sequences processed\n",
      "140000000 sequences processed\n",
      "150000000 sequences processed\n",
      "160000000 sequences processed\n",
      "170000000 sequences processed\n",
      "180000000 sequences processed\n",
      "190000000 sequences processed\n",
      "200000000 sequences processed\n",
      "210000000 sequences processed\n",
      "This is cutadapt 1.9.1 with Python 2.7.12\n",
      "Command line parameters: -f fastq -e 0.1 -q 20 -O 1 -a AGATCGGAAGAGC /home/sam/data/geoduck_illumina/trimmed/AD002_S9_L002_R2_001_val_2.fq.gz\n",
      "Trimming 1 adapter with at most 10.0% errors in single-end mode ...\n",
      "Finished in 5230.58 s (25 us/read; 2.44 M reads/minute).\n",
      "\n",
      "=== Summary ===\n",
      "\n",
      "Total reads processed:             212,571,014\n",
      "Reads with adapters:                84,686,365 (39.8%)\n",
      "Reads written (passing filters):   212,571,014 (100.0%)\n",
      "\n",
      "Total basepairs processed: 27,177,135,971 bp\n",
      "Quality-trimmed:              13,245,075 bp (0.0%)\n",
      "Total written (filtered):  26,838,125,885 bp (98.8%)\n",
      "\n",
      "=== Adapter 1 ===\n",
      "\n",
      "Sequence: AGATCGGAAGAGC; Type: regular 3'; Length: 13; Trimmed: 84686365 times.\n",
      "\n",
      "No. of allowed errors:\n",
      "0-9 bp: 0; 10-13 bp: 1\n",
      "\n",
      "Bases preceding removed adapters:\n",
      "  A: 33.1%\n",
      "  C: 22.4%\n",
      "  G: 14.6%\n",
      "  T: 29.9%\n",
      "  none/other: 0.1%\n",
      "\n",
      "Overview of removed sequences\n",
      "length\tcount\texpect\tmax.err\terror counts\n",
      "1\t60674168\t53142753.5\t0\t60674168\n",
      "2\t14784594\t13285688.4\t0\t14784594\n",
      "3\t3967718\t3321422.1\t0\t3967718\n",
      "4\t1480924\t830355.5\t0\t1480924\n",
      "5\t146347\t207588.9\t0\t146347\n",
      "6\t48580\t51897.2\t0\t48580\n",
      "7\t33791\t12974.3\t0\t33791\n",
      "8\t36656\t3243.6\t0\t36656\n",
      "9\t26598\t810.9\t0\t23963 2635\n",
      "10\t44418\t202.7\t1\t29185 15233\n",
      "11\t38922\t50.7\t1\t26762 12160\n",
      "12\t69792\t12.7\t1\t53960 15832\n",
      "13\t1195\t3.2\t1\t438 757\n",
      "14\t47339\t3.2\t1\t35576 11763\n",
      "15\t52770\t3.2\t1\t41608 11162\n",
      "16\t634\t3.2\t1\t201 433\n",
      "17\t42726\t3.2\t1\t33263 9463\n",
      "18\t22684\t3.2\t1\t17428 5256\n",
      "19\t47567\t3.2\t1\t36469 11098\n",
      "20\t31403\t3.2\t1\t24000 7403\n",
      "21\t28364\t3.2\t1\t21698 6666\n",
      "22\t34047\t3.2\t1\t26354 7693\n",
      "23\t36561\t3.2\t1\t28207 8354\n",
      "24\t65463\t3.2\t1\t49951 15512\n",
      "25\t34947\t3.2\t1\t27542 7405\n",
      "26\t34962\t3.2\t1\t27304 7658\n",
      "27\t29006\t3.2\t1\t21592 7414\n",
      "28\t53205\t3.2\t1\t43379 9826\n",
      "29\t20582\t3.2\t1\t14366 6216\n",
      "30\t53904\t3.2\t1\t43886 10018\n",
      "31\t25572\t3.2\t1\t18856 6716\n",
      "32\t104215\t3.2\t1\t89702 14513\n",
      "33\t48556\t3.2\t1\t38303 10253\n",
      "34\t20322\t3.2\t1\t14286 6036\n",
      "35\t57948\t3.2\t1\t48742 9206\n",
      "36\t17162\t3.2\t1\t12263 4899\n",
      "37\t31403\t3.2\t1\t23930 7473\n",
      "38\t10608\t3.2\t1\t7550 3058\n",
      "39\t18784\t3.2\t1\t13894 4890\n",
      "40\t6633\t3.2\t1\t4568 2065\n",
      "41\t32112\t3.2\t1\t26002 6110\n",
      "42\t45616\t3.2\t1\t37841 7775\n",
      "43\t21704\t3.2\t1\t16865 4839\n",
      "44\t37574\t3.2\t1\t30256 7318\n",
      "45\t87545\t3.2\t1\t75914 11631\n",
      "46\t36246\t3.2\t1\t28464 7782\n",
      "47\t16572\t3.2\t1\t12732 3840\n",
      "48\t80742\t3.2\t1\t69143 11599\n",
      "49\t21381\t3.2\t1\t16623 4758\n",
      "50\t8037\t3.2\t1\t5550 2487\n",
      "51\t63171\t3.2\t1\t53351 9820\n",
      "52\t96340\t3.2\t1\t86234 10106\n",
      "53\t23443\t3.2\t1\t19108 4335\n",
      "54\t36014\t3.2\t1\t29600 6414\n",
      "55\t15334\t3.2\t1\t12053 3281\n",
      "56\t10515\t3.2\t1\t7531 2984\n",
      "57\t18137\t3.2\t1\t14127 4010\n",
      "58\t22631\t3.2\t1\t17766 4865\n",
      "59\t11751\t3.2\t1\t8432 3319\n",
      "60\t10054\t3.2\t1\t6973 3081\n",
      "61\t9870\t3.2\t1\t6812 3058\n",
      "62\t9763\t3.2\t1\t6789 2974\n",
      "63\t10788\t3.2\t1\t7883 2905\n",
      "64\t17688\t3.2\t1\t14248 3440\n",
      "65\t25196\t3.2\t1\t20885 4311\n",
      "66\t29017\t3.2\t1\t24067 4950\n",
      "67\t30511\t3.2\t1\t25231 5280\n",
      "68\t31480\t3.2\t1\t25807 5673\n",
      "69\t32138\t3.2\t1\t26359 5779\n",
      "70\t32297\t3.2\t1\t26434 5863\n",
      "71\t32854\t3.2\t1\t26895 5959\n",
      "72\t32640\t3.2\t1\t26633 6007\n",
      "73\t32787\t3.2\t1\t26654 6133\n",
      "74\t32692\t3.2\t1\t26640 6052\n",
      "75\t32484\t3.2\t1\t26426 6058\n",
      "76\t32184\t3.2\t1\t26187 5997\n",
      "77\t32358\t3.2\t1\t26268 6090\n",
      "78\t31723\t3.2\t1\t25744 5979\n",
      "79\t31696\t3.2\t1\t25821 5875\n",
      "80\t31836\t3.2\t1\t25842 5994\n",
      "81\t31424\t3.2\t1\t25584 5840\n",
      "82\t31042\t3.2\t1\t25270 5772\n",
      "83\t30875\t3.2\t1\t25140 5735\n",
      "84\t30591\t3.2\t1\t24859 5732\n",
      "85\t30931\t3.2\t1\t25246 5685\n",
      "86\t30493\t3.2\t1\t24853 5640\n",
      "87\t30152\t3.2\t1\t24619 5533\n",
      "88\t29712\t3.2\t1\t24263 5449\n",
      "89\t29210\t3.2\t1\t23885 5325\n",
      "90\t28789\t3.2\t1\t23402 5387\n",
      "91\t28484\t3.2\t1\t23115 5369\n",
      "92\t28346\t3.2\t1\t23073 5273\n",
      "93\t27777\t3.2\t1\t22496 5281\n",
      "94\t27264\t3.2\t1\t22205 5059\n",
      "95\t27108\t3.2\t1\t22001 5107\n",
      "96\t26638\t3.2\t1\t21556 5082\n",
      "97\t26212\t3.2\t1\t21295 4917\n",
      "98\t25833\t3.2\t1\t21100 4733\n",
      "99\t25454\t3.2\t1\t20687 4767\n",
      "100\t24923\t3.2\t1\t20366 4557\n",
      "101\t24287\t3.2\t1\t19828 4459\n",
      "102\t23797\t3.2\t1\t19329 4468\n",
      "103\t23126\t3.2\t1\t18853 4273\n",
      "104\t22695\t3.2\t1\t18494 4201\n",
      "105\t22093\t3.2\t1\t17881 4212\n",
      "106\t21097\t3.2\t1\t17034 4063\n",
      "107\t20219\t3.2\t1\t16232 3987\n",
      "108\t19562\t3.2\t1\t15645 3917\n",
      "109\t18798\t3.2\t1\t14996 3802\n",
      "110\t18084\t3.2\t1\t14372 3712\n",
      "111\t17805\t3.2\t1\t14243 3562\n",
      "112\t17170\t3.2\t1\t13696 3474\n",
      "113\t16335\t3.2\t1\t13209 3126\n",
      "114\t15585\t3.2\t1\t12555 3030\n",
      "115\t15021\t3.2\t1\t12002 3019\n",
      "116\t14681\t3.2\t1\t11658 3023\n",
      "117\t14216\t3.2\t1\t11363 2853\n",
      "118\t13499\t3.2\t1\t10813 2686\n",
      "119\t13000\t3.2\t1\t10405 2595\n",
      "120\t12029\t3.2\t1\t9603 2426\n",
      "121\t11664\t3.2\t1\t9343 2321\n",
      "122\t10909\t3.2\t1\t8712 2197\n",
      "123\t10511\t3.2\t1\t8406 2105\n",
      "124\t9754\t3.2\t1\t7760 1994\n",
      "125\t9173\t3.2\t1\t7375 1798\n",
      "126\t8573\t3.2\t1\t6946 1627\n",
      "127\t7964\t3.2\t1\t6426 1538\n",
      "128\t6884\t3.2\t1\t5641 1243\n",
      "129\t6554\t3.2\t1\t5541 1013\n",
      "130\t5155\t3.2\t1\t4404 751\n",
      "131\t4812\t3.2\t1\t4172 640\n",
      "132\t3624\t3.2\t1\t3207 417\n",
      "133\t3284\t3.2\t1\t2958 326\n",
      "134\t2667\t3.2\t1\t2388 279\n",
      "135\t2436\t3.2\t1\t2207 229\n",
      "136\t1933\t3.2\t1\t1774 159\n",
      "137\t1689\t3.2\t1\t1566 123\n",
      "138\t1293\t3.2\t1\t1168 125\n",
      "139\t1070\t3.2\t1\t960 110\n",
      "140\t763\t3.2\t1\t676 87\n",
      "141\t591\t3.2\t1\t514 77\n",
      "142\t474\t3.2\t1\t374 100\n",
      "143\t386\t3.2\t1\t316 70\n",
      "144\t374\t3.2\t1\t285 89\n",
      "145\t336\t3.2\t1\t247 89\n",
      "146\t280\t3.2\t1\t196 84\n",
      "147\t260\t3.2\t1\t184 76\n",
      "148\t209\t3.2\t1\t132 77\n",
      "149\t233\t3.2\t1\t143 90\n",
      "150\t460\t3.2\t1\t349 111\n",
      "151\t59702\t3.2\t1\t58455 1247\n",
      "\n",
      "\n",
      "RUN STATISTICS FOR INPUT FILE: /home/sam/data/geoduck_illumina/trimmed/AD002_S9_L002_R2_001_val_2.fq.gz\n",
      "=============================================\n",
      "212571014 sequences processed in total\n",
      "The length threshold of paired-end sequences gets evaluated later on (in the validation step)\n",
      "\n",
      "Validate paired-end files AD002_S9_L002_R1_001_val_1_trimmed.fq.gz and AD002_S9_L002_R2_001_val_2_trimmed.fq.gz\n",
      "file_1: AD002_S9_L002_R1_001_val_1_trimmed.fq.gz, file_2: AD002_S9_L002_R2_001_val_2_trimmed.fq.gz\n",
      "\n",
      "\n",
      ">>>>> Now validing the length of the 2 paired-end infiles: AD002_S9_L002_R1_001_val_1_trimmed.fq.gz and AD002_S9_L002_R2_001_val_2_trimmed.fq.gz <<<<<\n",
      "Writing validated paired-end read 1 reads to AD002_S9_L002_R1_001_val_1_val_1.fq.gz\n",
      "Writing validated paired-end read 2 reads to AD002_S9_L002_R2_001_val_2_val_2.fq.gz\n",
      "\n",
      "Total number of sequences analysed: 212571014\n",
      "\n",
      "Number of sequence pairs removed because at least one read was shorter than the length cutoff (20 bp): 679425 (0.32%)\n",
      "\n",
      "Deleting both intermediate output files AD002_S9_L002_R1_001_val_1_trimmed.fq.gz and AD002_S9_L002_R2_001_val_2_trimmed.fq.gz\n",
      "\n",
      "====================================================================================================\n",
      "\n",
      "Writing report to '/home/sam/data/geoduck_illumina/trimmed/20180129_trimmed_again/NR005_S4_L001_R1_001_val_1.fq.gz_trimming_report.txt'\n",
      "\n",
      "SUMMARISING RUN PARAMETERS\n",
      "==========================\n",
      "Input filename: /home/sam/data/geoduck_illumina/trimmed/NR005_S4_L001_R1_001_val_1.fq.gz\n",
      "Trimming mode: paired-end\n",
      "Trim Galore version: 0.4.4\n",
      "Cutadapt version: 1.9.1\n",
      "Quality Phred score cutoff: 20\n",
      "Quality encoding type selected: ASCII+33\n",
      "Adapter sequence: 'AGATCGGAAGAGC' (Illumina TruSeq, Sanger iPCR; auto-detected)\n",
      "Maximum trimming error rate: 0.1 (default)\n",
      "Minimum required adapter overlap (stringency): 1 bp\n",
      "Minimum required sequence length for both reads before a sequence pair gets removed: 20 bp\n",
      "Output file(s) will be GZIP compressed\n",
      "\n",
      "Writing final adapter and quality trimmed output to NR005_S4_L001_R1_001_val_1_trimmed.fq.gz\n",
      "\n",
      "\n",
      "  >>> Now performing quality (cutoff 20) and adapter trimming in a single pass for the adapter sequence: 'AGATCGGAAGAGC' from file /home/sam/data/geoduck_illumina/trimmed/NR005_S4_L001_R1_001_val_1.fq.gz <<< \n",
      "This is cutadapt 1.9.1 with Python 2.7.12\n",
      "Command line parameters: -f fastq -e 0.1 -q 20 -O 1 -a AGATCGGAAGAGC /home/sam/data/geoduck_illumina/trimmed/NR005_S4_L001_R1_001_val_1.fq.gz\n",
      "Trimming 1 adapter with at most 10.0% errors in single-end mode ...\n",
      "Finished in 1.14 s (27 us/read; 2.19 M reads/minute).\n",
      "\n",
      "=== Summary ===\n",
      "\n",
      "Total reads processed:                  41,460\n",
      "Reads with adapters:                    26,139 (63.0%)\n",
      "Reads written (passing filters):        41,460 (100.0%)\n",
      "\n",
      "Total basepairs processed:     6,228,334 bp\n",
      "Quality-trimmed:                   1,175 bp (0.0%)\n",
      "Total written (filtered):      5,826,371 bp (93.5%)\n",
      "\n",
      "=== Adapter 1 ===\n",
      "\n",
      "Sequence: AGATCGGAAGAGC; Type: regular 3'; Length: 13; Trimmed: 26139 times.\n",
      "\n",
      "No. of allowed errors:\n",
      "0-9 bp: 0; 10-13 bp: 1\n",
      "\n",
      "Bases preceding removed adapters:\n",
      "  A: 27.4%\n",
      "  C: 30.9%\n",
      "  G: 21.2%\n",
      "  T: 19.5%\n",
      "  none/other: 1.0%\n",
      "\n",
      "Overview of removed sequences\n",
      "length\tcount\texpect\tmax.err\terror counts\n",
      "1\t11126\t10365.0\t0\t11126\n",
      "2\t2116\t2591.2\t0\t2116\n",
      "3\t918\t647.8\t0\t918\n",
      "4\t706\t162.0\t0\t706\n",
      "5\t5\t40.5\t0\t5\n",
      "6\t298\t10.1\t0\t298\n",
      "7\t258\t2.5\t0\t258\n",
      "8\t279\t0.6\t0\t279\n",
      "9\t264\t0.2\t0\t263 1\n",
      "10\t306\t0.0\t1\t288 18\n",
      "11\t303\t0.0\t1\t269 34\n",
      "12\t492\t0.0\t1\t461 31\n",
      "13\t14\t0.0\t1\t8 6\n",
      "14\t526\t0.0\t1\t491 35\n",
      "15\t9\t0.0\t1\t6 3\n",
      "16\t551\t0.0\t1\t504 47\n",
      "17\t18\t0.0\t1\t12 6\n",
      "18\t333\t0.0\t1\t306 27\n",
      "19\t837\t0.0\t1\t788 49\n",
      "20\t1\t0.0\t1\t1\n",
      "21\t1\t0.0\t1\t0 1\n",
      "22\t3\t0.0\t1\t1 2\n",
      "23\t299\t0.0\t1\t271 28\n",
      "24\t625\t0.0\t1\t580 45\n",
      "25\t11\t0.0\t1\t7 4\n",
      "26\t249\t0.0\t1\t227 22\n",
      "27\t208\t0.0\t1\t197 11\n",
      "28\t6\t0.0\t1\t3 3\n",
      "29\t254\t0.0\t1\t235 19\n",
      "30\t264\t0.0\t1\t245 19\n",
      "31\t292\t0.0\t1\t276 16\n",
      "32\t13\t0.0\t1\t8 5\n",
      "33\t370\t0.0\t1\t345 25\n",
      "34\t6\t0.0\t1\t3 3\n",
      "35\t297\t0.0\t1\t273 24\n",
      "36\t4\t0.0\t1\t2 2\n",
      "37\t166\t0.0\t1\t151 15\n",
      "38\t196\t0.0\t1\t182 14\n",
      "39\t159\t0.0\t1\t153 6\n",
      "40\t131\t0.0\t1\t125 6\n",
      "41\t191\t0.0\t1\t179 12\n",
      "42\t360\t0.0\t1\t333 27\n",
      "43\t61\t0.0\t1\t55 6\n",
      "44\t62\t0.0\t1\t58 4\n",
      "45\t1\t0.0\t1\t0 1\n",
      "46\t170\t0.0\t1\t160 10\n",
      "47\t103\t0.0\t1\t96 7\n",
      "48\t109\t0.0\t1\t99 10\n",
      "49\t123\t0.0\t1\t111 12\n",
      "50\t176\t0.0\t1\t165 11\n",
      "51\t81\t0.0\t1\t77 4\n",
      "52\t5\t0.0\t1\t4 1\n",
      "53\t163\t0.0\t1\t153 10\n",
      "54\t207\t0.0\t1\t194 13\n",
      "55\t18\t0.0\t1\t15 3\n",
      "56\t37\t0.0\t1\t31 6\n",
      "57\t168\t0.0\t1\t161 7\n",
      "58\t3\t0.0\t1\t2 1\n",
      "59\t9\t0.0\t1\t7 2\n",
      "60\t104\t0.0\t1\t102 2\n",
      "61\t1\t0.0\t1\t1\n",
      "62\t5\t0.0\t1\t4 1\n",
      "63\t19\t0.0\t1\t16 3\n",
      "64\t70\t0.0\t1\t68 2\n",
      "65\t49\t0.0\t1\t46 3\n",
      "66\t68\t0.0\t1\t66 2\n",
      "67\t37\t0.0\t1\t35 2\n",
      "68\t29\t0.0\t1\t26 3\n",
      "69\t13\t0.0\t1\t12 1\n",
      "70\t19\t0.0\t1\t18 1\n",
      "71\t20\t0.0\t1\t20\n",
      "72\t20\t0.0\t1\t19 1\n",
      "73\t25\t0.0\t1\t24 1\n",
      "74\t22\t0.0\t1\t21 1\n",
      "75\t17\t0.0\t1\t17\n",
      "76\t17\t0.0\t1\t16 1\n",
      "77\t18\t0.0\t1\t17 1\n",
      "78\t20\t0.0\t1\t19 1\n",
      "79\t20\t0.0\t1\t19 1\n",
      "80\t14\t0.0\t1\t13 1\n",
      "81\t15\t0.0\t1\t14 1\n",
      "82\t15\t0.0\t1\t15\n",
      "83\t12\t0.0\t1\t11 1\n",
      "84\t7\t0.0\t1\t7\n",
      "85\t10\t0.0\t1\t10\n",
      "86\t8\t0.0\t1\t7 1\n",
      "87\t8\t0.0\t1\t8\n",
      "88\t11\t0.0\t1\t10 1\n",
      "89\t14\t0.0\t1\t13 1\n",
      "90\t6\t0.0\t1\t5 1\n",
      "91\t6\t0.0\t1\t5 1\n",
      "92\t5\t0.0\t1\t5\n",
      "93\t1\t0.0\t1\t1\n",
      "94\t5\t0.0\t1\t4 1\n",
      "95\t3\t0.0\t1\t3\n",
      "96\t3\t0.0\t1\t3\n",
      "97\t9\t0.0\t1\t9\n",
      "98\t5\t0.0\t1\t5\n",
      "99\t4\t0.0\t1\t4\n",
      "100\t3\t0.0\t1\t3\n",
      "101\t7\t0.0\t1\t6 1\n",
      "102\t7\t0.0\t1\t7\n",
      "104\t2\t0.0\t1\t2\n",
      "105\t3\t0.0\t1\t3\n",
      "106\t2\t0.0\t1\t2\n",
      "107\t1\t0.0\t1\t1\n",
      "109\t2\t0.0\t1\t2\n",
      "110\t4\t0.0\t1\t3 1\n",
      "111\t1\t0.0\t1\t1\n",
      "113\t2\t0.0\t1\t2\n",
      "114\t2\t0.0\t1\t2\n",
      "117\t1\t0.0\t1\t1\n",
      "119\t1\t0.0\t1\t1\n",
      "120\t2\t0.0\t1\t2\n",
      "121\t1\t0.0\t1\t1\n",
      "123\t2\t0.0\t1\t2\n",
      "124\t2\t0.0\t1\t2\n",
      "125\t1\t0.0\t1\t1\n",
      "126\t2\t0.0\t1\t2\n",
      "127\t2\t0.0\t1\t2\n",
      "128\t3\t0.0\t1\t3\n",
      "129\t6\t0.0\t1\t5 1\n",
      "130\t5\t0.0\t1\t5\n",
      "131\t3\t0.0\t1\t3\n",
      "132\t3\t0.0\t1\t3\n",
      "133\t4\t0.0\t1\t3 1\n",
      "134\t9\t0.0\t1\t9\n",
      "135\t7\t0.0\t1\t6 1\n",
      "136\t8\t0.0\t1\t8\n",
      "137\t5\t0.0\t1\t5\n",
      "138\t6\t0.0\t1\t5 1\n",
      "139\t2\t0.0\t1\t2\n",
      "140\t3\t0.0\t1\t3\n",
      "141\t4\t0.0\t1\t3 1\n",
      "142\t9\t0.0\t1\t9\n",
      "143\t3\t0.0\t1\t3\n",
      "145\t1\t0.0\t1\t1\n",
      "146\t6\t0.0\t1\t6\n",
      "147\t1\t0.0\t1\t1\n",
      "148\t11\t0.0\t1\t11\n",
      "149\t3\t0.0\t1\t3\n",
      "150\t1\t0.0\t1\t1\n",
      "151\t271\t0.0\t1\t5 266\n",
      "\n",
      "\n",
      "RUN STATISTICS FOR INPUT FILE: /home/sam/data/geoduck_illumina/trimmed/NR005_S4_L001_R1_001_val_1.fq.gz\n",
      "=============================================\n",
      "41460 sequences processed in total\n",
      "The length threshold of paired-end sequences gets evaluated later on (in the validation step)\n",
      "\n",
      "Writing report to '/home/sam/data/geoduck_illumina/trimmed/20180129_trimmed_again/NR005_S4_L001_R2_001_val_2.fq.gz_trimming_report.txt'\n",
      "\n",
      "SUMMARISING RUN PARAMETERS\n",
      "==========================\n",
      "Input filename: /home/sam/data/geoduck_illumina/trimmed/NR005_S4_L001_R2_001_val_2.fq.gz\n",
      "Trimming mode: paired-end\n",
      "Trim Galore version: 0.4.4\n",
      "Cutadapt version: 1.9.1\n",
      "Quality Phred score cutoff: 20\n",
      "Quality encoding type selected: ASCII+33\n",
      "Adapter sequence: 'AGATCGGAAGAGC' (Illumina TruSeq, Sanger iPCR; auto-detected)\n",
      "Maximum trimming error rate: 0.1 (default)\n",
      "Minimum required adapter overlap (stringency): 1 bp\n",
      "Minimum required sequence length for both reads before a sequence pair gets removed: 20 bp\n",
      "Output file(s) will be GZIP compressed\n",
      "\n",
      "Writing final adapter and quality trimmed output to NR005_S4_L001_R2_001_val_2_trimmed.fq.gz\n",
      "\n",
      "\n",
      "  >>> Now performing quality (cutoff 20) and adapter trimming in a single pass for the adapter sequence: 'AGATCGGAAGAGC' from file /home/sam/data/geoduck_illumina/trimmed/NR005_S4_L001_R2_001_val_2.fq.gz <<< \n",
      "This is cutadapt 1.9.1 with Python 2.7.12\n",
      "Command line parameters: -f fastq -e 0.1 -q 20 -O 1 -a AGATCGGAAGAGC /home/sam/data/geoduck_illumina/trimmed/NR005_S4_L001_R2_001_val_2.fq.gz\n",
      "Trimming 1 adapter with at most 10.0% errors in single-end mode ...\n",
      "Finished in 1.12 s (27 us/read; 2.22 M reads/minute).\n",
      "\n",
      "=== Summary ===\n",
      "\n",
      "Total reads processed:                  41,460\n",
      "Reads with adapters:                    26,148 (63.1%)\n",
      "Reads written (passing filters):        41,460 (100.0%)\n",
      "\n",
      "Total basepairs processed:     6,219,292 bp\n",
      "Quality-trimmed:                   1,420 bp (0.0%)\n",
      "Total written (filtered):      5,865,672 bp (94.3%)\n",
      "\n",
      "=== Adapter 1 ===\n",
      "\n",
      "Sequence: AGATCGGAAGAGC; Type: regular 3'; Length: 13; Trimmed: 26148 times.\n",
      "\n",
      "No. of allowed errors:\n",
      "0-9 bp: 0; 10-13 bp: 1\n",
      "\n",
      "Bases preceding removed adapters:\n",
      "  A: 29.5%\n",
      "  C: 27.8%\n",
      "  G: 22.1%\n",
      "  T: 20.6%\n",
      "  none/other: 0.0%\n",
      "\n",
      "Overview of removed sequences\n",
      "length\tcount\texpect\tmax.err\terror counts\n",
      "1\t11262\t10365.0\t0\t11262\n",
      "2\t2324\t2591.2\t0\t2324\n",
      "3\t1069\t647.8\t0\t1069\n",
      "4\t756\t162.0\t0\t756\n",
      "5\t3\t40.5\t0\t3\n",
      "6\t271\t10.1\t0\t271\n",
      "7\t266\t2.5\t0\t266\n",
      "8\t308\t0.6\t0\t308\n",
      "9\t236\t0.2\t0\t235 1\n",
      "10\t314\t0.0\t1\t290 24\n",
      "11\t297\t0.0\t1\t265 32\n",
      "12\t507\t0.0\t1\t474 33\n",
      "13\t3\t0.0\t1\t1 2\n",
      "14\t297\t0.0\t1\t267 30\n",
      "15\t464\t0.0\t1\t435 29\n",
      "17\t309\t0.0\t1\t284 25\n",
      "18\t198\t0.0\t1\t185 13\n",
      "19\t298\t0.0\t1\t275 23\n",
      "20\t242\t0.0\t1\t221 21\n",
      "21\t207\t0.0\t1\t183 24\n",
      "22\t191\t0.0\t1\t177 14\n",
      "23\t267\t0.0\t1\t250 17\n",
      "24\t396\t0.0\t1\t366 30\n",
      "25\t220\t0.0\t1\t200 20\n",
      "26\t175\t0.0\t1\t162 13\n",
      "27\t159\t0.0\t1\t149 10\n",
      "28\t361\t0.0\t1\t334 27\n",
      "29\t83\t0.0\t1\t74 9\n",
      "30\t303\t0.0\t1\t284 19\n",
      "31\t112\t0.0\t1\t102 10\n",
      "32\t492\t0.0\t1\t464 28\n",
      "33\t176\t0.0\t1\t162 14\n",
      "34\t66\t0.0\t1\t56 10\n",
      "35\t265\t0.0\t1\t252 13\n",
      "36\t74\t0.0\t1\t71 3\n",
      "37\t121\t0.0\t1\t114 7\n",
      "38\t83\t0.0\t1\t82 1\n",
      "39\t95\t0.0\t1\t92 3\n",
      "40\t33\t0.0\t1\t30 3\n",
      "41\t123\t0.0\t1\t118 5\n",
      "42\t192\t0.0\t1\t181 11\n",
      "43\t80\t0.0\t1\t73 7\n",
      "44\t123\t0.0\t1\t119 4\n",
      "45\t226\t0.0\t1\t217 9\n",
      "46\t125\t0.0\t1\t118 7\n",
      "47\t75\t0.0\t1\t71 4\n",
      "48\t217\t0.0\t1\t203 14\n",
      "49\t70\t0.0\t1\t68 2\n",
      "50\t21\t0.0\t1\t20 1\n",
      "51\t147\t0.0\t1\t138 9\n",
      "52\t218\t0.0\t1\t204 14\n",
      "53\t86\t0.0\t1\t82 4\n",
      "54\t90\t0.0\t1\t83 7\n",
      "55\t52\t0.0\t1\t50 2\n",
      "56\t20\t0.0\t1\t18 2\n",
      "57\t53\t0.0\t1\t52 1\n",
      "58\t58\t0.0\t1\t58\n",
      "59\t28\t0.0\t1\t21 7\n",
      "60\t16\t0.0\t1\t15 1\n",
      "61\t11\t0.0\t1\t10 1\n",
      "62\t18\t0.0\t1\t15 3\n",
      "63\t21\t0.0\t1\t21\n",
      "64\t27\t0.0\t1\t27\n",
      "65\t30\t0.0\t1\t28 2\n",
      "66\t43\t0.0\t1\t42 1\n",
      "67\t52\t0.0\t1\t51 1\n",
      "68\t45\t0.0\t1\t43 2\n",
      "69\t38\t0.0\t1\t35 3\n",
      "70\t43\t0.0\t1\t42 1\n",
      "71\t24\t0.0\t1\t23 1\n",
      "72\t21\t0.0\t1\t20 1\n",
      "73\t25\t0.0\t1\t23 2\n",
      "74\t23\t0.0\t1\t23\n",
      "75\t17\t0.0\t1\t17\n",
      "76\t17\t0.0\t1\t17\n",
      "77\t19\t0.0\t1\t18 1\n",
      "78\t19\t0.0\t1\t19\n",
      "79\t16\t0.0\t1\t15 1\n",
      "80\t16\t0.0\t1\t15 1\n",
      "81\t16\t0.0\t1\t15 1\n",
      "82\t13\t0.0\t1\t13\n",
      "83\t12\t0.0\t1\t12\n",
      "84\t8\t0.0\t1\t8\n",
      "85\t10\t0.0\t1\t10\n",
      "86\t6\t0.0\t1\t5 1\n",
      "87\t7\t0.0\t1\t7\n",
      "88\t10\t0.0\t1\t10\n",
      "89\t14\t0.0\t1\t14\n",
      "90\t6\t0.0\t1\t6\n",
      "91\t5\t0.0\t1\t5\n",
      "92\t7\t0.0\t1\t7\n",
      "93\t2\t0.0\t1\t2\n",
      "94\t5\t0.0\t1\t5\n",
      "95\t3\t0.0\t1\t3\n",
      "96\t3\t0.0\t1\t3\n",
      "97\t9\t0.0\t1\t9\n",
      "98\t5\t0.0\t1\t5\n",
      "99\t4\t0.0\t1\t4\n",
      "100\t3\t0.0\t1\t3\n",
      "101\t7\t0.0\t1\t7\n",
      "102\t7\t0.0\t1\t7\n",
      "104\t2\t0.0\t1\t2\n",
      "105\t3\t0.0\t1\t3\n",
      "106\t2\t0.0\t1\t2\n",
      "107\t1\t0.0\t1\t1\n",
      "109\t2\t0.0\t1\t2\n",
      "110\t4\t0.0\t1\t3 1\n",
      "111\t1\t0.0\t1\t1\n",
      "113\t2\t0.0\t1\t2\n",
      "114\t2\t0.0\t1\t2\n",
      "118\t1\t0.0\t1\t1\n",
      "119\t1\t0.0\t1\t1\n",
      "120\t2\t0.0\t1\t2\n",
      "121\t1\t0.0\t1\t1\n",
      "123\t2\t0.0\t1\t2\n",
      "124\t2\t0.0\t1\t1 1\n",
      "126\t2\t0.0\t1\t2\n",
      "127\t2\t0.0\t1\t2\n",
      "128\t4\t0.0\t1\t4\n",
      "129\t6\t0.0\t1\t6\n",
      "130\t4\t0.0\t1\t4\n",
      "131\t3\t0.0\t1\t3\n",
      "132\t2\t0.0\t1\t2\n",
      "133\t3\t0.0\t1\t2 1\n",
      "134\t10\t0.0\t1\t10\n",
      "135\t7\t0.0\t1\t6 1\n",
      "136\t8\t0.0\t1\t8\n",
      "137\t5\t0.0\t1\t5\n",
      "138\t6\t0.0\t1\t6\n",
      "139\t2\t0.0\t1\t2\n",
      "140\t3\t0.0\t1\t3\n",
      "141\t3\t0.0\t1\t0 3\n",
      "142\t9\t0.0\t1\t7 2\n",
      "143\t3\t0.0\t1\t2 1\n",
      "145\t1\t0.0\t1\t0 1\n",
      "146\t6\t0.0\t1\t2 4\n",
      "147\t1\t0.0\t1\t0 1\n",
      "148\t11\t0.0\t1\t7 4\n",
      "149\t2\t0.0\t1\t1 1\n",
      "150\t2\t0.0\t1\t0 2\n",
      "151\t5\t0.0\t1\t2 3\n",
      "\n",
      "\n",
      "RUN STATISTICS FOR INPUT FILE: /home/sam/data/geoduck_illumina/trimmed/NR005_S4_L001_R2_001_val_2.fq.gz\n",
      "=============================================\n",
      "41460 sequences processed in total\n",
      "The length threshold of paired-end sequences gets evaluated later on (in the validation step)\n",
      "\n",
      "Validate paired-end files NR005_S4_L001_R1_001_val_1_trimmed.fq.gz and NR005_S4_L001_R2_001_val_2_trimmed.fq.gz\n",
      "file_1: NR005_S4_L001_R1_001_val_1_trimmed.fq.gz, file_2: NR005_S4_L001_R2_001_val_2_trimmed.fq.gz\n",
      "\n",
      "\n",
      ">>>>> Now validing the length of the 2 paired-end infiles: NR005_S4_L001_R1_001_val_1_trimmed.fq.gz and NR005_S4_L001_R2_001_val_2_trimmed.fq.gz <<<<<\n",
      "Writing validated paired-end read 1 reads to NR005_S4_L001_R1_001_val_1_val_1.fq.gz\n",
      "Writing validated paired-end read 2 reads to NR005_S4_L001_R2_001_val_2_val_2.fq.gz\n",
      "\n",
      "Total number of sequences analysed: 41460\n",
      "\n",
      "Number of sequence pairs removed because at least one read was shorter than the length cutoff (20 bp): 361 (0.87%)\n",
      "\n",
      "Deleting both intermediate output files NR005_S4_L001_R1_001_val_1_trimmed.fq.gz and NR005_S4_L001_R2_001_val_2_trimmed.fq.gz\n",
      "\n",
      "====================================================================================================\n",
      "\n",
      "Writing report to '/home/sam/data/geoduck_illumina/trimmed/20180129_trimmed_again/NR005_S4_L002_R1_001_val_1.fq.gz_trimming_report.txt'\n",
      "\n",
      "SUMMARISING RUN PARAMETERS\n",
      "==========================\n",
      "Input filename: /home/sam/data/geoduck_illumina/trimmed/NR005_S4_L002_R1_001_val_1.fq.gz\n",
      "Trimming mode: paired-end\n",
      "Trim Galore version: 0.4.4\n",
      "Cutadapt version: 1.9.1\n",
      "Quality Phred score cutoff: 20\n",
      "Quality encoding type selected: ASCII+33\n",
      "Adapter sequence: 'AGATCGGAAGAGC' (Illumina TruSeq, Sanger iPCR; auto-detected)\n",
      "Maximum trimming error rate: 0.1 (default)\n",
      "Minimum required adapter overlap (stringency): 1 bp\n",
      "Minimum required sequence length for both reads before a sequence pair gets removed: 20 bp\n",
      "Output file(s) will be GZIP compressed\n",
      "\n",
      "Writing final adapter and quality trimmed output to NR005_S4_L002_R1_001_val_1_trimmed.fq.gz\n",
      "\n",
      "\n",
      "  >>> Now performing quality (cutoff 20) and adapter trimming in a single pass for the adapter sequence: 'AGATCGGAAGAGC' from file /home/sam/data/geoduck_illumina/trimmed/NR005_S4_L002_R1_001_val_1.fq.gz <<< \n",
      "This is cutadapt 1.9.1 with Python 2.7.12\n",
      "Command line parameters: -f fastq -e 0.1 -q 20 -O 1 -a AGATCGGAAGAGC /home/sam/data/geoduck_illumina/trimmed/NR005_S4_L002_R1_001_val_1.fq.gz\n",
      "Trimming 1 adapter with at most 10.0% errors in single-end mode ...\n",
      "Finished in 0.99 s (27 us/read; 2.20 M reads/minute).\n",
      "\n",
      "=== Summary ===\n",
      "\n",
      "Total reads processed:                  36,456\n",
      "Reads with adapters:                    23,392 (64.2%)\n",
      "Reads written (passing filters):        36,456 (100.0%)\n",
      "\n",
      "Total basepairs processed:     5,477,053 bp\n",
      "Quality-trimmed:                     660 bp (0.0%)\n",
      "Total written (filtered):      5,060,825 bp (92.4%)\n",
      "\n",
      "=== Adapter 1 ===\n",
      "\n",
      "Sequence: AGATCGGAAGAGC; Type: regular 3'; Length: 13; Trimmed: 23392 times.\n",
      "\n",
      "No. of allowed errors:\n",
      "0-9 bp: 0; 10-13 bp: 1\n",
      "\n",
      "Bases preceding removed adapters:\n",
      "  A: 26.8%\n",
      "  C: 31.3%\n",
      "  G: 21.4%\n",
      "  T: 18.8%\n",
      "  none/other: 1.6%\n",
      "\n",
      "Overview of removed sequences\n",
      "length\tcount\texpect\tmax.err\terror counts\n",
      "1\t9374\t9114.0\t0\t9374\n",
      "2\t1817\t2278.5\t0\t1817\n",
      "3\t794\t569.6\t0\t794\n",
      "4\t622\t142.4\t0\t622\n",
      "5\t2\t35.6\t0\t2\n",
      "6\t253\t8.9\t0\t253\n",
      "7\t249\t2.2\t0\t249\n",
      "8\t292\t0.6\t0\t292\n",
      "9\t244\t0.1\t0\t243 1\n",
      "10\t228\t0.0\t1\t218 10\n",
      "11\t253\t0.0\t1\t234 19\n",
      "12\t483\t0.0\t1\t457 26\n",
      "13\t7\t0.0\t1\t3 4\n",
      "14\t426\t0.0\t1\t402 24\n",
      "15\t8\t0.0\t1\t2 6\n",
      "16\t499\t0.0\t1\t464 35\n",
      "17\t9\t0.0\t1\t6 3\n",
      "18\t269\t0.0\t1\t252 17\n",
      "19\t848\t0.0\t1\t812 36\n",
      "20\t6\t0.0\t1\t3 3\n",
      "22\t2\t0.0\t1\t2\n",
      "23\t231\t0.0\t1\t221 10\n",
      "24\t600\t0.0\t1\t570 30\n",
      "25\t7\t0.0\t1\t4 3\n",
      "26\t203\t0.0\t1\t191 12\n",
      "27\t194\t0.0\t1\t191 3\n",
      "28\t5\t0.0\t1\t4 1\n",
      "29\t219\t0.0\t1\t212 7\n",
      "30\t216\t0.0\t1\t206 10\n",
      "31\t289\t0.0\t1\t283 6\n",
      "32\t5\t0.0\t1\t3 2\n",
      "33\t325\t0.0\t1\t302 23\n",
      "34\t9\t0.0\t1\t5 4\n",
      "35\t313\t0.0\t1\t303 10\n",
      "36\t3\t0.0\t1\t3\n",
      "37\t169\t0.0\t1\t156 13\n",
      "38\t177\t0.0\t1\t168 9\n",
      "39\t145\t0.0\t1\t138 7\n",
      "40\t125\t0.0\t1\t119 6\n",
      "41\t177\t0.0\t1\t164 13\n",
      "42\t362\t0.0\t1\t338 24\n",
      "43\t60\t0.0\t1\t56 4\n",
      "44\t58\t0.0\t1\t58\n",
      "45\t1\t0.0\t1\t1\n",
      "46\t118\t0.0\t1\t115 3\n",
      "47\t109\t0.0\t1\t101 8\n",
      "48\t116\t0.0\t1\t114 2\n",
      "49\t103\t0.0\t1\t92 11\n",
      "50\t178\t0.0\t1\t170 8\n",
      "51\t101\t0.0\t1\t100 1\n",
      "52\t7\t0.0\t1\t3 4\n",
      "53\t140\t0.0\t1\t128 12\n",
      "54\t218\t0.0\t1\t208 10\n",
      "55\t10\t0.0\t1\t9 1\n",
      "56\t32\t0.0\t1\t27 5\n",
      "57\t225\t0.0\t1\t217 8\n",
      "58\t2\t0.0\t1\t2\n",
      "59\t3\t0.0\t1\t2 1\n",
      "60\t113\t0.0\t1\t110 3\n",
      "61\t2\t0.0\t1\t2\n",
      "63\t51\t0.0\t1\t50 1\n",
      "64\t49\t0.0\t1\t47 2\n",
      "65\t50\t0.0\t1\t46 4\n",
      "66\t64\t0.0\t1\t61 3\n",
      "67\t44\t0.0\t1\t43 1\n",
      "68\t50\t0.0\t1\t49 1\n",
      "69\t8\t0.0\t1\t8\n",
      "70\t11\t0.0\t1\t9 2\n",
      "71\t28\t0.0\t1\t28\n",
      "72\t27\t0.0\t1\t26 1\n",
      "73\t26\t0.0\t1\t24 2\n",
      "74\t17\t0.0\t1\t16 1\n",
      "75\t23\t0.0\t1\t21 2\n",
      "76\t20\t0.0\t1\t19 1\n",
      "77\t19\t0.0\t1\t19\n",
      "78\t26\t0.0\t1\t26\n",
      "79\t18\t0.0\t1\t16 2\n",
      "80\t20\t0.0\t1\t18 2\n",
      "81\t17\t0.0\t1\t17\n",
      "82\t14\t0.0\t1\t13 1\n",
      "83\t9\t0.0\t1\t9\n",
      "84\t14\t0.0\t1\t13 1\n",
      "85\t13\t0.0\t1\t12 1\n",
      "86\t18\t0.0\t1\t17 1\n",
      "87\t13\t0.0\t1\t13\n",
      "88\t11\t0.0\t1\t10 1\n",
      "89\t10\t0.0\t1\t9 1\n",
      "90\t15\t0.0\t1\t15\n",
      "91\t5\t0.0\t1\t5\n",
      "92\t10\t0.0\t1\t10\n",
      "93\t9\t0.0\t1\t9\n",
      "94\t3\t0.0\t1\t2 1\n",
      "95\t7\t0.0\t1\t7\n",
      "96\t5\t0.0\t1\t5\n",
      "97\t5\t0.0\t1\t5\n",
      "98\t3\t0.0\t1\t3\n",
      "99\t6\t0.0\t1\t6\n",
      "100\t8\t0.0\t1\t8\n",
      "101\t1\t0.0\t1\t1\n",
      "102\t6\t0.0\t1\t5 1\n",
      "103\t1\t0.0\t1\t1\n",
      "104\t4\t0.0\t1\t3 1\n",
      "105\t3\t0.0\t1\t3\n",
      "106\t1\t0.0\t1\t1\n",
      "107\t2\t0.0\t1\t2\n",
      "108\t2\t0.0\t1\t2\n",
      "109\t3\t0.0\t1\t2 1\n",
      "110\t1\t0.0\t1\t1\n",
      "112\t2\t0.0\t1\t2\n",
      "113\t1\t0.0\t1\t1\n",
      "114\t2\t0.0\t1\t2\n",
      "115\t2\t0.0\t1\t2\n",
      "116\t2\t0.0\t1\t2\n",
      "117\t4\t0.0\t1\t3 1\n",
      "118\t2\t0.0\t1\t2\n",
      "120\t3\t0.0\t1\t3\n",
      "121\t4\t0.0\t1\t4\n",
      "122\t4\t0.0\t1\t2 2\n",
      "123\t1\t0.0\t1\t1\n",
      "124\t2\t0.0\t1\t2\n",
      "125\t3\t0.0\t1\t2 1\n",
      "126\t4\t0.0\t1\t3 1\n",
      "127\t4\t0.0\t1\t3 1\n",
      "128\t5\t0.0\t1\t5\n",
      "129\t8\t0.0\t1\t7 1\n",
      "130\t5\t0.0\t1\t4 1\n",
      "131\t6\t0.0\t1\t6\n",
      "132\t5\t0.0\t1\t5\n",
      "133\t7\t0.0\t1\t7\n",
      "134\t19\t0.0\t1\t19\n",
      "135\t10\t0.0\t1\t10\n",
      "136\t3\t0.0\t1\t3\n",
      "137\t6\t0.0\t1\t6\n",
      "138\t8\t0.0\t1\t8\n",
      "139\t10\t0.0\t1\t8 2\n",
      "140\t5\t0.0\t1\t5\n",
      "141\t4\t0.0\t1\t4\n",
      "142\t9\t0.0\t1\t8 1\n",
      "143\t5\t0.0\t1\t5\n",
      "144\t1\t0.0\t1\t1\n",
      "145\t3\t0.0\t1\t3\n",
      "146\t9\t0.0\t1\t9\n",
      "147\t3\t0.0\t1\t3\n",
      "148\t8\t0.0\t1\t8\n",
      "149\t4\t0.0\t1\t4\n",
      "150\t3\t0.0\t1\t1 2\n",
      "151\t381\t0.0\t1\t3 378\n",
      "\n",
      "\n",
      "RUN STATISTICS FOR INPUT FILE: /home/sam/data/geoduck_illumina/trimmed/NR005_S4_L002_R1_001_val_1.fq.gz\n",
      "=============================================\n",
      "36456 sequences processed in total\n",
      "The length threshold of paired-end sequences gets evaluated later on (in the validation step)\n",
      "\n",
      "Writing report to '/home/sam/data/geoduck_illumina/trimmed/20180129_trimmed_again/NR005_S4_L002_R2_001_val_2.fq.gz_trimming_report.txt'\n",
      "\n",
      "SUMMARISING RUN PARAMETERS\n",
      "==========================\n",
      "Input filename: /home/sam/data/geoduck_illumina/trimmed/NR005_S4_L002_R2_001_val_2.fq.gz\n",
      "Trimming mode: paired-end\n",
      "Trim Galore version: 0.4.4\n",
      "Cutadapt version: 1.9.1\n",
      "Quality Phred score cutoff: 20\n",
      "Quality encoding type selected: ASCII+33\n",
      "Adapter sequence: 'AGATCGGAAGAGC' (Illumina TruSeq, Sanger iPCR; auto-detected)\n",
      "Maximum trimming error rate: 0.1 (default)\n",
      "Minimum required adapter overlap (stringency): 1 bp\n",
      "Minimum required sequence length for both reads before a sequence pair gets removed: 20 bp\n",
      "Output file(s) will be GZIP compressed\n",
      "\n",
      "Writing final adapter and quality trimmed output to NR005_S4_L002_R2_001_val_2_trimmed.fq.gz\n",
      "\n",
      "\n",
      "  >>> Now performing quality (cutoff 20) and adapter trimming in a single pass for the adapter sequence: 'AGATCGGAAGAGC' from file /home/sam/data/geoduck_illumina/trimmed/NR005_S4_L002_R2_001_val_2.fq.gz <<< \n",
      "This is cutadapt 1.9.1 with Python 2.7.12\n",
      "Command line parameters: -f fastq -e 0.1 -q 20 -O 1 -a AGATCGGAAGAGC /home/sam/data/geoduck_illumina/trimmed/NR005_S4_L002_R2_001_val_2.fq.gz\n",
      "Trimming 1 adapter with at most 10.0% errors in single-end mode ...\n",
      "Finished in 0.98 s (27 us/read; 2.23 M reads/minute).\n",
      "\n",
      "=== Summary ===\n",
      "\n",
      "Total reads processed:                  36,456\n",
      "Reads with adapters:                    23,277 (63.8%)\n",
      "Reads written (passing filters):        36,456 (100.0%)\n",
      "\n",
      "Total basepairs processed:     5,468,053 bp\n",
      "Quality-trimmed:                     843 bp (0.0%)\n",
      "Total written (filtered):      5,116,805 bp (93.6%)\n",
      "\n",
      "=== Adapter 1 ===\n",
      "\n",
      "Sequence: AGATCGGAAGAGC; Type: regular 3'; Length: 13; Trimmed: 23277 times.\n",
      "\n",
      "No. of allowed errors:\n",
      "0-9 bp: 0; 10-13 bp: 1\n",
      "\n",
      "Bases preceding removed adapters:\n",
      "  A: 29.1%\n",
      "  C: 28.0%\n",
      "  G: 22.5%\n",
      "  T: 20.4%\n",
      "  none/other: 0.0%\n",
      "\n",
      "Overview of removed sequences\n",
      "length\tcount\texpect\tmax.err\terror counts\n",
      "1\t9539\t9114.0\t0\t9539\n",
      "2\t1891\t2278.5\t0\t1891\n",
      "3\t931\t569.6\t0\t931\n",
      "4\t660\t142.4\t0\t660\n",
      "5\t9\t35.6\t0\t9\n",
      "6\t246\t8.9\t0\t246\n",
      "7\t265\t2.2\t0\t265\n",
      "8\t305\t0.6\t0\t305\n",
      "9\t225\t0.1\t0\t225\n",
      "10\t229\t0.0\t1\t218 11\n",
      "11\t238\t0.0\t1\t226 12\n",
      "12\t498\t0.0\t1\t473 25\n",
      "13\t2\t0.0\t1\t1 1\n",
      "14\t260\t0.0\t1\t249 11\n",
      "15\t402\t0.0\t1\t380 22\n",
      "16\t3\t0.0\t1\t1 2\n",
      "17\t268\t0.0\t1\t261 7\n",
      "18\t165\t0.0\t1\t158 7\n",
      "19\t294\t0.0\t1\t284 10\n",
      "20\t234\t0.0\t1\t227 7\n",
      "21\t181\t0.0\t1\t175 6\n",
      "22\t212\t0.0\t1\t206 6\n",
      "23\t223\t0.0\t1\t218 5\n",
      "24\t286\t0.0\t1\t278 8\n",
      "25\t204\t0.0\t1\t197 7\n",
      "26\t212\t0.0\t1\t200 12\n",
      "27\t130\t0.0\t1\t121 9\n",
      "28\t329\t0.0\t1\t322 7\n",
      "29\t74\t0.0\t1\t72 2\n",
      "30\t260\t0.0\t1\t253 7\n",
      "31\t91\t0.0\t1\t87 4\n",
      "32\t597\t0.0\t1\t589 8\n",
      "33\t169\t0.0\t1\t159 10\n",
      "34\t46\t0.0\t1\t44 2\n",
      "35\t290\t0.0\t1\t284 6\n",
      "36\t63\t0.0\t1\t59 4\n",
      "37\t125\t0.0\t1\t123 2\n",
      "38\t46\t0.0\t1\t45 1\n",
      "39\t56\t0.0\t1\t56\n",
      "40\t18\t0.0\t1\t17 1\n",
      "41\t130\t0.0\t1\t129 1\n",
      "42\t209\t0.0\t1\t202 7\n",
      "43\t84\t0.0\t1\t78 6\n",
      "44\t100\t0.0\t1\t97 3\n",
      "45\t265\t0.0\t1\t260 5\n",
      "46\t96\t0.0\t1\t91 5\n",
      "47\t61\t0.0\t1\t57 4\n",
      "48\t253\t0.0\t1\t250 3\n",
      "49\t62\t0.0\t1\t61 1\n",
      "50\t14\t0.0\t1\t12 2\n",
      "51\t156\t0.0\t1\t148 8\n",
      "52\t307\t0.0\t1\t302 5\n",
      "53\t95\t0.0\t1\t91 4\n",
      "54\t93\t0.0\t1\t89 4\n",
      "55\t32\t0.0\t1\t32\n",
      "56\t19\t0.0\t1\t19\n",
      "57\t34\t0.0\t1\t32 2\n",
      "58\t41\t0.0\t1\t37 4\n",
      "59\t24\t0.0\t1\t24\n",
      "60\t13\t0.0\t1\t12 1\n",
      "61\t9\t0.0\t1\t8 1\n",
      "62\t11\t0.0\t1\t10 1\n",
      "63\t16\t0.0\t1\t15 1\n",
      "64\t30\t0.0\t1\t30\n",
      "65\t38\t0.0\t1\t33 5\n",
      "66\t31\t0.0\t1\t31\n",
      "67\t45\t0.0\t1\t45\n",
      "68\t39\t0.0\t1\t38 1\n",
      "69\t24\t0.0\t1\t24\n",
      "70\t30\t0.0\t1\t30\n",
      "71\t41\t0.0\t1\t39 2\n",
      "72\t33\t0.0\t1\t33\n",
      "73\t27\t0.0\t1\t26 1\n",
      "74\t16\t0.0\t1\t16\n",
      "75\t22\t0.0\t1\t21 1\n",
      "76\t20\t0.0\t1\t20\n",
      "77\t19\t0.0\t1\t19\n",
      "78\t24\t0.0\t1\t24\n",
      "79\t19\t0.0\t1\t18 1\n",
      "80\t17\t0.0\t1\t17\n",
      "81\t18\t0.0\t1\t18\n",
      "82\t13\t0.0\t1\t13\n",
      "83\t10\t0.0\t1\t9 1\n",
      "84\t12\t0.0\t1\t12\n",
      "85\t13\t0.0\t1\t13\n",
      "86\t17\t0.0\t1\t17\n",
      "87\t14\t0.0\t1\t14\n",
      "88\t12\t0.0\t1\t10 2\n",
      "89\t11\t0.0\t1\t11\n",
      "90\t13\t0.0\t1\t13\n",
      "91\t6\t0.0\t1\t6\n",
      "92\t9\t0.0\t1\t9\n",
      "93\t9\t0.0\t1\t9\n",
      "94\t2\t0.0\t1\t2\n",
      "95\t7\t0.0\t1\t6 1\n",
      "96\t4\t0.0\t1\t4\n",
      "97\t5\t0.0\t1\t5\n",
      "98\t4\t0.0\t1\t4\n",
      "99\t5\t0.0\t1\t5\n",
      "100\t7\t0.0\t1\t7\n",
      "101\t2\t0.0\t1\t2\n",
      "102\t6\t0.0\t1\t6\n",
      "103\t1\t0.0\t1\t1\n",
      "104\t5\t0.0\t1\t4 1\n",
      "105\t3\t0.0\t1\t3\n",
      "106\t1\t0.0\t1\t1\n",
      "107\t2\t0.0\t1\t2\n",
      "108\t2\t0.0\t1\t2\n",
      "109\t3\t0.0\t1\t3\n",
      "110\t1\t0.0\t1\t1\n",
      "112\t2\t0.0\t1\t2\n",
      "113\t1\t0.0\t1\t1\n",
      "114\t2\t0.0\t1\t2\n",
      "115\t2\t0.0\t1\t1 1\n",
      "116\t2\t0.0\t1\t2\n",
      "117\t3\t0.0\t1\t2 1\n",
      "118\t2\t0.0\t1\t2\n",
      "120\t3\t0.0\t1\t2 1\n",
      "121\t3\t0.0\t1\t3\n",
      "122\t4\t0.0\t1\t4\n",
      "123\t2\t0.0\t1\t2\n",
      "124\t2\t0.0\t1\t2\n",
      "125\t3\t0.0\t1\t3\n",
      "126\t4\t0.0\t1\t4\n",
      "127\t4\t0.0\t1\t4\n",
      "128\t5\t0.0\t1\t4 1\n",
      "129\t8\t0.0\t1\t8\n",
      "130\t5\t0.0\t1\t5\n",
      "131\t6\t0.0\t1\t6\n",
      "132\t5\t0.0\t1\t4 1\n",
      "133\t7\t0.0\t1\t7\n",
      "134\t19\t0.0\t1\t19\n",
      "135\t10\t0.0\t1\t9 1\n",
      "136\t3\t0.0\t1\t3\n",
      "137\t6\t0.0\t1\t5 1\n",
      "138\t8\t0.0\t1\t7 1\n",
      "139\t9\t0.0\t1\t9\n",
      "140\t5\t0.0\t1\t2 3\n",
      "141\t4\t0.0\t1\t4\n",
      "142\t9\t0.0\t1\t8 1\n",
      "143\t5\t0.0\t1\t3 2\n",
      "144\t1\t0.0\t1\t1\n",
      "145\t3\t0.0\t1\t2 1\n",
      "146\t8\t0.0\t1\t2 6\n",
      "147\t3\t0.0\t1\t2 1\n",
      "148\t8\t0.0\t1\t5 3\n",
      "149\t4\t0.0\t1\t4\n",
      "150\t1\t0.0\t1\t0 1\n",
      "151\t4\t0.0\t1\t3 1\n",
      "\n",
      "\n",
      "RUN STATISTICS FOR INPUT FILE: /home/sam/data/geoduck_illumina/trimmed/NR005_S4_L002_R2_001_val_2.fq.gz\n",
      "=============================================\n",
      "36456 sequences processed in total\n",
      "The length threshold of paired-end sequences gets evaluated later on (in the validation step)\n",
      "\n",
      "Validate paired-end files NR005_S4_L002_R1_001_val_1_trimmed.fq.gz and NR005_S4_L002_R2_001_val_2_trimmed.fq.gz\n",
      "file_1: NR005_S4_L002_R1_001_val_1_trimmed.fq.gz, file_2: NR005_S4_L002_R2_001_val_2_trimmed.fq.gz\n",
      "\n",
      "\n",
      ">>>>> Now validing the length of the 2 paired-end infiles: NR005_S4_L002_R1_001_val_1_trimmed.fq.gz and NR005_S4_L002_R2_001_val_2_trimmed.fq.gz <<<<<\n",
      "Writing validated paired-end read 1 reads to NR005_S4_L002_R1_001_val_1_val_1.fq.gz\n",
      "Writing validated paired-end read 2 reads to NR005_S4_L002_R2_001_val_2_val_2.fq.gz\n",
      "\n",
      "Total number of sequences analysed: 36456\n",
      "\n",
      "Number of sequence pairs removed because at least one read was shorter than the length cutoff (20 bp): 506 (1.39%)\n",
      "\n",
      "Deleting both intermediate output files NR005_S4_L002_R1_001_val_1_trimmed.fq.gz and NR005_S4_L002_R2_001_val_2_trimmed.fq.gz\n",
      "\n",
      "====================================================================================================\n",
      "\n",
      "Writing report to '/home/sam/data/geoduck_illumina/trimmed/20180129_trimmed_again/NR006_S3_L001_R1_001_val_1.fq.gz_trimming_report.txt'\n",
      "\n",
      "SUMMARISING RUN PARAMETERS\n",
      "==========================\n",
      "Input filename: /home/sam/data/geoduck_illumina/trimmed/NR006_S3_L001_R1_001_val_1.fq.gz\n",
      "Trimming mode: paired-end\n",
      "Trim Galore version: 0.4.4\n",
      "Cutadapt version: 1.9.1\n",
      "Quality Phred score cutoff: 20\n",
      "Quality encoding type selected: ASCII+33\n",
      "Adapter sequence: 'AGATCGGAAGAGC' (Illumina TruSeq, Sanger iPCR; auto-detected)\n",
      "Maximum trimming error rate: 0.1 (default)\n",
      "Minimum required adapter overlap (stringency): 1 bp\n",
      "Minimum required sequence length for both reads before a sequence pair gets removed: 20 bp\n",
      "Output file(s) will be GZIP compressed\n",
      "\n",
      "Writing final adapter and quality trimmed output to NR006_S3_L001_R1_001_val_1_trimmed.fq.gz\n",
      "\n",
      "\n",
      "  >>> Now performing quality (cutoff 20) and adapter trimming in a single pass for the adapter sequence: 'AGATCGGAAGAGC' from file /home/sam/data/geoduck_illumina/trimmed/NR006_S3_L001_R1_001_val_1.fq.gz <<< \n",
      "This is cutadapt 1.9.1 with Python 2.7.12\n",
      "Command line parameters: -f fastq -e 0.1 -q 20 -O 1 -a AGATCGGAAGAGC /home/sam/data/geoduck_illumina/trimmed/NR006_S3_L001_R1_001_val_1.fq.gz\n",
      "Trimming 1 adapter with at most 10.0% errors in single-end mode ...\n",
      "Finished in 89.69 s (26 us/read; 2.28 M reads/minute).\n",
      "\n",
      "=== Summary ===\n",
      "\n",
      "Total reads processed:               3,414,678\n",
      "Reads with adapters:                 2,142,823 (62.8%)\n",
      "Reads written (passing filters):     3,414,678 (100.0%)\n",
      "\n",
      "Total basepairs processed:   513,251,944 bp\n",
      "Quality-trimmed:                  91,200 bp (0.0%)\n",
      "Total written (filtered):    475,253,670 bp (92.6%)\n",
      "\n",
      "=== Adapter 1 ===\n",
      "\n",
      "Sequence: AGATCGGAAGAGC; Type: regular 3'; Length: 13; Trimmed: 2142823 times.\n",
      "\n",
      "No. of allowed errors:\n",
      "0-9 bp: 0; 10-13 bp: 1\n",
      "\n",
      "Bases preceding removed adapters:\n",
      "  A: 30.9%\n",
      "  C: 25.9%\n",
      "  G: 18.7%\n",
      "  T: 20.0%\n",
      "  none/other: 4.5%\n",
      "\n",
      "Overview of removed sequences\n",
      "length\tcount\texpect\tmax.err\terror counts\n",
      "1\t937450\t853669.5\t0\t937450\n",
      "2\t176594\t213417.4\t0\t176594\n",
      "3\t79181\t53354.3\t0\t79181\n",
      "4\t61172\t13338.6\t0\t61172\n",
      "5\t575\t3334.6\t0\t575\n",
      "6\t23757\t833.7\t0\t23757\n",
      "7\t20513\t208.4\t0\t20513\n",
      "8\t24548\t52.1\t0\t24548\n",
      "9\t21333\t13.0\t0\t21171 162\n",
      "10\t22763\t3.3\t1\t20813 1950\n",
      "11\t24508\t0.8\t1\t21879 2629\n",
      "12\t45457\t0.2\t1\t41600 3857\n",
      "13\t1018\t0.1\t1\t561 457\n",
      "14\t44363\t0.1\t1\t40404 3959\n",
      "15\t823\t0.1\t1\t411 412\n",
      "16\t45163\t0.1\t1\t40778 4385\n",
      "17\t1267\t0.1\t1\t786 481\n",
      "18\t28010\t0.1\t1\t25271 2739\n",
      "19\t69106\t0.1\t1\t63733 5373\n",
      "20\t477\t0.1\t1\t296 181\n",
      "21\t152\t0.1\t1\t112 40\n",
      "22\t379\t0.1\t1\t225 154\n",
      "23\t20800\t0.1\t1\t18900 1900\n",
      "24\t50883\t0.1\t1\t46328 4555\n",
      "25\t1164\t0.1\t1\t785 379\n",
      "26\t17657\t0.1\t1\t16052 1605\n",
      "27\t16136\t0.1\t1\t14812 1324\n",
      "28\t390\t0.1\t1\t248 142\n",
      "29\t16537\t0.1\t1\t14907 1630\n",
      "30\t18828\t0.1\t1\t17036 1792\n",
      "31\t25091\t0.1\t1\t23130 1961\n",
      "32\t818\t0.1\t1\t543 275\n",
      "33\t26222\t0.1\t1\t23915 2307\n",
      "34\t1845\t0.1\t1\t1442 403\n",
      "35\t22648\t0.1\t1\t20769 1879\n",
      "36\t11708\t0.1\t1\t10747 961\n",
      "37\t296\t0.1\t1\t188 108\n",
      "38\t10935\t0.1\t1\t9946 989\n",
      "39\t11259\t0.1\t1\t10184 1075\n",
      "40\t9228\t0.1\t1\t8389 839\n",
      "41\t11460\t0.1\t1\t10202 1258\n",
      "42\t22714\t0.1\t1\t20755 1959\n",
      "43\t4407\t0.1\t1\t3986 421\n",
      "44\t4019\t0.1\t1\t3591 428\n",
      "45\t179\t0.1\t1\t119 60\n",
      "46\t7619\t0.1\t1\t6925 694\n",
      "47\t6412\t0.1\t1\t5845 567\n",
      "48\t6077\t0.1\t1\t5463 614\n",
      "49\t6620\t0.1\t1\t5890 730\n",
      "50\t10543\t0.1\t1\t9613 930\n",
      "51\t4405\t0.1\t1\t4030 375\n",
      "52\t259\t0.1\t1\t177 82\n",
      "53\t7095\t0.1\t1\t6434 661\n",
      "54\t10793\t0.1\t1\t9775 1018\n",
      "55\t462\t0.1\t1\t388 74\n",
      "56\t1856\t0.1\t1\t1649 207\n",
      "57\t8200\t0.1\t1\t7510 690\n",
      "58\t148\t0.1\t1\t102 46\n",
      "59\t300\t0.1\t1\t224 76\n",
      "60\t5469\t0.1\t1\t5029 440\n",
      "61\t83\t0.1\t1\t52 31\n",
      "62\t89\t0.1\t1\t64 25\n",
      "63\t1331\t0.1\t1\t1206 125\n",
      "64\t2777\t0.1\t1\t2523 254\n",
      "65\t2619\t0.1\t1\t2410 209\n",
      "66\t2941\t0.1\t1\t2671 270\n",
      "67\t1194\t0.1\t1\t1080 114\n",
      "68\t866\t0.1\t1\t770 96\n",
      "69\t572\t0.1\t1\t506 66\n",
      "70\t746\t0.1\t1\t685 61\n",
      "71\t1065\t0.1\t1\t979 86\n",
      "72\t1080\t0.1\t1\t978 102\n",
      "73\t979\t0.1\t1\t881 98\n",
      "74\t963\t0.1\t1\t897 66\n",
      "75\t847\t0.1\t1\t771 76\n",
      "76\t894\t0.1\t1\t815 79\n",
      "77\t809\t0.1\t1\t743 66\n",
      "78\t755\t0.1\t1\t681 74\n",
      "79\t707\t0.1\t1\t625 82\n",
      "80\t697\t0.1\t1\t620 77\n",
      "81\t662\t0.1\t1\t588 74\n",
      "82\t553\t0.1\t1\t498 55\n",
      "83\t540\t0.1\t1\t479 61\n",
      "84\t554\t0.1\t1\t510 44\n",
      "85\t470\t0.1\t1\t420 50\n",
      "86\t455\t0.1\t1\t416 39\n",
      "87\t372\t0.1\t1\t340 32\n",
      "88\t399\t0.1\t1\t361 38\n",
      "89\t381\t0.1\t1\t348 33\n",
      "90\t360\t0.1\t1\t322 38\n",
      "91\t305\t0.1\t1\t264 41\n",
      "92\t313\t0.1\t1\t259 54\n",
      "93\t292\t0.1\t1\t253 39\n",
      "94\t244\t0.1\t1\t205 39\n",
      "95\t396\t0.1\t1\t371 25\n",
      "96\t233\t0.1\t1\t193 40\n",
      "97\t290\t0.1\t1\t202 88\n",
      "98\t513\t0.1\t1\t412 101\n",
      "99\t343\t0.1\t1\t246 97\n",
      "100\t379\t0.1\t1\t196 183\n",
      "101\t272\t0.1\t1\t215 57\n",
      "102\t338\t0.1\t1\t165 173\n",
      "103\t161\t0.1\t1\t127 34\n",
      "104\t159\t0.1\t1\t140 19\n",
      "105\t131\t0.1\t1\t115 16\n",
      "106\t119\t0.1\t1\t103 16\n",
      "107\t314\t0.1\t1\t286 28\n",
      "108\t113\t0.1\t1\t98 15\n",
      "109\t105\t0.1\t1\t97 8\n",
      "110\t107\t0.1\t1\t97 10\n",
      "111\t108\t0.1\t1\t93 15\n",
      "112\t99\t0.1\t1\t88 11\n",
      "113\t102\t0.1\t1\t93 9\n",
      "114\t77\t0.1\t1\t69 8\n",
      "115\t124\t0.1\t1\t115 9\n",
      "116\t109\t0.1\t1\t97 12\n",
      "117\t108\t0.1\t1\t101 7\n",
      "118\t94\t0.1\t1\t87 7\n",
      "119\t107\t0.1\t1\t89 18\n",
      "120\t94\t0.1\t1\t84 10\n",
      "121\t90\t0.1\t1\t82 8\n",
      "122\t99\t0.1\t1\t89 10\n",
      "123\t96\t0.1\t1\t91 5\n",
      "124\t91\t0.1\t1\t79 12\n",
      "125\t119\t0.1\t1\t112 7\n",
      "126\t106\t0.1\t1\t99 7\n",
      "127\t117\t0.1\t1\t111 6\n",
      "128\t121\t0.1\t1\t115 6\n",
      "129\t78\t0.1\t1\t74 4\n",
      "130\t142\t0.1\t1\t128 14\n",
      "131\t124\t0.1\t1\t116 8\n",
      "132\t102\t0.1\t1\t98 4\n",
      "133\t87\t0.1\t1\t83 4\n",
      "134\t132\t0.1\t1\t123 9\n",
      "135\t103\t0.1\t1\t100 3\n",
      "136\t107\t0.1\t1\t100 7\n",
      "137\t103\t0.1\t1\t94 9\n",
      "138\t87\t0.1\t1\t83 4\n",
      "139\t92\t0.1\t1\t86 6\n",
      "140\t51\t0.1\t1\t46 5\n",
      "141\t59\t0.1\t1\t57 2\n",
      "142\t81\t0.1\t1\t74 7\n",
      "143\t33\t0.1\t1\t29 4\n",
      "144\t36\t0.1\t1\t24 12\n",
      "145\t15\t0.1\t1\t12 3\n",
      "146\t24\t0.1\t1\t21 3\n",
      "147\t31\t0.1\t1\t23 8\n",
      "148\t78\t0.1\t1\t51 27\n",
      "149\t54\t0.1\t1\t18 36\n",
      "150\t414\t0.1\t1\t14 400\n",
      "151\t96951\t0.1\t1\t29 96922\n",
      "\n",
      "\n",
      "RUN STATISTICS FOR INPUT FILE: /home/sam/data/geoduck_illumina/trimmed/NR006_S3_L001_R1_001_val_1.fq.gz\n",
      "=============================================\n",
      "3414678 sequences processed in total\n",
      "The length threshold of paired-end sequences gets evaluated later on (in the validation step)\n",
      "\n",
      "Writing report to '/home/sam/data/geoduck_illumina/trimmed/20180129_trimmed_again/NR006_S3_L001_R2_001_val_2.fq.gz_trimming_report.txt'\n",
      "\n",
      "SUMMARISING RUN PARAMETERS\n",
      "==========================\n",
      "Input filename: /home/sam/data/geoduck_illumina/trimmed/NR006_S3_L001_R2_001_val_2.fq.gz\n",
      "Trimming mode: paired-end\n",
      "Trim Galore version: 0.4.4\n",
      "Cutadapt version: 1.9.1\n",
      "Quality Phred score cutoff: 20\n",
      "Quality encoding type selected: ASCII+33\n",
      "Adapter sequence: 'AGATCGGAAGAGC' (Illumina TruSeq, Sanger iPCR; auto-detected)\n",
      "Maximum trimming error rate: 0.1 (default)\n",
      "Minimum required adapter overlap (stringency): 1 bp\n",
      "Minimum required sequence length for both reads before a sequence pair gets removed: 20 bp\n",
      "Output file(s) will be GZIP compressed\n",
      "\n",
      "Writing final adapter and quality trimmed output to NR006_S3_L001_R2_001_val_2_trimmed.fq.gz\n",
      "\n",
      "\n",
      "  >>> Now performing quality (cutoff 20) and adapter trimming in a single pass for the adapter sequence: 'AGATCGGAAGAGC' from file /home/sam/data/geoduck_illumina/trimmed/NR006_S3_L001_R2_001_val_2.fq.gz <<< \n",
      "This is cutadapt 1.9.1 with Python 2.7.12\n",
      "Command line parameters: -f fastq -e 0.1 -q 20 -O 1 -a AGATCGGAAGAGC /home/sam/data/geoduck_illumina/trimmed/NR006_S3_L001_R2_001_val_2.fq.gz\n",
      "Trimming 1 adapter with at most 10.0% errors in single-end mode ...\n",
      "Finished in 90.38 s (26 us/read; 2.27 M reads/minute).\n",
      "\n",
      "=== Summary ===\n",
      "\n",
      "Total reads processed:               3,414,678\n",
      "Reads with adapters:                 2,052,960 (60.1%)\n",
      "Reads written (passing filters):     3,414,678 (100.0%)\n",
      "\n",
      "Total basepairs processed:   511,788,024 bp\n",
      "Quality-trimmed:                 118,164 bp (0.0%)\n",
      "Total written (filtered):    489,484,820 bp (95.6%)\n",
      "\n",
      "=== Adapter 1 ===\n",
      "\n",
      "Sequence: AGATCGGAAGAGC; Type: regular 3'; Length: 13; Trimmed: 2052960 times.\n",
      "\n",
      "No. of allowed errors:\n",
      "0-9 bp: 0; 10-13 bp: 1\n",
      "\n",
      "Bases preceding removed adapters:\n",
      "  A: 33.3%\n",
      "  C: 24.6%\n",
      "  G: 20.0%\n",
      "  T: 22.0%\n",
      "  none/other: 0.0%\n",
      "\n",
      "Overview of removed sequences\n",
      "length\tcount\texpect\tmax.err\terror counts\n",
      "1\t957570\t853669.5\t0\t957570\n",
      "2\t185662\t213417.4\t0\t185662\n",
      "3\t88214\t53354.3\t0\t88214\n",
      "4\t62857\t13338.6\t0\t62857\n",
      "5\t648\t3334.6\t0\t648\n",
      "6\t22936\t833.7\t0\t22936\n",
      "7\t21575\t208.4\t0\t21575\n",
      "8\t26642\t52.1\t0\t26642\n",
      "9\t18194\t13.0\t0\t18044 150\n",
      "10\t23184\t3.3\t1\t21583 1601\n",
      "11\t22594\t0.8\t1\t20464 2130\n",
      "12\t44904\t0.2\t1\t41486 3418\n",
      "13\t394\t0.1\t1\t140 254\n",
      "14\t26379\t0.1\t1\t23897 2482\n",
      "15\t37633\t0.1\t1\t34681 2952\n",
      "16\t156\t0.1\t1\t67 89\n",
      "17\t24969\t0.1\t1\t23055 1914\n",
      "18\t15222\t0.1\t1\t13957 1265\n",
      "19\t25320\t0.1\t1\t23370 1950\n",
      "20\t18350\t0.1\t1\t16863 1487\n",
      "21\t15725\t0.1\t1\t14432 1293\n",
      "22\t17997\t0.1\t1\t16711 1286\n",
      "23\t18683\t0.1\t1\t17160 1523\n",
      "24\t31064\t0.1\t1\t28646 2418\n",
      "25\t16194\t0.1\t1\t15046 1148\n",
      "26\t15991\t0.1\t1\t14743 1248\n",
      "27\t9854\t0.1\t1\t8749 1105\n",
      "28\t25467\t0.1\t1\t24052 1415\n",
      "29\t5203\t0.1\t1\t4539 664\n",
      "30\t22296\t0.1\t1\t21075 1221\n",
      "31\t7804\t0.1\t1\t7015 789\n",
      "32\t37956\t0.1\t1\t36145 1811\n",
      "33\t13298\t0.1\t1\t12180 1118\n",
      "34\t4051\t0.1\t1\t3571 480\n",
      "35\t18620\t0.1\t1\t17659 961\n",
      "36\t5530\t0.1\t1\t5038 492\n",
      "37\t7956\t0.1\t1\t7339 617\n",
      "38\t4368\t0.1\t1\t4085 283\n",
      "39\t5419\t0.1\t1\t5047 372\n",
      "40\t2384\t0.1\t1\t2228 156\n",
      "41\t8956\t0.1\t1\t8433 523\n",
      "42\t11614\t0.1\t1\t11004 610\n",
      "43\t5102\t0.1\t1\t4746 356\n",
      "44\t7806\t0.1\t1\t7359 447\n",
      "45\t12470\t0.1\t1\t11850 620\n",
      "46\t5755\t0.1\t1\t5352 403\n",
      "47\t4015\t0.1\t1\t3771 244\n",
      "48\t10930\t0.1\t1\t10468 462\n",
      "49\t4516\t0.1\t1\t4246 270\n",
      "50\t1545\t0.1\t1\t1442 103\n",
      "51\t7427\t0.1\t1\t7059 368\n",
      "52\t11021\t0.1\t1\t10573 448\n",
      "53\t4130\t0.1\t1\t3937 193\n",
      "54\t4451\t0.1\t1\t4236 215\n",
      "55\t2750\t0.1\t1\t2622 128\n",
      "56\t1357\t0.1\t1\t1273 84\n",
      "57\t2213\t0.1\t1\t2091 122\n",
      "58\t2275\t0.1\t1\t2152 123\n",
      "59\t1161\t0.1\t1\t1103 58\n",
      "60\t739\t0.1\t1\t682 57\n",
      "61\t628\t0.1\t1\t577 51\n",
      "62\t684\t0.1\t1\t638 46\n",
      "63\t847\t0.1\t1\t799 48\n",
      "64\t1373\t0.1\t1\t1321 52\n",
      "65\t1579\t0.1\t1\t1493 86\n",
      "66\t1616\t0.1\t1\t1543 73\n",
      "67\t1569\t0.1\t1\t1490 79\n",
      "68\t1449\t0.1\t1\t1378 71\n",
      "69\t1380\t0.1\t1\t1323 57\n",
      "70\t1355\t0.1\t1\t1285 70\n",
      "71\t1219\t0.1\t1\t1169 50\n",
      "72\t1120\t0.1\t1\t1065 55\n",
      "73\t989\t0.1\t1\t950 39\n",
      "74\t939\t0.1\t1\t898 41\n",
      "75\t833\t0.1\t1\t798 35\n",
      "76\t880\t0.1\t1\t836 44\n",
      "77\t792\t0.1\t1\t760 32\n",
      "78\t719\t0.1\t1\t680 39\n",
      "79\t660\t0.1\t1\t631 29\n",
      "80\t658\t0.1\t1\t626 32\n",
      "81\t635\t0.1\t1\t608 27\n",
      "82\t550\t0.1\t1\t524 26\n",
      "83\t496\t0.1\t1\t467 29\n",
      "84\t532\t0.1\t1\t505 27\n",
      "85\t454\t0.1\t1\t432 22\n",
      "86\t439\t0.1\t1\t419 20\n",
      "87\t375\t0.1\t1\t359 16\n",
      "88\t384\t0.1\t1\t363 21\n",
      "89\t367\t0.1\t1\t356 11\n",
      "90\t357\t0.1\t1\t347 10\n",
      "91\t323\t0.1\t1\t308 15\n",
      "92\t278\t0.1\t1\t267 11\n",
      "93\t275\t0.1\t1\t263 12\n",
      "94\t227\t0.1\t1\t214 13\n",
      "95\t234\t0.1\t1\t224 10\n",
      "96\t199\t0.1\t1\t183 16\n",
      "97\t214\t0.1\t1\t194 20\n",
      "98\t342\t0.1\t1\t285 57\n",
      "99\t253\t0.1\t1\t223 30\n",
      "100\t225\t0.1\t1\t211 14\n",
      "101\t211\t0.1\t1\t182 29\n",
      "102\t180\t0.1\t1\t168 12\n",
      "103\t137\t0.1\t1\t126 11\n",
      "104\t155\t0.1\t1\t143 12\n",
      "105\t124\t0.1\t1\t122 2\n",
      "106\t113\t0.1\t1\t104 9\n",
      "107\t296\t0.1\t1\t288 8\n",
      "108\t113\t0.1\t1\t107 6\n",
      "109\t111\t0.1\t1\t107 4\n",
      "110\t108\t0.1\t1\t102 6\n",
      "111\t110\t0.1\t1\t104 6\n",
      "112\t105\t0.1\t1\t101 4\n",
      "113\t106\t0.1\t1\t97 9\n",
      "114\t81\t0.1\t1\t76 5\n",
      "115\t114\t0.1\t1\t109 5\n",
      "116\t105\t0.1\t1\t94 11\n",
      "117\t96\t0.1\t1\t87 9\n",
      "118\t97\t0.1\t1\t88 9\n",
      "119\t92\t0.1\t1\t87 5\n",
      "120\t93\t0.1\t1\t89 4\n",
      "121\t83\t0.1\t1\t77 6\n",
      "122\t91\t0.1\t1\t88 3\n",
      "123\t99\t0.1\t1\t91 8\n",
      "124\t107\t0.1\t1\t92 15\n",
      "125\t112\t0.1\t1\t104 8\n",
      "126\t95\t0.1\t1\t89 6\n",
      "127\t115\t0.1\t1\t104 11\n",
      "128\t105\t0.1\t1\t101 4\n",
      "129\t79\t0.1\t1\t76 3\n",
      "130\t138\t0.1\t1\t127 11\n",
      "131\t122\t0.1\t1\t113 9\n",
      "132\t107\t0.1\t1\t97 10\n",
      "133\t87\t0.1\t1\t80 7\n",
      "134\t123\t0.1\t1\t109 14\n",
      "135\t102\t0.1\t1\t93 9\n",
      "136\t106\t0.1\t1\t92 14\n",
      "137\t93\t0.1\t1\t78 15\n",
      "138\t89\t0.1\t1\t76 13\n",
      "139\t92\t0.1\t1\t77 15\n",
      "140\t45\t0.1\t1\t35 10\n",
      "141\t57\t0.1\t1\t43 14\n",
      "142\t78\t0.1\t1\t61 17\n",
      "143\t29\t0.1\t1\t15 14\n",
      "144\t27\t0.1\t1\t13 14\n",
      "145\t13\t0.1\t1\t7 6\n",
      "146\t19\t0.1\t1\t9 10\n",
      "147\t21\t0.1\t1\t9 12\n",
      "148\t50\t0.1\t1\t29 21\n",
      "149\t16\t0.1\t1\t8 8\n",
      "150\t19\t0.1\t1\t10 9\n",
      "151\t150\t0.1\t1\t26 124\n",
      "\n",
      "\n",
      "RUN STATISTICS FOR INPUT FILE: /home/sam/data/geoduck_illumina/trimmed/NR006_S3_L001_R2_001_val_2.fq.gz\n",
      "=============================================\n",
      "3414678 sequences processed in total\n",
      "The length threshold of paired-end sequences gets evaluated later on (in the validation step)\n",
      "\n",
      "Validate paired-end files NR006_S3_L001_R1_001_val_1_trimmed.fq.gz and NR006_S3_L001_R2_001_val_2_trimmed.fq.gz\n",
      "file_1: NR006_S3_L001_R1_001_val_1_trimmed.fq.gz, file_2: NR006_S3_L001_R2_001_val_2_trimmed.fq.gz\n",
      "\n",
      "\n",
      ">>>>> Now validing the length of the 2 paired-end infiles: NR006_S3_L001_R1_001_val_1_trimmed.fq.gz and NR006_S3_L001_R2_001_val_2_trimmed.fq.gz <<<<<\n",
      "Writing validated paired-end read 1 reads to NR006_S3_L001_R1_001_val_1_val_1.fq.gz\n",
      "Writing validated paired-end read 2 reads to NR006_S3_L001_R2_001_val_2_val_2.fq.gz\n",
      "\n",
      "Total number of sequences analysed: 3414678\n",
      "\n",
      "Number of sequence pairs removed because at least one read was shorter than the length cutoff (20 bp): 99013 (2.90%)\n",
      "\n",
      "Deleting both intermediate output files NR006_S3_L001_R1_001_val_1_trimmed.fq.gz and NR006_S3_L001_R2_001_val_2_trimmed.fq.gz\n",
      "\n",
      "====================================================================================================\n",
      "\n",
      "Writing report to '/home/sam/data/geoduck_illumina/trimmed/20180129_trimmed_again/NR006_S3_L002_R1_001_val_1.fq.gz_trimming_report.txt'\n",
      "\n",
      "SUMMARISING RUN PARAMETERS\n",
      "==========================\n",
      "Input filename: /home/sam/data/geoduck_illumina/trimmed/NR006_S3_L002_R1_001_val_1.fq.gz\n",
      "Trimming mode: paired-end\n",
      "Trim Galore version: 0.4.4\n",
      "Cutadapt version: 1.9.1\n",
      "Quality Phred score cutoff: 20\n",
      "Quality encoding type selected: ASCII+33\n",
      "Adapter sequence: 'AGATCGGAAGAGC' (Illumina TruSeq, Sanger iPCR; auto-detected)\n",
      "Maximum trimming error rate: 0.1 (default)\n",
      "Minimum required adapter overlap (stringency): 1 bp\n",
      "Minimum required sequence length for both reads before a sequence pair gets removed: 20 bp\n",
      "Output file(s) will be GZIP compressed\n",
      "\n",
      "Writing final adapter and quality trimmed output to NR006_S3_L002_R1_001_val_1_trimmed.fq.gz\n",
      "\n",
      "\n",
      "  >>> Now performing quality (cutoff 20) and adapter trimming in a single pass for the adapter sequence: 'AGATCGGAAGAGC' from file /home/sam/data/geoduck_illumina/trimmed/NR006_S3_L002_R1_001_val_1.fq.gz <<< \n",
      "This is cutadapt 1.9.1 with Python 2.7.12\n",
      "Command line parameters: -f fastq -e 0.1 -q 20 -O 1 -a AGATCGGAAGAGC /home/sam/data/geoduck_illumina/trimmed/NR006_S3_L002_R1_001_val_1.fq.gz\n",
      "Trimming 1 adapter with at most 10.0% errors in single-end mode ...\n",
      "Finished in 79.63 s (26 us/read; 2.28 M reads/minute).\n",
      "\n",
      "=== Summary ===\n",
      "\n",
      "Total reads processed:               3,031,025\n",
      "Reads with adapters:                 1,929,386 (63.7%)\n",
      "Reads written (passing filters):     3,031,025 (100.0%)\n",
      "\n",
      "Total basepairs processed:   455,473,203 bp\n",
      "Quality-trimmed:                  59,204 bp (0.0%)\n",
      "Total written (filtered):    412,084,346 bp (90.5%)\n",
      "\n",
      "=== Adapter 1 ===\n",
      "\n",
      "Sequence: AGATCGGAAGAGC; Type: regular 3'; Length: 13; Trimmed: 1929386 times.\n",
      "\n",
      "No. of allowed errors:\n",
      "0-9 bp: 0; 10-13 bp: 1\n",
      "\n",
      "Bases preceding removed adapters:\n",
      "  A: 29.7%\n",
      "  C: 25.2%\n",
      "  G: 18.5%\n",
      "  T: 19.2%\n",
      "  none/other: 7.3%\n",
      "\n",
      "Overview of removed sequences\n",
      "length\tcount\texpect\tmax.err\terror counts\n",
      "1\t788408\t757756.2\t0\t788408\n",
      "2\t151285\t189439.1\t0\t151285\n",
      "3\t67064\t47359.8\t0\t67064\n",
      "4\t53544\t11839.9\t0\t53544\n",
      "5\t487\t2960.0\t0\t487\n",
      "6\t20856\t740.0\t0\t20856\n",
      "7\t19031\t185.0\t0\t19031\n",
      "8\t21515\t46.2\t0\t21515\n",
      "9\t19602\t11.6\t0\t19467 135\n",
      "10\t20380\t2.9\t1\t19099 1281\n",
      "11\t21123\t0.7\t1\t19494 1629\n",
      "12\t40181\t0.2\t1\t37607 2574\n",
      "13\t635\t0.0\t1\t259 376\n",
      "14\t39626\t0.0\t1\t36980 2646\n",
      "15\t546\t0.0\t1\t216 330\n",
      "16\t40328\t0.0\t1\t37445 2883\n",
      "17\t781\t0.0\t1\t428 353\n",
      "18\t24020\t0.0\t1\t22173 1847\n",
      "19\t64281\t0.0\t1\t60433 3848\n",
      "20\t390\t0.0\t1\t219 171\n",
      "21\t136\t0.0\t1\t107 29\n",
      "22\t298\t0.0\t1\t212 86\n",
      "23\t18040\t0.0\t1\t16632 1408\n",
      "24\t47127\t0.0\t1\t43895 3232\n",
      "25\t820\t0.0\t1\t521 299\n",
      "26\t15602\t0.0\t1\t14517 1085\n",
      "27\t14558\t0.0\t1\t13600 958\n",
      "28\t317\t0.0\t1\t213 104\n",
      "29\t14674\t0.0\t1\t13589 1085\n",
      "30\t16895\t0.0\t1\t15657 1238\n",
      "31\t23644\t0.0\t1\t22177 1467\n",
      "32\t688\t0.0\t1\t446 242\n",
      "33\t23967\t0.0\t1\t22293 1674\n",
      "34\t1270\t0.0\t1\t965 305\n",
      "35\t21680\t0.0\t1\t20211 1469\n",
      "36\t10887\t0.0\t1\t10143 744\n",
      "37\t243\t0.0\t1\t157 86\n",
      "38\t9846\t0.0\t1\t9114 732\n",
      "39\t10438\t0.0\t1\t9648 790\n",
      "40\t8976\t0.0\t1\t8310 666\n",
      "41\t10247\t0.0\t1\t9296 951\n",
      "42\t22213\t0.0\t1\t20655 1558\n",
      "43\t4212\t0.0\t1\t3853 359\n",
      "44\t3399\t0.0\t1\t3115 284\n",
      "45\t122\t0.0\t1\t89 33\n",
      "46\t6912\t0.0\t1\t6349 563\n",
      "47\t6086\t0.0\t1\t5620 466\n",
      "48\t6045\t0.0\t1\t5510 535\n",
      "49\t6310\t0.0\t1\t5804 506\n",
      "50\t10220\t0.0\t1\t9462 758\n",
      "51\t4524\t0.0\t1\t4143 381\n",
      "52\t191\t0.0\t1\t121 70\n",
      "53\t5734\t0.0\t1\t5264 470\n",
      "54\t11519\t0.0\t1\t10583 936\n",
      "55\t261\t0.0\t1\t210 51\n",
      "56\t1303\t0.0\t1\t1144 159\n",
      "57\t9194\t0.0\t1\t8511 683\n",
      "58\t118\t0.0\t1\t84 34\n",
      "59\t170\t0.0\t1\t104 66\n",
      "60\t5353\t0.0\t1\t4961 392\n",
      "61\t68\t0.0\t1\t39 29\n",
      "62\t55\t0.0\t1\t36 19\n",
      "63\t1544\t0.0\t1\t1403 141\n",
      "64\t2235\t0.0\t1\t2041 194\n",
      "65\t2238\t0.0\t1\t2057 181\n",
      "66\t2678\t0.0\t1\t2481 197\n",
      "67\t1852\t0.0\t1\t1687 165\n",
      "68\t2019\t0.0\t1\t1822 197\n",
      "69\t391\t0.0\t1\t347 44\n",
      "70\t605\t0.0\t1\t551 54\n",
      "71\t1093\t0.0\t1\t1007 86\n",
      "72\t1059\t0.0\t1\t980 79\n",
      "73\t1081\t0.0\t1\t983 98\n",
      "74\t1093\t0.0\t1\t991 102\n",
      "75\t962\t0.0\t1\t884 78\n",
      "76\t906\t0.0\t1\t823 83\n",
      "77\t876\t0.0\t1\t798 78\n",
      "78\t823\t0.0\t1\t723 100\n",
      "79\t822\t0.0\t1\t745 77\n",
      "80\t790\t0.0\t1\t713 77\n",
      "81\t698\t0.0\t1\t629 69\n",
      "82\t635\t0.0\t1\t575 60\n",
      "83\t596\t0.0\t1\t528 68\n",
      "84\t583\t0.0\t1\t530 53\n",
      "85\t491\t0.0\t1\t442 49\n",
      "86\t475\t0.0\t1\t424 51\n",
      "87\t442\t0.0\t1\t395 47\n",
      "88\t480\t0.0\t1\t412 68\n",
      "89\t416\t0.0\t1\t373 43\n",
      "90\t372\t0.0\t1\t332 40\n",
      "91\t335\t0.0\t1\t297 38\n",
      "92\t347\t0.0\t1\t300 47\n",
      "93\t309\t0.0\t1\t274 35\n",
      "94\t312\t0.0\t1\t265 47\n",
      "95\t472\t0.0\t1\t443 29\n",
      "96\t279\t0.0\t1\t253 26\n",
      "97\t332\t0.0\t1\t233 99\n",
      "98\t520\t0.0\t1\t407 113\n",
      "99\t445\t0.0\t1\t326 119\n",
      "100\t435\t0.0\t1\t249 186\n",
      "101\t350\t0.0\t1\t280 70\n",
      "102\t357\t0.0\t1\t182 175\n",
      "103\t219\t0.0\t1\t171 48\n",
      "104\t248\t0.0\t1\t222 26\n",
      "105\t153\t0.0\t1\t129 24\n",
      "106\t144\t0.0\t1\t127 17\n",
      "107\t348\t0.0\t1\t326 22\n",
      "108\t186\t0.0\t1\t156 30\n",
      "109\t175\t0.0\t1\t157 18\n",
      "110\t163\t0.0\t1\t148 15\n",
      "111\t147\t0.0\t1\t126 21\n",
      "112\t146\t0.0\t1\t137 9\n",
      "113\t124\t0.0\t1\t115 9\n",
      "114\t124\t0.0\t1\t112 12\n",
      "115\t165\t0.0\t1\t152 13\n",
      "116\t161\t0.0\t1\t142 19\n",
      "117\t138\t0.0\t1\t128 10\n",
      "118\t144\t0.0\t1\t132 12\n",
      "119\t139\t0.0\t1\t127 12\n",
      "120\t132\t0.0\t1\t114 18\n",
      "121\t122\t0.0\t1\t108 14\n",
      "122\t147\t0.0\t1\t137 10\n",
      "123\t169\t0.0\t1\t152 17\n",
      "124\t133\t0.0\t1\t126 7\n",
      "125\t170\t0.0\t1\t158 12\n",
      "126\t123\t0.0\t1\t109 14\n",
      "127\t176\t0.0\t1\t169 7\n",
      "128\t143\t0.0\t1\t135 8\n",
      "129\t173\t0.0\t1\t168 5\n",
      "130\t186\t0.0\t1\t169 17\n",
      "131\t142\t0.0\t1\t134 8\n",
      "132\t129\t0.0\t1\t123 6\n",
      "133\t152\t0.0\t1\t145 7\n",
      "134\t164\t0.0\t1\t158 6\n",
      "135\t129\t0.0\t1\t117 12\n",
      "136\t152\t0.0\t1\t140 12\n",
      "137\t133\t0.0\t1\t126 7\n",
      "138\t125\t0.0\t1\t119 6\n",
      "139\t129\t0.0\t1\t124 5\n",
      "140\t75\t0.0\t1\t66 9\n",
      "141\t90\t0.0\t1\t81 9\n",
      "142\t137\t0.0\t1\t125 12\n",
      "143\t60\t0.0\t1\t56 4\n",
      "144\t50\t0.0\t1\t42 8\n",
      "145\t25\t0.0\t1\t23 2\n",
      "146\t51\t0.0\t1\t45 6\n",
      "147\t51\t0.0\t1\t37 14\n",
      "148\t107\t0.0\t1\t79 28\n",
      "149\t79\t0.0\t1\t36 43\n",
      "150\t395\t0.0\t1\t14 381\n",
      "151\t141220\t0.0\t1\t56 141164\n",
      "\n",
      "\n",
      "RUN STATISTICS FOR INPUT FILE: /home/sam/data/geoduck_illumina/trimmed/NR006_S3_L002_R1_001_val_1.fq.gz\n",
      "=============================================\n",
      "3031025 sequences processed in total\n",
      "The length threshold of paired-end sequences gets evaluated later on (in the validation step)\n",
      "\n",
      "Writing report to '/home/sam/data/geoduck_illumina/trimmed/20180129_trimmed_again/NR006_S3_L002_R2_001_val_2.fq.gz_trimming_report.txt'\n",
      "\n",
      "SUMMARISING RUN PARAMETERS\n",
      "==========================\n",
      "Input filename: /home/sam/data/geoduck_illumina/trimmed/NR006_S3_L002_R2_001_val_2.fq.gz\n",
      "Trimming mode: paired-end\n",
      "Trim Galore version: 0.4.4\n",
      "Cutadapt version: 1.9.1\n",
      "Quality Phred score cutoff: 20\n",
      "Quality encoding type selected: ASCII+33\n",
      "Adapter sequence: 'AGATCGGAAGAGC' (Illumina TruSeq, Sanger iPCR; auto-detected)\n",
      "Maximum trimming error rate: 0.1 (default)\n",
      "Minimum required adapter overlap (stringency): 1 bp\n",
      "Minimum required sequence length for both reads before a sequence pair gets removed: 20 bp\n",
      "Output file(s) will be GZIP compressed\n",
      "\n",
      "Writing final adapter and quality trimmed output to NR006_S3_L002_R2_001_val_2_trimmed.fq.gz\n",
      "\n",
      "\n",
      "  >>> Now performing quality (cutoff 20) and adapter trimming in a single pass for the adapter sequence: 'AGATCGGAAGAGC' from file /home/sam/data/geoduck_illumina/trimmed/NR006_S3_L002_R2_001_val_2.fq.gz <<< \n",
      "This is cutadapt 1.9.1 with Python 2.7.12\n",
      "Command line parameters: -f fastq -e 0.1 -q 20 -O 1 -a AGATCGGAAGAGC /home/sam/data/geoduck_illumina/trimmed/NR006_S3_L002_R2_001_val_2.fq.gz\n",
      "Trimming 1 adapter with at most 10.0% errors in single-end mode ...\n",
      "Finished in 80.21 s (26 us/read; 2.27 M reads/minute).\n",
      "\n",
      "=== Summary ===\n",
      "\n",
      "Total reads processed:               3,031,025\n",
      "Reads with adapters:                 1,797,618 (59.3%)\n",
      "Reads written (passing filters):     3,031,025 (100.0%)\n",
      "\n",
      "Total basepairs processed:   454,130,095 bp\n",
      "Quality-trimmed:                  81,905 bp (0.0%)\n",
      "Total written (filtered):    432,938,122 bp (95.3%)\n",
      "\n",
      "=== Adapter 1 ===\n",
      "\n",
      "Sequence: AGATCGGAAGAGC; Type: regular 3'; Length: 13; Trimmed: 1797618 times.\n",
      "\n",
      "No. of allowed errors:\n",
      "0-9 bp: 0; 10-13 bp: 1\n",
      "\n",
      "Bases preceding removed adapters:\n",
      "  A: 32.9%\n",
      "  C: 25.0%\n",
      "  G: 20.4%\n",
      "  T: 21.7%\n",
      "  none/other: 0.0%\n",
      "\n",
      "Overview of removed sequences\n",
      "length\tcount\texpect\tmax.err\terror counts\n",
      "1\t803750\t757756.2\t0\t803750\n",
      "2\t159777\t189439.1\t0\t159777\n",
      "3\t74983\t47359.8\t0\t74983\n",
      "4\t55464\t11839.9\t0\t55464\n",
      "5\t534\t2960.0\t0\t534\n",
      "6\t20415\t740.0\t0\t20415\n",
      "7\t19928\t185.0\t0\t19928\n",
      "8\t23459\t46.2\t0\t23459\n",
      "9\t17406\t11.6\t0\t17255 151\n",
      "10\t20562\t2.9\t1\t19800 762\n",
      "11\t19822\t0.7\t1\t18851 971\n",
      "12\t39785\t0.2\t1\t38357 1428\n",
      "13\t251\t0.0\t1\t63 188\n",
      "14\t22463\t0.0\t1\t21347 1116\n",
      "15\t35189\t0.0\t1\t33798 1391\n",
      "16\t162\t0.0\t1\t69 93\n",
      "17\t22058\t0.0\t1\t21192 866\n",
      "18\t14401\t0.0\t1\t13797 604\n",
      "19\t21417\t0.0\t1\t20543 874\n",
      "20\t16595\t0.0\t1\t15928 667\n",
      "21\t15273\t0.0\t1\t14570 703\n",
      "22\t16562\t0.0\t1\t15897 665\n",
      "23\t16572\t0.0\t1\t15858 714\n",
      "24\t23886\t0.0\t1\t22883 1003\n",
      "25\t15834\t0.0\t1\t15311 523\n",
      "26\t15356\t0.0\t1\t14732 624\n",
      "27\t9520\t0.0\t1\t8970 550\n",
      "28\t23049\t0.0\t1\t22321 728\n",
      "29\t4963\t0.0\t1\t4603 360\n",
      "30\t21010\t0.0\t1\t20327 683\n",
      "31\t6944\t0.0\t1\t6487 457\n",
      "32\t40614\t0.0\t1\t39579 1035\n",
      "33\t11790\t0.0\t1\t11296 494\n",
      "34\t3553\t0.0\t1\t3261 292\n",
      "35\t19784\t0.0\t1\t19297 487\n",
      "36\t3727\t0.0\t1\t3499 228\n",
      "37\t7832\t0.0\t1\t7453 379\n",
      "38\t2360\t0.0\t1\t2247 113\n",
      "39\t4177\t0.0\t1\t4003 174\n",
      "40\t1155\t0.0\t1\t1084 71\n",
      "41\t8529\t0.0\t1\t8285 244\n",
      "42\t11564\t0.0\t1\t11239 325\n",
      "43\t4405\t0.0\t1\t4220 185\n",
      "44\t7519\t0.0\t1\t7261 258\n",
      "45\t14908\t0.0\t1\t14491 417\n",
      "46\t5316\t0.0\t1\t5076 240\n",
      "47\t3010\t0.0\t1\t2866 144\n",
      "48\t12584\t0.0\t1\t12240 344\n",
      "49\t3165\t0.0\t1\t3063 102\n",
      "50\t895\t0.0\t1\t841 54\n",
      "51\t7922\t0.0\t1\t7682 240\n",
      "52\t13903\t0.0\t1\t13616 287\n",
      "53\t3428\t0.0\t1\t3297 131\n",
      "54\t4044\t0.0\t1\t3901 143\n",
      "55\t1734\t0.0\t1\t1680 54\n",
      "56\t889\t0.0\t1\t818 71\n",
      "57\t1690\t0.0\t1\t1620 70\n",
      "58\t1719\t0.0\t1\t1649 70\n",
      "59\t761\t0.0\t1\t721 40\n",
      "60\t547\t0.0\t1\t521 26\n",
      "61\t484\t0.0\t1\t454 30\n",
      "62\t497\t0.0\t1\t468 29\n",
      "63\t572\t0.0\t1\t547 25\n",
      "64\t1256\t0.0\t1\t1215 41\n",
      "65\t1584\t0.0\t1\t1533 51\n",
      "66\t1636\t0.0\t1\t1586 50\n",
      "67\t1562\t0.0\t1\t1515 47\n",
      "68\t1413\t0.0\t1\t1368 45\n",
      "69\t1431\t0.0\t1\t1386 45\n",
      "70\t1373\t0.0\t1\t1324 49\n",
      "71\t1279\t0.0\t1\t1241 38\n",
      "72\t1137\t0.0\t1\t1092 45\n",
      "73\t1089\t0.0\t1\t1040 49\n",
      "74\t1049\t0.0\t1\t1008 41\n",
      "75\t952\t0.0\t1\t930 22\n",
      "76\t884\t0.0\t1\t844 40\n",
      "77\t850\t0.0\t1\t825 25\n",
      "78\t785\t0.0\t1\t757 28\n",
      "79\t789\t0.0\t1\t761 28\n",
      "80\t759\t0.0\t1\t730 29\n",
      "81\t660\t0.0\t1\t633 27\n",
      "82\t621\t0.0\t1\t595 26\n",
      "83\t561\t0.0\t1\t528 33\n",
      "84\t581\t0.0\t1\t551 30\n",
      "85\t472\t0.0\t1\t452 20\n",
      "86\t464\t0.0\t1\t446 18\n",
      "87\t455\t0.0\t1\t439 16\n",
      "88\t459\t0.0\t1\t439 20\n",
      "89\t409\t0.0\t1\t390 19\n",
      "90\t391\t0.0\t1\t367 24\n",
      "91\t381\t0.0\t1\t362 19\n",
      "92\t325\t0.0\t1\t308 17\n",
      "93\t325\t0.0\t1\t309 16\n",
      "94\t299\t0.0\t1\t289 10\n",
      "95\t301\t0.0\t1\t289 12\n",
      "96\t283\t0.0\t1\t261 22\n",
      "97\t265\t0.0\t1\t249 16\n",
      "98\t369\t0.0\t1\t308 61\n",
      "99\t341\t0.0\t1\t305 36\n",
      "100\t291\t0.0\t1\t278 13\n",
      "101\t288\t0.0\t1\t257 31\n",
      "102\t207\t0.0\t1\t199 8\n",
      "103\t196\t0.0\t1\t185 11\n",
      "104\t233\t0.0\t1\t216 17\n",
      "105\t164\t0.0\t1\t154 10\n",
      "106\t146\t0.0\t1\t136 10\n",
      "107\t328\t0.0\t1\t316 12\n",
      "108\t188\t0.0\t1\t176 12\n",
      "109\t171\t0.0\t1\t162 9\n",
      "110\t162\t0.0\t1\t153 9\n",
      "111\t151\t0.0\t1\t143 8\n",
      "112\t147\t0.0\t1\t138 9\n",
      "113\t141\t0.0\t1\t132 9\n",
      "114\t129\t0.0\t1\t120 9\n",
      "115\t154\t0.0\t1\t147 7\n",
      "116\t156\t0.0\t1\t154 2\n",
      "117\t138\t0.0\t1\t130 8\n",
      "118\t144\t0.0\t1\t134 10\n",
      "119\t137\t0.0\t1\t130 7\n",
      "120\t126\t0.0\t1\t121 5\n",
      "121\t120\t0.0\t1\t115 5\n",
      "122\t148\t0.0\t1\t138 10\n",
      "123\t164\t0.0\t1\t153 11\n",
      "124\t154\t0.0\t1\t139 15\n",
      "125\t162\t0.0\t1\t158 4\n",
      "126\t113\t0.0\t1\t106 7\n",
      "127\t176\t0.0\t1\t166 10\n",
      "128\t141\t0.0\t1\t134 7\n",
      "129\t174\t0.0\t1\t169 5\n",
      "130\t175\t0.0\t1\t158 17\n",
      "131\t139\t0.0\t1\t132 7\n",
      "132\t127\t0.0\t1\t122 5\n",
      "133\t150\t0.0\t1\t142 8\n",
      "134\t159\t0.0\t1\t148 11\n",
      "135\t120\t0.0\t1\t114 6\n",
      "136\t146\t0.0\t1\t131 15\n",
      "137\t127\t0.0\t1\t116 11\n",
      "138\t123\t0.0\t1\t101 22\n",
      "139\t125\t0.0\t1\t112 13\n",
      "140\t69\t0.0\t1\t59 10\n",
      "141\t86\t0.0\t1\t73 13\n",
      "142\t125\t0.0\t1\t108 17\n",
      "143\t56\t0.0\t1\t42 14\n",
      "144\t43\t0.0\t1\t29 14\n",
      "145\t25\t0.0\t1\t13 12\n",
      "146\t39\t0.0\t1\t30 9\n",
      "147\t31\t0.0\t1\t14 17\n",
      "148\t81\t0.0\t1\t38 43\n",
      "149\t37\t0.0\t1\t21 16\n",
      "150\t18\t0.0\t1\t7 11\n",
      "151\t211\t0.0\t1\t46 165\n",
      "\n",
      "\n",
      "RUN STATISTICS FOR INPUT FILE: /home/sam/data/geoduck_illumina/trimmed/NR006_S3_L002_R2_001_val_2.fq.gz\n",
      "=============================================\n",
      "3031025 sequences processed in total\n",
      "The length threshold of paired-end sequences gets evaluated later on (in the validation step)\n",
      "\n",
      "Validate paired-end files NR006_S3_L002_R1_001_val_1_trimmed.fq.gz and NR006_S3_L002_R2_001_val_2_trimmed.fq.gz\n",
      "file_1: NR006_S3_L002_R1_001_val_1_trimmed.fq.gz, file_2: NR006_S3_L002_R2_001_val_2_trimmed.fq.gz\n",
      "\n",
      "\n",
      ">>>>> Now validing the length of the 2 paired-end infiles: NR006_S3_L002_R1_001_val_1_trimmed.fq.gz and NR006_S3_L002_R2_001_val_2_trimmed.fq.gz <<<<<\n",
      "Writing validated paired-end read 1 reads to NR006_S3_L002_R1_001_val_1_val_1.fq.gz\n",
      "Writing validated paired-end read 2 reads to NR006_S3_L002_R2_001_val_2_val_2.fq.gz\n",
      "\n",
      "Total number of sequences analysed: 3031025\n",
      "\n",
      "Number of sequence pairs removed because at least one read was shorter than the length cutoff (20 bp): 143798 (4.74%)\n",
      "\n",
      "Deleting both intermediate output files NR006_S3_L002_R1_001_val_1_trimmed.fq.gz and NR006_S3_L002_R2_001_val_2_trimmed.fq.gz\n",
      "\n",
      "====================================================================================================\n",
      "\n",
      "Writing report to '/home/sam/data/geoduck_illumina/trimmed/20180129_trimmed_again/NR012_S1_L001_R1_001_val_1.fq.gz_trimming_report.txt'\n",
      "\n",
      "SUMMARISING RUN PARAMETERS\n",
      "==========================\n",
      "Input filename: /home/sam/data/geoduck_illumina/trimmed/NR012_S1_L001_R1_001_val_1.fq.gz\n",
      "Trimming mode: paired-end\n",
      "Trim Galore version: 0.4.4\n",
      "Cutadapt version: 1.9.1\n",
      "Quality Phred score cutoff: 20\n",
      "Quality encoding type selected: ASCII+33\n",
      "Adapter sequence: 'AGATCGGAAGAGC' (Illumina TruSeq, Sanger iPCR; auto-detected)\n",
      "Maximum trimming error rate: 0.1 (default)\n",
      "Minimum required adapter overlap (stringency): 1 bp\n",
      "Minimum required sequence length for both reads before a sequence pair gets removed: 20 bp\n",
      "Output file(s) will be GZIP compressed\n",
      "\n",
      "Writing final adapter and quality trimmed output to NR012_S1_L001_R1_001_val_1_trimmed.fq.gz\n",
      "\n",
      "\n",
      "  >>> Now performing quality (cutoff 20) and adapter trimming in a single pass for the adapter sequence: 'AGATCGGAAGAGC' from file /home/sam/data/geoduck_illumina/trimmed/NR012_S1_L001_R1_001_val_1.fq.gz <<< \n",
      "This is cutadapt 1.9.1 with Python 2.7.12\n",
      "Command line parameters: -f fastq -e 0.1 -q 20 -O 1 -a AGATCGGAAGAGC /home/sam/data/geoduck_illumina/trimmed/NR012_S1_L001_R1_001_val_1.fq.gz\n",
      "Trimming 1 adapter with at most 10.0% errors in single-end mode ...\n",
      "Finished in 7.24 s (26 us/read; 2.33 M reads/minute).\n",
      "\n",
      "=== Summary ===\n",
      "\n",
      "Total reads processed:                 281,472\n",
      "Reads with adapters:                   121,688 (43.2%)\n",
      "Reads written (passing filters):       281,472 (100.0%)\n",
      "\n",
      "Total basepairs processed:    35,695,401 bp\n",
      "Quality-trimmed:                  15,812 bp (0.0%)\n",
      "Total written (filtered):     34,569,512 bp (96.8%)\n",
      "\n",
      "=== Adapter 1 ===\n",
      "\n",
      "Sequence: AGATCGGAAGAGC; Type: regular 3'; Length: 13; Trimmed: 121688 times.\n",
      "\n",
      "No. of allowed errors:\n",
      "0-9 bp: 0; 10-13 bp: 1\n",
      "\n",
      "Bases preceding removed adapters:\n",
      "  A: 31.3%\n",
      "  C: 23.7%\n",
      "  G: 15.8%\n",
      "  T: 26.8%\n",
      "  none/other: 2.5%\n",
      "\n",
      "Overview of removed sequences\n",
      "length\tcount\texpect\tmax.err\terror counts\n",
      "1\t75815\t70368.0\t0\t75815\n",
      "2\t19000\t17592.0\t0\t19000\n",
      "3\t5354\t4398.0\t0\t5354\n",
      "4\t2554\t1099.5\t0\t2554\n",
      "5\t243\t274.9\t0\t243\n",
      "6\t426\t68.7\t0\t426\n",
      "7\t331\t17.2\t0\t331\n",
      "8\t402\t4.3\t0\t402\n",
      "9\t348\t1.1\t0\t346 2\n",
      "10\t401\t0.3\t1\t365 36\n",
      "11\t394\t0.1\t1\t352 42\n",
      "12\t746\t0.0\t1\t695 51\n",
      "13\t21\t0.0\t1\t11 10\n",
      "14\t750\t0.0\t1\t686 64\n",
      "15\t14\t0.0\t1\t8 6\n",
      "16\t792\t0.0\t1\t723 69\n",
      "17\t29\t0.0\t1\t20 9\n",
      "18\t456\t0.0\t1\t419 37\n",
      "19\t1132\t0.0\t1\t1070 62\n",
      "20\t9\t0.0\t1\t5 4\n",
      "21\t3\t0.0\t1\t1 2\n",
      "22\t9\t0.0\t1\t6 3\n",
      "23\t386\t0.0\t1\t351 35\n",
      "24\t901\t0.0\t1\t828 73\n",
      "25\t15\t0.0\t1\t10 5\n",
      "26\t308\t0.0\t1\t288 20\n",
      "27\t287\t0.0\t1\t258 29\n",
      "28\t7\t0.0\t1\t5 2\n",
      "29\t301\t0.0\t1\t264 37\n",
      "30\t340\t0.0\t1\t310 30\n",
      "31\t454\t0.0\t1\t425 29\n",
      "32\t19\t0.0\t1\t16 3\n",
      "33\t507\t0.0\t1\t457 50\n",
      "34\t456\t0.0\t1\t431 25\n",
      "35\t18\t0.0\t1\t16 2\n",
      "36\t60\t0.0\t1\t51 9\n",
      "37\t216\t0.0\t1\t198 18\n",
      "38\t218\t0.0\t1\t197 21\n",
      "39\t255\t0.0\t1\t242 13\n",
      "40\t185\t0.0\t1\t168 17\n",
      "41\t200\t0.0\t1\t172 28\n",
      "42\t414\t0.0\t1\t387 27\n",
      "43\t106\t0.0\t1\t94 12\n",
      "44\t98\t0.0\t1\t87 11\n",
      "45\t96\t0.0\t1\t78 18\n",
      "46\t237\t0.0\t1\t220 17\n",
      "47\t156\t0.0\t1\t143 13\n",
      "48\t113\t0.0\t1\t103 10\n",
      "49\t342\t0.0\t1\t323 19\n",
      "50\t243\t0.0\t1\t220 23\n",
      "51\t78\t0.0\t1\t74 4\n",
      "52\t71\t0.0\t1\t66 5\n",
      "53\t244\t0.0\t1\t229 15\n",
      "54\t161\t0.0\t1\t153 8\n",
      "55\t30\t0.0\t1\t26 4\n",
      "56\t46\t0.0\t1\t41 5\n",
      "57\t182\t0.0\t1\t167 15\n",
      "58\t9\t0.0\t1\t8 1\n",
      "59\t34\t0.0\t1\t24 10\n",
      "60\t87\t0.0\t1\t84 3\n",
      "61\t3\t0.0\t1\t3\n",
      "62\t16\t0.0\t1\t10 6\n",
      "63\t56\t0.0\t1\t48 8\n",
      "64\t85\t0.0\t1\t79 6\n",
      "65\t52\t0.0\t1\t40 12\n",
      "66\t55\t0.0\t1\t54 1\n",
      "67\t35\t0.0\t1\t32 3\n",
      "68\t20\t0.0\t1\t18 2\n",
      "69\t16\t0.0\t1\t8 8\n",
      "70\t40\t0.0\t1\t32 8\n",
      "71\t30\t0.0\t1\t27 3\n",
      "72\t32\t0.0\t1\t30 2\n",
      "73\t24\t0.0\t1\t20 4\n",
      "74\t27\t0.0\t1\t27\n",
      "75\t38\t0.0\t1\t34 4\n",
      "76\t39\t0.0\t1\t34 5\n",
      "77\t41\t0.0\t1\t39 2\n",
      "78\t39\t0.0\t1\t35 4\n",
      "79\t36\t0.0\t1\t34 2\n",
      "80\t30\t0.0\t1\t27 3\n",
      "81\t32\t0.0\t1\t30 2\n",
      "82\t25\t0.0\t1\t24 1\n",
      "83\t29\t0.0\t1\t24 5\n",
      "84\t30\t0.0\t1\t26 4\n",
      "85\t27\t0.0\t1\t27\n",
      "86\t22\t0.0\t1\t19 3\n",
      "87\t23\t0.0\t1\t18 5\n",
      "88\t19\t0.0\t1\t14 5\n",
      "89\t33\t0.0\t1\t31 2\n",
      "90\t27\t0.0\t1\t24 3\n",
      "91\t22\t0.0\t1\t18 4\n",
      "92\t19\t0.0\t1\t16 3\n",
      "93\t20\t0.0\t1\t16 4\n",
      "94\t26\t0.0\t1\t25 1\n",
      "95\t18\t0.0\t1\t16 2\n",
      "96\t13\t0.0\t1\t11 2\n",
      "97\t14\t0.0\t1\t14\n",
      "98\t16\t0.0\t1\t13 3\n",
      "99\t23\t0.0\t1\t17 6\n",
      "100\t9\t0.0\t1\t9\n",
      "101\t14\t0.0\t1\t13 1\n",
      "102\t22\t0.0\t1\t19 3\n",
      "103\t25\t0.0\t1\t22 3\n",
      "104\t17\t0.0\t1\t16 1\n",
      "105\t12\t0.0\t1\t12\n",
      "106\t12\t0.0\t1\t11 1\n",
      "107\t8\t0.0\t1\t7 1\n",
      "108\t7\t0.0\t1\t6 1\n",
      "109\t11\t0.0\t1\t9 2\n",
      "110\t6\t0.0\t1\t3 3\n",
      "111\t16\t0.0\t1\t14 2\n",
      "112\t10\t0.0\t1\t9 1\n",
      "113\t9\t0.0\t1\t8 1\n",
      "114\t6\t0.0\t1\t5 1\n",
      "115\t10\t0.0\t1\t9 1\n",
      "116\t8\t0.0\t1\t8\n",
      "117\t4\t0.0\t1\t3 1\n",
      "118\t4\t0.0\t1\t3 1\n",
      "119\t7\t0.0\t1\t7\n",
      "120\t6\t0.0\t1\t5 1\n",
      "121\t11\t0.0\t1\t8 3\n",
      "122\t3\t0.0\t1\t3\n",
      "123\t7\t0.0\t1\t5 2\n",
      "124\t8\t0.0\t1\t6 2\n",
      "125\t7\t0.0\t1\t7\n",
      "126\t11\t0.0\t1\t9 2\n",
      "127\t6\t0.0\t1\t5 1\n",
      "128\t8\t0.0\t1\t6 2\n",
      "129\t4\t0.0\t1\t4\n",
      "130\t9\t0.0\t1\t9\n",
      "131\t8\t0.0\t1\t8\n",
      "132\t11\t0.0\t1\t11\n",
      "133\t6\t0.0\t1\t6\n",
      "134\t6\t0.0\t1\t6\n",
      "135\t7\t0.0\t1\t7\n",
      "136\t5\t0.0\t1\t5\n",
      "137\t10\t0.0\t1\t10\n",
      "138\t5\t0.0\t1\t5\n",
      "139\t7\t0.0\t1\t7\n",
      "140\t2\t0.0\t1\t2\n",
      "141\t5\t0.0\t1\t5\n",
      "142\t4\t0.0\t1\t4\n",
      "143\t3\t0.0\t1\t3\n",
      "144\t4\t0.0\t1\t3 1\n",
      "145\t5\t0.0\t1\t4 1\n",
      "146\t3\t0.0\t1\t3\n",
      "147\t4\t0.0\t1\t3 1\n",
      "148\t9\t0.0\t1\t8 1\n",
      "149\t3\t0.0\t1\t0 3\n",
      "150\t18\t0.0\t1\t0 18\n",
      "151\t3005\t0.0\t1\t83 2922\n",
      "\n",
      "\n",
      "RUN STATISTICS FOR INPUT FILE: /home/sam/data/geoduck_illumina/trimmed/NR012_S1_L001_R1_001_val_1.fq.gz\n",
      "=============================================\n",
      "281472 sequences processed in total\n",
      "The length threshold of paired-end sequences gets evaluated later on (in the validation step)\n",
      "\n",
      "Writing report to '/home/sam/data/geoduck_illumina/trimmed/20180129_trimmed_again/NR012_S1_L001_R2_001_val_2.fq.gz_trimming_report.txt'\n",
      "\n",
      "SUMMARISING RUN PARAMETERS\n",
      "==========================\n",
      "Input filename: /home/sam/data/geoduck_illumina/trimmed/NR012_S1_L001_R2_001_val_2.fq.gz\n",
      "Trimming mode: paired-end\n",
      "Trim Galore version: 0.4.4\n",
      "Cutadapt version: 1.9.1\n",
      "Quality Phred score cutoff: 20\n",
      "Quality encoding type selected: ASCII+33\n",
      "Adapter sequence: 'AGATCGGAAGAGC' (Illumina TruSeq, Sanger iPCR; auto-detected)\n",
      "Maximum trimming error rate: 0.1 (default)\n",
      "Minimum required adapter overlap (stringency): 1 bp\n",
      "Minimum required sequence length for both reads before a sequence pair gets removed: 20 bp\n",
      "Output file(s) will be GZIP compressed\n",
      "\n",
      "Writing final adapter and quality trimmed output to NR012_S1_L001_R2_001_val_2_trimmed.fq.gz\n",
      "\n",
      "\n",
      "  >>> Now performing quality (cutoff 20) and adapter trimming in a single pass for the adapter sequence: 'AGATCGGAAGAGC' from file /home/sam/data/geoduck_illumina/trimmed/NR012_S1_L001_R2_001_val_2.fq.gz <<< \n",
      "This is cutadapt 1.9.1 with Python 2.7.12\n",
      "Command line parameters: -f fastq -e 0.1 -q 20 -O 1 -a AGATCGGAAGAGC /home/sam/data/geoduck_illumina/trimmed/NR012_S1_L001_R2_001_val_2.fq.gz\n",
      "Trimming 1 adapter with at most 10.0% errors in single-end mode ...\n",
      "Finished in 7.21 s (26 us/read; 2.34 M reads/minute).\n",
      "\n",
      "=== Summary ===\n",
      "\n",
      "Total reads processed:                 281,472\n",
      "Reads with adapters:                   120,782 (42.9%)\n",
      "Reads written (passing filters):       281,472 (100.0%)\n",
      "\n",
      "Total basepairs processed:    36,447,504 bp\n",
      "Quality-trimmed:                  19,787 bp (0.1%)\n",
      "Total written (filtered):     35,723,779 bp (98.0%)\n",
      "\n",
      "=== Adapter 1 ===\n",
      "\n",
      "Sequence: AGATCGGAAGAGC; Type: regular 3'; Length: 13; Trimmed: 120782 times.\n",
      "\n",
      "No. of allowed errors:\n",
      "0-9 bp: 0; 10-13 bp: 1\n",
      "\n",
      "Bases preceding removed adapters:\n",
      "  A: 32.5%\n",
      "  C: 23.1%\n",
      "  G: 16.7%\n",
      "  T: 27.7%\n",
      "  none/other: 0.1%\n",
      "\n",
      "Overview of removed sequences\n",
      "length\tcount\texpect\tmax.err\terror counts\n",
      "1\t76746\t70368.0\t0\t76746\n",
      "2\t19123\t17592.0\t0\t19123\n",
      "3\t5743\t4398.0\t0\t5743\n",
      "4\t2596\t1099.5\t0\t2596\n",
      "5\t190\t274.9\t0\t190\n",
      "6\t434\t68.7\t0\t434\n",
      "7\t361\t17.2\t0\t361\n",
      "8\t438\t4.3\t0\t438\n",
      "9\t303\t1.1\t0\t300 3\n",
      "10\t412\t0.3\t1\t367 45\n",
      "11\t385\t0.1\t1\t338 47\n",
      "12\t762\t0.0\t1\t686 76\n",
      "13\t9\t0.0\t1\t4 5\n",
      "14\t484\t0.0\t1\t437 47\n",
      "15\t692\t0.0\t1\t623 69\n",
      "16\t2\t0.0\t1\t1 1\n",
      "17\t452\t0.0\t1\t404 48\n",
      "18\t257\t0.0\t1\t228 29\n",
      "19\t453\t0.0\t1\t405 48\n",
      "20\t332\t0.0\t1\t290 42\n",
      "21\t244\t0.0\t1\t219 25\n",
      "22\t305\t0.0\t1\t278 27\n",
      "23\t372\t0.0\t1\t340 32\n",
      "24\t581\t0.0\t1\t538 43\n",
      "25\t340\t0.0\t1\t304 36\n",
      "26\t316\t0.0\t1\t282 34\n",
      "27\t168\t0.0\t1\t149 19\n",
      "28\t453\t0.0\t1\t408 45\n",
      "29\t108\t0.0\t1\t88 20\n",
      "30\t457\t0.0\t1\t428 29\n",
      "31\t177\t0.0\t1\t150 27\n",
      "32\t759\t0.0\t1\t693 66\n",
      "33\t298\t0.0\t1\t274 24\n",
      "34\t81\t0.0\t1\t70 11\n",
      "35\t413\t0.0\t1\t381 32\n",
      "36\t145\t0.0\t1\t130 15\n",
      "37\t181\t0.0\t1\t163 18\n",
      "38\t105\t0.0\t1\t96 9\n",
      "39\t113\t0.0\t1\t105 8\n",
      "40\t51\t0.0\t1\t44 7\n",
      "41\t173\t0.0\t1\t163 10\n",
      "42\t263\t0.0\t1\t248 15\n",
      "43\t116\t0.0\t1\t100 16\n",
      "44\t154\t0.0\t1\t144 10\n",
      "45\t330\t0.0\t1\t308 22\n",
      "46\t171\t0.0\t1\t156 15\n",
      "47\t112\t0.0\t1\t103 9\n",
      "48\t305\t0.0\t1\t276 29\n",
      "49\t134\t0.0\t1\t123 11\n",
      "50\t41\t0.0\t1\t36 5\n",
      "51\t210\t0.0\t1\t199 11\n",
      "52\t314\t0.0\t1\t297 17\n",
      "53\t117\t0.0\t1\t103 14\n",
      "54\t134\t0.0\t1\t123 11\n",
      "55\t94\t0.0\t1\t86 8\n",
      "56\t42\t0.0\t1\t39 3\n",
      "57\t73\t0.0\t1\t64 9\n",
      "58\t70\t0.0\t1\t65 5\n",
      "59\t42\t0.0\t1\t41 1\n",
      "60\t31\t0.0\t1\t29 2\n",
      "61\t24\t0.0\t1\t19 5\n",
      "62\t27\t0.0\t1\t20 7\n",
      "63\t27\t0.0\t1\t25 2\n",
      "64\t48\t0.0\t1\t44 4\n",
      "65\t55\t0.0\t1\t53 2\n",
      "66\t76\t0.0\t1\t74 2\n",
      "67\t59\t0.0\t1\t59\n",
      "68\t50\t0.0\t1\t48 2\n",
      "69\t54\t0.0\t1\t51 3\n",
      "70\t60\t0.0\t1\t53 7\n",
      "71\t46\t0.0\t1\t44 2\n",
      "72\t48\t0.0\t1\t45 3\n",
      "73\t46\t0.0\t1\t43 3\n",
      "74\t51\t0.0\t1\t47 4\n",
      "75\t48\t0.0\t1\t44 4\n",
      "76\t51\t0.0\t1\t46 5\n",
      "77\t46\t0.0\t1\t42 4\n",
      "78\t42\t0.0\t1\t37 5\n",
      "79\t39\t0.0\t1\t34 5\n",
      "80\t38\t0.0\t1\t32 6\n",
      "81\t41\t0.0\t1\t35 6\n",
      "82\t44\t0.0\t1\t40 4\n",
      "83\t40\t0.0\t1\t33 7\n",
      "84\t33\t0.0\t1\t27 6\n",
      "85\t30\t0.0\t1\t26 4\n",
      "86\t30\t0.0\t1\t25 5\n",
      "87\t32\t0.0\t1\t29 3\n",
      "88\t25\t0.0\t1\t23 2\n",
      "89\t32\t0.0\t1\t28 4\n",
      "90\t33\t0.0\t1\t28 5\n",
      "91\t25\t0.0\t1\t20 5\n",
      "92\t19\t0.0\t1\t17 2\n",
      "93\t22\t0.0\t1\t20 2\n",
      "94\t25\t0.0\t1\t24 1\n",
      "95\t27\t0.0\t1\t23 4\n",
      "96\t23\t0.0\t1\t19 4\n",
      "97\t18\t0.0\t1\t17 1\n",
      "98\t22\t0.0\t1\t18 4\n",
      "99\t26\t0.0\t1\t22 4\n",
      "100\t13\t0.0\t1\t12 1\n",
      "101\t22\t0.0\t1\t16 6\n",
      "102\t19\t0.0\t1\t17 2\n",
      "103\t26\t0.0\t1\t23 3\n",
      "104\t16\t0.0\t1\t15 1\n",
      "105\t15\t0.0\t1\t13 2\n",
      "106\t16\t0.0\t1\t15 1\n",
      "107\t10\t0.0\t1\t7 3\n",
      "108\t11\t0.0\t1\t9 2\n",
      "109\t15\t0.0\t1\t11 4\n",
      "110\t7\t0.0\t1\t7\n",
      "111\t16\t0.0\t1\t15 1\n",
      "112\t11\t0.0\t1\t10 1\n",
      "113\t9\t0.0\t1\t8 1\n",
      "114\t7\t0.0\t1\t7\n",
      "115\t10\t0.0\t1\t9 1\n",
      "116\t11\t0.0\t1\t9 2\n",
      "117\t10\t0.0\t1\t5 5\n",
      "118\t4\t0.0\t1\t1 3\n",
      "119\t9\t0.0\t1\t7 2\n",
      "120\t9\t0.0\t1\t8 1\n",
      "121\t9\t0.0\t1\t8 1\n",
      "122\t3\t0.0\t1\t3\n",
      "123\t6\t0.0\t1\t5 1\n",
      "124\t8\t0.0\t1\t6 2\n",
      "125\t4\t0.0\t1\t4\n",
      "126\t14\t0.0\t1\t11 3\n",
      "127\t6\t0.0\t1\t6\n",
      "128\t8\t0.0\t1\t8\n",
      "129\t4\t0.0\t1\t3 1\n",
      "130\t9\t0.0\t1\t9\n",
      "131\t8\t0.0\t1\t8\n",
      "132\t11\t0.0\t1\t8 3\n",
      "133\t6\t0.0\t1\t6\n",
      "134\t6\t0.0\t1\t6\n",
      "135\t8\t0.0\t1\t7 1\n",
      "136\t5\t0.0\t1\t5\n",
      "137\t9\t0.0\t1\t9\n",
      "138\t5\t0.0\t1\t5\n",
      "139\t7\t0.0\t1\t7\n",
      "140\t2\t0.0\t1\t2\n",
      "141\t4\t0.0\t1\t2 2\n",
      "142\t4\t0.0\t1\t3 1\n",
      "143\t3\t0.0\t1\t3\n",
      "144\t3\t0.0\t1\t2 1\n",
      "145\t5\t0.0\t1\t0 5\n",
      "146\t4\t0.0\t1\t3 1\n",
      "147\t2\t0.0\t1\t0 2\n",
      "148\t9\t0.0\t1\t3 6\n",
      "151\t95\t0.0\t1\t83 12\n",
      "\n",
      "\n",
      "RUN STATISTICS FOR INPUT FILE: /home/sam/data/geoduck_illumina/trimmed/NR012_S1_L001_R2_001_val_2.fq.gz\n",
      "=============================================\n",
      "281472 sequences processed in total\n",
      "The length threshold of paired-end sequences gets evaluated later on (in the validation step)\n",
      "\n",
      "Validate paired-end files NR012_S1_L001_R1_001_val_1_trimmed.fq.gz and NR012_S1_L001_R2_001_val_2_trimmed.fq.gz\n",
      "file_1: NR012_S1_L001_R1_001_val_1_trimmed.fq.gz, file_2: NR012_S1_L001_R2_001_val_2_trimmed.fq.gz\n",
      "\n",
      "\n",
      ">>>>> Now validing the length of the 2 paired-end infiles: NR012_S1_L001_R1_001_val_1_trimmed.fq.gz and NR012_S1_L001_R2_001_val_2_trimmed.fq.gz <<<<<\n",
      "Writing validated paired-end read 1 reads to NR012_S1_L001_R1_001_val_1_val_1.fq.gz\n",
      "Writing validated paired-end read 2 reads to NR012_S1_L001_R2_001_val_2_val_2.fq.gz\n",
      "\n",
      "Total number of sequences analysed: 281472\n",
      "\n",
      "Number of sequence pairs removed because at least one read was shorter than the length cutoff (20 bp): 3813 (1.35%)\n",
      "\n",
      "Deleting both intermediate output files NR012_S1_L001_R1_001_val_1_trimmed.fq.gz and NR012_S1_L001_R2_001_val_2_trimmed.fq.gz\n",
      "\n",
      "====================================================================================================\n",
      "\n",
      "Writing report to '/home/sam/data/geoduck_illumina/trimmed/20180129_trimmed_again/NR012_S1_L002_R1_001_val_1.fq.gz_trimming_report.txt'\n",
      "\n",
      "SUMMARISING RUN PARAMETERS\n",
      "==========================\n",
      "Input filename: /home/sam/data/geoduck_illumina/trimmed/NR012_S1_L002_R1_001_val_1.fq.gz\n",
      "Trimming mode: paired-end\n",
      "Trim Galore version: 0.4.4\n",
      "Cutadapt version: 1.9.1\n",
      "Quality Phred score cutoff: 20\n",
      "Quality encoding type selected: ASCII+33\n",
      "Adapter sequence: 'AGATCGGAAGAGC' (Illumina TruSeq, Sanger iPCR; auto-detected)\n",
      "Maximum trimming error rate: 0.1 (default)\n",
      "Minimum required adapter overlap (stringency): 1 bp\n",
      "Minimum required sequence length for both reads before a sequence pair gets removed: 20 bp\n",
      "Output file(s) will be GZIP compressed\n",
      "\n",
      "Writing final adapter and quality trimmed output to NR012_S1_L002_R1_001_val_1_trimmed.fq.gz\n",
      "\n",
      "\n",
      "  >>> Now performing quality (cutoff 20) and adapter trimming in a single pass for the adapter sequence: 'AGATCGGAAGAGC' from file /home/sam/data/geoduck_illumina/trimmed/NR012_S1_L002_R1_001_val_1.fq.gz <<< \n",
      "This is cutadapt 1.9.1 with Python 2.7.12\n",
      "Command line parameters: -f fastq -e 0.1 -q 20 -O 1 -a AGATCGGAAGAGC /home/sam/data/geoduck_illumina/trimmed/NR012_S1_L002_R1_001_val_1.fq.gz\n",
      "Trimming 1 adapter with at most 10.0% errors in single-end mode ...\n",
      "Finished in 8.48 s (26 us/read; 2.31 M reads/minute).\n",
      "\n",
      "=== Summary ===\n",
      "\n",
      "Total reads processed:                 326,252\n",
      "Reads with adapters:                   140,395 (43.0%)\n",
      "Reads written (passing filters):       326,252 (100.0%)\n",
      "\n",
      "Total basepairs processed:    41,596,018 bp\n",
      "Quality-trimmed:                  15,103 bp (0.0%)\n",
      "Total written (filtered):     40,182,403 bp (96.6%)\n",
      "\n",
      "=== Adapter 1 ===\n",
      "\n",
      "Sequence: AGATCGGAAGAGC; Type: regular 3'; Length: 13; Trimmed: 140395 times.\n",
      "\n",
      "No. of allowed errors:\n",
      "0-9 bp: 0; 10-13 bp: 1\n",
      "\n",
      "Bases preceding removed adapters:\n",
      "  A: 30.8%\n",
      "  C: 23.3%\n",
      "  G: 15.5%\n",
      "  T: 27.4%\n",
      "  none/other: 3.0%\n",
      "\n",
      "Overview of removed sequences\n",
      "length\tcount\texpect\tmax.err\terror counts\n",
      "1\t89057\t81563.0\t0\t89057\n",
      "2\t21691\t20390.8\t0\t21691\n",
      "3\t6146\t5097.7\t0\t6146\n",
      "4\t2757\t1274.4\t0\t2757\n",
      "5\t208\t318.6\t0\t208\n",
      "6\t391\t79.7\t0\t391\n",
      "7\t349\t19.9\t0\t349\n",
      "8\t371\t5.0\t0\t371\n",
      "9\t332\t1.2\t0\t327 5\n",
      "10\t366\t0.3\t1\t334 32\n",
      "11\t354\t0.1\t1\t331 23\n",
      "12\t685\t0.0\t1\t644 41\n",
      "13\t17\t0.0\t1\t12 5\n",
      "14\t670\t0.0\t1\t629 41\n",
      "15\t12\t0.0\t1\t6 6\n",
      "16\t719\t0.0\t1\t672 47\n",
      "17\t24\t0.0\t1\t18 6\n",
      "18\t430\t0.0\t1\t401 29\n",
      "19\t1193\t0.0\t1\t1140 53\n",
      "20\t4\t0.0\t1\t3 1\n",
      "21\t3\t0.0\t1\t0 3\n",
      "22\t10\t0.0\t1\t6 4\n",
      "23\t335\t0.0\t1\t305 30\n",
      "24\t840\t0.0\t1\t795 45\n",
      "25\t23\t0.0\t1\t18 5\n",
      "26\t324\t0.0\t1\t298 26\n",
      "27\t263\t0.0\t1\t244 19\n",
      "28\t1\t0.0\t1\t0 1\n",
      "29\t279\t0.0\t1\t257 22\n",
      "30\t328\t0.0\t1\t297 31\n",
      "31\t474\t0.0\t1\t448 26\n",
      "32\t15\t0.0\t1\t9 6\n",
      "33\t498\t0.0\t1\t460 38\n",
      "34\t453\t0.0\t1\t429 24\n",
      "35\t34\t0.0\t1\t30 4\n",
      "36\t70\t0.0\t1\t62 8\n",
      "37\t206\t0.0\t1\t190 16\n",
      "38\t233\t0.0\t1\t226 7\n",
      "39\t231\t0.0\t1\t216 15\n",
      "40\t180\t0.0\t1\t165 15\n",
      "41\t237\t0.0\t1\t218 19\n",
      "42\t398\t0.0\t1\t371 27\n",
      "43\t102\t0.0\t1\t89 13\n",
      "44\t87\t0.0\t1\t70 17\n",
      "45\t113\t0.0\t1\t95 18\n",
      "46\t178\t0.0\t1\t161 17\n",
      "47\t171\t0.0\t1\t156 15\n",
      "48\t141\t0.0\t1\t120 21\n",
      "49\t406\t0.0\t1\t380 26\n",
      "50\t244\t0.0\t1\t236 8\n",
      "51\t89\t0.0\t1\t83 6\n",
      "52\t91\t0.0\t1\t80 11\n",
      "53\t260\t0.0\t1\t237 23\n",
      "54\t197\t0.0\t1\t192 5\n",
      "55\t40\t0.0\t1\t27 13\n",
      "56\t88\t0.0\t1\t75 13\n",
      "57\t191\t0.0\t1\t185 6\n",
      "58\t6\t0.0\t1\t4 2\n",
      "59\t54\t0.0\t1\t45 9\n",
      "60\t82\t0.0\t1\t75 7\n",
      "61\t7\t0.0\t1\t6 1\n",
      "62\t19\t0.0\t1\t13 6\n",
      "63\t63\t0.0\t1\t56 7\n",
      "64\t91\t0.0\t1\t81 10\n",
      "65\t71\t0.0\t1\t63 8\n",
      "66\t71\t0.0\t1\t61 10\n",
      "67\t75\t0.0\t1\t63 12\n",
      "68\t49\t0.0\t1\t43 6\n",
      "69\t17\t0.0\t1\t15 2\n",
      "70\t35\t0.0\t1\t33 2\n",
      "71\t46\t0.0\t1\t44 2\n",
      "72\t41\t0.0\t1\t40 1\n",
      "73\t44\t0.0\t1\t39 5\n",
      "74\t40\t0.0\t1\t32 8\n",
      "75\t57\t0.0\t1\t51 6\n",
      "76\t53\t0.0\t1\t49 4\n",
      "77\t54\t0.0\t1\t49 5\n",
      "78\t44\t0.0\t1\t41 3\n",
      "79\t37\t0.0\t1\t35 2\n",
      "80\t57\t0.0\t1\t49 8\n",
      "81\t47\t0.0\t1\t38 9\n",
      "82\t52\t0.0\t1\t49 3\n",
      "83\t36\t0.0\t1\t29 7\n",
      "84\t46\t0.0\t1\t34 12\n",
      "85\t56\t0.0\t1\t48 8\n",
      "86\t47\t0.0\t1\t40 7\n",
      "87\t32\t0.0\t1\t28 4\n",
      "88\t49\t0.0\t1\t41 8\n",
      "89\t44\t0.0\t1\t35 9\n",
      "90\t41\t0.0\t1\t33 8\n",
      "91\t38\t0.0\t1\t33 5\n",
      "92\t41\t0.0\t1\t35 6\n",
      "93\t42\t0.0\t1\t36 6\n",
      "94\t32\t0.0\t1\t27 5\n",
      "95\t28\t0.0\t1\t23 5\n",
      "96\t29\t0.0\t1\t28 1\n",
      "97\t41\t0.0\t1\t35 6\n",
      "98\t35\t0.0\t1\t26 9\n",
      "99\t23\t0.0\t1\t21 2\n",
      "100\t25\t0.0\t1\t22 3\n",
      "101\t33\t0.0\t1\t29 4\n",
      "102\t19\t0.0\t1\t18 1\n",
      "103\t26\t0.0\t1\t21 5\n",
      "104\t17\t0.0\t1\t14 3\n",
      "105\t30\t0.0\t1\t25 5\n",
      "106\t26\t0.0\t1\t20 6\n",
      "107\t18\t0.0\t1\t18\n",
      "108\t20\t0.0\t1\t14 6\n",
      "109\t17\t0.0\t1\t16 1\n",
      "110\t14\t0.0\t1\t11 3\n",
      "111\t15\t0.0\t1\t14 1\n",
      "112\t20\t0.0\t1\t16 4\n",
      "113\t17\t0.0\t1\t12 5\n",
      "114\t17\t0.0\t1\t9 8\n",
      "115\t21\t0.0\t1\t18 3\n",
      "116\t16\t0.0\t1\t11 5\n",
      "117\t14\t0.0\t1\t11 3\n",
      "118\t13\t0.0\t1\t11 2\n",
      "119\t24\t0.0\t1\t21 3\n",
      "120\t12\t0.0\t1\t9 3\n",
      "121\t10\t0.0\t1\t6 4\n",
      "122\t17\t0.0\t1\t12 5\n",
      "123\t11\t0.0\t1\t9 2\n",
      "124\t6\t0.0\t1\t3 3\n",
      "125\t12\t0.0\t1\t9 3\n",
      "126\t14\t0.0\t1\t12 2\n",
      "127\t14\t0.0\t1\t11 3\n",
      "128\t6\t0.0\t1\t6\n",
      "129\t9\t0.0\t1\t7 2\n",
      "130\t13\t0.0\t1\t11 2\n",
      "131\t14\t0.0\t1\t13 1\n",
      "132\t7\t0.0\t1\t7\n",
      "133\t9\t0.0\t1\t9\n",
      "134\t14\t0.0\t1\t13 1\n",
      "135\t7\t0.0\t1\t6 1\n",
      "136\t5\t0.0\t1\t5\n",
      "137\t10\t0.0\t1\t10\n",
      "138\t6\t0.0\t1\t6\n",
      "139\t8\t0.0\t1\t8\n",
      "140\t5\t0.0\t1\t5\n",
      "141\t3\t0.0\t1\t3\n",
      "142\t5\t0.0\t1\t4 1\n",
      "143\t4\t0.0\t1\t4\n",
      "144\t2\t0.0\t1\t2\n",
      "145\t3\t0.0\t1\t2 1\n",
      "146\t4\t0.0\t1\t4\n",
      "147\t4\t0.0\t1\t2 2\n",
      "148\t9\t0.0\t1\t8 1\n",
      "149\t3\t0.0\t1\t1 2\n",
      "150\t23\t0.0\t1\t4 19\n",
      "151\t4243\t0.0\t1\t117 4126\n",
      "\n",
      "\n",
      "RUN STATISTICS FOR INPUT FILE: /home/sam/data/geoduck_illumina/trimmed/NR012_S1_L002_R1_001_val_1.fq.gz\n",
      "=============================================\n",
      "326252 sequences processed in total\n",
      "The length threshold of paired-end sequences gets evaluated later on (in the validation step)\n",
      "\n",
      "Writing report to '/home/sam/data/geoduck_illumina/trimmed/20180129_trimmed_again/NR012_S1_L002_R2_001_val_2.fq.gz_trimming_report.txt'\n",
      "\n",
      "SUMMARISING RUN PARAMETERS\n",
      "==========================\n",
      "Input filename: /home/sam/data/geoduck_illumina/trimmed/NR012_S1_L002_R2_001_val_2.fq.gz\n",
      "Trimming mode: paired-end\n",
      "Trim Galore version: 0.4.4\n",
      "Cutadapt version: 1.9.1\n",
      "Quality Phred score cutoff: 20\n",
      "Quality encoding type selected: ASCII+33\n",
      "Adapter sequence: 'AGATCGGAAGAGC' (Illumina TruSeq, Sanger iPCR; auto-detected)\n",
      "Maximum trimming error rate: 0.1 (default)\n",
      "Minimum required adapter overlap (stringency): 1 bp\n",
      "Minimum required sequence length for both reads before a sequence pair gets removed: 20 bp\n",
      "Output file(s) will be GZIP compressed\n",
      "\n",
      "Writing final adapter and quality trimmed output to NR012_S1_L002_R2_001_val_2_trimmed.fq.gz\n",
      "\n",
      "\n",
      "  >>> Now performing quality (cutoff 20) and adapter trimming in a single pass for the adapter sequence: 'AGATCGGAAGAGC' from file /home/sam/data/geoduck_illumina/trimmed/NR012_S1_L002_R2_001_val_2.fq.gz <<< \n",
      "This is cutadapt 1.9.1 with Python 2.7.12\n",
      "Command line parameters: -f fastq -e 0.1 -q 20 -O 1 -a AGATCGGAAGAGC /home/sam/data/geoduck_illumina/trimmed/NR012_S1_L002_R2_001_val_2.fq.gz\n",
      "Trimming 1 adapter with at most 10.0% errors in single-end mode ...\n",
      "Finished in 8.52 s (26 us/read; 2.30 M reads/minute).\n",
      "\n",
      "=== Summary ===\n",
      "\n",
      "Total reads processed:                 326,252\n",
      "Reads with adapters:                   139,337 (42.7%)\n",
      "Reads written (passing filters):       326,252 (100.0%)\n",
      "\n",
      "Total basepairs processed:    42,562,422 bp\n",
      "Quality-trimmed:                  18,976 bp (0.0%)\n",
      "Total written (filtered):     41,698,391 bp (98.0%)\n",
      "\n",
      "=== Adapter 1 ===\n",
      "\n",
      "Sequence: AGATCGGAAGAGC; Type: regular 3'; Length: 13; Trimmed: 139337 times.\n",
      "\n",
      "No. of allowed errors:\n",
      "0-9 bp: 0; 10-13 bp: 1\n",
      "\n",
      "Bases preceding removed adapters:\n",
      "  A: 32.2%\n",
      "  C: 23.0%\n",
      "  G: 16.3%\n",
      "  T: 28.4%\n",
      "  none/other: 0.1%\n",
      "\n",
      "Overview of removed sequences\n",
      "length\tcount\texpect\tmax.err\terror counts\n",
      "1\t90369\t81563.0\t0\t90369\n",
      "2\t21799\t20390.8\t0\t21799\n",
      "3\t6520\t5097.7\t0\t6520\n",
      "4\t2800\t1274.4\t0\t2800\n",
      "5\t171\t318.6\t0\t171\n",
      "6\t409\t79.7\t0\t409\n",
      "7\t410\t19.9\t0\t410\n",
      "8\t426\t5.0\t0\t426\n",
      "9\t296\t1.2\t0\t293 3\n",
      "10\t393\t0.3\t1\t365 28\n",
      "11\t373\t0.1\t1\t327 46\n",
      "12\t713\t0.0\t1\t666 47\n",
      "13\t1\t0.0\t1\t1\n",
      "14\t417\t0.0\t1\t385 32\n",
      "15\t651\t0.0\t1\t606 45\n",
      "16\t4\t0.0\t1\t2 2\n",
      "17\t417\t0.0\t1\t387 30\n",
      "18\t271\t0.0\t1\t253 18\n",
      "19\t440\t0.0\t1\t414 26\n",
      "20\t340\t0.0\t1\t332 8\n",
      "21\t291\t0.0\t1\t273 18\n",
      "22\t366\t0.0\t1\t341 25\n",
      "23\t319\t0.0\t1\t299 20\n",
      "24\t493\t0.0\t1\t459 34\n",
      "25\t339\t0.0\t1\t303 36\n",
      "26\t319\t0.0\t1\t302 17\n",
      "27\t190\t0.0\t1\t175 15\n",
      "28\t444\t0.0\t1\t430 14\n",
      "29\t123\t0.0\t1\t107 16\n",
      "30\t446\t0.0\t1\t422 24\n",
      "31\t180\t0.0\t1\t156 24\n",
      "32\t875\t0.0\t1\t837 38\n",
      "33\t307\t0.0\t1\t287 20\n",
      "34\t94\t0.0\t1\t85 9\n",
      "35\t482\t0.0\t1\t454 28\n",
      "36\t93\t0.0\t1\t84 9\n",
      "37\t215\t0.0\t1\t203 12\n",
      "38\t67\t0.0\t1\t64 3\n",
      "39\t102\t0.0\t1\t93 9\n",
      "40\t35\t0.0\t1\t31 4\n",
      "41\t180\t0.0\t1\t171 9\n",
      "42\t278\t0.0\t1\t265 13\n",
      "43\t122\t0.0\t1\t114 8\n",
      "44\t173\t0.0\t1\t158 15\n",
      "45\t360\t0.0\t1\t338 22\n",
      "46\t177\t0.0\t1\t165 12\n",
      "47\t85\t0.0\t1\t77 8\n",
      "48\t420\t0.0\t1\t397 23\n",
      "49\t99\t0.0\t1\t95 4\n",
      "50\t38\t0.0\t1\t28 10\n",
      "51\t242\t0.0\t1\t218 24\n",
      "52\t450\t0.0\t1\t439 11\n",
      "53\t126\t0.0\t1\t119 7\n",
      "54\t140\t0.0\t1\t129 11\n",
      "55\t65\t0.0\t1\t62 3\n",
      "56\t46\t0.0\t1\t38 8\n",
      "57\t71\t0.0\t1\t65 6\n",
      "58\t90\t0.0\t1\t86 4\n",
      "59\t36\t0.0\t1\t31 5\n",
      "60\t23\t0.0\t1\t21 2\n",
      "61\t20\t0.0\t1\t16 4\n",
      "62\t24\t0.0\t1\t21 3\n",
      "63\t30\t0.0\t1\t25 5\n",
      "64\t37\t0.0\t1\t34 3\n",
      "65\t67\t0.0\t1\t60 7\n",
      "66\t79\t0.0\t1\t71 8\n",
      "67\t89\t0.0\t1\t83 6\n",
      "68\t91\t0.0\t1\t82 9\n",
      "69\t72\t0.0\t1\t69 3\n",
      "70\t79\t0.0\t1\t68 11\n",
      "71\t72\t0.0\t1\t57 15\n",
      "72\t73\t0.0\t1\t68 5\n",
      "73\t72\t0.0\t1\t63 9\n",
      "74\t61\t0.0\t1\t54 7\n",
      "75\t81\t0.0\t1\t75 6\n",
      "76\t71\t0.0\t1\t62 9\n",
      "77\t68\t0.0\t1\t62 6\n",
      "78\t65\t0.0\t1\t59 6\n",
      "79\t56\t0.0\t1\t49 7\n",
      "80\t66\t0.0\t1\t59 7\n",
      "81\t57\t0.0\t1\t51 6\n",
      "82\t60\t0.0\t1\t53 7\n",
      "83\t48\t0.0\t1\t41 7\n",
      "84\t47\t0.0\t1\t41 6\n",
      "85\t61\t0.0\t1\t52 9\n",
      "86\t47\t0.0\t1\t40 7\n",
      "87\t41\t0.0\t1\t36 5\n",
      "88\t60\t0.0\t1\t56 4\n",
      "89\t49\t0.0\t1\t43 6\n",
      "90\t54\t0.0\t1\t52 2\n",
      "91\t51\t0.0\t1\t49 2\n",
      "92\t50\t0.0\t1\t43 7\n",
      "93\t52\t0.0\t1\t47 5\n",
      "94\t49\t0.0\t1\t37 12\n",
      "95\t38\t0.0\t1\t34 4\n",
      "96\t36\t0.0\t1\t33 3\n",
      "97\t53\t0.0\t1\t48 5\n",
      "98\t40\t0.0\t1\t32 8\n",
      "99\t34\t0.0\t1\t28 6\n",
      "100\t33\t0.0\t1\t26 7\n",
      "101\t34\t0.0\t1\t29 5\n",
      "102\t33\t0.0\t1\t28 5\n",
      "103\t26\t0.0\t1\t22 4\n",
      "104\t27\t0.0\t1\t20 7\n",
      "105\t25\t0.0\t1\t23 2\n",
      "106\t31\t0.0\t1\t26 5\n",
      "107\t23\t0.0\t1\t21 2\n",
      "108\t17\t0.0\t1\t15 2\n",
      "109\t25\t0.0\t1\t23 2\n",
      "110\t20\t0.0\t1\t15 5\n",
      "111\t19\t0.0\t1\t14 5\n",
      "112\t30\t0.0\t1\t24 6\n",
      "113\t19\t0.0\t1\t18 1\n",
      "114\t27\t0.0\t1\t18 9\n",
      "115\t26\t0.0\t1\t23 3\n",
      "116\t20\t0.0\t1\t17 3\n",
      "117\t20\t0.0\t1\t17 3\n",
      "118\t28\t0.0\t1\t18 10\n",
      "119\t27\t0.0\t1\t24 3\n",
      "120\t15\t0.0\t1\t12 3\n",
      "121\t15\t0.0\t1\t12 3\n",
      "122\t19\t0.0\t1\t12 7\n",
      "123\t20\t0.0\t1\t14 6\n",
      "124\t6\t0.0\t1\t4 2\n",
      "125\t16\t0.0\t1\t14 2\n",
      "126\t13\t0.0\t1\t12 1\n",
      "127\t17\t0.0\t1\t15 2\n",
      "128\t8\t0.0\t1\t6 2\n",
      "129\t8\t0.0\t1\t8\n",
      "130\t15\t0.0\t1\t14 1\n",
      "131\t14\t0.0\t1\t14\n",
      "132\t8\t0.0\t1\t7 1\n",
      "133\t9\t0.0\t1\t9\n",
      "134\t15\t0.0\t1\t12 3\n",
      "135\t8\t0.0\t1\t7 1\n",
      "136\t5\t0.0\t1\t5\n",
      "137\t10\t0.0\t1\t9 1\n",
      "138\t7\t0.0\t1\t7\n",
      "139\t8\t0.0\t1\t8\n",
      "140\t5\t0.0\t1\t5\n",
      "141\t3\t0.0\t1\t2 1\n",
      "142\t4\t0.0\t1\t3 1\n",
      "143\t4\t0.0\t1\t2 2\n",
      "144\t2\t0.0\t1\t2\n",
      "145\t2\t0.0\t1\t1 1\n",
      "146\t4\t0.0\t1\t2 2\n",
      "147\t2\t0.0\t1\t2\n",
      "148\t8\t0.0\t1\t5 3\n",
      "150\t4\t0.0\t1\t3 1\n",
      "151\t132\t0.0\t1\t122 10\n",
      "\n",
      "\n",
      "RUN STATISTICS FOR INPUT FILE: /home/sam/data/geoduck_illumina/trimmed/NR012_S1_L002_R2_001_val_2.fq.gz\n",
      "=============================================\n",
      "326252 sequences processed in total\n",
      "The length threshold of paired-end sequences gets evaluated later on (in the validation step)\n",
      "\n",
      "Validate paired-end files NR012_S1_L002_R1_001_val_1_trimmed.fq.gz and NR012_S1_L002_R2_001_val_2_trimmed.fq.gz\n",
      "file_1: NR012_S1_L002_R1_001_val_1_trimmed.fq.gz, file_2: NR012_S1_L002_R2_001_val_2_trimmed.fq.gz\n",
      "\n",
      "\n",
      ">>>>> Now validing the length of the 2 paired-end infiles: NR012_S1_L002_R1_001_val_1_trimmed.fq.gz and NR012_S1_L002_R2_001_val_2_trimmed.fq.gz <<<<<\n",
      "Writing validated paired-end read 1 reads to NR012_S1_L002_R1_001_val_1_val_1.fq.gz\n",
      "Writing validated paired-end read 2 reads to NR012_S1_L002_R2_001_val_2_val_2.fq.gz\n",
      "\n",
      "Total number of sequences analysed: 326252\n",
      "\n",
      "Number of sequence pairs removed because at least one read was shorter than the length cutoff (20 bp): 5128 (1.57%)\n",
      "\n",
      "Deleting both intermediate output files NR012_S1_L002_R1_001_val_1_trimmed.fq.gz and NR012_S1_L002_R2_001_val_2_trimmed.fq.gz\n",
      "\n",
      "====================================================================================================\n",
      "\n",
      "Writing report to '/home/sam/data/geoduck_illumina/trimmed/20180129_trimmed_again/NR013_AD013_S2_L001_R1_001_val_1.fq.gz_trimming_report.txt'\n",
      "\n",
      "SUMMARISING RUN PARAMETERS\n",
      "==========================\n",
      "Input filename: /home/sam/data/geoduck_illumina/trimmed/NR013_AD013_S2_L001_R1_001_val_1.fq.gz\n",
      "Trimming mode: paired-end\n",
      "Trim Galore version: 0.4.4\n",
      "Cutadapt version: 1.9.1\n",
      "Quality Phred score cutoff: 20\n",
      "Quality encoding type selected: ASCII+33\n",
      "Adapter sequence: 'AGATCGGAAGAGC' (Illumina TruSeq, Sanger iPCR; auto-detected)\n",
      "Maximum trimming error rate: 0.1 (default)\n",
      "Minimum required adapter overlap (stringency): 1 bp\n",
      "Minimum required sequence length for both reads before a sequence pair gets removed: 20 bp\n",
      "Output file(s) will be GZIP compressed\n",
      "\n",
      "Writing final adapter and quality trimmed output to NR013_AD013_S2_L001_R1_001_val_1_trimmed.fq.gz\n",
      "\n",
      "\n",
      "  >>> Now performing quality (cutoff 20) and adapter trimming in a single pass for the adapter sequence: 'AGATCGGAAGAGC' from file /home/sam/data/geoduck_illumina/trimmed/NR013_AD013_S2_L001_R1_001_val_1.fq.gz <<< \n",
      "10000000 sequences processed\n",
      "20000000 sequences processed\n",
      "30000000 sequences processed\n",
      "40000000 sequences processed\n",
      "50000000 sequences processed\n",
      "60000000 sequences processed\n",
      "70000000 sequences processed\n",
      "80000000 sequences processed\n",
      "90000000 sequences processed\n",
      "100000000 sequences processed\n",
      "110000000 sequences processed\n",
      "120000000 sequences processed\n",
      "130000000 sequences processed\n",
      "140000000 sequences processed\n",
      "150000000 sequences processed\n",
      "160000000 sequences processed\n",
      "170000000 sequences processed\n",
      "180000000 sequences processed\n",
      "190000000 sequences processed\n",
      "200000000 sequences processed\n",
      "210000000 sequences processed\n",
      "220000000 sequences processed\n",
      "230000000 sequences processed\n",
      "240000000 sequences processed\n",
      "250000000 sequences processed\n",
      "260000000 sequences processed\n",
      "270000000 sequences processed\n",
      "280000000 sequences processed\n",
      "290000000 sequences processed\n",
      "300000000 sequences processed\n",
      "This is cutadapt 1.9.1 with Python 2.7.12\n",
      "Command line parameters: -f fastq -e 0.1 -q 20 -O 1 -a AGATCGGAAGAGC /home/sam/data/geoduck_illumina/trimmed/NR013_AD013_S2_L001_R1_001_val_1.fq.gz\n",
      "Trimming 1 adapter with at most 10.0% errors in single-end mode ...\n",
      "Finished in 7650.23 s (25 us/read; 2.43 M reads/minute).\n",
      "\n",
      "=== Summary ===\n",
      "\n",
      "Total reads processed:             309,877,704\n",
      "Reads with adapters:               125,600,847 (40.5%)\n",
      "Reads written (passing filters):   309,877,704 (100.0%)\n",
      "\n",
      "Total basepairs processed: 39,009,495,129 bp\n",
      "Quality-trimmed:              24,971,797 bp (0.1%)\n",
      "Total written (filtered):  38,552,981,389 bp (98.8%)\n",
      "\n",
      "=== Adapter 1 ===\n",
      "\n",
      "Sequence: AGATCGGAAGAGC; Type: regular 3'; Length: 13; Trimmed: 125600847 times.\n",
      "\n",
      "No. of allowed errors:\n",
      "0-9 bp: 0; 10-13 bp: 1\n",
      "\n",
      "Bases preceding removed adapters:\n",
      "  A: 32.1%\n",
      "  C: 23.7%\n",
      "  G: 14.5%\n",
      "  T: 29.6%\n",
      "  none/other: 0.1%\n",
      "\n",
      "Overview of removed sequences\n",
      "length\tcount\texpect\tmax.err\terror counts\n",
      "1\t89424517\t77469426.0\t0\t89424517\n",
      "2\t22164935\t19367356.5\t0\t22164935\n",
      "3\t6130509\t4841839.1\t0\t6130509\n",
      "4\t2218083\t1210459.8\t0\t2218083\n",
      "5\t219378\t302614.9\t0\t219378\n",
      "6\t99427\t75653.7\t0\t99427\n",
      "7\t69831\t18913.4\t0\t69831\n",
      "8\t81144\t4728.4\t0\t81144\n",
      "9\t69327\t1182.1\t0\t65673 3654\n",
      "10\t86254\t295.5\t1\t64771 21483\n",
      "11\t86689\t73.9\t1\t66818 19871\n",
      "12\t152618\t18.5\t1\t127131 25487\n",
      "13\t8000\t4.6\t1\t4400 3600\n",
      "14\t142876\t4.6\t1\t121534 21342\n",
      "15\t6164\t4.6\t1\t3175 2989\n",
      "16\t157524\t4.6\t1\t129484 28040\n",
      "17\t9368\t4.6\t1\t5281 4087\n",
      "18\t110322\t4.6\t1\t89737 20585\n",
      "19\t227175\t4.6\t1\t198294 28881\n",
      "20\t2286\t4.6\t1\t1173 1113\n",
      "21\t2562\t4.6\t1\t970 1592\n",
      "22\t5827\t4.6\t1\t2657 3170\n",
      "23\t88216\t4.6\t1\t71209 17007\n",
      "24\t186505\t4.6\t1\t158602 27903\n",
      "25\t8216\t4.6\t1\t4608 3608\n",
      "26\t76802\t4.6\t1\t61671 15131\n",
      "27\t65197\t4.6\t1\t55637 9560\n",
      "28\t2895\t4.6\t1\t1382 1513\n",
      "29\t67938\t4.6\t1\t55476 12462\n",
      "30\t88509\t4.6\t1\t73248 15261\n",
      "31\t107470\t4.6\t1\t90166 17304\n",
      "32\t9768\t4.6\t1\t5863 3905\n",
      "33\t124009\t4.6\t1\t102313 21696\n",
      "34\t3169\t4.6\t1\t1701 1468\n",
      "35\t65831\t4.6\t1\t52431 13400\n",
      "36\t86774\t4.6\t1\t70190 16584\n",
      "37\t100365\t4.6\t1\t83307 17058\n",
      "38\t4565\t4.6\t1\t2666 1899\n",
      "39\t79416\t4.6\t1\t62736 16680\n",
      "40\t117161\t4.6\t1\t92493 24668\n",
      "41\t6185\t4.6\t1\t3649 2536\n",
      "42\t79241\t4.6\t1\t60043 19198\n",
      "43\t98328\t4.6\t1\t73247 25081\n",
      "44\t150109\t4.6\t1\t119150 30959\n",
      "45\t31815\t4.6\t1\t25768 6047\n",
      "46\t43480\t4.6\t1\t31378 12102\n",
      "47\t4867\t4.6\t1\t2963 1904\n",
      "48\t117730\t4.6\t1\t95666 22064\n",
      "49\t64655\t4.6\t1\t50519 14136\n",
      "50\t49998\t4.6\t1\t37908 12090\n",
      "51\t80945\t4.6\t1\t64124 16821\n",
      "52\t100481\t4.6\t1\t82422 18059\n",
      "53\t35864\t4.6\t1\t29124 6740\n",
      "54\t8281\t4.6\t1\t5909 2372\n",
      "55\t114512\t4.6\t1\t100445 14067\n",
      "56\t88023\t4.6\t1\t74060 13963\n",
      "57\t9008\t4.6\t1\t6800 2208\n",
      "58\t16584\t4.6\t1\t11832 4752\n",
      "59\t79509\t4.6\t1\t68186 11323\n",
      "60\t5679\t4.6\t1\t3243 2436\n",
      "61\t13090\t4.6\t1\t8529 4561\n",
      "62\t64280\t4.6\t1\t54366 9914\n",
      "63\t3785\t4.6\t1\t2002 1783\n",
      "64\t9323\t4.6\t1\t5891 3432\n",
      "65\t22289\t4.6\t1\t17071 5218\n",
      "66\t58348\t4.6\t1\t49571 8777\n",
      "67\t27565\t4.6\t1\t21718 5847\n",
      "68\t37093\t4.6\t1\t30329 6764\n",
      "69\t29029\t4.6\t1\t23079 5950\n",
      "70\t25492\t4.6\t1\t19547 5945\n",
      "71\t12307\t4.6\t1\t8398 3909\n",
      "72\t16100\t4.6\t1\t12236 3864\n",
      "73\t24087\t4.6\t1\t19483 4604\n",
      "74\t28372\t4.6\t1\t23063 5309\n",
      "75\t30767\t4.6\t1\t24926 5841\n",
      "76\t32489\t4.6\t1\t26238 6251\n",
      "77\t32880\t4.6\t1\t26516 6364\n",
      "78\t33544\t4.6\t1\t26942 6602\n",
      "79\t33279\t4.6\t1\t26423 6856\n",
      "80\t33181\t4.6\t1\t26360 6821\n",
      "81\t33195\t4.6\t1\t26291 6904\n",
      "82\t33217\t4.6\t1\t26339 6878\n",
      "83\t32639\t4.6\t1\t25624 7015\n",
      "84\t32054\t4.6\t1\t25344 6710\n",
      "85\t31961\t4.6\t1\t25110 6851\n",
      "86\t31461\t4.6\t1\t24659 6802\n",
      "87\t31056\t4.6\t1\t24375 6681\n",
      "88\t30928\t4.6\t1\t24346 6582\n",
      "89\t29979\t4.6\t1\t23608 6371\n",
      "90\t29702\t4.6\t1\t23396 6306\n",
      "91\t29509\t4.6\t1\t23084 6425\n",
      "92\t28807\t4.6\t1\t22489 6318\n",
      "93\t27764\t4.6\t1\t21767 5997\n",
      "94\t27551\t4.6\t1\t21509 6042\n",
      "95\t26674\t4.6\t1\t20797 5877\n",
      "96\t26566\t4.6\t1\t20764 5802\n",
      "97\t25897\t4.6\t1\t20320 5577\n",
      "98\t25130\t4.6\t1\t19615 5515\n",
      "99\t24662\t4.6\t1\t19241 5421\n",
      "100\t23805\t4.6\t1\t18513 5292\n",
      "101\t23057\t4.6\t1\t17989 5068\n",
      "102\t22484\t4.6\t1\t17512 4972\n",
      "103\t22162\t4.6\t1\t17259 4903\n",
      "104\t21057\t4.6\t1\t16304 4753\n",
      "105\t20635\t4.6\t1\t15939 4696\n",
      "106\t19695\t4.6\t1\t15133 4562\n",
      "107\t18768\t4.6\t1\t14393 4375\n",
      "108\t17844\t4.6\t1\t13675 4169\n",
      "109\t17123\t4.6\t1\t13124 3999\n",
      "110\t16499\t4.6\t1\t12532 3967\n",
      "111\t15593\t4.6\t1\t11916 3677\n",
      "112\t15113\t4.6\t1\t11560 3553\n",
      "113\t14481\t4.6\t1\t11021 3460\n",
      "114\t13889\t4.6\t1\t10658 3231\n",
      "115\t13496\t4.6\t1\t10426 3070\n",
      "116\t12857\t4.6\t1\t9829 3028\n",
      "117\t12269\t4.6\t1\t9412 2857\n",
      "118\t11754\t4.6\t1\t8975 2779\n",
      "119\t11278\t4.6\t1\t8643 2635\n",
      "120\t10276\t4.6\t1\t7870 2406\n",
      "121\t10062\t4.6\t1\t7696 2366\n",
      "122\t8939\t4.6\t1\t6836 2103\n",
      "123\t8804\t4.6\t1\t6816 1988\n",
      "124\t7987\t4.6\t1\t6117 1870\n",
      "125\t7642\t4.6\t1\t5918 1724\n",
      "126\t7156\t4.6\t1\t5526 1630\n",
      "127\t6575\t4.6\t1\t5123 1452\n",
      "128\t5937\t4.6\t1\t4721 1216\n",
      "129\t6086\t4.6\t1\t5006 1080\n",
      "130\t4836\t4.6\t1\t3909 927\n",
      "131\t4800\t4.6\t1\t4045 755\n",
      "132\t3687\t4.6\t1\t3116 571\n",
      "133\t3273\t4.6\t1\t2805 468\n",
      "134\t2904\t4.6\t1\t2528 376\n",
      "135\t2541\t4.6\t1\t2231 310\n",
      "136\t2071\t4.6\t1\t1822 249\n",
      "137\t1835\t4.6\t1\t1635 200\n",
      "138\t1525\t4.6\t1\t1343 182\n",
      "139\t1248\t4.6\t1\t1099 149\n",
      "140\t1075\t4.6\t1\t952 123\n",
      "141\t817\t4.6\t1\t727 90\n",
      "142\t833\t4.6\t1\t735 98\n",
      "143\t705\t4.6\t1\t627 78\n",
      "144\t566\t4.6\t1\t506 60\n",
      "145\t527\t4.6\t1\t448 79\n",
      "146\t584\t4.6\t1\t496 88\n",
      "147\t490\t4.6\t1\t431 59\n",
      "148\t592\t4.6\t1\t486 106\n",
      "149\t548\t4.6\t1\t429 119\n",
      "150\t899\t4.6\t1\t543 356\n",
      "151\t72200\t4.6\t1\t36351 35849\n",
      "\n",
      "\n",
      "RUN STATISTICS FOR INPUT FILE: /home/sam/data/geoduck_illumina/trimmed/NR013_AD013_S2_L001_R1_001_val_1.fq.gz\n",
      "=============================================\n",
      "309877704 sequences processed in total\n",
      "The length threshold of paired-end sequences gets evaluated later on (in the validation step)\n",
      "\n",
      "Writing report to '/home/sam/data/geoduck_illumina/trimmed/20180129_trimmed_again/NR013_AD013_S2_L001_R2_001_val_2.fq.gz_trimming_report.txt'\n",
      "\n",
      "SUMMARISING RUN PARAMETERS\n",
      "==========================\n",
      "Input filename: /home/sam/data/geoduck_illumina/trimmed/NR013_AD013_S2_L001_R2_001_val_2.fq.gz\n",
      "Trimming mode: paired-end\n",
      "Trim Galore version: 0.4.4\n",
      "Cutadapt version: 1.9.1\n",
      "Quality Phred score cutoff: 20\n",
      "Quality encoding type selected: ASCII+33\n",
      "Adapter sequence: 'AGATCGGAAGAGC' (Illumina TruSeq, Sanger iPCR; auto-detected)\n",
      "Maximum trimming error rate: 0.1 (default)\n",
      "Minimum required adapter overlap (stringency): 1 bp\n",
      "Minimum required sequence length for both reads before a sequence pair gets removed: 20 bp\n",
      "Output file(s) will be GZIP compressed\n",
      "\n",
      "Writing final adapter and quality trimmed output to NR013_AD013_S2_L001_R2_001_val_2_trimmed.fq.gz\n",
      "\n",
      "\n",
      "  >>> Now performing quality (cutoff 20) and adapter trimming in a single pass for the adapter sequence: 'AGATCGGAAGAGC' from file /home/sam/data/geoduck_illumina/trimmed/NR013_AD013_S2_L001_R2_001_val_2.fq.gz <<< \n",
      "10000000 sequences processed\n",
      "20000000 sequences processed\n",
      "30000000 sequences processed\n",
      "40000000 sequences processed\n",
      "50000000 sequences processed\n",
      "60000000 sequences processed\n",
      "70000000 sequences processed\n",
      "80000000 sequences processed\n",
      "90000000 sequences processed\n",
      "100000000 sequences processed\n",
      "110000000 sequences processed\n",
      "120000000 sequences processed\n",
      "130000000 sequences processed\n",
      "140000000 sequences processed\n",
      "150000000 sequences processed\n",
      "160000000 sequences processed\n",
      "170000000 sequences processed\n",
      "180000000 sequences processed\n",
      "190000000 sequences processed\n",
      "200000000 sequences processed\n",
      "210000000 sequences processed\n",
      "220000000 sequences processed\n",
      "230000000 sequences processed\n",
      "240000000 sequences processed\n",
      "250000000 sequences processed\n",
      "260000000 sequences processed\n",
      "270000000 sequences processed\n",
      "280000000 sequences processed\n",
      "290000000 sequences processed\n",
      "300000000 sequences processed\n",
      "This is cutadapt 1.9.1 with Python 2.7.12\n",
      "Command line parameters: -f fastq -e 0.1 -q 20 -O 1 -a AGATCGGAAGAGC /home/sam/data/geoduck_illumina/trimmed/NR013_AD013_S2_L001_R2_001_val_2.fq.gz\n",
      "Trimming 1 adapter with at most 10.0% errors in single-end mode ...\n",
      "Finished in 7839.82 s (25 us/read; 2.37 M reads/minute).\n",
      "\n",
      "=== Summary ===\n",
      "\n",
      "Total reads processed:             309,877,704\n",
      "Reads with adapters:               125,884,928 (40.6%)\n",
      "Reads written (passing filters):   309,877,704 (100.0%)\n",
      "\n",
      "Total basepairs processed: 39,922,240,804 bp\n",
      "Quality-trimmed:              30,422,029 bp (0.1%)\n",
      "Total written (filtered):  39,426,232,820 bp (98.8%)\n",
      "\n",
      "=== Adapter 1 ===\n",
      "\n",
      "Sequence: AGATCGGAAGAGC; Type: regular 3'; Length: 13; Trimmed: 125884928 times.\n",
      "\n",
      "No. of allowed errors:\n",
      "0-9 bp: 0; 10-13 bp: 1\n",
      "\n",
      "Bases preceding removed adapters:\n",
      "  A: 32.1%\n",
      "  C: 23.1%\n",
      "  G: 15.3%\n",
      "  T: 29.6%\n",
      "  none/other: 0.0%\n",
      "\n",
      "Overview of removed sequences\n",
      "length\tcount\texpect\tmax.err\terror counts\n",
      "1\t88845909\t77469426.0\t0\t88845909\n",
      "2\t22125586\t19367356.5\t0\t22125586\n",
      "3\t6391789\t4841839.1\t0\t6391789\n",
      "4\t2247257\t1210459.8\t0\t2247257\n",
      "5\t199820\t302614.9\t0\t199820\n",
      "6\t110118\t75653.7\t0\t110118\n",
      "7\t87861\t18913.4\t0\t87861\n",
      "8\t95299\t4728.4\t0\t95299\n",
      "9\t63978\t1182.1\t0\t59832 4146\n",
      "10\t105395\t295.5\t1\t75720 29675\n",
      "11\t89578\t73.9\t1\t66659 22919\n",
      "12\t173465\t18.5\t1\t138196 35269\n",
      "13\t2389\t4.6\t1\t887 1502\n",
      "14\t113395\t4.6\t1\t87505 25890\n",
      "15\t135634\t4.6\t1\t111210 24424\n",
      "16\t1253\t4.6\t1\t397 856\n",
      "17\t104209\t4.6\t1\t83034 21175\n",
      "18\t56976\t4.6\t1\t45269 11707\n",
      "19\t114574\t4.6\t1\t90053 24521\n",
      "20\t75891\t4.6\t1\t60021 15870\n",
      "21\t61297\t4.6\t1\t49070 12227\n",
      "22\t78310\t4.6\t1\t62289 16021\n",
      "23\t85866\t4.6\t1\t66959 18907\n",
      "24\t167954\t4.6\t1\t130753 37201\n",
      "25\t73002\t4.6\t1\t58910 14092\n",
      "26\t70090\t4.6\t1\t56535 13555\n",
      "27\t47016\t4.6\t1\t35691 11325\n",
      "28\t117783\t4.6\t1\t97694 20089\n",
      "29\t33129\t4.6\t1\t23153 9976\n",
      "30\t112586\t4.6\t1\t92542 20044\n",
      "31\t48002\t4.6\t1\t35771 12231\n",
      "32\t193298\t4.6\t1\t165534 27764\n",
      "33\t87671\t4.6\t1\t68658 19013\n",
      "34\t32056\t4.6\t1\t22676 9380\n",
      "35\t109513\t4.6\t1\t91086 18427\n",
      "36\t39661\t4.6\t1\t30104 9557\n",
      "37\t57789\t4.6\t1\t44412 13377\n",
      "38\t30027\t4.6\t1\t23398 6629\n",
      "39\t42527\t4.6\t1\t32645 9882\n",
      "40\t18198\t4.6\t1\t13575 4623\n",
      "41\t61586\t4.6\t1\t49843 11743\n",
      "42\t83440\t4.6\t1\t68575 14865\n",
      "43\t42319\t4.6\t1\t32603 9716\n",
      "44\t66878\t4.6\t1\t52939 13939\n",
      "45\t115546\t4.6\t1\t95285 20261\n",
      "46\t62010\t4.6\t1\t47751 14259\n",
      "47\t39415\t4.6\t1\t31070 8345\n",
      "48\t114841\t4.6\t1\t94405 20436\n",
      "49\t50075\t4.6\t1\t39582 10493\n",
      "50\t22207\t4.6\t1\t15934 6273\n",
      "51\t93791\t4.6\t1\t76265 17526\n",
      "52\t129976\t4.6\t1\t111177 18799\n",
      "53\t52101\t4.6\t1\t42431 9670\n",
      "54\t70208\t4.6\t1\t57057 13151\n",
      "55\t41534\t4.6\t1\t33481 8053\n",
      "56\t22914\t4.6\t1\t17128 5786\n",
      "57\t38141\t4.6\t1\t29953 8188\n",
      "58\t44206\t4.6\t1\t34293 9913\n",
      "59\t25694\t4.6\t1\t18853 6841\n",
      "60\t18689\t4.6\t1\t13090 5599\n",
      "61\t16750\t4.6\t1\t11632 5118\n",
      "62\t17922\t4.6\t1\t12604 5318\n",
      "63\t19917\t4.6\t1\t14689 5228\n",
      "64\t30669\t4.6\t1\t23972 6697\n",
      "65\t39466\t4.6\t1\t31318 8148\n",
      "66\t43956\t4.6\t1\t34720 9236\n",
      "67\t45097\t4.6\t1\t35359 9738\n",
      "68\t45768\t4.6\t1\t35779 9989\n",
      "69\t45954\t4.6\t1\t35668 10286\n",
      "70\t46047\t4.6\t1\t35770 10277\n",
      "71\t45745\t4.6\t1\t35257 10488\n",
      "72\t45607\t4.6\t1\t35117 10490\n",
      "73\t45024\t4.6\t1\t34813 10211\n",
      "74\t44498\t4.6\t1\t34307 10191\n",
      "75\t44017\t4.6\t1\t34003 10014\n",
      "76\t43540\t4.6\t1\t33431 10109\n",
      "77\t43090\t4.6\t1\t33125 9965\n",
      "78\t42637\t4.6\t1\t32752 9885\n",
      "79\t41853\t4.6\t1\t32164 9689\n",
      "80\t41286\t4.6\t1\t31721 9565\n",
      "81\t40929\t4.6\t1\t31441 9488\n",
      "82\t40505\t4.6\t1\t31156 9349\n",
      "83\t39595\t4.6\t1\t30373 9222\n",
      "84\t39003\t4.6\t1\t29871 9132\n",
      "85\t38856\t4.6\t1\t29840 9016\n",
      "86\t38308\t4.6\t1\t29168 9140\n",
      "87\t37828\t4.6\t1\t28974 8854\n",
      "88\t37411\t4.6\t1\t28579 8832\n",
      "89\t36119\t4.6\t1\t27688 8431\n",
      "90\t35621\t4.6\t1\t27246 8375\n",
      "91\t35130\t4.6\t1\t26933 8197\n",
      "92\t34667\t4.6\t1\t26480 8187\n",
      "93\t33625\t4.6\t1\t25771 7854\n",
      "94\t33191\t4.6\t1\t25409 7782\n",
      "95\t32138\t4.6\t1\t24554 7584\n",
      "96\t32105\t4.6\t1\t24550 7555\n",
      "97\t31392\t4.6\t1\t24148 7244\n",
      "98\t30486\t4.6\t1\t23462 7024\n",
      "99\t29633\t4.6\t1\t22786 6847\n",
      "100\t28698\t4.6\t1\t22036 6662\n",
      "101\t28001\t4.6\t1\t21491 6510\n",
      "102\t27082\t4.6\t1\t20760 6322\n",
      "103\t26784\t4.6\t1\t20505 6279\n",
      "104\t25275\t4.6\t1\t19359 5916\n",
      "105\t24641\t4.6\t1\t18707 5934\n",
      "106\t23433\t4.6\t1\t17836 5597\n",
      "107\t22242\t4.6\t1\t16750 5492\n",
      "108\t21167\t4.6\t1\t15837 5330\n",
      "109\t20140\t4.6\t1\t15032 5108\n",
      "110\t19354\t4.6\t1\t14351 5003\n",
      "111\t18430\t4.6\t1\t13727 4703\n",
      "112\t17600\t4.6\t1\t13157 4443\n",
      "113\t16947\t4.6\t1\t12800 4147\n",
      "114\t16325\t4.6\t1\t12306 4019\n",
      "115\t15895\t4.6\t1\t11983 3912\n",
      "116\t15307\t4.6\t1\t11534 3773\n",
      "117\t14514\t4.6\t1\t10941 3573\n",
      "118\t13913\t4.6\t1\t10507 3406\n",
      "119\t13245\t4.6\t1\t10078 3167\n",
      "120\t12231\t4.6\t1\t9245 2986\n",
      "121\t11796\t4.6\t1\t8977 2819\n",
      "122\t10638\t4.6\t1\t8118 2520\n",
      "123\t10402\t4.6\t1\t7982 2420\n",
      "124\t9446\t4.6\t1\t7197 2249\n",
      "125\t9015\t4.6\t1\t6866 2149\n",
      "126\t8486\t4.6\t1\t6462 2024\n",
      "127\t7804\t4.6\t1\t6024 1780\n",
      "128\t6749\t4.6\t1\t5384 1365\n",
      "129\t6747\t4.6\t1\t5588 1159\n",
      "130\t5261\t4.6\t1\t4337 924\n",
      "131\t5109\t4.6\t1\t4384 725\n",
      "132\t3835\t4.6\t1\t3252 583\n",
      "133\t3353\t4.6\t1\t2896 457\n",
      "134\t2940\t4.6\t1\t2598 342\n",
      "135\t2590\t4.6\t1\t2262 328\n",
      "136\t2123\t4.6\t1\t1857 266\n",
      "137\t1846\t4.6\t1\t1629 217\n",
      "138\t1530\t4.6\t1\t1332 198\n",
      "139\t1244\t4.6\t1\t1106 138\n",
      "140\t1074\t4.6\t1\t916 158\n",
      "141\t858\t4.6\t1\t679 179\n",
      "142\t851\t4.6\t1\t632 219\n",
      "143\t728\t4.6\t1\t535 193\n",
      "144\t621\t4.6\t1\t433 188\n",
      "145\t543\t4.6\t1\t369 174\n",
      "146\t592\t4.6\t1\t362 230\n",
      "147\t522\t4.6\t1\t326 196\n",
      "148\t596\t4.6\t1\t302 294\n",
      "149\t511\t4.6\t1\t299 212\n",
      "150\t928\t4.6\t1\t667 261\n",
      "151\t37859\t4.6\t1\t36758 1101\n",
      "\n",
      "\n",
      "RUN STATISTICS FOR INPUT FILE: /home/sam/data/geoduck_illumina/trimmed/NR013_AD013_S2_L001_R2_001_val_2.fq.gz\n",
      "=============================================\n",
      "309877704 sequences processed in total\n",
      "The length threshold of paired-end sequences gets evaluated later on (in the validation step)\n",
      "\n",
      "Validate paired-end files NR013_AD013_S2_L001_R1_001_val_1_trimmed.fq.gz and NR013_AD013_S2_L001_R2_001_val_2_trimmed.fq.gz\n",
      "file_1: NR013_AD013_S2_L001_R1_001_val_1_trimmed.fq.gz, file_2: NR013_AD013_S2_L001_R2_001_val_2_trimmed.fq.gz\n",
      "\n",
      "\n",
      ">>>>> Now validing the length of the 2 paired-end infiles: NR013_AD013_S2_L001_R1_001_val_1_trimmed.fq.gz and NR013_AD013_S2_L001_R2_001_val_2_trimmed.fq.gz <<<<<\n",
      "Writing validated paired-end read 1 reads to NR013_AD013_S2_L001_R1_001_val_1_val_1.fq.gz\n",
      "Writing validated paired-end read 2 reads to NR013_AD013_S2_L001_R2_001_val_2_val_2.fq.gz\n",
      "\n",
      "Total number of sequences analysed: 309877704\n",
      "\n",
      "Number of sequence pairs removed because at least one read was shorter than the length cutoff (20 bp): 827042 (0.27%)\n",
      "\n",
      "Deleting both intermediate output files NR013_AD013_S2_L001_R1_001_val_1_trimmed.fq.gz and NR013_AD013_S2_L001_R2_001_val_2_trimmed.fq.gz\n",
      "\n",
      "====================================================================================================\n",
      "\n",
      "Writing report to '/home/sam/data/geoduck_illumina/trimmed/20180129_trimmed_again/NR013_AD013_S2_L002_R1_001_val_1.fq.gz_trimming_report.txt'\n",
      "\n",
      "SUMMARISING RUN PARAMETERS\n",
      "==========================\n",
      "Input filename: /home/sam/data/geoduck_illumina/trimmed/NR013_AD013_S2_L002_R1_001_val_1.fq.gz\n",
      "Trimming mode: paired-end\n",
      "Trim Galore version: 0.4.4\n",
      "Cutadapt version: 1.9.1\n",
      "Quality Phred score cutoff: 20\n",
      "Quality encoding type selected: ASCII+33\n",
      "Adapter sequence: 'AGATCGGAAGAGC' (Illumina TruSeq, Sanger iPCR; auto-detected)\n",
      "Maximum trimming error rate: 0.1 (default)\n",
      "Minimum required adapter overlap (stringency): 1 bp\n",
      "Minimum required sequence length for both reads before a sequence pair gets removed: 20 bp\n",
      "Output file(s) will be GZIP compressed\n",
      "\n",
      "Writing final adapter and quality trimmed output to NR013_AD013_S2_L002_R1_001_val_1_trimmed.fq.gz\n",
      "\n",
      "\n",
      "  >>> Now performing quality (cutoff 20) and adapter trimming in a single pass for the adapter sequence: 'AGATCGGAAGAGC' from file /home/sam/data/geoduck_illumina/trimmed/NR013_AD013_S2_L002_R1_001_val_1.fq.gz <<< \n",
      "10000000 sequences processed\n",
      "20000000 sequences processed\n",
      "30000000 sequences processed\n",
      "40000000 sequences processed\n",
      "50000000 sequences processed\n",
      "60000000 sequences processed\n",
      "70000000 sequences processed\n",
      "80000000 sequences processed\n",
      "90000000 sequences processed\n",
      "100000000 sequences processed\n",
      "110000000 sequences processed\n",
      "120000000 sequences processed\n",
      "130000000 sequences processed\n",
      "140000000 sequences processed\n",
      "150000000 sequences processed\n",
      "160000000 sequences processed\n",
      "170000000 sequences processed\n",
      "180000000 sequences processed\n",
      "190000000 sequences processed\n",
      "200000000 sequences processed\n",
      "210000000 sequences processed\n",
      "220000000 sequences processed\n",
      "230000000 sequences processed\n",
      "240000000 sequences processed\n",
      "250000000 sequences processed\n",
      "260000000 sequences processed\n",
      "270000000 sequences processed\n",
      "280000000 sequences processed\n",
      "290000000 sequences processed\n",
      "300000000 sequences processed\n",
      "310000000 sequences processed\n",
      "320000000 sequences processed\n",
      "330000000 sequences processed\n",
      "340000000 sequences processed\n",
      "350000000 sequences processed\n",
      "360000000 sequences processed\n",
      "370000000 sequences processed\n",
      "This is cutadapt 1.9.1 with Python 2.7.12\n",
      "Command line parameters: -f fastq -e 0.1 -q 20 -O 1 -a AGATCGGAAGAGC /home/sam/data/geoduck_illumina/trimmed/NR013_AD013_S2_L002_R1_001_val_1.fq.gz\n",
      "Trimming 1 adapter with at most 10.0% errors in single-end mode ...\n",
      "Finished in 9397.15 s (25 us/read; 2.42 M reads/minute).\n",
      "\n",
      "=== Summary ===\n",
      "\n",
      "Total reads processed:             379,535,476\n",
      "Reads with adapters:               157,413,392 (41.5%)\n",
      "Reads written (passing filters):   379,535,476 (100.0%)\n",
      "\n",
      "Total basepairs processed: 48,465,312,024 bp\n",
      "Quality-trimmed:              27,720,181 bp (0.1%)\n",
      "Total written (filtered):  47,689,069,992 bp (98.4%)\n",
      "\n",
      "=== Adapter 1 ===\n",
      "\n",
      "Sequence: AGATCGGAAGAGC; Type: regular 3'; Length: 13; Trimmed: 157413392 times.\n",
      "\n",
      "No. of allowed errors:\n",
      "0-9 bp: 0; 10-13 bp: 1\n",
      "\n",
      "Bases preceding removed adapters:\n",
      "  A: 32.2%\n",
      "  C: 23.6%\n",
      "  G: 14.5%\n",
      "  T: 29.6%\n",
      "  none/other: 0.1%\n",
      "\n",
      "Overview of removed sequences\n",
      "length\tcount\texpect\tmax.err\terror counts\n",
      "1\t110667740\t94883869.0\t0\t110667740\n",
      "2\t26663149\t23720967.2\t0\t26663149\n",
      "3\t7393392\t5930241.8\t0\t7393392\n",
      "4\t2653451\t1482560.5\t0\t2653451\n",
      "5\t254168\t370640.1\t0\t254168\n",
      "6\t128511\t92660.0\t0\t128511\n",
      "7\t88935\t23165.0\t0\t88935\n",
      "8\t105904\t5791.3\t0\t105904\n",
      "9\t92612\t1447.8\t0\t88170 4442\n",
      "10\t114990\t362.0\t1\t85274 29716\n",
      "11\t123270\t90.5\t1\t91224 32046\n",
      "12\t208554\t22.6\t1\t169873 38681\n",
      "13\t11285\t5.7\t1\t6138 5147\n",
      "14\t194395\t5.7\t1\t161652 32743\n",
      "15\t8680\t5.7\t1\t4299 4381\n",
      "16\t223910\t5.7\t1\t178592 45318\n",
      "17\t15102\t5.7\t1\t8422 6680\n",
      "18\t154776\t5.7\t1\t123797 30979\n",
      "19\t313056\t5.7\t1\t268766 44290\n",
      "20\t3143\t5.7\t1\t1583 1560\n",
      "21\t3876\t5.7\t1\t1369 2507\n",
      "22\t8827\t5.7\t1\t3921 4906\n",
      "23\t127107\t5.7\t1\t99437 27670\n",
      "24\t274885\t5.7\t1\t228223 46662\n",
      "25\t12392\t5.7\t1\t6883 5509\n",
      "26\t115765\t5.7\t1\t90881 24884\n",
      "27\t92682\t5.7\t1\t78305 14377\n",
      "28\t4103\t5.7\t1\t1994 2109\n",
      "29\t103986\t5.7\t1\t82555 21431\n",
      "30\t131306\t5.7\t1\t107052 24254\n",
      "31\t162960\t5.7\t1\t133400 29560\n",
      "32\t14793\t5.7\t1\t8688 6105\n",
      "33\t194807\t5.7\t1\t157828 36979\n",
      "34\t4998\t5.7\t1\t2683 2315\n",
      "35\t105963\t5.7\t1\t82175 23788\n",
      "36\t137630\t5.7\t1\t110316 27314\n",
      "37\t159200\t5.7\t1\t129691 29509\n",
      "38\t6660\t5.7\t1\t3982 2678\n",
      "39\t127764\t5.7\t1\t98518 29246\n",
      "40\t199028\t5.7\t1\t153865 45163\n",
      "41\t9971\t5.7\t1\t5999 3972\n",
      "42\t136600\t5.7\t1\t101299 35301\n",
      "43\t178520\t5.7\t1\t129574 48946\n",
      "44\t262367\t5.7\t1\t204880 57487\n",
      "45\t52027\t5.7\t1\t41943 10084\n",
      "46\t73660\t5.7\t1\t52825 20835\n",
      "47\t8441\t5.7\t1\t5143 3298\n",
      "48\t174363\t5.7\t1\t139828 34535\n",
      "49\t108253\t5.7\t1\t83992 24261\n",
      "50\t108727\t5.7\t1\t83186 25541\n",
      "51\t161423\t5.7\t1\t124375 37048\n",
      "52\t175183\t5.7\t1\t140943 34240\n",
      "53\t74326\t5.7\t1\t61229 13097\n",
      "54\t16107\t5.7\t1\t11671 4436\n",
      "55\t194176\t5.7\t1\t167170 27006\n",
      "56\t185187\t5.7\t1\t155240 29947\n",
      "57\t17080\t5.7\t1\t13135 3945\n",
      "58\t31574\t5.7\t1\t22573 9001\n",
      "59\t182957\t5.7\t1\t156639 26318\n",
      "60\t10462\t5.7\t1\t6373 4089\n",
      "61\t24520\t5.7\t1\t15594 8926\n",
      "62\t128782\t5.7\t1\t109288 19494\n",
      "63\t5151\t5.7\t1\t2694 2457\n",
      "64\t13518\t5.7\t1\t8092 5426\n",
      "65\t51853\t5.7\t1\t40861 10992\n",
      "66\t103903\t5.7\t1\t87611 16292\n",
      "67\t66440\t5.7\t1\t53152 13288\n",
      "68\t88880\t5.7\t1\t72897 15983\n",
      "69\t82619\t5.7\t1\t66722 15897\n",
      "70\t96138\t5.7\t1\t77885 18253\n",
      "71\t25150\t5.7\t1\t16131 9019\n",
      "72\t27241\t5.7\t1\t19801 7440\n",
      "73\t45344\t5.7\t1\t36516 8828\n",
      "74\t57484\t5.7\t1\t47405 10079\n",
      "75\t64067\t5.7\t1\t52679 11388\n",
      "76\t68089\t5.7\t1\t55772 12317\n",
      "77\t70462\t5.7\t1\t57543 12919\n",
      "78\t72065\t5.7\t1\t58687 13378\n",
      "79\t72745\t5.7\t1\t58952 13793\n",
      "80\t72785\t5.7\t1\t58704 14081\n",
      "81\t73449\t5.7\t1\t59389 14060\n",
      "82\t73456\t5.7\t1\t59165 14291\n",
      "83\t72939\t5.7\t1\t58314 14625\n",
      "84\t72785\t5.7\t1\t58258 14527\n",
      "85\t72185\t5.7\t1\t57856 14329\n",
      "86\t71613\t5.7\t1\t57074 14539\n",
      "87\t70864\t5.7\t1\t56677 14187\n",
      "88\t71145\t5.7\t1\t56961 14184\n",
      "89\t69194\t5.7\t1\t55323 13871\n",
      "90\t69287\t5.7\t1\t55262 14025\n",
      "91\t68129\t5.7\t1\t54295 13834\n",
      "92\t67451\t5.7\t1\t53886 13565\n",
      "93\t65864\t5.7\t1\t52572 13292\n",
      "94\t64955\t5.7\t1\t51832 13123\n",
      "95\t64272\t5.7\t1\t51150 13122\n",
      "96\t63370\t5.7\t1\t50414 12956\n",
      "97\t61971\t5.7\t1\t49449 12522\n",
      "98\t61411\t5.7\t1\t48937 12474\n",
      "99\t60080\t5.7\t1\t47918 12162\n",
      "100\t59076\t5.7\t1\t46891 12185\n",
      "101\t57691\t5.7\t1\t45837 11854\n",
      "102\t56341\t5.7\t1\t44779 11562\n",
      "103\t54946\t5.7\t1\t43824 11122\n",
      "104\t53301\t5.7\t1\t42151 11150\n",
      "105\t51774\t5.7\t1\t41142 10632\n",
      "106\t50125\t5.7\t1\t39682 10443\n",
      "107\t48776\t5.7\t1\t38650 10126\n",
      "108\t47554\t5.7\t1\t37248 10306\n",
      "109\t45662\t5.7\t1\t36058 9604\n",
      "110\t43668\t5.7\t1\t34299 9369\n",
      "111\t42524\t5.7\t1\t33304 9220\n",
      "112\t41006\t5.7\t1\t32171 8835\n",
      "113\t39805\t5.7\t1\t31283 8522\n",
      "114\t38467\t5.7\t1\t30150 8317\n",
      "115\t36917\t5.7\t1\t28981 7936\n",
      "116\t35474\t5.7\t1\t27902 7572\n",
      "117\t33548\t5.7\t1\t26390 7158\n",
      "118\t32583\t5.7\t1\t25558 7025\n",
      "119\t31091\t5.7\t1\t24329 6762\n",
      "120\t29303\t5.7\t1\t22921 6382\n",
      "121\t27728\t5.7\t1\t21825 5903\n",
      "122\t25972\t5.7\t1\t20442 5530\n",
      "123\t24465\t5.7\t1\t19214 5251\n",
      "124\t22898\t5.7\t1\t17993 4905\n",
      "125\t21621\t5.7\t1\t16995 4626\n",
      "126\t20014\t5.7\t1\t15694 4320\n",
      "127\t18987\t5.7\t1\t15019 3968\n",
      "128\t16876\t5.7\t1\t13491 3385\n",
      "129\t16593\t5.7\t1\t13625 2968\n",
      "130\t13421\t5.7\t1\t10974 2447\n",
      "131\t12382\t5.7\t1\t10375 2007\n",
      "132\t9875\t5.7\t1\t8394 1481\n",
      "133\t8952\t5.7\t1\t7759 1193\n",
      "134\t7501\t5.7\t1\t6524 977\n",
      "135\t6534\t5.7\t1\t5675 859\n",
      "136\t5217\t5.7\t1\t4621 596\n",
      "137\t4587\t5.7\t1\t4093 494\n",
      "138\t3588\t5.7\t1\t3198 390\n",
      "139\t2812\t5.7\t1\t2534 278\n",
      "140\t2065\t5.7\t1\t1837 228\n",
      "141\t1568\t5.7\t1\t1398 170\n",
      "142\t1250\t5.7\t1\t1114 136\n",
      "143\t1100\t5.7\t1\t961 139\n",
      "144\t839\t5.7\t1\t710 129\n",
      "145\t842\t5.7\t1\t725 117\n",
      "146\t755\t5.7\t1\t637 118\n",
      "147\t688\t5.7\t1\t589 99\n",
      "148\t799\t5.7\t1\t669 130\n",
      "149\t662\t5.7\t1\t532 130\n",
      "150\t933\t5.7\t1\t583 350\n",
      "151\t101421\t5.7\t1\t50281 51140\n",
      "\n",
      "\n",
      "RUN STATISTICS FOR INPUT FILE: /home/sam/data/geoduck_illumina/trimmed/NR013_AD013_S2_L002_R1_001_val_1.fq.gz\n",
      "=============================================\n",
      "379535476 sequences processed in total\n",
      "The length threshold of paired-end sequences gets evaluated later on (in the validation step)\n",
      "\n",
      "Writing report to '/home/sam/data/geoduck_illumina/trimmed/20180129_trimmed_again/NR013_AD013_S2_L002_R2_001_val_2.fq.gz_trimming_report.txt'\n",
      "\n",
      "SUMMARISING RUN PARAMETERS\n",
      "==========================\n",
      "Input filename: /home/sam/data/geoduck_illumina/trimmed/NR013_AD013_S2_L002_R2_001_val_2.fq.gz\n",
      "Trimming mode: paired-end\n",
      "Trim Galore version: 0.4.4\n",
      "Cutadapt version: 1.9.1\n",
      "Quality Phred score cutoff: 20\n",
      "Quality encoding type selected: ASCII+33\n",
      "Adapter sequence: 'AGATCGGAAGAGC' (Illumina TruSeq, Sanger iPCR; auto-detected)\n",
      "Maximum trimming error rate: 0.1 (default)\n",
      "Minimum required adapter overlap (stringency): 1 bp\n",
      "Minimum required sequence length for both reads before a sequence pair gets removed: 20 bp\n",
      "Output file(s) will be GZIP compressed\n",
      "\n",
      "Writing final adapter and quality trimmed output to NR013_AD013_S2_L002_R2_001_val_2_trimmed.fq.gz\n",
      "\n",
      "\n",
      "  >>> Now performing quality (cutoff 20) and adapter trimming in a single pass for the adapter sequence: 'AGATCGGAAGAGC' from file /home/sam/data/geoduck_illumina/trimmed/NR013_AD013_S2_L002_R2_001_val_2.fq.gz <<< \n",
      "10000000 sequences processed\n",
      "20000000 sequences processed\n",
      "30000000 sequences processed\n",
      "40000000 sequences processed\n",
      "50000000 sequences processed\n",
      "60000000 sequences processed\n",
      "70000000 sequences processed\n",
      "80000000 sequences processed\n",
      "90000000 sequences processed\n",
      "100000000 sequences processed\n",
      "110000000 sequences processed\n",
      "120000000 sequences processed\n",
      "130000000 sequences processed\n",
      "140000000 sequences processed\n",
      "150000000 sequences processed\n",
      "160000000 sequences processed\n",
      "170000000 sequences processed\n",
      "180000000 sequences processed\n",
      "190000000 sequences processed\n",
      "200000000 sequences processed\n",
      "210000000 sequences processed\n",
      "220000000 sequences processed\n",
      "230000000 sequences processed\n",
      "240000000 sequences processed\n",
      "250000000 sequences processed\n",
      "260000000 sequences processed\n",
      "270000000 sequences processed\n",
      "280000000 sequences processed\n",
      "290000000 sequences processed\n",
      "300000000 sequences processed\n",
      "310000000 sequences processed\n",
      "320000000 sequences processed\n",
      "330000000 sequences processed\n",
      "340000000 sequences processed\n",
      "350000000 sequences processed\n",
      "360000000 sequences processed\n",
      "370000000 sequences processed\n",
      "This is cutadapt 1.9.1 with Python 2.7.12\n",
      "Command line parameters: -f fastq -e 0.1 -q 20 -O 1 -a AGATCGGAAGAGC /home/sam/data/geoduck_illumina/trimmed/NR013_AD013_S2_L002_R2_001_val_2.fq.gz\n",
      "Trimming 1 adapter with at most 10.0% errors in single-end mode ...\n",
      "Finished in 9482.79 s (25 us/read; 2.40 M reads/minute).\n",
      "\n",
      "=== Summary ===\n",
      "\n",
      "Total reads processed:             379,535,476\n",
      "Reads with adapters:               158,202,642 (41.7%)\n",
      "Reads written (passing filters):   379,535,476 (100.0%)\n",
      "\n",
      "Total basepairs processed: 49,672,679,182 bp\n",
      "Quality-trimmed:              30,573,885 bp (0.1%)\n",
      "Total written (filtered):  48,791,766,215 bp (98.2%)\n",
      "\n",
      "=== Adapter 1 ===\n",
      "\n",
      "Sequence: AGATCGGAAGAGC; Type: regular 3'; Length: 13; Trimmed: 158202642 times.\n",
      "\n",
      "No. of allowed errors:\n",
      "0-9 bp: 0; 10-13 bp: 1\n",
      "\n",
      "Bases preceding removed adapters:\n",
      "  A: 32.0%\n",
      "  C: 22.9%\n",
      "  G: 15.3%\n",
      "  T: 29.8%\n",
      "  none/other: 0.0%\n",
      "\n",
      "Overview of removed sequences\n",
      "length\tcount\texpect\tmax.err\terror counts\n",
      "1\t109580322\t94883869.0\t0\t109580322\n",
      "2\t26576548\t23720967.2\t0\t26576548\n",
      "3\t7597769\t5930241.8\t0\t7597769\n",
      "4\t2680993\t1482560.5\t0\t2680993\n",
      "5\t226898\t370640.1\t0\t226898\n",
      "6\t150660\t92660.0\t0\t150660\n",
      "7\t122142\t23165.0\t0\t122142\n",
      "8\t131733\t5791.3\t0\t131733\n",
      "9\t90844\t1447.8\t0\t86022 4822\n",
      "10\t150905\t362.0\t1\t107502 43403\n",
      "11\t135349\t90.5\t1\t97607 37742\n",
      "12\t252946\t22.6\t1\t199121 53825\n",
      "13\t3530\t5.7\t1\t1331 2199\n",
      "14\t168638\t5.7\t1\t128925 39713\n",
      "15\t195609\t5.7\t1\t158466 37143\n",
      "16\t1950\t5.7\t1\t608 1342\n",
      "17\t154612\t5.7\t1\t121576 33036\n",
      "18\t83683\t5.7\t1\t65735 17948\n",
      "19\t170549\t5.7\t1\t131905 38644\n",
      "20\t113819\t5.7\t1\t88764 25055\n",
      "21\t99079\t5.7\t1\t77374 21705\n",
      "22\t121474\t5.7\t1\t94946 26528\n",
      "23\t131152\t5.7\t1\t101603 29549\n",
      "24\t248335\t5.7\t1\t190265 58070\n",
      "25\t123183\t5.7\t1\t98568 24615\n",
      "26\t116509\t5.7\t1\t93505 23004\n",
      "27\t85593\t5.7\t1\t63827 21766\n",
      "28\t187504\t5.7\t1\t154503 33001\n",
      "29\t63599\t5.7\t1\t44467 19132\n",
      "30\t187277\t5.7\t1\t152585 34692\n",
      "31\t82799\t5.7\t1\t60937 21862\n",
      "32\t355185\t5.7\t1\t306261 48924\n",
      "33\t158641\t5.7\t1\t124646 33995\n",
      "34\t64822\t5.7\t1\t45842 18980\n",
      "35\t195418\t5.7\t1\t164137 31281\n",
      "36\t58936\t5.7\t1\t42343 16593\n",
      "37\t108180\t5.7\t1\t83522 24658\n",
      "38\t35568\t5.7\t1\t25724 9844\n",
      "39\t64341\t5.7\t1\t47554 16787\n",
      "40\t20127\t5.7\t1\t13603 6524\n",
      "41\t106196\t5.7\t1\t85625 20571\n",
      "42\t153301\t5.7\t1\t126849 26452\n",
      "43\t68625\t5.7\t1\t52544 16081\n",
      "44\t121927\t5.7\t1\t96858 25069\n",
      "45\t262336\t5.7\t1\t224010 38326\n",
      "46\t116328\t5.7\t1\t90525 25803\n",
      "47\t58043\t5.7\t1\t44637 13406\n",
      "48\t248028\t5.7\t1\t209929 38099\n",
      "49\t75167\t5.7\t1\t58530 16637\n",
      "50\t31491\t5.7\t1\t21928 9563\n",
      "51\t198900\t5.7\t1\t165930 32970\n",
      "52\t287281\t5.7\t1\t253422 33859\n",
      "53\t78856\t5.7\t1\t63865 14991\n",
      "54\t119169\t5.7\t1\t97179 21990\n",
      "55\t55266\t5.7\t1\t43255 12011\n",
      "56\t35422\t5.7\t1\t25390 10032\n",
      "57\t60972\t5.7\t1\t46938 14034\n",
      "58\t72809\t5.7\t1\t56312 16497\n",
      "59\t38555\t5.7\t1\t27522 11033\n",
      "60\t31487\t5.7\t1\t21444 10043\n",
      "61\t31013\t5.7\t1\t21204 9809\n",
      "62\t31200\t5.7\t1\t21560 9640\n",
      "63\t34441\t5.7\t1\t24740 9701\n",
      "64\t55775\t5.7\t1\t44093 11682\n",
      "65\t78736\t5.7\t1\t63960 14776\n",
      "66\t90620\t5.7\t1\t73458 17162\n",
      "67\t95517\t5.7\t1\t76780 18737\n",
      "68\t97188\t5.7\t1\t77764 19424\n",
      "69\t99204\t5.7\t1\t79345 19859\n",
      "70\t99963\t5.7\t1\t79546 20417\n",
      "71\t100406\t5.7\t1\t79933 20473\n",
      "72\t99798\t5.7\t1\t79578 20220\n",
      "73\t99683\t5.7\t1\t79316 20367\n",
      "74\t99331\t5.7\t1\t78979 20352\n",
      "75\t98621\t5.7\t1\t78425 20196\n",
      "76\t97932\t5.7\t1\t77457 20475\n",
      "77\t97135\t5.7\t1\t77151 19984\n",
      "78\t97028\t5.7\t1\t77056 19972\n",
      "79\t96260\t5.7\t1\t76466 19794\n",
      "80\t94810\t5.7\t1\t75454 19356\n",
      "81\t94602\t5.7\t1\t75244 19358\n",
      "82\t94004\t5.7\t1\t74771 19233\n",
      "83\t92945\t5.7\t1\t73823 19122\n",
      "84\t92419\t5.7\t1\t73182 19237\n",
      "85\t91632\t5.7\t1\t72790 18842\n",
      "86\t91035\t5.7\t1\t72317 18718\n",
      "87\t90050\t5.7\t1\t71548 18502\n",
      "88\t89894\t5.7\t1\t71506 18388\n",
      "89\t87725\t5.7\t1\t69845 17880\n",
      "90\t87185\t5.7\t1\t69399 17786\n",
      "91\t84618\t5.7\t1\t66960 17658\n",
      "92\t84186\t5.7\t1\t66945 17241\n",
      "93\t82900\t5.7\t1\t65834 17066\n",
      "94\t81370\t5.7\t1\t64473 16897\n",
      "95\t80429\t5.7\t1\t63919 16510\n",
      "96\t79457\t5.7\t1\t63127 16330\n",
      "97\t78199\t5.7\t1\t62179 16020\n",
      "98\t77342\t5.7\t1\t61569 15773\n",
      "99\t75264\t5.7\t1\t59801 15463\n",
      "100\t74042\t5.7\t1\t58980 15062\n",
      "101\t72181\t5.7\t1\t57310 14871\n",
      "102\t70827\t5.7\t1\t56161 14666\n",
      "103\t69379\t5.7\t1\t55477 13902\n",
      "104\t66924\t5.7\t1\t53369 13555\n",
      "105\t64860\t5.7\t1\t51608 13252\n",
      "106\t63171\t5.7\t1\t50167 13004\n",
      "107\t60966\t5.7\t1\t48666 12300\n",
      "108\t59572\t5.7\t1\t47230 12342\n",
      "109\t57458\t5.7\t1\t45484 11974\n",
      "110\t54410\t5.7\t1\t43114 11296\n",
      "111\t53126\t5.7\t1\t41972 11154\n",
      "112\t51049\t5.7\t1\t40361 10688\n",
      "113\t49939\t5.7\t1\t39585 10354\n",
      "114\t48143\t5.7\t1\t38288 9855\n",
      "115\t46204\t5.7\t1\t36627 9577\n",
      "116\t44751\t5.7\t1\t35508 9243\n",
      "117\t42469\t5.7\t1\t33654 8815\n",
      "118\t41251\t5.7\t1\t32738 8513\n",
      "119\t39131\t5.7\t1\t31188 7943\n",
      "120\t37000\t5.7\t1\t29225 7775\n",
      "121\t34921\t5.7\t1\t27767 7154\n",
      "122\t32786\t5.7\t1\t26075 6711\n",
      "123\t30986\t5.7\t1\t24656 6330\n",
      "124\t29117\t5.7\t1\t23102 6015\n",
      "125\t27519\t5.7\t1\t21859 5660\n",
      "126\t25495\t5.7\t1\t20373 5122\n",
      "127\t23925\t5.7\t1\t19306 4619\n",
      "128\t20431\t5.7\t1\t16762 3669\n",
      "129\t19497\t5.7\t1\t16497 3000\n",
      "130\t15325\t5.7\t1\t13045 2280\n",
      "131\t13886\t5.7\t1\t12078 1808\n",
      "132\t10761\t5.7\t1\t9519 1242\n",
      "133\t9402\t5.7\t1\t8413 989\n",
      "134\t7873\t5.7\t1\t7096 777\n",
      "135\t6890\t5.7\t1\t6211 679\n",
      "136\t5416\t5.7\t1\t4918 498\n",
      "137\t4724\t5.7\t1\t4317 407\n",
      "138\t3681\t5.7\t1\t3347 334\n",
      "139\t2838\t5.7\t1\t2579 259\n",
      "140\t2088\t5.7\t1\t1837 251\n",
      "141\t1592\t5.7\t1\t1358 234\n",
      "142\t1256\t5.7\t1\t990 266\n",
      "143\t1102\t5.7\t1\t852 250\n",
      "144\t825\t5.7\t1\t603 222\n",
      "145\t848\t5.7\t1\t562 286\n",
      "146\t752\t5.7\t1\t465 287\n",
      "147\t697\t5.7\t1\t449 248\n",
      "148\t750\t5.7\t1\t456 294\n",
      "149\t606\t5.7\t1\t356 250\n",
      "150\t664\t5.7\t1\t440 224\n",
      "151\t51839\t5.7\t1\t50778 1061\n",
      "\n",
      "\n",
      "RUN STATISTICS FOR INPUT FILE: /home/sam/data/geoduck_illumina/trimmed/NR013_AD013_S2_L002_R2_001_val_2.fq.gz\n",
      "=============================================\n",
      "379535476 sequences processed in total\n",
      "The length threshold of paired-end sequences gets evaluated later on (in the validation step)\n",
      "\n",
      "Validate paired-end files NR013_AD013_S2_L002_R1_001_val_1_trimmed.fq.gz and NR013_AD013_S2_L002_R2_001_val_2_trimmed.fq.gz\n",
      "file_1: NR013_AD013_S2_L002_R1_001_val_1_trimmed.fq.gz, file_2: NR013_AD013_S2_L002_R2_001_val_2_trimmed.fq.gz\n",
      "\n",
      "\n",
      ">>>>> Now validing the length of the 2 paired-end infiles: NR013_AD013_S2_L002_R1_001_val_1_trimmed.fq.gz and NR013_AD013_S2_L002_R2_001_val_2_trimmed.fq.gz <<<<<\n",
      "Writing validated paired-end read 1 reads to NR013_AD013_S2_L002_R1_001_val_1_val_1.fq.gz\n",
      "Writing validated paired-end read 2 reads to NR013_AD013_S2_L002_R2_001_val_2_val_2.fq.gz\n",
      "\n",
      "Total number of sequences analysed: 379535476\n",
      "\n",
      "Number of sequence pairs removed because at least one read was shorter than the length cutoff (20 bp): 942696 (0.25%)\n",
      "\n",
      "Deleting both intermediate output files NR013_AD013_S2_L002_R1_001_val_1_trimmed.fq.gz and NR013_AD013_S2_L002_R2_001_val_2_trimmed.fq.gz\n",
      "\n",
      "====================================================================================================\n",
      "\n",
      "Writing report to '/home/sam/data/geoduck_illumina/trimmed/20180129_trimmed_again/NR014_AD014_S5_L001_R1_001_val_1.fq.gz_trimming_report.txt'\n",
      "\n",
      "SUMMARISING RUN PARAMETERS\n",
      "==========================\n",
      "Input filename: /home/sam/data/geoduck_illumina/trimmed/NR014_AD014_S5_L001_R1_001_val_1.fq.gz\n",
      "Trimming mode: paired-end\n",
      "Trim Galore version: 0.4.4\n",
      "Cutadapt version: 1.9.1\n",
      "Quality Phred score cutoff: 20\n",
      "Quality encoding type selected: ASCII+33\n",
      "Adapter sequence: 'AGATCGGAAGAGC' (Illumina TruSeq, Sanger iPCR; auto-detected)\n",
      "Maximum trimming error rate: 0.1 (default)\n",
      "Minimum required adapter overlap (stringency): 1 bp\n",
      "Minimum required sequence length for both reads before a sequence pair gets removed: 20 bp\n",
      "Output file(s) will be GZIP compressed\n",
      "\n",
      "Writing final adapter and quality trimmed output to NR014_AD014_S5_L001_R1_001_val_1_trimmed.fq.gz\n",
      "\n",
      "\n",
      "  >>> Now performing quality (cutoff 20) and adapter trimming in a single pass for the adapter sequence: 'AGATCGGAAGAGC' from file /home/sam/data/geoduck_illumina/trimmed/NR014_AD014_S5_L001_R1_001_val_1.fq.gz <<< \n",
      "10000000 sequences processed\n",
      "20000000 sequences processed\n",
      "30000000 sequences processed\n",
      "40000000 sequences processed\n",
      "50000000 sequences processed\n",
      "60000000 sequences processed\n",
      "70000000 sequences processed\n",
      "80000000 sequences processed\n",
      "90000000 sequences processed\n",
      "100000000 sequences processed\n",
      "110000000 sequences processed\n",
      "120000000 sequences processed\n",
      "130000000 sequences processed\n",
      "140000000 sequences processed\n",
      "150000000 sequences processed\n",
      "160000000 sequences processed\n",
      "170000000 sequences processed\n",
      "180000000 sequences processed\n",
      "190000000 sequences processed\n",
      "200000000 sequences processed\n",
      "210000000 sequences processed\n",
      "220000000 sequences processed\n",
      "230000000 sequences processed\n",
      "240000000 sequences processed\n",
      "250000000 sequences processed\n",
      "260000000 sequences processed\n",
      "270000000 sequences processed\n",
      "280000000 sequences processed\n",
      "290000000 sequences processed\n",
      "300000000 sequences processed\n",
      "310000000 sequences processed\n",
      "320000000 sequences processed\n",
      "330000000 sequences processed\n",
      "340000000 sequences processed\n",
      "350000000 sequences processed\n",
      "360000000 sequences processed\n",
      "370000000 sequences processed\n",
      "380000000 sequences processed\n",
      "390000000 sequences processed\n",
      "400000000 sequences processed\n",
      "410000000 sequences processed\n",
      "420000000 sequences processed\n",
      "430000000 sequences processed\n",
      "440000000 sequences processed\n",
      "450000000 sequences processed\n",
      "460000000 sequences processed\n",
      "470000000 sequences processed\n",
      "480000000 sequences processed\n",
      "490000000 sequences processed\n",
      "500000000 sequences processed\n",
      "510000000 sequences processed\n",
      "520000000 sequences processed\n",
      "530000000 sequences processed\n",
      "540000000 sequences processed\n",
      "550000000 sequences processed\n",
      "560000000 sequences processed\n",
      "570000000 sequences processed\n",
      "580000000 sequences processed\n",
      "590000000 sequences processed\n",
      "600000000 sequences processed\n",
      "610000000 sequences processed\n",
      "620000000 sequences processed\n",
      "630000000 sequences processed\n",
      "640000000 sequences processed\n",
      "650000000 sequences processed\n",
      "660000000 sequences processed\n",
      "670000000 sequences processed\n",
      "680000000 sequences processed\n",
      "690000000 sequences processed\n",
      "700000000 sequences processed\n",
      "710000000 sequences processed\n",
      "720000000 sequences processed\n",
      "730000000 sequences processed\n",
      "740000000 sequences processed\n",
      "750000000 sequences processed\n",
      "This is cutadapt 1.9.1 with Python 2.7.12\n",
      "Command line parameters: -f fastq -e 0.1 -q 20 -O 1 -a AGATCGGAAGAGC /home/sam/data/geoduck_illumina/trimmed/NR014_AD014_S5_L001_R1_001_val_1.fq.gz\n",
      "Trimming 1 adapter with at most 10.0% errors in single-end mode ...\n",
      "Finished in 17866.55 s (24 us/read; 2.52 M reads/minute).\n",
      "\n",
      "=== Summary ===\n",
      "\n",
      "Total reads processed:             750,570,443\n",
      "Reads with adapters:               289,187,209 (38.5%)\n",
      "Reads written (passing filters):   750,570,443 (100.0%)\n",
      "\n",
      "Total basepairs processed: 88,547,063,621 bp\n",
      "Quality-trimmed:              51,479,249 bp (0.1%)\n",
      "Total written (filtered):  86,834,377,140 bp (98.1%)\n",
      "\n",
      "=== Adapter 1 ===\n",
      "\n",
      "Sequence: AGATCGGAAGAGC; Type: regular 3'; Length: 13; Trimmed: 289187209 times.\n",
      "\n",
      "No. of allowed errors:\n",
      "0-9 bp: 0; 10-13 bp: 1\n",
      "\n",
      "Bases preceding removed adapters:\n",
      "  A: 31.0%\n",
      "  C: 23.5%\n",
      "  G: 15.1%\n",
      "  T: 29.7%\n",
      "  none/other: 0.7%\n",
      "\n",
      "Overview of removed sequences\n",
      "length\tcount\texpect\tmax.err\terror counts\n",
      "1\t192909668\t187642610.8\t0\t192909668\n",
      "2\t54350674\t46910652.7\t0\t54350674\n",
      "3\t14030638\t11727663.2\t0\t14030638\n",
      "4\t5645418\t2931915.8\t0\t5645418\n",
      "5\t644053\t732978.9\t0\t644053\n",
      "6\t309787\t183244.7\t0\t309787\n",
      "7\t237428\t45811.2\t0\t237428\n",
      "8\t281248\t11452.8\t0\t281248\n",
      "9\t249313\t2863.2\t0\t241008 8305\n",
      "10\t279192\t715.8\t1\t232507 46685\n",
      "11\t286431\t178.9\t1\t241423 45008\n",
      "12\t515773\t44.7\t1\t463019 52754\n",
      "13\t18367\t11.2\t1\t11839 6528\n",
      "14\t508262\t11.2\t1\t456715 51547\n",
      "15\t14833\t11.2\t1\t8724 6109\n",
      "16\t544414\t11.2\t1\t480512 63902\n",
      "17\t23775\t11.2\t1\t14558 9217\n",
      "18\t359075\t11.2\t1\t315603 43472\n",
      "19\t825945\t11.2\t1\t762424 63521\n",
      "20\t5665\t11.2\t1\t3478 2187\n",
      "21\t4990\t11.2\t1\t2183 2807\n",
      "22\t10358\t11.2\t1\t5510 4848\n",
      "23\t299789\t11.2\t1\t264190 35599\n",
      "24\t667172\t11.2\t1\t605345 61827\n",
      "25\t19949\t11.2\t1\t12651 7298\n",
      "26\t256354\t11.2\t1\t226389 29965\n",
      "27\t230460\t11.2\t1\t210212 20248\n",
      "28\t6568\t11.2\t1\t3872 2696\n",
      "29\t245569\t11.2\t1\t216886 28683\n",
      "30\t317069\t11.2\t1\t286417 30652\n",
      "31\t374473\t11.2\t1\t339859 34614\n",
      "32\t19468\t11.2\t1\t12962 6506\n",
      "33\t450331\t11.2\t1\t403885 46446\n",
      "34\t7586\t11.2\t1\t4925 2661\n",
      "35\t238662\t11.2\t1\t207824 30838\n",
      "36\t250935\t11.2\t1\t221628 29307\n",
      "37\t254545\t11.2\t1\t225209 29336\n",
      "38\t414582\t11.2\t1\t369817 44765\n",
      "39\t210154\t11.2\t1\t191129 19025\n",
      "40\t7717\t11.2\t1\t4738 2979\n",
      "41\t255519\t11.2\t1\t224930 30589\n",
      "42\t220673\t11.2\t1\t196986 23687\n",
      "43\t253997\t11.2\t1\t221561 32436\n",
      "44\t538821\t11.2\t1\t489058 49763\n",
      "45\t109189\t11.2\t1\t100829 8360\n",
      "46\t120125\t11.2\t1\t102249 17876\n",
      "47\t6331\t11.2\t1\t3868 2463\n",
      "48\t235082\t11.2\t1\t208233 26849\n",
      "49\t198823\t11.2\t1\t177604 21219\n",
      "50\t193998\t11.2\t1\t170789 23209\n",
      "51\t239801\t11.2\t1\t208357 31444\n",
      "52\t380896\t11.2\t1\t343749 37147\n",
      "53\t166334\t11.2\t1\t152005 14329\n",
      "54\t16602\t11.2\t1\t12011 4591\n",
      "55\t330481\t11.2\t1\t297411 33070\n",
      "56\t505188\t11.2\t1\t464589 40599\n",
      "57\t31464\t11.2\t1\t26508 4956\n",
      "58\t89482\t11.2\t1\t78503 10979\n",
      "59\t471636\t11.2\t1\t438787 32849\n",
      "60\t13294\t11.2\t1\t9635 3659\n",
      "61\t31799\t11.2\t1\t22753 9046\n",
      "62\t348119\t11.2\t1\t323874 24245\n",
      "63\t6467\t11.2\t1\t4271 2196\n",
      "64\t13543\t11.2\t1\t8483 5060\n",
      "65\t105779\t11.2\t1\t94646 11133\n",
      "66\t228594\t11.2\t1\t209330 19264\n",
      "67\t202694\t11.2\t1\t185615 17079\n",
      "68\t246807\t11.2\t1\t228830 17977\n",
      "69\t144456\t11.2\t1\t130812 13644\n",
      "70\t116936\t11.2\t1\t103663 13273\n",
      "71\t54946\t11.2\t1\t47030 7916\n",
      "72\t75718\t11.2\t1\t67992 7726\n",
      "73\t114178\t11.2\t1\t105273 8905\n",
      "74\t127551\t11.2\t1\t117131 10420\n",
      "75\t133108\t11.2\t1\t121830 11278\n",
      "76\t135663\t11.2\t1\t124030 11633\n",
      "77\t135833\t11.2\t1\t123792 12041\n",
      "78\t135999\t11.2\t1\t123523 12476\n",
      "79\t134513\t11.2\t1\t122024 12489\n",
      "80\t132353\t11.2\t1\t119792 12561\n",
      "81\t131502\t11.2\t1\t119043 12459\n",
      "82\t129476\t11.2\t1\t117100 12376\n",
      "83\t127413\t11.2\t1\t115100 12313\n",
      "84\t125209\t11.2\t1\t113013 12196\n",
      "85\t124152\t11.2\t1\t112064 12088\n",
      "86\t122198\t11.2\t1\t110146 12052\n",
      "87\t119141\t11.2\t1\t107493 11648\n",
      "88\t118327\t11.2\t1\t106680 11647\n",
      "89\t114684\t11.2\t1\t103226 11458\n",
      "90\t112928\t11.2\t1\t101561 11367\n",
      "91\t110169\t11.2\t1\t99211 10958\n",
      "92\t107113\t11.2\t1\t96201 10912\n",
      "93\t105103\t11.2\t1\t94518 10585\n",
      "94\t103054\t11.2\t1\t92695 10359\n",
      "95\t101527\t11.2\t1\t91241 10286\n",
      "96\t98938\t11.2\t1\t88802 10136\n",
      "97\t96670\t11.2\t1\t87063 9607\n",
      "98\t93458\t11.2\t1\t83912 9546\n",
      "99\t91216\t11.2\t1\t82070 9146\n",
      "100\t88277\t11.2\t1\t79256 9021\n",
      "101\t84267\t11.2\t1\t75500 8767\n",
      "102\t81040\t11.2\t1\t72488 8552\n",
      "103\t76881\t11.2\t1\t68764 8117\n",
      "104\t71322\t11.2\t1\t63577 7745\n",
      "105\t66845\t11.2\t1\t59357 7488\n",
      "106\t61236\t11.2\t1\t54084 7152\n",
      "107\t54672\t11.2\t1\t47874 6798\n",
      "108\t49926\t11.2\t1\t43230 6696\n",
      "109\t43801\t11.2\t1\t37446 6355\n",
      "110\t40718\t11.2\t1\t34799 5919\n",
      "111\t39329\t11.2\t1\t33521 5808\n",
      "112\t37285\t11.2\t1\t31675 5610\n",
      "113\t36474\t11.2\t1\t31067 5407\n",
      "114\t34974\t11.2\t1\t29828 5146\n",
      "115\t34356\t11.2\t1\t29518 4838\n",
      "116\t33052\t11.2\t1\t28418 4634\n",
      "117\t32570\t11.2\t1\t28040 4530\n",
      "118\t31661\t11.2\t1\t27284 4377\n",
      "119\t31114\t11.2\t1\t27029 4085\n",
      "120\t30169\t11.2\t1\t26088 4081\n",
      "121\t29542\t11.2\t1\t25780 3762\n",
      "122\t27978\t11.2\t1\t24484 3494\n",
      "123\t27542\t11.2\t1\t24306 3236\n",
      "124\t26034\t11.2\t1\t22997 3037\n",
      "125\t27797\t11.2\t1\t24853 2944\n",
      "126\t24952\t11.2\t1\t22306 2646\n",
      "127\t27223\t11.2\t1\t24604 2619\n",
      "128\t24870\t11.2\t1\t22619 2251\n",
      "129\t44965\t11.2\t1\t42042 2923\n",
      "130\t23224\t11.2\t1\t21466 1758\n",
      "131\t35351\t11.2\t1\t33515 1836\n",
      "132\t21416\t11.2\t1\t20253 1163\n",
      "133\t23072\t11.2\t1\t22010 1062\n",
      "134\t19541\t11.2\t1\t18686 855\n",
      "135\t22239\t11.2\t1\t21349 890\n",
      "136\t18813\t11.2\t1\t18114 699\n",
      "137\t20313\t11.2\t1\t19612 701\n",
      "138\t17790\t11.2\t1\t17295 495\n",
      "139\t17086\t11.2\t1\t16597 489\n",
      "140\t15218\t11.2\t1\t14783 435\n",
      "141\t14618\t11.2\t1\t14200 418\n",
      "142\t12159\t11.2\t1\t11781 378\n",
      "143\t13967\t11.2\t1\t13572 395\n",
      "144\t11090\t11.2\t1\t10754 336\n",
      "145\t11962\t11.2\t1\t11529 433\n",
      "146\t9230\t11.2\t1\t8851 379\n",
      "147\t8773\t11.2\t1\t8302 471\n",
      "148\t7357\t11.2\t1\t6629 728\n",
      "149\t8441\t11.2\t1\t7255 1186\n",
      "150\t18295\t11.2\t1\t9802 8493\n",
      "151\t2054622\t11.2\t1\t682481 1372141\n",
      "\n",
      "\n",
      "RUN STATISTICS FOR INPUT FILE: /home/sam/data/geoduck_illumina/trimmed/NR014_AD014_S5_L001_R1_001_val_1.fq.gz\n",
      "=============================================\n",
      "750570443 sequences processed in total\n",
      "The length threshold of paired-end sequences gets evaluated later on (in the validation step)\n",
      "\n",
      "Writing report to '/home/sam/data/geoduck_illumina/trimmed/20180129_trimmed_again/NR014_AD014_S5_L001_R2_001_val_2.fq.gz_trimming_report.txt'\n",
      "\n",
      "SUMMARISING RUN PARAMETERS\n",
      "==========================\n",
      "Input filename: /home/sam/data/geoduck_illumina/trimmed/NR014_AD014_S5_L001_R2_001_val_2.fq.gz\n",
      "Trimming mode: paired-end\n",
      "Trim Galore version: 0.4.4\n",
      "Cutadapt version: 1.9.1\n",
      "Quality Phred score cutoff: 20\n",
      "Quality encoding type selected: ASCII+33\n",
      "Adapter sequence: 'AGATCGGAAGAGC' (Illumina TruSeq, Sanger iPCR; auto-detected)\n",
      "Maximum trimming error rate: 0.1 (default)\n",
      "Minimum required adapter overlap (stringency): 1 bp\n",
      "Minimum required sequence length for both reads before a sequence pair gets removed: 20 bp\n",
      "Output file(s) will be GZIP compressed\n",
      "\n",
      "Writing final adapter and quality trimmed output to NR014_AD014_S5_L001_R2_001_val_2_trimmed.fq.gz\n",
      "\n",
      "\n",
      "  >>> Now performing quality (cutoff 20) and adapter trimming in a single pass for the adapter sequence: 'AGATCGGAAGAGC' from file /home/sam/data/geoduck_illumina/trimmed/NR014_AD014_S5_L001_R2_001_val_2.fq.gz <<< \n",
      "10000000 sequences processed\n",
      "20000000 sequences processed\n",
      "30000000 sequences processed\n",
      "40000000 sequences processed\n",
      "50000000 sequences processed\n",
      "60000000 sequences processed\n",
      "70000000 sequences processed\n",
      "80000000 sequences processed\n",
      "90000000 sequences processed\n",
      "100000000 sequences processed\n",
      "110000000 sequences processed\n",
      "120000000 sequences processed\n",
      "130000000 sequences processed\n",
      "140000000 sequences processed\n",
      "150000000 sequences processed\n",
      "160000000 sequences processed\n",
      "170000000 sequences processed\n",
      "180000000 sequences processed\n",
      "190000000 sequences processed\n",
      "200000000 sequences processed\n",
      "210000000 sequences processed\n",
      "220000000 sequences processed\n",
      "230000000 sequences processed\n",
      "240000000 sequences processed\n",
      "250000000 sequences processed\n",
      "260000000 sequences processed\n",
      "270000000 sequences processed\n",
      "280000000 sequences processed\n",
      "290000000 sequences processed\n",
      "300000000 sequences processed\n",
      "310000000 sequences processed\n",
      "320000000 sequences processed\n",
      "330000000 sequences processed\n",
      "340000000 sequences processed\n",
      "350000000 sequences processed\n",
      "360000000 sequences processed\n",
      "370000000 sequences processed\n",
      "380000000 sequences processed\n",
      "390000000 sequences processed\n",
      "400000000 sequences processed\n",
      "410000000 sequences processed\n",
      "420000000 sequences processed\n",
      "430000000 sequences processed\n",
      "440000000 sequences processed\n",
      "450000000 sequences processed\n",
      "460000000 sequences processed\n",
      "470000000 sequences processed\n",
      "480000000 sequences processed\n",
      "490000000 sequences processed\n",
      "500000000 sequences processed\n",
      "510000000 sequences processed\n",
      "520000000 sequences processed\n",
      "530000000 sequences processed\n",
      "540000000 sequences processed\n",
      "550000000 sequences processed\n",
      "560000000 sequences processed\n",
      "570000000 sequences processed\n",
      "580000000 sequences processed\n",
      "590000000 sequences processed\n",
      "600000000 sequences processed\n",
      "610000000 sequences processed\n",
      "620000000 sequences processed\n",
      "630000000 sequences processed\n",
      "640000000 sequences processed\n",
      "650000000 sequences processed\n",
      "660000000 sequences processed\n",
      "670000000 sequences processed\n",
      "680000000 sequences processed\n",
      "690000000 sequences processed\n",
      "700000000 sequences processed\n",
      "710000000 sequences processed\n",
      "720000000 sequences processed\n",
      "730000000 sequences processed\n",
      "740000000 sequences processed\n",
      "750000000 sequences processed\n",
      "This is cutadapt 1.9.1 with Python 2.7.12\n",
      "Command line parameters: -f fastq -e 0.1 -q 20 -O 1 -a AGATCGGAAGAGC /home/sam/data/geoduck_illumina/trimmed/NR014_AD014_S5_L001_R2_001_val_2.fq.gz\n",
      "Trimming 1 adapter with at most 10.0% errors in single-end mode ...\n",
      "Finished in 18489.76 s (25 us/read; 2.44 M reads/minute).\n",
      "\n",
      "=== Summary ===\n",
      "\n",
      "Total reads processed:             750,570,443\n",
      "Reads with adapters:               299,526,533 (39.9%)\n",
      "Reads written (passing filters):   750,570,443 (100.0%)\n",
      "\n",
      "Total basepairs processed: 91,326,369,816 bp\n",
      "Quality-trimmed:              58,430,294 bp (0.1%)\n",
      "Total written (filtered):  89,411,521,810 bp (97.9%)\n",
      "\n",
      "=== Adapter 1 ===\n",
      "\n",
      "Sequence: AGATCGGAAGAGC; Type: regular 3'; Length: 13; Trimmed: 299526533 times.\n",
      "\n",
      "No. of allowed errors:\n",
      "0-9 bp: 0; 10-13 bp: 1\n",
      "\n",
      "Bases preceding removed adapters:\n",
      "  A: 31.5%\n",
      "  C: 22.9%\n",
      "  G: 15.6%\n",
      "  T: 29.8%\n",
      "  none/other: 0.2%\n",
      "\n",
      "Overview of removed sequences\n",
      "length\tcount\texpect\tmax.err\terror counts\n",
      "1\t195508272\t187642610.8\t0\t195508272\n",
      "2\t53701405\t46910652.7\t0\t53701405\n",
      "3\t14830289\t11727663.2\t0\t14830289\n",
      "4\t5884365\t2931915.8\t0\t5884365\n",
      "5\t580478\t732978.9\t0\t580478\n",
      "6\t433990\t183244.7\t0\t433990\n",
      "7\t373366\t45811.2\t0\t373366\n",
      "8\t443852\t11452.8\t0\t443852\n",
      "9\t319608\t2863.2\t0\t310462 9146\n",
      "10\t425138\t715.8\t1\t356030 69108\n",
      "11\t405396\t178.9\t1\t341837 63559\n",
      "12\t791929\t44.7\t1\t699759 92170\n",
      "13\t7085\t11.2\t1\t3337 3748\n",
      "14\t479779\t11.2\t1\t414019 65760\n",
      "15\t666605\t11.2\t1\t597374 69231\n",
      "16\t3296\t11.2\t1\t1405 1891\n",
      "17\t447935\t11.2\t1\t395018 52917\n",
      "18\t283150\t11.2\t1\t249797 33353\n",
      "19\t479301\t11.2\t1\t420748 58553\n",
      "20\t348868\t11.2\t1\t308297 40571\n",
      "21\t318834\t11.2\t1\t280791 38043\n",
      "22\t359445\t11.2\t1\t319023 40422\n",
      "23\t383741\t11.2\t1\t337154 46587\n",
      "24\t609570\t11.2\t1\t532241 77329\n",
      "25\t355609\t11.2\t1\t317427 38182\n",
      "26\t376193\t11.2\t1\t333823 42370\n",
      "27\t287577\t11.2\t1\t246800 40777\n",
      "28\t564034\t11.2\t1\t513550 50484\n",
      "29\t164754\t11.2\t1\t134266 30488\n",
      "30\t537878\t11.2\t1\t487749 50129\n",
      "31\t235999\t11.2\t1\t202289 33710\n",
      "32\t1053810\t11.2\t1\t982245 71565\n",
      "33\t439075\t11.2\t1\t388688 50387\n",
      "34\t143201\t11.2\t1\t117029 26172\n",
      "35\t549014\t11.2\t1\t503741 45273\n",
      "36\t171248\t11.2\t1\t147222 24026\n",
      "37\t269301\t11.2\t1\t234512 34789\n",
      "38\t138046\t11.2\t1\t122921 15125\n",
      "39\t190077\t11.2\t1\t166107 23970\n",
      "40\t72430\t11.2\t1\t62897 9533\n",
      "41\t307972\t11.2\t1\t280936 27036\n",
      "42\t435964\t11.2\t1\t401434 34530\n",
      "43\t197707\t11.2\t1\t175590 22117\n",
      "44\t332032\t11.2\t1\t299779 32253\n",
      "45\t743685\t11.2\t1\t691844 51841\n",
      "46\t293743\t11.2\t1\t258266 35477\n",
      "47\t147612\t11.2\t1\t131958 15654\n",
      "48\t642368\t11.2\t1\t594726 47642\n",
      "49\t205842\t11.2\t1\t185331 20511\n",
      "50\t69850\t11.2\t1\t59300 10550\n",
      "51\t482885\t11.2\t1\t444980 37905\n",
      "52\t819026\t11.2\t1\t777011 42015\n",
      "53\t279581\t11.2\t1\t258697 20884\n",
      "54\t371032\t11.2\t1\t342874 28158\n",
      "55\t201835\t11.2\t1\t186009 15826\n",
      "56\t93525\t11.2\t1\t81149 12376\n",
      "57\t181125\t11.2\t1\t164075 17050\n",
      "58\t201528\t11.2\t1\t181670 19858\n",
      "59\t106205\t11.2\t1\t92181 14024\n",
      "60\t72144\t11.2\t1\t60589 11555\n",
      "61\t67276\t11.2\t1\t56173 11103\n",
      "62\t73654\t11.2\t1\t62651 11003\n",
      "63\t88612\t11.2\t1\t77301 11311\n",
      "64\t160280\t11.2\t1\t146493 13787\n",
      "65\t211526\t11.2\t1\t194467 17059\n",
      "66\t230118\t11.2\t1\t211041 19077\n",
      "67\t233193\t11.2\t1\t213309 19884\n",
      "68\t232801\t11.2\t1\t212622 20179\n",
      "69\t232837\t11.2\t1\t212661 20176\n",
      "70\t230956\t11.2\t1\t210679 20277\n",
      "71\t230712\t11.2\t1\t210035 20677\n",
      "72\t227427\t11.2\t1\t206864 20563\n",
      "73\t224834\t11.2\t1\t204801 20033\n",
      "74\t220854\t11.2\t1\t200966 19888\n",
      "75\t217348\t11.2\t1\t197730 19618\n",
      "76\t213212\t11.2\t1\t193853 19359\n",
      "77\t210130\t11.2\t1\t191101 19029\n",
      "78\t207107\t11.2\t1\t188351 18756\n",
      "79\t201762\t11.2\t1\t183564 18198\n",
      "80\t198242\t11.2\t1\t180336 17906\n",
      "81\t195772\t11.2\t1\t178095 17677\n",
      "82\t191787\t11.2\t1\t174331 17456\n",
      "83\t188512\t11.2\t1\t171336 17176\n",
      "84\t186114\t11.2\t1\t169107 17007\n",
      "85\t183104\t11.2\t1\t166525 16579\n",
      "86\t180176\t11.2\t1\t164018 16158\n",
      "87\t176875\t11.2\t1\t161108 15767\n",
      "88\t173208\t11.2\t1\t157369 15839\n",
      "89\t168365\t11.2\t1\t153113 15252\n",
      "90\t163173\t11.2\t1\t148130 15043\n",
      "91\t159873\t11.2\t1\t145102 14771\n",
      "92\t154865\t11.2\t1\t140836 14029\n",
      "93\t152411\t11.2\t1\t138467 13944\n",
      "94\t148681\t11.2\t1\t135244 13437\n",
      "95\t145839\t11.2\t1\t132601 13238\n",
      "96\t141809\t11.2\t1\t129026 12783\n",
      "97\t137914\t11.2\t1\t125518 12396\n",
      "98\t133616\t11.2\t1\t121504 12112\n",
      "99\t130159\t11.2\t1\t118199 11960\n",
      "100\t125571\t11.2\t1\t114128 11443\n",
      "101\t120631\t11.2\t1\t109465 11166\n",
      "102\t114670\t11.2\t1\t103928 10742\n",
      "103\t108518\t11.2\t1\t98139 10379\n",
      "104\t98959\t11.2\t1\t89090 9869\n",
      "105\t90807\t11.2\t1\t81324 9483\n",
      "106\t80925\t11.2\t1\t72014 8911\n",
      "107\t69986\t11.2\t1\t61403 8583\n",
      "108\t61928\t11.2\t1\t53842 8086\n",
      "109\t51721\t11.2\t1\t44112 7609\n",
      "110\t47044\t11.2\t1\t39803 7241\n",
      "111\t44643\t11.2\t1\t37536 7107\n",
      "112\t42188\t11.2\t1\t35304 6884\n",
      "113\t40737\t11.2\t1\t34327 6410\n",
      "114\t39256\t11.2\t1\t33071 6185\n",
      "115\t38548\t11.2\t1\t32442 6106\n",
      "116\t37109\t11.2\t1\t31381 5728\n",
      "117\t36299\t11.2\t1\t30884 5415\n",
      "118\t35146\t11.2\t1\t29963 5183\n",
      "119\t34570\t11.2\t1\t29658 4912\n",
      "120\t33134\t11.2\t1\t28543 4591\n",
      "121\t32475\t11.2\t1\t28113 4362\n",
      "122\t30765\t11.2\t1\t26673 4092\n",
      "123\t30432\t11.2\t1\t26520 3912\n",
      "124\t28512\t11.2\t1\t24762 3750\n",
      "125\t30138\t11.2\t1\t26594 3544\n",
      "126\t27221\t11.2\t1\t24034 3187\n",
      "127\t29101\t11.2\t1\t26126 2975\n",
      "128\t26261\t11.2\t1\t23747 2514\n",
      "129\t46487\t11.2\t1\t44023 2464\n",
      "130\t23963\t11.2\t1\t22173 1790\n",
      "131\t35973\t11.2\t1\t34287 1686\n",
      "132\t21567\t11.2\t1\t20281 1286\n",
      "133\t23281\t11.2\t1\t22074 1207\n",
      "134\t19695\t11.2\t1\t18575 1120\n",
      "135\t22376\t11.2\t1\t21283 1093\n",
      "136\t18882\t11.2\t1\t17760 1122\n",
      "137\t20353\t11.2\t1\t19196 1157\n",
      "138\t17856\t11.2\t1\t16607 1249\n",
      "139\t17077\t11.2\t1\t15902 1175\n",
      "140\t15173\t11.2\t1\t13937 1236\n",
      "141\t14561\t11.2\t1\t13203 1358\n",
      "142\t12110\t11.2\t1\t10652 1458\n",
      "143\t13944\t11.2\t1\t12264 1680\n",
      "144\t11169\t11.2\t1\t9336 1833\n",
      "145\t11834\t11.2\t1\t9757 2077\n",
      "146\t9067\t11.2\t1\t7040 2027\n",
      "147\t8590\t11.2\t1\t6651 1939\n",
      "148\t6792\t11.2\t1\t5016 1776\n",
      "149\t7588\t11.2\t1\t5615 1973\n",
      "150\t14895\t11.2\t1\t12078 2817\n",
      "151\t701267\t11.2\t1\t683500 17767\n",
      "\n",
      "\n",
      "RUN STATISTICS FOR INPUT FILE: /home/sam/data/geoduck_illumina/trimmed/NR014_AD014_S5_L001_R2_001_val_2.fq.gz\n",
      "=============================================\n",
      "750570443 sequences processed in total\n",
      "The length threshold of paired-end sequences gets evaluated later on (in the validation step)\n",
      "\n",
      "Validate paired-end files NR014_AD014_S5_L001_R1_001_val_1_trimmed.fq.gz and NR014_AD014_S5_L001_R2_001_val_2_trimmed.fq.gz\n",
      "file_1: NR014_AD014_S5_L001_R1_001_val_1_trimmed.fq.gz, file_2: NR014_AD014_S5_L001_R2_001_val_2_trimmed.fq.gz\n",
      "\n",
      "\n",
      ">>>>> Now validing the length of the 2 paired-end infiles: NR014_AD014_S5_L001_R1_001_val_1_trimmed.fq.gz and NR014_AD014_S5_L001_R2_001_val_2_trimmed.fq.gz <<<<<\n",
      "Writing validated paired-end read 1 reads to NR014_AD014_S5_L001_R1_001_val_1_val_1.fq.gz\n",
      "Writing validated paired-end read 2 reads to NR014_AD014_S5_L001_R2_001_val_2_val_2.fq.gz\n",
      "\n",
      "Total number of sequences analysed: 750570443\n",
      "\n",
      "Number of sequence pairs removed because at least one read was shorter than the length cutoff (20 bp): 4761104 (0.63%)\n",
      "\n",
      "Deleting both intermediate output files NR014_AD014_S5_L001_R1_001_val_1_trimmed.fq.gz and NR014_AD014_S5_L001_R2_001_val_2_trimmed.fq.gz\n",
      "\n",
      "====================================================================================================\n",
      "\n",
      "Writing report to '/home/sam/data/geoduck_illumina/trimmed/20180129_trimmed_again/NR014_AD014_S5_L002_R1_001_val_1.fq.gz_trimming_report.txt'\n",
      "\n",
      "SUMMARISING RUN PARAMETERS\n",
      "==========================\n",
      "Input filename: /home/sam/data/geoduck_illumina/trimmed/NR014_AD014_S5_L002_R1_001_val_1.fq.gz\n",
      "Trimming mode: paired-end\n",
      "Trim Galore version: 0.4.4\n",
      "Cutadapt version: 1.9.1\n",
      "Quality Phred score cutoff: 20\n",
      "Quality encoding type selected: ASCII+33\n",
      "Adapter sequence: 'AGATCGGAAGAGC' (Illumina TruSeq, Sanger iPCR; auto-detected)\n",
      "Maximum trimming error rate: 0.1 (default)\n",
      "Minimum required adapter overlap (stringency): 1 bp\n",
      "Minimum required sequence length for both reads before a sequence pair gets removed: 20 bp\n",
      "Output file(s) will be GZIP compressed\n",
      "\n",
      "Writing final adapter and quality trimmed output to NR014_AD014_S5_L002_R1_001_val_1_trimmed.fq.gz\n",
      "\n",
      "\n",
      "  >>> Now performing quality (cutoff 20) and adapter trimming in a single pass for the adapter sequence: 'AGATCGGAAGAGC' from file /home/sam/data/geoduck_illumina/trimmed/NR014_AD014_S5_L002_R1_001_val_1.fq.gz <<< \n",
      "10000000 sequences processed\n",
      "20000000 sequences processed\n",
      "30000000 sequences processed\n",
      "40000000 sequences processed\n",
      "50000000 sequences processed\n",
      "60000000 sequences processed\n",
      "70000000 sequences processed\n",
      "80000000 sequences processed\n",
      "90000000 sequences processed\n",
      "100000000 sequences processed\n",
      "110000000 sequences processed\n",
      "120000000 sequences processed\n",
      "130000000 sequences processed\n",
      "140000000 sequences processed\n",
      "150000000 sequences processed\n",
      "160000000 sequences processed\n",
      "170000000 sequences processed\n",
      "180000000 sequences processed\n",
      "190000000 sequences processed\n",
      "200000000 sequences processed\n",
      "210000000 sequences processed\n",
      "220000000 sequences processed\n",
      "230000000 sequences processed\n",
      "240000000 sequences processed\n",
      "250000000 sequences processed\n",
      "260000000 sequences processed\n",
      "270000000 sequences processed\n",
      "280000000 sequences processed\n",
      "290000000 sequences processed\n",
      "300000000 sequences processed\n",
      "310000000 sequences processed\n",
      "320000000 sequences processed\n",
      "330000000 sequences processed\n",
      "340000000 sequences processed\n",
      "350000000 sequences processed\n",
      "360000000 sequences processed\n",
      "370000000 sequences processed\n",
      "380000000 sequences processed\n",
      "390000000 sequences processed\n",
      "400000000 sequences processed\n",
      "410000000 sequences processed\n",
      "420000000 sequences processed\n",
      "430000000 sequences processed\n",
      "440000000 sequences processed\n",
      "450000000 sequences processed\n",
      "460000000 sequences processed\n",
      "470000000 sequences processed\n",
      "480000000 sequences processed\n",
      "490000000 sequences processed\n",
      "500000000 sequences processed\n",
      "510000000 sequences processed\n",
      "520000000 sequences processed\n",
      "530000000 sequences processed\n",
      "540000000 sequences processed\n",
      "550000000 sequences processed\n",
      "560000000 sequences processed\n",
      "570000000 sequences processed\n",
      "580000000 sequences processed\n",
      "590000000 sequences processed\n",
      "600000000 sequences processed\n",
      "610000000 sequences processed\n",
      "620000000 sequences processed\n",
      "630000000 sequences processed\n",
      "640000000 sequences processed\n",
      "650000000 sequences processed\n",
      "660000000 sequences processed\n",
      "670000000 sequences processed\n",
      "680000000 sequences processed\n",
      "690000000 sequences processed\n",
      "700000000 sequences processed\n",
      "710000000 sequences processed\n",
      "720000000 sequences processed\n",
      "730000000 sequences processed\n",
      "740000000 sequences processed\n",
      "750000000 sequences processed\n",
      "760000000 sequences processed\n",
      "770000000 sequences processed\n",
      "780000000 sequences processed\n",
      "790000000 sequences processed\n",
      "800000000 sequences processed\n",
      "810000000 sequences processed\n",
      "820000000 sequences processed\n",
      "830000000 sequences processed\n",
      "840000000 sequences processed\n",
      "850000000 sequences processed\n",
      "860000000 sequences processed\n",
      "870000000 sequences processed\n",
      "880000000 sequences processed\n",
      "This is cutadapt 1.9.1 with Python 2.7.12\n",
      "Command line parameters: -f fastq -e 0.1 -q 20 -O 1 -a AGATCGGAAGAGC /home/sam/data/geoduck_illumina/trimmed/NR014_AD014_S5_L002_R1_001_val_1.fq.gz\n",
      "Trimming 1 adapter with at most 10.0% errors in single-end mode ...\n",
      "Finished in 21050.32 s (24 us/read; 2.51 M reads/minute).\n",
      "\n",
      "=== Summary ===\n",
      "\n",
      "Total reads processed:             882,029,208\n",
      "Reads with adapters:               346,789,678 (39.3%)\n",
      "Reads written (passing filters):   882,029,208 (100.0%)\n",
      "\n",
      "Total basepairs processed: 106,455,935,222 bp\n",
      "Quality-trimmed:              52,712,555 bp (0.0%)\n",
      "Total written (filtered):  104,195,667,043 bp (97.9%)\n",
      "\n",
      "=== Adapter 1 ===\n",
      "\n",
      "Sequence: AGATCGGAAGAGC; Type: regular 3'; Length: 13; Trimmed: 346789678 times.\n",
      "\n",
      "No. of allowed errors:\n",
      "0-9 bp: 0; 10-13 bp: 1\n",
      "\n",
      "Bases preceding removed adapters:\n",
      "  A: 31.0%\n",
      "  C: 23.5%\n",
      "  G: 15.0%\n",
      "  T: 29.7%\n",
      "  none/other: 0.8%\n",
      "\n",
      "Overview of removed sequences\n",
      "length\tcount\texpect\tmax.err\terror counts\n",
      "1\t232624048\t220507302.0\t0\t232624048\n",
      "2\t63171145\t55126825.5\t0\t63171145\n",
      "3\t16364041\t13781706.4\t0\t16364041\n",
      "4\t6416328\t3445426.6\t0\t6416328\n",
      "5\t712237\t861356.6\t0\t712237\n",
      "6\t338291\t215339.2\t0\t338291\n",
      "7\t256798\t53834.8\t0\t256798\n",
      "8\t299385\t13458.7\t0\t299385\n",
      "9\t269984\t3364.7\t0\t260119 9865\n",
      "10\t307146\t841.2\t1\t253539 53607\n",
      "11\t315713\t210.3\t1\t264417 51296\n",
      "12\t562598\t52.6\t1\t503159 59439\n",
      "13\t19773\t13.1\t1\t12157 7616\n",
      "14\t552117\t13.1\t1\t496927 55190\n",
      "15\t16227\t13.1\t1\t8906 7321\n",
      "16\t596749\t13.1\t1\t526191 70558\n",
      "17\t27757\t13.1\t1\t16526 11231\n",
      "18\t393755\t13.1\t1\t344817 48938\n",
      "19\t910749\t13.1\t1\t842499 68250\n",
      "20\t6613\t13.1\t1\t4009 2604\n",
      "21\t6467\t13.1\t1\t2594 3873\n",
      "22\t12592\t13.1\t1\t6281 6311\n",
      "23\t329115\t13.1\t1\t286324 42791\n",
      "24\t753373\t13.1\t1\t681700 71673\n",
      "25\t22850\t13.1\t1\t14018 8832\n",
      "26\t290173\t13.1\t1\t254343 35830\n",
      "27\t256563\t13.1\t1\t234123 22440\n",
      "28\t7769\t13.1\t1\t4433 3336\n",
      "29\t277229\t13.1\t1\t243555 33674\n",
      "30\t351843\t13.1\t1\t315504 36339\n",
      "31\t436338\t13.1\t1\t392748 43590\n",
      "32\t23439\t13.1\t1\t14948 8491\n",
      "33\t525669\t13.1\t1\t466840 58829\n",
      "34\t9392\t13.1\t1\t5925 3467\n",
      "35\t281455\t13.1\t1\t240818 40637\n",
      "36\t294552\t13.1\t1\t257293 37259\n",
      "37\t297296\t13.1\t1\t259132 38164\n",
      "38\t493224\t13.1\t1\t434828 58396\n",
      "39\t243583\t13.1\t1\t220510 23073\n",
      "40\t9618\t13.1\t1\t5695 3923\n",
      "41\t303835\t13.1\t1\t265494 38341\n",
      "42\t254452\t13.1\t1\t224354 30098\n",
      "43\t308613\t13.1\t1\t264246 44367\n",
      "44\t654339\t13.1\t1\t589647 64692\n",
      "45\t128733\t13.1\t1\t118218 10515\n",
      "46\t140916\t13.1\t1\t118230 22686\n",
      "47\t7614\t13.1\t1\t4438 3176\n",
      "48\t271899\t13.1\t1\t238607 33292\n",
      "49\t234233\t13.1\t1\t207020 27213\n",
      "50\t249540\t13.1\t1\t216599 32941\n",
      "51\t294191\t13.1\t1\t248847 45344\n",
      "52\t475128\t13.1\t1\t423307 51821\n",
      "53\t212764\t13.1\t1\t192266 20498\n",
      "54\t20848\t13.1\t1\t14202 6646\n",
      "55\t373944\t13.1\t1\t327143 46801\n",
      "56\t656108\t13.1\t1\t595438 60670\n",
      "57\t35360\t13.1\t1\t28048 7312\n",
      "58\t102150\t13.1\t1\t84835 17315\n",
      "59\t650539\t13.1\t1\t597527 53012\n",
      "60\t16480\t13.1\t1\t11648 4832\n",
      "61\t38073\t13.1\t1\t25058 13015\n",
      "62\t443604\t13.1\t1\t406917 36687\n",
      "63\t7517\t13.1\t1\t4909 2608\n",
      "64\t13872\t13.1\t1\t7818 6054\n",
      "65\t151378\t13.1\t1\t133918 17460\n",
      "66\t259795\t13.1\t1\t232048 27747\n",
      "67\t261332\t13.1\t1\t232395 28937\n",
      "68\t305222\t13.1\t1\t274651 30571\n",
      "69\t238490\t13.1\t1\t212414 26076\n",
      "70\t300674\t13.1\t1\t271942 28732\n",
      "71\t60371\t13.1\t1\t47219 13152\n",
      "72\t78053\t13.1\t1\t66578 11475\n",
      "73\t145094\t13.1\t1\t131164 13930\n",
      "74\t168443\t13.1\t1\t152721 15722\n",
      "75\t178369\t13.1\t1\t161202 17167\n",
      "76\t183267\t13.1\t1\t165070 18197\n",
      "77\t185332\t13.1\t1\t166059 19273\n",
      "78\t185669\t13.1\t1\t166032 19637\n",
      "79\t184689\t13.1\t1\t164958 19731\n",
      "80\t183758\t13.1\t1\t163905 19853\n",
      "81\t182961\t13.1\t1\t162933 20028\n",
      "82\t181147\t13.1\t1\t161071 20076\n",
      "83\t179647\t13.1\t1\t159268 20379\n",
      "84\t179288\t13.1\t1\t158666 20622\n",
      "85\t176739\t13.1\t1\t156220 20519\n",
      "86\t175156\t13.1\t1\t154930 20226\n",
      "87\t172269\t13.1\t1\t152575 19694\n",
      "88\t171226\t13.1\t1\t151246 19980\n",
      "89\t167198\t13.1\t1\t147679 19519\n",
      "90\t165543\t13.1\t1\t145909 19634\n",
      "91\t162119\t13.1\t1\t142744 19375\n",
      "92\t159287\t13.1\t1\t140282 19005\n",
      "93\t157565\t13.1\t1\t138859 18706\n",
      "94\t155466\t13.1\t1\t136859 18607\n",
      "95\t151105\t13.1\t1\t133065 18040\n",
      "96\t150049\t13.1\t1\t132085 17964\n",
      "97\t146097\t13.1\t1\t128574 17523\n",
      "98\t143252\t13.1\t1\t125855 17397\n",
      "99\t139784\t13.1\t1\t122884 16900\n",
      "100\t136912\t13.1\t1\t120038 16874\n",
      "101\t132459\t13.1\t1\t116012 16447\n",
      "102\t129407\t13.1\t1\t113040 16367\n",
      "103\t123250\t13.1\t1\t107686 15564\n",
      "104\t116965\t13.1\t1\t101765 15200\n",
      "105\t111444\t13.1\t1\t96645 14799\n",
      "106\t104382\t13.1\t1\t89841 14541\n",
      "107\t95932\t13.1\t1\t82076 13856\n",
      "108\t90857\t13.1\t1\t76743 14114\n",
      "109\t82767\t13.1\t1\t69467 13300\n",
      "110\t78135\t13.1\t1\t65334 12801\n",
      "111\t75752\t13.1\t1\t63276 12476\n",
      "112\t73108\t13.1\t1\t61040 12068\n",
      "113\t70775\t13.1\t1\t59264 11511\n",
      "114\t68614\t13.1\t1\t57411 11203\n",
      "115\t67336\t13.1\t1\t56534 10802\n",
      "116\t64527\t13.1\t1\t54287 10240\n",
      "117\t63276\t13.1\t1\t53412 9864\n",
      "118\t61181\t13.1\t1\t51519 9662\n",
      "119\t59921\t13.1\t1\t50861 9060\n",
      "120\t57524\t13.1\t1\t48564 8960\n",
      "121\t56580\t13.1\t1\t48099 8481\n",
      "122\t53484\t13.1\t1\t45411 8073\n",
      "123\t51725\t13.1\t1\t44332 7393\n",
      "124\t48421\t13.1\t1\t41518 6903\n",
      "125\t50894\t13.1\t1\t44326 6568\n",
      "126\t46143\t13.1\t1\t39934 6209\n",
      "127\t48871\t13.1\t1\t42933 5938\n",
      "128\t43008\t13.1\t1\t38170 4838\n",
      "129\t72254\t13.1\t1\t66995 5259\n",
      "130\t39005\t13.1\t1\t35333 3672\n",
      "131\t55946\t13.1\t1\t52525 3421\n",
      "132\t34239\t13.1\t1\t31874 2365\n",
      "133\t36791\t13.1\t1\t34839 1952\n",
      "134\t29888\t13.1\t1\t28301 1587\n",
      "135\t32842\t13.1\t1\t31413 1429\n",
      "136\t27536\t13.1\t1\t26434 1102\n",
      "137\t28779\t13.1\t1\t27737 1042\n",
      "138\t24740\t13.1\t1\t23988 752\n",
      "139\t23461\t13.1\t1\t22794 667\n",
      "140\t20629\t13.1\t1\t20112 517\n",
      "141\t19394\t13.1\t1\t18916 478\n",
      "142\t15752\t13.1\t1\t15304 448\n",
      "143\t18237\t13.1\t1\t17814 423\n",
      "144\t14377\t13.1\t1\t13941 436\n",
      "145\t15679\t13.1\t1\t15212 467\n",
      "146\t11999\t13.1\t1\t11562 437\n",
      "147\t11281\t13.1\t1\t10783 498\n",
      "148\t9039\t13.1\t1\t8390 649\n",
      "149\t10320\t13.1\t1\t9341 979\n",
      "150\t16689\t13.1\t1\t10308 6381\n",
      "151\t2792571\t13.1\t1\t930915 1861656\n",
      "\n",
      "\n",
      "RUN STATISTICS FOR INPUT FILE: /home/sam/data/geoduck_illumina/trimmed/NR014_AD014_S5_L002_R1_001_val_1.fq.gz\n",
      "=============================================\n",
      "882029208 sequences processed in total\n",
      "The length threshold of paired-end sequences gets evaluated later on (in the validation step)\n",
      "\n",
      "Writing report to '/home/sam/data/geoduck_illumina/trimmed/20180129_trimmed_again/NR014_AD014_S5_L002_R2_001_val_2.fq.gz_trimming_report.txt'\n",
      "\n",
      "SUMMARISING RUN PARAMETERS\n",
      "==========================\n",
      "Input filename: /home/sam/data/geoduck_illumina/trimmed/NR014_AD014_S5_L002_R2_001_val_2.fq.gz\n",
      "Trimming mode: paired-end\n",
      "Trim Galore version: 0.4.4\n",
      "Cutadapt version: 1.9.1\n",
      "Quality Phred score cutoff: 20\n",
      "Quality encoding type selected: ASCII+33\n",
      "Adapter sequence: 'AGATCGGAAGAGC' (Illumina TruSeq, Sanger iPCR; auto-detected)\n",
      "Maximum trimming error rate: 0.1 (default)\n",
      "Minimum required adapter overlap (stringency): 1 bp\n",
      "Minimum required sequence length for both reads before a sequence pair gets removed: 20 bp\n",
      "Output file(s) will be GZIP compressed\n",
      "\n",
      "Writing final adapter and quality trimmed output to NR014_AD014_S5_L002_R2_001_val_2_trimmed.fq.gz\n",
      "\n",
      "\n",
      "  >>> Now performing quality (cutoff 20) and adapter trimming in a single pass for the adapter sequence: 'AGATCGGAAGAGC' from file /home/sam/data/geoduck_illumina/trimmed/NR014_AD014_S5_L002_R2_001_val_2.fq.gz <<< \n",
      "10000000 sequences processed\n",
      "20000000 sequences processed\n",
      "30000000 sequences processed\n",
      "40000000 sequences processed\n",
      "50000000 sequences processed\n",
      "60000000 sequences processed\n",
      "70000000 sequences processed\n",
      "80000000 sequences processed\n",
      "90000000 sequences processed\n",
      "100000000 sequences processed\n",
      "110000000 sequences processed\n",
      "120000000 sequences processed\n",
      "130000000 sequences processed\n",
      "140000000 sequences processed\n",
      "150000000 sequences processed\n",
      "160000000 sequences processed\n",
      "170000000 sequences processed\n",
      "180000000 sequences processed\n",
      "190000000 sequences processed\n",
      "200000000 sequences processed\n",
      "210000000 sequences processed\n",
      "220000000 sequences processed\n",
      "230000000 sequences processed\n",
      "240000000 sequences processed\n",
      "250000000 sequences processed\n",
      "260000000 sequences processed\n",
      "270000000 sequences processed\n",
      "280000000 sequences processed\n",
      "290000000 sequences processed\n",
      "300000000 sequences processed\n",
      "310000000 sequences processed\n",
      "320000000 sequences processed\n",
      "330000000 sequences processed\n",
      "340000000 sequences processed\n",
      "350000000 sequences processed\n",
      "360000000 sequences processed\n",
      "370000000 sequences processed\n",
      "380000000 sequences processed\n",
      "390000000 sequences processed\n",
      "400000000 sequences processed\n",
      "410000000 sequences processed\n",
      "420000000 sequences processed\n",
      "430000000 sequences processed\n",
      "440000000 sequences processed\n",
      "450000000 sequences processed\n",
      "460000000 sequences processed\n",
      "470000000 sequences processed\n",
      "480000000 sequences processed\n",
      "490000000 sequences processed\n",
      "500000000 sequences processed\n",
      "510000000 sequences processed\n",
      "520000000 sequences processed\n",
      "530000000 sequences processed\n",
      "540000000 sequences processed\n",
      "550000000 sequences processed\n",
      "560000000 sequences processed\n",
      "570000000 sequences processed\n",
      "580000000 sequences processed\n",
      "590000000 sequences processed\n",
      "600000000 sequences processed\n",
      "610000000 sequences processed\n",
      "620000000 sequences processed\n",
      "630000000 sequences processed\n",
      "640000000 sequences processed\n",
      "650000000 sequences processed\n",
      "660000000 sequences processed\n",
      "670000000 sequences processed\n",
      "680000000 sequences processed\n",
      "690000000 sequences processed\n",
      "700000000 sequences processed\n",
      "710000000 sequences processed\n",
      "720000000 sequences processed\n",
      "730000000 sequences processed\n",
      "740000000 sequences processed\n",
      "750000000 sequences processed\n",
      "760000000 sequences processed\n",
      "770000000 sequences processed\n",
      "780000000 sequences processed\n",
      "790000000 sequences processed\n",
      "800000000 sequences processed\n",
      "810000000 sequences processed\n",
      "820000000 sequences processed\n",
      "830000000 sequences processed\n",
      "840000000 sequences processed\n",
      "850000000 sequences processed\n",
      "860000000 sequences processed\n",
      "870000000 sequences processed\n",
      "880000000 sequences processed\n",
      "This is cutadapt 1.9.1 with Python 2.7.12\n",
      "Command line parameters: -f fastq -e 0.1 -q 20 -O 1 -a AGATCGGAAGAGC /home/sam/data/geoduck_illumina/trimmed/NR014_AD014_S5_L002_R2_001_val_2.fq.gz\n",
      "Trimming 1 adapter with at most 10.0% errors in single-end mode ...\n",
      "Finished in 21417.12 s (24 us/read; 2.47 M reads/minute).\n",
      "\n",
      "=== Summary ===\n",
      "\n",
      "Total reads processed:             882,029,208\n",
      "Reads with adapters:               357,804,367 (40.6%)\n",
      "Reads written (passing filters):   882,029,208 (100.0%)\n",
      "\n",
      "Total basepairs processed: 109,686,944,991 bp\n",
      "Quality-trimmed:              57,793,498 bp (0.1%)\n",
      "Total written (filtered):  107,213,258,754 bp (97.7%)\n",
      "\n",
      "=== Adapter 1 ===\n",
      "\n",
      "Sequence: AGATCGGAAGAGC; Type: regular 3'; Length: 13; Trimmed: 357804367 times.\n",
      "\n",
      "No. of allowed errors:\n",
      "0-9 bp: 0; 10-13 bp: 1\n",
      "\n",
      "Bases preceding removed adapters:\n",
      "  A: 31.4%\n",
      "  C: 23.0%\n",
      "  G: 15.5%\n",
      "  T: 29.8%\n",
      "  none/other: 0.3%\n",
      "\n",
      "Overview of removed sequences\n",
      "length\tcount\texpect\tmax.err\terror counts\n",
      "1\t235099366\t220507302.0\t0\t235099366\n",
      "2\t62543836\t55126825.5\t0\t62543836\n",
      "3\t17144245\t13781706.4\t0\t17144245\n",
      "4\t6646447\t3445426.6\t0\t6646447\n",
      "5\t639554\t861356.6\t0\t639554\n",
      "6\t474891\t215339.2\t0\t474891\n",
      "7\t405448\t53834.8\t0\t405448\n",
      "8\t466848\t13458.7\t0\t466848\n",
      "9\t348446\t3364.7\t0\t337673 10773\n",
      "10\t461105\t841.2\t1\t386657 74448\n",
      "11\t439665\t210.3\t1\t373444 66221\n",
      "12\t854670\t52.6\t1\t764149 90521\n",
      "13\t7135\t13.1\t1\t2811 4324\n",
      "14\t514702\t13.1\t1\t447955 66747\n",
      "15\t722304\t13.1\t1\t657406 64898\n",
      "16\t3921\t13.1\t1\t1665 2256\n",
      "17\t483893\t13.1\t1\t430625 53268\n",
      "18\t311726\t13.1\t1\t280786 30940\n",
      "19\t508462\t13.1\t1\t447409 61053\n",
      "20\t376124\t13.1\t1\t336875 39249\n",
      "21\t359883\t13.1\t1\t322088 37795\n",
      "22\t398674\t13.1\t1\t356521 42153\n",
      "23\t416959\t13.1\t1\t370051 46908\n",
      "24\t611113\t13.1\t1\t529458 81655\n",
      "25\t410127\t13.1\t1\t370095 40032\n",
      "26\t415762\t13.1\t1\t375188 40574\n",
      "27\t327612\t13.1\t1\t286723 40889\n",
      "28\t609543\t13.1\t1\t556927 52616\n",
      "29\t205545\t13.1\t1\t171345 34200\n",
      "30\t596568\t13.1\t1\t542203 54365\n",
      "31\t263996\t13.1\t1\t228833 35163\n",
      "32\t1289400\t13.1\t1\t1212993 76407\n",
      "33\t512660\t13.1\t1\t459500 53160\n",
      "34\t184716\t13.1\t1\t153208 31508\n",
      "35\t692795\t13.1\t1\t644767 48028\n",
      "36\t158296\t13.1\t1\t131258 27038\n",
      "37\t342624\t13.1\t1\t302501 40123\n",
      "38\t97006\t13.1\t1\t81760 15246\n",
      "39\t183704\t13.1\t1\t157551 26153\n",
      "40\t45254\t13.1\t1\t35800 9454\n",
      "41\t355563\t13.1\t1\t324511 31052\n",
      "42\t528472\t13.1\t1\t488465 40007\n",
      "43\t208242\t13.1\t1\t182929 25313\n",
      "44\t390129\t13.1\t1\t351993 38136\n",
      "45\t1128825\t13.1\t1\t1066223 62602\n",
      "46\t345095\t13.1\t1\t302774 42321\n",
      "47\t117084\t13.1\t1\t100592 16492\n",
      "48\t873716\t13.1\t1\t814718 58998\n",
      "49\t169284\t13.1\t1\t146835 22449\n",
      "50\t52750\t13.1\t1\t40938 11812\n",
      "51\t646744\t13.1\t1\t599590 47154\n",
      "52\t1193783\t13.1\t1\t1142866 50917\n",
      "53\t250287\t13.1\t1\t227517 22770\n",
      "54\t380803\t13.1\t1\t348438 32365\n",
      "55\t138543\t13.1\t1\t122337 16206\n",
      "56\t90468\t13.1\t1\t75051 15417\n",
      "57\t169985\t13.1\t1\t149829 20156\n",
      "58\t203858\t13.1\t1\t180530 23328\n",
      "59\t93592\t13.1\t1\t77303 16289\n",
      "60\t78787\t13.1\t1\t63686 15101\n",
      "61\t82444\t13.1\t1\t67562 14882\n",
      "62\t79746\t13.1\t1\t65366 14380\n",
      "63\t94126\t13.1\t1\t79609 14517\n",
      "64\t179996\t13.1\t1\t162738 17258\n",
      "65\t257597\t13.1\t1\t235879 21718\n",
      "66\t285315\t13.1\t1\t261166 24149\n",
      "67\t292126\t13.1\t1\t266275 25851\n",
      "68\t295475\t13.1\t1\t268387 27088\n",
      "69\t295990\t13.1\t1\t268443 27547\n",
      "70\t296714\t13.1\t1\t268419 28295\n",
      "71\t300117\t13.1\t1\t271207 28910\n",
      "72\t295102\t13.1\t1\t266729 28373\n",
      "73\t297780\t13.1\t1\t269028 28752\n",
      "74\t291630\t13.1\t1\t263445 28185\n",
      "75\t287606\t13.1\t1\t259336 28270\n",
      "76\t282934\t13.1\t1\t254551 28383\n",
      "77\t279808\t13.1\t1\t251854 27954\n",
      "78\t276410\t13.1\t1\t248663 27747\n",
      "79\t271068\t13.1\t1\t243712 27356\n",
      "80\t266965\t13.1\t1\t240369 26596\n",
      "81\t264294\t13.1\t1\t238046 26248\n",
      "82\t260238\t13.1\t1\t233644 26594\n",
      "83\t258826\t13.1\t1\t232425 26401\n",
      "84\t255117\t13.1\t1\t229078 26039\n",
      "85\t252997\t13.1\t1\t227278 25719\n",
      "86\t250320\t13.1\t1\t224751 25569\n",
      "87\t246703\t13.1\t1\t221934 24769\n",
      "88\t243894\t13.1\t1\t219163 24731\n",
      "89\t237391\t13.1\t1\t212725 24666\n",
      "90\t232269\t13.1\t1\t207825 24444\n",
      "91\t227208\t13.1\t1\t203062 24146\n",
      "92\t222180\t13.1\t1\t198614 23566\n",
      "93\t219038\t13.1\t1\t195701 23337\n",
      "94\t215630\t13.1\t1\t192779 22851\n",
      "95\t212171\t13.1\t1\t189594 22577\n",
      "96\t208724\t13.1\t1\t186441 22283\n",
      "97\t203591\t13.1\t1\t181707 21884\n",
      "98\t198733\t13.1\t1\t177288 21445\n",
      "99\t195082\t13.1\t1\t174344 20738\n",
      "100\t190416\t13.1\t1\t169973 20443\n",
      "101\t185717\t13.1\t1\t165304 20413\n",
      "102\t177447\t13.1\t1\t157905 19542\n",
      "103\t169476\t13.1\t1\t150354 19122\n",
      "104\t158041\t13.1\t1\t139594 18447\n",
      "105\t149050\t13.1\t1\t130821 18229\n",
      "106\t137972\t13.1\t1\t120204 17768\n",
      "107\t124510\t13.1\t1\t107417 17093\n",
      "108\t115302\t13.1\t1\t98640 16662\n",
      "109\t103287\t13.1\t1\t87118 16169\n",
      "110\t95710\t13.1\t1\t80359 15351\n",
      "111\t92391\t13.1\t1\t77278 15113\n",
      "112\t88978\t13.1\t1\t74373 14605\n",
      "113\t86338\t13.1\t1\t72405 13933\n",
      "114\t83339\t13.1\t1\t69681 13658\n",
      "115\t81009\t13.1\t1\t68155 12854\n",
      "116\t78418\t13.1\t1\t65893 12525\n",
      "117\t76604\t13.1\t1\t64480 12124\n",
      "118\t73600\t13.1\t1\t61978 11622\n",
      "119\t72003\t13.1\t1\t60976 11027\n",
      "120\t68625\t13.1\t1\t57886 10739\n",
      "121\t67484\t13.1\t1\t57305 10179\n",
      "122\t63641\t13.1\t1\t53989 9652\n",
      "123\t61549\t13.1\t1\t52545 9004\n",
      "124\t57810\t13.1\t1\t49218 8592\n",
      "125\t59751\t13.1\t1\t51688 8063\n",
      "126\t54119\t13.1\t1\t46626 7493\n",
      "127\t56196\t13.1\t1\t49390 6806\n",
      "128\t48450\t13.1\t1\t43017 5433\n",
      "129\t77040\t13.1\t1\t72208 4832\n",
      "130\t41906\t13.1\t1\t38198 3708\n",
      "131\t58208\t13.1\t1\t55170 3038\n",
      "132\t35365\t13.1\t1\t33000 2365\n",
      "133\t37604\t13.1\t1\t35650 1954\n",
      "134\t30524\t13.1\t1\t28823 1701\n",
      "135\t33388\t13.1\t1\t31868 1520\n",
      "136\t27789\t13.1\t1\t26293 1496\n",
      "137\t28933\t13.1\t1\t27576 1357\n",
      "138\t24891\t13.1\t1\t23429 1462\n",
      "139\t23476\t13.1\t1\t22117 1359\n",
      "140\t20658\t13.1\t1\t19184 1474\n",
      "141\t19395\t13.1\t1\t17726 1669\n",
      "142\t15659\t13.1\t1\t13873 1786\n",
      "143\t18154\t13.1\t1\t16108 2046\n",
      "144\t14333\t13.1\t1\t12303 2030\n",
      "145\t15517\t13.1\t1\t12894 2623\n",
      "146\t11673\t13.1\t1\t9220 2453\n",
      "147\t10903\t13.1\t1\t8531 2372\n",
      "148\t8478\t13.1\t1\t6288 2190\n",
      "149\t9455\t13.1\t1\t6948 2507\n",
      "150\t11337\t13.1\t1\t8415 2922\n",
      "151\t948058\t13.1\t1\t929339 18719\n",
      "\n",
      "\n",
      "RUN STATISTICS FOR INPUT FILE: /home/sam/data/geoduck_illumina/trimmed/NR014_AD014_S5_L002_R2_001_val_2.fq.gz\n",
      "=============================================\n",
      "882029208 sequences processed in total\n",
      "The length threshold of paired-end sequences gets evaluated later on (in the validation step)\n",
      "\n",
      "Validate paired-end files NR014_AD014_S5_L002_R1_001_val_1_trimmed.fq.gz and NR014_AD014_S5_L002_R2_001_val_2_trimmed.fq.gz\n",
      "file_1: NR014_AD014_S5_L002_R1_001_val_1_trimmed.fq.gz, file_2: NR014_AD014_S5_L002_R2_001_val_2_trimmed.fq.gz\n",
      "\n",
      "\n",
      ">>>>> Now validing the length of the 2 paired-end infiles: NR014_AD014_S5_L002_R1_001_val_1_trimmed.fq.gz and NR014_AD014_S5_L002_R2_001_val_2_trimmed.fq.gz <<<<<\n",
      "Writing validated paired-end read 1 reads to NR014_AD014_S5_L002_R1_001_val_1_val_1.fq.gz\n",
      "Writing validated paired-end read 2 reads to NR014_AD014_S5_L002_R2_001_val_2_val_2.fq.gz\n",
      "\n",
      "Total number of sequences analysed: 882029208\n",
      "\n",
      "Number of sequence pairs removed because at least one read was shorter than the length cutoff (20 bp): 5672977 (0.64%)\n",
      "\n",
      "Deleting both intermediate output files NR014_AD014_S5_L002_R1_001_val_1_trimmed.fq.gz and NR014_AD014_S5_L002_R2_001_val_2_trimmed.fq.gz\n",
      "\n",
      "====================================================================================================\n",
      "\n",
      "Writing report to '/home/sam/data/geoduck_illumina/trimmed/20180129_trimmed_again/NR015_AD015_S6_L001_R1_001_val_1.fq.gz_trimming_report.txt'\n",
      "\n",
      "SUMMARISING RUN PARAMETERS\n",
      "==========================\n",
      "Input filename: /home/sam/data/geoduck_illumina/trimmed/NR015_AD015_S6_L001_R1_001_val_1.fq.gz\n",
      "Trimming mode: paired-end\n",
      "Trim Galore version: 0.4.4\n",
      "Cutadapt version: 1.9.1\n",
      "Quality Phred score cutoff: 20\n",
      "Quality encoding type selected: ASCII+33\n",
      "Adapter sequence: 'AGATCGGAAGAGC' (Illumina TruSeq, Sanger iPCR; auto-detected)\n",
      "Maximum trimming error rate: 0.1 (default)\n",
      "Minimum required adapter overlap (stringency): 1 bp\n",
      "Minimum required sequence length for both reads before a sequence pair gets removed: 20 bp\n",
      "Output file(s) will be GZIP compressed\n",
      "\n",
      "Writing final adapter and quality trimmed output to NR015_AD015_S6_L001_R1_001_val_1_trimmed.fq.gz\n",
      "\n",
      "\n",
      "  >>> Now performing quality (cutoff 20) and adapter trimming in a single pass for the adapter sequence: 'AGATCGGAAGAGC' from file /home/sam/data/geoduck_illumina/trimmed/NR015_AD015_S6_L001_R1_001_val_1.fq.gz <<< \n",
      "10000000 sequences processed\n",
      "20000000 sequences processed\n",
      "30000000 sequences processed\n",
      "40000000 sequences processed\n",
      "50000000 sequences processed\n",
      "60000000 sequences processed\n",
      "70000000 sequences processed\n",
      "80000000 sequences processed\n",
      "90000000 sequences processed\n",
      "100000000 sequences processed\n",
      "110000000 sequences processed\n",
      "120000000 sequences processed\n",
      "130000000 sequences processed\n",
      "140000000 sequences processed\n",
      "150000000 sequences processed\n",
      "160000000 sequences processed\n",
      "170000000 sequences processed\n",
      "180000000 sequences processed\n",
      "190000000 sequences processed\n",
      "200000000 sequences processed\n",
      "210000000 sequences processed\n",
      "220000000 sequences processed\n",
      "230000000 sequences processed\n",
      "240000000 sequences processed\n",
      "250000000 sequences processed\n",
      "This is cutadapt 1.9.1 with Python 2.7.12\n",
      "Command line parameters: -f fastq -e 0.1 -q 20 -O 1 -a AGATCGGAAGAGC /home/sam/data/geoduck_illumina/trimmed/NR015_AD015_S6_L001_R1_001_val_1.fq.gz\n",
      "Trimming 1 adapter with at most 10.0% errors in single-end mode ...\n",
      "Finished in 5993.33 s (24 us/read; 2.52 M reads/minute).\n",
      "\n",
      "=== Summary ===\n",
      "\n",
      "Total reads processed:             251,960,784\n",
      "Reads with adapters:                98,150,549 (39.0%)\n",
      "Reads written (passing filters):   251,960,784 (100.0%)\n",
      "\n",
      "Total basepairs processed: 30,292,882,260 bp\n",
      "Quality-trimmed:              14,880,179 bp (0.0%)\n",
      "Total written (filtered):  29,870,391,799 bp (98.6%)\n",
      "\n",
      "=== Adapter 1 ===\n",
      "\n",
      "Sequence: AGATCGGAAGAGC; Type: regular 3'; Length: 13; Trimmed: 98150549 times.\n",
      "\n",
      "No. of allowed errors:\n",
      "0-9 bp: 0; 10-13 bp: 1\n",
      "\n",
      "Bases preceding removed adapters:\n",
      "  A: 31.6%\n",
      "  C: 23.5%\n",
      "  G: 14.7%\n",
      "  T: 29.7%\n",
      "  none/other: 0.4%\n",
      "\n",
      "Overview of removed sequences\n",
      "length\tcount\texpect\tmax.err\terror counts\n",
      "1\t67647589\t62990196.0\t0\t67647589\n",
      "2\t18152045\t15747549.0\t0\t18152045\n",
      "3\t4775986\t3936887.2\t0\t4775986\n",
      "4\t1890969\t984221.8\t0\t1890969\n",
      "5\t207664\t246055.5\t0\t207664\n",
      "6\t98537\t61513.9\t0\t98537\n",
      "7\t74846\t15378.5\t0\t74846\n",
      "8\t88166\t3844.6\t0\t88166\n",
      "9\t80952\t961.2\t0\t78131 2821\n",
      "10\t87335\t240.3\t1\t73607 13728\n",
      "11\t89943\t60.1\t1\t76934 13009\n",
      "12\t160853\t15.0\t1\t146405 14448\n",
      "13\t4569\t3.8\t1\t2624 1945\n",
      "14\t159804\t3.8\t1\t145131 14673\n",
      "15\t3985\t3.8\t1\t2223 1762\n",
      "16\t170314\t3.8\t1\t151711 18603\n",
      "17\t7416\t3.8\t1\t4591 2825\n",
      "18\t108590\t3.8\t1\t97058 11532\n",
      "19\t256789\t3.8\t1\t238071 18718\n",
      "20\t1865\t3.8\t1\t1095 770\n",
      "21\t1505\t3.8\t1\t703 802\n",
      "22\t3383\t3.8\t1\t1937 1446\n",
      "23\t95821\t3.8\t1\t85516 10305\n",
      "24\t198167\t3.8\t1\t181758 16409\n",
      "25\t6239\t3.8\t1\t4186 2053\n",
      "26\t75965\t3.8\t1\t68109 7856\n",
      "27\t68321\t3.8\t1\t63163 5158\n",
      "28\t2319\t3.8\t1\t1409 910\n",
      "29\t72530\t3.8\t1\t64801 7729\n",
      "30\t89129\t3.8\t1\t80867 8262\n",
      "31\t111877\t3.8\t1\t103485 8392\n",
      "32\t5769\t3.8\t1\t4052 1717\n",
      "33\t127778\t3.8\t1\t116285 11493\n",
      "34\t2818\t3.8\t1\t1949 869\n",
      "35\t69936\t3.8\t1\t61722 8214\n",
      "36\t66317\t3.8\t1\t59395 6922\n",
      "37\t71065\t3.8\t1\t64414 6651\n",
      "38\t107872\t3.8\t1\t99240 8632\n",
      "39\t2490\t3.8\t1\t1707 783\n",
      "40\t62577\t3.8\t1\t55534 7043\n",
      "41\t60445\t3.8\t1\t53846 6599\n",
      "42\t72395\t3.8\t1\t63133 9262\n",
      "43\t79730\t3.8\t1\t67556 12174\n",
      "44\t174322\t3.8\t1\t155148 19174\n",
      "45\t33434\t3.8\t1\t29737 3697\n",
      "46\t48312\t3.8\t1\t40644 7668\n",
      "47\t8868\t3.8\t1\t7169 1699\n",
      "48\t363712\t3.8\t1\t346160 17552\n",
      "49\t159049\t3.8\t1\t148984 10065\n",
      "50\t28469\t3.8\t1\t24000 4469\n",
      "51\t87615\t3.8\t1\t81660 5955\n",
      "52\t83971\t3.8\t1\t77843 6128\n",
      "53\t16110\t3.8\t1\t13174 2936\n",
      "54\t5077\t3.8\t1\t3616 1461\n",
      "55\t40249\t3.8\t1\t37342 2907\n",
      "56\t25311\t3.8\t1\t22425 2886\n",
      "57\t4914\t3.8\t1\t3521 1393\n",
      "58\t9531\t3.8\t1\t7750 1781\n",
      "59\t15683\t3.8\t1\t13352 2331\n",
      "60\t6108\t3.8\t1\t4537 1571\n",
      "61\t9553\t3.8\t1\t7756 1797\n",
      "62\t30625\t3.8\t1\t28085 2540\n",
      "63\t6646\t3.8\t1\t5279 1367\n",
      "64\t10476\t3.8\t1\t8893 1583\n",
      "65\t12129\t3.8\t1\t10311 1818\n",
      "66\t32496\t3.8\t1\t30129 2367\n",
      "67\t9348\t3.8\t1\t7566 1782\n",
      "68\t14758\t3.8\t1\t12818 1940\n",
      "69\t17456\t3.8\t1\t15424 2032\n",
      "70\t20192\t3.8\t1\t18162 2030\n",
      "71\t8547\t3.8\t1\t6817 1730\n",
      "72\t7395\t3.8\t1\t5884 1511\n",
      "73\t13127\t3.8\t1\t11639 1488\n",
      "74\t20669\t3.8\t1\t18903 1766\n",
      "75\t24963\t3.8\t1\t22902 2061\n",
      "76\t27047\t3.8\t1\t24893 2154\n",
      "77\t27841\t3.8\t1\t25567 2274\n",
      "78\t28052\t3.8\t1\t25700 2352\n",
      "79\t28023\t3.8\t1\t25545 2478\n",
      "80\t27896\t3.8\t1\t25399 2497\n",
      "81\t27733\t3.8\t1\t25291 2442\n",
      "82\t27390\t3.8\t1\t24949 2441\n",
      "83\t26697\t3.8\t1\t24265 2432\n",
      "84\t26722\t3.8\t1\t24290 2432\n",
      "85\t25927\t3.8\t1\t23433 2494\n",
      "86\t25310\t3.8\t1\t22865 2445\n",
      "87\t25046\t3.8\t1\t22605 2441\n",
      "88\t24529\t3.8\t1\t22179 2350\n",
      "89\t24483\t3.8\t1\t22077 2406\n",
      "90\t23988\t3.8\t1\t21625 2363\n",
      "91\t22741\t3.8\t1\t20617 2124\n",
      "92\t22201\t3.8\t1\t20056 2145\n",
      "93\t21627\t3.8\t1\t19471 2156\n",
      "94\t20771\t3.8\t1\t18675 2096\n",
      "95\t20353\t3.8\t1\t18316 2037\n",
      "96\t20197\t3.8\t1\t18120 2077\n",
      "97\t19416\t3.8\t1\t17499 1917\n",
      "98\t18926\t3.8\t1\t17057 1869\n",
      "99\t18141\t3.8\t1\t16272 1869\n",
      "100\t17620\t3.8\t1\t15816 1804\n",
      "101\t17151\t3.8\t1\t15427 1724\n",
      "102\t16211\t3.8\t1\t14534 1677\n",
      "103\t15119\t3.8\t1\t13497 1622\n",
      "104\t13828\t3.8\t1\t12253 1575\n",
      "105\t12712\t3.8\t1\t11213 1499\n",
      "106\t11640\t3.8\t1\t10264 1376\n",
      "107\t10351\t3.8\t1\t9001 1350\n",
      "108\t9064\t3.8\t1\t7742 1322\n",
      "109\t8038\t3.8\t1\t6793 1245\n",
      "110\t7411\t3.8\t1\t6192 1219\n",
      "111\t6953\t3.8\t1\t5785 1168\n",
      "112\t6592\t3.8\t1\t5468 1124\n",
      "113\t6501\t3.8\t1\t5458 1043\n",
      "114\t6362\t3.8\t1\t5319 1043\n",
      "115\t6222\t3.8\t1\t5249 973\n",
      "116\t5855\t3.8\t1\t4900 955\n",
      "117\t6026\t3.8\t1\t5076 950\n",
      "118\t5657\t3.8\t1\t4820 837\n",
      "119\t5546\t3.8\t1\t4708 838\n",
      "120\t5427\t3.8\t1\t4637 790\n",
      "121\t5303\t3.8\t1\t4551 752\n",
      "122\t4956\t3.8\t1\t4286 670\n",
      "123\t5000\t3.8\t1\t4290 710\n",
      "124\t4630\t3.8\t1\t4031 599\n",
      "125\t5010\t3.8\t1\t4393 617\n",
      "126\t4540\t3.8\t1\t3960 580\n",
      "127\t4791\t3.8\t1\t4274 517\n",
      "128\t4506\t3.8\t1\t4038 468\n",
      "129\t7579\t3.8\t1\t7009 570\n",
      "130\t4174\t3.8\t1\t3794 380\n",
      "131\t6400\t3.8\t1\t6002 398\n",
      "132\t4137\t3.8\t1\t3882 255\n",
      "133\t4484\t3.8\t1\t4219 265\n",
      "134\t3766\t3.8\t1\t3564 202\n",
      "135\t4268\t3.8\t1\t4046 222\n",
      "136\t3911\t3.8\t1\t3717 194\n",
      "137\t4096\t3.8\t1\t3915 181\n",
      "138\t3517\t3.8\t1\t3375 142\n",
      "139\t3308\t3.8\t1\t3157 151\n",
      "140\t3107\t3.8\t1\t2996 111\n",
      "141\t3026\t3.8\t1\t2901 125\n",
      "142\t2772\t3.8\t1\t2650 122\n",
      "143\t3177\t3.8\t1\t3054 123\n",
      "144\t2245\t3.8\t1\t2121 124\n",
      "145\t2434\t3.8\t1\t2302 132\n",
      "146\t2314\t3.8\t1\t2195 119\n",
      "147\t2036\t3.8\t1\t1898 138\n",
      "148\t1674\t3.8\t1\t1487 187\n",
      "149\t1931\t3.8\t1\t1679 252\n",
      "150\t4377\t3.8\t1\t3025 1352\n",
      "151\t379957\t3.8\t1\t180456 199501\n",
      "\n",
      "\n",
      "RUN STATISTICS FOR INPUT FILE: /home/sam/data/geoduck_illumina/trimmed/NR015_AD015_S6_L001_R1_001_val_1.fq.gz\n",
      "=============================================\n",
      "251960784 sequences processed in total\n",
      "The length threshold of paired-end sequences gets evaluated later on (in the validation step)\n",
      "\n",
      "Writing report to '/home/sam/data/geoduck_illumina/trimmed/20180129_trimmed_again/NR015_AD015_S6_L001_R2_001_val_2.fq.gz_trimming_report.txt'\n",
      "\n",
      "SUMMARISING RUN PARAMETERS\n",
      "==========================\n",
      "Input filename: /home/sam/data/geoduck_illumina/trimmed/NR015_AD015_S6_L001_R2_001_val_2.fq.gz\n",
      "Trimming mode: paired-end\n",
      "Trim Galore version: 0.4.4\n",
      "Cutadapt version: 1.9.1\n",
      "Quality Phred score cutoff: 20\n",
      "Quality encoding type selected: ASCII+33\n",
      "Adapter sequence: 'AGATCGGAAGAGC' (Illumina TruSeq, Sanger iPCR; auto-detected)\n",
      "Maximum trimming error rate: 0.1 (default)\n",
      "Minimum required adapter overlap (stringency): 1 bp\n",
      "Minimum required sequence length for both reads before a sequence pair gets removed: 20 bp\n",
      "Output file(s) will be GZIP compressed\n",
      "\n",
      "Writing final adapter and quality trimmed output to NR015_AD015_S6_L001_R2_001_val_2_trimmed.fq.gz\n",
      "\n",
      "\n",
      "  >>> Now performing quality (cutoff 20) and adapter trimming in a single pass for the adapter sequence: 'AGATCGGAAGAGC' from file /home/sam/data/geoduck_illumina/trimmed/NR015_AD015_S6_L001_R2_001_val_2.fq.gz <<< \n",
      "10000000 sequences processed\n",
      "20000000 sequences processed\n",
      "30000000 sequences processed\n",
      "40000000 sequences processed\n",
      "50000000 sequences processed\n",
      "60000000 sequences processed\n",
      "70000000 sequences processed\n",
      "80000000 sequences processed\n",
      "90000000 sequences processed\n",
      "100000000 sequences processed\n",
      "110000000 sequences processed\n",
      "120000000 sequences processed\n",
      "130000000 sequences processed\n",
      "140000000 sequences processed\n",
      "150000000 sequences processed\n",
      "160000000 sequences processed\n",
      "170000000 sequences processed\n",
      "180000000 sequences processed\n",
      "190000000 sequences processed\n",
      "200000000 sequences processed\n",
      "210000000 sequences processed\n",
      "220000000 sequences processed\n",
      "230000000 sequences processed\n",
      "240000000 sequences processed\n",
      "250000000 sequences processed\n",
      "This is cutadapt 1.9.1 with Python 2.7.12\n",
      "Command line parameters: -f fastq -e 0.1 -q 20 -O 1 -a AGATCGGAAGAGC /home/sam/data/geoduck_illumina/trimmed/NR015_AD015_S6_L001_R2_001_val_2.fq.gz\n",
      "Trimming 1 adapter with at most 10.0% errors in single-end mode ...\n",
      "Finished in 6193.60 s (25 us/read; 2.44 M reads/minute).\n",
      "\n",
      "=== Summary ===\n",
      "\n",
      "Total reads processed:             251,960,784\n",
      "Reads with adapters:               100,979,622 (40.1%)\n",
      "Reads written (passing filters):   251,960,784 (100.0%)\n",
      "\n",
      "Total basepairs processed: 31,155,879,786 bp\n",
      "Quality-trimmed:              21,545,872 bp (0.1%)\n",
      "Total written (filtered):  30,661,317,524 bp (98.4%)\n",
      "\n",
      "=== Adapter 1 ===\n",
      "\n",
      "Sequence: AGATCGGAAGAGC; Type: regular 3'; Length: 13; Trimmed: 100979622 times.\n",
      "\n",
      "No. of allowed errors:\n",
      "0-9 bp: 0; 10-13 bp: 1\n",
      "\n",
      "Bases preceding removed adapters:\n",
      "  A: 31.8%\n",
      "  C: 23.0%\n",
      "  G: 15.4%\n",
      "  T: 29.6%\n",
      "  none/other: 0.2%\n",
      "\n",
      "Overview of removed sequences\n",
      "length\tcount\texpect\tmax.err\terror counts\n",
      "1\t68313199\t62990196.0\t0\t68313199\n",
      "2\t18143166\t15747549.0\t0\t18143166\n",
      "3\t5124109\t3936887.2\t0\t5124109\n",
      "4\t1954723\t984221.8\t0\t1954723\n",
      "5\t188739\t246055.5\t0\t188739\n",
      "6\t129140\t61513.9\t0\t129140\n",
      "7\t108832\t15378.5\t0\t108832\n",
      "8\t128886\t3844.6\t0\t128886\n",
      "9\t91260\t961.2\t0\t87928 3332\n",
      "10\t126203\t240.3\t1\t104830 21373\n",
      "11\t115077\t60.1\t1\t96709 18368\n",
      "12\t226745\t15.0\t1\t199323 27422\n",
      "13\t2194\t3.8\t1\t959 1235\n",
      "14\t138238\t3.8\t1\t118664 19574\n",
      "15\t191130\t3.8\t1\t169718 21412\n",
      "16\t995\t3.8\t1\t353 642\n",
      "17\t129094\t3.8\t1\t113633 15461\n",
      "18\t79704\t3.8\t1\t69960 9744\n",
      "19\t138252\t3.8\t1\t121219 17033\n",
      "20\t100096\t3.8\t1\t87894 12202\n",
      "21\t86564\t3.8\t1\t75650 10914\n",
      "22\t101219\t3.8\t1\t89374 11845\n",
      "23\t107692\t3.8\t1\t94087 13605\n",
      "24\t189955\t3.8\t1\t167177 22778\n",
      "25\t98024\t3.8\t1\t87166 10858\n",
      "26\t98295\t3.8\t1\t86572 11723\n",
      "27\t64815\t3.8\t1\t54515 10300\n",
      "28\t155492\t3.8\t1\t141126 14366\n",
      "29\t40546\t3.8\t1\t32691 7855\n",
      "30\t146697\t3.8\t1\t132730 13967\n",
      "31\t60422\t3.8\t1\t51220 9202\n",
      "32\t268474\t3.8\t1\t249368 19106\n",
      "33\t110795\t3.8\t1\t97259 13536\n",
      "34\t38204\t3.8\t1\t31226 6978\n",
      "35\t143638\t3.8\t1\t131450 12188\n",
      "36\t47890\t3.8\t1\t41399 6491\n",
      "37\t72520\t3.8\t1\t63288 9232\n",
      "38\t37867\t3.8\t1\t33785 4082\n",
      "39\t50949\t3.8\t1\t44733 6216\n",
      "40\t20848\t3.8\t1\t18228 2620\n",
      "41\t79085\t3.8\t1\t71910 7175\n",
      "42\t111036\t3.8\t1\t101963 9073\n",
      "43\t50564\t3.8\t1\t44806 5758\n",
      "44\t82737\t3.8\t1\t74481 8256\n",
      "45\t151990\t3.8\t1\t139894 12096\n",
      "46\t71723\t3.8\t1\t63298 8425\n",
      "47\t48816\t3.8\t1\t44185 4631\n",
      "48\t148377\t3.8\t1\t136657 11720\n",
      "49\t61498\t3.8\t1\t55998 5500\n",
      "50\t23050\t3.8\t1\t20044 3006\n",
      "51\t119426\t3.8\t1\t109607 9819\n",
      "52\t183281\t3.8\t1\t172910 10371\n",
      "53\t71389\t3.8\t1\t66191 5198\n",
      "54\t89936\t3.8\t1\t82844 7092\n",
      "55\t52537\t3.8\t1\t48707 3830\n",
      "56\t25438\t3.8\t1\t22287 3151\n",
      "57\t46738\t3.8\t1\t42457 4281\n",
      "58\t50540\t3.8\t1\t45618 4922\n",
      "59\t28544\t3.8\t1\t25164 3380\n",
      "60\t18858\t3.8\t1\t16100 2758\n",
      "61\t17190\t3.8\t1\t14584 2606\n",
      "62\t18991\t3.8\t1\t16327 2664\n",
      "63\t22581\t3.8\t1\t19907 2674\n",
      "64\t38653\t3.8\t1\t35535 3118\n",
      "65\t48769\t3.8\t1\t44750 4019\n",
      "66\t52547\t3.8\t1\t48219 4328\n",
      "67\t52683\t3.8\t1\t48131 4552\n",
      "68\t52201\t3.8\t1\t47603 4598\n",
      "69\t52101\t3.8\t1\t47443 4658\n",
      "70\t51853\t3.8\t1\t47277 4576\n",
      "71\t51340\t3.8\t1\t46653 4687\n",
      "72\t50223\t3.8\t1\t45673 4550\n",
      "73\t49914\t3.8\t1\t45383 4531\n",
      "74\t48369\t3.8\t1\t43891 4478\n",
      "75\t47815\t3.8\t1\t43411 4404\n",
      "76\t46803\t3.8\t1\t42428 4375\n",
      "77\t46237\t3.8\t1\t42021 4216\n",
      "78\t44826\t3.8\t1\t40634 4192\n",
      "79\t43489\t3.8\t1\t39420 4069\n",
      "80\t43261\t3.8\t1\t39363 3898\n",
      "81\t42442\t3.8\t1\t38480 3962\n",
      "82\t41492\t3.8\t1\t37628 3864\n",
      "83\t40374\t3.8\t1\t36728 3646\n",
      "84\t39695\t3.8\t1\t36017 3678\n",
      "85\t39208\t3.8\t1\t35580 3628\n",
      "86\t38171\t3.8\t1\t34599 3572\n",
      "87\t38016\t3.8\t1\t34553 3463\n",
      "88\t36525\t3.8\t1\t33092 3433\n",
      "89\t41258\t3.8\t1\t33003 8255\n",
      "90\t34752\t3.8\t1\t31007 3745\n",
      "91\t33218\t3.8\t1\t30100 3118\n",
      "92\t32601\t3.8\t1\t29475 3126\n",
      "93\t31952\t3.8\t1\t28947 3005\n",
      "94\t31006\t3.8\t1\t28057 2949\n",
      "95\t30263\t3.8\t1\t27427 2836\n",
      "96\t29561\t3.8\t1\t26776 2785\n",
      "97\t28672\t3.8\t1\t25981 2691\n",
      "98\t27549\t3.8\t1\t24930 2619\n",
      "99\t26632\t3.8\t1\t24067 2565\n",
      "100\t25886\t3.8\t1\t23427 2459\n",
      "101\t24490\t3.8\t1\t22115 2375\n",
      "102\t23311\t3.8\t1\t21015 2296\n",
      "103\t21482\t3.8\t1\t19313 2169\n",
      "104\t19671\t3.8\t1\t17609 2062\n",
      "105\t17882\t3.8\t1\t15906 1976\n",
      "106\t15683\t3.8\t1\t13909 1774\n",
      "107\t13366\t3.8\t1\t11584 1782\n",
      "108\t11524\t3.8\t1\t9775 1749\n",
      "109\t9805\t3.8\t1\t8171 1634\n",
      "110\t8591\t3.8\t1\t7096 1495\n",
      "111\t8086\t3.8\t1\t6610 1476\n",
      "112\t7566\t3.8\t1\t6240 1326\n",
      "113\t7509\t3.8\t1\t6118 1391\n",
      "114\t7162\t3.8\t1\t5878 1284\n",
      "115\t7027\t3.8\t1\t5812 1215\n",
      "116\t6711\t3.8\t1\t5506 1205\n",
      "117\t6602\t3.8\t1\t5438 1164\n",
      "118\t6295\t3.8\t1\t5252 1043\n",
      "119\t6307\t3.8\t1\t5204 1103\n",
      "120\t6026\t3.8\t1\t5073 953\n",
      "121\t5936\t3.8\t1\t5017 919\n",
      "122\t5503\t3.8\t1\t4654 849\n",
      "123\t5535\t3.8\t1\t4715 820\n",
      "124\t5175\t3.8\t1\t4422 753\n",
      "125\t5396\t3.8\t1\t4729 667\n",
      "126\t5052\t3.8\t1\t4370 682\n",
      "127\t5194\t3.8\t1\t4586 608\n",
      "128\t4771\t3.8\t1\t4258 513\n",
      "129\t7872\t3.8\t1\t7315 557\n",
      "130\t4352\t3.8\t1\t3923 429\n",
      "131\t6565\t3.8\t1\t6168 397\n",
      "132\t4192\t3.8\t1\t3862 330\n",
      "133\t4550\t3.8\t1\t4240 310\n",
      "134\t3843\t3.8\t1\t3560 283\n",
      "135\t4287\t3.8\t1\t4041 246\n",
      "136\t3693\t3.8\t1\t3402 291\n",
      "137\t4048\t3.8\t1\t3789 259\n",
      "138\t3506\t3.8\t1\t3254 252\n",
      "139\t3304\t3.8\t1\t3036 268\n",
      "140\t3120\t3.8\t1\t2843 277\n",
      "141\t3010\t3.8\t1\t2684 326\n",
      "142\t2563\t3.8\t1\t2214 349\n",
      "143\t2946\t3.8\t1\t2557 389\n",
      "144\t2247\t3.8\t1\t1857 390\n",
      "145\t2380\t3.8\t1\t1931 449\n",
      "146\t1903\t3.8\t1\t1439 464\n",
      "147\t1848\t3.8\t1\t1414 434\n",
      "148\t1435\t3.8\t1\t985 450\n",
      "149\t1587\t3.8\t1\t1131 456\n",
      "150\t3495\t3.8\t1\t2865 630\n",
      "151\t169848\t3.8\t1\t165158 4690\n",
      "\n",
      "\n",
      "RUN STATISTICS FOR INPUT FILE: /home/sam/data/geoduck_illumina/trimmed/NR015_AD015_S6_L001_R2_001_val_2.fq.gz\n",
      "=============================================\n",
      "251960784 sequences processed in total\n",
      "The length threshold of paired-end sequences gets evaluated later on (in the validation step)\n",
      "\n",
      "Validate paired-end files NR015_AD015_S6_L001_R1_001_val_1_trimmed.fq.gz and NR015_AD015_S6_L001_R2_001_val_2_trimmed.fq.gz\n",
      "file_1: NR015_AD015_S6_L001_R1_001_val_1_trimmed.fq.gz, file_2: NR015_AD015_S6_L001_R2_001_val_2_trimmed.fq.gz\n",
      "\n",
      "\n",
      ">>>>> Now validing the length of the 2 paired-end infiles: NR015_AD015_S6_L001_R1_001_val_1_trimmed.fq.gz and NR015_AD015_S6_L001_R2_001_val_2_trimmed.fq.gz <<<<<\n",
      "Writing validated paired-end read 1 reads to NR015_AD015_S6_L001_R1_001_val_1_val_1.fq.gz\n",
      "Writing validated paired-end read 2 reads to NR015_AD015_S6_L001_R2_001_val_2_val_2.fq.gz\n",
      "\n",
      "Total number of sequences analysed: 251960784\n",
      "\n",
      "Number of sequence pairs removed because at least one read was shorter than the length cutoff (20 bp): 1195829 (0.47%)\n",
      "\n",
      "Deleting both intermediate output files NR015_AD015_S6_L001_R1_001_val_1_trimmed.fq.gz and NR015_AD015_S6_L001_R2_001_val_2_trimmed.fq.gz\n",
      "\n",
      "====================================================================================================\n",
      "\n",
      "Writing report to '/home/sam/data/geoduck_illumina/trimmed/20180129_trimmed_again/NR015_AD015_S6_L002_R1_001_val_1.fq.gz_trimming_report.txt'\n",
      "\n",
      "SUMMARISING RUN PARAMETERS\n",
      "==========================\n",
      "Input filename: /home/sam/data/geoduck_illumina/trimmed/NR015_AD015_S6_L002_R1_001_val_1.fq.gz\n",
      "Trimming mode: paired-end\n",
      "Trim Galore version: 0.4.4\n",
      "Cutadapt version: 1.9.1\n",
      "Quality Phred score cutoff: 20\n",
      "Quality encoding type selected: ASCII+33\n",
      "Adapter sequence: 'AGATCGGAAGAGC' (Illumina TruSeq, Sanger iPCR; auto-detected)\n",
      "Maximum trimming error rate: 0.1 (default)\n",
      "Minimum required adapter overlap (stringency): 1 bp\n",
      "Minimum required sequence length for both reads before a sequence pair gets removed: 20 bp\n",
      "Output file(s) will be GZIP compressed\n",
      "\n",
      "Writing final adapter and quality trimmed output to NR015_AD015_S6_L002_R1_001_val_1_trimmed.fq.gz\n",
      "\n",
      "\n",
      "  >>> Now performing quality (cutoff 20) and adapter trimming in a single pass for the adapter sequence: 'AGATCGGAAGAGC' from file /home/sam/data/geoduck_illumina/trimmed/NR015_AD015_S6_L002_R1_001_val_1.fq.gz <<< \n",
      "10000000 sequences processed\n",
      "20000000 sequences processed\n",
      "30000000 sequences processed\n",
      "40000000 sequences processed\n",
      "50000000 sequences processed\n",
      "60000000 sequences processed\n",
      "70000000 sequences processed\n",
      "80000000 sequences processed\n",
      "90000000 sequences processed\n",
      "100000000 sequences processed\n",
      "110000000 sequences processed\n",
      "120000000 sequences processed\n",
      "130000000 sequences processed\n",
      "140000000 sequences processed\n",
      "150000000 sequences processed\n",
      "160000000 sequences processed\n",
      "170000000 sequences processed\n",
      "180000000 sequences processed\n",
      "190000000 sequences processed\n",
      "200000000 sequences processed\n",
      "210000000 sequences processed\n",
      "220000000 sequences processed\n",
      "230000000 sequences processed\n",
      "240000000 sequences processed\n",
      "250000000 sequences processed\n",
      "260000000 sequences processed\n",
      "270000000 sequences processed\n",
      "280000000 sequences processed\n",
      "This is cutadapt 1.9.1 with Python 2.7.12\n",
      "Command line parameters: -f fastq -e 0.1 -q 20 -O 1 -a AGATCGGAAGAGC /home/sam/data/geoduck_illumina/trimmed/NR015_AD015_S6_L002_R1_001_val_1.fq.gz\n",
      "Trimming 1 adapter with at most 10.0% errors in single-end mode ...\n",
      "Finished in 6925.44 s (24 us/read; 2.48 M reads/minute).\n",
      "\n",
      "=== Summary ===\n",
      "\n",
      "Total reads processed:             286,345,514\n",
      "Reads with adapters:               112,959,695 (39.4%)\n",
      "Reads written (passing filters):   286,345,514 (100.0%)\n",
      "\n",
      "Total basepairs processed: 35,038,018,789 bp\n",
      "Quality-trimmed:              13,805,282 bp (0.0%)\n",
      "Total written (filtered):  34,518,982,225 bp (98.5%)\n",
      "\n",
      "=== Adapter 1 ===\n",
      "\n",
      "Sequence: AGATCGGAAGAGC; Type: regular 3'; Length: 13; Trimmed: 112959695 times.\n",
      "\n",
      "No. of allowed errors:\n",
      "0-9 bp: 0; 10-13 bp: 1\n",
      "\n",
      "Bases preceding removed adapters:\n",
      "  A: 31.6%\n",
      "  C: 23.5%\n",
      "  G: 14.7%\n",
      "  T: 29.7%\n",
      "  none/other: 0.5%\n",
      "\n",
      "Overview of removed sequences\n",
      "length\tcount\texpect\tmax.err\terror counts\n",
      "1\t78396135\t71586378.5\t0\t78396135\n",
      "2\t20488392\t17896594.6\t0\t20488392\n",
      "3\t5379582\t4474148.7\t0\t5379582\n",
      "4\t2080584\t1118537.2\t0\t2080584\n",
      "5\t223491\t279634.3\t0\t223491\n",
      "6\t100154\t69908.6\t0\t100154\n",
      "7\t75732\t17477.1\t0\t75732\n",
      "8\t86633\t4369.3\t0\t86633\n",
      "9\t80998\t1092.3\t0\t77642 3356\n",
      "10\t88682\t273.1\t1\t74287 14395\n",
      "11\t90629\t68.3\t1\t77841 12788\n",
      "12\t161905\t17.1\t1\t148361 13544\n",
      "13\t4301\t4.3\t1\t2285 2016\n",
      "14\t160121\t4.3\t1\t146934 13187\n",
      "15\t3804\t4.3\t1\t1935 1869\n",
      "16\t171383\t4.3\t1\t153699 17684\n",
      "17\t7255\t4.3\t1\t4345 2910\n",
      "18\t107431\t4.3\t1\t96486 10945\n",
      "19\t265068\t4.3\t1\t247099 17969\n",
      "20\t1979\t4.3\t1\t1118 861\n",
      "21\t1742\t4.3\t1\t801 941\n",
      "22\t3763\t4.3\t1\t2101 1662\n",
      "23\t95718\t4.3\t1\t85159 10559\n",
      "24\t206250\t4.3\t1\t190203 16047\n",
      "25\t6546\t4.3\t1\t4203 2343\n",
      "26\t78594\t4.3\t1\t70732 7862\n",
      "27\t70370\t4.3\t1\t65520 4850\n",
      "28\t2519\t4.3\t1\t1512 1007\n",
      "29\t75709\t4.3\t1\t67862 7847\n",
      "30\t91570\t4.3\t1\t83304 8266\n",
      "31\t117793\t4.3\t1\t109087 8706\n",
      "32\t6339\t4.3\t1\t4304 2035\n",
      "33\t137342\t4.3\t1\t124884 12458\n",
      "34\t3254\t4.3\t1\t2205 1049\n",
      "35\t75697\t4.3\t1\t66263 9434\n",
      "36\t70974\t4.3\t1\t63418 7556\n",
      "37\t76030\t4.3\t1\t68817 7213\n",
      "38\t118275\t4.3\t1\t108426 9849\n",
      "39\t2841\t4.3\t1\t1941 900\n",
      "40\t69529\t4.3\t1\t61210 8319\n",
      "41\t68776\t4.3\t1\t60720 8056\n",
      "42\t82760\t4.3\t1\t70525 12235\n",
      "43\t94151\t4.3\t1\t77798 16353\n",
      "44\t204930\t4.3\t1\t180380 24550\n",
      "45\t38013\t4.3\t1\t33457 4556\n",
      "46\t54282\t4.3\t1\t44999 9283\n",
      "47\t11755\t4.3\t1\t9625 2130\n",
      "48\t492920\t4.3\t1\t471229 21691\n",
      "49\t144283\t4.3\t1\t132748 11535\n",
      "50\t29210\t4.3\t1\t23840 5370\n",
      "51\t111750\t4.3\t1\t103977 7773\n",
      "52\t84336\t4.3\t1\t76899 7437\n",
      "53\t14904\t4.3\t1\t11273 3631\n",
      "54\t6028\t4.3\t1\t4207 1821\n",
      "55\t38270\t4.3\t1\t34920 3350\n",
      "56\t20143\t4.3\t1\t16575 3568\n",
      "57\t4741\t4.3\t1\t3048 1693\n",
      "58\t9097\t4.3\t1\t6744 2353\n",
      "59\t20031\t4.3\t1\t17035 2996\n",
      "60\t6411\t4.3\t1\t4486 1925\n",
      "61\t9287\t4.3\t1\t6932 2355\n",
      "62\t31197\t4.3\t1\t28051 3146\n",
      "63\t5622\t4.3\t1\t4034 1588\n",
      "64\t9211\t4.3\t1\t7161 2050\n",
      "65\t14746\t4.3\t1\t12258 2488\n",
      "66\t33048\t4.3\t1\t29926 3122\n",
      "67\t11214\t4.3\t1\t8789 2425\n",
      "68\t24371\t4.3\t1\t21564 2807\n",
      "69\t30605\t4.3\t1\t27502 3103\n",
      "70\t44875\t4.3\t1\t41184 3691\n",
      "71\t17999\t4.3\t1\t15248 2751\n",
      "72\t8543\t4.3\t1\t6252 2291\n",
      "73\t12632\t4.3\t1\t10441 2191\n",
      "74\t22706\t4.3\t1\t20346 2360\n",
      "75\t30225\t4.3\t1\t27494 2731\n",
      "76\t33537\t4.3\t1\t30537 3000\n",
      "77\t35713\t4.3\t1\t32516 3197\n",
      "78\t36158\t4.3\t1\t32668 3490\n",
      "79\t36710\t4.3\t1\t33136 3574\n",
      "80\t36902\t4.3\t1\t33217 3685\n",
      "81\t36541\t4.3\t1\t32897 3644\n",
      "82\t36501\t4.3\t1\t32621 3880\n",
      "83\t35623\t4.3\t1\t31806 3817\n",
      "84\t35658\t4.3\t1\t31851 3807\n",
      "85\t35471\t4.3\t1\t31655 3816\n",
      "86\t34536\t4.3\t1\t30760 3776\n",
      "87\t33782\t4.3\t1\t30227 3555\n",
      "88\t33922\t4.3\t1\t30047 3875\n",
      "89\t33565\t4.3\t1\t29789 3776\n",
      "90\t33858\t4.3\t1\t30148 3710\n",
      "91\t32373\t4.3\t1\t28635 3738\n",
      "92\t31605\t4.3\t1\t28042 3563\n",
      "93\t30634\t4.3\t1\t27154 3480\n",
      "94\t30011\t4.3\t1\t26517 3494\n",
      "95\t29452\t4.3\t1\t25995 3457\n",
      "96\t29070\t4.3\t1\t25623 3447\n",
      "97\t28471\t4.3\t1\t25113 3358\n",
      "98\t27941\t4.3\t1\t24728 3213\n",
      "99\t27386\t4.3\t1\t24187 3199\n",
      "100\t26522\t4.3\t1\t23386 3136\n",
      "101\t25249\t4.3\t1\t22180 3069\n",
      "102\t24525\t4.3\t1\t21350 3175\n",
      "103\t23581\t4.3\t1\t20579 3002\n",
      "104\t22163\t4.3\t1\t19173 2990\n",
      "105\t20739\t4.3\t1\t17886 2853\n",
      "106\t19555\t4.3\t1\t16798 2757\n",
      "107\t17970\t4.3\t1\t15280 2690\n",
      "108\t16213\t4.3\t1\t13719 2494\n",
      "109\t14971\t4.3\t1\t12491 2480\n",
      "110\t14369\t4.3\t1\t11949 2420\n",
      "111\t13676\t4.3\t1\t11409 2267\n",
      "112\t13279\t4.3\t1\t10889 2390\n",
      "113\t12986\t4.3\t1\t10823 2163\n",
      "114\t12529\t4.3\t1\t10316 2213\n",
      "115\t12359\t4.3\t1\t10273 2086\n",
      "116\t11824\t4.3\t1\t9721 2103\n",
      "117\t11521\t4.3\t1\t9680 1841\n",
      "118\t11341\t4.3\t1\t9443 1898\n",
      "119\t10814\t4.3\t1\t9101 1713\n",
      "120\t10536\t4.3\t1\t8822 1714\n",
      "121\t10223\t4.3\t1\t8608 1615\n",
      "122\t9654\t4.3\t1\t8123 1531\n",
      "123\t9335\t4.3\t1\t7885 1450\n",
      "124\t8797\t4.3\t1\t7489 1308\n",
      "125\t8983\t4.3\t1\t7661 1322\n",
      "126\t8427\t4.3\t1\t7191 1236\n",
      "127\t8872\t4.3\t1\t7688 1184\n",
      "128\t7881\t4.3\t1\t6883 998\n",
      "129\t11912\t4.3\t1\t10868 1044\n",
      "130\t7024\t4.3\t1\t6278 746\n",
      "131\t10003\t4.3\t1\t9240 763\n",
      "132\t6500\t4.3\t1\t5991 509\n",
      "133\t6890\t4.3\t1\t6514 376\n",
      "134\t5695\t4.3\t1\t5351 344\n",
      "135\t6358\t4.3\t1\t6012 346\n",
      "136\t5449\t4.3\t1\t5171 278\n",
      "137\t5778\t4.3\t1\t5554 224\n",
      "138\t4758\t4.3\t1\t4570 188\n",
      "139\t4546\t4.3\t1\t4383 163\n",
      "140\t4169\t4.3\t1\t3998 171\n",
      "141\t3895\t4.3\t1\t3759 136\n",
      "142\t3337\t4.3\t1\t3215 122\n",
      "143\t3888\t4.3\t1\t3761 127\n",
      "144\t3019\t4.3\t1\t2893 126\n",
      "145\t3195\t4.3\t1\t3069 126\n",
      "146\t2750\t4.3\t1\t2613 137\n",
      "147\t2411\t4.3\t1\t2277 134\n",
      "148\t1979\t4.3\t1\t1816 163\n",
      "149\t2200\t4.3\t1\t2002 198\n",
      "150\t3790\t4.3\t1\t2743 1047\n",
      "151\t507649\t4.3\t1\t240906 266743\n",
      "\n",
      "\n",
      "RUN STATISTICS FOR INPUT FILE: /home/sam/data/geoduck_illumina/trimmed/NR015_AD015_S6_L002_R1_001_val_1.fq.gz\n",
      "=============================================\n",
      "286345514 sequences processed in total\n",
      "The length threshold of paired-end sequences gets evaluated later on (in the validation step)\n",
      "\n",
      "Writing report to '/home/sam/data/geoduck_illumina/trimmed/20180129_trimmed_again/NR015_AD015_S6_L002_R2_001_val_2.fq.gz_trimming_report.txt'\n",
      "\n",
      "SUMMARISING RUN PARAMETERS\n",
      "==========================\n",
      "Input filename: /home/sam/data/geoduck_illumina/trimmed/NR015_AD015_S6_L002_R2_001_val_2.fq.gz\n",
      "Trimming mode: paired-end\n",
      "Trim Galore version: 0.4.4\n",
      "Cutadapt version: 1.9.1\n",
      "Quality Phred score cutoff: 20\n",
      "Quality encoding type selected: ASCII+33\n",
      "Adapter sequence: 'AGATCGGAAGAGC' (Illumina TruSeq, Sanger iPCR; auto-detected)\n",
      "Maximum trimming error rate: 0.1 (default)\n",
      "Minimum required adapter overlap (stringency): 1 bp\n",
      "Minimum required sequence length for both reads before a sequence pair gets removed: 20 bp\n",
      "Output file(s) will be GZIP compressed\n",
      "\n",
      "Writing final adapter and quality trimmed output to NR015_AD015_S6_L002_R2_001_val_2_trimmed.fq.gz\n",
      "\n",
      "\n",
      "  >>> Now performing quality (cutoff 20) and adapter trimming in a single pass for the adapter sequence: 'AGATCGGAAGAGC' from file /home/sam/data/geoduck_illumina/trimmed/NR015_AD015_S6_L002_R2_001_val_2.fq.gz <<< \n",
      "10000000 sequences processed\n",
      "20000000 sequences processed\n",
      "30000000 sequences processed\n",
      "40000000 sequences processed\n",
      "50000000 sequences processed\n",
      "60000000 sequences processed\n",
      "70000000 sequences processed\n",
      "80000000 sequences processed\n",
      "90000000 sequences processed\n",
      "100000000 sequences processed\n",
      "110000000 sequences processed\n",
      "120000000 sequences processed\n",
      "130000000 sequences processed\n",
      "140000000 sequences processed\n",
      "150000000 sequences processed\n",
      "160000000 sequences processed\n",
      "170000000 sequences processed\n",
      "180000000 sequences processed\n",
      "190000000 sequences processed\n",
      "200000000 sequences processed\n",
      "210000000 sequences processed\n",
      "220000000 sequences processed\n",
      "230000000 sequences processed\n",
      "240000000 sequences processed\n",
      "250000000 sequences processed\n",
      "260000000 sequences processed\n",
      "270000000 sequences processed\n",
      "280000000 sequences processed\n",
      "This is cutadapt 1.9.1 with Python 2.7.12\n",
      "Command line parameters: -f fastq -e 0.1 -q 20 -O 1 -a AGATCGGAAGAGC /home/sam/data/geoduck_illumina/trimmed/NR015_AD015_S6_L002_R2_001_val_2.fq.gz\n",
      "Trimming 1 adapter with at most 10.0% errors in single-end mode ...\n",
      "Finished in 7061.45 s (25 us/read; 2.43 M reads/minute).\n",
      "\n",
      "=== Summary ===\n",
      "\n",
      "Total reads processed:             286,345,514\n",
      "Reads with adapters:               115,866,451 (40.5%)\n",
      "Reads written (passing filters):   286,345,514 (100.0%)\n",
      "\n",
      "Total basepairs processed: 35,998,085,991 bp\n",
      "Quality-trimmed:              19,015,689 bp (0.1%)\n",
      "Total written (filtered):  35,401,915,461 bp (98.3%)\n",
      "\n",
      "=== Adapter 1 ===\n",
      "\n",
      "Sequence: AGATCGGAAGAGC; Type: regular 3'; Length: 13; Trimmed: 115866451 times.\n",
      "\n",
      "No. of allowed errors:\n",
      "0-9 bp: 0; 10-13 bp: 1\n",
      "\n",
      "Bases preceding removed adapters:\n",
      "  A: 31.7%\n",
      "  C: 23.1%\n",
      "  G: 15.2%\n",
      "  T: 29.8%\n",
      "  none/other: 0.2%\n",
      "\n",
      "Overview of removed sequences\n",
      "length\tcount\texpect\tmax.err\terror counts\n",
      "1\t79078292\t71586378.5\t0\t79078292\n",
      "2\t20457949\t17896594.6\t0\t20457949\n",
      "3\t5667589\t4474148.7\t0\t5667589\n",
      "4\t2136345\t1118537.2\t0\t2136345\n",
      "5\t203396\t279634.3\t0\t203396\n",
      "6\t132501\t69908.6\t0\t132501\n",
      "7\t111365\t17477.1\t0\t111365\n",
      "8\t128366\t4369.3\t0\t128366\n",
      "9\t94532\t1092.3\t0\t90994 3538\n",
      "10\t126018\t273.1\t1\t105952 20066\n",
      "11\t117446\t68.3\t1\t100812 16634\n",
      "12\t229409\t17.1\t1\t207524 21885\n",
      "13\t1726\t4.3\t1\t637 1089\n",
      "14\t137420\t4.3\t1\t121594 15826\n",
      "15\t195930\t4.3\t1\t179871 16059\n",
      "16\t1019\t4.3\t1\t406 613\n",
      "17\t129235\t4.3\t1\t116497 12738\n",
      "18\t83174\t4.3\t1\t75604 7570\n",
      "19\t134882\t4.3\t1\t120454 14428\n",
      "20\t102012\t4.3\t1\t92451 9561\n",
      "21\t94539\t4.3\t1\t85162 9377\n",
      "22\t104963\t4.3\t1\t94978 9985\n",
      "23\t109241\t4.3\t1\t97982 11259\n",
      "24\t169194\t4.3\t1\t149865 19329\n",
      "25\t108190\t4.3\t1\t98765 9425\n",
      "26\t106700\t4.3\t1\t97468 9232\n",
      "27\t72292\t4.3\t1\t63329 8963\n",
      "28\t159462\t4.3\t1\t147411 12051\n",
      "29\t47180\t4.3\t1\t39638 7542\n",
      "30\t153244\t4.3\t1\t140978 12266\n",
      "31\t63196\t4.3\t1\t55159 8037\n",
      "32\t317234\t4.3\t1\t300225 17009\n",
      "33\t121994\t4.3\t1\t109974 12020\n",
      "34\t43615\t4.3\t1\t36635 6980\n",
      "35\t170310\t4.3\t1\t159614 10696\n",
      "36\t39610\t4.3\t1\t33583 6027\n",
      "37\t87306\t4.3\t1\t77969 9337\n",
      "38\t24050\t4.3\t1\t20856 3194\n",
      "39\t45657\t4.3\t1\t39827 5830\n",
      "40\t11498\t4.3\t1\t9445 2053\n",
      "41\t86757\t4.3\t1\t79909 6848\n",
      "42\t126694\t4.3\t1\t117946 8748\n",
      "43\t49611\t4.3\t1\t44116 5495\n",
      "44\t91218\t4.3\t1\t83312 7906\n",
      "45\t219772\t4.3\t1\t207168 12604\n",
      "46\t80363\t4.3\t1\t71838 8525\n",
      "47\t41363\t4.3\t1\t37265 4098\n",
      "48\t199800\t4.3\t1\t187096 12704\n",
      "49\t51044\t4.3\t1\t45831 5213\n",
      "50\t15676\t4.3\t1\t13062 2614\n",
      "51\t148760\t4.3\t1\t138677 10083\n",
      "52\t266390\t4.3\t1\t255506 10884\n",
      "53\t63589\t4.3\t1\t58598 4991\n",
      "54\t90051\t4.3\t1\t83057 6994\n",
      "55\t36487\t4.3\t1\t32986 3501\n",
      "56\t21520\t4.3\t1\t18377 3143\n",
      "57\t40871\t4.3\t1\t36643 4228\n",
      "58\t47319\t4.3\t1\t42462 4857\n",
      "59\t22625\t4.3\t1\t19152 3473\n",
      "60\t17420\t4.3\t1\t14369 3051\n",
      "61\t17831\t4.3\t1\t14783 3048\n",
      "62\t17975\t4.3\t1\t15025 2950\n",
      "63\t21863\t4.3\t1\t18777 3086\n",
      "64\t41409\t4.3\t1\t37784 3625\n",
      "65\t56629\t4.3\t1\t52276 4353\n",
      "66\t61832\t4.3\t1\t56984 4848\n",
      "67\t62658\t4.3\t1\t57537 5121\n",
      "68\t62434\t4.3\t1\t56987 5447\n",
      "69\t62709\t4.3\t1\t57086 5623\n",
      "70\t62901\t4.3\t1\t57266 5635\n",
      "71\t63182\t4.3\t1\t57483 5699\n",
      "72\t61596\t4.3\t1\t55910 5686\n",
      "73\t61973\t4.3\t1\t56417 5556\n",
      "74\t60575\t4.3\t1\t54925 5650\n",
      "75\t59734\t4.3\t1\t54144 5590\n",
      "76\t58840\t4.3\t1\t53256 5584\n",
      "77\t58095\t4.3\t1\t52527 5568\n",
      "78\t56618\t4.3\t1\t51156 5462\n",
      "79\t55726\t4.3\t1\t50288 5438\n",
      "80\t55092\t4.3\t1\t49744 5348\n",
      "81\t53834\t4.3\t1\t48554 5280\n",
      "82\t53585\t4.3\t1\t48277 5308\n",
      "83\t52377\t4.3\t1\t47209 5168\n",
      "84\t51678\t4.3\t1\t46445 5233\n",
      "85\t51400\t4.3\t1\t46389 5011\n",
      "86\t50090\t4.3\t1\t45163 4927\n",
      "87\t49563\t4.3\t1\t44686 4877\n",
      "88\t49536\t4.3\t1\t44418 5118\n",
      "89\t54300\t4.3\t1\t43261 11039\n",
      "90\t46966\t4.3\t1\t41887 5079\n",
      "91\t45879\t4.3\t1\t41024 4855\n",
      "92\t44543\t4.3\t1\t39873 4670\n",
      "93\t43755\t4.3\t1\t39186 4569\n",
      "94\t42755\t4.3\t1\t38214 4541\n",
      "95\t42335\t4.3\t1\t37965 4370\n",
      "96\t41082\t4.3\t1\t36743 4339\n",
      "97\t40279\t4.3\t1\t36050 4229\n",
      "98\t39428\t4.3\t1\t35127 4301\n",
      "99\t38622\t4.3\t1\t34464 4158\n",
      "100\t37168\t4.3\t1\t33149 4019\n",
      "101\t35893\t4.3\t1\t31952 3941\n",
      "102\t34270\t4.3\t1\t30464 3806\n",
      "103\t33129\t4.3\t1\t29302 3827\n",
      "104\t30511\t4.3\t1\t26863 3648\n",
      "105\t28506\t4.3\t1\t24942 3564\n",
      "106\t26155\t4.3\t1\t22782 3373\n",
      "107\t23714\t4.3\t1\t20265 3449\n",
      "108\t21008\t4.3\t1\t17830 3178\n",
      "109\t18868\t4.3\t1\t15779 3089\n",
      "110\t17607\t4.3\t1\t14520 3087\n",
      "111\t16874\t4.3\t1\t13908 2966\n",
      "112\t16298\t4.3\t1\t13432 2866\n",
      "113\t16100\t4.3\t1\t13251 2849\n",
      "114\t15299\t4.3\t1\t12620 2679\n",
      "115\t15136\t4.3\t1\t12454 2682\n",
      "116\t14617\t4.3\t1\t12112 2505\n",
      "117\t13972\t4.3\t1\t11656 2316\n",
      "118\t13701\t4.3\t1\t11408 2293\n",
      "119\t13361\t4.3\t1\t11090 2271\n",
      "120\t12786\t4.3\t1\t10659 2127\n",
      "121\t12428\t4.3\t1\t10405 2023\n",
      "122\t11591\t4.3\t1\t9643 1948\n",
      "123\t11405\t4.3\t1\t9595 1810\n",
      "124\t10712\t4.3\t1\t9030 1682\n",
      "125\t10763\t4.3\t1\t9166 1597\n",
      "126\t10062\t4.3\t1\t8468 1594\n",
      "127\t10400\t4.3\t1\t9011 1389\n",
      "128\t8928\t4.3\t1\t7828 1100\n",
      "129\t12877\t4.3\t1\t11900 977\n",
      "130\t7703\t4.3\t1\t6939 764\n",
      "131\t10433\t4.3\t1\t9779 654\n",
      "132\t6733\t4.3\t1\t6237 496\n",
      "133\t7084\t4.3\t1\t6711 373\n",
      "134\t5808\t4.3\t1\t5437 371\n",
      "135\t6472\t4.3\t1\t6149 323\n",
      "136\t5338\t4.3\t1\t5038 300\n",
      "137\t5793\t4.3\t1\t5496 297\n",
      "138\t4805\t4.3\t1\t4489 316\n",
      "139\t4583\t4.3\t1\t4269 314\n",
      "140\t4150\t4.3\t1\t3832 318\n",
      "141\t3861\t4.3\t1\t3532 329\n",
      "142\t3205\t4.3\t1\t2860 345\n",
      "143\t3735\t4.3\t1\t3317 418\n",
      "144\t2991\t4.3\t1\t2562 429\n",
      "145\t3152\t4.3\t1\t2627 525\n",
      "146\t2370\t4.3\t1\t1827 543\n",
      "147\t2265\t4.3\t1\t1773 492\n",
      "148\t1796\t4.3\t1\t1247 549\n",
      "149\t1873\t4.3\t1\t1345 528\n",
      "150\t2305\t4.3\t1\t1737 568\n",
      "151\t228767\t4.3\t1\t224062 4705\n",
      "\n",
      "\n",
      "RUN STATISTICS FOR INPUT FILE: /home/sam/data/geoduck_illumina/trimmed/NR015_AD015_S6_L002_R2_001_val_2.fq.gz\n",
      "=============================================\n",
      "286345514 sequences processed in total\n",
      "The length threshold of paired-end sequences gets evaluated later on (in the validation step)\n",
      "\n",
      "Validate paired-end files NR015_AD015_S6_L002_R1_001_val_1_trimmed.fq.gz and NR015_AD015_S6_L002_R2_001_val_2_trimmed.fq.gz\n",
      "file_1: NR015_AD015_S6_L002_R1_001_val_1_trimmed.fq.gz, file_2: NR015_AD015_S6_L002_R2_001_val_2_trimmed.fq.gz\n",
      "\n",
      "\n",
      ">>>>> Now validing the length of the 2 paired-end infiles: NR015_AD015_S6_L002_R1_001_val_1_trimmed.fq.gz and NR015_AD015_S6_L002_R2_001_val_2_trimmed.fq.gz <<<<<\n",
      "Writing validated paired-end read 1 reads to NR015_AD015_S6_L002_R1_001_val_1_val_1.fq.gz\n",
      "Writing validated paired-end read 2 reads to NR015_AD015_S6_L002_R2_001_val_2_val_2.fq.gz\n",
      "\n",
      "Total number of sequences analysed: 286345514\n",
      "\n",
      "Number of sequence pairs removed because at least one read was shorter than the length cutoff (20 bp): 1343740 (0.47%)\n",
      "\n",
      "Deleting both intermediate output files NR015_AD015_S6_L002_R1_001_val_1_trimmed.fq.gz and NR015_AD015_S6_L002_R2_001_val_2_trimmed.fq.gz\n",
      "\n",
      "====================================================================================================\n",
      "\n",
      "Writing report to '/home/sam/data/geoduck_illumina/trimmed/20180129_trimmed_again/NR019_S7_L001_R1_001_val_1.fq.gz_trimming_report.txt'\n",
      "\n",
      "SUMMARISING RUN PARAMETERS\n",
      "==========================\n",
      "Input filename: /home/sam/data/geoduck_illumina/trimmed/NR019_S7_L001_R1_001_val_1.fq.gz\n",
      "Trimming mode: paired-end\n",
      "Trim Galore version: 0.4.4\n",
      "Cutadapt version: 1.9.1\n",
      "Quality Phred score cutoff: 20\n",
      "Quality encoding type selected: ASCII+33\n",
      "Adapter sequence: 'AGATCGGAAGAGC' (Illumina TruSeq, Sanger iPCR; auto-detected)\n",
      "Maximum trimming error rate: 0.1 (default)\n",
      "Minimum required adapter overlap (stringency): 1 bp\n",
      "Minimum required sequence length for both reads before a sequence pair gets removed: 20 bp\n",
      "Output file(s) will be GZIP compressed\n",
      "\n",
      "Writing final adapter and quality trimmed output to NR019_S7_L001_R1_001_val_1_trimmed.fq.gz\n",
      "\n",
      "\n",
      "  >>> Now performing quality (cutoff 20) and adapter trimming in a single pass for the adapter sequence: 'AGATCGGAAGAGC' from file /home/sam/data/geoduck_illumina/trimmed/NR019_S7_L001_R1_001_val_1.fq.gz <<< \n",
      "This is cutadapt 1.9.1 with Python 2.7.12\n",
      "Command line parameters: -f fastq -e 0.1 -q 20 -O 1 -a AGATCGGAAGAGC /home/sam/data/geoduck_illumina/trimmed/NR019_S7_L001_R1_001_val_1.fq.gz\n",
      "Trimming 1 adapter with at most 10.0% errors in single-end mode ...\n",
      "Finished in 1.13 s (27 us/read; 2.19 M reads/minute).\n",
      "\n",
      "=== Summary ===\n",
      "\n",
      "Total reads processed:                  41,244\n",
      "Reads with adapters:                    22,174 (53.8%)\n",
      "Reads written (passing filters):        41,244 (100.0%)\n",
      "\n",
      "Total basepairs processed:     5,571,801 bp\n",
      "Quality-trimmed:                  10,796 bp (0.2%)\n",
      "Total written (filtered):      5,496,658 bp (98.7%)\n",
      "\n",
      "=== Adapter 1 ===\n",
      "\n",
      "Sequence: AGATCGGAAGAGC; Type: regular 3'; Length: 13; Trimmed: 22174 times.\n",
      "\n",
      "No. of allowed errors:\n",
      "0-9 bp: 0; 10-13 bp: 1\n",
      "\n",
      "Bases preceding removed adapters:\n",
      "  A: 41.0%\n",
      "  C: 22.0%\n",
      "  G: 10.3%\n",
      "  T: 26.5%\n",
      "  none/other: 0.2%\n",
      "\n",
      "Overview of removed sequences\n",
      "length\tcount\texpect\tmax.err\terror counts\n",
      "1\t17489\t10311.0\t0\t17489\n",
      "2\t2472\t2577.8\t0\t2472\n",
      "3\t1088\t644.4\t0\t1088\n",
      "4\t354\t161.1\t0\t354\n",
      "5\t17\t40.3\t0\t17\n",
      "6\t10\t10.1\t0\t10\n",
      "7\t9\t2.5\t0\t9\n",
      "8\t9\t0.6\t0\t9\n",
      "9\t9\t0.2\t0\t9\n",
      "10\t16\t0.0\t1\t11 5\n",
      "11\t16\t0.0\t1\t12 4\n",
      "12\t18\t0.0\t1\t16 2\n",
      "13\t2\t0.0\t1\t2\n",
      "14\t23\t0.0\t1\t15 8\n",
      "15\t1\t0.0\t1\t1\n",
      "16\t25\t0.0\t1\t11 14\n",
      "17\t1\t0.0\t1\t1\n",
      "18\t29\t0.0\t1\t20 9\n",
      "19\t29\t0.0\t1\t19 10\n",
      "21\t1\t0.0\t1\t1\n",
      "22\t1\t0.0\t1\t0 1\n",
      "23\t15\t0.0\t1\t11 4\n",
      "24\t32\t0.0\t1\t20 12\n",
      "25\t1\t0.0\t1\t1\n",
      "26\t7\t0.0\t1\t6 1\n",
      "27\t10\t0.0\t1\t9 1\n",
      "29\t9\t0.0\t1\t6 3\n",
      "30\t13\t0.0\t1\t11 2\n",
      "31\t15\t0.0\t1\t10 5\n",
      "32\t2\t0.0\t1\t1 1\n",
      "33\t24\t0.0\t1\t21 3\n",
      "35\t18\t0.0\t1\t13 5\n",
      "36\t11\t0.0\t1\t9 2\n",
      "37\t5\t0.0\t1\t3 2\n",
      "38\t8\t0.0\t1\t5 3\n",
      "39\t11\t0.0\t1\t8 3\n",
      "40\t12\t0.0\t1\t11 1\n",
      "41\t1\t0.0\t1\t0 1\n",
      "42\t12\t0.0\t1\t8 4\n",
      "43\t20\t0.0\t1\t10 10\n",
      "44\t27\t0.0\t1\t20 7\n",
      "45\t6\t0.0\t1\t5 1\n",
      "46\t8\t0.0\t1\t5 3\n",
      "48\t32\t0.0\t1\t18 14\n",
      "49\t14\t0.0\t1\t10 4\n",
      "50\t6\t0.0\t1\t4 2\n",
      "51\t9\t0.0\t1\t8 1\n",
      "52\t10\t0.0\t1\t8 2\n",
      "53\t3\t0.0\t1\t2 1\n",
      "55\t2\t0.0\t1\t2\n",
      "56\t9\t0.0\t1\t7 2\n",
      "58\t3\t0.0\t1\t3\n",
      "59\t6\t0.0\t1\t3 3\n",
      "61\t1\t0.0\t1\t1\n",
      "62\t3\t0.0\t1\t3\n",
      "63\t1\t0.0\t1\t1\n",
      "64\t1\t0.0\t1\t1\n",
      "65\t1\t0.0\t1\t0 1\n",
      "66\t3\t0.0\t1\t2 1\n",
      "67\t4\t0.0\t1\t3 1\n",
      "68\t2\t0.0\t1\t1 1\n",
      "69\t2\t0.0\t1\t1 1\n",
      "70\t3\t0.0\t1\t3\n",
      "71\t2\t0.0\t1\t0 2\n",
      "72\t1\t0.0\t1\t0 1\n",
      "73\t4\t0.0\t1\t4\n",
      "74\t5\t0.0\t1\t3 2\n",
      "75\t1\t0.0\t1\t0 1\n",
      "76\t1\t0.0\t1\t0 1\n",
      "77\t2\t0.0\t1\t2\n",
      "78\t3\t0.0\t1\t2 1\n",
      "79\t4\t0.0\t1\t2 2\n",
      "80\t3\t0.0\t1\t3\n",
      "81\t4\t0.0\t1\t3 1\n",
      "82\t7\t0.0\t1\t7\n",
      "83\t6\t0.0\t1\t5 1\n",
      "84\t3\t0.0\t1\t2 1\n",
      "85\t4\t0.0\t1\t3 1\n",
      "86\t6\t0.0\t1\t5 1\n",
      "87\t2\t0.0\t1\t1 1\n",
      "88\t1\t0.0\t1\t1\n",
      "89\t3\t0.0\t1\t2 1\n",
      "90\t7\t0.0\t1\t5 2\n",
      "91\t4\t0.0\t1\t3 1\n",
      "93\t3\t0.0\t1\t2 1\n",
      "94\t1\t0.0\t1\t1\n",
      "95\t3\t0.0\t1\t2 1\n",
      "96\t3\t0.0\t1\t2 1\n",
      "97\t1\t0.0\t1\t1\n",
      "98\t4\t0.0\t1\t4\n",
      "100\t2\t0.0\t1\t1 1\n",
      "101\t1\t0.0\t1\t0 1\n",
      "102\t1\t0.0\t1\t1\n",
      "104\t3\t0.0\t1\t1 2\n",
      "105\t1\t0.0\t1\t1\n",
      "106\t2\t0.0\t1\t1 1\n",
      "107\t2\t0.0\t1\t0 2\n",
      "109\t2\t0.0\t1\t2\n",
      "111\t3\t0.0\t1\t2 1\n",
      "115\t1\t0.0\t1\t1\n",
      "116\t2\t0.0\t1\t2\n",
      "119\t2\t0.0\t1\t1 1\n",
      "120\t1\t0.0\t1\t1\n",
      "121\t2\t0.0\t1\t0 2\n",
      "126\t3\t0.0\t1\t2 1\n",
      "127\t1\t0.0\t1\t0 1\n",
      "128\t1\t0.0\t1\t0 1\n",
      "130\t1\t0.0\t1\t0 1\n",
      "131\t1\t0.0\t1\t1\n",
      "135\t1\t0.0\t1\t1\n",
      "139\t1\t0.0\t1\t1\n",
      "140\t1\t0.0\t1\t1\n",
      "143\t1\t0.0\t1\t0 1\n",
      "144\t1\t0.0\t1\t1\n",
      "150\t3\t0.0\t1\t3\n",
      "151\t45\t0.0\t1\t16 29\n",
      "\n",
      "\n",
      "RUN STATISTICS FOR INPUT FILE: /home/sam/data/geoduck_illumina/trimmed/NR019_S7_L001_R1_001_val_1.fq.gz\n",
      "=============================================\n",
      "41244 sequences processed in total\n",
      "The length threshold of paired-end sequences gets evaluated later on (in the validation step)\n",
      "\n",
      "Writing report to '/home/sam/data/geoduck_illumina/trimmed/20180129_trimmed_again/NR019_S7_L001_R2_001_val_2.fq.gz_trimming_report.txt'\n",
      "\n",
      "SUMMARISING RUN PARAMETERS\n",
      "==========================\n",
      "Input filename: /home/sam/data/geoduck_illumina/trimmed/NR019_S7_L001_R2_001_val_2.fq.gz\n",
      "Trimming mode: paired-end\n",
      "Trim Galore version: 0.4.4\n",
      "Cutadapt version: 1.9.1\n",
      "Quality Phred score cutoff: 20\n",
      "Quality encoding type selected: ASCII+33\n",
      "Adapter sequence: 'AGATCGGAAGAGC' (Illumina TruSeq, Sanger iPCR; auto-detected)\n",
      "Maximum trimming error rate: 0.1 (default)\n",
      "Minimum required adapter overlap (stringency): 1 bp\n",
      "Minimum required sequence length for both reads before a sequence pair gets removed: 20 bp\n",
      "Output file(s) will be GZIP compressed\n",
      "\n",
      "Writing final adapter and quality trimmed output to NR019_S7_L001_R2_001_val_2_trimmed.fq.gz\n",
      "\n",
      "\n",
      "  >>> Now performing quality (cutoff 20) and adapter trimming in a single pass for the adapter sequence: 'AGATCGGAAGAGC' from file /home/sam/data/geoduck_illumina/trimmed/NR019_S7_L001_R2_001_val_2.fq.gz <<< \n",
      "This is cutadapt 1.9.1 with Python 2.7.12\n",
      "Command line parameters: -f fastq -e 0.1 -q 20 -O 1 -a AGATCGGAAGAGC /home/sam/data/geoduck_illumina/trimmed/NR019_S7_L001_R2_001_val_2.fq.gz\n",
      "Trimming 1 adapter with at most 10.0% errors in single-end mode ...\n",
      "Finished in 1.10 s (27 us/read; 2.25 M reads/minute).\n",
      "\n",
      "=== Summary ===\n",
      "\n",
      "Total reads processed:                  41,244\n",
      "Reads with adapters:                    20,983 (50.9%)\n",
      "Reads written (passing filters):        41,244 (100.0%)\n",
      "\n",
      "Total basepairs processed:     5,533,654 bp\n",
      "Quality-trimmed:                  11,962 bp (0.2%)\n",
      "Total written (filtered):      5,452,710 bp (98.5%)\n",
      "\n",
      "=== Adapter 1 ===\n",
      "\n",
      "Sequence: AGATCGGAAGAGC; Type: regular 3'; Length: 13; Trimmed: 20983 times.\n",
      "\n",
      "No. of allowed errors:\n",
      "0-9 bp: 0; 10-13 bp: 1\n",
      "\n",
      "Bases preceding removed adapters:\n",
      "  A: 38.6%\n",
      "  C: 21.7%\n",
      "  G: 13.3%\n",
      "  T: 26.4%\n",
      "  none/other: 0.1%\n",
      "\n",
      "Overview of removed sequences\n",
      "length\tcount\texpect\tmax.err\terror counts\n",
      "1\t15569\t10311.0\t0\t15569\n",
      "2\t2875\t2577.8\t0\t2875\n",
      "3\t1260\t644.4\t0\t1260\n",
      "4\t347\t161.1\t0\t347\n",
      "5\t17\t40.3\t0\t17\n",
      "6\t20\t10.1\t0\t20\n",
      "7\t14\t2.5\t0\t14\n",
      "8\t9\t0.6\t0\t9\n",
      "9\t9\t0.2\t0\t8 1\n",
      "10\t22\t0.0\t1\t15 7\n",
      "11\t19\t0.0\t1\t13 6\n",
      "12\t26\t0.0\t1\t15 11\n",
      "14\t23\t0.0\t1\t12 11\n",
      "15\t25\t0.0\t1\t16 9\n",
      "17\t14\t0.0\t1\t11 3\n",
      "18\t17\t0.0\t1\t14 3\n",
      "19\t23\t0.0\t1\t16 7\n",
      "20\t14\t0.0\t1\t10 4\n",
      "21\t10\t0.0\t1\t6 4\n",
      "22\t19\t0.0\t1\t13 6\n",
      "23\t16\t0.0\t1\t10 6\n",
      "24\t28\t0.0\t1\t20 8\n",
      "25\t13\t0.0\t1\t9 4\n",
      "26\t10\t0.0\t1\t8 2\n",
      "27\t10\t0.0\t1\t5 5\n",
      "28\t15\t0.0\t1\t13 2\n",
      "29\t6\t0.0\t1\t4 2\n",
      "30\t22\t0.0\t1\t18 4\n",
      "31\t11\t0.0\t1\t8 3\n",
      "32\t18\t0.0\t1\t13 5\n",
      "33\t20\t0.0\t1\t17 3\n",
      "34\t10\t0.0\t1\t6 4\n",
      "35\t14\t0.0\t1\t7 7\n",
      "36\t10\t0.0\t1\t8 2\n",
      "37\t9\t0.0\t1\t8 1\n",
      "38\t6\t0.0\t1\t2 4\n",
      "39\t7\t0.0\t1\t5 2\n",
      "41\t11\t0.0\t1\t10 1\n",
      "42\t15\t0.0\t1\t11 4\n",
      "43\t7\t0.0\t1\t5 2\n",
      "44\t11\t0.0\t1\t7 4\n",
      "45\t18\t0.0\t1\t14 4\n",
      "46\t9\t0.0\t1\t6 3\n",
      "47\t7\t0.0\t1\t4 3\n",
      "48\t23\t0.0\t1\t20 3\n",
      "49\t8\t0.0\t1\t5 3\n",
      "50\t1\t0.0\t1\t1\n",
      "51\t13\t0.0\t1\t11 2\n",
      "52\t8\t0.0\t1\t7 1\n",
      "53\t11\t0.0\t1\t6 5\n",
      "54\t10\t0.0\t1\t8 2\n",
      "55\t2\t0.0\t1\t2\n",
      "56\t1\t0.0\t1\t0 1\n",
      "57\t5\t0.0\t1\t3 2\n",
      "58\t4\t0.0\t1\t4\n",
      "59\t3\t0.0\t1\t2 1\n",
      "60\t4\t0.0\t1\t3 1\n",
      "62\t4\t0.0\t1\t3 1\n",
      "63\t4\t0.0\t1\t4\n",
      "64\t5\t0.0\t1\t4 1\n",
      "65\t2\t0.0\t1\t0 2\n",
      "66\t9\t0.0\t1\t4 5\n",
      "67\t6\t0.0\t1\t4 2\n",
      "68\t6\t0.0\t1\t5 1\n",
      "69\t5\t0.0\t1\t3 2\n",
      "70\t6\t0.0\t1\t5 1\n",
      "71\t5\t0.0\t1\t4 1\n",
      "72\t6\t0.0\t1\t3 3\n",
      "73\t7\t0.0\t1\t5 2\n",
      "74\t8\t0.0\t1\t6 2\n",
      "75\t1\t0.0\t1\t0 1\n",
      "76\t5\t0.0\t1\t4 1\n",
      "77\t4\t0.0\t1\t2 2\n",
      "78\t7\t0.0\t1\t7\n",
      "79\t5\t0.0\t1\t4 1\n",
      "80\t5\t0.0\t1\t3 2\n",
      "81\t7\t0.0\t1\t5 2\n",
      "82\t4\t0.0\t1\t4\n",
      "83\t6\t0.0\t1\t5 1\n",
      "84\t3\t0.0\t1\t2 1\n",
      "85\t3\t0.0\t1\t3\n",
      "86\t6\t0.0\t1\t4 2\n",
      "87\t4\t0.0\t1\t3 1\n",
      "88\t5\t0.0\t1\t3 2\n",
      "89\t9\t0.0\t1\t4 5\n",
      "90\t5\t0.0\t1\t4 1\n",
      "91\t4\t0.0\t1\t3 1\n",
      "92\t2\t0.0\t1\t1 1\n",
      "93\t2\t0.0\t1\t1 1\n",
      "94\t2\t0.0\t1\t0 2\n",
      "95\t3\t0.0\t1\t1 2\n",
      "96\t8\t0.0\t1\t5 3\n",
      "97\t2\t0.0\t1\t2\n",
      "98\t7\t0.0\t1\t6 1\n",
      "99\t1\t0.0\t1\t1\n",
      "100\t2\t0.0\t1\t1 1\n",
      "101\t2\t0.0\t1\t1 1\n",
      "102\t2\t0.0\t1\t2\n",
      "103\t4\t0.0\t1\t3 1\n",
      "104\t1\t0.0\t1\t1\n",
      "105\t1\t0.0\t1\t1\n",
      "106\t3\t0.0\t1\t2 1\n",
      "107\t2\t0.0\t1\t0 2\n",
      "109\t3\t0.0\t1\t3\n",
      "111\t1\t0.0\t1\t1\n",
      "112\t1\t0.0\t1\t1\n",
      "113\t1\t0.0\t1\t0 1\n",
      "115\t3\t0.0\t1\t1 2\n",
      "118\t1\t0.0\t1\t0 1\n",
      "119\t3\t0.0\t1\t1 2\n",
      "120\t1\t0.0\t1\t1\n",
      "121\t1\t0.0\t1\t0 1\n",
      "122\t1\t0.0\t1\t1\n",
      "124\t1\t0.0\t1\t1\n",
      "126\t2\t0.0\t1\t1 1\n",
      "127\t3\t0.0\t1\t2 1\n",
      "128\t1\t0.0\t1\t1\n",
      "130\t1\t0.0\t1\t0 1\n",
      "131\t3\t0.0\t1\t2 1\n",
      "135\t1\t0.0\t1\t1\n",
      "137\t1\t0.0\t1\t0 1\n",
      "138\t1\t0.0\t1\t0 1\n",
      "139\t1\t0.0\t1\t1\n",
      "140\t1\t0.0\t1\t1\n",
      "150\t2\t0.0\t1\t1 1\n",
      "151\t21\t0.0\t1\t11 10\n",
      "\n",
      "\n",
      "RUN STATISTICS FOR INPUT FILE: /home/sam/data/geoduck_illumina/trimmed/NR019_S7_L001_R2_001_val_2.fq.gz\n",
      "=============================================\n",
      "41244 sequences processed in total\n",
      "The length threshold of paired-end sequences gets evaluated later on (in the validation step)\n",
      "\n",
      "Validate paired-end files NR019_S7_L001_R1_001_val_1_trimmed.fq.gz and NR019_S7_L001_R2_001_val_2_trimmed.fq.gz\n",
      "file_1: NR019_S7_L001_R1_001_val_1_trimmed.fq.gz, file_2: NR019_S7_L001_R2_001_val_2_trimmed.fq.gz\n",
      "\n",
      "\n",
      ">>>>> Now validing the length of the 2 paired-end infiles: NR019_S7_L001_R1_001_val_1_trimmed.fq.gz and NR019_S7_L001_R2_001_val_2_trimmed.fq.gz <<<<<\n",
      "Writing validated paired-end read 1 reads to NR019_S7_L001_R1_001_val_1_val_1.fq.gz\n",
      "Writing validated paired-end read 2 reads to NR019_S7_L001_R2_001_val_2_val_2.fq.gz\n",
      "\n",
      "Total number of sequences analysed: 41244\n",
      "\n",
      "Number of sequence pairs removed because at least one read was shorter than the length cutoff (20 bp): 208 (0.50%)\n",
      "\n",
      "Deleting both intermediate output files NR019_S7_L001_R1_001_val_1_trimmed.fq.gz and NR019_S7_L001_R2_001_val_2_trimmed.fq.gz\n",
      "\n",
      "====================================================================================================\n",
      "\n",
      "Writing report to '/home/sam/data/geoduck_illumina/trimmed/20180129_trimmed_again/NR019_S7_L002_R1_001_val_1.fq.gz_trimming_report.txt'\n",
      "\n",
      "SUMMARISING RUN PARAMETERS\n",
      "==========================\n",
      "Input filename: /home/sam/data/geoduck_illumina/trimmed/NR019_S7_L002_R1_001_val_1.fq.gz\n",
      "Trimming mode: paired-end\n",
      "Trim Galore version: 0.4.4\n",
      "Cutadapt version: 1.9.1\n",
      "Quality Phred score cutoff: 20\n",
      "Quality encoding type selected: ASCII+33\n",
      "Adapter sequence: 'AGATCGGAAGAGC' (Illumina TruSeq, Sanger iPCR; auto-detected)\n",
      "Maximum trimming error rate: 0.1 (default)\n",
      "Minimum required adapter overlap (stringency): 1 bp\n",
      "Minimum required sequence length for both reads before a sequence pair gets removed: 20 bp\n",
      "Output file(s) will be GZIP compressed\n",
      "\n",
      "Writing final adapter and quality trimmed output to NR019_S7_L002_R1_001_val_1_trimmed.fq.gz\n",
      "\n",
      "\n",
      "  >>> Now performing quality (cutoff 20) and adapter trimming in a single pass for the adapter sequence: 'AGATCGGAAGAGC' from file /home/sam/data/geoduck_illumina/trimmed/NR019_S7_L002_R1_001_val_1.fq.gz <<< \n",
      "This is cutadapt 1.9.1 with Python 2.7.12\n",
      "Command line parameters: -f fastq -e 0.1 -q 20 -O 1 -a AGATCGGAAGAGC /home/sam/data/geoduck_illumina/trimmed/NR019_S7_L002_R1_001_val_1.fq.gz\n",
      "Trimming 1 adapter with at most 10.0% errors in single-end mode ...\n",
      "Finished in 0.61 s (27 us/read; 2.20 M reads/minute).\n",
      "\n",
      "=== Summary ===\n",
      "\n",
      "Total reads processed:                  22,310\n",
      "Reads with adapters:                    11,594 (52.0%)\n",
      "Reads written (passing filters):        22,310 (100.0%)\n",
      "\n",
      "Total basepairs processed:     3,005,909 bp\n",
      "Quality-trimmed:                   4,316 bp (0.1%)\n",
      "Total written (filtered):      2,944,393 bp (98.0%)\n",
      "\n",
      "=== Adapter 1 ===\n",
      "\n",
      "Sequence: AGATCGGAAGAGC; Type: regular 3'; Length: 13; Trimmed: 11594 times.\n",
      "\n",
      "No. of allowed errors:\n",
      "0-9 bp: 0; 10-13 bp: 1\n",
      "\n",
      "Bases preceding removed adapters:\n",
      "  A: 39.7%\n",
      "  C: 22.0%\n",
      "  G: 11.3%\n",
      "  T: 26.4%\n",
      "  none/other: 0.6%\n",
      "\n",
      "Overview of removed sequences\n",
      "length\tcount\texpect\tmax.err\terror counts\n",
      "1\t8570\t5577.5\t0\t8570\n",
      "2\t1444\t1394.4\t0\t1444\n",
      "3\t596\t348.6\t0\t596\n",
      "4\t178\t87.1\t0\t178\n",
      "5\t11\t21.8\t0\t11\n",
      "6\t13\t5.4\t0\t13\n",
      "7\t8\t1.4\t0\t8\n",
      "8\t9\t0.3\t0\t9\n",
      "9\t8\t0.1\t0\t7 1\n",
      "10\t23\t0.0\t1\t18 5\n",
      "11\t15\t0.0\t1\t7 8\n",
      "12\t24\t0.0\t1\t18 6\n",
      "13\t2\t0.0\t1\t1 1\n",
      "14\t19\t0.0\t1\t16 3\n",
      "16\t23\t0.0\t1\t16 7\n",
      "17\t1\t0.0\t1\t1\n",
      "18\t22\t0.0\t1\t19 3\n",
      "19\t34\t0.0\t1\t30 4\n",
      "22\t1\t0.0\t1\t1\n",
      "23\t14\t0.0\t1\t10 4\n",
      "24\t24\t0.0\t1\t18 6\n",
      "25\t2\t0.0\t1\t1 1\n",
      "26\t7\t0.0\t1\t5 2\n",
      "27\t10\t0.0\t1\t8 2\n",
      "29\t16\t0.0\t1\t14 2\n",
      "30\t12\t0.0\t1\t10 2\n",
      "31\t12\t0.0\t1\t7 5\n",
      "32\t3\t0.0\t1\t0 3\n",
      "33\t25\t0.0\t1\t24 1\n",
      "34\t1\t0.0\t1\t1\n",
      "35\t20\t0.0\t1\t12 8\n",
      "36\t12\t0.0\t1\t9 3\n",
      "37\t14\t0.0\t1\t9 5\n",
      "38\t6\t0.0\t1\t4 2\n",
      "39\t3\t0.0\t1\t2 1\n",
      "40\t15\t0.0\t1\t8 7\n",
      "41\t2\t0.0\t1\t1 1\n",
      "42\t12\t0.0\t1\t8 4\n",
      "43\t13\t0.0\t1\t11 2\n",
      "44\t16\t0.0\t1\t13 3\n",
      "45\t3\t0.0\t1\t3\n",
      "46\t6\t0.0\t1\t4 2\n",
      "47\t1\t0.0\t1\t0 1\n",
      "48\t19\t0.0\t1\t16 3\n",
      "49\t8\t0.0\t1\t8\n",
      "50\t2\t0.0\t1\t2\n",
      "51\t8\t0.0\t1\t6 2\n",
      "52\t11\t0.0\t1\t9 2\n",
      "53\t2\t0.0\t1\t1 1\n",
      "54\t3\t0.0\t1\t3\n",
      "55\t13\t0.0\t1\t11 2\n",
      "56\t7\t0.0\t1\t5 2\n",
      "57\t3\t0.0\t1\t2 1\n",
      "58\t2\t0.0\t1\t1 1\n",
      "59\t11\t0.0\t1\t9 2\n",
      "61\t7\t0.0\t1\t3 4\n",
      "62\t6\t0.0\t1\t5 1\n",
      "63\t3\t0.0\t1\t2 1\n",
      "64\t1\t0.0\t1\t1\n",
      "65\t4\t0.0\t1\t1 3\n",
      "66\t6\t0.0\t1\t3 3\n",
      "67\t2\t0.0\t1\t2\n",
      "68\t3\t0.0\t1\t2 1\n",
      "69\t4\t0.0\t1\t3 1\n",
      "70\t6\t0.0\t1\t4 2\n",
      "71\t1\t0.0\t1\t0 1\n",
      "72\t1\t0.0\t1\t1\n",
      "73\t5\t0.0\t1\t4 1\n",
      "74\t2\t0.0\t1\t2\n",
      "75\t3\t0.0\t1\t3\n",
      "76\t2\t0.0\t1\t1 1\n",
      "77\t4\t0.0\t1\t3 1\n",
      "78\t4\t0.0\t1\t4\n",
      "79\t4\t0.0\t1\t4\n",
      "80\t4\t0.0\t1\t3 1\n",
      "81\t8\t0.0\t1\t8\n",
      "82\t1\t0.0\t1\t1\n",
      "83\t4\t0.0\t1\t4\n",
      "84\t1\t0.0\t1\t1\n",
      "85\t4\t0.0\t1\t4\n",
      "86\t5\t0.0\t1\t4 1\n",
      "87\t1\t0.0\t1\t0 1\n",
      "89\t4\t0.0\t1\t4\n",
      "90\t4\t0.0\t1\t4\n",
      "91\t4\t0.0\t1\t4\n",
      "93\t2\t0.0\t1\t1 1\n",
      "94\t2\t0.0\t1\t2\n",
      "95\t7\t0.0\t1\t5 2\n",
      "96\t2\t0.0\t1\t1 1\n",
      "97\t4\t0.0\t1\t4\n",
      "98\t2\t0.0\t1\t2\n",
      "99\t2\t0.0\t1\t1 1\n",
      "100\t1\t0.0\t1\t0 1\n",
      "101\t5\t0.0\t1\t4 1\n",
      "102\t3\t0.0\t1\t2 1\n",
      "103\t2\t0.0\t1\t2\n",
      "107\t1\t0.0\t1\t1\n",
      "108\t1\t0.0\t1\t1\n",
      "109\t1\t0.0\t1\t1\n",
      "110\t4\t0.0\t1\t3 1\n",
      "111\t2\t0.0\t1\t1 1\n",
      "112\t1\t0.0\t1\t1\n",
      "116\t2\t0.0\t1\t2\n",
      "117\t1\t0.0\t1\t1\n",
      "118\t2\t0.0\t1\t2\n",
      "119\t3\t0.0\t1\t3\n",
      "120\t2\t0.0\t1\t1 1\n",
      "121\t2\t0.0\t1\t2\n",
      "123\t1\t0.0\t1\t1\n",
      "124\t1\t0.0\t1\t1\n",
      "126\t1\t0.0\t1\t1\n",
      "127\t2\t0.0\t1\t1 1\n",
      "129\t2\t0.0\t1\t1 1\n",
      "131\t4\t0.0\t1\t3 1\n",
      "132\t1\t0.0\t1\t1\n",
      "133\t3\t0.0\t1\t2 1\n",
      "135\t3\t0.0\t1\t3\n",
      "136\t1\t0.0\t1\t0 1\n",
      "137\t3\t0.0\t1\t3\n",
      "138\t1\t0.0\t1\t1\n",
      "139\t1\t0.0\t1\t1\n",
      "143\t1\t0.0\t1\t1\n",
      "146\t1\t0.0\t1\t1\n",
      "149\t1\t0.0\t1\t0 1\n",
      "150\t2\t0.0\t1\t2\n",
      "151\t64\t0.0\t1\t36 28\n",
      "\n",
      "\n",
      "RUN STATISTICS FOR INPUT FILE: /home/sam/data/geoduck_illumina/trimmed/NR019_S7_L002_R1_001_val_1.fq.gz\n",
      "=============================================\n",
      "22310 sequences processed in total\n",
      "The length threshold of paired-end sequences gets evaluated later on (in the validation step)\n",
      "\n",
      "Writing report to '/home/sam/data/geoduck_illumina/trimmed/20180129_trimmed_again/NR019_S7_L002_R2_001_val_2.fq.gz_trimming_report.txt'\n",
      "\n",
      "SUMMARISING RUN PARAMETERS\n",
      "==========================\n",
      "Input filename: /home/sam/data/geoduck_illumina/trimmed/NR019_S7_L002_R2_001_val_2.fq.gz\n",
      "Trimming mode: paired-end\n",
      "Trim Galore version: 0.4.4\n",
      "Cutadapt version: 1.9.1\n",
      "Quality Phred score cutoff: 20\n",
      "Quality encoding type selected: ASCII+33\n",
      "Adapter sequence: 'AGATCGGAAGAGC' (Illumina TruSeq, Sanger iPCR; auto-detected)\n",
      "Maximum trimming error rate: 0.1 (default)\n",
      "Minimum required adapter overlap (stringency): 1 bp\n",
      "Minimum required sequence length for both reads before a sequence pair gets removed: 20 bp\n",
      "Output file(s) will be GZIP compressed\n",
      "\n",
      "Writing final adapter and quality trimmed output to NR019_S7_L002_R2_001_val_2_trimmed.fq.gz\n",
      "\n",
      "\n",
      "  >>> Now performing quality (cutoff 20) and adapter trimming in a single pass for the adapter sequence: 'AGATCGGAAGAGC' from file /home/sam/data/geoduck_illumina/trimmed/NR019_S7_L002_R2_001_val_2.fq.gz <<< \n",
      "This is cutadapt 1.9.1 with Python 2.7.12\n",
      "Command line parameters: -f fastq -e 0.1 -q 20 -O 1 -a AGATCGGAAGAGC /home/sam/data/geoduck_illumina/trimmed/NR019_S7_L002_R2_001_val_2.fq.gz\n",
      "Trimming 1 adapter with at most 10.0% errors in single-end mode ...\n",
      "Finished in 0.61 s (27 us/read; 2.18 M reads/minute).\n",
      "\n",
      "=== Summary ===\n",
      "\n",
      "Total reads processed:                  22,310\n",
      "Reads with adapters:                    11,102 (49.8%)\n",
      "Reads written (passing filters):        22,310 (100.0%)\n",
      "\n",
      "Total basepairs processed:     2,948,137 bp\n",
      "Quality-trimmed:                   5,764 bp (0.2%)\n",
      "Total written (filtered):      2,883,202 bp (97.8%)\n",
      "\n",
      "=== Adapter 1 ===\n",
      "\n",
      "Sequence: AGATCGGAAGAGC; Type: regular 3'; Length: 13; Trimmed: 11102 times.\n",
      "\n",
      "No. of allowed errors:\n",
      "0-9 bp: 0; 10-13 bp: 1\n",
      "\n",
      "Bases preceding removed adapters:\n",
      "  A: 37.2%\n",
      "  C: 22.4%\n",
      "  G: 13.8%\n",
      "  T: 26.4%\n",
      "  none/other: 0.3%\n",
      "\n",
      "Overview of removed sequences\n",
      "length\tcount\texpect\tmax.err\terror counts\n",
      "1\t7858\t5577.5\t0\t7858\n",
      "2\t1557\t1394.4\t0\t1557\n",
      "3\t609\t348.6\t0\t609\n",
      "4\t171\t87.1\t0\t171\n",
      "5\t12\t21.8\t0\t12\n",
      "6\t17\t5.4\t0\t17\n",
      "7\t12\t1.4\t0\t12\n",
      "8\t12\t0.3\t0\t12\n",
      "9\t5\t0.1\t0\t5\n",
      "10\t18\t0.0\t1\t15 3\n",
      "11\t10\t0.0\t1\t9 1\n",
      "12\t22\t0.0\t1\t15 7\n",
      "14\t17\t0.0\t1\t13 4\n",
      "15\t21\t0.0\t1\t19 2\n",
      "17\t26\t0.0\t1\t19 7\n",
      "18\t15\t0.0\t1\t13 2\n",
      "19\t21\t0.0\t1\t16 5\n",
      "20\t15\t0.0\t1\t13 2\n",
      "21\t11\t0.0\t1\t9 2\n",
      "22\t10\t0.0\t1\t5 5\n",
      "23\t18\t0.0\t1\t14 4\n",
      "24\t15\t0.0\t1\t8 7\n",
      "25\t13\t0.0\t1\t11 2\n",
      "26\t12\t0.0\t1\t9 3\n",
      "27\t10\t0.0\t1\t6 4\n",
      "28\t21\t0.0\t1\t18 3\n",
      "29\t12\t0.0\t1\t8 4\n",
      "30\t17\t0.0\t1\t15 2\n",
      "31\t8\t0.0\t1\t6 2\n",
      "32\t20\t0.0\t1\t18 2\n",
      "33\t13\t0.0\t1\t9 4\n",
      "34\t3\t0.0\t1\t3\n",
      "35\t21\t0.0\t1\t17 4\n",
      "36\t13\t0.0\t1\t10 3\n",
      "37\t13\t0.0\t1\t9 4\n",
      "38\t2\t0.0\t1\t2\n",
      "39\t8\t0.0\t1\t3 5\n",
      "40\t1\t0.0\t1\t0 1\n",
      "41\t6\t0.0\t1\t5 1\n",
      "42\t8\t0.0\t1\t6 2\n",
      "43\t4\t0.0\t1\t4\n",
      "44\t9\t0.0\t1\t7 2\n",
      "45\t21\t0.0\t1\t18 3\n",
      "46\t9\t0.0\t1\t6 3\n",
      "47\t5\t0.0\t1\t3 2\n",
      "48\t18\t0.0\t1\t15 3\n",
      "49\t3\t0.0\t1\t3\n",
      "50\t2\t0.0\t1\t1 1\n",
      "51\t10\t0.0\t1\t10\n",
      "52\t25\t0.0\t1\t20 5\n",
      "53\t10\t0.0\t1\t8 2\n",
      "54\t7\t0.0\t1\t5 2\n",
      "55\t4\t0.0\t1\t3 1\n",
      "56\t4\t0.0\t1\t4\n",
      "57\t5\t0.0\t1\t2 3\n",
      "58\t9\t0.0\t1\t6 3\n",
      "59\t1\t0.0\t1\t1\n",
      "60\t4\t0.0\t1\t3 1\n",
      "61\t3\t0.0\t1\t3\n",
      "62\t1\t0.0\t1\t1\n",
      "63\t3\t0.0\t1\t3\n",
      "64\t3\t0.0\t1\t2 1\n",
      "65\t3\t0.0\t1\t2 1\n",
      "66\t8\t0.0\t1\t6 2\n",
      "67\t3\t0.0\t1\t3\n",
      "68\t7\t0.0\t1\t7\n",
      "69\t11\t0.0\t1\t7 4\n",
      "70\t3\t0.0\t1\t3\n",
      "71\t4\t0.0\t1\t3 1\n",
      "72\t5\t0.0\t1\t3 2\n",
      "73\t4\t0.0\t1\t4\n",
      "74\t7\t0.0\t1\t6 1\n",
      "75\t2\t0.0\t1\t2\n",
      "76\t3\t0.0\t1\t3\n",
      "77\t4\t0.0\t1\t2 2\n",
      "78\t6\t0.0\t1\t6\n",
      "79\t5\t0.0\t1\t4 1\n",
      "80\t6\t0.0\t1\t4 2\n",
      "81\t6\t0.0\t1\t4 2\n",
      "82\t3\t0.0\t1\t3\n",
      "83\t6\t0.0\t1\t4 2\n",
      "84\t2\t0.0\t1\t1 1\n",
      "85\t6\t0.0\t1\t6\n",
      "86\t5\t0.0\t1\t2 3\n",
      "87\t2\t0.0\t1\t1 1\n",
      "88\t1\t0.0\t1\t0 1\n",
      "89\t10\t0.0\t1\t4 6\n",
      "90\t5\t0.0\t1\t4 1\n",
      "91\t6\t0.0\t1\t3 3\n",
      "92\t2\t0.0\t1\t1 1\n",
      "93\t1\t0.0\t1\t1\n",
      "94\t3\t0.0\t1\t1 2\n",
      "95\t9\t0.0\t1\t7 2\n",
      "96\t3\t0.0\t1\t3\n",
      "97\t3\t0.0\t1\t2 1\n",
      "98\t3\t0.0\t1\t3\n",
      "99\t5\t0.0\t1\t5\n",
      "100\t4\t0.0\t1\t1 3\n",
      "101\t4\t0.0\t1\t3 1\n",
      "102\t2\t0.0\t1\t2\n",
      "103\t2\t0.0\t1\t2\n",
      "104\t1\t0.0\t1\t0 1\n",
      "107\t4\t0.0\t1\t3 1\n",
      "108\t3\t0.0\t1\t3\n",
      "109\t1\t0.0\t1\t1\n",
      "110\t5\t0.0\t1\t4 1\n",
      "111\t1\t0.0\t1\t1\n",
      "116\t2\t0.0\t1\t1 1\n",
      "117\t2\t0.0\t1\t2\n",
      "118\t3\t0.0\t1\t2 1\n",
      "119\t3\t0.0\t1\t3\n",
      "120\t2\t0.0\t1\t0 2\n",
      "121\t5\t0.0\t1\t5\n",
      "123\t2\t0.0\t1\t2\n",
      "124\t1\t0.0\t1\t1\n",
      "126\t3\t0.0\t1\t3\n",
      "127\t2\t0.0\t1\t1 1\n",
      "129\t1\t0.0\t1\t1\n",
      "131\t4\t0.0\t1\t3 1\n",
      "132\t2\t0.0\t1\t2\n",
      "133\t2\t0.0\t1\t2\n",
      "134\t1\t0.0\t1\t0 1\n",
      "135\t5\t0.0\t1\t4 1\n",
      "137\t4\t0.0\t1\t4\n",
      "139\t1\t0.0\t1\t1\n",
      "143\t1\t0.0\t1\t0 1\n",
      "144\t1\t0.0\t1\t0 1\n",
      "146\t1\t0.0\t1\t1\n",
      "148\t1\t0.0\t1\t0 1\n",
      "151\t31\t0.0\t1\t22 9\n",
      "\n",
      "\n",
      "RUN STATISTICS FOR INPUT FILE: /home/sam/data/geoduck_illumina/trimmed/NR019_S7_L002_R2_001_val_2.fq.gz\n",
      "=============================================\n",
      "22310 sequences processed in total\n",
      "The length threshold of paired-end sequences gets evaluated later on (in the validation step)\n",
      "\n",
      "Validate paired-end files NR019_S7_L002_R1_001_val_1_trimmed.fq.gz and NR019_S7_L002_R2_001_val_2_trimmed.fq.gz\n",
      "file_1: NR019_S7_L002_R1_001_val_1_trimmed.fq.gz, file_2: NR019_S7_L002_R2_001_val_2_trimmed.fq.gz\n",
      "\n",
      "\n",
      ">>>>> Now validing the length of the 2 paired-end infiles: NR019_S7_L002_R1_001_val_1_trimmed.fq.gz and NR019_S7_L002_R2_001_val_2_trimmed.fq.gz <<<<<\n",
      "Writing validated paired-end read 1 reads to NR019_S7_L002_R1_001_val_1_val_1.fq.gz\n",
      "Writing validated paired-end read 2 reads to NR019_S7_L002_R2_001_val_2_val_2.fq.gz\n",
      "\n",
      "Total number of sequences analysed: 22310\n",
      "\n",
      "Number of sequence pairs removed because at least one read was shorter than the length cutoff (20 bp): 159 (0.71%)\n",
      "\n",
      "Deleting both intermediate output files NR019_S7_L002_R1_001_val_1_trimmed.fq.gz and NR019_S7_L002_R2_001_val_2_trimmed.fq.gz\n",
      "\n",
      "====================================================================================================\n",
      "\n",
      "Writing report to '/home/sam/data/geoduck_illumina/trimmed/20180129_trimmed_again/NR021_S8_L001_R1_001_val_1.fq.gz_trimming_report.txt'\n",
      "\n",
      "SUMMARISING RUN PARAMETERS\n",
      "==========================\n",
      "Input filename: /home/sam/data/geoduck_illumina/trimmed/NR021_S8_L001_R1_001_val_1.fq.gz\n",
      "Trimming mode: paired-end\n",
      "Trim Galore version: 0.4.4\n",
      "Cutadapt version: 1.9.1\n",
      "Quality Phred score cutoff: 20\n",
      "Quality encoding type selected: ASCII+33\n",
      "Adapter sequence: 'AGATCGGAAGAGC' (Illumina TruSeq, Sanger iPCR; auto-detected)\n",
      "Maximum trimming error rate: 0.1 (default)\n",
      "Minimum required adapter overlap (stringency): 1 bp\n",
      "Minimum required sequence length for both reads before a sequence pair gets removed: 20 bp\n",
      "Output file(s) will be GZIP compressed\n",
      "\n",
      "Writing final adapter and quality trimmed output to NR021_S8_L001_R1_001_val_1_trimmed.fq.gz\n",
      "\n",
      "\n",
      "  >>> Now performing quality (cutoff 20) and adapter trimming in a single pass for the adapter sequence: 'AGATCGGAAGAGC' from file /home/sam/data/geoduck_illumina/trimmed/NR021_S8_L001_R1_001_val_1.fq.gz <<< \n",
      "This is cutadapt 1.9.1 with Python 2.7.12\n",
      "Command line parameters: -f fastq -e 0.1 -q 20 -O 1 -a AGATCGGAAGAGC /home/sam/data/geoduck_illumina/trimmed/NR021_S8_L001_R1_001_val_1.fq.gz\n",
      "Trimming 1 adapter with at most 10.0% errors in single-end mode ...\n",
      "Finished in 105.81 s (26 us/read; 2.27 M reads/minute).\n",
      "\n",
      "=== Summary ===\n",
      "\n",
      "Total reads processed:               3,996,315\n",
      "Reads with adapters:                 2,414,527 (60.4%)\n",
      "Reads written (passing filters):     3,996,315 (100.0%)\n",
      "\n",
      "Total basepairs processed:   593,062,263 bp\n",
      "Quality-trimmed:                 140,474 bp (0.0%)\n",
      "Total written (filtered):    563,799,510 bp (95.1%)\n",
      "\n",
      "=== Adapter 1 ===\n",
      "\n",
      "Sequence: AGATCGGAAGAGC; Type: regular 3'; Length: 13; Trimmed: 2414527 times.\n",
      "\n",
      "No. of allowed errors:\n",
      "0-9 bp: 0; 10-13 bp: 1\n",
      "\n",
      "Bases preceding removed adapters:\n",
      "  A: 30.7%\n",
      "  C: 27.6%\n",
      "  G: 20.3%\n",
      "  T: 20.3%\n",
      "  none/other: 1.0%\n",
      "\n",
      "Overview of removed sequences\n",
      "length\tcount\texpect\tmax.err\terror counts\n",
      "1\t1131131\t999078.8\t0\t1131131\n",
      "2\t218514\t249769.7\t0\t218514\n",
      "3\t92634\t62442.4\t0\t92634\n",
      "4\t69282\t15610.6\t0\t69282\n",
      "5\t947\t3902.7\t0\t947\n",
      "6\t26669\t975.7\t0\t26669\n",
      "7\t23335\t243.9\t0\t23335\n",
      "8\t27529\t61.0\t0\t27529\n",
      "9\t24387\t15.2\t0\t24268 119\n",
      "10\t25314\t3.8\t1\t23754 1560\n",
      "11\t26283\t1.0\t1\t23938 2345\n",
      "12\t49032\t0.2\t1\t45917 3115\n",
      "13\t1383\t0.1\t1\t889 494\n",
      "14\t48217\t0.1\t1\t44794 3423\n",
      "15\t1073\t0.1\t1\t659 414\n",
      "16\t50306\t0.1\t1\t46248 4058\n",
      "17\t1566\t0.1\t1\t1024 542\n",
      "18\t30527\t0.1\t1\t28070 2457\n",
      "19\t76707\t0.1\t1\t72330 4377\n",
      "20\t483\t0.1\t1\t293 190\n",
      "21\t170\t0.1\t1\t116 54\n",
      "22\t411\t0.1\t1\t266 145\n",
      "23\t23809\t0.1\t1\t22004 1805\n",
      "24\t56486\t0.1\t1\t52715 3771\n",
      "25\t1264\t0.1\t1\t847 417\n",
      "26\t20025\t0.1\t1\t18597 1428\n",
      "27\t17927\t0.1\t1\t16880 1047\n",
      "28\t351\t0.1\t1\t223 128\n",
      "29\t17811\t0.1\t1\t16451 1360\n",
      "30\t21046\t0.1\t1\t19611 1435\n",
      "31\t27126\t0.1\t1\t25585 1541\n",
      "32\t916\t0.1\t1\t652 264\n",
      "33\t29781\t0.1\t1\t27735 2046\n",
      "34\t397\t0.1\t1\t263 134\n",
      "35\t13349\t0.1\t1\t12548 801\n",
      "36\t13531\t0.1\t1\t12645 886\n",
      "37\t14307\t0.1\t1\t13290 1017\n",
      "38\t22347\t0.1\t1\t20934 1413\n",
      "39\t332\t0.1\t1\t246 86\n",
      "40\t11397\t0.1\t1\t10684 713\n",
      "41\t9994\t0.1\t1\t9415 579\n",
      "42\t12897\t0.1\t1\t12021 876\n",
      "43\t9858\t0.1\t1\t8895 963\n",
      "44\t23703\t0.1\t1\t22141 1562\n",
      "45\t4875\t0.1\t1\t4557 318\n",
      "46\t4502\t0.1\t1\t4081 421\n",
      "47\t589\t0.1\t1\t397 192\n",
      "48\t31741\t0.1\t1\t30092 1649\n",
      "49\t16981\t0.1\t1\t16147 834\n",
      "50\t2554\t0.1\t1\t2298 256\n",
      "51\t7633\t0.1\t1\t7163 470\n",
      "52\t7621\t0.1\t1\t7210 411\n",
      "53\t1435\t0.1\t1\t1306 129\n",
      "54\t296\t0.1\t1\t239 57\n",
      "55\t4039\t0.1\t1\t3828 211\n",
      "56\t2670\t0.1\t1\t2494 176\n",
      "57\t335\t0.1\t1\t292 43\n",
      "58\t887\t0.1\t1\t818 69\n",
      "59\t1073\t0.1\t1\t980 93\n",
      "60\t420\t0.1\t1\t369 51\n",
      "61\t755\t0.1\t1\t685 70\n",
      "62\t2440\t0.1\t1\t2318 122\n",
      "63\t473\t0.1\t1\t428 45\n",
      "64\t769\t0.1\t1\t725 44\n",
      "65\t690\t0.1\t1\t625 65\n",
      "66\t2129\t0.1\t1\t2034 95\n",
      "67\t407\t0.1\t1\t365 42\n",
      "68\t567\t0.1\t1\t529 38\n",
      "69\t620\t0.1\t1\t561 59\n",
      "70\t613\t0.1\t1\t564 49\n",
      "71\t236\t0.1\t1\t214 22\n",
      "72\t340\t0.1\t1\t305 35\n",
      "73\t568\t0.1\t1\t529 39\n",
      "74\t809\t0.1\t1\t763 46\n",
      "75\t909\t0.1\t1\t847 62\n",
      "76\t869\t0.1\t1\t818 51\n",
      "77\t850\t0.1\t1\t806 44\n",
      "78\t779\t0.1\t1\t728 51\n",
      "79\t785\t0.1\t1\t753 32\n",
      "80\t767\t0.1\t1\t722 45\n",
      "81\t665\t0.1\t1\t622 43\n",
      "82\t614\t0.1\t1\t569 45\n",
      "83\t595\t0.1\t1\t557 38\n",
      "84\t523\t0.1\t1\t491 32\n",
      "85\t468\t0.1\t1\t441 27\n",
      "86\t442\t0.1\t1\t408 34\n",
      "87\t418\t0.1\t1\t376 42\n",
      "88\t418\t0.1\t1\t393 25\n",
      "89\t362\t0.1\t1\t339 23\n",
      "90\t368\t0.1\t1\t341 27\n",
      "91\t365\t0.1\t1\t321 44\n",
      "92\t279\t0.1\t1\t261 18\n",
      "93\t253\t0.1\t1\t238 15\n",
      "94\t279\t0.1\t1\t262 17\n",
      "95\t199\t0.1\t1\t180 19\n",
      "96\t252\t0.1\t1\t226 26\n",
      "97\t220\t0.1\t1\t204 16\n",
      "98\t201\t0.1\t1\t191 10\n",
      "99\t180\t0.1\t1\t163 17\n",
      "100\t173\t0.1\t1\t158 15\n",
      "101\t153\t0.1\t1\t143 10\n",
      "102\t113\t0.1\t1\t105 8\n",
      "103\t147\t0.1\t1\t131 16\n",
      "104\t129\t0.1\t1\t114 15\n",
      "105\t101\t0.1\t1\t95 6\n",
      "106\t102\t0.1\t1\t92 10\n",
      "107\t98\t0.1\t1\t85 13\n",
      "108\t70\t0.1\t1\t62 8\n",
      "109\t95\t0.1\t1\t86 9\n",
      "110\t76\t0.1\t1\t63 13\n",
      "111\t84\t0.1\t1\t77 7\n",
      "112\t76\t0.1\t1\t69 7\n",
      "113\t89\t0.1\t1\t82 7\n",
      "114\t51\t0.1\t1\t43 8\n",
      "115\t79\t0.1\t1\t67 12\n",
      "116\t56\t0.1\t1\t52 4\n",
      "117\t50\t0.1\t1\t45 5\n",
      "118\t70\t0.1\t1\t64 6\n",
      "119\t55\t0.1\t1\t49 6\n",
      "120\t74\t0.1\t1\t66 8\n",
      "121\t52\t0.1\t1\t41 11\n",
      "122\t71\t0.1\t1\t69 2\n",
      "123\t47\t0.1\t1\t42 5\n",
      "124\t67\t0.1\t1\t63 4\n",
      "125\t73\t0.1\t1\t71 2\n",
      "126\t91\t0.1\t1\t86 5\n",
      "127\t66\t0.1\t1\t62 4\n",
      "128\t106\t0.1\t1\t100 6\n",
      "129\t102\t0.1\t1\t100 2\n",
      "130\t70\t0.1\t1\t64 6\n",
      "131\t90\t0.1\t1\t84 6\n",
      "132\t90\t0.1\t1\t84 6\n",
      "133\t85\t0.1\t1\t83 2\n",
      "134\t91\t0.1\t1\t86 5\n",
      "135\t99\t0.1\t1\t91 8\n",
      "136\t79\t0.1\t1\t74 5\n",
      "137\t108\t0.1\t1\t102 6\n",
      "138\t76\t0.1\t1\t74 2\n",
      "139\t97\t0.1\t1\t93 4\n",
      "140\t46\t0.1\t1\t43 3\n",
      "141\t50\t0.1\t1\t49 1\n",
      "142\t69\t0.1\t1\t66 3\n",
      "143\t52\t0.1\t1\t49 3\n",
      "144\t30\t0.1\t1\t28 2\n",
      "145\t32\t0.1\t1\t27 5\n",
      "146\t64\t0.1\t1\t56 8\n",
      "147\t41\t0.1\t1\t34 7\n",
      "148\t113\t0.1\t1\t102 11\n",
      "149\t47\t0.1\t1\t32 15\n",
      "150\t176\t0.1\t1\t16 160\n",
      "151\t24625\t0.1\t1\t199 24426\n",
      "\n",
      "\n",
      "RUN STATISTICS FOR INPUT FILE: /home/sam/data/geoduck_illumina/trimmed/NR021_S8_L001_R1_001_val_1.fq.gz\n",
      "=============================================\n",
      "3996315 sequences processed in total\n",
      "The length threshold of paired-end sequences gets evaluated later on (in the validation step)\n",
      "\n",
      "Writing report to '/home/sam/data/geoduck_illumina/trimmed/20180129_trimmed_again/NR021_S8_L001_R2_001_val_2.fq.gz_trimming_report.txt'\n",
      "\n",
      "SUMMARISING RUN PARAMETERS\n",
      "==========================\n",
      "Input filename: /home/sam/data/geoduck_illumina/trimmed/NR021_S8_L001_R2_001_val_2.fq.gz\n",
      "Trimming mode: paired-end\n",
      "Trim Galore version: 0.4.4\n",
      "Cutadapt version: 1.9.1\n",
      "Quality Phred score cutoff: 20\n",
      "Quality encoding type selected: ASCII+33\n",
      "Adapter sequence: 'AGATCGGAAGAGC' (Illumina TruSeq, Sanger iPCR; auto-detected)\n",
      "Maximum trimming error rate: 0.1 (default)\n",
      "Minimum required adapter overlap (stringency): 1 bp\n",
      "Minimum required sequence length for both reads before a sequence pair gets removed: 20 bp\n",
      "Output file(s) will be GZIP compressed\n",
      "\n",
      "Writing final adapter and quality trimmed output to NR021_S8_L001_R2_001_val_2_trimmed.fq.gz\n",
      "\n",
      "\n",
      "  >>> Now performing quality (cutoff 20) and adapter trimming in a single pass for the adapter sequence: 'AGATCGGAAGAGC' from file /home/sam/data/geoduck_illumina/trimmed/NR021_S8_L001_R2_001_val_2.fq.gz <<< \n",
      "This is cutadapt 1.9.1 with Python 2.7.12\n",
      "Command line parameters: -f fastq -e 0.1 -q 20 -O 1 -a AGATCGGAAGAGC /home/sam/data/geoduck_illumina/trimmed/NR021_S8_L001_R2_001_val_2.fq.gz\n",
      "Trimming 1 adapter with at most 10.0% errors in single-end mode ...\n",
      "Finished in 106.50 s (27 us/read; 2.25 M reads/minute).\n",
      "\n",
      "=== Summary ===\n",
      "\n",
      "Total reads processed:               3,996,315\n",
      "Reads with adapters:                 2,365,417 (59.2%)\n",
      "Reads written (passing filters):     3,996,315 (100.0%)\n",
      "\n",
      "Total basepairs processed:   593,688,021 bp\n",
      "Quality-trimmed:                 161,140 bp (0.0%)\n",
      "Total written (filtered):    568,208,959 bp (95.7%)\n",
      "\n",
      "=== Adapter 1 ===\n",
      "\n",
      "Sequence: AGATCGGAAGAGC; Type: regular 3'; Length: 13; Trimmed: 2365417 times.\n",
      "\n",
      "No. of allowed errors:\n",
      "0-9 bp: 0; 10-13 bp: 1\n",
      "\n",
      "Bases preceding removed adapters:\n",
      "  A: 30.9%\n",
      "  C: 23.9%\n",
      "  G: 22.7%\n",
      "  T: 22.5%\n",
      "  none/other: 0.0%\n",
      "\n",
      "Overview of removed sequences\n",
      "length\tcount\texpect\tmax.err\terror counts\n",
      "1\t1092119\t999078.8\t0\t1092119\n",
      "2\t237188\t249769.7\t0\t237188\n",
      "3\t105726\t62442.4\t0\t105726\n",
      "4\t70512\t15610.6\t0\t70512\n",
      "5\t833\t3902.7\t0\t833\n",
      "6\t25608\t975.7\t0\t25608\n",
      "7\t24439\t243.9\t0\t24439\n",
      "8\t30007\t61.0\t0\t30007\n",
      "9\t19423\t15.2\t0\t19324 99\n",
      "10\t27007\t3.8\t1\t24438 2569\n",
      "11\t24193\t1.0\t1\t21508 2685\n",
      "12\t49848\t0.2\t1\t44992 4856\n",
      "13\t417\t0.1\t1\t163 254\n",
      "14\t30409\t0.1\t1\t26982 3427\n",
      "15\t41366\t0.1\t1\t37549 3817\n",
      "16\t176\t0.1\t1\t77 99\n",
      "17\t27857\t0.1\t1\t25162 2695\n",
      "18\t16568\t0.1\t1\t14861 1707\n",
      "19\t29519\t0.1\t1\t26578 2941\n",
      "20\t21091\t0.1\t1\t19067 2024\n",
      "21\t16792\t0.1\t1\t14962 1830\n",
      "22\t20201\t0.1\t1\t18364 1837\n",
      "23\t21276\t0.1\t1\t19178 2098\n",
      "24\t37690\t0.1\t1\t34176 3514\n",
      "25\t18430\t0.1\t1\t16766 1664\n",
      "26\t17597\t0.1\t1\t15864 1733\n",
      "27\t10354\t0.1\t1\t8997 1357\n",
      "28\t28466\t0.1\t1\t26325 2141\n",
      "29\t5853\t0.1\t1\t4981 872\n",
      "30\t25322\t0.1\t1\t23457 1865\n",
      "31\t8773\t0.1\t1\t7711 1062\n",
      "32\t40024\t0.1\t1\t37612 2412\n",
      "33\t15124\t0.1\t1\t13686 1438\n",
      "34\t5040\t0.1\t1\t4387 653\n",
      "35\t21707\t0.1\t1\t20267 1440\n",
      "36\t7141\t0.1\t1\t6402 739\n",
      "37\t9565\t0.1\t1\t8710 855\n",
      "38\t5456\t0.1\t1\t5052 404\n",
      "39\t7199\t0.1\t1\t6631 568\n",
      "40\t2942\t0.1\t1\t2733 209\n",
      "41\t9827\t0.1\t1\t9198 629\n",
      "42\t12807\t0.1\t1\t11985 822\n",
      "43\t5830\t0.1\t1\t5380 450\n",
      "44\t8947\t0.1\t1\t8286 661\n",
      "45\t14044\t0.1\t1\t13176 868\n",
      "46\t7086\t0.1\t1\t6524 562\n",
      "47\t4907\t0.1\t1\t4585 322\n",
      "48\t12935\t0.1\t1\t12161 774\n",
      "49\t5201\t0.1\t1\t4855 346\n",
      "50\t2042\t0.1\t1\t1867 175\n",
      "51\t8681\t0.1\t1\t8150 531\n",
      "52\t11816\t0.1\t1\t11253 563\n",
      "53\t4853\t0.1\t1\t4571 282\n",
      "54\t5760\t0.1\t1\t5434 326\n",
      "55\t3489\t0.1\t1\t3286 203\n",
      "56\t1741\t0.1\t1\t1609 132\n",
      "57\t2940\t0.1\t1\t2759 181\n",
      "58\t3041\t0.1\t1\t2850 191\n",
      "59\t1674\t0.1\t1\t1550 124\n",
      "60\t1051\t0.1\t1\t942 109\n",
      "61\t883\t0.1\t1\t805 78\n",
      "62\t943\t0.1\t1\t873 70\n",
      "63\t1034\t0.1\t1\t960 74\n",
      "64\t1718\t0.1\t1\t1633 85\n",
      "65\t1982\t0.1\t1\t1863 119\n",
      "66\t1986\t0.1\t1\t1876 110\n",
      "67\t1871\t0.1\t1\t1749 122\n",
      "68\t1762\t0.1\t1\t1657 105\n",
      "69\t1718\t0.1\t1\t1620 98\n",
      "70\t1574\t0.1\t1\t1476 98\n",
      "71\t1413\t0.1\t1\t1317 96\n",
      "72\t1343\t0.1\t1\t1269 74\n",
      "73\t1190\t0.1\t1\t1122 68\n",
      "74\t1114\t0.1\t1\t1048 66\n",
      "75\t1075\t0.1\t1\t1009 66\n",
      "76\t935\t0.1\t1\t881 54\n",
      "77\t910\t0.1\t1\t852 58\n",
      "78\t809\t0.1\t1\t773 36\n",
      "79\t788\t0.1\t1\t753 35\n",
      "80\t783\t0.1\t1\t742 41\n",
      "81\t682\t0.1\t1\t644 38\n",
      "82\t640\t0.1\t1\t583 57\n",
      "83\t582\t0.1\t1\t557 25\n",
      "84\t530\t0.1\t1\t496 34\n",
      "85\t475\t0.1\t1\t450 25\n",
      "86\t434\t0.1\t1\t407 27\n",
      "87\t422\t0.1\t1\t396 26\n",
      "88\t440\t0.1\t1\t409 31\n",
      "89\t363\t0.1\t1\t343 20\n",
      "90\t367\t0.1\t1\t338 29\n",
      "91\t370\t0.1\t1\t350 20\n",
      "92\t288\t0.1\t1\t265 23\n",
      "93\t266\t0.1\t1\t246 20\n",
      "94\t285\t0.1\t1\t268 17\n",
      "95\t198\t0.1\t1\t180 18\n",
      "96\t259\t0.1\t1\t242 17\n",
      "97\t224\t0.1\t1\t206 18\n",
      "98\t211\t0.1\t1\t201 10\n",
      "99\t195\t0.1\t1\t180 15\n",
      "100\t179\t0.1\t1\t160 19\n",
      "101\t176\t0.1\t1\t157 19\n",
      "102\t129\t0.1\t1\t118 11\n",
      "103\t144\t0.1\t1\t128 16\n",
      "104\t130\t0.1\t1\t119 11\n",
      "105\t99\t0.1\t1\t92 7\n",
      "106\t105\t0.1\t1\t94 11\n",
      "107\t98\t0.1\t1\t92 6\n",
      "108\t78\t0.1\t1\t70 8\n",
      "109\t101\t0.1\t1\t92 9\n",
      "110\t71\t0.1\t1\t67 4\n",
      "111\t92\t0.1\t1\t89 3\n",
      "112\t81\t0.1\t1\t72 9\n",
      "113\t95\t0.1\t1\t91 4\n",
      "114\t53\t0.1\t1\t45 8\n",
      "115\t80\t0.1\t1\t69 11\n",
      "116\t66\t0.1\t1\t56 10\n",
      "117\t54\t0.1\t1\t46 8\n",
      "118\t77\t0.1\t1\t69 8\n",
      "119\t60\t0.1\t1\t53 7\n",
      "120\t77\t0.1\t1\t69 8\n",
      "121\t55\t0.1\t1\t46 9\n",
      "122\t73\t0.1\t1\t68 5\n",
      "123\t54\t0.1\t1\t49 5\n",
      "124\t66\t0.1\t1\t59 7\n",
      "125\t73\t0.1\t1\t69 4\n",
      "126\t97\t0.1\t1\t89 8\n",
      "127\t62\t0.1\t1\t60 2\n",
      "128\t116\t0.1\t1\t104 12\n",
      "129\t102\t0.1\t1\t90 12\n",
      "130\t77\t0.1\t1\t66 11\n",
      "131\t97\t0.1\t1\t83 14\n",
      "132\t85\t0.1\t1\t81 4\n",
      "133\t93\t0.1\t1\t83 10\n",
      "134\t94\t0.1\t1\t76 18\n",
      "135\t95\t0.1\t1\t81 14\n",
      "136\t77\t0.1\t1\t69 8\n",
      "137\t107\t0.1\t1\t99 8\n",
      "138\t79\t0.1\t1\t72 7\n",
      "139\t95\t0.1\t1\t79 16\n",
      "140\t45\t0.1\t1\t33 12\n",
      "141\t49\t0.1\t1\t34 15\n",
      "142\t67\t0.1\t1\t43 24\n",
      "143\t52\t0.1\t1\t38 14\n",
      "144\t30\t0.1\t1\t15 15\n",
      "145\t27\t0.1\t1\t16 11\n",
      "146\t60\t0.1\t1\t13 47\n",
      "147\t42\t0.1\t1\t21 21\n",
      "148\t106\t0.1\t1\t62 44\n",
      "149\t32\t0.1\t1\t21 11\n",
      "150\t21\t0.1\t1\t14 7\n",
      "151\t252\t0.1\t1\t185 67\n",
      "\n",
      "\n",
      "RUN STATISTICS FOR INPUT FILE: /home/sam/data/geoduck_illumina/trimmed/NR021_S8_L001_R2_001_val_2.fq.gz\n",
      "=============================================\n",
      "3996315 sequences processed in total\n",
      "The length threshold of paired-end sequences gets evaluated later on (in the validation step)\n",
      "\n",
      "Validate paired-end files NR021_S8_L001_R1_001_val_1_trimmed.fq.gz and NR021_S8_L001_R2_001_val_2_trimmed.fq.gz\n",
      "file_1: NR021_S8_L001_R1_001_val_1_trimmed.fq.gz, file_2: NR021_S8_L001_R2_001_val_2_trimmed.fq.gz\n",
      "\n",
      "\n",
      ">>>>> Now validing the length of the 2 paired-end infiles: NR021_S8_L001_R1_001_val_1_trimmed.fq.gz and NR021_S8_L001_R2_001_val_2_trimmed.fq.gz <<<<<\n",
      "Writing validated paired-end read 1 reads to NR021_S8_L001_R1_001_val_1_val_1.fq.gz\n",
      "Writing validated paired-end read 2 reads to NR021_S8_L001_R2_001_val_2_val_2.fq.gz\n",
      "\n",
      "Total number of sequences analysed: 3996315\n",
      "\n",
      "Number of sequence pairs removed because at least one read was shorter than the length cutoff (20 bp): 27040 (0.68%)\n",
      "\n",
      "Deleting both intermediate output files NR021_S8_L001_R1_001_val_1_trimmed.fq.gz and NR021_S8_L001_R2_001_val_2_trimmed.fq.gz\n",
      "\n",
      "====================================================================================================\n",
      "\n",
      "Writing report to '/home/sam/data/geoduck_illumina/trimmed/20180129_trimmed_again/NR021_S8_L002_R1_001_val_1.fq.gz_trimming_report.txt'\n",
      "\n",
      "SUMMARISING RUN PARAMETERS\n",
      "==========================\n",
      "Input filename: /home/sam/data/geoduck_illumina/trimmed/NR021_S8_L002_R1_001_val_1.fq.gz\n",
      "Trimming mode: paired-end\n",
      "Trim Galore version: 0.4.4\n",
      "Cutadapt version: 1.9.1\n",
      "Quality Phred score cutoff: 20\n",
      "Quality encoding type selected: ASCII+33\n",
      "Adapter sequence: 'AGATCGGAAGAGC' (Illumina TruSeq, Sanger iPCR; auto-detected)\n",
      "Maximum trimming error rate: 0.1 (default)\n",
      "Minimum required adapter overlap (stringency): 1 bp\n",
      "Minimum required sequence length for both reads before a sequence pair gets removed: 20 bp\n",
      "Output file(s) will be GZIP compressed\n",
      "\n",
      "Writing final adapter and quality trimmed output to NR021_S8_L002_R1_001_val_1_trimmed.fq.gz\n",
      "\n",
      "\n",
      "  >>> Now performing quality (cutoff 20) and adapter trimming in a single pass for the adapter sequence: 'AGATCGGAAGAGC' from file /home/sam/data/geoduck_illumina/trimmed/NR021_S8_L002_R1_001_val_1.fq.gz <<< \n",
      "This is cutadapt 1.9.1 with Python 2.7.12\n",
      "Command line parameters: -f fastq -e 0.1 -q 20 -O 1 -a AGATCGGAAGAGC /home/sam/data/geoduck_illumina/trimmed/NR021_S8_L002_R1_001_val_1.fq.gz\n",
      "Trimming 1 adapter with at most 10.0% errors in single-end mode ...\n",
      "Finished in 91.64 s (26 us/read; 2.31 M reads/minute).\n",
      "\n",
      "=== Summary ===\n",
      "\n",
      "Total reads processed:               3,532,417\n",
      "Reads with adapters:                 2,146,057 (60.8%)\n",
      "Reads written (passing filters):     3,532,417 (100.0%)\n",
      "\n",
      "Total basepairs processed:   522,802,866 bp\n",
      "Quality-trimmed:                 102,517 bp (0.0%)\n",
      "Total written (filtered):    493,335,822 bp (94.4%)\n",
      "\n",
      "=== Adapter 1 ===\n",
      "\n",
      "Sequence: AGATCGGAAGAGC; Type: regular 3'; Length: 13; Trimmed: 2146057 times.\n",
      "\n",
      "No. of allowed errors:\n",
      "0-9 bp: 0; 10-13 bp: 1\n",
      "\n",
      "Bases preceding removed adapters:\n",
      "  A: 30.2%\n",
      "  C: 27.5%\n",
      "  G: 20.7%\n",
      "  T: 20.0%\n",
      "  none/other: 1.6%\n",
      "\n",
      "Overview of removed sequences\n",
      "length\tcount\texpect\tmax.err\terror counts\n",
      "1\t962828\t883104.2\t0\t962828\n",
      "2\t190615\t220776.1\t0\t190615\n",
      "3\t79318\t55194.0\t0\t79318\n",
      "4\t61991\t13798.5\t0\t61991\n",
      "5\t878\t3449.6\t0\t878\n",
      "6\t24256\t862.4\t0\t24256\n",
      "7\t21651\t215.6\t0\t21651\n",
      "8\t24057\t53.9\t0\t24057\n",
      "9\t22585\t13.5\t0\t22500 85\n",
      "10\t22526\t3.4\t1\t21612 914\n",
      "11\t23516\t0.8\t1\t22109 1407\n",
      "12\t44734\t0.2\t1\t42931 1803\n",
      "13\t915\t0.1\t1\t541 374\n",
      "14\t43908\t0.1\t1\t41920 1988\n",
      "15\t629\t0.1\t1\t342 287\n",
      "16\t45177\t0.1\t1\t42939 2238\n",
      "17\t992\t0.1\t1\t649 343\n",
      "18\t26506\t0.1\t1\t25049 1457\n",
      "19\t72561\t0.1\t1\t69795 2766\n",
      "20\t462\t0.1\t1\t263 199\n",
      "21\t116\t0.1\t1\t93 23\n",
      "22\t353\t0.1\t1\t232 121\n",
      "23\t20249\t0.1\t1\t19187 1062\n",
      "24\t53005\t0.1\t1\t50663 2342\n",
      "25\t791\t0.1\t1\t527 264\n",
      "26\t17722\t0.1\t1\t16903 819\n",
      "27\t16734\t0.1\t1\t16040 694\n",
      "28\t327\t0.1\t1\t222 105\n",
      "29\t16504\t0.1\t1\t15682 822\n",
      "30\t18747\t0.1\t1\t17875 872\n",
      "31\t26015\t0.1\t1\t25060 955\n",
      "32\t648\t0.1\t1\t440 208\n",
      "33\t27918\t0.1\t1\t26622 1296\n",
      "34\t326\t0.1\t1\t210 116\n",
      "35\t12569\t0.1\t1\t12057 512\n",
      "36\t12619\t0.1\t1\t12021 598\n",
      "37\t13184\t0.1\t1\t12533 651\n",
      "38\t21713\t0.1\t1\t20828 885\n",
      "39\t339\t0.1\t1\t246 93\n",
      "40\t10499\t0.1\t1\t10040 459\n",
      "41\t9561\t0.1\t1\t9084 477\n",
      "42\t11337\t0.1\t1\t10789 548\n",
      "43\t9078\t0.1\t1\t8480 598\n",
      "44\t23113\t0.1\t1\t22033 1080\n",
      "45\t4765\t0.1\t1\t4517 248\n",
      "46\t3889\t0.1\t1\t3598 291\n",
      "47\t597\t0.1\t1\t398 199\n",
      "48\t39522\t0.1\t1\t38097 1425\n",
      "49\t12892\t0.1\t1\t12326 566\n",
      "50\t1792\t0.1\t1\t1603 189\n",
      "51\t9666\t0.1\t1\t9248 418\n",
      "52\t6354\t0.1\t1\t6086 268\n",
      "53\t740\t0.1\t1\t658 82\n",
      "54\t247\t0.1\t1\t216 31\n",
      "55\t3374\t0.1\t1\t3241 133\n",
      "56\t1493\t0.1\t1\t1419 74\n",
      "57\t137\t0.1\t1\t114 23\n",
      "58\t524\t0.1\t1\t470 54\n",
      "59\t965\t0.1\t1\t894 71\n",
      "60\t223\t0.1\t1\t184 39\n",
      "61\t416\t0.1\t1\t387 29\n",
      "62\t2071\t0.1\t1\t1977 94\n",
      "63\t195\t0.1\t1\t170 25\n",
      "64\t461\t0.1\t1\t425 36\n",
      "65\t562\t0.1\t1\t520 42\n",
      "66\t1783\t0.1\t1\t1723 60\n",
      "67\t320\t0.1\t1\t285 35\n",
      "68\t845\t0.1\t1\t795 50\n",
      "69\t1033\t0.1\t1\t974 59\n",
      "70\t1298\t0.1\t1\t1227 71\n",
      "71\t382\t0.1\t1\t338 44\n",
      "72\t162\t0.1\t1\t143 19\n",
      "73\t419\t0.1\t1\t400 19\n",
      "74\t737\t0.1\t1\t699 38\n",
      "75\t940\t0.1\t1\t884 56\n",
      "76\t991\t0.1\t1\t942 49\n",
      "77\t930\t0.1\t1\t877 53\n",
      "78\t909\t0.1\t1\t857 52\n",
      "79\t883\t0.1\t1\t825 58\n",
      "80\t831\t0.1\t1\t784 47\n",
      "81\t759\t0.1\t1\t714 45\n",
      "82\t727\t0.1\t1\t679 48\n",
      "83\t645\t0.1\t1\t611 34\n",
      "84\t585\t0.1\t1\t548 37\n",
      "85\t543\t0.1\t1\t505 38\n",
      "86\t507\t0.1\t1\t465 42\n",
      "87\t472\t0.1\t1\t439 33\n",
      "88\t457\t0.1\t1\t432 25\n",
      "89\t410\t0.1\t1\t381 29\n",
      "90\t403\t0.1\t1\t371 32\n",
      "91\t387\t0.1\t1\t355 32\n",
      "92\t362\t0.1\t1\t333 29\n",
      "93\t280\t0.1\t1\t260 20\n",
      "94\t317\t0.1\t1\t296 21\n",
      "95\t291\t0.1\t1\t264 27\n",
      "96\t284\t0.1\t1\t256 28\n",
      "97\t233\t0.1\t1\t215 18\n",
      "98\t248\t0.1\t1\t232 16\n",
      "99\t230\t0.1\t1\t214 16\n",
      "100\t227\t0.1\t1\t206 21\n",
      "101\t194\t0.1\t1\t178 16\n",
      "102\t185\t0.1\t1\t168 17\n",
      "103\t184\t0.1\t1\t162 22\n",
      "104\t182\t0.1\t1\t166 16\n",
      "105\t119\t0.1\t1\t109 10\n",
      "106\t126\t0.1\t1\t115 11\n",
      "107\t154\t0.1\t1\t136 18\n",
      "108\t142\t0.1\t1\t123 19\n",
      "109\t138\t0.1\t1\t130 8\n",
      "110\t132\t0.1\t1\t112 20\n",
      "111\t118\t0.1\t1\t108 10\n",
      "112\t85\t0.1\t1\t74 11\n",
      "113\t148\t0.1\t1\t137 11\n",
      "114\t91\t0.1\t1\t85 6\n",
      "115\t98\t0.1\t1\t86 12\n",
      "116\t101\t0.1\t1\t93 8\n",
      "117\t76\t0.1\t1\t69 7\n",
      "118\t91\t0.1\t1\t86 5\n",
      "119\t83\t0.1\t1\t77 6\n",
      "120\t99\t0.1\t1\t92 7\n",
      "121\t82\t0.1\t1\t73 9\n",
      "122\t87\t0.1\t1\t82 5\n",
      "123\t75\t0.1\t1\t69 6\n",
      "124\t79\t0.1\t1\t71 8\n",
      "125\t92\t0.1\t1\t82 10\n",
      "126\t158\t0.1\t1\t138 20\n",
      "127\t119\t0.1\t1\t112 7\n",
      "128\t165\t0.1\t1\t151 14\n",
      "129\t147\t0.1\t1\t139 8\n",
      "130\t130\t0.1\t1\t124 6\n",
      "131\t111\t0.1\t1\t106 5\n",
      "132\t152\t0.1\t1\t144 8\n",
      "133\t99\t0.1\t1\t91 8\n",
      "134\t149\t0.1\t1\t140 9\n",
      "135\t105\t0.1\t1\t101 4\n",
      "136\t125\t0.1\t1\t121 4\n",
      "137\t167\t0.1\t1\t159 8\n",
      "138\t107\t0.1\t1\t105 2\n",
      "139\t118\t0.1\t1\t114 4\n",
      "140\t89\t0.1\t1\t84 5\n",
      "141\t53\t0.1\t1\t51 2\n",
      "142\t108\t0.1\t1\t103 5\n",
      "143\t61\t0.1\t1\t57 4\n",
      "144\t46\t0.1\t1\t44 2\n",
      "145\t53\t0.1\t1\t49 4\n",
      "146\t99\t0.1\t1\t91 8\n",
      "147\t59\t0.1\t1\t46 13\n",
      "148\t177\t0.1\t1\t169 8\n",
      "149\t66\t0.1\t1\t52 14\n",
      "150\t129\t0.1\t1\t16 113\n",
      "151\t35017\t0.1\t1\t286 34731\n",
      "\n",
      "\n",
      "RUN STATISTICS FOR INPUT FILE: /home/sam/data/geoduck_illumina/trimmed/NR021_S8_L002_R1_001_val_1.fq.gz\n",
      "=============================================\n",
      "3532417 sequences processed in total\n",
      "The length threshold of paired-end sequences gets evaluated later on (in the validation step)\n",
      "\n",
      "Writing report to '/home/sam/data/geoduck_illumina/trimmed/20180129_trimmed_again/NR021_S8_L002_R2_001_val_2.fq.gz_trimming_report.txt'\n",
      "\n",
      "SUMMARISING RUN PARAMETERS\n",
      "==========================\n",
      "Input filename: /home/sam/data/geoduck_illumina/trimmed/NR021_S8_L002_R2_001_val_2.fq.gz\n",
      "Trimming mode: paired-end\n",
      "Trim Galore version: 0.4.4\n",
      "Cutadapt version: 1.9.1\n",
      "Quality Phred score cutoff: 20\n",
      "Quality encoding type selected: ASCII+33\n",
      "Adapter sequence: 'AGATCGGAAGAGC' (Illumina TruSeq, Sanger iPCR; auto-detected)\n",
      "Maximum trimming error rate: 0.1 (default)\n",
      "Minimum required adapter overlap (stringency): 1 bp\n",
      "Minimum required sequence length for both reads before a sequence pair gets removed: 20 bp\n",
      "Output file(s) will be GZIP compressed\n",
      "\n",
      "Writing final adapter and quality trimmed output to NR021_S8_L002_R2_001_val_2_trimmed.fq.gz\n",
      "\n",
      "\n",
      "  >>> Now performing quality (cutoff 20) and adapter trimming in a single pass for the adapter sequence: 'AGATCGGAAGAGC' from file /home/sam/data/geoduck_illumina/trimmed/NR021_S8_L002_R2_001_val_2.fq.gz <<< \n",
      "This is cutadapt 1.9.1 with Python 2.7.12\n",
      "Command line parameters: -f fastq -e 0.1 -q 20 -O 1 -a AGATCGGAAGAGC /home/sam/data/geoduck_illumina/trimmed/NR021_S8_L002_R2_001_val_2.fq.gz\n",
      "Trimming 1 adapter with at most 10.0% errors in single-end mode ...\n",
      "Finished in 92.95 s (26 us/read; 2.28 M reads/minute).\n",
      "\n",
      "=== Summary ===\n",
      "\n",
      "Total reads processed:               3,532,417\n",
      "Reads with adapters:                 2,104,235 (59.6%)\n",
      "Reads written (passing filters):     3,532,417 (100.0%)\n",
      "\n",
      "Total basepairs processed:   523,752,676 bp\n",
      "Quality-trimmed:                 107,379 bp (0.0%)\n",
      "Total written (filtered):    499,307,219 bp (95.3%)\n",
      "\n",
      "=== Adapter 1 ===\n",
      "\n",
      "Sequence: AGATCGGAAGAGC; Type: regular 3'; Length: 13; Trimmed: 2104235 times.\n",
      "\n",
      "No. of allowed errors:\n",
      "0-9 bp: 0; 10-13 bp: 1\n",
      "\n",
      "Bases preceding removed adapters:\n",
      "  A: 30.6%\n",
      "  C: 24.2%\n",
      "  G: 22.9%\n",
      "  T: 22.2%\n",
      "  none/other: 0.0%\n",
      "\n",
      "Overview of removed sequences\n",
      "length\tcount\texpect\tmax.err\terror counts\n",
      "1\t935006\t883104.2\t0\t935006\n",
      "2\t205452\t220776.1\t0\t205452\n",
      "3\t90078\t55194.0\t0\t90078\n",
      "4\t63626\t13798.5\t0\t63626\n",
      "5\t727\t3449.6\t0\t727\n",
      "6\t23747\t862.4\t0\t23747\n",
      "7\t22619\t215.6\t0\t22619\n",
      "8\t26870\t53.9\t0\t26870\n",
      "9\t19116\t13.5\t0\t19027 89\n",
      "10\t23453\t3.4\t1\t22285 1168\n",
      "11\t22380\t0.8\t1\t21062 1318\n",
      "12\t45677\t0.2\t1\t43499 2178\n",
      "13\t267\t0.1\t1\t86 181\n",
      "14\t26102\t0.1\t1\t24555 1547\n",
      "15\t39664\t0.1\t1\t37859 1805\n",
      "16\t155\t0.1\t1\t69 86\n",
      "17\t25075\t0.1\t1\t23887 1188\n",
      "18\t16252\t0.1\t1\t15419 833\n",
      "19\t25135\t0.1\t1\t23806 1329\n",
      "20\t19314\t0.1\t1\t18388 926\n",
      "21\t17365\t0.1\t1\t16477 888\n",
      "22\t18767\t0.1\t1\t17902 865\n",
      "23\t18776\t0.1\t1\t17846 930\n",
      "24\t28530\t0.1\t1\t27083 1447\n",
      "25\t18607\t0.1\t1\t17836 771\n",
      "26\t17570\t0.1\t1\t16733 837\n",
      "27\t10369\t0.1\t1\t9719 650\n",
      "28\t26829\t0.1\t1\t25809 1020\n",
      "29\t5986\t0.1\t1\t5499 487\n",
      "30\t23753\t0.1\t1\t22893 860\n",
      "31\t8001\t0.1\t1\t7469 532\n",
      "32\t44562\t0.1\t1\t43349 1213\n",
      "33\t13829\t0.1\t1\t13159 670\n",
      "34\t4413\t0.1\t1\t4065 348\n",
      "35\t23178\t0.1\t1\t22495 683\n",
      "36\t4811\t0.1\t1\t4512 299\n",
      "37\t9949\t0.1\t1\t9472 477\n",
      "38\t3118\t0.1\t1\t2940 178\n",
      "39\t5489\t0.1\t1\t5229 260\n",
      "40\t1309\t0.1\t1\t1223 86\n",
      "41\t9613\t0.1\t1\t9261 352\n",
      "42\t13101\t0.1\t1\t12699 402\n",
      "43\t4956\t0.1\t1\t4710 246\n",
      "44\t8731\t0.1\t1\t8370 361\n",
      "45\t16619\t0.1\t1\t16155 464\n",
      "46\t6813\t0.1\t1\t6510 303\n",
      "47\t3769\t0.1\t1\t3616 153\n",
      "48\t15026\t0.1\t1\t14589 437\n",
      "49\t3833\t0.1\t1\t3678 155\n",
      "50\t1098\t0.1\t1\t1019 79\n",
      "51\t9289\t0.1\t1\t9011 278\n",
      "52\t15274\t0.1\t1\t14904 370\n",
      "53\t4134\t0.1\t1\t3986 148\n",
      "54\t5251\t0.1\t1\t5085 166\n",
      "55\t2132\t0.1\t1\t2063 69\n",
      "56\t1191\t0.1\t1\t1138 53\n",
      "57\t2299\t0.1\t1\t2215 84\n",
      "58\t2389\t0.1\t1\t2297 92\n",
      "59\t1056\t0.1\t1\t1001 55\n",
      "60\t693\t0.1\t1\t647 46\n",
      "61\t708\t0.1\t1\t657 51\n",
      "62\t635\t0.1\t1\t588 47\n",
      "63\t777\t0.1\t1\t738 39\n",
      "64\t1588\t0.1\t1\t1524 64\n",
      "65\t1955\t0.1\t1\t1868 87\n",
      "66\t2047\t0.1\t1\t1964 83\n",
      "67\t1954\t0.1\t1\t1884 70\n",
      "68\t1866\t0.1\t1\t1781 85\n",
      "69\t1667\t0.1\t1\t1603 64\n",
      "70\t1558\t0.1\t1\t1492 66\n",
      "71\t1433\t0.1\t1\t1377 56\n",
      "72\t1397\t0.1\t1\t1343 54\n",
      "73\t1311\t0.1\t1\t1253 58\n",
      "74\t1112\t0.1\t1\t1053 59\n",
      "75\t1134\t0.1\t1\t1085 49\n",
      "76\t1075\t0.1\t1\t1029 46\n",
      "77\t961\t0.1\t1\t919 42\n",
      "78\t969\t0.1\t1\t932 37\n",
      "79\t908\t0.1\t1\t843 65\n",
      "80\t855\t0.1\t1\t827 28\n",
      "81\t788\t0.1\t1\t751 37\n",
      "82\t752\t0.1\t1\t717 35\n",
      "83\t664\t0.1\t1\t638 26\n",
      "84\t608\t0.1\t1\t580 28\n",
      "85\t572\t0.1\t1\t531 41\n",
      "86\t517\t0.1\t1\t491 26\n",
      "87\t500\t0.1\t1\t472 28\n",
      "88\t502\t0.1\t1\t477 25\n",
      "89\t425\t0.1\t1\t408 17\n",
      "90\t412\t0.1\t1\t384 28\n",
      "91\t406\t0.1\t1\t376 30\n",
      "92\t378\t0.1\t1\t364 14\n",
      "93\t311\t0.1\t1\t288 23\n",
      "94\t349\t0.1\t1\t317 32\n",
      "95\t314\t0.1\t1\t289 25\n",
      "96\t291\t0.1\t1\t270 21\n",
      "97\t264\t0.1\t1\t244 20\n",
      "98\t273\t0.1\t1\t258 15\n",
      "99\t249\t0.1\t1\t231 18\n",
      "100\t247\t0.1\t1\t231 16\n",
      "101\t235\t0.1\t1\t217 18\n",
      "102\t198\t0.1\t1\t182 16\n",
      "103\t191\t0.1\t1\t180 11\n",
      "104\t207\t0.1\t1\t191 16\n",
      "105\t136\t0.1\t1\t123 13\n",
      "106\t155\t0.1\t1\t138 17\n",
      "107\t158\t0.1\t1\t139 19\n",
      "108\t156\t0.1\t1\t138 18\n",
      "109\t149\t0.1\t1\t134 15\n",
      "110\t149\t0.1\t1\t132 17\n",
      "111\t131\t0.1\t1\t121 10\n",
      "112\t93\t0.1\t1\t83 10\n",
      "113\t159\t0.1\t1\t147 12\n",
      "114\t108\t0.1\t1\t96 12\n",
      "115\t101\t0.1\t1\t86 15\n",
      "116\t117\t0.1\t1\t105 12\n",
      "117\t88\t0.1\t1\t80 8\n",
      "118\t113\t0.1\t1\t106 7\n",
      "119\t91\t0.1\t1\t81 10\n",
      "120\t112\t0.1\t1\t105 7\n",
      "121\t89\t0.1\t1\t78 11\n",
      "122\t94\t0.1\t1\t88 6\n",
      "123\t90\t0.1\t1\t78 12\n",
      "124\t87\t0.1\t1\t79 8\n",
      "125\t95\t0.1\t1\t83 12\n",
      "126\t156\t0.1\t1\t149 7\n",
      "127\t127\t0.1\t1\t116 11\n",
      "128\t168\t0.1\t1\t156 12\n",
      "129\t151\t0.1\t1\t127 24\n",
      "130\t134\t0.1\t1\t121 13\n",
      "131\t117\t0.1\t1\t99 18\n",
      "132\t152\t0.1\t1\t143 9\n",
      "133\t115\t0.1\t1\t98 17\n",
      "134\t148\t0.1\t1\t126 22\n",
      "135\t102\t0.1\t1\t93 9\n",
      "136\t125\t0.1\t1\t103 22\n",
      "137\t166\t0.1\t1\t149 17\n",
      "138\t110\t0.1\t1\t99 11\n",
      "139\t119\t0.1\t1\t100 19\n",
      "140\t89\t0.1\t1\t72 17\n",
      "141\t49\t0.1\t1\t42 7\n",
      "142\t108\t0.1\t1\t81 27\n",
      "143\t58\t0.1\t1\t46 12\n",
      "144\t46\t0.1\t1\t30 16\n",
      "145\t49\t0.1\t1\t28 21\n",
      "146\t94\t0.1\t1\t39 55\n",
      "147\t49\t0.1\t1\t34 15\n",
      "148\t167\t0.1\t1\t115 52\n",
      "149\t50\t0.1\t1\t29 21\n",
      "150\t21\t0.1\t1\t11 10\n",
      "151\t338\t0.1\t1\t261 77\n",
      "\n",
      "\n",
      "RUN STATISTICS FOR INPUT FILE: /home/sam/data/geoduck_illumina/trimmed/NR021_S8_L002_R2_001_val_2.fq.gz\n",
      "=============================================\n",
      "3532417 sequences processed in total\n",
      "The length threshold of paired-end sequences gets evaluated later on (in the validation step)\n",
      "\n",
      "Validate paired-end files NR021_S8_L002_R1_001_val_1_trimmed.fq.gz and NR021_S8_L002_R2_001_val_2_trimmed.fq.gz\n",
      "file_1: NR021_S8_L002_R1_001_val_1_trimmed.fq.gz, file_2: NR021_S8_L002_R2_001_val_2_trimmed.fq.gz\n",
      "\n",
      "\n",
      ">>>>> Now validing the length of the 2 paired-end infiles: NR021_S8_L002_R1_001_val_1_trimmed.fq.gz and NR021_S8_L002_R2_001_val_2_trimmed.fq.gz <<<<<\n",
      "Writing validated paired-end read 1 reads to NR021_S8_L002_R1_001_val_1_val_1.fq.gz\n",
      "Writing validated paired-end read 2 reads to NR021_S8_L002_R2_001_val_2_val_2.fq.gz\n",
      "\n",
      "Total number of sequences analysed: 3532417\n",
      "\n",
      "Number of sequence pairs removed because at least one read was shorter than the length cutoff (20 bp): 37926 (1.07%)\n",
      "\n",
      "Deleting both intermediate output files NR021_S8_L002_R1_001_val_1_trimmed.fq.gz and NR021_S8_L002_R2_001_val_2_trimmed.fq.gz\n",
      "\n",
      "====================================================================================================\n",
      "\n",
      "\n",
      "real\t4901m40.787s\n",
      "user\t10209m3.532s\n",
      "sys\t213m39.764s\n"
     ]
    }
   ],
   "source": [
    "%%bash\n",
    "mkdir /home/sam/data/geoduck_illumina/trimmed/20180129_trimmed_again\n",
    "time /home/shared/trimgalore/trim_galore \\\n",
    "--paired \\\n",
    "--output_dir /home/sam/data/geoduck_illumina/trimmed/20180129_trimmed_again \\\n",
    "/home/sam/data/geoduck_illumina/trimmed/AD002_S9_L001_R1_001_val_1.fq.gz \\\n",
    "/home/sam/data/geoduck_illumina/trimmed/AD002_S9_L001_R2_001_val_2.fq.gz \\\n",
    "/home/sam/data/geoduck_illumina/trimmed/AD002_S9_L002_R1_001_val_1.fq.gz \\\n",
    "/home/sam/data/geoduck_illumina/trimmed/AD002_S9_L002_R2_001_val_2.fq.gz \\\n",
    "/home/sam/data/geoduck_illumina/trimmed/NR005_S4_L001_R1_001_val_1.fq.gz \\\n",
    "/home/sam/data/geoduck_illumina/trimmed/NR005_S4_L001_R2_001_val_2.fq.gz \\\n",
    "/home/sam/data/geoduck_illumina/trimmed/NR005_S4_L002_R1_001_val_1.fq.gz \\\n",
    "/home/sam/data/geoduck_illumina/trimmed/NR005_S4_L002_R2_001_val_2.fq.gz \\\n",
    "/home/sam/data/geoduck_illumina/trimmed/NR006_S3_L001_R1_001_val_1.fq.gz \\\n",
    "/home/sam/data/geoduck_illumina/trimmed/NR006_S3_L001_R2_001_val_2.fq.gz \\\n",
    "/home/sam/data/geoduck_illumina/trimmed/NR006_S3_L002_R1_001_val_1.fq.gz \\\n",
    "/home/sam/data/geoduck_illumina/trimmed/NR006_S3_L002_R2_001_val_2.fq.gz \\\n",
    "/home/sam/data/geoduck_illumina/trimmed/NR012_S1_L001_R1_001_val_1.fq.gz \\\n",
    "/home/sam/data/geoduck_illumina/trimmed/NR012_S1_L001_R2_001_val_2.fq.gz \\\n",
    "/home/sam/data/geoduck_illumina/trimmed/NR012_S1_L002_R1_001_val_1.fq.gz \\\n",
    "/home/sam/data/geoduck_illumina/trimmed/NR012_S1_L002_R2_001_val_2.fq.gz \\\n",
    "/home/sam/data/geoduck_illumina/trimmed/NR013_AD013_S2_L001_R1_001_val_1.fq.gz \\\n",
    "/home/sam/data/geoduck_illumina/trimmed/NR013_AD013_S2_L001_R2_001_val_2.fq.gz \\\n",
    "/home/sam/data/geoduck_illumina/trimmed/NR013_AD013_S2_L002_R1_001_val_1.fq.gz \\\n",
    "/home/sam/data/geoduck_illumina/trimmed/NR013_AD013_S2_L002_R2_001_val_2.fq.gz \\\n",
    "/home/sam/data/geoduck_illumina/trimmed/NR014_AD014_S5_L001_R1_001_val_1.fq.gz \\\n",
    "/home/sam/data/geoduck_illumina/trimmed/NR014_AD014_S5_L001_R2_001_val_2.fq.gz \\\n",
    "/home/sam/data/geoduck_illumina/trimmed/NR014_AD014_S5_L002_R1_001_val_1.fq.gz \\\n",
    "/home/sam/data/geoduck_illumina/trimmed/NR014_AD014_S5_L002_R2_001_val_2.fq.gz \\\n",
    "/home/sam/data/geoduck_illumina/trimmed/NR015_AD015_S6_L001_R1_001_val_1.fq.gz \\\n",
    "/home/sam/data/geoduck_illumina/trimmed/NR015_AD015_S6_L001_R2_001_val_2.fq.gz \\\n",
    "/home/sam/data/geoduck_illumina/trimmed/NR015_AD015_S6_L002_R1_001_val_1.fq.gz \\\n",
    "/home/sam/data/geoduck_illumina/trimmed/NR015_AD015_S6_L002_R2_001_val_2.fq.gz \\\n",
    "/home/sam/data/geoduck_illumina/trimmed/NR019_S7_L001_R1_001_val_1.fq.gz \\\n",
    "/home/sam/data/geoduck_illumina/trimmed/NR019_S7_L001_R2_001_val_2.fq.gz \\\n",
    "/home/sam/data/geoduck_illumina/trimmed/NR019_S7_L002_R1_001_val_1.fq.gz \\\n",
    "/home/sam/data/geoduck_illumina/trimmed/NR019_S7_L002_R2_001_val_2.fq.gz \\\n",
    "/home/sam/data/geoduck_illumina/trimmed/NR021_S8_L001_R1_001_val_1.fq.gz \\\n",
    "/home/sam/data/geoduck_illumina/trimmed/NR021_S8_L001_R2_001_val_2.fq.gz \\\n",
    "/home/sam/data/geoduck_illumina/trimmed/NR021_S8_L002_R1_001_val_1.fq.gz \\\n",
    "/home/sam/data/geoduck_illumina/trimmed/NR021_S8_L002_R2_001_val_2.fq.gz\n",
    "cat ~/.default-subject.mail | msmtp \"$EMAIL\""
   ]
  },
  {
   "cell_type": "code",
   "execution_count": 48,
   "metadata": {},
   "outputs": [
    {
     "name": "stdout",
     "output_type": "stream",
     "text": [
      "AD002_S9_L001_R1_001_val_1.fq.gz_trimming_report.txt\n",
      "AD002_S9_L001_R1_001_val_1_val_1.fq.gz\n",
      "AD002_S9_L001_R2_001_val_2.fq.gz_trimming_report.txt\n",
      "AD002_S9_L001_R2_001_val_2_val_2.fq.gz\n",
      "AD002_S9_L002_R1_001_val_1.fq.gz_trimming_report.txt\n",
      "AD002_S9_L002_R1_001_val_1_val_1.fq.gz\n",
      "AD002_S9_L002_R2_001_val_2.fq.gz_trimming_report.txt\n",
      "AD002_S9_L002_R2_001_val_2_val_2.fq.gz\n",
      "NR005_S4_L001_R1_001_val_1.fq.gz_trimming_report.txt\n",
      "NR005_S4_L001_R1_001_val_1_val_1.fq.gz\n",
      "NR005_S4_L001_R2_001_val_2.fq.gz_trimming_report.txt\n",
      "NR005_S4_L001_R2_001_val_2_val_2.fq.gz\n",
      "NR005_S4_L002_R1_001_val_1.fq.gz_trimming_report.txt\n",
      "NR005_S4_L002_R1_001_val_1_val_1.fq.gz\n",
      "NR005_S4_L002_R2_001_val_2.fq.gz_trimming_report.txt\n",
      "NR005_S4_L002_R2_001_val_2_val_2.fq.gz\n",
      "NR006_S3_L001_R1_001_val_1.fq.gz_trimming_report.txt\n",
      "NR006_S3_L001_R1_001_val_1_val_1.fq.gz\n",
      "NR006_S3_L001_R2_001_val_2.fq.gz_trimming_report.txt\n",
      "NR006_S3_L001_R2_001_val_2_val_2.fq.gz\n",
      "NR006_S3_L002_R1_001_val_1.fq.gz_trimming_report.txt\n",
      "NR006_S3_L002_R1_001_val_1_val_1.fq.gz\n",
      "NR006_S3_L002_R2_001_val_2.fq.gz_trimming_report.txt\n",
      "NR006_S3_L002_R2_001_val_2_val_2.fq.gz\n",
      "NR012_S1_L001_R1_001_val_1.fq.gz_trimming_report.txt\n",
      "NR012_S1_L001_R1_001_val_1_val_1.fq.gz\n",
      "NR012_S1_L001_R2_001_val_2.fq.gz_trimming_report.txt\n",
      "NR012_S1_L001_R2_001_val_2_val_2.fq.gz\n",
      "NR012_S1_L002_R1_001_val_1.fq.gz_trimming_report.txt\n",
      "NR012_S1_L002_R1_001_val_1_val_1.fq.gz\n",
      "NR012_S1_L002_R2_001_val_2.fq.gz_trimming_report.txt\n",
      "NR012_S1_L002_R2_001_val_2_val_2.fq.gz\n",
      "NR013_AD013_S2_L001_R1_001_val_1.fq.gz_trimming_report.txt\n",
      "NR013_AD013_S2_L001_R1_001_val_1_val_1.fq.gz\n",
      "NR013_AD013_S2_L001_R2_001_val_2.fq.gz_trimming_report.txt\n",
      "NR013_AD013_S2_L001_R2_001_val_2_val_2.fq.gz\n",
      "NR013_AD013_S2_L002_R1_001_val_1.fq.gz_trimming_report.txt\n",
      "NR013_AD013_S2_L002_R1_001_val_1_val_1.fq.gz\n",
      "NR013_AD013_S2_L002_R2_001_val_2.fq.gz_trimming_report.txt\n",
      "NR013_AD013_S2_L002_R2_001_val_2_val_2.fq.gz\n",
      "NR014_AD014_S5_L001_R1_001_val_1.fq.gz_trimming_report.txt\n",
      "NR014_AD014_S5_L001_R1_001_val_1_val_1.fq.gz\n",
      "NR014_AD014_S5_L001_R2_001_val_2.fq.gz_trimming_report.txt\n",
      "NR014_AD014_S5_L001_R2_001_val_2_val_2.fq.gz\n",
      "NR014_AD014_S5_L002_R1_001_val_1.fq.gz_trimming_report.txt\n",
      "NR014_AD014_S5_L002_R1_001_val_1_val_1.fq.gz\n",
      "NR014_AD014_S5_L002_R2_001_val_2.fq.gz_trimming_report.txt\n",
      "NR014_AD014_S5_L002_R2_001_val_2_val_2.fq.gz\n",
      "NR015_AD015_S6_L001_R1_001_val_1.fq.gz_trimming_report.txt\n",
      "NR015_AD015_S6_L001_R1_001_val_1_val_1.fq.gz\n",
      "NR015_AD015_S6_L001_R2_001_val_2.fq.gz_trimming_report.txt\n",
      "NR015_AD015_S6_L001_R2_001_val_2_val_2.fq.gz\n",
      "NR015_AD015_S6_L002_R1_001_val_1.fq.gz_trimming_report.txt\n",
      "NR015_AD015_S6_L002_R1_001_val_1_val_1.fq.gz\n",
      "NR015_AD015_S6_L002_R2_001_val_2.fq.gz_trimming_report.txt\n",
      "NR015_AD015_S6_L002_R2_001_val_2_val_2.fq.gz\n",
      "NR019_S7_L001_R1_001_val_1.fq.gz_trimming_report.txt\n",
      "NR019_S7_L001_R1_001_val_1_val_1.fq.gz\n",
      "NR019_S7_L001_R2_001_val_2.fq.gz_trimming_report.txt\n",
      "NR019_S7_L001_R2_001_val_2_val_2.fq.gz\n",
      "NR019_S7_L002_R1_001_val_1.fq.gz_trimming_report.txt\n",
      "NR019_S7_L002_R1_001_val_1_val_1.fq.gz\n",
      "NR019_S7_L002_R2_001_val_2.fq.gz_trimming_report.txt\n",
      "NR019_S7_L002_R2_001_val_2_val_2.fq.gz\n",
      "NR021_S8_L001_R1_001_val_1.fq.gz_trimming_report.txt\n",
      "NR021_S8_L001_R1_001_val_1_val_1.fq.gz\n",
      "NR021_S8_L001_R2_001_val_2.fq.gz_trimming_report.txt\n",
      "NR021_S8_L001_R2_001_val_2_val_2.fq.gz\n",
      "NR021_S8_L002_R1_001_val_1.fq.gz_trimming_report.txt\n",
      "NR021_S8_L002_R1_001_val_1_val_1.fq.gz\n",
      "NR021_S8_L002_R2_001_val_2.fq.gz_trimming_report.txt\n",
      "NR021_S8_L002_R2_001_val_2_val_2.fq.gz\n"
     ]
    }
   ],
   "source": [
    "%%bash\n",
    "ls /home/sam/data/geoduck_illumina/trimmed/20180129_trimmed_again/"
   ]
  },
  {
   "cell_type": "code",
   "execution_count": 49,
   "metadata": {},
   "outputs": [
    {
     "name": "stdout",
     "output_type": "stream",
     "text": [
      "/home/sam/data/geoduck_illumina/trimmed/20180129_trimmed_again\n"
     ]
    }
   ],
   "source": [
    "%%bash\n",
    "cd /home/sam/data/geoduck_illumina/trimmed/20180129_trimmed_again/\n",
    "pwd"
   ]
  },
  {
   "cell_type": "code",
   "execution_count": 50,
   "metadata": {},
   "outputs": [
    {
     "name": "stdout",
     "output_type": "stream",
     "text": [
      "/home/sam/data/geoduck_illumina/trimmed/20180129_trimmed_again/AD002_S9_L001_R1_001_val_1_val_1.fq.gz \\\n",
      "/home/sam/data/geoduck_illumina/trimmed/20180129_trimmed_again/AD002_S9_L001_R2_001_val_2_val_2.fq.gz \\\n",
      "/home/sam/data/geoduck_illumina/trimmed/20180129_trimmed_again/AD002_S9_L002_R1_001_val_1_val_1.fq.gz \\\n",
      "/home/sam/data/geoduck_illumina/trimmed/20180129_trimmed_again/AD002_S9_L002_R2_001_val_2_val_2.fq.gz \\\n",
      "/home/sam/data/geoduck_illumina/trimmed/20180129_trimmed_again/NR005_S4_L001_R1_001_val_1_val_1.fq.gz \\\n",
      "/home/sam/data/geoduck_illumina/trimmed/20180129_trimmed_again/NR005_S4_L001_R2_001_val_2_val_2.fq.gz \\\n",
      "/home/sam/data/geoduck_illumina/trimmed/20180129_trimmed_again/NR005_S4_L002_R1_001_val_1_val_1.fq.gz \\\n",
      "/home/sam/data/geoduck_illumina/trimmed/20180129_trimmed_again/NR005_S4_L002_R2_001_val_2_val_2.fq.gz \\\n",
      "/home/sam/data/geoduck_illumina/trimmed/20180129_trimmed_again/NR006_S3_L001_R1_001_val_1_val_1.fq.gz \\\n",
      "/home/sam/data/geoduck_illumina/trimmed/20180129_trimmed_again/NR006_S3_L001_R2_001_val_2_val_2.fq.gz \\\n",
      "/home/sam/data/geoduck_illumina/trimmed/20180129_trimmed_again/NR006_S3_L002_R1_001_val_1_val_1.fq.gz \\\n",
      "/home/sam/data/geoduck_illumina/trimmed/20180129_trimmed_again/NR006_S3_L002_R2_001_val_2_val_2.fq.gz \\\n",
      "/home/sam/data/geoduck_illumina/trimmed/20180129_trimmed_again/NR012_S1_L001_R1_001_val_1_val_1.fq.gz \\\n",
      "/home/sam/data/geoduck_illumina/trimmed/20180129_trimmed_again/NR012_S1_L001_R2_001_val_2_val_2.fq.gz \\\n",
      "/home/sam/data/geoduck_illumina/trimmed/20180129_trimmed_again/NR012_S1_L002_R1_001_val_1_val_1.fq.gz \\\n",
      "/home/sam/data/geoduck_illumina/trimmed/20180129_trimmed_again/NR012_S1_L002_R2_001_val_2_val_2.fq.gz \\\n",
      "/home/sam/data/geoduck_illumina/trimmed/20180129_trimmed_again/NR013_AD013_S2_L001_R1_001_val_1_val_1.fq.gz \\\n",
      "/home/sam/data/geoduck_illumina/trimmed/20180129_trimmed_again/NR013_AD013_S2_L001_R2_001_val_2_val_2.fq.gz \\\n",
      "/home/sam/data/geoduck_illumina/trimmed/20180129_trimmed_again/NR013_AD013_S2_L002_R1_001_val_1_val_1.fq.gz \\\n",
      "/home/sam/data/geoduck_illumina/trimmed/20180129_trimmed_again/NR013_AD013_S2_L002_R2_001_val_2_val_2.fq.gz \\\n",
      "/home/sam/data/geoduck_illumina/trimmed/20180129_trimmed_again/NR014_AD014_S5_L001_R1_001_val_1_val_1.fq.gz \\\n",
      "/home/sam/data/geoduck_illumina/trimmed/20180129_trimmed_again/NR014_AD014_S5_L001_R2_001_val_2_val_2.fq.gz \\\n",
      "/home/sam/data/geoduck_illumina/trimmed/20180129_trimmed_again/NR014_AD014_S5_L002_R1_001_val_1_val_1.fq.gz \\\n",
      "/home/sam/data/geoduck_illumina/trimmed/20180129_trimmed_again/NR014_AD014_S5_L002_R2_001_val_2_val_2.fq.gz \\\n",
      "/home/sam/data/geoduck_illumina/trimmed/20180129_trimmed_again/NR015_AD015_S6_L001_R1_001_val_1_val_1.fq.gz \\\n",
      "/home/sam/data/geoduck_illumina/trimmed/20180129_trimmed_again/NR015_AD015_S6_L001_R2_001_val_2_val_2.fq.gz \\\n",
      "/home/sam/data/geoduck_illumina/trimmed/20180129_trimmed_again/NR015_AD015_S6_L002_R1_001_val_1_val_1.fq.gz \\\n",
      "/home/sam/data/geoduck_illumina/trimmed/20180129_trimmed_again/NR015_AD015_S6_L002_R2_001_val_2_val_2.fq.gz \\\n",
      "/home/sam/data/geoduck_illumina/trimmed/20180129_trimmed_again/NR019_S7_L001_R1_001_val_1_val_1.fq.gz \\\n",
      "/home/sam/data/geoduck_illumina/trimmed/20180129_trimmed_again/NR019_S7_L001_R2_001_val_2_val_2.fq.gz \\\n",
      "/home/sam/data/geoduck_illumina/trimmed/20180129_trimmed_again/NR019_S7_L002_R1_001_val_1_val_1.fq.gz \\\n",
      "/home/sam/data/geoduck_illumina/trimmed/20180129_trimmed_again/NR019_S7_L002_R2_001_val_2_val_2.fq.gz \\\n",
      "/home/sam/data/geoduck_illumina/trimmed/20180129_trimmed_again/NR021_S8_L001_R1_001_val_1_val_1.fq.gz \\\n",
      "/home/sam/data/geoduck_illumina/trimmed/20180129_trimmed_again/NR021_S8_L001_R2_001_val_2_val_2.fq.gz \\\n",
      "/home/sam/data/geoduck_illumina/trimmed/20180129_trimmed_again/NR021_S8_L002_R1_001_val_1_val_1.fq.gz \\\n",
      "/home/sam/data/geoduck_illumina/trimmed/20180129_trimmed_again/NR021_S8_L002_R2_001_val_2_val_2.fq.gz \\\n"
     ]
    }
   ],
   "source": [
    "%%bash\n",
    "for file in /home/sam/data/geoduck_illumina/trimmed/20180129_trimmed_again/*.gz\n",
    "do\n",
    "printf \"%s\\n\" \"$file \\\\\"\n",
    "done"
   ]
  },
  {
   "cell_type": "code",
   "execution_count": 51,
   "metadata": {},
   "outputs": [
    {
     "name": "stdout",
     "output_type": "stream",
     "text": [
      "Analysis complete for NR005_S4_L001_R1_001_val_1_val_1.fq.gz\n",
      "Analysis complete for NR005_S4_L001_R2_001_val_2_val_2.fq.gz\n",
      "Analysis complete for NR005_S4_L002_R1_001_val_1_val_1.fq.gz\n",
      "Analysis complete for NR005_S4_L002_R2_001_val_2_val_2.fq.gz\n",
      "Analysis complete for NR012_S1_L001_R1_001_val_1_val_1.fq.gz\n",
      "Analysis complete for NR012_S1_L001_R2_001_val_2_val_2.fq.gz\n",
      "Analysis complete for NR012_S1_L002_R1_001_val_1_val_1.fq.gz\n",
      "Analysis complete for NR012_S1_L002_R2_001_val_2_val_2.fq.gz\n",
      "Analysis complete for NR006_S3_L002_R2_001_val_2_val_2.fq.gz\n",
      "Analysis complete for NR006_S3_L002_R1_001_val_1_val_1.fq.gz\n",
      "Analysis complete for NR006_S3_L001_R1_001_val_1_val_1.fq.gz\n",
      "Analysis complete for NR006_S3_L001_R2_001_val_2_val_2.fq.gz\n",
      "Analysis complete for AD002_S9_L001_R1_001_val_1_val_1.fq.gz\n",
      "Analysis complete for AD002_S9_L001_R2_001_val_2_val_2.fq.gz\n",
      "Analysis complete for AD002_S9_L002_R1_001_val_1_val_1.fq.gz\n",
      "Analysis complete for AD002_S9_L002_R2_001_val_2_val_2.fq.gz\n",
      "Analysis complete for NR013_AD013_S2_L001_R1_001_val_1_val_1.fq.gz\n",
      "Analysis complete for NR019_S7_L001_R1_001_val_1_val_1.fq.gz\n",
      "Analysis complete for NR019_S7_L001_R2_001_val_2_val_2.fq.gz\n",
      "Analysis complete for NR019_S7_L002_R1_001_val_1_val_1.fq.gz\n",
      "Analysis complete for NR019_S7_L002_R2_001_val_2_val_2.fq.gz\n",
      "Analysis complete for NR021_S8_L001_R1_001_val_1_val_1.fq.gz\n",
      "Analysis complete for NR021_S8_L001_R2_001_val_2_val_2.fq.gz\n",
      "Analysis complete for NR021_S8_L002_R1_001_val_1_val_1.fq.gz\n",
      "Analysis complete for NR021_S8_L002_R2_001_val_2_val_2.fq.gz\n",
      "Analysis complete for NR013_AD013_S2_L001_R2_001_val_2_val_2.fq.gz\n",
      "Analysis complete for NR013_AD013_S2_L002_R1_001_val_1_val_1.fq.gz\n",
      "Analysis complete for NR013_AD013_S2_L002_R2_001_val_2_val_2.fq.gz\n",
      "Analysis complete for NR015_AD015_S6_L001_R1_001_val_1_val_1.fq.gz\n",
      "Analysis complete for NR015_AD015_S6_L001_R2_001_val_2_val_2.fq.gz\n",
      "Analysis complete for NR015_AD015_S6_L002_R1_001_val_1_val_1.fq.gz\n",
      "Analysis complete for NR015_AD015_S6_L002_R2_001_val_2_val_2.fq.gz\n",
      "Analysis complete for NR014_AD014_S5_L001_R1_001_val_1_val_1.fq.gz\n",
      "Analysis complete for NR014_AD014_S5_L001_R2_001_val_2_val_2.fq.gz\n",
      "Analysis complete for NR014_AD014_S5_L002_R1_001_val_1_val_1.fq.gz\n",
      "Analysis complete for NR014_AD014_S5_L002_R2_001_val_2_val_2.fq.gz\n"
     ]
    },
    {
     "name": "stderr",
     "output_type": "stream",
     "text": [
      "Started analysis of AD002_S9_L001_R1_001_val_1_val_1.fq.gz\n",
      "Started analysis of AD002_S9_L001_R2_001_val_2_val_2.fq.gz\n",
      "Started analysis of AD002_S9_L002_R1_001_val_1_val_1.fq.gz\n",
      "Started analysis of AD002_S9_L002_R2_001_val_2_val_2.fq.gz\n",
      "Started analysis of NR005_S4_L001_R1_001_val_1_val_1.fq.gz\n",
      "Approx 5% complete for NR005_S4_L001_R1_001_val_1_val_1.fq.gz\n",
      "Approx 10% complete for NR005_S4_L001_R1_001_val_1_val_1.fq.gz\n",
      "Approx 15% complete for NR005_S4_L001_R1_001_val_1_val_1.fq.gz\n",
      "Approx 20% complete for NR005_S4_L001_R1_001_val_1_val_1.fq.gz\n",
      "Approx 25% complete for NR005_S4_L001_R1_001_val_1_val_1.fq.gz\n",
      "Approx 30% complete for NR005_S4_L001_R1_001_val_1_val_1.fq.gz\n",
      "Approx 35% complete for NR005_S4_L001_R1_001_val_1_val_1.fq.gz\n",
      "Approx 40% complete for NR005_S4_L001_R1_001_val_1_val_1.fq.gz\n",
      "Approx 45% complete for NR005_S4_L001_R1_001_val_1_val_1.fq.gz\n",
      "Approx 50% complete for NR005_S4_L001_R1_001_val_1_val_1.fq.gz\n",
      "Started analysis of NR005_S4_L001_R2_001_val_2_val_2.fq.gz\n",
      "Approx 55% complete for NR005_S4_L001_R1_001_val_1_val_1.fq.gz\n",
      "Approx 5% complete for NR005_S4_L001_R2_001_val_2_val_2.fq.gz\n",
      "Approx 60% complete for NR005_S4_L001_R1_001_val_1_val_1.fq.gz\n",
      "Approx 10% complete for NR005_S4_L001_R2_001_val_2_val_2.fq.gz\n",
      "Approx 65% complete for NR005_S4_L001_R1_001_val_1_val_1.fq.gz\n",
      "Approx 15% complete for NR005_S4_L001_R2_001_val_2_val_2.fq.gz\n",
      "Approx 70% complete for NR005_S4_L001_R1_001_val_1_val_1.fq.gz\n",
      "Approx 20% complete for NR005_S4_L001_R2_001_val_2_val_2.fq.gz\n",
      "Approx 75% complete for NR005_S4_L001_R1_001_val_1_val_1.fq.gz\n",
      "Approx 25% complete for NR005_S4_L001_R2_001_val_2_val_2.fq.gz\n",
      "Approx 80% complete for NR005_S4_L001_R1_001_val_1_val_1.fq.gz\n",
      "Approx 30% complete for NR005_S4_L001_R2_001_val_2_val_2.fq.gz\n",
      "Approx 85% complete for NR005_S4_L001_R1_001_val_1_val_1.fq.gz\n",
      "Approx 35% complete for NR005_S4_L001_R2_001_val_2_val_2.fq.gz\n",
      "Approx 90% complete for NR005_S4_L001_R1_001_val_1_val_1.fq.gz\n",
      "Started analysis of NR005_S4_L002_R1_001_val_1_val_1.fq.gz\n",
      "Approx 40% complete for NR005_S4_L001_R2_001_val_2_val_2.fq.gz\n",
      "Approx 95% complete for NR005_S4_L001_R1_001_val_1_val_1.fq.gz\n",
      "Approx 5% complete for NR005_S4_L002_R1_001_val_1_val_1.fq.gz\n",
      "Approx 45% complete for NR005_S4_L001_R2_001_val_2_val_2.fq.gz\n",
      "Approx 10% complete for NR005_S4_L002_R1_001_val_1_val_1.fq.gz\n",
      "Approx 50% complete for NR005_S4_L001_R2_001_val_2_val_2.fq.gz\n",
      "Approx 15% complete for NR005_S4_L002_R1_001_val_1_val_1.fq.gz\n",
      "Approx 55% complete for NR005_S4_L001_R2_001_val_2_val_2.fq.gz\n",
      "Approx 20% complete for NR005_S4_L002_R1_001_val_1_val_1.fq.gz\n",
      "Approx 25% complete for NR005_S4_L002_R1_001_val_1_val_1.fq.gz\n",
      "Approx 60% complete for NR005_S4_L001_R2_001_val_2_val_2.fq.gz\n",
      "Approx 30% complete for NR005_S4_L002_R1_001_val_1_val_1.fq.gz\n",
      "Approx 65% complete for NR005_S4_L001_R2_001_val_2_val_2.fq.gz\n",
      "Approx 35% complete for NR005_S4_L002_R1_001_val_1_val_1.fq.gz\n",
      "Approx 70% complete for NR005_S4_L001_R2_001_val_2_val_2.fq.gz\n",
      "Started analysis of NR005_S4_L002_R2_001_val_2_val_2.fq.gz\n",
      "Approx 40% complete for NR005_S4_L002_R1_001_val_1_val_1.fq.gz\n",
      "Approx 75% complete for NR005_S4_L001_R2_001_val_2_val_2.fq.gz\n",
      "Approx 5% complete for NR005_S4_L002_R2_001_val_2_val_2.fq.gz\n",
      "Approx 45% complete for NR005_S4_L002_R1_001_val_1_val_1.fq.gz\n",
      "Approx 80% complete for NR005_S4_L001_R2_001_val_2_val_2.fq.gz\n",
      "Approx 50% complete for NR005_S4_L002_R1_001_val_1_val_1.fq.gz\n",
      "Approx 10% complete for NR005_S4_L002_R2_001_val_2_val_2.fq.gz\n",
      "Approx 85% complete for NR005_S4_L001_R2_001_val_2_val_2.fq.gz\n",
      "Approx 55% complete for NR005_S4_L002_R1_001_val_1_val_1.fq.gz\n",
      "Approx 15% complete for NR005_S4_L002_R2_001_val_2_val_2.fq.gz\n",
      "Approx 20% complete for NR005_S4_L002_R2_001_val_2_val_2.fq.gz\n",
      "Approx 60% complete for NR005_S4_L002_R1_001_val_1_val_1.fq.gz\n",
      "Approx 90% complete for NR005_S4_L001_R2_001_val_2_val_2.fq.gz\n",
      "Approx 25% complete for NR005_S4_L002_R2_001_val_2_val_2.fq.gz\n",
      "Approx 65% complete for NR005_S4_L002_R1_001_val_1_val_1.fq.gz\n",
      "Approx 95% complete for NR005_S4_L001_R2_001_val_2_val_2.fq.gz\n",
      "Approx 30% complete for NR005_S4_L002_R2_001_val_2_val_2.fq.gz\n",
      "Approx 70% complete for NR005_S4_L002_R1_001_val_1_val_1.fq.gz\n",
      "Started analysis of NR006_S3_L001_R1_001_val_1_val_1.fq.gz\n",
      "Approx 75% complete for NR005_S4_L002_R1_001_val_1_val_1.fq.gz\n",
      "Approx 35% complete for NR005_S4_L002_R2_001_val_2_val_2.fq.gz\n",
      "Approx 80% complete for NR005_S4_L002_R1_001_val_1_val_1.fq.gz\n",
      "Approx 40% complete for NR005_S4_L002_R2_001_val_2_val_2.fq.gz\n",
      "Approx 85% complete for NR005_S4_L002_R1_001_val_1_val_1.fq.gz\n",
      "Approx 45% complete for NR005_S4_L002_R2_001_val_2_val_2.fq.gz\n",
      "Approx 50% complete for NR005_S4_L002_R2_001_val_2_val_2.fq.gz\n",
      "Approx 90% complete for NR005_S4_L002_R1_001_val_1_val_1.fq.gz\n",
      "Approx 55% complete for NR005_S4_L002_R2_001_val_2_val_2.fq.gz\n",
      "Approx 95% complete for NR005_S4_L002_R1_001_val_1_val_1.fq.gz\n",
      "Started analysis of NR006_S3_L001_R2_001_val_2_val_2.fq.gz\n",
      "Approx 60% complete for NR005_S4_L002_R2_001_val_2_val_2.fq.gz\n",
      "Approx 65% complete for NR005_S4_L002_R2_001_val_2_val_2.fq.gz\n",
      "Approx 70% complete for NR005_S4_L002_R2_001_val_2_val_2.fq.gz\n",
      "Approx 75% complete for NR005_S4_L002_R2_001_val_2_val_2.fq.gz\n",
      "Approx 80% complete for NR005_S4_L002_R2_001_val_2_val_2.fq.gz\n",
      "Approx 85% complete for NR005_S4_L002_R2_001_val_2_val_2.fq.gz\n",
      "Started analysis of NR006_S3_L002_R1_001_val_1_val_1.fq.gz\n",
      "Approx 90% complete for NR005_S4_L002_R2_001_val_2_val_2.fq.gz\n",
      "Approx 95% complete for NR005_S4_L002_R2_001_val_2_val_2.fq.gz\n",
      "Started analysis of NR006_S3_L002_R2_001_val_2_val_2.fq.gz\n",
      "Approx 5% complete for NR006_S3_L001_R1_001_val_1_val_1.fq.gz\n",
      "Approx 5% complete for NR006_S3_L002_R1_001_val_1_val_1.fq.gz\n",
      "Approx 5% complete for NR006_S3_L002_R2_001_val_2_val_2.fq.gz\n",
      "Approx 5% complete for NR006_S3_L001_R2_001_val_2_val_2.fq.gz\n",
      "Approx 10% complete for NR006_S3_L002_R2_001_val_2_val_2.fq.gz\n",
      "Approx 10% complete for NR006_S3_L002_R1_001_val_1_val_1.fq.gz\n",
      "Approx 10% complete for NR006_S3_L001_R1_001_val_1_val_1.fq.gz\n",
      "Approx 10% complete for NR006_S3_L001_R2_001_val_2_val_2.fq.gz\n",
      "Started analysis of NR012_S1_L001_R1_001_val_1_val_1.fq.gz\n",
      "Approx 15% complete for NR006_S3_L002_R2_001_val_2_val_2.fq.gz\n",
      "Approx 5% complete for NR012_S1_L001_R1_001_val_1_val_1.fq.gz\n",
      "Started analysis of NR012_S1_L001_R2_001_val_2_val_2.fq.gz\n",
      "Approx 10% complete for NR012_S1_L001_R1_001_val_1_val_1.fq.gz\n",
      "Approx 15% complete for NR006_S3_L002_R1_001_val_1_val_1.fq.gz\n",
      "Approx 5% complete for NR012_S1_L001_R2_001_val_2_val_2.fq.gz\n",
      "Started analysis of NR012_S1_L002_R1_001_val_1_val_1.fq.gz\n",
      "Approx 15% complete for NR012_S1_L001_R1_001_val_1_val_1.fq.gz\n",
      "Approx 10% complete for NR012_S1_L001_R2_001_val_2_val_2.fq.gz\n",
      "Approx 20% complete for NR012_S1_L001_R1_001_val_1_val_1.fq.gz\n",
      "Started analysis of NR012_S1_L002_R2_001_val_2_val_2.fq.gz\n",
      "Approx 5% complete for NR012_S1_L002_R1_001_val_1_val_1.fq.gz\n",
      "Approx 15% complete for NR012_S1_L001_R2_001_val_2_val_2.fq.gz\n",
      "Approx 15% complete for NR006_S3_L001_R1_001_val_1_val_1.fq.gz\n",
      "Approx 25% complete for NR012_S1_L001_R1_001_val_1_val_1.fq.gz\n",
      "Approx 5% complete for NR012_S1_L002_R2_001_val_2_val_2.fq.gz\n",
      "Approx 20% complete for NR012_S1_L001_R2_001_val_2_val_2.fq.gz\n",
      "Approx 10% complete for NR012_S1_L002_R1_001_val_1_val_1.fq.gz\n",
      "Approx 15% complete for NR006_S3_L001_R2_001_val_2_val_2.fq.gz\n",
      "Approx 25% complete for NR012_S1_L001_R2_001_val_2_val_2.fq.gz\n",
      "Approx 30% complete for NR012_S1_L001_R1_001_val_1_val_1.fq.gz\n",
      "Approx 15% complete for NR012_S1_L002_R1_001_val_1_val_1.fq.gz\n",
      "Approx 10% complete for NR012_S1_L002_R2_001_val_2_val_2.fq.gz\n",
      "Approx 30% complete for NR012_S1_L001_R2_001_val_2_val_2.fq.gz\n",
      "Approx 35% complete for NR012_S1_L001_R1_001_val_1_val_1.fq.gz\n",
      "Approx 20% complete for NR012_S1_L002_R1_001_val_1_val_1.fq.gz\n",
      "Approx 15% complete for NR012_S1_L002_R2_001_val_2_val_2.fq.gz\n",
      "Approx 35% complete for NR012_S1_L001_R2_001_val_2_val_2.fq.gz\n",
      "Approx 40% complete for NR012_S1_L001_R1_001_val_1_val_1.fq.gz\n",
      "Approx 25% complete for NR012_S1_L002_R1_001_val_1_val_1.fq.gz\n",
      "Approx 20% complete for NR012_S1_L002_R2_001_val_2_val_2.fq.gz\n",
      "Approx 40% complete for NR012_S1_L001_R2_001_val_2_val_2.fq.gz\n",
      "Approx 45% complete for NR012_S1_L001_R1_001_val_1_val_1.fq.gz\n",
      "Approx 30% complete for NR012_S1_L002_R1_001_val_1_val_1.fq.gz\n",
      "Approx 45% complete for NR012_S1_L001_R2_001_val_2_val_2.fq.gz\n",
      "Approx 50% complete for NR012_S1_L001_R1_001_val_1_val_1.fq.gz\n",
      "Approx 25% complete for NR012_S1_L002_R2_001_val_2_val_2.fq.gz\n",
      "Approx 50% complete for NR012_S1_L001_R2_001_val_2_val_2.fq.gz\n",
      "Approx 35% complete for NR012_S1_L002_R1_001_val_1_val_1.fq.gz\n",
      "Approx 20% complete for NR006_S3_L002_R2_001_val_2_val_2.fq.gz\n",
      "Approx 55% complete for NR012_S1_L001_R1_001_val_1_val_1.fq.gz\n",
      "Approx 30% complete for NR012_S1_L002_R2_001_val_2_val_2.fq.gz\n",
      "Approx 55% complete for NR012_S1_L001_R2_001_val_2_val_2.fq.gz\n",
      "Approx 40% complete for NR012_S1_L002_R1_001_val_1_val_1.fq.gz\n",
      "Approx 60% complete for NR012_S1_L001_R1_001_val_1_val_1.fq.gz\n",
      "Approx 35% complete for NR012_S1_L002_R2_001_val_2_val_2.fq.gz\n",
      "Approx 60% complete for NR012_S1_L001_R2_001_val_2_val_2.fq.gz\n",
      "Approx 45% complete for NR012_S1_L002_R1_001_val_1_val_1.fq.gz\n",
      "Approx 20% complete for NR006_S3_L002_R1_001_val_1_val_1.fq.gz\n",
      "Approx 65% complete for NR012_S1_L001_R1_001_val_1_val_1.fq.gz\n",
      "Approx 40% complete for NR012_S1_L002_R2_001_val_2_val_2.fq.gz\n",
      "Approx 65% complete for NR012_S1_L001_R2_001_val_2_val_2.fq.gz\n",
      "Approx 50% complete for NR012_S1_L002_R1_001_val_1_val_1.fq.gz\n",
      "Approx 70% complete for NR012_S1_L001_R1_001_val_1_val_1.fq.gz\n",
      "Approx 70% complete for NR012_S1_L001_R2_001_val_2_val_2.fq.gz\n",
      "Approx 45% complete for NR012_S1_L002_R2_001_val_2_val_2.fq.gz\n",
      "Approx 55% complete for NR012_S1_L002_R1_001_val_1_val_1.fq.gz\n",
      "Approx 75% complete for NR012_S1_L001_R1_001_val_1_val_1.fq.gz\n",
      "Approx 75% complete for NR012_S1_L001_R2_001_val_2_val_2.fq.gz\n",
      "Approx 50% complete for NR012_S1_L002_R2_001_val_2_val_2.fq.gz\n",
      "Approx 60% complete for NR012_S1_L002_R1_001_val_1_val_1.fq.gz\n",
      "Approx 80% complete for NR012_S1_L001_R2_001_val_2_val_2.fq.gz\n",
      "Approx 80% complete for NR012_S1_L001_R1_001_val_1_val_1.fq.gz\n",
      "Approx 20% complete for NR006_S3_L001_R1_001_val_1_val_1.fq.gz\n",
      "Approx 55% complete for NR012_S1_L002_R2_001_val_2_val_2.fq.gz\n",
      "Approx 65% complete for NR012_S1_L002_R1_001_val_1_val_1.fq.gz\n",
      "Approx 85% complete for NR012_S1_L001_R1_001_val_1_val_1.fq.gz\n",
      "Approx 85% complete for NR012_S1_L001_R2_001_val_2_val_2.fq.gz\n",
      "Approx 60% complete for NR012_S1_L002_R2_001_val_2_val_2.fq.gz\n",
      "Approx 70% complete for NR012_S1_L002_R1_001_val_1_val_1.fq.gz\n",
      "Approx 90% complete for NR012_S1_L001_R1_001_val_1_val_1.fq.gz\n",
      "Approx 20% complete for NR006_S3_L001_R2_001_val_2_val_2.fq.gz\n",
      "Approx 90% complete for NR012_S1_L001_R2_001_val_2_val_2.fq.gz\n",
      "Approx 65% complete for NR012_S1_L002_R2_001_val_2_val_2.fq.gz\n",
      "Approx 95% complete for NR012_S1_L001_R1_001_val_1_val_1.fq.gz\n",
      "Approx 75% complete for NR012_S1_L002_R1_001_val_1_val_1.fq.gz\n",
      "Approx 95% complete for NR012_S1_L001_R2_001_val_2_val_2.fq.gz\n",
      "Approx 70% complete for NR012_S1_L002_R2_001_val_2_val_2.fq.gz\n",
      "Approx 80% complete for NR012_S1_L002_R1_001_val_1_val_1.fq.gz\n",
      "Approx 25% complete for NR006_S3_L002_R2_001_val_2_val_2.fq.gz\n",
      "Approx 85% complete for NR012_S1_L002_R1_001_val_1_val_1.fq.gz\n",
      "Approx 75% complete for NR012_S1_L002_R2_001_val_2_val_2.fq.gz\n",
      "Approx 90% complete for NR012_S1_L002_R1_001_val_1_val_1.fq.gz\n",
      "Approx 80% complete for NR012_S1_L002_R2_001_val_2_val_2.fq.gz\n",
      "Approx 25% complete for NR006_S3_L002_R1_001_val_1_val_1.fq.gz\n",
      "Approx 95% complete for NR012_S1_L002_R1_001_val_1_val_1.fq.gz\n",
      "Approx 85% complete for NR012_S1_L002_R2_001_val_2_val_2.fq.gz\n",
      "Approx 90% complete for NR012_S1_L002_R2_001_val_2_val_2.fq.gz\n",
      "Approx 95% complete for NR012_S1_L002_R2_001_val_2_val_2.fq.gz\n",
      "Approx 25% complete for NR006_S3_L001_R1_001_val_1_val_1.fq.gz\n",
      "Started analysis of NR013_AD013_S2_L001_R1_001_val_1_val_1.fq.gz\n",
      "Approx 25% complete for NR006_S3_L001_R2_001_val_2_val_2.fq.gz\n",
      "Approx 30% complete for NR006_S3_L002_R2_001_val_2_val_2.fq.gz\n",
      "Started analysis of NR013_AD013_S2_L001_R2_001_val_2_val_2.fq.gz\n",
      "Approx 30% complete for NR006_S3_L002_R1_001_val_1_val_1.fq.gz\n",
      "Started analysis of NR013_AD013_S2_L002_R1_001_val_1_val_1.fq.gz\n",
      "Started analysis of NR013_AD013_S2_L002_R2_001_val_2_val_2.fq.gz\n",
      "Approx 30% complete for NR006_S3_L001_R1_001_val_1_val_1.fq.gz\n",
      "Approx 35% complete for NR006_S3_L002_R2_001_val_2_val_2.fq.gz\n",
      "Approx 30% complete for NR006_S3_L001_R2_001_val_2_val_2.fq.gz\n",
      "Approx 35% complete for NR006_S3_L002_R1_001_val_1_val_1.fq.gz\n",
      "Approx 40% complete for NR006_S3_L002_R1_001_val_1_val_1.fq.gz\n",
      "Approx 40% complete for NR006_S3_L002_R2_001_val_2_val_2.fq.gz\n",
      "Approx 35% complete for NR006_S3_L001_R1_001_val_1_val_1.fq.gz\n",
      "Approx 35% complete for NR006_S3_L001_R2_001_val_2_val_2.fq.gz\n",
      "Approx 45% complete for NR006_S3_L002_R1_001_val_1_val_1.fq.gz\n",
      "Approx 45% complete for NR006_S3_L002_R2_001_val_2_val_2.fq.gz\n",
      "Approx 40% complete for NR006_S3_L001_R1_001_val_1_val_1.fq.gz\n",
      "Approx 40% complete for NR006_S3_L001_R2_001_val_2_val_2.fq.gz\n",
      "Approx 50% complete for NR006_S3_L002_R2_001_val_2_val_2.fq.gz\n",
      "Approx 50% complete for NR006_S3_L002_R1_001_val_1_val_1.fq.gz\n",
      "Approx 45% complete for NR006_S3_L001_R1_001_val_1_val_1.fq.gz\n",
      "Approx 45% complete for NR006_S3_L001_R2_001_val_2_val_2.fq.gz\n",
      "Approx 55% complete for NR006_S3_L002_R1_001_val_1_val_1.fq.gz\n",
      "Approx 55% complete for NR006_S3_L002_R2_001_val_2_val_2.fq.gz\n",
      "Approx 50% complete for NR006_S3_L001_R1_001_val_1_val_1.fq.gz\n",
      "Approx 60% complete for NR006_S3_L002_R1_001_val_1_val_1.fq.gz\n",
      "Approx 50% complete for NR006_S3_L001_R2_001_val_2_val_2.fq.gz\n",
      "Approx 60% complete for NR006_S3_L002_R2_001_val_2_val_2.fq.gz\n",
      "Approx 55% complete for NR006_S3_L001_R1_001_val_1_val_1.fq.gz\n",
      "Approx 65% complete for NR006_S3_L002_R1_001_val_1_val_1.fq.gz\n",
      "Approx 65% complete for NR006_S3_L002_R2_001_val_2_val_2.fq.gz\n",
      "Approx 55% complete for NR006_S3_L001_R2_001_val_2_val_2.fq.gz\n",
      "Approx 70% complete for NR006_S3_L002_R1_001_val_1_val_1.fq.gz\n",
      "Approx 70% complete for NR006_S3_L002_R2_001_val_2_val_2.fq.gz\n",
      "Approx 60% complete for NR006_S3_L001_R1_001_val_1_val_1.fq.gz\n",
      "Approx 60% complete for NR006_S3_L001_R2_001_val_2_val_2.fq.gz\n",
      "Approx 75% complete for NR006_S3_L002_R1_001_val_1_val_1.fq.gz\n",
      "Approx 75% complete for NR006_S3_L002_R2_001_val_2_val_2.fq.gz\n",
      "Approx 65% complete for NR006_S3_L001_R1_001_val_1_val_1.fq.gz\n",
      "Approx 65% complete for NR006_S3_L001_R2_001_val_2_val_2.fq.gz\n",
      "Approx 80% complete for NR006_S3_L002_R1_001_val_1_val_1.fq.gz\n",
      "Approx 80% complete for NR006_S3_L002_R2_001_val_2_val_2.fq.gz\n",
      "Approx 70% complete for NR006_S3_L001_R1_001_val_1_val_1.fq.gz\n",
      "Approx 70% complete for NR006_S3_L001_R2_001_val_2_val_2.fq.gz\n",
      "Approx 85% complete for NR006_S3_L002_R2_001_val_2_val_2.fq.gz\n",
      "Approx 85% complete for NR006_S3_L002_R1_001_val_1_val_1.fq.gz\n",
      "Approx 75% complete for NR006_S3_L001_R1_001_val_1_val_1.fq.gz\n",
      "Approx 90% complete for NR006_S3_L002_R2_001_val_2_val_2.fq.gz\n",
      "Approx 75% complete for NR006_S3_L001_R2_001_val_2_val_2.fq.gz\n",
      "Approx 90% complete for NR006_S3_L002_R1_001_val_1_val_1.fq.gz\n",
      "Approx 80% complete for NR006_S3_L001_R1_001_val_1_val_1.fq.gz\n",
      "Approx 95% complete for NR006_S3_L002_R2_001_val_2_val_2.fq.gz\n",
      "Approx 95% complete for NR006_S3_L002_R1_001_val_1_val_1.fq.gz\n",
      "Approx 80% complete for NR006_S3_L001_R2_001_val_2_val_2.fq.gz\n",
      "Approx 85% complete for NR006_S3_L001_R1_001_val_1_val_1.fq.gz\n",
      "Approx 85% complete for NR006_S3_L001_R2_001_val_2_val_2.fq.gz\n",
      "Started analysis of NR014_AD014_S5_L001_R1_001_val_1_val_1.fq.gz\n",
      "Started analysis of NR014_AD014_S5_L001_R2_001_val_2_val_2.fq.gz\n",
      "Approx 90% complete for NR006_S3_L001_R1_001_val_1_val_1.fq.gz\n",
      "Approx 90% complete for NR006_S3_L001_R2_001_val_2_val_2.fq.gz\n",
      "Approx 95% complete for NR006_S3_L001_R1_001_val_1_val_1.fq.gz\n",
      "Approx 95% complete for NR006_S3_L001_R2_001_val_2_val_2.fq.gz\n",
      "Started analysis of NR014_AD014_S5_L002_R1_001_val_1_val_1.fq.gz\n",
      "Started analysis of NR014_AD014_S5_L002_R2_001_val_2_val_2.fq.gz\n",
      "Approx 5% complete for AD002_S9_L001_R1_001_val_1_val_1.fq.gz\n",
      "Approx 5% complete for AD002_S9_L001_R2_001_val_2_val_2.fq.gz\n",
      "Approx 5% complete for AD002_S9_L002_R1_001_val_1_val_1.fq.gz\n",
      "Approx 5% complete for AD002_S9_L002_R2_001_val_2_val_2.fq.gz\n",
      "Approx 5% complete for NR013_AD013_S2_L001_R1_001_val_1_val_1.fq.gz\n",
      "Approx 10% complete for AD002_S9_L001_R1_001_val_1_val_1.fq.gz\n",
      "Approx 10% complete for AD002_S9_L001_R2_001_val_2_val_2.fq.gz\n",
      "Approx 5% complete for NR013_AD013_S2_L001_R2_001_val_2_val_2.fq.gz\n",
      "Approx 5% complete for NR013_AD013_S2_L002_R1_001_val_1_val_1.fq.gz\n",
      "Approx 10% complete for AD002_S9_L002_R1_001_val_1_val_1.fq.gz\n",
      "Approx 5% complete for NR013_AD013_S2_L002_R2_001_val_2_val_2.fq.gz\n",
      "Approx 10% complete for AD002_S9_L002_R2_001_val_2_val_2.fq.gz\n",
      "Approx 15% complete for AD002_S9_L001_R1_001_val_1_val_1.fq.gz\n",
      "Approx 15% complete for AD002_S9_L001_R2_001_val_2_val_2.fq.gz\n",
      "Approx 15% complete for AD002_S9_L002_R1_001_val_1_val_1.fq.gz\n",
      "Approx 15% complete for AD002_S9_L002_R2_001_val_2_val_2.fq.gz\n",
      "Approx 10% complete for NR013_AD013_S2_L001_R1_001_val_1_val_1.fq.gz\n",
      "Approx 20% complete for AD002_S9_L001_R1_001_val_1_val_1.fq.gz\n",
      "Approx 10% complete for NR013_AD013_S2_L001_R2_001_val_2_val_2.fq.gz\n",
      "Approx 20% complete for AD002_S9_L001_R2_001_val_2_val_2.fq.gz\n",
      "Approx 10% complete for NR013_AD013_S2_L002_R1_001_val_1_val_1.fq.gz\n",
      "Approx 10% complete for NR013_AD013_S2_L002_R2_001_val_2_val_2.fq.gz\n",
      "Approx 20% complete for AD002_S9_L002_R1_001_val_1_val_1.fq.gz\n",
      "Approx 5% complete for NR014_AD014_S5_L001_R1_001_val_1_val_1.fq.gz\n",
      "Approx 20% complete for AD002_S9_L002_R2_001_val_2_val_2.fq.gz\n",
      "Approx 25% complete for AD002_S9_L001_R1_001_val_1_val_1.fq.gz\n",
      "Approx 5% complete for NR014_AD014_S5_L001_R2_001_val_2_val_2.fq.gz\n",
      "Approx 25% complete for AD002_S9_L001_R2_001_val_2_val_2.fq.gz\n",
      "Approx 15% complete for NR013_AD013_S2_L001_R1_001_val_1_val_1.fq.gz\n",
      "Approx 25% complete for AD002_S9_L002_R1_001_val_1_val_1.fq.gz\n",
      "Approx 15% complete for NR013_AD013_S2_L001_R2_001_val_2_val_2.fq.gz\n",
      "Approx 30% complete for AD002_S9_L001_R1_001_val_1_val_1.fq.gz\n",
      "Approx 25% complete for AD002_S9_L002_R2_001_val_2_val_2.fq.gz\n",
      "Approx 30% complete for AD002_S9_L001_R2_001_val_2_val_2.fq.gz\n",
      "Approx 15% complete for NR013_AD013_S2_L002_R1_001_val_1_val_1.fq.gz\n",
      "Approx 15% complete for NR013_AD013_S2_L002_R2_001_val_2_val_2.fq.gz\n",
      "Approx 35% complete for AD002_S9_L001_R1_001_val_1_val_1.fq.gz\n",
      "Approx 30% complete for AD002_S9_L002_R1_001_val_1_val_1.fq.gz\n",
      "Approx 35% complete for AD002_S9_L001_R2_001_val_2_val_2.fq.gz\n",
      "Approx 20% complete for NR013_AD013_S2_L001_R1_001_val_1_val_1.fq.gz\n",
      "Approx 30% complete for AD002_S9_L002_R2_001_val_2_val_2.fq.gz\n",
      "Approx 20% complete for NR013_AD013_S2_L001_R2_001_val_2_val_2.fq.gz\n",
      "Approx 40% complete for AD002_S9_L001_R1_001_val_1_val_1.fq.gz\n",
      "Approx 35% complete for AD002_S9_L002_R1_001_val_1_val_1.fq.gz\n",
      "Approx 40% complete for AD002_S9_L001_R2_001_val_2_val_2.fq.gz\n",
      "Approx 20% complete for NR013_AD013_S2_L002_R1_001_val_1_val_1.fq.gz\n",
      "Approx 35% complete for AD002_S9_L002_R2_001_val_2_val_2.fq.gz\n",
      "Approx 45% complete for AD002_S9_L001_R1_001_val_1_val_1.fq.gz\n",
      "Approx 20% complete for NR013_AD013_S2_L002_R2_001_val_2_val_2.fq.gz\n",
      "Approx 25% complete for NR013_AD013_S2_L001_R1_001_val_1_val_1.fq.gz\n",
      "Approx 10% complete for NR014_AD014_S5_L001_R1_001_val_1_val_1.fq.gz\n",
      "Approx 40% complete for AD002_S9_L002_R1_001_val_1_val_1.fq.gz\n",
      "Approx 45% complete for AD002_S9_L001_R2_001_val_2_val_2.fq.gz\n",
      "Approx 10% complete for NR014_AD014_S5_L001_R2_001_val_2_val_2.fq.gz\n",
      "Approx 25% complete for NR013_AD013_S2_L001_R2_001_val_2_val_2.fq.gz\n",
      "Approx 40% complete for AD002_S9_L002_R2_001_val_2_val_2.fq.gz\n",
      "Approx 50% complete for AD002_S9_L001_R1_001_val_1_val_1.fq.gz\n",
      "Approx 50% complete for AD002_S9_L001_R2_001_val_2_val_2.fq.gz\n",
      "Approx 45% complete for AD002_S9_L002_R1_001_val_1_val_1.fq.gz\n",
      "Approx 25% complete for NR013_AD013_S2_L002_R1_001_val_1_val_1.fq.gz\n",
      "Approx 30% complete for NR013_AD013_S2_L001_R1_001_val_1_val_1.fq.gz\n",
      "Approx 55% complete for AD002_S9_L001_R1_001_val_1_val_1.fq.gz\n",
      "Approx 45% complete for AD002_S9_L002_R2_001_val_2_val_2.fq.gz\n",
      "Approx 25% complete for NR013_AD013_S2_L002_R2_001_val_2_val_2.fq.gz\n",
      "Approx 55% complete for AD002_S9_L001_R2_001_val_2_val_2.fq.gz\n",
      "Approx 30% complete for NR013_AD013_S2_L001_R2_001_val_2_val_2.fq.gz\n",
      "Approx 50% complete for AD002_S9_L002_R1_001_val_1_val_1.fq.gz\n",
      "Approx 60% complete for AD002_S9_L001_R1_001_val_1_val_1.fq.gz\n",
      "Approx 50% complete for AD002_S9_L002_R2_001_val_2_val_2.fq.gz\n",
      "Approx 60% complete for AD002_S9_L001_R2_001_val_2_val_2.fq.gz\n",
      "Approx 35% complete for NR013_AD013_S2_L001_R1_001_val_1_val_1.fq.gz\n",
      "Approx 30% complete for NR013_AD013_S2_L002_R1_001_val_1_val_1.fq.gz\n",
      "Approx 55% complete for AD002_S9_L002_R1_001_val_1_val_1.fq.gz\n",
      "Approx 65% complete for AD002_S9_L001_R1_001_val_1_val_1.fq.gz\n",
      "Approx 55% complete for AD002_S9_L002_R2_001_val_2_val_2.fq.gz\n",
      "Approx 30% complete for NR013_AD013_S2_L002_R2_001_val_2_val_2.fq.gz\n",
      "Approx 15% complete for NR014_AD014_S5_L001_R1_001_val_1_val_1.fq.gz\n",
      "Approx 35% complete for NR013_AD013_S2_L001_R2_001_val_2_val_2.fq.gz\n",
      "Approx 65% complete for AD002_S9_L001_R2_001_val_2_val_2.fq.gz\n",
      "Approx 70% complete for AD002_S9_L001_R1_001_val_1_val_1.fq.gz\n",
      "Approx 60% complete for AD002_S9_L002_R1_001_val_1_val_1.fq.gz\n",
      "Approx 15% complete for NR014_AD014_S5_L001_R2_001_val_2_val_2.fq.gz\n",
      "Approx 40% complete for NR013_AD013_S2_L001_R1_001_val_1_val_1.fq.gz\n",
      "Approx 70% complete for AD002_S9_L001_R2_001_val_2_val_2.fq.gz\n",
      "Approx 60% complete for AD002_S9_L002_R2_001_val_2_val_2.fq.gz\n",
      "Approx 35% complete for NR013_AD013_S2_L002_R1_001_val_1_val_1.fq.gz\n",
      "Approx 75% complete for AD002_S9_L001_R1_001_val_1_val_1.fq.gz\n",
      "Approx 65% complete for AD002_S9_L002_R1_001_val_1_val_1.fq.gz\n",
      "Approx 40% complete for NR013_AD013_S2_L001_R2_001_val_2_val_2.fq.gz\n",
      "Approx 75% complete for AD002_S9_L001_R2_001_val_2_val_2.fq.gz\n",
      "Approx 35% complete for NR013_AD013_S2_L002_R2_001_val_2_val_2.fq.gz\n",
      "Approx 65% complete for AD002_S9_L002_R2_001_val_2_val_2.fq.gz\n",
      "Approx 80% complete for AD002_S9_L001_R1_001_val_1_val_1.fq.gz\n",
      "Approx 45% complete for NR013_AD013_S2_L001_R1_001_val_1_val_1.fq.gz\n",
      "Approx 70% complete for AD002_S9_L002_R1_001_val_1_val_1.fq.gz\n",
      "Approx 80% complete for AD002_S9_L001_R2_001_val_2_val_2.fq.gz\n",
      "Approx 85% complete for AD002_S9_L001_R1_001_val_1_val_1.fq.gz\n",
      "Approx 40% complete for NR013_AD013_S2_L002_R1_001_val_1_val_1.fq.gz\n",
      "Approx 70% complete for AD002_S9_L002_R2_001_val_2_val_2.fq.gz\n",
      "Approx 45% complete for NR013_AD013_S2_L001_R2_001_val_2_val_2.fq.gz\n",
      "Approx 75% complete for AD002_S9_L002_R1_001_val_1_val_1.fq.gz\n",
      "Approx 85% complete for AD002_S9_L001_R2_001_val_2_val_2.fq.gz\n",
      "Approx 90% complete for AD002_S9_L001_R1_001_val_1_val_1.fq.gz\n",
      "Approx 20% complete for NR014_AD014_S5_L001_R1_001_val_1_val_1.fq.gz\n",
      "Approx 40% complete for NR013_AD013_S2_L002_R2_001_val_2_val_2.fq.gz\n",
      "Approx 50% complete for NR013_AD013_S2_L001_R1_001_val_1_val_1.fq.gz\n",
      "Approx 75% complete for AD002_S9_L002_R2_001_val_2_val_2.fq.gz\n",
      "Approx 80% complete for AD002_S9_L002_R1_001_val_1_val_1.fq.gz\n",
      "Approx 90% complete for AD002_S9_L001_R2_001_val_2_val_2.fq.gz\n",
      "Approx 95% complete for AD002_S9_L001_R1_001_val_1_val_1.fq.gz\n",
      "Approx 20% complete for NR014_AD014_S5_L001_R2_001_val_2_val_2.fq.gz\n",
      "Approx 45% complete for NR013_AD013_S2_L002_R1_001_val_1_val_1.fq.gz\n",
      "Approx 50% complete for NR013_AD013_S2_L001_R2_001_val_2_val_2.fq.gz\n",
      "Approx 80% complete for AD002_S9_L002_R2_001_val_2_val_2.fq.gz\n",
      "Approx 95% complete for AD002_S9_L001_R2_001_val_2_val_2.fq.gz\n",
      "Started analysis of NR015_AD015_S6_L001_R1_001_val_1_val_1.fq.gz\n",
      "Approx 85% complete for AD002_S9_L002_R1_001_val_1_val_1.fq.gz\n",
      "Approx 55% complete for NR013_AD013_S2_L001_R1_001_val_1_val_1.fq.gz\n",
      "Approx 45% complete for NR013_AD013_S2_L002_R2_001_val_2_val_2.fq.gz\n",
      "Started analysis of NR015_AD015_S6_L001_R2_001_val_2_val_2.fq.gz\n",
      "Approx 85% complete for AD002_S9_L002_R2_001_val_2_val_2.fq.gz\n",
      "Approx 90% complete for AD002_S9_L002_R1_001_val_1_val_1.fq.gz\n",
      "Approx 50% complete for NR013_AD013_S2_L002_R1_001_val_1_val_1.fq.gz\n",
      "Approx 55% complete for NR013_AD013_S2_L001_R2_001_val_2_val_2.fq.gz\n",
      "Approx 60% complete for NR013_AD013_S2_L001_R1_001_val_1_val_1.fq.gz\n",
      "Approx 5% complete for NR015_AD015_S6_L001_R1_001_val_1_val_1.fq.gz\n",
      "Approx 90% complete for AD002_S9_L002_R2_001_val_2_val_2.fq.gz\n",
      "Approx 95% complete for AD002_S9_L002_R1_001_val_1_val_1.fq.gz\n",
      "Approx 25% complete for NR014_AD014_S5_L001_R1_001_val_1_val_1.fq.gz\n",
      "Approx 50% complete for NR013_AD013_S2_L002_R2_001_val_2_val_2.fq.gz\n",
      "Approx 5% complete for NR015_AD015_S6_L001_R2_001_val_2_val_2.fq.gz\n",
      "Approx 60% complete for NR013_AD013_S2_L001_R2_001_val_2_val_2.fq.gz\n",
      "Approx 55% complete for NR013_AD013_S2_L002_R1_001_val_1_val_1.fq.gz\n",
      "Approx 95% complete for AD002_S9_L002_R2_001_val_2_val_2.fq.gz\n",
      "Started analysis of NR015_AD015_S6_L002_R1_001_val_1_val_1.fq.gz\n",
      "Approx 25% complete for NR014_AD014_S5_L001_R2_001_val_2_val_2.fq.gz\n",
      "Approx 10% complete for NR015_AD015_S6_L001_R1_001_val_1_val_1.fq.gz\n",
      "Approx 65% complete for NR013_AD013_S2_L001_R1_001_val_1_val_1.fq.gz\n",
      "Approx 10% complete for NR015_AD015_S6_L001_R2_001_val_2_val_2.fq.gz\n",
      "Started analysis of NR015_AD015_S6_L002_R2_001_val_2_val_2.fq.gz\n",
      "Approx 55% complete for NR013_AD013_S2_L002_R2_001_val_2_val_2.fq.gz\n",
      "Approx 5% complete for NR015_AD015_S6_L002_R1_001_val_1_val_1.fq.gz\n",
      "Approx 15% complete for NR015_AD015_S6_L001_R1_001_val_1_val_1.fq.gz\n",
      "Approx 65% complete for NR013_AD013_S2_L001_R2_001_val_2_val_2.fq.gz\n",
      "Approx 70% complete for NR013_AD013_S2_L001_R1_001_val_1_val_1.fq.gz\n",
      "Approx 60% complete for NR013_AD013_S2_L002_R1_001_val_1_val_1.fq.gz\n",
      "Approx 15% complete for NR015_AD015_S6_L001_R2_001_val_2_val_2.fq.gz\n",
      "Approx 5% complete for NR015_AD015_S6_L002_R2_001_val_2_val_2.fq.gz\n",
      "Approx 20% complete for NR015_AD015_S6_L001_R1_001_val_1_val_1.fq.gz\n",
      "Approx 30% complete for NR014_AD014_S5_L001_R1_001_val_1_val_1.fq.gz\n",
      "Approx 10% complete for NR015_AD015_S6_L002_R1_001_val_1_val_1.fq.gz\n",
      "Approx 60% complete for NR013_AD013_S2_L002_R2_001_val_2_val_2.fq.gz\n",
      "Approx 70% complete for NR013_AD013_S2_L001_R2_001_val_2_val_2.fq.gz\n",
      "Approx 75% complete for NR013_AD013_S2_L001_R1_001_val_1_val_1.fq.gz\n",
      "Approx 20% complete for NR015_AD015_S6_L001_R2_001_val_2_val_2.fq.gz\n",
      "Approx 65% complete for NR013_AD013_S2_L002_R1_001_val_1_val_1.fq.gz\n",
      "Approx 30% complete for NR014_AD014_S5_L001_R2_001_val_2_val_2.fq.gz\n",
      "Approx 25% complete for NR015_AD015_S6_L001_R1_001_val_1_val_1.fq.gz\n",
      "Approx 10% complete for NR015_AD015_S6_L002_R2_001_val_2_val_2.fq.gz\n",
      "Approx 15% complete for NR015_AD015_S6_L002_R1_001_val_1_val_1.fq.gz\n",
      "Approx 75% complete for NR013_AD013_S2_L001_R2_001_val_2_val_2.fq.gz\n",
      "Approx 80% complete for NR013_AD013_S2_L001_R1_001_val_1_val_1.fq.gz\n",
      "Approx 25% complete for NR015_AD015_S6_L001_R2_001_val_2_val_2.fq.gz\n",
      "Approx 65% complete for NR013_AD013_S2_L002_R2_001_val_2_val_2.fq.gz\n",
      "Approx 30% complete for NR015_AD015_S6_L001_R1_001_val_1_val_1.fq.gz\n",
      "Approx 15% complete for NR015_AD015_S6_L002_R2_001_val_2_val_2.fq.gz\n",
      "Approx 20% complete for NR015_AD015_S6_L002_R1_001_val_1_val_1.fq.gz\n",
      "Approx 70% complete for NR013_AD013_S2_L002_R1_001_val_1_val_1.fq.gz\n",
      "Approx 5% complete for NR014_AD014_S5_L002_R1_001_val_1_val_1.fq.gz\n",
      "Approx 30% complete for NR015_AD015_S6_L001_R2_001_val_2_val_2.fq.gz\n",
      "Approx 85% complete for NR013_AD013_S2_L001_R1_001_val_1_val_1.fq.gz\n",
      "Approx 35% complete for NR014_AD014_S5_L001_R1_001_val_1_val_1.fq.gz\n",
      "Approx 80% complete for NR013_AD013_S2_L001_R2_001_val_2_val_2.fq.gz\n",
      "Approx 35% complete for NR015_AD015_S6_L001_R1_001_val_1_val_1.fq.gz\n",
      "Approx 20% complete for NR015_AD015_S6_L002_R2_001_val_2_val_2.fq.gz\n",
      "Approx 25% complete for NR015_AD015_S6_L002_R1_001_val_1_val_1.fq.gz\n",
      "Approx 5% complete for NR014_AD014_S5_L002_R2_001_val_2_val_2.fq.gz\n",
      "Approx 70% complete for NR013_AD013_S2_L002_R2_001_val_2_val_2.fq.gz\n",
      "Approx 75% complete for NR013_AD013_S2_L002_R1_001_val_1_val_1.fq.gz\n",
      "Approx 35% complete for NR015_AD015_S6_L001_R2_001_val_2_val_2.fq.gz\n",
      "Approx 40% complete for NR015_AD015_S6_L001_R1_001_val_1_val_1.fq.gz\n",
      "Approx 35% complete for NR014_AD014_S5_L001_R2_001_val_2_val_2.fq.gz\n",
      "Approx 90% complete for NR013_AD013_S2_L001_R1_001_val_1_val_1.fq.gz\n",
      "Approx 85% complete for NR013_AD013_S2_L001_R2_001_val_2_val_2.fq.gz\n",
      "Approx 30% complete for NR015_AD015_S6_L002_R1_001_val_1_val_1.fq.gz\n",
      "Approx 25% complete for NR015_AD015_S6_L002_R2_001_val_2_val_2.fq.gz\n",
      "Approx 45% complete for NR015_AD015_S6_L001_R1_001_val_1_val_1.fq.gz\n",
      "Approx 40% complete for NR015_AD015_S6_L001_R2_001_val_2_val_2.fq.gz\n",
      "Approx 75% complete for NR013_AD013_S2_L002_R2_001_val_2_val_2.fq.gz\n",
      "Approx 80% complete for NR013_AD013_S2_L002_R1_001_val_1_val_1.fq.gz\n",
      "Approx 95% complete for NR013_AD013_S2_L001_R1_001_val_1_val_1.fq.gz\n",
      "Approx 35% complete for NR015_AD015_S6_L002_R1_001_val_1_val_1.fq.gz\n",
      "Approx 90% complete for NR013_AD013_S2_L001_R2_001_val_2_val_2.fq.gz\n",
      "Approx 30% complete for NR015_AD015_S6_L002_R2_001_val_2_val_2.fq.gz\n",
      "Approx 40% complete for NR014_AD014_S5_L001_R1_001_val_1_val_1.fq.gz\n",
      "Approx 50% complete for NR015_AD015_S6_L001_R1_001_val_1_val_1.fq.gz\n",
      "Approx 45% complete for NR015_AD015_S6_L001_R2_001_val_2_val_2.fq.gz\n",
      "Approx 40% complete for NR015_AD015_S6_L002_R1_001_val_1_val_1.fq.gz\n",
      "Approx 80% complete for NR013_AD013_S2_L002_R2_001_val_2_val_2.fq.gz\n",
      "Started analysis of NR019_S7_L001_R1_001_val_1_val_1.fq.gz\n",
      "Approx 5% complete for NR019_S7_L001_R1_001_val_1_val_1.fq.gz\n",
      "Approx 10% complete for NR019_S7_L001_R1_001_val_1_val_1.fq.gz\n",
      "Approx 15% complete for NR019_S7_L001_R1_001_val_1_val_1.fq.gz\n",
      "Approx 20% complete for NR019_S7_L001_R1_001_val_1_val_1.fq.gz\n",
      "Approx 25% complete for NR019_S7_L001_R1_001_val_1_val_1.fq.gz\n",
      "Approx 30% complete for NR019_S7_L001_R1_001_val_1_val_1.fq.gz\n",
      "Approx 35% complete for NR019_S7_L001_R1_001_val_1_val_1.fq.gz\n",
      "Approx 40% complete for NR019_S7_L001_R1_001_val_1_val_1.fq.gz\n",
      "Approx 45% complete for NR019_S7_L001_R1_001_val_1_val_1.fq.gz\n",
      "Approx 50% complete for NR019_S7_L001_R1_001_val_1_val_1.fq.gz\n",
      "Approx 55% complete for NR019_S7_L001_R1_001_val_1_val_1.fq.gz\n",
      "Approx 60% complete for NR019_S7_L001_R1_001_val_1_val_1.fq.gz\n",
      "Approx 65% complete for NR019_S7_L001_R1_001_val_1_val_1.fq.gz\n",
      "Approx 70% complete for NR019_S7_L001_R1_001_val_1_val_1.fq.gz\n",
      "Approx 75% complete for NR019_S7_L001_R1_001_val_1_val_1.fq.gz\n",
      "Approx 80% complete for NR019_S7_L001_R1_001_val_1_val_1.fq.gz\n",
      "Approx 85% complete for NR019_S7_L001_R1_001_val_1_val_1.fq.gz\n",
      "Approx 90% complete for NR019_S7_L001_R1_001_val_1_val_1.fq.gz\n",
      "Approx 95% complete for NR019_S7_L001_R1_001_val_1_val_1.fq.gz\n",
      "Started analysis of NR019_S7_L001_R2_001_val_2_val_2.fq.gz\n",
      "Approx 5% complete for NR019_S7_L001_R2_001_val_2_val_2.fq.gz\n",
      "Approx 10% complete for NR019_S7_L001_R2_001_val_2_val_2.fq.gz\n",
      "Approx 15% complete for NR019_S7_L001_R2_001_val_2_val_2.fq.gz\n",
      "Approx 20% complete for NR019_S7_L001_R2_001_val_2_val_2.fq.gz\n",
      "Approx 25% complete for NR019_S7_L001_R2_001_val_2_val_2.fq.gz\n",
      "Approx 30% complete for NR019_S7_L001_R2_001_val_2_val_2.fq.gz\n",
      "Approx 35% complete for NR019_S7_L001_R2_001_val_2_val_2.fq.gz\n",
      "Approx 40% complete for NR019_S7_L001_R2_001_val_2_val_2.fq.gz\n",
      "Approx 45% complete for NR019_S7_L001_R2_001_val_2_val_2.fq.gz\n",
      "Approx 50% complete for NR019_S7_L001_R2_001_val_2_val_2.fq.gz\n",
      "Approx 55% complete for NR019_S7_L001_R2_001_val_2_val_2.fq.gz\n",
      "Approx 60% complete for NR019_S7_L001_R2_001_val_2_val_2.fq.gz\n",
      "Approx 65% complete for NR019_S7_L001_R2_001_val_2_val_2.fq.gz\n",
      "Approx 70% complete for NR019_S7_L001_R2_001_val_2_val_2.fq.gz\n",
      "Approx 75% complete for NR019_S7_L001_R2_001_val_2_val_2.fq.gz\n",
      "Approx 80% complete for NR019_S7_L001_R2_001_val_2_val_2.fq.gz\n",
      "Approx 85% complete for NR019_S7_L001_R2_001_val_2_val_2.fq.gz\n",
      "Approx 90% complete for NR019_S7_L001_R2_001_val_2_val_2.fq.gz\n",
      "Approx 95% complete for NR019_S7_L001_R2_001_val_2_val_2.fq.gz\n",
      "Started analysis of NR019_S7_L002_R1_001_val_1_val_1.fq.gz\n",
      "Approx 5% complete for NR019_S7_L002_R1_001_val_1_val_1.fq.gz\n",
      "Approx 10% complete for NR019_S7_L002_R1_001_val_1_val_1.fq.gz\n",
      "Approx 15% complete for NR019_S7_L002_R1_001_val_1_val_1.fq.gz\n",
      "Approx 20% complete for NR019_S7_L002_R1_001_val_1_val_1.fq.gz\n",
      "Approx 25% complete for NR019_S7_L002_R1_001_val_1_val_1.fq.gz\n",
      "Approx 30% complete for NR019_S7_L002_R1_001_val_1_val_1.fq.gz\n",
      "Approx 35% complete for NR019_S7_L002_R1_001_val_1_val_1.fq.gz\n",
      "Approx 40% complete for NR019_S7_L002_R1_001_val_1_val_1.fq.gz\n",
      "Approx 45% complete for NR019_S7_L002_R1_001_val_1_val_1.fq.gz\n",
      "Approx 50% complete for NR019_S7_L002_R1_001_val_1_val_1.fq.gz\n",
      "Approx 55% complete for NR019_S7_L002_R1_001_val_1_val_1.fq.gz\n",
      "Approx 60% complete for NR019_S7_L002_R1_001_val_1_val_1.fq.gz\n",
      "Approx 65% complete for NR019_S7_L002_R1_001_val_1_val_1.fq.gz\n",
      "Approx 70% complete for NR019_S7_L002_R1_001_val_1_val_1.fq.gz\n",
      "Approx 75% complete for NR019_S7_L002_R1_001_val_1_val_1.fq.gz\n",
      "Approx 80% complete for NR019_S7_L002_R1_001_val_1_val_1.fq.gz\n",
      "Approx 85% complete for NR019_S7_L002_R1_001_val_1_val_1.fq.gz\n",
      "Approx 90% complete for NR019_S7_L002_R1_001_val_1_val_1.fq.gz\n",
      "Approx 95% complete for NR019_S7_L002_R1_001_val_1_val_1.fq.gz\n",
      "Started analysis of NR019_S7_L002_R2_001_val_2_val_2.fq.gz\n",
      "Approx 5% complete for NR019_S7_L002_R2_001_val_2_val_2.fq.gz\n",
      "Approx 10% complete for NR019_S7_L002_R2_001_val_2_val_2.fq.gz\n",
      "Approx 15% complete for NR019_S7_L002_R2_001_val_2_val_2.fq.gz\n",
      "Approx 20% complete for NR019_S7_L002_R2_001_val_2_val_2.fq.gz\n",
      "Approx 25% complete for NR019_S7_L002_R2_001_val_2_val_2.fq.gz\n",
      "Approx 30% complete for NR019_S7_L002_R2_001_val_2_val_2.fq.gz\n",
      "Approx 35% complete for NR019_S7_L002_R2_001_val_2_val_2.fq.gz\n",
      "Approx 40% complete for NR019_S7_L002_R2_001_val_2_val_2.fq.gz\n",
      "Approx 45% complete for NR019_S7_L002_R2_001_val_2_val_2.fq.gz\n",
      "Approx 50% complete for NR019_S7_L002_R2_001_val_2_val_2.fq.gz\n",
      "Approx 55% complete for NR019_S7_L002_R2_001_val_2_val_2.fq.gz\n",
      "Approx 60% complete for NR019_S7_L002_R2_001_val_2_val_2.fq.gz\n",
      "Approx 65% complete for NR019_S7_L002_R2_001_val_2_val_2.fq.gz\n",
      "Approx 70% complete for NR019_S7_L002_R2_001_val_2_val_2.fq.gz\n",
      "Approx 75% complete for NR019_S7_L002_R2_001_val_2_val_2.fq.gz\n",
      "Approx 80% complete for NR019_S7_L002_R2_001_val_2_val_2.fq.gz\n",
      "Approx 85% complete for NR019_S7_L002_R2_001_val_2_val_2.fq.gz\n",
      "Approx 90% complete for NR019_S7_L002_R2_001_val_2_val_2.fq.gz\n",
      "Approx 95% complete for NR019_S7_L002_R2_001_val_2_val_2.fq.gz\n",
      "Started analysis of NR021_S8_L001_R1_001_val_1_val_1.fq.gz\n",
      "Approx 5% complete for NR021_S8_L001_R1_001_val_1_val_1.fq.gz\n",
      "Approx 85% complete for NR013_AD013_S2_L002_R1_001_val_1_val_1.fq.gz\n",
      "Approx 10% complete for NR021_S8_L001_R1_001_val_1_val_1.fq.gz\n",
      "Approx 15% complete for NR021_S8_L001_R1_001_val_1_val_1.fq.gz\n",
      "Approx 20% complete for NR021_S8_L001_R1_001_val_1_val_1.fq.gz\n",
      "Approx 25% complete for NR021_S8_L001_R1_001_val_1_val_1.fq.gz\n",
      "Approx 30% complete for NR021_S8_L001_R1_001_val_1_val_1.fq.gz\n",
      "Approx 35% complete for NR015_AD015_S6_L002_R2_001_val_2_val_2.fq.gz\n",
      "Approx 35% complete for NR021_S8_L001_R1_001_val_1_val_1.fq.gz\n",
      "Approx 40% complete for NR021_S8_L001_R1_001_val_1_val_1.fq.gz\n",
      "Approx 45% complete for NR021_S8_L001_R1_001_val_1_val_1.fq.gz\n",
      "Approx 50% complete for NR021_S8_L001_R1_001_val_1_val_1.fq.gz\n",
      "Approx 55% complete for NR015_AD015_S6_L001_R1_001_val_1_val_1.fq.gz\n",
      "Approx 55% complete for NR021_S8_L001_R1_001_val_1_val_1.fq.gz\n",
      "Approx 95% complete for NR013_AD013_S2_L001_R2_001_val_2_val_2.fq.gz\n",
      "Approx 60% complete for NR021_S8_L001_R1_001_val_1_val_1.fq.gz\n",
      "Approx 65% complete for NR021_S8_L001_R1_001_val_1_val_1.fq.gz\n",
      "Approx 70% complete for NR021_S8_L001_R1_001_val_1_val_1.fq.gz\n",
      "Approx 75% complete for NR021_S8_L001_R1_001_val_1_val_1.fq.gz\n",
      "Approx 80% complete for NR021_S8_L001_R1_001_val_1_val_1.fq.gz\n",
      "Approx 85% complete for NR021_S8_L001_R1_001_val_1_val_1.fq.gz\n",
      "Approx 90% complete for NR021_S8_L001_R1_001_val_1_val_1.fq.gz\n",
      "Approx 95% complete for NR021_S8_L001_R1_001_val_1_val_1.fq.gz\n",
      "Started analysis of NR021_S8_L001_R2_001_val_2_val_2.fq.gz\n",
      "Approx 50% complete for NR015_AD015_S6_L001_R2_001_val_2_val_2.fq.gz\n",
      "Approx 5% complete for NR021_S8_L001_R2_001_val_2_val_2.fq.gz\n",
      "Approx 40% complete for NR014_AD014_S5_L001_R2_001_val_2_val_2.fq.gz\n",
      "Approx 10% complete for NR021_S8_L001_R2_001_val_2_val_2.fq.gz\n",
      "Approx 15% complete for NR021_S8_L001_R2_001_val_2_val_2.fq.gz\n",
      "Approx 20% complete for NR021_S8_L001_R2_001_val_2_val_2.fq.gz\n",
      "Approx 25% complete for NR021_S8_L001_R2_001_val_2_val_2.fq.gz\n",
      "Approx 30% complete for NR021_S8_L001_R2_001_val_2_val_2.fq.gz\n",
      "Approx 35% complete for NR021_S8_L001_R2_001_val_2_val_2.fq.gz\n",
      "Approx 40% complete for NR021_S8_L001_R2_001_val_2_val_2.fq.gz\n",
      "Approx 45% complete for NR021_S8_L001_R2_001_val_2_val_2.fq.gz\n",
      "Approx 50% complete for NR021_S8_L001_R2_001_val_2_val_2.fq.gz\n",
      "Approx 55% complete for NR021_S8_L001_R2_001_val_2_val_2.fq.gz\n",
      "Approx 60% complete for NR021_S8_L001_R2_001_val_2_val_2.fq.gz\n",
      "Approx 65% complete for NR021_S8_L001_R2_001_val_2_val_2.fq.gz\n",
      "Approx 70% complete for NR021_S8_L001_R2_001_val_2_val_2.fq.gz\n",
      "Approx 75% complete for NR021_S8_L001_R2_001_val_2_val_2.fq.gz\n",
      "Approx 80% complete for NR021_S8_L001_R2_001_val_2_val_2.fq.gz\n",
      "Approx 85% complete for NR021_S8_L001_R2_001_val_2_val_2.fq.gz\n",
      "Approx 90% complete for NR021_S8_L001_R2_001_val_2_val_2.fq.gz\n",
      "Approx 95% complete for NR021_S8_L001_R2_001_val_2_val_2.fq.gz\n",
      "Started analysis of NR021_S8_L002_R1_001_val_1_val_1.fq.gz\n",
      "Approx 5% complete for NR021_S8_L002_R1_001_val_1_val_1.fq.gz\n",
      "Approx 10% complete for NR021_S8_L002_R1_001_val_1_val_1.fq.gz\n",
      "Approx 15% complete for NR021_S8_L002_R1_001_val_1_val_1.fq.gz\n",
      "Approx 20% complete for NR021_S8_L002_R1_001_val_1_val_1.fq.gz\n",
      "Approx 25% complete for NR021_S8_L002_R1_001_val_1_val_1.fq.gz\n",
      "Approx 30% complete for NR021_S8_L002_R1_001_val_1_val_1.fq.gz\n",
      "Approx 35% complete for NR021_S8_L002_R1_001_val_1_val_1.fq.gz\n",
      "Approx 40% complete for NR021_S8_L002_R1_001_val_1_val_1.fq.gz\n",
      "Approx 45% complete for NR021_S8_L002_R1_001_val_1_val_1.fq.gz\n",
      "Approx 50% complete for NR021_S8_L002_R1_001_val_1_val_1.fq.gz\n",
      "Approx 55% complete for NR021_S8_L002_R1_001_val_1_val_1.fq.gz\n",
      "Approx 60% complete for NR021_S8_L002_R1_001_val_1_val_1.fq.gz\n",
      "Approx 65% complete for NR021_S8_L002_R1_001_val_1_val_1.fq.gz\n",
      "Approx 45% complete for NR015_AD015_S6_L002_R1_001_val_1_val_1.fq.gz\n",
      "Approx 70% complete for NR021_S8_L002_R1_001_val_1_val_1.fq.gz\n",
      "Approx 75% complete for NR021_S8_L002_R1_001_val_1_val_1.fq.gz\n",
      "Approx 80% complete for NR021_S8_L002_R1_001_val_1_val_1.fq.gz\n",
      "Approx 85% complete for NR021_S8_L002_R1_001_val_1_val_1.fq.gz\n",
      "Approx 90% complete for NR021_S8_L002_R1_001_val_1_val_1.fq.gz\n",
      "Approx 95% complete for NR021_S8_L002_R1_001_val_1_val_1.fq.gz\n",
      "Started analysis of NR021_S8_L002_R2_001_val_2_val_2.fq.gz\n",
      "Approx 5% complete for NR021_S8_L002_R2_001_val_2_val_2.fq.gz\n",
      "Approx 10% complete for NR021_S8_L002_R2_001_val_2_val_2.fq.gz\n",
      "Approx 15% complete for NR021_S8_L002_R2_001_val_2_val_2.fq.gz\n",
      "Approx 20% complete for NR021_S8_L002_R2_001_val_2_val_2.fq.gz\n",
      "Approx 25% complete for NR021_S8_L002_R2_001_val_2_val_2.fq.gz\n",
      "Approx 30% complete for NR021_S8_L002_R2_001_val_2_val_2.fq.gz\n",
      "Approx 35% complete for NR021_S8_L002_R2_001_val_2_val_2.fq.gz\n",
      "Approx 40% complete for NR021_S8_L002_R2_001_val_2_val_2.fq.gz\n",
      "Approx 60% complete for NR015_AD015_S6_L001_R1_001_val_1_val_1.fq.gz\n",
      "Approx 45% complete for NR021_S8_L002_R2_001_val_2_val_2.fq.gz\n",
      "Approx 50% complete for NR021_S8_L002_R2_001_val_2_val_2.fq.gz\n",
      "Approx 55% complete for NR021_S8_L002_R2_001_val_2_val_2.fq.gz\n",
      "Approx 60% complete for NR021_S8_L002_R2_001_val_2_val_2.fq.gz\n",
      "Approx 65% complete for NR021_S8_L002_R2_001_val_2_val_2.fq.gz\n",
      "Approx 40% complete for NR015_AD015_S6_L002_R2_001_val_2_val_2.fq.gz\n",
      "Approx 70% complete for NR021_S8_L002_R2_001_val_2_val_2.fq.gz\n",
      "Approx 75% complete for NR021_S8_L002_R2_001_val_2_val_2.fq.gz\n",
      "Approx 80% complete for NR021_S8_L002_R2_001_val_2_val_2.fq.gz\n",
      "Approx 85% complete for NR021_S8_L002_R2_001_val_2_val_2.fq.gz\n",
      "Approx 90% complete for NR021_S8_L002_R2_001_val_2_val_2.fq.gz\n",
      "Approx 95% complete for NR021_S8_L002_R2_001_val_2_val_2.fq.gz\n",
      "Approx 55% complete for NR015_AD015_S6_L001_R2_001_val_2_val_2.fq.gz\n",
      "Approx 85% complete for NR013_AD013_S2_L002_R2_001_val_2_val_2.fq.gz\n",
      "Approx 90% complete for NR013_AD013_S2_L002_R1_001_val_1_val_1.fq.gz\n",
      "Approx 50% complete for NR015_AD015_S6_L002_R1_001_val_1_val_1.fq.gz\n",
      "Approx 45% complete for NR014_AD014_S5_L001_R1_001_val_1_val_1.fq.gz\n",
      "Approx 65% complete for NR015_AD015_S6_L001_R1_001_val_1_val_1.fq.gz\n",
      "Approx 45% complete for NR015_AD015_S6_L002_R2_001_val_2_val_2.fq.gz\n",
      "Approx 60% complete for NR015_AD015_S6_L001_R2_001_val_2_val_2.fq.gz\n",
      "Approx 90% complete for NR013_AD013_S2_L002_R2_001_val_2_val_2.fq.gz\n",
      "Approx 95% complete for NR013_AD013_S2_L002_R1_001_val_1_val_1.fq.gz\n",
      "Approx 55% complete for NR015_AD015_S6_L002_R1_001_val_1_val_1.fq.gz\n",
      "Approx 70% complete for NR015_AD015_S6_L001_R1_001_val_1_val_1.fq.gz\n",
      "Approx 65% complete for NR015_AD015_S6_L001_R2_001_val_2_val_2.fq.gz\n",
      "Approx 45% complete for NR014_AD014_S5_L001_R2_001_val_2_val_2.fq.gz\n",
      "Approx 50% complete for NR015_AD015_S6_L002_R2_001_val_2_val_2.fq.gz\n",
      "Approx 75% complete for NR015_AD015_S6_L001_R1_001_val_1_val_1.fq.gz\n",
      "Approx 60% complete for NR015_AD015_S6_L002_R1_001_val_1_val_1.fq.gz\n",
      "Approx 70% complete for NR015_AD015_S6_L001_R2_001_val_2_val_2.fq.gz\n",
      "Approx 95% complete for NR013_AD013_S2_L002_R2_001_val_2_val_2.fq.gz\n",
      "Approx 55% complete for NR015_AD015_S6_L002_R2_001_val_2_val_2.fq.gz\n",
      "Approx 10% complete for NR014_AD014_S5_L002_R1_001_val_1_val_1.fq.gz\n",
      "Approx 50% complete for NR014_AD014_S5_L001_R1_001_val_1_val_1.fq.gz\n",
      "Approx 80% complete for NR015_AD015_S6_L001_R1_001_val_1_val_1.fq.gz\n",
      "Approx 10% complete for NR014_AD014_S5_L002_R2_001_val_2_val_2.fq.gz\n",
      "Approx 65% complete for NR015_AD015_S6_L002_R1_001_val_1_val_1.fq.gz\n",
      "Approx 75% complete for NR015_AD015_S6_L001_R2_001_val_2_val_2.fq.gz\n",
      "Approx 60% complete for NR015_AD015_S6_L002_R2_001_val_2_val_2.fq.gz\n",
      "Approx 85% complete for NR015_AD015_S6_L001_R1_001_val_1_val_1.fq.gz\n",
      "Approx 50% complete for NR014_AD014_S5_L001_R2_001_val_2_val_2.fq.gz\n",
      "Approx 70% complete for NR015_AD015_S6_L002_R1_001_val_1_val_1.fq.gz\n",
      "Approx 80% complete for NR015_AD015_S6_L001_R2_001_val_2_val_2.fq.gz\n",
      "Approx 65% complete for NR015_AD015_S6_L002_R2_001_val_2_val_2.fq.gz\n",
      "Approx 90% complete for NR015_AD015_S6_L001_R1_001_val_1_val_1.fq.gz\n",
      "Approx 85% complete for NR015_AD015_S6_L001_R2_001_val_2_val_2.fq.gz\n",
      "Approx 75% complete for NR015_AD015_S6_L002_R1_001_val_1_val_1.fq.gz\n",
      "Approx 55% complete for NR014_AD014_S5_L001_R1_001_val_1_val_1.fq.gz\n",
      "Approx 95% complete for NR015_AD015_S6_L001_R1_001_val_1_val_1.fq.gz\n",
      "Approx 70% complete for NR015_AD015_S6_L002_R2_001_val_2_val_2.fq.gz\n",
      "Approx 15% complete for NR014_AD014_S5_L002_R1_001_val_1_val_1.fq.gz\n",
      "Approx 90% complete for NR015_AD015_S6_L001_R2_001_val_2_val_2.fq.gz\n",
      "Approx 80% complete for NR015_AD015_S6_L002_R1_001_val_1_val_1.fq.gz\n",
      "Approx 15% complete for NR014_AD014_S5_L002_R2_001_val_2_val_2.fq.gz\n",
      "Approx 75% complete for NR015_AD015_S6_L002_R2_001_val_2_val_2.fq.gz\n",
      "Approx 55% complete for NR014_AD014_S5_L001_R2_001_val_2_val_2.fq.gz\n",
      "Approx 95% complete for NR015_AD015_S6_L001_R2_001_val_2_val_2.fq.gz\n",
      "Approx 85% complete for NR015_AD015_S6_L002_R1_001_val_1_val_1.fq.gz\n",
      "Approx 80% complete for NR015_AD015_S6_L002_R2_001_val_2_val_2.fq.gz\n",
      "Approx 60% complete for NR014_AD014_S5_L001_R1_001_val_1_val_1.fq.gz\n",
      "Approx 90% complete for NR015_AD015_S6_L002_R1_001_val_1_val_1.fq.gz\n",
      "Approx 85% complete for NR015_AD015_S6_L002_R2_001_val_2_val_2.fq.gz\n",
      "Approx 20% complete for NR014_AD014_S5_L002_R1_001_val_1_val_1.fq.gz\n",
      "Approx 95% complete for NR015_AD015_S6_L002_R1_001_val_1_val_1.fq.gz\n",
      "Approx 20% complete for NR014_AD014_S5_L002_R2_001_val_2_val_2.fq.gz\n",
      "Approx 60% complete for NR014_AD014_S5_L001_R2_001_val_2_val_2.fq.gz\n",
      "Approx 90% complete for NR015_AD015_S6_L002_R2_001_val_2_val_2.fq.gz\n",
      "Approx 65% complete for NR014_AD014_S5_L001_R1_001_val_1_val_1.fq.gz\n",
      "Approx 95% complete for NR015_AD015_S6_L002_R2_001_val_2_val_2.fq.gz\n",
      "Approx 25% complete for NR014_AD014_S5_L002_R1_001_val_1_val_1.fq.gz\n",
      "Approx 65% complete for NR014_AD014_S5_L001_R2_001_val_2_val_2.fq.gz\n",
      "Approx 25% complete for NR014_AD014_S5_L002_R2_001_val_2_val_2.fq.gz\n",
      "Approx 70% complete for NR014_AD014_S5_L001_R1_001_val_1_val_1.fq.gz\n",
      "Approx 70% complete for NR014_AD014_S5_L001_R2_001_val_2_val_2.fq.gz\n",
      "Approx 30% complete for NR014_AD014_S5_L002_R1_001_val_1_val_1.fq.gz\n",
      "Approx 30% complete for NR014_AD014_S5_L002_R2_001_val_2_val_2.fq.gz\n",
      "Approx 75% complete for NR014_AD014_S5_L001_R1_001_val_1_val_1.fq.gz\n",
      "Approx 75% complete for NR014_AD014_S5_L001_R2_001_val_2_val_2.fq.gz\n",
      "Approx 35% complete for NR014_AD014_S5_L002_R1_001_val_1_val_1.fq.gz\n",
      "Approx 80% complete for NR014_AD014_S5_L001_R1_001_val_1_val_1.fq.gz\n",
      "Approx 35% complete for NR014_AD014_S5_L002_R2_001_val_2_val_2.fq.gz\n",
      "Approx 80% complete for NR014_AD014_S5_L001_R2_001_val_2_val_2.fq.gz\n",
      "Approx 85% complete for NR014_AD014_S5_L001_R1_001_val_1_val_1.fq.gz\n",
      "Approx 40% complete for NR014_AD014_S5_L002_R1_001_val_1_val_1.fq.gz\n",
      "Approx 40% complete for NR014_AD014_S5_L002_R2_001_val_2_val_2.fq.gz\n",
      "Approx 85% complete for NR014_AD014_S5_L001_R2_001_val_2_val_2.fq.gz\n",
      "Approx 90% complete for NR014_AD014_S5_L001_R1_001_val_1_val_1.fq.gz\n",
      "Approx 45% complete for NR014_AD014_S5_L002_R1_001_val_1_val_1.fq.gz\n",
      "Approx 45% complete for NR014_AD014_S5_L002_R2_001_val_2_val_2.fq.gz\n",
      "Approx 90% complete for NR014_AD014_S5_L001_R2_001_val_2_val_2.fq.gz\n",
      "Approx 95% complete for NR014_AD014_S5_L001_R1_001_val_1_val_1.fq.gz\n",
      "Approx 50% complete for NR014_AD014_S5_L002_R1_001_val_1_val_1.fq.gz\n",
      "Approx 95% complete for NR014_AD014_S5_L001_R2_001_val_2_val_2.fq.gz\n",
      "Approx 50% complete for NR014_AD014_S5_L002_R2_001_val_2_val_2.fq.gz\n",
      "Approx 55% complete for NR014_AD014_S5_L002_R1_001_val_1_val_1.fq.gz\n",
      "Approx 55% complete for NR014_AD014_S5_L002_R2_001_val_2_val_2.fq.gz\n",
      "Approx 60% complete for NR014_AD014_S5_L002_R1_001_val_1_val_1.fq.gz\n",
      "Approx 60% complete for NR014_AD014_S5_L002_R2_001_val_2_val_2.fq.gz\n",
      "Approx 65% complete for NR014_AD014_S5_L002_R1_001_val_1_val_1.fq.gz\n",
      "Approx 65% complete for NR014_AD014_S5_L002_R2_001_val_2_val_2.fq.gz\n",
      "Approx 70% complete for NR014_AD014_S5_L002_R1_001_val_1_val_1.fq.gz\n",
      "Approx 70% complete for NR014_AD014_S5_L002_R2_001_val_2_val_2.fq.gz\n",
      "Approx 75% complete for NR014_AD014_S5_L002_R1_001_val_1_val_1.fq.gz\n",
      "Approx 75% complete for NR014_AD014_S5_L002_R2_001_val_2_val_2.fq.gz\n",
      "Approx 80% complete for NR014_AD014_S5_L002_R1_001_val_1_val_1.fq.gz\n",
      "Approx 80% complete for NR014_AD014_S5_L002_R2_001_val_2_val_2.fq.gz\n",
      "Approx 85% complete for NR014_AD014_S5_L002_R1_001_val_1_val_1.fq.gz\n",
      "Approx 85% complete for NR014_AD014_S5_L002_R2_001_val_2_val_2.fq.gz\n",
      "Approx 90% complete for NR014_AD014_S5_L002_R1_001_val_1_val_1.fq.gz\n",
      "Approx 90% complete for NR014_AD014_S5_L002_R2_001_val_2_val_2.fq.gz\n",
      "Approx 95% complete for NR014_AD014_S5_L002_R1_001_val_1_val_1.fq.gz\n",
      "Approx 95% complete for NR014_AD014_S5_L002_R2_001_val_2_val_2.fq.gz\n",
      "\n",
      "real\t302m11.414s\n",
      "user\t1299m46.092s\n",
      "sys\t25m18.036s\n"
     ]
    }
   ],
   "source": [
    "%%bash\n",
    "time fastqc --threads 12 \\\n",
    "/home/sam/data/geoduck_illumina/trimmed/20180129_trimmed_again/AD002_S9_L001_R1_001_val_1_val_1.fq.gz \\\n",
    "/home/sam/data/geoduck_illumina/trimmed/20180129_trimmed_again/AD002_S9_L001_R2_001_val_2_val_2.fq.gz \\\n",
    "/home/sam/data/geoduck_illumina/trimmed/20180129_trimmed_again/AD002_S9_L002_R1_001_val_1_val_1.fq.gz \\\n",
    "/home/sam/data/geoduck_illumina/trimmed/20180129_trimmed_again/AD002_S9_L002_R2_001_val_2_val_2.fq.gz \\\n",
    "/home/sam/data/geoduck_illumina/trimmed/20180129_trimmed_again/NR005_S4_L001_R1_001_val_1_val_1.fq.gz \\\n",
    "/home/sam/data/geoduck_illumina/trimmed/20180129_trimmed_again/NR005_S4_L001_R2_001_val_2_val_2.fq.gz \\\n",
    "/home/sam/data/geoduck_illumina/trimmed/20180129_trimmed_again/NR005_S4_L002_R1_001_val_1_val_1.fq.gz \\\n",
    "/home/sam/data/geoduck_illumina/trimmed/20180129_trimmed_again/NR005_S4_L002_R2_001_val_2_val_2.fq.gz \\\n",
    "/home/sam/data/geoduck_illumina/trimmed/20180129_trimmed_again/NR006_S3_L001_R1_001_val_1_val_1.fq.gz \\\n",
    "/home/sam/data/geoduck_illumina/trimmed/20180129_trimmed_again/NR006_S3_L001_R2_001_val_2_val_2.fq.gz \\\n",
    "/home/sam/data/geoduck_illumina/trimmed/20180129_trimmed_again/NR006_S3_L002_R1_001_val_1_val_1.fq.gz \\\n",
    "/home/sam/data/geoduck_illumina/trimmed/20180129_trimmed_again/NR006_S3_L002_R2_001_val_2_val_2.fq.gz \\\n",
    "/home/sam/data/geoduck_illumina/trimmed/20180129_trimmed_again/NR012_S1_L001_R1_001_val_1_val_1.fq.gz \\\n",
    "/home/sam/data/geoduck_illumina/trimmed/20180129_trimmed_again/NR012_S1_L001_R2_001_val_2_val_2.fq.gz \\\n",
    "/home/sam/data/geoduck_illumina/trimmed/20180129_trimmed_again/NR012_S1_L002_R1_001_val_1_val_1.fq.gz \\\n",
    "/home/sam/data/geoduck_illumina/trimmed/20180129_trimmed_again/NR012_S1_L002_R2_001_val_2_val_2.fq.gz \\\n",
    "/home/sam/data/geoduck_illumina/trimmed/20180129_trimmed_again/NR013_AD013_S2_L001_R1_001_val_1_val_1.fq.gz \\\n",
    "/home/sam/data/geoduck_illumina/trimmed/20180129_trimmed_again/NR013_AD013_S2_L001_R2_001_val_2_val_2.fq.gz \\\n",
    "/home/sam/data/geoduck_illumina/trimmed/20180129_trimmed_again/NR013_AD013_S2_L002_R1_001_val_1_val_1.fq.gz \\\n",
    "/home/sam/data/geoduck_illumina/trimmed/20180129_trimmed_again/NR013_AD013_S2_L002_R2_001_val_2_val_2.fq.gz \\\n",
    "/home/sam/data/geoduck_illumina/trimmed/20180129_trimmed_again/NR014_AD014_S5_L001_R1_001_val_1_val_1.fq.gz \\\n",
    "/home/sam/data/geoduck_illumina/trimmed/20180129_trimmed_again/NR014_AD014_S5_L001_R2_001_val_2_val_2.fq.gz \\\n",
    "/home/sam/data/geoduck_illumina/trimmed/20180129_trimmed_again/NR014_AD014_S5_L002_R1_001_val_1_val_1.fq.gz \\\n",
    "/home/sam/data/geoduck_illumina/trimmed/20180129_trimmed_again/NR014_AD014_S5_L002_R2_001_val_2_val_2.fq.gz \\\n",
    "/home/sam/data/geoduck_illumina/trimmed/20180129_trimmed_again/NR015_AD015_S6_L001_R1_001_val_1_val_1.fq.gz \\\n",
    "/home/sam/data/geoduck_illumina/trimmed/20180129_trimmed_again/NR015_AD015_S6_L001_R2_001_val_2_val_2.fq.gz \\\n",
    "/home/sam/data/geoduck_illumina/trimmed/20180129_trimmed_again/NR015_AD015_S6_L002_R1_001_val_1_val_1.fq.gz \\\n",
    "/home/sam/data/geoduck_illumina/trimmed/20180129_trimmed_again/NR015_AD015_S6_L002_R2_001_val_2_val_2.fq.gz \\\n",
    "/home/sam/data/geoduck_illumina/trimmed/20180129_trimmed_again/NR019_S7_L001_R1_001_val_1_val_1.fq.gz \\\n",
    "/home/sam/data/geoduck_illumina/trimmed/20180129_trimmed_again/NR019_S7_L001_R2_001_val_2_val_2.fq.gz \\\n",
    "/home/sam/data/geoduck_illumina/trimmed/20180129_trimmed_again/NR019_S7_L002_R1_001_val_1_val_1.fq.gz \\\n",
    "/home/sam/data/geoduck_illumina/trimmed/20180129_trimmed_again/NR019_S7_L002_R2_001_val_2_val_2.fq.gz \\\n",
    "/home/sam/data/geoduck_illumina/trimmed/20180129_trimmed_again/NR021_S8_L001_R1_001_val_1_val_1.fq.gz \\\n",
    "/home/sam/data/geoduck_illumina/trimmed/20180129_trimmed_again/NR021_S8_L001_R2_001_val_2_val_2.fq.gz \\\n",
    "/home/sam/data/geoduck_illumina/trimmed/20180129_trimmed_again/NR021_S8_L002_R1_001_val_1_val_1.fq.gz \\\n",
    "/home/sam/data/geoduck_illumina/trimmed/20180129_trimmed_again/NR021_S8_L002_R2_001_val_2_val_2.fq.gz\n",
    "cat ~/.default-subject.mail | msmtp \"$EMAIL\""
   ]
  },
  {
   "cell_type": "code",
   "execution_count": 52,
   "metadata": {},
   "outputs": [
    {
     "name": "stdout",
     "output_type": "stream",
     "text": [
      "total 0\n"
     ]
    },
    {
     "name": "stderr",
     "output_type": "stream",
     "text": [
      "mv: cannot stat '/home/sam/data/geoduck_illumina/trimmed/trimmed_again/*fastqc*': No such file or directory\n"
     ]
    }
   ],
   "source": [
    "%%bash\n",
    "mkdir /home/sam/data/geoduck_illumina/trimmed/20180205_trimmed_fastqc_02\n",
    "for fastqc in /home/sam/data/geoduck_illumina/trimmed/trimmed_agai/*fastqc*\n",
    "do\n",
    "mv \"$fastqc\" /home/sam/data/geoduck_illumina/trimmed/20180205_trimmed_fastqc_02\n",
    "done\n",
    "ls -ltr /home/sam/data/geoduck_illumina/trimmed/20180205_trimmed_fastqc_02"
   ]
  },
  {
   "cell_type": "code",
   "execution_count": 53,
   "metadata": {},
   "outputs": [
    {
     "name": "stdout",
     "output_type": "stream",
     "text": [
      "total 28176\n",
      "-rw-rw-r-- 1 sam sam 432759 Feb  5 06:47 NR005_S4_L002_R2_001_val_2_val_2_fastqc.zip\n",
      "-rw-rw-r-- 1 sam sam 428273 Feb  5 06:47 NR005_S4_L002_R1_001_val_1_val_1_fastqc.zip\n",
      "-rw-rw-r-- 1 sam sam 473117 Feb  5 06:47 NR005_S4_L001_R2_001_val_2_val_2_fastqc.zip\n",
      "-rw-rw-r-- 1 sam sam 467261 Feb  5 06:47 NR005_S4_L001_R1_001_val_1_val_1_fastqc.zip\n",
      "-rw-rw-r-- 1 sam sam 263344 Feb  5 06:47 NR005_S4_L002_R2_001_val_2_val_2_fastqc.html\n",
      "-rw-rw-r-- 1 sam sam 264908 Feb  5 06:47 NR005_S4_L002_R1_001_val_1_val_1_fastqc.html\n",
      "-rw-rw-r-- 1 sam sam 272103 Feb  5 06:47 NR005_S4_L001_R2_001_val_2_val_2_fastqc.html\n",
      "-rw-rw-r-- 1 sam sam 272816 Feb  5 06:47 NR005_S4_L001_R1_001_val_1_val_1_fastqc.html\n",
      "-rw-rw-r-- 1 sam sam 569441 Feb  5 06:47 NR012_S1_L001_R1_001_val_1_val_1_fastqc.zip\n",
      "-rw-rw-r-- 1 sam sam 249922 Feb  5 06:47 NR012_S1_L001_R1_001_val_1_val_1_fastqc.html\n",
      "-rw-rw-r-- 1 sam sam 574263 Feb  5 06:47 NR012_S1_L001_R2_001_val_2_val_2_fastqc.zip\n",
      "-rw-rw-r-- 1 sam sam 253596 Feb  5 06:47 NR012_S1_L001_R2_001_val_2_val_2_fastqc.html\n",
      "-rw-rw-r-- 1 sam sam 568629 Feb  5 06:47 NR012_S1_L002_R1_001_val_1_val_1_fastqc.zip\n",
      "-rw-rw-r-- 1 sam sam 245651 Feb  5 06:47 NR012_S1_L002_R1_001_val_1_val_1_fastqc.html\n",
      "-rw-rw-r-- 1 sam sam 571514 Feb  5 06:47 NR012_S1_L002_R2_001_val_2_val_2_fastqc.zip\n",
      "-rw-rw-r-- 1 sam sam 245277 Feb  5 06:47 NR012_S1_L002_R2_001_val_2_val_2_fastqc.html\n",
      "-rw-rw-r-- 1 sam sam 598763 Feb  5 06:48 NR006_S3_L002_R2_001_val_2_val_2_fastqc.zip\n",
      "-rw-rw-r-- 1 sam sam 248346 Feb  5 06:48 NR006_S3_L002_R2_001_val_2_val_2_fastqc.html\n",
      "-rw-rw-r-- 1 sam sam 599179 Feb  5 06:48 NR006_S3_L002_R1_001_val_1_val_1_fastqc.zip\n",
      "-rw-rw-r-- 1 sam sam 248355 Feb  5 06:48 NR006_S3_L002_R1_001_val_1_val_1_fastqc.html\n",
      "-rw-rw-r-- 1 sam sam 601016 Feb  5 06:49 NR006_S3_L001_R1_001_val_1_val_1_fastqc.zip\n",
      "-rw-rw-r-- 1 sam sam 252349 Feb  5 06:49 NR006_S3_L001_R1_001_val_1_val_1_fastqc.html\n",
      "-rw-rw-r-- 1 sam sam 597042 Feb  5 06:49 NR006_S3_L001_R2_001_val_2_val_2_fastqc.zip\n",
      "-rw-rw-r-- 1 sam sam 251931 Feb  5 06:49 NR006_S3_L001_R2_001_val_2_val_2_fastqc.html\n",
      "-rw-rw-r-- 1 sam sam 565380 Feb  5 07:56 AD002_S9_L001_R1_001_val_1_val_1_fastqc.zip\n",
      "-rw-rw-r-- 1 sam sam 234393 Feb  5 07:56 AD002_S9_L001_R1_001_val_1_val_1_fastqc.html\n",
      "-rw-rw-r-- 1 sam sam 567265 Feb  5 08:00 AD002_S9_L001_R2_001_val_2_val_2_fastqc.zip\n",
      "-rw-rw-r-- 1 sam sam 236565 Feb  5 08:00 AD002_S9_L001_R2_001_val_2_val_2_fastqc.html\n",
      "-rw-rw-r-- 1 sam sam 565041 Feb  5 08:09 AD002_S9_L002_R1_001_val_1_val_1_fastqc.zip\n",
      "-rw-rw-r-- 1 sam sam 230950 Feb  5 08:09 AD002_S9_L002_R1_001_val_1_val_1_fastqc.html\n",
      "-rw-rw-r-- 1 sam sam 569229 Feb  5 08:13 AD002_S9_L002_R2_001_val_2_val_2_fastqc.zip\n",
      "-rw-rw-r-- 1 sam sam 234354 Feb  5 08:13 AD002_S9_L002_R2_001_val_2_val_2_fastqc.html\n",
      "-rw-rw-r-- 1 sam sam 565214 Feb  5 08:56 NR013_AD013_S2_L001_R1_001_val_1_val_1_fastqc.zip\n",
      "-rw-rw-r-- 1 sam sam 234139 Feb  5 08:56 NR013_AD013_S2_L001_R1_001_val_1_val_1_fastqc.html\n",
      "-rw-rw-r-- 1 sam sam 491407 Feb  5 08:56 NR019_S7_L001_R1_001_val_1_val_1_fastqc.zip\n",
      "-rw-rw-r-- 1 sam sam 261531 Feb  5 08:56 NR019_S7_L001_R1_001_val_1_val_1_fastqc.html\n",
      "-rw-rw-r-- 1 sam sam 490020 Feb  5 08:56 NR019_S7_L001_R2_001_val_2_val_2_fastqc.zip\n",
      "-rw-rw-r-- 1 sam sam 263448 Feb  5 08:56 NR019_S7_L001_R2_001_val_2_val_2_fastqc.html\n",
      "-rw-rw-r-- 1 sam sam 464351 Feb  5 08:56 NR019_S7_L002_R1_001_val_1_val_1_fastqc.zip\n",
      "-rw-rw-r-- 1 sam sam 268708 Feb  5 08:56 NR019_S7_L002_R1_001_val_1_val_1_fastqc.html\n",
      "-rw-rw-r-- 1 sam sam 460821 Feb  5 08:56 NR019_S7_L002_R2_001_val_2_val_2_fastqc.zip\n",
      "-rw-rw-r-- 1 sam sam 267356 Feb  5 08:56 NR019_S7_L002_R2_001_val_2_val_2_fastqc.html\n",
      "-rw-rw-r-- 1 sam sam 596763 Feb  5 08:58 NR021_S8_L001_R1_001_val_1_val_1_fastqc.zip\n",
      "-rw-rw-r-- 1 sam sam 251757 Feb  5 08:58 NR021_S8_L001_R1_001_val_1_val_1_fastqc.html\n",
      "-rw-rw-r-- 1 sam sam 596186 Feb  5 09:00 NR021_S8_L001_R2_001_val_2_val_2_fastqc.zip\n",
      "-rw-rw-r-- 1 sam sam 253280 Feb  5 09:00 NR021_S8_L001_R2_001_val_2_val_2_fastqc.html\n",
      "-rw-rw-r-- 1 sam sam 590527 Feb  5 09:02 NR021_S8_L002_R1_001_val_1_val_1_fastqc.zip\n",
      "-rw-rw-r-- 1 sam sam 244933 Feb  5 09:02 NR021_S8_L002_R1_001_val_1_val_1_fastqc.html\n",
      "-rw-rw-r-- 1 sam sam 598134 Feb  5 09:04 NR021_S8_L002_R2_001_val_2_val_2_fastqc.zip\n",
      "-rw-rw-r-- 1 sam sam 250190 Feb  5 09:04 NR021_S8_L002_R2_001_val_2_val_2_fastqc.html\n",
      "-rw-rw-r-- 1 sam sam 568372 Feb  5 09:05 NR013_AD013_S2_L001_R2_001_val_2_val_2_fastqc.zip\n",
      "-rw-rw-r-- 1 sam sam 236771 Feb  5 09:05 NR013_AD013_S2_L001_R2_001_val_2_val_2_fastqc.html\n",
      "-rw-rw-r-- 1 sam sam 570468 Feb  5 09:15 NR013_AD013_S2_L002_R1_001_val_1_val_1_fastqc.zip\n",
      "-rw-rw-r-- 1 sam sam 234176 Feb  5 09:15 NR013_AD013_S2_L002_R1_001_val_1_val_1_fastqc.html\n",
      "-rw-rw-r-- 1 sam sam 568442 Feb  5 09:20 NR013_AD013_S2_L002_R2_001_val_2_val_2_fastqc.zip\n",
      "-rw-rw-r-- 1 sam sam 233208 Feb  5 09:20 NR013_AD013_S2_L002_R2_001_val_2_val_2_fastqc.html\n",
      "-rw-rw-r-- 1 sam sam 566327 Feb  5 09:28 NR015_AD015_S6_L001_R1_001_val_1_val_1_fastqc.zip\n",
      "-rw-rw-r-- 1 sam sam 234855 Feb  5 09:28 NR015_AD015_S6_L001_R1_001_val_1_val_1_fastqc.html\n",
      "-rw-rw-r-- 1 sam sam 567567 Feb  5 09:32 NR015_AD015_S6_L001_R2_001_val_2_val_2_fastqc.zip\n",
      "-rw-rw-r-- 1 sam sam 236195 Feb  5 09:32 NR015_AD015_S6_L001_R2_001_val_2_val_2_fastqc.html\n",
      "-rw-rw-r-- 1 sam sam 563250 Feb  5 09:39 NR015_AD015_S6_L002_R1_001_val_1_val_1_fastqc.zip\n",
      "-rw-rw-r-- 1 sam sam 229424 Feb  5 09:39 NR015_AD015_S6_L002_R1_001_val_1_val_1_fastqc.html\n",
      "-rw-rw-r-- 1 sam sam 568920 Feb  5 09:43 NR015_AD015_S6_L002_R2_001_val_2_val_2_fastqc.zip\n",
      "-rw-rw-r-- 1 sam sam 234268 Feb  5 09:43 NR015_AD015_S6_L002_R2_001_val_2_val_2_fastqc.html\n",
      "-rw-rw-r-- 1 sam sam 563439 Feb  5 10:28 NR014_AD014_S5_L001_R1_001_val_1_val_1_fastqc.zip\n",
      "-rw-rw-r-- 1 sam sam 232903 Feb  5 10:28 NR014_AD014_S5_L001_R1_001_val_1_val_1_fastqc.html\n",
      "-rw-rw-r-- 1 sam sam 567511 Feb  5 10:33 NR014_AD014_S5_L001_R2_001_val_2_val_2_fastqc.zip\n",
      "-rw-rw-r-- 1 sam sam 236851 Feb  5 10:33 NR014_AD014_S5_L001_R2_001_val_2_val_2_fastqc.html\n",
      "-rw-rw-r-- 1 sam sam 566778 Feb  5 11:45 NR014_AD014_S5_L002_R1_001_val_1_val_1_fastqc.zip\n",
      "-rw-rw-r-- 1 sam sam 231140 Feb  5 11:45 NR014_AD014_S5_L002_R1_001_val_1_val_1_fastqc.html\n",
      "-rw-rw-r-- 1 sam sam 566274 Feb  5 11:49 NR014_AD014_S5_L002_R2_001_val_2_val_2_fastqc.zip\n",
      "-rw-rw-r-- 1 sam sam 232224 Feb  5 11:49 NR014_AD014_S5_L002_R2_001_val_2_val_2_fastqc.html\n"
     ]
    },
    {
     "name": "stderr",
     "output_type": "stream",
     "text": [
      "mkdir: cannot create directory /home/sam/data/geoduck_illumina/trimmed/20180205_trimmed_fastqc_02: File exists\n"
     ]
    }
   ],
   "source": [
    "%%bash\n",
    "mkdir /home/sam/data/geoduck_illumina/trimmed/20180205_trimmed_fastqc_02\n",
    "for fastqc in /home/sam/data/geoduck_illumina/trimmed/20180129_trimmed_again/*fastqc*\n",
    "do\n",
    "mv \"$fastqc\" /home/sam/data/geoduck_illumina/trimmed/20180205_trimmed_fastqc_02\n",
    "done\n",
    "ls -ltr /home/sam/data/geoduck_illumina/trimmed/20180205_trimmed_fastqc_02"
   ]
  },
  {
   "cell_type": "code",
   "execution_count": 54,
   "metadata": {},
   "outputs": [
    {
     "name": "stdout",
     "output_type": "stream",
     "text": [
      "Searching 72 files..\n"
     ]
    },
    {
     "name": "stderr",
     "output_type": "stream",
     "text": [
      "[INFO   ]         multiqc : This is MultiQC v1.5.dev0\n",
      "[INFO   ]         multiqc : Template    : default\n",
      "[INFO   ]         multiqc : Searching '/home/sam/data/geoduck_illumina/trimmed/20180205_trimmed_fastqc_02/'\n",
      "[INFO   ]          fastqc : Found 36 reports\n",
      "[INFO   ]         multiqc : Compressing plot data\n",
      "[INFO   ]         multiqc : Report      : multiqc_report.html\n",
      "[INFO   ]         multiqc : Data        : multiqc_data\n",
      "[INFO   ]         multiqc : MultiQC complete\n"
     ]
    }
   ],
   "source": [
    "%%bash\n",
    "cd /home/sam/data/geoduck_illumina/trimmed/20180205_trimmed_fastqc_02/\n",
    "/home/shared/anaconda3/bin/multiqc /home/sam/data/geoduck_illumina/trimmed/20180205_trimmed_fastqc_02/"
   ]
  },
  {
   "cell_type": "code",
   "execution_count": 55,
   "metadata": {},
   "outputs": [
    {
     "name": "stdout",
     "output_type": "stream",
     "text": [
      "total 1444\n",
      "-rw-rw-r-- 1 sam sam 1474437 Feb  5 13:01 multiqc_report.html\n",
      "drwxrwxr-x 2 sam sam    4096 Feb  5 13:01 multiqc_data\n"
     ]
    }
   ],
   "source": [
    "%%bash\n",
    "mkdir /home/sam/data/geoduck_illumina/trimmed/20180205_trimmed_multiqc_fastqc_02\n",
    "mv /home/sam/data/geoduck_illumina/trimmed/20180205_trimmed_fastqc_02/multiqc* /home/sam/data/geoduck_illumina/trimmed/20180205_trimmed_multiqc_fastqc_02\n",
    "ls -ltr /home/sam/data/geoduck_illumina/trimmed/20180205_trimmed_multiqc_fastqc_02"
   ]
  },
  {
   "cell_type": "markdown",
   "metadata": {
    "collapsed": true
   },
   "source": [
    "Updating this to show new file structure/locations on Owl."
   ]
  },
  {
   "cell_type": "code",
   "execution_count": 1,
   "metadata": {},
   "outputs": [
    {
     "ename": "SyntaxError",
     "evalue": "invalid token (<ipython-input-1-40adfff0d201>, line 2)",
     "output_type": "error",
     "traceback": [
      "\u001b[0;36m  File \u001b[0;32m\"<ipython-input-1-40adfff0d201>\"\u001b[0;36m, line \u001b[0;32m2\u001b[0m\n\u001b[0;31m    ls /home/sam/owl/Athaliana/20180125_geoduck_novaseq/\u001b[0m\n\u001b[0m                                       ^\u001b[0m\n\u001b[0;31mSyntaxError\u001b[0m\u001b[0;31m:\u001b[0m invalid token\n"
     ]
    }
   ],
   "source": [
    "%bash\n",
    "ls /home/sam/owl/Athaliana/20180125_geoduck_novaseq/"
   ]
  },
  {
   "cell_type": "code",
   "execution_count": 2,
   "metadata": {},
   "outputs": [
    {
     "name": "stdout",
     "output_type": "stream",
     "text": [
      "\u001b[0m\u001b[01;34m20180125_trim_galore_reports\u001b[0m/\r\n",
      "\u001b[01;34m20180129_multiqc_trim01\u001b[0m/\r\n",
      "\u001b[01;34m20180129_trimmed_fastqc_01\u001b[0m/\r\n",
      "\u001b[01;34m20180129_trimmed_multiqc_fastqc_01\u001b[0m/\r\n",
      "\u001b[34;42m20180205_trim_galore_reports\u001b[0m/\r\n",
      "\u001b[01;34m20180205_trimmed_fastqc_02\u001b[0m/\r\n",
      "\u001b[01;34m20180205_trimmed_multiqc_fastqc_02\u001b[0m/\r\n",
      "\u001b[34;42m20180206_kmergenie\u001b[0m/\r\n",
      "\u001b[01;32mAD002_S9_L001_R1_001_val_1_val_1.fq.gz\u001b[0m*\r\n",
      "\u001b[01;32mAD002_S9_L001_R2_001_val_2_val_2.fq.gz\u001b[0m*\r\n",
      "\u001b[01;32mAD002_S9_L002_R1_001_val_1_val_1.fq.gz\u001b[0m*\r\n",
      "\u001b[01;32mAD002_S9_L002_R2_001_val_2_val_2.fq.gz\u001b[0m*\r\n",
      "\u001b[01;32mNR005_S4_L001_R1_001_val_1_val_1.fq.gz\u001b[0m*\r\n",
      "\u001b[01;32mNR005_S4_L001_R2_001_val_2_val_2.fq.gz\u001b[0m*\r\n",
      "\u001b[01;32mNR005_S4_L002_R1_001_val_1_val_1.fq.gz\u001b[0m*\r\n",
      "\u001b[01;32mNR005_S4_L002_R2_001_val_2_val_2.fq.gz\u001b[0m*\r\n",
      "\u001b[01;32mNR006_S3_L001_R1_001_val_1_val_1.fq.gz\u001b[0m*\r\n",
      "\u001b[01;32mNR006_S3_L001_R2_001_val_2_val_2.fq.gz\u001b[0m*\r\n",
      "\u001b[01;32mNR006_S3_L002_R1_001_val_1_val_1.fq.gz\u001b[0m*\r\n",
      "\u001b[01;32mNR006_S3_L002_R2_001_val_2_val_2.fq.gz\u001b[0m*\r\n",
      "\u001b[01;32mNR012_S1_L001_R1_001_val_1_val_1.fq.gz\u001b[0m*\r\n",
      "\u001b[01;32mNR012_S1_L001_R2_001_val_2_val_2.fq.gz\u001b[0m*\r\n",
      "\u001b[01;32mNR012_S1_L002_R1_001_val_1_val_1.fq.gz\u001b[0m*\r\n",
      "\u001b[01;32mNR012_S1_L002_R2_001_val_2_val_2.fq.gz\u001b[0m*\r\n",
      "\u001b[01;32mNR013_AD013_S2_L001_R1_001_val_1_val_1.fq.gz\u001b[0m*\r\n",
      "\u001b[01;32mNR013_AD013_S2_L001_R2_001_val_2_val_2.fq.gz\u001b[0m*\r\n",
      "\u001b[01;32mNR013_AD013_S2_L002_R1_001_val_1_val_1.fq.gz\u001b[0m*\r\n",
      "\u001b[01;32mNR013_AD013_S2_L002_R2_001_val_2_val_2.fq.gz\u001b[0m*\r\n"
     ]
    }
   ],
   "source": [
    "ls /home/sam/owl/Athaliana/20180125_geoduck_novaseq/"
   ]
  },
  {
   "cell_type": "code",
   "execution_count": null,
   "metadata": {
    "collapsed": true
   },
   "outputs": [],
   "source": []
  }
 ],
 "metadata": {
  "kernelspec": {
   "display_name": "Python 3",
   "language": "python",
   "name": "python3"
  },
  "language_info": {
   "codemirror_mode": {
    "name": "ipython",
    "version": 3
   },
   "file_extension": ".py",
   "mimetype": "text/x-python",
   "name": "python",
   "nbconvert_exporter": "python",
   "pygments_lexer": "ipython3",
   "version": "3.6.3"
  }
 },
 "nbformat": 4,
 "nbformat_minor": 2
}
